{"nbformat":4,"nbformat_minor":0,"metadata":{"colab":{"name":"sentence-transformers.ipynb","provenance":[],"collapsed_sections":[],"authorship_tag":"ABX9TyODBuGmGsdWNzIJSdVAKNIN"},"kernelspec":{"name":"python3","display_name":"Python 3"},"accelerator":"GPU"},"cells":[{"cell_type":"code","metadata":{"id":"5vUgopsW1ReA","colab_type":"code","outputId":"2ca5f276-a475-48a7-ab5d-9f497669d8ff","executionInfo":{"status":"ok","timestamp":1588766058788,"user_tz":-120,"elapsed":3708,"user":{"displayName":"Marko Cvitković","photoUrl":"","userId":"14016951488981011334"}},"colab":{"base_uri":"https://localhost:8080/","height":34}},"source":["import torch\n","torch.cuda.get_device_name(0)\n","# 'Tesla P100-PCIE-16GB' worked great for me"],"execution_count":0,"outputs":[{"output_type":"execute_result","data":{"text/plain":["'Tesla P100-PCIE-16GB'"]},"metadata":{"tags":[]},"execution_count":1}]},{"cell_type":"code","metadata":{"id":"y2F97CrP4WSe","colab_type":"code","colab":{}},"source":["\"\"\"\n","INSTRUCTIONS:\n","- upload data.py, util.py, preprocessing.py (optional) in ./content and edit them in a way relative paths work(when importing)\n","- i managed to build an engine by creating corpus(DataFrame) of abstracts back in the python console localy, then serialize it \n","  with pickle, upload it here in /content and then use it here along with paper_ids in query_engine.fit method\n","- the rest is basically executing the code given here with possible minor tweaks\n","\"\"\""],"execution_count":0,"outputs":[]},{"cell_type":"code","metadata":{"id":"kXf3BQzry9hH","colab_type":"code","outputId":"9ef4268d-c963-492c-d16b-b4d2a81e60e2","executionInfo":{"status":"ok","timestamp":1588766068585,"user_tz":-120,"elapsed":8123,"user":{"displayName":"Marko Cvitković","photoUrl":"","userId":"14016951488981011334"}},"colab":{"base_uri":"https://localhost:8080/","height":898}},"source":["!pip install sentence-transformers"],"execution_count":0,"outputs":[{"output_type":"stream","text":["Collecting sentence-transformers\n","\u001b[?25l  Downloading https://files.pythonhosted.org/packages/b9/46/b7d6c37d92d1bd65319220beabe4df845434930e3f30e42d3cfaecb74dc4/sentence-transformers-0.2.6.1.tar.gz (55kB)\n","\u001b[K     |████████████████████████████████| 61kB 3.2MB/s \n","\u001b[?25hCollecting transformers>=2.8.0\n","\u001b[?25l  Downloading https://files.pythonhosted.org/packages/a3/78/92cedda05552398352ed9784908b834ee32a0bd071a9b32de287327370b7/transformers-2.8.0-py3-none-any.whl (563kB)\n","\u001b[K     |████████████████████████████████| 573kB 13.9MB/s \n","\u001b[?25hRequirement already satisfied: tqdm in /usr/local/lib/python3.6/dist-packages (from sentence-transformers) (4.38.0)\n","Requirement already satisfied: torch>=1.0.1 in /usr/local/lib/python3.6/dist-packages (from sentence-transformers) (1.5.0+cu101)\n","Requirement already satisfied: numpy in /usr/local/lib/python3.6/dist-packages (from sentence-transformers) (1.18.3)\n","Requirement already satisfied: scikit-learn in /usr/local/lib/python3.6/dist-packages (from sentence-transformers) (0.22.2.post1)\n","Requirement already satisfied: scipy in /usr/local/lib/python3.6/dist-packages (from sentence-transformers) (1.4.1)\n","Requirement already satisfied: nltk in /usr/local/lib/python3.6/dist-packages (from sentence-transformers) (3.2.5)\n","Requirement already satisfied: boto3 in /usr/local/lib/python3.6/dist-packages (from transformers>=2.8.0->sentence-transformers) (1.13.1)\n","Requirement already satisfied: requests in /usr/local/lib/python3.6/dist-packages (from transformers>=2.8.0->sentence-transformers) (2.23.0)\n","Collecting tokenizers==0.5.2\n","\u001b[?25l  Downloading https://files.pythonhosted.org/packages/d1/3f/73c881ea4723e43c1e9acf317cf407fab3a278daab3a69c98dcac511c04f/tokenizers-0.5.2-cp36-cp36m-manylinux1_x86_64.whl (3.7MB)\n","\u001b[K     |████████████████████████████████| 3.7MB 57.6MB/s \n","\u001b[?25hCollecting sacremoses\n","\u001b[?25l  Downloading https://files.pythonhosted.org/packages/7d/34/09d19aff26edcc8eb2a01bed8e98f13a1537005d31e95233fd48216eed10/sacremoses-0.0.43.tar.gz (883kB)\n","\u001b[K     |████████████████████████████████| 890kB 57.4MB/s \n","\u001b[?25hRequirement already satisfied: dataclasses; python_version < \"3.7\" in /usr/local/lib/python3.6/dist-packages (from transformers>=2.8.0->sentence-transformers) (0.7)\n","Requirement already satisfied: regex!=2019.12.17 in /usr/local/lib/python3.6/dist-packages (from transformers>=2.8.0->sentence-transformers) (2019.12.20)\n","Requirement already satisfied: filelock in /usr/local/lib/python3.6/dist-packages (from transformers>=2.8.0->sentence-transformers) (3.0.12)\n","Collecting sentencepiece\n","\u001b[?25l  Downloading https://files.pythonhosted.org/packages/98/2c/8df20f3ac6c22ac224fff307ebc102818206c53fc454ecd37d8ac2060df5/sentencepiece-0.1.86-cp36-cp36m-manylinux1_x86_64.whl (1.0MB)\n","\u001b[K     |████████████████████████████████| 1.0MB 46.2MB/s \n","\u001b[?25hRequirement already satisfied: future in /usr/local/lib/python3.6/dist-packages (from torch>=1.0.1->sentence-transformers) (0.16.0)\n","Requirement already satisfied: joblib>=0.11 in /usr/local/lib/python3.6/dist-packages (from scikit-learn->sentence-transformers) (0.14.1)\n","Requirement already satisfied: six in /usr/local/lib/python3.6/dist-packages (from nltk->sentence-transformers) (1.12.0)\n","Requirement already satisfied: s3transfer<0.4.0,>=0.3.0 in /usr/local/lib/python3.6/dist-packages (from boto3->transformers>=2.8.0->sentence-transformers) (0.3.3)\n","Requirement already satisfied: botocore<1.17.0,>=1.16.1 in /usr/local/lib/python3.6/dist-packages (from boto3->transformers>=2.8.0->sentence-transformers) (1.16.1)\n","Requirement already satisfied: jmespath<1.0.0,>=0.7.1 in /usr/local/lib/python3.6/dist-packages (from boto3->transformers>=2.8.0->sentence-transformers) (0.9.5)\n","Requirement already satisfied: idna<3,>=2.5 in /usr/local/lib/python3.6/dist-packages (from requests->transformers>=2.8.0->sentence-transformers) (2.9)\n","Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.6/dist-packages (from requests->transformers>=2.8.0->sentence-transformers) (2020.4.5.1)\n","Requirement already satisfied: chardet<4,>=3.0.2 in /usr/local/lib/python3.6/dist-packages (from requests->transformers>=2.8.0->sentence-transformers) (3.0.4)\n","Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /usr/local/lib/python3.6/dist-packages (from requests->transformers>=2.8.0->sentence-transformers) (1.24.3)\n","Requirement already satisfied: click in /usr/local/lib/python3.6/dist-packages (from sacremoses->transformers>=2.8.0->sentence-transformers) (7.1.2)\n","Requirement already satisfied: python-dateutil<3.0.0,>=2.1 in /usr/local/lib/python3.6/dist-packages (from botocore<1.17.0,>=1.16.1->boto3->transformers>=2.8.0->sentence-transformers) (2.8.1)\n","Requirement already satisfied: docutils<0.16,>=0.10 in /usr/local/lib/python3.6/dist-packages (from botocore<1.17.0,>=1.16.1->boto3->transformers>=2.8.0->sentence-transformers) (0.15.2)\n","Building wheels for collected packages: sentence-transformers, sacremoses\n","  Building wheel for sentence-transformers (setup.py) ... \u001b[?25l\u001b[?25hdone\n","  Created wheel for sentence-transformers: filename=sentence_transformers-0.2.6.1-cp36-none-any.whl size=74031 sha256=202b31ce2c0702d11be4ac0ad402f02184312a8ab16833bc0315b28b7c346ea2\n","  Stored in directory: /root/.cache/pip/wheels/d7/fa/17/2b081a8cd8b0a86753fb0e9826b3cc19f0207062c0b2da7008\n","  Building wheel for sacremoses (setup.py) ... \u001b[?25l\u001b[?25hdone\n","  Created wheel for sacremoses: filename=sacremoses-0.0.43-cp36-none-any.whl size=893260 sha256=65e5a5a8e72511c7b7a8d3e14e546079b267272ad7b41d729c6f4d7f7ebaf5c5\n","  Stored in directory: /root/.cache/pip/wheels/29/3c/fd/7ce5c3f0666dab31a50123635e6fb5e19ceb42ce38d4e58f45\n","Successfully built sentence-transformers sacremoses\n","Installing collected packages: tokenizers, sacremoses, sentencepiece, transformers, sentence-transformers\n","Successfully installed sacremoses-0.0.43 sentence-transformers-0.2.6.1 sentencepiece-0.1.86 tokenizers-0.5.2 transformers-2.8.0\n"],"name":"stdout"}]},{"cell_type":"code","metadata":{"id":"93Xeht7k1mEQ","colab_type":"code","colab":{}},"source":["from sentence_transformers import SentenceTransformer\n","from sentence_transformers import models\n","from nltk import sent_tokenize\n","import numpy as np"],"execution_count":0,"outputs":[]},{"cell_type":"code","metadata":{"id":"Sv6nma8p1p6h","colab_type":"code","colab":{}},"source":["def normalize(embeddings):\n","        \"\"\"\n","        Normalizes embeddings using L2 normalization.\n","        Args:\n","            embeddings: input embeddings matrix\n","        Returns:\n","            normalized embeddings\n","        \"\"\"\n","        # Calculation is different for matrices vs vectors\n","        if len(embeddings.shape) > 1:\n","            return embeddings / np.linalg.norm(embeddings, axis=1).reshape(-1, 1)\n","\n","        else:\n","            return embeddings / np.linalg.norm(embeddings)"],"execution_count":0,"outputs":[]},{"cell_type":"code","metadata":{"id":"Co0i4sZu14CY","colab_type":"code","colab":{}},"source":["def BERT_sentence_embeddings(data, query=False):\n","    \n","    \"\"\"\n","    Input:\n","        corpus: DataFrame containing information about paragraphs : paper_id, section, text\n","        query: if True, import is one sentence - a query\n","    Returns:\n","        corpus embeddings: numpy array containing paragraph embeddings for each text paragraph in input\n","        which is obtained by averaging over sentence embeddings(try #1 - until a better idea arrives (probably not so great))\n","        -dimensions: n x 768 where n represents number of input paragraphs\n","    \n","    References\n","    ----------\n","    {\n","    reimers-2019-sentence-bert,\n","    title = \"Sentence-BERT: Sentence Embeddings using Siamese BERT-Networks\",\n","    author = \"Reimers, Nils and Gurevych, Iryna\",\n","    booktitle = \"Proceedings of the 2019 Conference on Empirical Methods in Natural Language Processing\",\n","    month = \"11\",\n","    year = \"2019\",\n","    publisher = \"Association for Computational Linguistics\",\n","    url = \"http://arxiv.org/abs/1908.10084\",\n","    }\n","\n","    \"\"\"\n","    \n","    #pre-trained model on semantic text similarity task\n","    model = SentenceTransformer('bert-base-nli-stsb-mean-tokens') \n","    \n","    if query:\n","        return normalize(np.array(model.encode([data])).reshape(1,768))\n","    \n","    else:\n","        text_paragraphs = [paragraph for paragraph in list(data['text'])]\n","        n=len(text_paragraphs)\n","        \n","        corpus_embeddings=[]\n","        for paragraph in text_paragraphs:\n","            sentences = sent_tokenize(paragraph)\n","            sent_embeddings = normalize(np.array(model.encode(sentences)).reshape(-1,768))#shape = no_of_sents_in_paragraph X 768\n","            corpus_embeddings.append(np.mean(sent_embeddings,axis=0).reshape(1,768)) \n","        \n","        return normalize(np.array(corpus_embeddings).reshape(n,768))"],"execution_count":0,"outputs":[]},{"cell_type":"code","metadata":{"id":"rknHhIIo1-Nf","colab_type":"code","outputId":"1f8bb13b-d53b-4a37-a656-b908b296fe8b","executionInfo":{"status":"ok","timestamp":1588766477497,"user_tz":-120,"elapsed":36966,"user":{"displayName":"Marko Cvitković","photoUrl":"","userId":"14016951488981011334"}},"colab":{"base_uri":"https://localhost:8080/","height":34}},"source":["emb1 = BERT_sentence_embeddings('What do we know about COVID-19 risk factors?', query=True)\n","emb2 = BERT_sentence_embeddings('Do co-existing respiratory/viral infections make the virus more transmissible or virulent?', query=True)\n","emb3 = BERT_sentence_embeddings('The text is small and will load quickly and easily fit into memory.', query=True)"],"execution_count":0,"outputs":[{"output_type":"stream","text":["100%|██████████| 405M/405M [00:24<00:00, 16.3MB/s]\n"],"name":"stderr"}]},{"cell_type":"code","metadata":{"id":"dB1slJUt2rMs","colab_type":"code","outputId":"becc3184-f905-40d1-db3f-693aff78d1f5","executionInfo":{"status":"ok","timestamp":1588766592814,"user_tz":-120,"elapsed":622,"user":{"displayName":"Marko Cvitković","photoUrl":"","userId":"14016951488981011334"}},"colab":{"base_uri":"https://localhost:8080/","height":34}},"source":["#testing sentence embeddings\n","np.dot(emb1,emb2.T)"],"execution_count":0,"outputs":[{"output_type":"execute_result","data":{"text/plain":["array([[0.32969183]], dtype=float32)"]},"metadata":{"tags":[]},"execution_count":12}]},{"cell_type":"code","metadata":{"id":"lSfKg-v72uuZ","colab_type":"code","outputId":"3cc1a597-70b7-4fa7-a972-5859d9cc4efd","executionInfo":{"status":"ok","timestamp":1588766594128,"user_tz":-120,"elapsed":584,"user":{"displayName":"Marko Cvitković","photoUrl":"","userId":"14016951488981011334"}},"colab":{"base_uri":"https://localhost:8080/","height":34}},"source":["np.dot(emb1,emb3.T)"],"execution_count":0,"outputs":[{"output_type":"execute_result","data":{"text/plain":["array([[-0.01860096]], dtype=float32)"]},"metadata":{"tags":[]},"execution_count":13}]},{"cell_type":"code","metadata":{"id":"BWRDi25w2xqc","colab_type":"code","outputId":"d0b5a383-7ee0-4475-d674-1b145d4dc792","executionInfo":{"status":"ok","timestamp":1588766595591,"user_tz":-120,"elapsed":600,"user":{"displayName":"Marko Cvitković","photoUrl":"","userId":"14016951488981011334"}},"colab":{"base_uri":"https://localhost:8080/","height":34}},"source":["np.dot(emb3,emb2.T)"],"execution_count":0,"outputs":[{"output_type":"execute_result","data":{"text/plain":["array([[0.11308034]], dtype=float32)"]},"metadata":{"tags":[]},"execution_count":14}]},{"cell_type":"code","metadata":{"id":"xwcb0D2X5kJ_","colab_type":"code","outputId":"1eb8e7da-335b-4e66-dac6-b53b44b1a084","executionInfo":{"status":"ok","timestamp":1588767157358,"user_tz":-120,"elapsed":617,"user":{"displayName":"Marko Cvitković","photoUrl":"","userId":"14016951488981011334"}},"colab":{"base_uri":"https://localhost:8080/","height":70}},"source":["import nltk\n","nltk.download('stopwords')"],"execution_count":0,"outputs":[{"output_type":"stream","text":["[nltk_data] Downloading package stopwords to /root/nltk_data...\n","[nltk_data]   Unzipping corpora/stopwords.zip.\n"],"name":"stdout"},{"output_type":"execute_result","data":{"text/plain":["True"]},"metadata":{"tags":[]},"execution_count":18}]},{"cell_type":"code","metadata":{"id":"nZEx_6Kf3ld5","colab_type":"code","colab":{}},"source":["from data import CovidDataLoader\n","from nltk import sent_tokenize\n","\n","import pickle\n","import pandas as pd\n","import numpy as np\n","import time"],"execution_count":0,"outputs":[]},{"cell_type":"code","metadata":{"id":"n9KqA4hD54vR","colab_type":"code","colab":{}},"source":["class QueryEngine_BERT():\n","\n","    def __init__(self):\n","        self.corpus = None\n","        self.ids = None\n","\n","    def fit(self, corpus, document_ids=None):\n","        \"\"\"\n","        Builds the query engine on the given corpus.\n","\n","        Args:\n","            corpus: list of documents to build the model on\n","            document_ids: optional, if given it will associate the given id's to each document given in the corpus\n","        \"\"\"\n","        self.ids = document_ids\n","        self.corpus = [paragraph for paragraph in corpus['text']]\n","        self.corpus_embeddings = BERT_sentence_embeddings(corpus, query=False)\n","        \n","    def __create_query_result(self, query, similarities, n):\n","        \"\"\"\n","\n","        Args:\n","            similarities: sparse matrix containing cosine similarities between the query vector and documents from corpus\n","            n: number of most similar documents to include in the result\n","\n","        Returns:\n","            pandas DataFrame containing query, document, similarity\n","        \"\"\"\n","\n","        result = {\n","            'query': query * len(similarities),\n","            'text': self.corpus,\n","            'sim': np.squeeze(similarities)\n","        }\n","        if self.ids:\n","            result.update({'id': self.ids})\n","\n","        result = pd.DataFrame(result).sort_values(by='sim', ascending=False)[:n]\n","\n","        return result[result['sim'] > 0]\n","\n","    def run_query(self, query, n=5):\n","        \"\"\"\n","        Runs the given query, returns max n most similar documents from the corpus on which the model was build.\n","\n","        Args:\n","            query: query to run\n","            n: max number of results returned\n","\n","        Returns:\n","            n(or less) most similar documents from the corpus on which the model was build\n","        \"\"\"\n","        if self.corpus is None:\n","            raise AttributeError('Model not built yet, please call the fit method before running queries!')\n","\n","        assert type(query) == str\n","            \n","\n","        query_embedding = BERT_sentence_embeddings(query, query=True)\n","        similarities = np.dot(self.corpus_embeddings,query_embedding.T)  # TODO: check if this already sorts values\n","        \n","        return self.__create_query_result(query, similarities, n)\n","\n","    def save(self, dir_path, name):\n","        \"\"\"\n","        Serializes the object to file(name.dat) to the directory defined by the path.\n","\n","        Args:\n","            dir_path: path of the directory to save the object to\n","            name: name of the file without any extensions\n","        \"\"\"\n","        pickle_path = dir_path + name + '.dat'\n","        print('Writing object to %s' % pickle_path)\n","        with open(pickle_path, 'wb') as f:\n","            pickle.dump(self, f)\n","\n","    @staticmethod\n","    def load(pickle_path):\n","        \"\"\"\n","        Loads(de-serializes) QueryEngine object from the given path.\n","\n","        Args:\n","            pickle_path: path to QueryEngine pickle\n","\n","        Returns:\n","            QueryEngine object\n","        \"\"\"\n","        with open(pickle_path, 'rb') as f:\n","            query_engine = pickle.load(f)\n","            if type(query_engine) != QueryEngine_BERT:\n","                raise ValueError('Path to non QueryEngine_BERT object!')\n","            return query_engine"],"execution_count":0,"outputs":[]},{"cell_type":"code","metadata":{"id":"paM2onP26kwp","colab_type":"code","colab":{}},"source":["with open(\"test.txt\", \"rb\") as fp:\n","  abstracts = pickle.load(fp)"],"execution_count":0,"outputs":[]},{"cell_type":"code","metadata":{"id":"AmCUXmdMBb0n","colab_type":"code","outputId":"b29ed346-27c2-4122-cc6d-ce4d6fee7c8d","executionInfo":{"status":"ok","timestamp":1588770217089,"user_tz":-120,"elapsed":1457,"user":{"displayName":"Marko Cvitković","photoUrl":"","userId":"14016951488981011334"}},"colab":{"base_uri":"https://localhost:8080/","height":1000}},"source":["abstracts"],"execution_count":0,"outputs":[{"output_type":"execute_result","data":{"text/plain":["[{'paper_id': '0015023cc06b5362d332b3baf348d11567ca2fbb',\n","  'section': 'Abstract',\n","  'text': 'word count: 194 22 Text word count: 5168 23 24 25 author/funder. All rights reserved. No reuse allowed without permission. Abstract 27 The positive stranded RNA genomes of picornaviruses comprise a single large open reading 28 frame flanked by 5′ and 3′ untranslated regions (UTRs). Foot-and-mouth disease virus (FMDV) 29 has an unusually large 5′ UTR (1.3 kb) containing five structural domains. These include the 30 internal ribosome entry site (IRES), which facilitates initiation of translation, and the cis-acting 31 replication element (cre). Less well characterised structures are a 5′ terminal 360 nucleotide 32 stem-loop, a variable length poly-C-tract of approximately 100-200 nucleotides and a series of 33 two to four tandemly repeated pseudoknots (PKs). We investigated the structures of the PKs 34 by selective 2′ hydroxyl acetylation analysed by primer extension (SHAPE) analysis and 35 determined their contribution to genome replication by mutation and deletion experiments. 36 SHAPE and mutation experiments confirmed the importance of the previously predicted PK 37 structures for their function. Deletion experiments showed that although PKs are not essential 38 for replication, they provide genomes with a competitive advantage. However, although 39 replicons and full-length genomes lacking all PKs were replication competent, no infectious 40 virus was rescued from genomes containing less than one PK copy. This is consistent with our 41 earlier report describing the presence of putative packaging signals in the PK region. 42 43 author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '00340eea543336d54adda18236424de6a5e91c9d',\n","  'section': 'Abstract',\n","  'text': 'During the past three months, a new coronavirus (SARS-CoV-2) epidemic has been growing exponentially, affecting over 100 thousand people worldwide, and causing enormous distress to economies and societies of affected countries. A plethora of analyses based on viral sequences has already been published, in scientific journals as well as through non-peer reviewed channels, to investigate SARS-CoV-2 genetic heterogeneity and spatiotemporal dissemination. We examined all full genome sequences currently available to assess the presence of sufficient information for reliable phylogenetic and phylogeographic studies. Our analysis clearly shows severe limitations in the present data, in light of which any finding should be considered, at the very best, preliminary and hypothesis-generating. Hence the need for avoiding stigmatization based on partial information, and for continuing concerted efforts to increase number and quality of the sequences required for robust tracing of the epidemic. . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': '00911cf4f99a3d5ae5e5b787675646a743574496',\n","  'section': 'Abstract',\n","  'text': 'The fast accumulation of viral metagenomic data has contributed significantly to new RNA virus discovery. However, the short read size, complex composition, and large data size can all make taxonomic analysis difficult. In particular, commonly used alignment-based methods are not ideal choices for detecting new viral species. In this work, we present a novel hierarchical classification model named CHEER, which can conduct read-level taxonomic classification from order to genus for new species. By combining k-mer embedding-based encoding, hierarchically organized CNNs, and carefully trained rejection layer, CHEER is able to assign correct taxonomic labels for reads from new species. We tested CHEER on both simulated and real sequencing data. The results show that CHEER can achieve higher accuracy than popular alignmentbased and alignment-free taxonomic assignment tools. The source code, scripts, and pre-trained parameters for CHEER are available via GitHub: https://github.com/KennthShang/CHEER. '},\n"," {'paper_id': '00d16927588fb04d4be0e6b269fc02f0d3c2aa7b',\n","  'section': 'Abstract',\n","  'text': \"Infectious bronchitis (IB) causes significant economic losses in the global poultry industry. Control of infectious bronchitis is hindered by the genetic diversity of the causative agent, infectious bronchitis virus (IBV), which has led to the emergence of several serotypes that lack complete serologic cross-protection. While serotyping by definition requires immunologic characterization, genotyping is an efficient means to identify IBVs detected in samples. Sanger sequencing of the S1 subunit of the spike gene is currently used to genotype IBV; however, the universal S1 PCR was created to work from cultured IBV and it is inefficient at detecting mixed isolates. This paper describes a MinION-based AmpSeq method that genetically typed IBV from clinical samples, including samples with multiple isolates. Total RNA was extracted from fifteen tracheal scrapings and choanal cleft swab samples, randomly reverse transcribed, and PCR amplified using modified S1-targeted primers. Amplicons were barcoded to allow for pooling of samples, processed per manufacturer's instructions into a 1D MinION sequencing library, and sequenced on the MinION. The AmpSeq method detected IBV in 13 of 14 IBV-positive samples. AmpSeq accurately detected and genotyped both IBV lineages in three of five samples containing two IBV lineages. Additionally, one sample contained three IBV lineages, and AmpSeq accurately detected two of the three. Strain identification, including detection of different strains from the same lineage, was also possible with this AmpSeq method. The results demonstrate the feasibility of using MinION-based AmpSeq for rapid and accurate identification and lineage typing of IBV from oral swab samples. Infectious bronchitis virus is an enveloped, pleomorphic gammacoronavirus with an unsegmented, single-stranded, positive-sense, 26-27.8 Kb, RNA genome that encodes the nonstructural polyprotein,1a and 1b, and several structural proteins: spike (S), envelop (E), membrane (M) and nucleocapsid (N). 38,41 In addition, two accessory genes, expressing 3a, 3b and 5a and 5b, respectively, have also been described. 6, 14, 38 The S protein is highly glycosylated and post-translational cleavage leads to two subunits: S1 and S2. 10,48 Besides acting as the viral attachment protein, the S1 protein is a major target of neutralizing antibodies. 7 As with many attachment proteins that are targets of virus-neutralizing antibodies, the S1 subunit is highly diverse with almost 50% of the amino acids differing among IBV serotypes. 2,21,38 Such variation leads to important biological differences between IBV serotypes and the emergence of novel variants. More than 60 serotypes of IBV have been reported, but the most common serotypes of All rights reserved. No reuse allowed without permission. Massachusetts. 16 This genetic diversity leads to emergence of new serotypes and a lack of complete cross-serotype protection by vaccines. 29 The correlation of IBV genotypes and serotypes of IBV has been reported 45 ; therefore, accurate genotypic identification of IBV is an important step to diagnose IBV in clinical respiratory cases, ensure selection of proper IBV vaccines for use in vaccination programs, and to better understand the epidemiology of this global virus. One of the recent comprehensive classification schemes for IBV uses S1 gene sequencebased phylogeny of IBV and identified six genotypes (I-VI), 32 sub-genotypic lineages, and a number of inter-lineage recombinants in global strains of IBV. 44 Among the six genotypes, genotype I (G-I) is the most diverse group of viruses, with 27 unique lineages. 44 As such, sequencing of the S1 subunit provides important information regarding the classification of IBV within samples. The genetic classification of IBV has relied on genotype-specific RT-qPCR assays, serotype-specific S1 RT-qPCR 32,40 and/or pan-IBV S1 RT-PCR assays coupled with Sanger sequencing. 8,31,49 Genotype-specific RT-qPCRs are limited to short fragments, which may miss important changes in the S1 (~1.6 kb) 44 outside the short target. Additionally, since the target is short, the primers lie within the variable regions of the S1 and may require a different assay for each genotype. Lastly, the pan-IBV S1 primers 1, 17, 22, 23, 25, 33 , while only requiring one reaction, have relatively low sensitivity and are typically used only on egg-cultured virus, which adds an additional, time consuming step and many diagnostic labs do not have SPF embryos readily available. Lastly, cloning of PCR products has been used to detect multiple IBV sub-populations in the sample 30 ; however, this is inefficient when potentially dealing with multiple (three or four) All rights reserved. No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/634600 doi: bioRxiv preprint IBV types. As such, the development of a pan-IBV sequencing method to rapidly determine the genotype(s) in samples would aid in lineage typing and tracking of IBV genotypes circulating in poultry flocks. Recently, third generation sequencing technology has been used for detection of viral nucleotides and sequencing ultra-long DNA molecules. 34 The MinION Nanopore sequencer, a new DNA sequencing technology that allows for rapid, in-house, real-time technique detection and differentiation of IBV lineages, may be cost effective and useful in the field. 19 Ampliconbased sequencing has also been used to amplify specific regions of Newcastle disease 3,13 , infectious laryngotracheitis virus 39 Zika 34 , Ebola 35 and avian influenza viruses 46 , by simple RT-PCR and then sequencing on the MinION device. Real-time data analysis, the lack of significant start-up cost investment or maintenance expenses, simultaneous and sequential multiplexing unique to MinION, and the ability to sequence long DNA molecules so that primers are in conserved regions while the product contains the variable region are the features that make this technology highly feasible to be used for disease diagnostics. Accurate identification of IBV genotypes from samples, including detection of multiple isolates from a single sample, is crucial to respiratory disease diagnosis, selection of appropriate IBV vaccines, and epidemiologic studies. Therefore, the aim of the current study is to create a single, amplicon-based protocol to sequence IBV-S1 gene and develop a sequence analysis workflow to identify IBV isolates from clinical swab samples. This method provides a useful assay for IBV and a model for the development of future amplicon sequencing (AmpSeq) based assays. All rights reserved. No reuse allowed without permission. A universal S1 primer set, 27 tailed with the MinION universal adapter sequence of 22 nucleotides (underlined, Table 2 ) to allow barcoding of amplicons, was used for targeted amplification of the IBV S1 gene for IBV. The PCR reaction mixture (Expand High Fidelity PCR system, Roche Diagnostics, Mannheim, Germany) was composed of 10 µl of cDNA, 1 µl of 10 µM forward primer, 1 µl of 10 µM reverse primer, 2.5 µl of 10× Expand high fidelity buffer, 1 µl of Expand High Fidelity Enzyme Mix, and 1 µl of 10mM dNTPs, to a final volume to 25 µl with 9 µl of nuclease-free water. The following thermocycling conditions were used for amplicon synthesis: denaturation at 95 °C for 2 min; 30 cycles of 94 °C for 1 min, 45 °C for 2 min, and 72 °C for 2 minutes, and final extension at 72 °C for 10 min. See Table 2 for details about all three primer sets used in this study. PCR products were electrophoresed in 1.5% agarose with SYBR Safe DNA gel stain (Invitrogen) for visual evaluation. Amplified DNA was purified by Agencourt AMPure XP beads (Beckman Coulter, USA) at 1.6:1 (volumetric bead:DNA) and quantified using the dsDNA High Sensitivity Assay Kit on a Qubit® Fluorimeter 3.0. The amplicons obtained from the tailed S1 primer set were then used to prepare MinIONcompatible DNA libraries. Briefly, each of the amplicons were diluted to 0.5 nM and amplified using barcoding primers (1D PCR Barcoding Amplicon Kit, Oxford Nanopore Technologies, [ONT], UK) and LongAmp Taq 2X Master Mix (New England Biolabs, MA, USA) with the following conditions: 95 °C for 3 min; 15 cycles of 95 °C for 15 seconds, 62 °C for 15 seconds, 65 °C for 80 seconds; and 65 °C for 80 seconds. The barcoded amplicons were bead purified (1:1.4; bead:solution), pooled (Run1; pool of 10 samples, Run2; pool of 5 samples) into a single All rights reserved. No reuse allowed without permission. Centrifuge 24 , Kraken 50 , QIIME 5 and Mothur 37 have been used to identify and profile microbial species; however, the high error rate 20 of Nanopore reads translates to poor classification accuracy for many of these tools. Alternatively, de novo 3,26 or reference-based 18 assembly methods have been used in MinION and other deep sequencing platforms. Using a strategy similar to other unbiased laboratory tests (e.g., standard bacterial cultures), an approach was developed to maximize usage of reads (i.e., reads are not discarded based on pre-set length or abundance requirements, similar to how a single colony may be interpreted as a significant result). This approach uses read-based classification against a database containing an equal number of representative sequences per lineage to detect and classify the reads based on their lineage, before conducting read-based classification to the isolate level by using all available IBV sequences in a separate database. Finally, lineage-clustered isolate-level read assignments are interpreted by a veterinary diagnostician to result in reads available for consensus building. The use of a final consensus alignment helps to overcome the individual error rate of MinION sequencing. 3 Similarly, the absence of predetermined metrics used in de novo assembly allows for the informed decision as to how many consensus sequences to build, a bioinformatics problem when dealing with clinical cases that can contain more than one isolate (e.g., similar to how there is not a predetermined number of significant bacterial colonies). Confirmatory followup tests (e.g., RT-PCR) may be needed when dealing with low number of reads in clinical samples; however, future testing of this new technology will allow for creating standards for such confirmatory testing. Taken together, these results suggest the application of MinION-based AmpSeq, specifically for detecting IBV isolates from clinical samples within a few days, compared to several days to weeks and multiple diagnostic assays to culture and detect multiple IBV All rights reserved. No reuse allowed without permission. \"},\n"," {'paper_id': '0139ea4ca580af99b602c6435368e7fdbefacb03',\n","  'section': 'Abstract',\n","  'text': \"Nipah Virus (NiV) came into limelight recently due to an outbreak in Kerala, India. NiV causes severe disease and death in people with over 75% case fatality rate. It is a public health concern and has the potential to become a global pandemic. Lack of treatment has forced the containment methods to be restricted to isolation and surveillance. WHO's 'R&D Blueprint list of priority diseases' (2018) indicates that there is an urgent need for accelerated research & development for addressing NiV. In the quest for druglike NiV inhibitors (NVIs) a thorough literature search followed by systematic data curation was conducted. Rigorous data analysis was done with curated NVIs for prioritizing druglike compounds. For the same, more than 1800 descriptors of NVIs were computed and comparative analysis was performed with the FDA approved small molecules and antivirals. These compounds were further evaluated through PAINS filter to study their toxicity profile. Simultaneously, compounds were also prioritized based on the robustness of the assays through which they were identified. Our efforts lead to the creation of a well-curated structured knowledgebase of 182 NVIs with 98 small molecule inhibitors. The reported IC50/EC50 values for some of these inhibitors are in the nanomolar range -as low as 0.47 nM. In order to prioritize these inhibitors, we performed several tests and applied filters to identify drug-like non-toxic compounds. Of 98, a few compounds passed DruLito & PAINS filters exhibiting drug-like properties and were also prioritized in an independent screen based only the assay robustness. The NVIs have diverse structural features and offer a wide spectrum of ways in which they can be developed further as druglike molecules. We report a knowledgebase for furthering the development of NVIs. The platform has a diverse set of 98 NVIs of which a few have been prioritized based on a combined evidence strategy. The platform has the provision to submit new inhibitors as and when reported by the community for further enhancement of NiV inhibitor landscape. \"},\n"," {'paper_id': '013d9d1cba8a54d5d3718c229b812d7cf91b6c89',\n","  'section': 'Abstract',\n","  'text': 'Background: A novel coronavirus (2019-nCoV) emerged in Wuhan City, China, at the end of 2019 and has caused an outbreak of human-to-human '},\n"," {'paper_id': '018fb5e62fbbcae07d57d94d29ac630dcc4dccf9',\n","  'section': 'Abstract',\n","  'text': 'All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '01d162d7fae6aaba8e6e60e563ef4c2fca7b0e18',\n","  'section': 'Abstract',\n","  'text': 'Faced with the current large-scale public health emergency, collecting, sorting, and analyzing biomedical information related to the \"coronavirus\" should be done as quickly as possible to gain a global perspective, which is a basic requirement for strengthening epidemic control capacity. However, for human researchers studying the viruses and the hosts, the vast amount of information available cannot be processed effectively and in a timely manner, particularly when the scientific understanding may be limited, which can further lower the information processing efficiency. We present TWIRLS, a method that can automatically acquire, organize, and classify information. Additionally, independent functional data sources can be added to build an inference system using a machine-based approach, which can provide relevant knowledge to help human researchers quickly establish subject cognition and to make more effective decisions. TWIRLS can automatically analyze more than three million words in more than 14,000 literature articles in only 4 hours. Combining with generalized gene interaction databases creates a data interface that can help researchers to further analyze the information. Using the TWIRLS system, we found that an important regulatory factor angiotensin-converting enzyme 2 (ACE2) may be involved in the All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. : medRxiv preprint host pathological changes on binding to the coronavirus after infection. After triggering functional changes in ACE2/AT2R, an imbalance in the steady-state cytokine regulatory axis involving the Renin-Angiotensin System and IP-10 leads to a cytokine storm. '},\n"," {'paper_id': '01e3b313e78a352593be2ff64927192af66619b5',\n","  'section': 'Abstract',\n","  'text': 'Viruses interact with hundreds to thousands of proteins in mammals, yet adaptation 6 against viruses has only been studied in a few proteins specialized in antiviral defense. Whether adaptation to viruses typically involves only specialized antiviral proteins or 8 affects a broad array of proteins is unknown. Here, we analyze adaptation in ~1,300 9 virus-interacting proteins manually curated from a set of 9,900 proteins conserved 10 across mammals. We show that viruses (i) use the more evolutionarily constrained 11 proteins from the cellular functions they hijack and that (ii) despite this high constraint, 12 virus-interacting proteins account for a high proportion of all protein adaptation in 13 humans and other mammals. Adaptation is elevated in virus-interacting proteins across 14 all functional categories, including both immune and non-immune functions. Our results 15 demonstrate that viruses are one of the most dominant drivers of evolutionary change 16 across mammalian and human proteomes. 17 '},\n"," {'paper_id': '02201e4601ab0eb70b6c26480cf2bfeae2625193',\n","  'section': 'Abstract',\n","  'text': 'School closure is often considered as an option to mitigate influenza epidemics because of its potential to reduce transmission in children and then in the community. The policy is still however highly debated because of controversial evidence. Moreover, the specific mechanisms leading to mitigation are not clearly identified. We introduced a stochastic spatial age-specific metapopulation model to assess the role of holiday-associated behavioral changes and how '},\n"," {'paper_id': '0255ea4b2f26a51a3bfa3bd8f3e1978c82c976d5',\n","  'section': 'Abstract',\n","  'text': 'Post-transcriptional gene silencing (PTGS) is a powerful tool to understand and control plant metabolic pathways, which is central to plant biotechnology. PTGS is commonly accomplished through delivery of small interfering RNA (siRNA) into cells. Standard plant siRNA delivery methods (Agrobacterium and viruses) involve coding siRNA into DNA vectors, and are only tractable for certain plant species. Herein, we develop a nanotube-based platform for direct delivery of siRNA, and show high silencing efficiency in intact plant cells. We demonstrate that nanotubes successfully deliver siRNA and silence endogenous genes owing to effective intracellular delivery and nanotubeinduced protection of siRNA from nuclease degradation. This study establishes that nanotubes could enable a myriad of plant biotechnology applications that rely on RNA delivery to intact cells. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '0313c3faa16cd66d64f31ae37e40fb70695d69fb',\n","  'section': 'Abstract',\n","  'text': 'Recent outbreak of coronavirus disease 2019 in China has lead a global pandemic around the world. For controlling COVID-19 outbreaks, most countries take two typical intervention strategies: suppression approach like immediately lock-downing cities at epicentre of and mitigation; or mitigation approach that slows down but not stopping epidemic of COVID-19 for reducing peak healthcare demand. Both strategies have their apparent merits and limitations; it becomes extremely hard to conduct one intervention strategy as the most feasible way to certain country. Targeting at this problem, this paper conducts a feasibility study by defining a mathematical model named SEMCR that access the effectiveness of mitigation, suppression and hybrid interventions for controlling COVID-19 outbreaks in London and Wuhan. The model first extends traditional SEIR (Susceptible-Exposed-Infectious-Recovered) model by considering one key fact: there is a direct link between Exposed and Recovered population. Then, it defines threshold and parameters to classify two stages of COVID control: active contain by isolation of cases and contacts, passive contain by suppression or mitigation. The model was fitted and evaluated with public dataset containing daily number of confirmed active cases including Wuhan, London, Hubei province and the UK during January, 2020 and March 2020. The simulated results show that 1) Suppression taken in Wuhan significantly reduces total exposed and infectious populations, but it has to be consistently maintained at least a period of 90 days (by the middle of April 2020) where its success relies on sufficient support from other parts of China. 2) In London, it is possible to take a hybrid intervention of suppression and mitigation for every 2 or 3 weeks over a longer period. While the total infectious populations will be doubled, the economic performance of London will be less affected due to limited mobility constraints. 3) Both in Wuhan and London cases, '},\n"," {'paper_id': '033ea7af3e6137df652de026f0751ac435327b75',\n","  'section': 'Abstract',\n","  'text': \"Quantitative reverse transcription polymerase chain reaction (RT-qPCR) assay is the gold standard recommended to test for acute SARS-CoV-2 infection. [1] [2] [3] [4] It has been used by the Centers for Disease Control and Prevention (CDC) and several other companies in their Emergency Use Authorization (EUA) assays. With many PCR-based molecular assays, an extraction step is routinely used as part of the protocol. This step can take up a significant amount of time and labor, especially if the extraction is performed manually. Long assay time, partly caused by slow sample preparation steps, has created a large backlog when testing patient samples suspected of COVID-19. Using flu and RSV clinical specimens, we have collected evidence that the RT-qPCR assay can be performed directly on patient sample material from a nasal swab immersed in virus transport medium (VTM) without an RNA extraction step. We have also used this approach to test for the direct detection of SARS-CoV-2 reference materials spiked in VTM. Our data, while preliminary, suggest that using a few microliters of these untreated samples still can lead to sensitive test results. If RNA extraction steps can be omitted without significantly affecting clinical sensitivity, the turnaround time of COVID-19 tests and the backlog we currently experience can be reduced drastically. Next, we will confirm our findings using patient samples. The sample preparation step is generally time-consuming, regardless of whether it is done manually or automated. In addition, there is a current shortage of the recommended viral RNA extraction kits needed for the Centers for Disease Control and Prevention (CDC) RT-qPCR assay to diagnose SARS-CoV-2. During a study in developing a rapid protocol for influenza (Inf) and respiratory syncytial virus (RSV) diagnostics, we investigated the feasibility of omitting the sample preparation steps to expedite the test without significantly impacting the test's sensitivity. Using Inf and RSV clinical specimens, we successfully performed RT-qPCR reactions by simply adding a few microliters of the unprocessed sample in viral transport medium (VTM) directly into the RT-qPCR assay master mix. We then tested the approach using SARS-CoV-2 plasmid and SeraCare AccuPlex reference materials. \"},\n"," {'paper_id': '03813d8657ba43ea382788caec2d14257b26d8fd',\n","  'section': 'Abstract',\n","  'text': \"The spread of COVID-19 engulfs almost all the countries and territories of the planet, and infections and fatality are increasing rapidly. The first epi-center of its' massive spread was in Wuhan, Hubei province, China having a temperate weather, but the spread has got an unprecedented momentum in European temperate countries mainly in Italy and Spain (as of March 30, 2020). However, Malaysia and Singapore and the neighboring tropical countries of China got relatively low spread and fatality that created a research interest on whether there are potential impacts of weather condition on COVID-19 spread. Adopting the SIR (Susceptible Infected Removed) deviated model to predict potential cases and death in the coming days from COVID-19 was done using the secondary and official sources of data. This study shows that COVID-19 spread and fatality tend to be high across the world but compared to tropical countries, it is going to be incredibly high in the temperate countries having lower temperature (7-16°C) and humidity (80-90%) in last March. However, some literature predicted that this might not to be true, rather irrespective of weather conditions there might be a continuous spread and death. Moreover, a large number of asymptotic COVID-19 carrier in both temperate and tropical countries may re-outbreak in the coming winter. Therefore, a comprehensive global program with the leadership of WHO for testing of entire population of the world is required, which will be very useful for the individual states to take proper political action, social movement and medical services. \"},\n"," {'paper_id': '03ce432f27c7df6af22b92245a614db2ecb5de5f',\n","  'section': 'Abstract',\n","  'text': 'Human astroviruses are small nonenveloped viruses with positive-sense single-stranded RNA genomes that contain three main open reading frames: ORF1a, ORF1b and ORF2. Astroviruses cause acute gastroenteritis in children worldwide and have been associated with encephalitis and meningitis in immunocompromised individuals. Through comparative genomic analysis of >400 astrovirus sequences, we identified a conserved \"ORFX\" overlapping the capsid-encoding ORF2 in genogroup I, III and IV astroviruses. ORFX appears to be subject to purifying selection, consistent with it encoding a functional protein product, termed XP. Using ribosome profiling of cells infected with human astrovirus 1, we confirm initiation at the ORFX AUG. XP-knockout astroviruses are strongly attenuated and after passaging can partly restore viral titer via pseudo-reversions, thus demonstrating that XP plays an important role in virus growth. To further investigate XP, we developed an astrovirus replicon system. We demonstrate that XP has only minor effects on RNA replication and structural protein production. Instead, XP associates with the plasma membrane with an extracellular N-terminus topology and promotes efficient virus release. Using two different assays, we show that expression of human or related astrovirus XPs leads to cell permeabilization, suggesting a viroporin-like activity. The discovery of XP advances our knowledge of these important human viruses and opens a new direction of research into astrovirus replication and pathogenesis. They are considered to be the second most common cause of gastroenteritis in infants after rotavirus 1 . Recently, and especially in immunocompromised individuals, astroviruses have also . CC-BY 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/661579 doi: bioRxiv preprint been associated with extra-intestinal infections including fatal meningitis and encephalitis 2 . Despite their undoubted importance, astroviruses still represent one of the least studied groups of human positive-sense RNA viruses. The genome contains three main open reading frames: ORF1a encoding a nonstructural polyprotein (nsP1a), ORF1b encoding the RNA-dependent RNA polymerase (RdRp), and ORF2 encoding the capsid protein (CP) (Fig. 1A) 3 . ORF1a and ORF1b are translated from the genomic RNA (gRNA), with expression of ORF1b depending on programmed ribosomal frameshifting whereby a proportion of ribosomes translating ORF1a make a −1 nucleotide shift into ORF1b. Frameshifting occurs at a conserved A_AAA_AAC sequence within the ORF1a/ORF1b overlap region and depends on a 3′-adjacent stimulatory RNA stem-loop structure 4 . ORF2 is translated from a subgenomic RNA (sgRNA) that is produced during virus infection. Previously, using comparative genomics we identified a conserved fourth ORF (ORFX) in genogroup I astroviruses, which appears to be subject to purifying selection and therefore is likely to encode a functional protein product, termed the X protein (XP, 12 kDa, 112 aa) 5 . ORFX overlaps the 5′ region of the capsid-encoding ORF2 and is predicted to be translatable via ribosomal leaky scanning, which is expected to be enhanced by the very short leader on the sgRNA 3,6,7 . However, despite the comparative genomic predictions, the existence, relevance, and function of XP have never been demonstrated. Here we perform a functional molecular dissection of ORFX, providing insight into the simultaneous orchestration of several overlapping functional elements in a small human-pathogenic RNA virus. We show that ORFX is translated during virus infection, XP knockout mutant viruses are highly attenuated, and XP is a transmembrane (TM) protein that promotes efficient virus release via a viroporin-like activity. Through an extended bioinformatic analysis, we predict that related XP proteins are widely encoded in other mammalian astroviruses including the divergent \"MLB\" group of human-infecting astroviruses. Comparative genomic analysis reveals a potential ORFX in genogroup I, III and IV astroviruses. Since there are many more astrovirus sequences available now than in 2010, we began by repeating our previous comparative genomic analysis but this time also extending to other astrovirus genogroups. All mammalian astrovirus complete or nearly complete genome sequences were obtained from the '},\n"," {'paper_id': '03ea3a614b56409d3f099c9ad764864293132540',\n","  'section': 'Abstract',\n","  'text': 'Ribosomal frameshifting during the translation of RNA is implicated in both human disease and viral infection. While previous work has uncovered many mechanistic details about single RNA frameshifting kinetics in vitro, very little is known about how single RNA frameshift in living systems. To confront this problem, we have developed technology to quantify live-cell single RNA translation dynamics in frameshifted open reading frames. Applying this technology to RNA encoding the HIV-1 frameshift sequence reveals a small subset (~8%) of the translating pool robustly frameshift in living cells. Frameshifting RNA are preferentially in multi-RNA \"translation factories,\" are translated at about the same All rights reserved. No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/478040 doi: bioRxiv preprint 2 rate as non-frameshifting RNA (~2 aa/sec), and can continuously frameshift for more than four rounds of translation. Fits to a bursty model of frameshifting constrain frameshifting kinetic rates and demonstrate how ribosomal traffic jams contribute to the persistence of the frameshifting state. These data provide novel insight into retroviral frameshifting and could lead to new strategies to perturb the process in living cells. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '03ea9ad47ebe9a599205b99390c45490e6724024',\n","  'section': 'Abstract',\n","  'text': 'Since the introduction of the novel Corona Virus to the Chinese city Wuhan in the Hubei province during the late December 2019, the effectiveness of the deadly disease, its human infection, spreading severity and the mortality rate of the infection has been an issue of debate. The outbreak of the virus along the time has become a massive threat to the global public health security and has been declared as a pandemic. Accounting the radical number of increases in the infected cases and the death due to COVID-19 infections around the globe, there is a need to predict the infections among the people by making proper optimization and using various Infectious Disease modelling (IDM) methods, in order to challenge the outcome. In comparison with previous diseases like SARS and Ebola viruses, the new corona virus infections are infectious during the incubation period. In addition to that, naturally produced droplets from humans (e.g. droplets produced by breathing, talking, sneezing, coughing) and Person-to-person contact transmission are reported to be the foremost ways of transmission of novel corona virus. By considering the above two factors, a modified SEIR (Susceptibility-Exposure-Infection-Recovery) method have been used for predicting the spread of the infections in the state of Tamil Nadu which is located in the southern part of India. Further, we have utilized the current surveillance data from Health and Family Welfare Department -Government of Tamil Nadu to accurately predict the spreading trend of the infection on a state level. '},\n"," {'paper_id': '04030bba3035a58c7725ae267973206f6eb6c0b4',\n","  'section': 'Abstract',\n","  'text': 'The recent outbreak of infections by the 2019 novel coronavirus (2019-nCoV), the third 36 zoonotic CoV has raised great public health concern. The demand for rapid and accurate 37 diagnosis of this novel pathogen brought significant clinical and technological challenges. 38 Currently, metagenomic next-generation sequencing (mNGS) and reverse-transcription PCR 39 (RT-PCR) are the most widely used molecular diagnostics for 2019-nCoV. 40 Methods 41 2019-nCoV infections were confirmed in 52 specimens by mNGS. Genomic information was 42 analyzed and used for the design and development of an isothermal, CRISPR-based 43 diagnostic for the novel virus. The diagnostic performance of CRISPR-nCoV was assessed 44 and also compared across three technology platforms (mNGS, RT-PCR and CRISPR) 45 Results 46 2019-nCoVs sequenced in our study were conserved with the Wuhan strain, and shared 47 certain genetic similarity with SARS-CoV. A high degree of variation in the level of viral 48 All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '041bae0a6de2b69979d39460b3f2ee8946534ec2',\n","  'section': 'Abstract',\n","  'text': 'In the note, the logistic growth regression model is used for the estimation of the final size and its peak time of the coronavirus epidemic in China, South Korea, and the rest of the World. In the previous article [1], we try to estimate the final size of the epidemic for the whole World using the logistic model and SIR model. The estimation was about 83000 cases. Both models show that the outbreak is moderating; however, new data showed a linear upward trend. It turns out that the epidemy in China was slowing but is begin to spread elsewhere in the World. In this note, we will give forecasting epidemic size for China, South Korea, and the rest of the World and daily predictions using the logistic model. Full daily reports and counties outside of China generated are available at https://www.researchgate.net/publication/339912313_Forecasting_of_final_COVID -19_epidemic_size The MATLAB program fitVirus used for calculations is freely available from https://www.mathworks.com/matlabcentral/fileexchange/74411-fitvirus We note that logistic models give similar results as the SIR model (at least for the case of China and South Korea). However, the logistic model is given by explicit formula and is thus much simpler for regression analysis than the SIR model, where one must on each optimization step solve a system of ordinary differential equations. (One may, '},\n"," {'paper_id': '052bf4fb7deaf593862991af3b118b5f11a9fbe1',\n","  'section': 'Abstract',\n","  'text': 'Herewith we present a computational model for the forecasting of cumulative diagnosed cases of Covid-19 pneumonia within a country. The only explicit parameter of the model is the population density. The implicit parameter is a moving average ambient temperature, currently integrated into the kinetic constants. Other finer details pertaining to the mechanism of the pathogen SARS-CoV-2 spread within a given region are implicitly manifested in the exponent parameters derived from the non-linear fitting of the published data on Covid-19 occurrence. The performance of the model is demonstrated on a few selected countries, and on the Diamond Princess cruising ship outbreak. The model might be used as an aiding tool for the policy makers regarding the decisions on the containment measures and quarantine regime required. '},\n"," {'paper_id': '0537863d6c059cc9cf5ec8dd0beb27dec5d9d801',\n","  'section': 'Abstract',\n","  'text': 'The paper evaluates total inclusive costs of three public health approaches to address the COVID-19 epidemic in the US based on epidemiological projections in Ferguson et al (2020). We calculate and add costs of lost productivity and costs of mortality measured through the value of statistical life. We find that the aggressive approach which involves strict suppression measures and a drastic reduction of economic activity for three months with extensive testing and case tracking afterwards results in the lowest total costs for the society. The approach of doing no non-pharmaceutical measures results in the lowest total costs if the infection fatality rate falls below 0.15%. '},\n"," {'paper_id': '05d99c07db59b6948e39bfa62c2cbbf62944059a',\n","  'section': 'Abstract',\n","  'text': 'An outbreak of a novel coronavirus, named CoVID-19, was first reported in China on 31 December 2019. As of 9 February 2020, cases have been reported in 25 countries, including probable cases of human-to-human transmission in England. We adapted an existing national-scale metapopulation model to capture the spread of CoVID-19 in England and Wales. We used 2011 census data to capture population sizes and population movement, together with parameter estimates from the current outbreak in China. We predict that a CoVID-19 outbreak will peak 126 to 147 days (~4 months) after the start of person-to-person transmission in England and Wales in the absence of controls, assuming biological parameters remain unchanged. Therefore, if person-to-person transmission persists from February, we predict the epidemic peak would occur in June. The starting location has minimal impact on peak timing, and model stochasticity varies peak timing by 10 days. Incorporating realistic parameter uncertainty leads to estimates of peak time ranging from 78 days to 241 days after person-to-person transmission has been established. Seasonal changes in transmission rate substantially impact the timing and size of the epidemic peak, as well as the total attack rate. We provide initial estimates of the potential course of CoVID-19 in England and Wales in the absence of control measures. These results can be refined with improved estimates of epidemiological parameters, and permit investigation of control measures and cost effectiveness analyses. Seasonal changes in transmission rate could shift the timing of the peak into winter months, which will have important implications for health-care capacity planning. '},\n"," {'paper_id': '061ffcdd4d674c4d7ce24e4aa7c5037c68596864',\n","  'section': 'Abstract',\n","  'text': 'infectious disease caused by severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) with multiple organ injuries. The aim of this study was to analyze SARS-CoV-2-induced acute liver injury (ALI), its association with death risk and prognosis after discharge. Methods: Three-hundred and fifty-five COVID-19 patients were recruited. Clinical data were collected from electronic medical records. ALI was evaluated and its prognosis was tracked. The association between ALI and death risk was analyzed. Results: Of 355 COVID-19 patients, 211 were common, 88 severe, and 51 critical ill cases, respectively. On admission, 223 (62.8%) patients were with hypoproteinemia, 151(42.5%) with cholestasis, and 101 (28.5%) with hepatocellular injury. As expected, ALI was more common in critical ill patients. By multivariate logistic regression, male, older age and lymphocyte reduction were three important independent risk factors predicting ALI among COVID-19 patients. Death risk analysis shows that fatality rate was higher among patients with hypoproteinemia than those without hypoproteinemia (RR=9.471, P<0.001). Moreover, fatality rate was higher among patients with cholestasis than those without cholestasis (RR=2.182, P<0.05). Follow-up observation found that more than one hepatic functional indexes of two-third patients remained abnormal 14 days after discharge. Conclusions: ALI at early stage elevates death risk of COVID-19 patients. SARS-CoV-2-induced ALI has not recovered completely 14 days after discharge. '},\n"," {'paper_id': '0624a12abfe85c8b5070850d912a2db4cd453236',\n","  'section': 'Abstract',\n","  'text': 'Long-read sequencing technologies are invaluable for determining complex RNA transcript architectures but are error-prone. Numerous \"hybrid correction\" algorithms have been developed for genomic data that correct long reads by exploiting the accuracy and depth of short reads sequenced from the same sample. These algorithms are not suited for correcting more complex transcriptome sequencing data. We have created a novel algorithm called TALC (Transcription Aware Long Read Correction) which models changes in RNA expression and isoform representation in a weighted De-Bruijn graph to correct long reads from transcriptome studies. We show that transcription aware correction by TALC improves the accuracy of the whole spectrum of downstream RNA-seq applications and is thus necessary for transcriptome analyses that use long read technology. '},\n"," {'paper_id': '06837008df793f872a6fb830dfb83c9525edb7c4',\n","  'section': 'Abstract',\n","  'text': 'To combat the pandemic of the coronavirus disease , numerous governments have established phone hotlines to prescreen potential cases. These hotlines have struggled with the volume of callers, leading to wait times of hours or, even, an inability to contact health authorities. Symptoma is a symptom-to-disease digital health assistant that can differentiate more than 20,000 diseases with an accuracy of more than 90%. We tested the accuracy of Symptoma to identify COVID-19 using a set of diverse clinical cases combined with case reports of COVID-19. We showed that Symptoma can accurately distinguish COVID-19 in 96.32% of clinical cases. When considering only COVID-19 symptoms and risk factors, Symptoma identified 100% of those infected when presented with only three signs. Lastly, we showed that Symptoma\\'s accuracy far exceeds that of simple \"yes-no\" questionnaires widely available online. In summary, Symptoma provides unparalleled accuracy in systematically identifying cases of COVID-19 while also considering over 20,000 other diseases. Furthermore, Symptoma allows free text input, furthered with disease-specific follow up questions, in 36 languages. Combined, these results and accessibility give Symptoma the potential to be a key tool in the global fight against COVID-19. The Symptoma predictor is freely available online at https://www. '},\n"," {'paper_id': '06a1002f9fbea7179ac3572843f66b14568af6e4',\n","  'section': 'Abstract',\n","  'text': '2019-nCov has caused more than 80 deaths as of 27 January 2020 in China, and infection cases have been reported in more than 10 countries. However, there is no approved drug to treat the disease. 2019-nCov M pro is a potential drug target to combat the virus. We built homology models based on SARS M pro structures, and docked 1903 small molecule drugs to the models. Based on the docking score and the 3D similarity of the binding mode to the known M pro ligands, 4 drugs were selected for binding free energy calculations. Both MM/GBSA and SIE methods voted for nelfinavir, with the binding free energy of -24.69±0.52 kcal/mol and -9.42±0.04 kcal/mol, respectively. Therefore, we suggested that nelfinavir might be a potential inhibitor against 2019-nCov M pro . author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '06c1b3535b83251cf92c01258b5048beeab7a460',\n","  'section': 'Abstract',\n","  'text': 'The basic reproductive number -R 0 -is one of the most common and most commonly misapplied numbers in public health. Nevertheless, estimating R 0 for every transmissible pathogen, emerging or endemic, remains a priority for epidemiologists the world over. Although often used to compare outbreaks and forecast pandemic risk, this single number belies the complexity that two different pathogens can exhibit, even when they have the same R 0 . Here, we show how predicting outbreak size requires both an estimate of R 0 and an estimate of the heterogeneity in the number of secondary infections. To facilitate rapid determination of outbreak risk, we propose a reformulation of a classic result from random network theory that relies on contact tracing data to simultaneously determine the first moment (R 0 ) and the higher moments (representing the heterogeneity) in the distribution of secondary infections. Further, we show how this framework is robust in the face of the typically limited amount of data for emerging pathogens. Lastly, we demonstrate that without data on the heterogeneity in secondary infections for emerging pathogens like 2019-nCoV, the uncertainty in outbreak size ranges dramatically, in the case of 2019-nCoV from 5-40% of susceptible individuals. Taken together, our work highlights the critical need for contact tracing during emerging infectious disease outbreaks and the need to look beyond R 0 when predicting epidemic size. '},\n"," {'paper_id': '06d12dc5ac32d82387c65370d0a600e13059122d',\n","  'section': 'Abstract',\n","  'text': 'To ultimately combat the emerging COVID-19 pandemic, it is desired to develop an 17 effective and safe vaccine against this highly contagious disease caused by the SARS-CoV-2 18 coronavirus. Our literature and clinical trial survey showed that the whole virus, as well as the 19 spike (S) protein, nucleocapsid (N) protein, and membrane (M) protein, have been tested for 20 vaccine development against SARS and MERS. However, these vaccine candidates might lack 21 the induction of complete protection and have safety concerns. We then applied the Vaxign 22 reverse vaccinology tool and the newly developed Vaxign-ML machine learning tool to predict 23 COVID-19 vaccine candidates. By investigating the entire proteome of SARS-CoV-2, six 24 proteins, including the S protein and five non-structural proteins (nsp3, 3CL-pro, and nsp8-10), 25 were predicted to be adhesins, which are crucial to the viral adhering and host invasion. The S, 26 nsp3, and nsp8 proteins were also predicted by Vaxign-ML to induce high protective 27 antigenicity. Besides the commonly used S protein, the nsp3 protein has not been tested in any 28 coronavirus vaccine studies and was selected for further investigation. The nsp3 was found to be 29 more conserved among SARS-CoV-2, SARS-CoV, and MERS-CoV than among 15 30 coronaviruses infecting human and other animals. The protein was also predicted to contain 31 promiscuous MHC-I and MHC-II T-cell epitopes, and linear B-cell epitopes localized in specific 32 locations and functional domains of the protein. By applying reverse vaccinology and machine 33 learning, we predicted potential vaccine targets for effective and safe COVID-19 vaccine 34 development. We then propose that an \"Sp/Nsp cocktail vaccine\" containing a structural 35 protein(s) (Sp) and a non-structural protein(s) (Nsp) would stimulate effective complementary 36 immune responses. 37 38 39 40 '},\n"," {'paper_id': '073d74442e2655d79b0b3f764a627ec667ad422c',\n","  'section': 'Abstract',\n","  'text': \"The newly emergent human virus SARS-CoV-2 is resulting in high fatality rates and incapacitated health systems. Preventing further transmission is a priority. We analysed key parameters of epidemic spread to estimate the contribution of different transmission routes and determine requirements for case isolation and contact-tracing needed to stop the epidemic. We conclude that viral spread is too fast to be contained by manual contact tracing, but could be controlled if this process was faster, more efficient and happened at scale. A contact-tracing App which builds a memory of proximity contacts and immediately notifies contacts of positive cases can achieve epidemic control if used by enough people. By targeting recommendations to only those at risk, epidemics could be contained without need for mass quarantines ('lock-downs') that are harmful to society. We discuss the ethical requirements for an intervention of this kind. \"},\n"," {'paper_id': '07e833d0917cace550853f72923856d0fe1a7120',\n","  'section': 'Abstract',\n","  'text': 'Many pathogens take advantage of the dependence of the host on the interaction of hundreds of extracellular proteins with the glycosaminoglycans heparan sulfate to regulate homeostasis and use heparan sulfate as a means to adhere and gain access to cells. Moreover, mucosal epithelia such as that of the respiratory tract are protected by a layer of mucin polysaccharides, which are usually sulfated. Consequently, the polydisperse, natural products of heparan sulfate and the allied polysaccharide, heparin have been found to be involved and prevent infection by a range of viruses including S-associated coronavirus strain HSR1. Here we use surface plasmon resonance and circular dichroism to measure the interaction between the SARS-CoV-2 Spike S1 protein receptor binding domain (SARS-CoV-2 S1 RBD) and heparin. The data demonstrate an interaction between the recombinant surface receptor binding domain and the polysaccharide. This has implications for the rapid development of a first-line therapeutic by repurposing heparin and for next-generation, tailor-made, GAG-based antivirals. '},\n"," {'paper_id': '08660499ee722a74043f8417faee3e1eeb9d0f5f',\n","  'section': 'Abstract',\n","  'text': \"The ongoing Zika epidemic in the Americas has challenged public health surveillance, response, and control systems. Even as the epidemic appears to be near its end in the Americas, it is unclear whether substantial Zika virus transmission may still be ongoing. This issue is exacerbated by large discrepancies in local case reporting and significant delays in detecting outbreaks due to surveillance gaps. To uncover locations with lingering outbreaks in the Americas, we investigated travel-associated Zika cases diagnosed in the United States and Europe to identify signatures of transmission dynamics that were not captured by local reporting. We found that a large and unreported Zika outbreak occurred in Cuba during 2017, a year after peak transmission in neighboring countries, with cases still appearing in 2018. By sequencing Zika virus from infected travelers, we show that the 2017 outbreak in Cuba was sparked by longlived lineages of Zika virus introduced from multiple places in the Americas a year prior. Our data suggest that while aggressive mosquito control in Cuba may initially have been effective at mitigating Zika virus transmission, in the absence of vaccines, herd immunity, or strong international coordination, such control measures may need to be maintained to be effective. Our study highlights how Zika virus may still be 'silently' spreading in the Americas and provides a framework for more accurately understanding outbreak dynamics. \"},\n"," {'paper_id': '08826d0596a01a2a482eb6d80edb0e87cddc304e',\n","  'section': 'Abstract',\n","  'text': 'The evolution of the pandemic caused by COVID-19, its high reproductive number and the associated clinical needs, is overwhelming national health systems. We propose a method for predicting both ICU requirements and the number of deaths, and which will enable the health authorities of the countries involved to plan the resources needed to face the pandemic as many days in advance as possible. We use official Spanish data to predict ICU admissions and deaths based on the number of infections. We employ OLS to perform the econometric estimation, and through RMSE, MSE, MAPE, and SMAPE forecast performance measures we select the best lagged predictor of both dependent variables. For Spain, our prediction shows that the best predictor of ICU admissions is the number of people infected eight days before, and that the best predictor of deaths is the number of people infected five days before. In the first case, we obtain a 98% coefficient of determination, and in the second a 97% coefficient. The estimated delayed elasticities find that a 1% increase in the number of cases today will imply a 0.72% increase in ICU patients eight days later and a 1.09% increase in the number of deaths five days later. The model is not intended to analyse the epidemiology of COVID-19. Our objective is rather to estimate a leading indicator of clinical needs. Having a forecast model available several days in advance can enable governments to more effectively face the gap between needs and resources triggered by the outbreak and thus reduce the deaths caused by COVID-19. No funding Evidence before this study Daily news regarding the exponential growth of those affected by COVID-19 shows that healthcare resources are being overwhelmed by clinical needs in many countries. In particular, serious problems are arising in the most affected countries due to the shortage of ICU beds and the large number of deaths that the authorities are unable to deal with. National health authorities do not have adequate prediction mechanisms to facilitate clinical crisis management. We have performed bibliographic searches of the usual terms used to designate COVID-19, together with those of \"prediction\", \"estimation\", \"ICU\", \"mortality\" and the like, both in Pubmed and in Google Scholar. The predictive literature related to COVID-19 remains very sparse and the few models that do exist are based on exponential adjustments for forecasting the population affected. However, these models lose their predictive accuracy when the growth rate of infections decreases, added to which such models fail to determine the most statistically efficient maximum prediction time. We apply a previously unused method based on predictions through delayed logarithmic estimates of ICU admissions and deaths based on the number of infections. For Spain, we estimate that the best predictor of ICU admissions is the number of people infected eight days before and that the best predictor of deaths is the number of those infected five days before. The findings herald a step forward that improves the possibility of managing the health crisis. We provide a method to estimate a leading indicator of needs, which thus far has been unavailable to health authorities and which should allow them to plan for the resources required. Furthermore, it is a versatile and simple method that is applicable to any country, state, region, city or hospital area as well as to any type of health care need associated with the COVID-19 pandemic and similar future ones. All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '08911cdc65e71e6398ca79b46806e6c8b2b730ae',\n","  'section': 'Abstract',\n","  'text': 'medRxiv preprint immunology 2015; 156(1): 19-27. 15. Gu J, Gong E, Zhang B, et al. Multiple organ infection and the pathogenesis of SARS. J Exp Med 2005; 202(3): 415-24. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '08a22278486e12768ce186677a6a89663d24586f',\n","  'section': 'Abstract',\n","  'text': 'In this study, we reported for the first time the existence of complemented palindrome small RNAs 26 (cpsRNAs) and proposed cpsRNAs and palindrome small RNAs (psRNAs) as a novel class of small RNAs. 27 The first discovered cpsRNA UCUUUAACAAGCUUGUUAAAGA from SARS coronavirus named SARS-28 CoV-cpsR-22 contained 22 nucleotides perfectly matching its reverse complementary sequence. Further 29 sequence analysis supported that SARS-CoV-cpsR-22 originated from bat betacoronavirus. The results of 30 RNAi experiments showed that one 19-nt segment of SARS-CoV-cpsR-22 significantly induced cell 31 apoptosis. These results suggested that SARS-CoV-cpsR-22 could play a role in SARS-CoV infection or 32 pathogenicity. The discovery of psRNAs and cpsRNAs paved the way to find new markers for pathogen 33 detection and reveal the mechanisms in the infection or pathogenicity from a different point of view. The 34 discovery of psRNAs and cpsRNAs also broaden the understanding of palindrome motifs in animal of plant 35 genomes. '},\n"," {'paper_id': '090b6c8b3df30bc248221869f673a2d970caa1b9',\n","  'section': 'Abstract',\n","  'text': \"In the wake of the Rohingya population's mass migration from Myanmar, one of the world's largest 10 refugee settlements was constructed in Cox's Bazar, Bangladesh to accommodate nearly 900,000 new 11 refugees. Refugee populations are particularly vulnerable to infectious disease outbreaks due to many 12 population and environmental factors. A large measles outbreak, with over 2,500 cases, occurred among 13 the Rohingya population between September and December 2017. Here, we estimate key epidemiological 14 parameters and use a dynamic mathematical model of measles transmission to evaluate the effectiveness 15 of the reactive vaccination campaigns in the refugee camps. We also estimate the potential for subsequent 16 outbreaks under different vaccination coverage scenarios. Our modeling results highlight the success of 17 the vaccination campaigns in rapidly curbing transmission and emphasize the public health importance 18 of maintaining high levels of vaccination in this population, where high birth rates and historically low 19 vaccination coverage rates create suitable conditions for future measles outbreaks. 20 \"},\n"," {'paper_id': '091a8e9a61e19e88caeb039f0e3888d111b20439',\n","  'section': 'Abstract',\n","  'text': \"Based on publicly released data for 1212 patients, we investigated the epidemiological characteristics of COVID-19 in Henan of China. The following findings are obtained: 1) COVID-19 patients in Henan show gender (55% vs 45%) and age (81% aged between 21 and 60) preferences, possible causes were explored; 2) Statistical analysis on 483 patients reveals that the estimated average, mode and median incubation periods are 7.4, 4 and 7 days; Incubation periods of 92% patients were no more than 14 days; 3) The epidemic of COVID-19 in Henan has undergone three stages and showed high correlations with the numbers of patients that recently return from Wuhan; 4) Network analysis on the aggregate outbreak phenomena of COVID-19 revealed that 208 cases were clustering infected, and various people's Hospital are the main force in treating patients. The related investigations have potential implications for the prevention and control of COVID-19. \"},\n"," {'paper_id': '09892e597bdc1ded9eb5922b4f3d41e041d6634a',\n","  'section': 'Abstract',\n","  'text': \"The novel coronavirus whose outbreak took place in December 2019 continues to spread at a rapid rate worldwide. In the absence of an effective vaccine, inhibitor repurposing or de novo drug design may offer a longer-term strategy to combat this and future infections due to similar viruses. Here, we report on detailed classical and mix-solvent molecular dynamics simulations of the main protease (Mpro) enriched by evolutionary and stability analysis of the protein. The results were compared with those for a highly similar SARS Mpro protein. In spite of a high level of sequence similarity, the active sites in both proteins show major differences in both shape and size indicating that repurposing SARS drugs for COVID-19 may be futile. Furthermore, analysis of the binding site's conformational changes during the simulation time indicates its flexibility and plasticity, which dashes hopes for rapid and reliable drug design. Conversely, structural stability of the protein with respect to flexible loop mutations indicates that the virus' mutability will pose a further challenge to the rational design of small-molecule inhibitors. However, few residues contribute significantly to the protein stability and thus can be considered as key anchoring residues for Mpro inhibitor design. \"},\n"," {'paper_id': '09b6706748f0c1ae0da436ac2dfac9052b84e4ea',\n","  'section': 'Abstract',\n","  'text': 'A new porcine coronavirus SADS-CoV was recently identified from suckling piglets 20 with severe diarrhea in southern China and its genome sequence is most identical (~95% 21 identity) to that of bat α-coronavirus HKU2. It again indicates bats are the natural 22 reservoir of many coronaviruses that have great potential for cross-species transmission 23 to animals and humans by recombination and/or mutation. Here we report the cryo-EM 24 structures of HKU2 and SADS-CoV spike glycoprotein trimers at 2.38 Å and 2.83 Å 25 resolution, respectively. HKU2 and SADS-CoV spikes exhibit very high structural 26 similarity, with subtle differences mainly distributed in the NTD and CTD of the S1 27 subunit responsible for cell attachment and receptor binding. We systematically 28 analyzed and compared the NTD, CTD, SD1 and SD2 domains of the S1 subunit and 29 the S2 subunit of HKU2 spike with those of α-, β-, γ-, and δ-coronavirus spikes. The 30 results show that the NTD and CTD of HKU2/SADS-CoV are probably the most 31 ancestral in the evolution of spike. Although the S2 subunit mediating membrane fusion 32 is highly conserved, the connecting region after fusion peptide in HKU2/SADS-CoV 33 S2 subunit also adopts a conformation distinct from other coronaviruses. These results 34 structurally demonstrate a close evolutionary relationship between HKU2 /SADS-CoV 35 and β-coronavirus spikes and provide new insights into the evolution and cross-species 36 transmission of coronaviruses. : bioRxiv preprint 39 Coronaviruses, categorized into the order Nidovirales family Coronaviridae and 40 subfamily Coronavirinae, are a large group of viral pathogens with a wide host range 1 . 41 Their infections in humans, other mammals and birds can cause respiratory, hepatic, 42 enteric and neurological diseases with varying severity 2 . The severe acute respiratory 43 syndrome coronavirus (SARS-CoV) and Middle East respiratory syndrome coronavirus 44 (MERS-CoV) have posed severe threats to human health in the 21 st century 1,3 . In the 45 meantime, coronaviruses infecting domestic animals also bring substantial economic 46 losses 4 . For example, the swine acute diarrhea syndrome coronavirus (SADS-CoV) 47 (also known as SeACoV and PEAV) isolated in 2017 caused outbreaks of severe watery 48 diarrhea of suckling piglets with a mortality up to 90% in several commercial pig farms 49 in Guangdong Province of China 5-10 . SADS-CoV is an α-coronavirus and other 50 representative members in the α-genus are porcine epidemic diarrhea virus 51 (PEDV), porcine transmissible gastroenteritis coronavirus (TGEV), porcine respiratory 52 coronavirus (PRCV), human NL63 and 229E coronaviruses (HCoV-NL63 and HCoV-53 229E) 1 . Representative members in other three genera include mouse hepatitis 54 coronavirus (MHV), bovine coronavirus (BCoV), SARS-CoV, MERS-CoV, HCoV-55 OC43 and HCoV-HKU1 in the β-genus, avian infectious bronchitis virus (IBV) in the 56 γ-genus and porcine deltacoronavirus (PdCoV) in the δ-genus 1 . 57 Cross-species transmission promoted by genetic recombination and/or mutations 58 underlies the host range expansion of coronaviruses 3,11-13 . Bats are the natural reservoir 59 of more than 30 different α-and β-coronaviruses that have great potential for 60 interspecies transmission by recombination and/or mutation 12,14-16 . Data on genetic 61 evolution, receptor binding and pathogenesis have demonstrated that human SARS-62 CoV and MERS-CoV most likely originate from bats 1 . Palm civets and dromedary 63 camels are the intermediate hosts of SARS-CoV and MERS-CoV from bats to humans 1 , 64 respectively. The newly identified porcine SADS-CoV isolates are also found to share 65 ~95% sequence identity with Rhinolophus bat coronavirus HKU2, and this further 66 stressed the severe results of coronavirus spillover from bats to domestic animals 5-10 . '},\n"," {'paper_id': '09c9fcabc66a106e01ef42247cbd86b6d85bd67f',\n","  'section': 'Abstract',\n","  'text': '24 A subset of intracellular mono-ADP-ribosyltransferases diphtheria toxin-like (ARTDs, aka 25 mono-PARPs) is induced by type I interferons. Some of these mono-ARTDs feature antiviral 26 activity while certain RNA viruses, including Chikungunya virus (CHIKV), encode mono-ADP-27 ribosylhydrolases, suggesting a role for mono-ADP-ribosylation (MARylation) in host-virus 28 conflicts. CHIKV expresses four non-structural proteins (nsP1-nsP4), with nsP3 containing a 29 macrodomain that hydrolyzes and thereby reverses protein MARylation in vitro and in cells. 30 This de-MARylation activity is essential as hydrolase inactivating mutations result in 31 replication defective virus. However, the substrates of MARylation during CHIKV infection 32 are unknown and thus it is unclear how the macrodomain contributes to virus replication 33 and how mono-ARTD-dependent MARylation confers antiviral immunity. We identified 34 ARTD10 and ARTD12 as restriction factors for CHIKV replication in a catalytic activity-35 dependent manner. CHIKV replication requires processing of the non-structural polyprotein 36 nsP1-4 by the nsP2-encoded protease and the assembly of the four individual nsPs into a 37 functional replication complex. Expression of ARTD10 and ARTD12 resulted in a reduction 38 of processed nsPs. Similarly, MAR hydrolase inactive CHIKV replicon mutants revealed a 39 decrease in processed nsPs, comparable to an nsP2 protease defective mutant. This 40 suggested that the macrodomain contributes to nsP2 protease activity. In support, a 41 hydrolase-deficient virus was complemented by a protease-deficient virus. We 42 hypothesized that MARylation regulates the proteolytic function of nsP2. Indeed, we found 43 that nsP2 is MARylated by ARTD10. This inhibited nsP2 protease activity, thereby preventing 44 polyprotein processing and consequently virus replication. This inhibition was antagonized 45 by the MAR hydrolase activity of nsP3. Together, our findings provide a mechanistic 46 explanation for the need of the viral MAR hydrolase for efficient replication of CHIKV. 47 Author Summary 48 Infectious diseases still pose major health threats. Especially fast evolving viruses find ever 49 new strategies to manipulate the immune response. With climate warming and increased 50 human mobility vector-borne pathogens like Chikungunya virus (CHIKV) spread and cause 51 world-wide epidemics. Beyond the acute phase, CHIKV patients regularly suffer from chronic 52 rheumatism. This entails a decline in life quality and an economic burden. To date no drugs 53 are approved and the mode of pathogenesis remains elusive. Here we describe a mechanistic 54 function of the CHIKV nsP3 macrodomain. We found that the viral nsP2 is mono-ADP-55 ribosylated interfering with its auto-proteolytic function. The nsP3 macrodomain removes this 56 modification and restores the protease activity that is essential for replication. Because 57 macrodomains are highly conserved they might represent broad antiviral targets. 58 59 Introduction 60 Upon viral infection host cells initiate an antiviral immune response. Because the regulation 61 of protein function through post-translational modifications (PTMs) is among the quickest 62 mechanisms to adapt to challenges such as viral infections, PTMs display an essential part of 63 antiviral signaling. Viruses have developed multifaceted strategies to evade or even hijack 64 cellular mechanisms, e.g. encoding proteins that regulate PTMs and thereby counteract the 65 antiviral reaction of the host [1-5]. Recent findings have elaborated on the role of intracellular 66 ADP-ribosylation at the host-pathogen interface [6-12]. ADP-ribosylation is an ancient PTM of 67 proteins and intracellularly mainly catalyzed by members of the ADP-ribosyltransferase 68 diphtheria toxin-like (ARTD) family (also known as poly-ADP-ribosylpolymerases (PARPs)), 69 which is composed of seventeen proteins [1, 13]. These enzymes use NAD + as co-factor to 70 transfer ADP-ribose onto a substrate protein with release of nicotinamide. Based on their 71 . : bioRxiv preprint 4 catalytic features the ARTDs are subdivided into three classes. The first class contains ARTD1, 72 2, 5, and 6, which are capable of transferring multiple ADP-ribose moieties in an iterative 73 process and thereby forming long polymers. This results in poly-ADP-ribosylation (PARylation) 74 of substrate proteins. The second and largest class is defined by members that are restricted 75 to mono-ADP-ribosylation (MARylation) (ARTD3, 4,(7)(8)(9)(10)(11)(12)(14)(15)(16)(17), whereas the third 76 group is defined by ARTD13, which lacks the ability to bind NAD + and thus is catalytically 77 inactive [18]. 78 79 PARylation, arguably best known for its function in DNA damage response, has also been 80 linked to chromatin organization, ribosome biogenesis, telomere maintenance, signaling 81 processes and cell death [1, 19, 20]. In contrast functions of MARylation are less well 82 understood but expanding with links to DNA damage repair, gene expression, signaling, stress 83 response and cell death [1, 21]. Recent findings indicate a role for MARylation in host-84 pathogen conflicts [1, 21, 22]. The expression of several mono-ARTDs is triggered by type I 85 interferons (IFNs) or pathogen-associated molecular patterns (PAMPs), like LPS, as part of an 86 innate immune response to pathogens [6, 8, 11,[23][24][25]. Among these mono-ARTDs are ARTD7, 87 ARTD10 and ARTD12, which have been identified to mediate clearance of Venezuelan Equine 88 Encephalitis Virus (VEEV) by interfering with viral replication [7, 8]. In line with this, Zika virus 89 replication is restricted by ARTD12 [26] . In addition to these molecular and cell-based studies, 90 evolutionary analysis suggests a role for several ARTD family members at the host-pathogen 91 interface, among them the macrodomain containing ARTDs (ARTD7-9) and ARTD13 [27]. 92 Although catalytically inactive, the latter is best studied for being restrictive for viral 93 replication by recognizing foreign CG-rich RNA through its Zinc finger domains [28, 29]. 94 95 . CC-BY-NC-ND 4.0 '},\n"," {'paper_id': '09fd48d0e1f60fa69b68ebd54bd5d71fc08dec96',\n","  'section': 'Abstract',\n","  'text': 'We analyzed age-specific and sex-specific morbidity and mortality data from SARS-COV-2 pandemic in China and Republic of Korea (ROK). Data from China exhibit a standard Gaussian distribution with peak morbidity in the 50-59 years cohort, while the ROK data have a bimodal distribution with highest morbidity in the 20-29 years cohort. '},\n"," {'paper_id': '0a27cb2cd52229472fcfc3e49d3a3cb7179867e4',\n","  'section': 'Abstract',\n","  'text': 'Strict interventions were successful to control the novel coronavirus (COVID-19) outbreak in China. As transmission intensifies in other countries, the interplay between age, contact patterns, social distancing, susceptibility to infection and disease, and COVID-19 dynamics remains unclear. To answer these questions, we analyze contact surveys data for Wuhan and Shanghai before and during the outbreak and contact tracing information from Hunan Province. Daily contacts were reduced 7-9 fold during the COVID-19 social distancing period, with most interactions restricted to the household. Children 0-14 years were 59% (95% CI 7-82%) less susceptible than individuals 65 years and over. A transmission model calibrated against these data indicates that social distancing alone, as implemented in China during the outbreak, is sufficient to control COVID-19. While proactive school closures cannot interrupt transmission on their own, they reduce peak incidence by half and delay the epidemic. These findings can help guide global intervention policies. '},\n"," {'paper_id': '0a2a28cb82e7a03af0a9fad4fd4c68c9fdac2477',\n","  'section': 'Abstract',\n","  'text': 'The discovery of epitopes is helpful to the development of SARS-CoV-2 vaccine. The sequences of the surface protein of SARS-CoV-2 and its proximal sequences were obtained by BLAST, the sequences of the whole genome of SARS-CoV-2 were obtained from the GenBank. Based on the NCBI Reference Sequence: NC_045512.2, the conformational and linear B cell epitopes of the surface protein were predicted separately by various prediction methods. Furthermore, the conservation of the epitopes, the adaptability and other evolutionary characteristics were also analyzed. 7 epitopes were predicted, including 5 linear epitopes and 2 conformational epitopes, one of the linear and one of the conformational were coincide. The epitope D mutated easily, but the other epitopes were very conservative and the epitope C was the most conservative. It is worth mentioning that all of the 6 dominated epitopes were absolutely conservative in nearly 1000 SARS-CoV-2 genomes, and they deserved further study. The findings would facilitate the vaccine development, had the potential to be directly applied on the treatment in this disease, but also have the potential to prevent the possible threats caused by other types of coronavirus. Results The primary structure and physicochemical properties of the S/E/M protein were analyzed. The results revealed that the S protein has an average hydrophilic index of -author/funder. All rights reserved. No reuse allowed without permission. : bioRxiv preprint 0.079( Figure S1A ). On the basis of hydrophilicity, it also showed amphoteric properties. There was an outside-in transmembrane helix in 23 residues from position 1214 th to position 1236 th at the N-terminal( Figure S2A ). The protein instability index was 33.01, which revealed the S protein was stable. The E protein has an average hydrophilic index of 1.128( Figure S1B ). It was hydrophobic. An inside-out transmembrane helix in 23 residues from position 12 th to position 34 th at the N-terminal was predicted( Figure S2B ). The protein instability index was 38.68, which revealed the E protein was stable. The M protein has an average hydrophilic index of 0.446( Figure   S1C ). On the basis of hydrophobicity, it also showed amphoteric properties. There were two outside-in transmembrane helices, one was in 20 residues from position 20 th to position 39 th , the another one was in 23 residues from position 78 th to position 100 th , and an inside-out transmembrane helix in 20 residues from position 51 st to position 73 rd , at the N-terminal( Figure S2C ). The protein instability index was 39.14, which revealed the M protein was stable. The optimal template for homology modeling of the S protein of SARS-CoV-2 was the S protein of SARS(PDB ID: 6acc.1), with the sequence identity of 76.47% and the GMQE score of 0.73. According to the evaluation of the structure by Ramachandran plot (Figure 1A) , 99.3% of the residues were located in the most favoured regions and the allowed regions, 0.7% of the residues were located in the disallowed regions, the high-energy regions (Table 1) , which was possibly due to some energy was spent in the protein processing to make these residues enter the high-energy regions [5] . The result generally showed that the structure was reliable. The structure of S protein( Figure 1B) of SARS-CoV-2 is a trimer, which can be divided into a tightly curled tail and a distributed head. The head is mainly composed of β-sheet, irregular curl and turn, which is exposed to the envelope of the virus, contributes to the formation of epitopes. The tail is mainly composed of several α-helices, part of which is embedded in the envelope, hinders the formation of epitopes. The optimal template for homology modeling of the E protein of SARS-CoV-2 was the E protein of SARS(PDB ID: 5x29.1), with the sequence identity of 91.38% and author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '0a43046c154d0e521a6c425df215d90f3c62681e',\n","  'section': 'Abstract',\n","  'text': '300 words) 33 Quantification of aerosolized influenza virus is used for determining inhalation exposure. Several 34 bioaerosol samplers and analytical methods have been used; however, the detection and 35 quantification of influenza virus among aerosol samples remains challenging. Therefore, 36 improved viral aerosol measurement methods are needed. This study evaluated influenza virus 37 recovery among three filter types polytetrafluoroethylene, polyvinylchloride and polystyrene. 38 Polytetrafluoroethylene, polyvinylchloride are fabricated filter materials and commonly used in 39 the scientific literature to sample for viral aerosols. A novel, electrospun polystyrene filter 40 material may improve viral aerosol recovery during filter-based air sampling. The filter materials 41 were compared across the following conditions: treated with or without air, filter wash buffer 42 (HBSS or PBS), and viral RNA extraction method (QIAamp Viral RNA Mini Kit or Trizol). 43 Twenty trials were completed in a chamber and samples were analyzed using RT-qPCR. Viral 44 recovery was significantly different (p-value < .0001) by filter type. Polystyrene filter use 45 resulted in recovery of the most viral RNA. Air sampling did not affect the recovery of viral 46 RNA from the filter materials (p-values > 0.05). Viral RNA concentrations were significantly 47 different across extraction methods for all comparisons (p-values < 0.05). Our results 48 demonstrated that the novel polystyrene filter material resulted in the highest concentration of 49 extracted RNA compared to the commonly used polytetrafluoroethylene and polyvinylchloride, 50 which we speculate may be related to the chemical composition of the filter material (e.g., 51 polystyrene is an aromatic hydrocarbon whereas polytetrafluoroethylene and polyvinylchloride 52 contain more polar, and thus potentially reactive, carbon-halogen bonds). Air sampling did not 53 have an effect on viral RNA recovery. Using Hanks Balanced Salt Solution with QIAamp Viral . CC-BY 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/441154 doi: bioRxiv preprint 54 RNA Mini Kit, and Phosphate-buffered saline with the Trizol extraction, resulted in the most 55 viral RNA recovery. 56 57 58 '},\n"," {'paper_id': '0a54bc995e2c2b9bd0eff65aa7c00815fc16008d',\n","  'section': 'Abstract',\n","  'text': \"The aim of the present work is to estimate capacities of Brazil's health system and their demand as a result of predicted incoming severe cases of the novel coronavirus disease outbreak. Three incrementing levels of hospital equipment usage are considered: (1) in terms of available intensive care unit (ICU) beds; (2) available ICU beds and existing surgery operating rooms; and (3) available ICU beds and existing surgery operating rooms and respirators located in other hospital areas. The average available (adult) ICU beds in a hospital has been computed deducting from the number of existing beds the equivalent number of beds that has been demanded during 2019 on a monthly average. Based on a reference mean duration of hospitalization for the disease, it is computed the daily admission capacity of infected patients per state and of the entire country for each of the three referred levels of hospital equipment usage. Furthermore, an exponential regression model is fitted using the daily available data for the number of patients who have been documented as infected in Brazil. It can then be predicted the three dates at which the health system might begin to demonstrate stress for the maximum demand of the three incrementing groups of hospital equipments. The necessary public data are analyzed by means of the Python programming language and are made available on the internet by the \"},\n"," {'paper_id': '0ab795fc615df6457551a8e231dce1f268eef9d2',\n","  'section': 'Abstract',\n","  'text': 'With the emergence of 4 rd generation transmission, the prevention and treatment of the novel coronavirus disease 2019 has entered a new period. We aimed to report several changes in the clinical characteristics at admission of patients with COVID-19. Clinical records and laboratory results of patients suffering from COVID-19 were retrospectively reviewed and matched with the admission dates to analyze the changes in characteristics at the onset of illness. . Of the 89 affected patients, 31 [34.8%] patients were admitted from January 16 to 22, and 58 [65.2%] were admitted from January 23 to 29. Patients were admitted with more systemic symptoms, such as fever (21 [67.7%] of 31), fatigue (13 [41.9%] of 31), and myalgia (7 [22.6%] of 31), before January 23. More patients (10 [32.3%] of 31) admitted before January 23 had a small amount of sputum production compared with a smaller proportion (4 [6.9%] of 58) of the patients admitted after January 23. Other symptoms, such as cough, nausea, diarrhea, and chest tightness, were not significantly different between the two groups. In addition, the group admitted before January 23 had a larger proportion of patients with reduced lymphocyte (13 [54.2%] of 24), CD3 (11 [54.4%] of 21), and CD8 (9 [42.9%] of 21) counts and elevated serum amyloid A (SAA, 18 [75%] of 24). Conclusions. The initial symptoms of recently infected patients seem more insidious, indicating that the new coronavirus may gradually evolve into a virus similar to influenza and latent in asymptomatic carriers for a long time. '},\n"," {'paper_id': '0acc1f9a1c333a9a6b2dbba4a252d7576f024783',\n","  'section': 'Abstract',\n","  'text': 'Motivation: Understanding the relationship between the sequence, structure, binding energy, binding kinetics and binding thermodynamics of protein-protein interactions is crucial to understanding cellular signaling, the assembly and regulation of molecular complexes, the mechanisms through which mutations lead to disease, and protein engineering. Results: We present SKEMPI 2.0, a major update to our database of binding free energy changes upon mutation for structurally resolved protein-protein interactions. This version now contains manually curated binding data for 7085 mutations, an increase of 133%, including changes in kinetics for 1844 mutations, enthalpy and entropy changes for 443 mutations, and 440 mutations which abolish detectable binding. Availability: The database is available at '},\n"," {'paper_id': '0aff00101d5ccc6592987185ab833f95d842f98b',\n","  'section': 'Abstract',\n","  'text': 'The World Health Organization (WHO) has declared the 2019 novel coronavirus (2019-nCoV) infection outbreak a global health emergency. Currently, there is no effective anti-2019-nCoV medication. The sequence identity of the 3CL proteases of 2019-nCoV and SARS is 96%, which provides a sound foundation for structural-based drug repositioning (SBDR). Based on a SARS 3CL protease X-ray crystal structure, we construct a 3D homology structure of 2019-nCoV 3CL protease. Based on this structure and existing experimental datasets for SARS 3CL protease inhibitors, we develop an SBDR model based on machine learning and mathematics to screen 1465 drugs in the DrugBank that have been approved by the U.S. Food and Drug Administration (FDA). We found that many FDA approved drugs are potentially highly potent to 2019-nCoV. '},\n"," {'paper_id': '0b282573f5c63c943021c10ca39a1ed21acfb429',\n","  'section': 'Abstract',\n","  'text': 'Two months after it was firstly reported, the novel coronavirus disease COVID-19 has already spread worldwide. However, the vast majority of reported infections have occurred in China. To assess the effect of early travel restrictions adopted by the health authorities in China, we have implemented an epidemic metapopulation model that is fed with mobility data corresponding to 2019 and 2020. This allows to compare two radically different scenarios, one with no travel restrictions and another in which mobility is reduced by a travel ban. Our findings indicate that i) travel restrictions are an effective measure in the short term, however, ii) they are ineffective when it comes to completely eliminate the disease. The latter is due to the impossibility of removing the risk of seeding the disease to other regions. Our study also highlights the importance of developing more realistic models of behavioral changes when a disease outbreak is unfolding. '},\n"," {'paper_id': '0b460e73926eb107001c95c0dfeae2362b4251b0',\n","  'section': 'Abstract',\n","  'text': 'With the ongoing outbreak of Coronavirus Disease 2019 , infected patients within and beyond the epidemic area, Wuhan, China, showed different epidemiological and clinical characteristics. There is a paucity of data concerning coinfection with other common respiratory pathogens in COVID-19 patients outside of Wuhan. We conducted a double-centre study recruiting 68 patients with severe acute respiratory coronavirus 2 (SARS-CoV-2) infection confirmed by nucleic acid testing in Qingdao and Wuhan from January 17 to February 16, 2020. Indirect immunofluorescence was performed to detect the specific IgM antibody against common respiratory pathogens in collected acute phase serum. : Of the 68 patients with SARS-CoV-2 infection, 30 (44.12%) were from Qingdao. The median age of Qingdao and Wuhan patients were 50 (IQR: 37-59) and 31 (IQR: 28-38) years, respectively, and the majority of patients were female in Qingdao (60.00%) and Wuhan (55.26%). Among COVID-19 patients in Qingdao, 24 (80.00%) of them had IgM antibodies against at least one respiratory pathogen, whereas only one (2.63%) of the patients in Wuhan had positive results for serum IgM antibody detection (P<0.0001). The most common respiratory pathogens detected in Qingdao COVID-19 patients were influenza virus A (60.00%) and influenza virus B (53.33%), followed by mycoplasma pneumoniae (23.33%) and legionella pneumophila (20.00%). While the pattern for coinfection in patients with community-acquired pneumonia in Qingdao was quite different, with a positive rate of only 20.90%. Interpretation: We reported a large proportion of COVID-19 patients with coinfection of seasonal respiratory pathogens in Qingdao, northeast China, which differed greatly from the patients in Wuhan, central China. Precautions are needed when dealing with COVID-19 patients beyond the epidemic centre who have coinfection with other respiratory pathogens. We highly recommend adding SARS-CoV-2 to routine diagnostic testing in capable hospitals to prevent misdetection of the virus. '},\n"," {'paper_id': '0b4b4e5bb8d0d5167eec1e203b5dad283bd364a5',\n","  'section': 'Abstract',\n","  'text': 'doi: bioRxiv preprint 106 components. ALLAN and MET, by contrast, are more variable and very close to each other and to 107 untreated control (Fig. 2b, Extended data Fig. 4c). 108 109 Drug synergies and lifespan extension 110 We next determine effect of different drug combinations on lifespan in nematodes. We began with 111 MET and RAP due to their translational importance in humans 14,27 . However, RAP combined with 112 MET at their optimal doses did not show any further benefit on lifespan (Extended data Fig. 2j,p). We 113 carried out additional trials, exploring all possible combinations of optimal and half-optimal dose for 114 MET and RAP (Extended data Fig. 2j-p). Combination of RAP and MET each at their half-optimal 115 dose result in further extension in maximum lifespan (Fig. 2e, Extended data Fig. 2m,s) but did not 116 result in further mean lifespan extension (Extended data 2p). These weak benefits of combining MET 117 with RAP are consistent with results in mice where the existing evidence suggests that addition of Figures S3 lifespan comparison of triple combinations supplementary for figure 2. a, RAP+RIF+PSORA did not show further benefit compared to the dual synergistic combinations RAP+RIF and RIF+PSORA. b, RAP+RIF+MET is toxic compared to RAP+RIF. More than 50 worms per condition.. c, RAP+PSORA+ALLAN did not show further benefit compared to single drug constituents. d, RAP+RIF+ALLAN and RIF+PSORA+ALLAN resulted in a significant lifespan extension compared to RAP+RIF and RIF+PSORA respectively. log-rank with proper adjustment. e, Mean and f, Maximum lifespan of figure d showed a monotonic increase from single drugs to double and triple combinations. One-way ANOVA, Bonferroni multiple comparison *doi: bioRxiv preprint 235 12 Ejsing, C. S. et al. Collision-induced dissociation pathways of yeast sphingolipids and their 236 molecular profiling in total lipid extracts: a study by quadrupole TOF and linear ion trap-237 orbitrap mass spectrometry. '},\n"," {'paper_id': '0bd443591b5e10934beef050b516afeca6b668fe',\n","  'section': 'Abstract',\n","  'text': 'Since COVID-19 emerged in early December, 2019 in Wuhan and swept across China Mainland, a series of large-scale public health interventions, especially Wuhan lock-down combined with nationwide traffic restrictions and Stay At Home Movement, have been taken by the government to control the epidemic. Based on Baidu Migration data and the confirmed cases data, we identified two key factors affecting the later (e.g February 27, 2020) cumulative confirmed cases in non-Wuhan region (y). One is the sum travelers from Wuhan during January 20 to January 26 (x1), which had higher infected probability but lower transmission ability because the human-to-human transmission risk of COVID-19 was confirmed and announced on January 20. The other is the \"seed cases\" from Wuhan before January 19, which had higher transmission ability and could be represented with the confirmed cases before January 29 (x2) due to a mean 10-day delay between infection and detection. A simple yet effective regression model then was established as follow: y= 70.0916+0.0054×x1+2.3455×x2 (n = 44, R 2 = 0.9330, P<10 -7 ). Even the lock-down date only delay or in advance 3 days, the estimated confirmed cases by February 27 in non-Wuhan region will increase 35.21% or reduce 30.74% -48.59%. Although the above interventions greatly reduced the human mobility, Wuhan lock-down combined with nationwide traffic restrictions and Stay At Home Movement do have a determining effect on the ongoing spread of COVID-19 across China Mainland. The strategy adopted by China has changed the fast-rising curve of newly diagnosed cases, the international community should learn from lessons of Wuhan and experience from China. Efforts of 29 Provinces and 44 prefecture-level cities against COVID-19 were also assessed preliminarily according to the interpretive model. Big data has played and will continue playing an important role in public health. '},\n"," {'paper_id': '0c27c0ddaa4761f6155838df81a88d24619720f8',\n","  'section': 'Abstract',\n","  'text': 'Genome Detective is a web-based, user-friendly software application to quickly and accurately assemble all known virus genomes from next generation sequencing datasets. This application allows the identification of phylogenetic clusters and genotypes from assembled genomes in FASTA format. Since its release in 2019, we have produced a number of typing tools for emergent viruses that have caused large outbreaks, such as Zika and Yellow Fever Virus in Brazil. Here, we present The Genome Detective Coronavirus Typing Tool that can accurately identify novel coronavirus (2019-nCoV) sequences isolated in China and around the world. The tool can accept up to 2,000 sequences per submission and the analysis of a new whole genome sequence will take approximately one minute. The tool has been tested and validated with hundreds of whole genomes from ten coronavirus species, and correctly classified all of the SARS-related coronavirus (SARSr-CoV) and all of the available public data for 2019-nCoV. The tool also allows tracking of new viral mutations as the outbreak expands globally, which may help to accelerate the development of novel diagnostics, drugs and vaccines. '},\n"," {'paper_id': '0cad40f50e41144e8b64744251515eca8cfac5f7',\n","  'section': 'Abstract',\n","  'text': \"In this paper, we estimate the reproductive number R 0 of COVID-19 based on Wallinga and Lipsitch framework [11] and a novel statistical time delay dynamic system. We use the observed data reported in CCDC's paper to estimate distribution of the generation interval of the infection and apply the simulation results from the time delay dynamic system as well as released data from CCDC to fit the growth rate. The conclusion is: Based our Fudan-CCDC model, the growth rate r of COVID-19 is almost in [0.30, 0.32] which is larger than the growth rate 0.1 estimated by CCDC [9], and the reproductive number R 0 of COVID-19 is estimated by 3.25 ≤ R 0 ≤ 3.4 if we simply use R = 1 + r * T c with T c = 7.5, which is bigger than that of SARS. Some evolutions and predictions are listed. \"},\n"," {'paper_id': '0cb9c296684ca5e71462d825cab2827854a01544',\n","  'section': 'Abstract',\n","  'text': 'Summary Statement: DUX4 is thought to mediate cytopathology through p53. Here, DUX4 is shown to kill primary myoblasts and promote pathological phenotypes in the iDUX4[2.7] mouse model on the p53-null background, calling into question this notion. FSHD is a genetically dominant myopathy caused by mutations that cause expression of the normally silent DUX4 gene. This transcription factor has been shown to interfere with myogenesis when misexpressed at very low levels in myoblasts, and to cause cell death when overexpressed at high levels. A previous report using adeno-associated virus to deliver high levels of DUX4 to mouse skeletal muscle demonstrated severe pathology that was suppressed on a p53 knockout background, implying that DUX4 acted through the p53 pathway. Here, we investigate the p53-dependence of DUX4 using both in vitro cellular and the transgenic iDUX4[2.7] mouse models. We find that inhibiting p53 has no effect on the cytoxicity of DUX4 in vitro. When crossed onto the p53 null background, we find no suppression of the male-specific lethality or skin phenotypes of the DUX4 transgene, and find that primary myoblasts from this mouse are still killed by DUX4 expression. These data challenge the notion that the p53 pathway is central to the pathogenicity of DUX4. 166 words. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '0d0bb3af5f5665e4b529c7307b56ea40202ebc5d',\n","  'section': 'Abstract',\n","  'text': 'We used moving regression to model the growth rate (cases/day) and acceleration (cases/day²) of COVID-19 cases in 123 countries as of March 25 th 2020. In countries entering stationary growth (China and South Korea) decline in acceleration was observable up to 1 week after severe restriction to human mobility was adopted as a preventive measure. Deceleration was detectable within 2 weeks, whereas stationary growth was reached within 6 weeks. These results corroborate that mass social isolation is a highly effective measure against the dissemination of SARS-CoV-2. We also found that the impact of public health measures could be evaluated in seemingly real-time by monitoring COVID-19 growth curves. Moreover, reasonable daily predictions of new cases were obtained (R² ~ 0.95). Apart from the analysis of prevalence partitioned by country, moving regression can be easily applied to city, state, region or arbitrary territory data to help monitoring the local behavior of COVID-19 cases. '},\n"," {'paper_id': '0d4432c32daefa6f0d711ca35eca81188e0bccdf',\n","  'section': 'Abstract',\n","  'text': '17 years after the SARS-CoV epidemic, the world is facing the COVID-19 pandemic. COVID-19 is caused by a coronavirus named SARS-CoV-2. Given the most optimistic projections estimating that it will take more than a year to develop a vaccine, our best short term strategy may lie in identifying virus-specific targets for small molecule interventions. All coronaviruses utilize a molecular mechanism called -1 PRF to control the relative expression of their proteins. Prior analyses of SARS-CoV revealed that it utilizes a structurally unique three-stemmed mRNA pseudoknot to stimulate high rates of -1 PRF, that it also harbors a -1 PRF attenuation element. Altering -1 PRF activity negatively impacts virus replication, suggesting that this molecular mechanism may be therapeutically targeted. Here we present a comparative analysis of the original SARS-CoV and SARS-CoV-2 frameshift signals. Structural analyses reveal that the core -1 PRF signal, composed of the U UUA AAC slippery site and three-stemmed mRNA pseudoknot is highly conserved. In contrast, the upstream attenuator hairpin is less well conserved. Functional assays revealed that both elements promote similar rates of -1 PRF and that silent coding mutations in the slippery site strongly ablate -1 PRF activity. We suggest that molecules that were previously identified as inhibiting SARS-CoV mediated -1 PRF may serve as lead compounds to counter the current pandemic. '},\n"," {'paper_id': '0d7964c665ff0107c674bc6dab8f252cab08038e',\n","  'section': 'Abstract',\n","  'text': \"Honey bees are eusocial insects, living in a colony that is often described as a superorganism. RNA mobility among cells of an organism has been documented in plants and animals. Here we show that RNA spreads further in honey bees, and is horizontally transferred between individuals and across generations. We found that honey bees share biologically active RNA through secretion and ingestion of worker and royal jellies. Such RNA initiates RNA interference, which is a known defense mechanism against viral infection. Furthermore, we characterized diverse RNA profiles of worker and royal jelly, including fragmented viral RNA. Our findings demonstrate a transmissible RNA pathway with potential roles in social immunity and epigenetic signaling among members of the hive. One of the characteristics of RNA interference (RNAi) is systemic spread of the silencing signal among cells and tissues throughout the organism. Systemic RNAi, initiated by double-stranded RNA (dsRNA) ingestion, has been reported in diverse invertebrates, including honey bees, demonstrating environmental RNA uptake that undermines homologous gene expression. However, the question why any organism would take up RNA from the environment has remained largely unanswered. Here, we report on horizontal RNA flow among honey bees mediated by secretion and ingestion of worker and royal jelly diets. We show that ingested dsRNA spreads through the bee's hemolymph associated with a protein complex. The systemic dsRNA is secreted with the jelly and delivered to larvae via ingestion. Furthermore, we demonstrate that transmission of jelly-secreted dsRNA to larvae is biologically active and triggers gene knockdown that lasts into adulthood. Finally, RNA extracted from worker and royal jellies harbor differential naturally occurring RNA populations. Some of these RNAs corresponded to honey bee protein coding genes, transposable elements, non-coding RNA as well as bacteria, fungi and viruses. These results reveal an inherent property of honey bees to share RNA among individuals and generations. Thus, our findings suggest a transmissible RNA pathway, playing a role in social immunity and epigenetic signaling between honey bees and potentially among other closely interacting organisms. \"},\n"," {'paper_id': '0da2ec30d7dfdef624833a36890f0297f19d09ec',\n","  'section': 'Abstract',\n","  'text': 'Compound heterozygotes occur when different mutations at the same locus on both maternal and paternal chromosomes produce a recessive trait. Here we present the tool VarCount for the quantification of mutations at the individual level. We used VarCount to characterize compound heterozygous coding variants in patients with epileptic encephalopathy and in the 1000 genomes participants. The Epi4k data contains variants identified by whole exome sequencing in patients with either Lennox-Gastaut Syndrome (LGS) or Infantile Spasms (IS), as well as their parents. We queried the Epi4k dataset (264 trios) and the phased 1000 genomes data (2504 participants) for recessive variants. To assess enrichment, transcript counts were compared between the Epi4k and 1000 genomes participants using minor allele frequency (MAF) cutoffs of 0.5% and 1.0%, and including all ancestries or only probands of European ancestry. In the Epi4k participants, we found enrichment for rare, compound heterozygous mutations in six genes, including three involved in neuronal growth and development -PRTG (p=0.00086, 1% MAF, combined ancestries), TNC (p=0.0221% MAF, combined ancestries), and MACF1 (p=0.0245, 0.5% MAF, EU ancestry). Due the total number of transcripts considered in these analyses, the enrichment detected was not significant after correction for multiple testing and higher powered or prospective studies are necessary to validate the candidacy of these genes. However, PRTG, TNC, and MACF1 are potential novel recessive epilepsy genes and our results highlight that compound heterozygous mutations should be considered in sporadic epilepsy. '},\n"," {'paper_id': '0e154f30a23e2d11bc5674354f3f7c74fb6a0008',\n","  'section': 'Abstract',\n","  'text': 'COVID-19 (SARS-CoV-2 '},\n"," {'paper_id': '0e2ba405f636e06821999876cd82c00e26404b43',\n","  'section': 'Abstract',\n","  'text': \"Members of the Nidovirales order have (+)RNA genomes amongst the largest in size in the RNA virus world. Expression of their genes is promoted through reading of genomic RNA and mRNA transcripts by the ribosome of the infected cell. The 5'-end of these RNAs is supposedly protected by an RNA-cap structure (m7GpppNm) whose most synthesis steps remain elusive. In Eukaryotes, the RNA-cap structure is methylated by RNA methyltransferases (MTases) at the RNA-cap N7-guanine position as well as the 2'-O methyl position of the first transcribed nucleotide. In Coronaviridae, two separate enzymes (nsp14 and nsp16) perform the N7-guanine and the 2'-OH methylation, respectively. One salient feature of the Nidovirales N7-guanine MTase nsp14 is that it is the only example of non-Rossman fold viral MTase known so far. Conversely, all other Nidovirales nsp16-like MTases have a canonical Rossman fold. Many Nidovirales members lack either any RNA MTase signature sequence (e.g., Arteriviridae), or lack a N7-guanine MTase signature sequence (e.g., Tobaniviridae, Euroniviridae, Roniviridae, Medioniviridae). Both nsp14-and nsp16-like enzyme genes are usually located in Orf1b encoding for the replication machinery. Here, we report the discovery of a putative Rossman fold RNA MTase in the Orf1a of ten Tobaniviridae members. Multiple sequence alignments and structural analyses identify this novel gene as a typical RNA-cap N7-guanine MTase with substrate specificity and active-site organization similar to the canonical eukaryotic RNA-cap N7-guanine MTase. author/funder. All rights reserved. No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/639369 doi: bioRxiv preprint Up to recently, four viral families, Arteriviridae, Coronaviridae, Roniviridae, and Mesoniviridae constituted the Nidovirales order. The Coronaviridae family was further divided in two subfamilies, Coronavirinae and Torovirinae. The Torovirinae family contained two genus Torovirus and Bafinivirus, plus few unassigned species. Recently, the ICTV proposed a novel classification into the Nidovirales order, with the creation of nine families [1] . In this novel ICTV proposal, ratified on February 2019, Coronaviridae do not encompass both Coronavirinae and Toronavirinae subfamilies. Rather, the new suborder Tornidovirinae (including toroviruses and bafiniviruses) contains a novel family, Tobaniviridae, at the same level as the Coronaviridae, which is now within suborder Cornidovirineae. This new taxonomy is used here, as depicted in Fig. 1 . Nidovirales have attracted much attention in the past decades for being causative agents of serious diseases in humans and animals. In humans, Severe Acute Respiratory Syndrome (SARS) and the Middle-East Respiratory Syndrome (MERS) coronaviruses (CoVs) have been associated with high fatality/case ratios of ∼10% and ∼35 %, respectively (reviewed in [2]). In animals, pathogens of significant economical and societal impact include Equine Arteritis virus (EAV, Alphaarterivirus equid) and Porcine Reproductive Respiratory Syndrome virus (PRSSV 1-2, Betaarterivirus suid 1-2). Two salient features of these viruses are, amongst others, their complex and still obscure replication/transcription mechanism as well as their broad genome-size range. Genomes of Arteriviridae (12.7 -15.7 kb in length), Mesoniviridae (∼20 kb), Medioniviridae (20.2 -25 kb), Euroniviridae (∼24.5 kb), Roniviridae (∼26 kb) Tobaniviridae and Coronaviridae (∼27-32 kb), Abyssoviridae (35.9 kb), and Mononiviridae (41.1 kb) members, share a similar genomic organisation generating viral mRNAs from a fixed genomic location, the 'nest', which has inspired the family name (Nido : latin for 'nest') [3, 4] . Nidovirales genomic and mRNAs are presumably capped, although direct demonstration of the presence of an RNA cap structure is still missing for many of these Nidovirales members. In Eukaryotes, the RNA cap structure is thought to be synthesized post-transcriptionally [5] . Most of (+)RNA viruses supposedly follow a conventional RNA capping pathway [6] in which nascent viral 5'-triphosphate genomic (or mRNAs) is thought to be processed through three enzymatic reactions to yield an RNA cap whose structure is indistinguishable from that of cellular mRNAs. The capping pathway implies that the 5'-triphosphate RNA is hydrolyzed into 5'-diphosphate by an RNA triphosphatase, a GMP residue (the « cap », coming from GTP) is next covalently transferred to the 5'-diphosphate RNA in the 5' to 5' orientation by a guanlylyltransferase (GTase), releasing inorganic pyrophosphate. Both the cap and the first transcribed nucleotide are then methylated at the N7-guanine (mGpppN-RNA, so-called Cap0 structure) and the 2'-oxygen position (mGpppNm-RNA, Cap1 structure) by one or two RNA methyltransferases (MTases), respectively. The MTase uses S-Adenosyl-Methionine (SAM) as the methyl donor and releases S-Adenosyl-homocysteine (SAH). There is no high-resolution structural analysis available of Nidovirales RNA caps. The evidence for their presence comes from immunological detection in Torovirus genomic and mRNAs [7] , and co-migration analysis in low-resolution chromatographic techniques for MHV [8] and Simian Hemorragic fever virus [9] . In other instances, Nidovirales cap structures supposedly conform to the canonical eukaryotic RNA capping pathway because enzymes usually deemed necessary to perform RNA capping are present in cognate genomes (reviewed in [6]). Bioinformatic analyses have been used to detect signature-sequences typically associated with RNA capping/modifying enzymes i.e guanlylyltransferase (GTase), polymerase, helicase, nuclease, and methyltransferase (MTase). These signature-sequences have served in many instances to isolate corresponding enzymes endowed with the predicted activity. In Nidovirales, this approach has been pioneered for Coronavirus enzymes, yet author/funder. All rights reserved. No reuse allowed without permission. \"},\n"," {'paper_id': '0e68648eb34965522da908cb05869e0176fd4307',\n","  'section': 'Abstract',\n","  'text': 'Seasonal temperature variation may impact the trajectories of COVID-19 in different global regions. Cumulative data reported by the World Health Organization, for dates up to March 27, 2020 1 , show association between COVID-19 incidence and regions at or above 30 o latitude. Historic climate data also show significant reduction of case rates with mean maximum temperature above approximately 22.5 degrees Celsius. Variance at the local level, however, could not be well explained by geography and temperature. These preliminary findings support continued countermeasures and study of SARS-CoV-2/COVID-19 transmission rates by temperature and humidity. '},\n"," {'paper_id': '0e88d4fb23740a8d59ec3b36256db8f99231929d',\n","  'section': 'Abstract',\n","  'text': 'This study shows that the disease free equilibrium ( ) for COVID-19 coronavirus does not satisfy the criteria for a locally or globally asymptotic stability. This implies that as a pandemic as declared by WHO (2020) the COVID-19 coronavirus does not have a curative vaccine yet and precautionary measures are advised through quarantine and observatory procedures. Also, the Basic Reproductive number ( 0 < 1) by Equation (33) shows that there is a chance of decline of secondary infections when the ratio between the incidence rate in the population and the total number of infected population quarantined with observatory procedure. The effort to evaluate the disease equilibrium shows that unless there is a dedicated effort from government, decision makers and stakeholders, the world would hardly be reed of the COVID-19 coronavirus and further spread is eminent and the rate of infection will continue to increase despite the increased rate of recovery because of the absence of vaccine at the moment. '},\n"," {'paper_id': '0eda8331214ca028350b07e2953702f3078a105e',\n","  'section': 'Abstract',\n","  'text': 'Estimation of the fraction and contagiousness of undocumented novel coronavirus infections is critical for understanding the overall prevalence and pandemic potential of this disease. Many mild infections are typically not reported and, depending on their contagiousness, may support stealth transmission and the spread of documented infection. Here we use observations of reported infection and spread within China in conjunction with mobility data, a networked dynamic metapopulation model and Bayesian inference, to infer critical epidemiological characteristics associated with the emerging coronavirus, including the fraction of undocumented infections and their contagiousness. We estimate 86% of all infections were undocumented (95% CI: [82%-90%]) prior to the Wuhan travel shutdown (January 23, 2020). Per person, these undocumented . CC-BY 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': '0ee75b3dd25efc988f0171d5cc9b3586c5ab46c3',\n","  'section': 'Abstract',\n","  'text': 'This paper deals with an advanced analytical epidemic diffusion model which is capable to predict the status of epidemic impacts. This newly propose model well describes an epidemic growth and it could be widely applied into various topics including pathology, epidemiology, business and data sciences. The Advanced Analytical Epidemic Diffusion Model (AAEDM) is a dynamic diffusion prediction model which is theoretically intuitive and its tractable closed formula could be easily adapted into versatile Bigdata driven analytics including the machine learning system. This dynamic model is still an analytical model but the periods of prediction are segmented for adapting the values from the dataset when the data is available. The epidemiologically vital parameters which effect on the AAEDM are also introduced in this paper. The evaluation of this theoretical model based on the Covid-19 data in Korea has been accomplished with relative fair future prediction accuracies. Although this analytical model has been designed from a basic exponential growth model, the performance of the AAEDM is competitive with other Bigdata based simulation models. Since the AAEDM is relatively simple and handy, anyone can use this model into analyzing outbreak situations in his daily life. '},\n"," {'paper_id': '0eee9760aee0ca2fdf8d1d215c3689d5f8d84df5',\n","  'section': 'Abstract',\n","  'text': 'Background With the spread of COVID-19 from Wuhan, Hubei Province to other areas of the country, medical staff in Fever Clinics faced the challenge of identifying suspected cases among febrile patients with acute respiratory infections. We aimed to describe the prevalence and clinical features of COVID-19 as compared to pneumonias of other etiologies in a Fever Clinic in Beijing. '},\n"," {'paper_id': '0f1faecb3cb3cf441c0092fa858da8616c274a73',\n","  'section': 'Abstract',\n","  'text': \"The novel coronavirus, COVID-19, has sparked an outflow of scientific research seeking to understand the virus, its spread, and best practices in prevention and treatment. If this international research effort is going to be as swift and effective as possible, it will need to rely on a principle of open science. When researchers share data, code, and software and generally make their work as transparent as possible, it allows other researchers to verify and expand upon their work. Furthermore, it allows public officials to make informed decisions. In this study, we analyzed 535 preprint articles related to COVID-19 for eight transparency criteria and recorded study location and funding information. We found that individual researchers have lined up to help during this crisis, quickly tackling important public health questions, often without funding or support from outside organizations. However, most authors could improve their data sharing and scientific reporting practices. The contrast between researchers' commitment to doing important research and their reporting practices reveals underlying weaknesses in the research community's reporting habits, but not necessarily their science. \"},\n"," {'paper_id': '0f3402fa5b44e121d410ec73dfc21937074e5fa3',\n","  'section': 'Abstract',\n","  'text': 'Background: To investigate the prevalence and outcomes of acute gastrointestinal injury (AGI) in critically ill patients with coronavirus disease 2019 . In this clinical retrospective study, demographic data, laboratory parameters, AGI grades, clinical severity and outcomes were collected. The primary endpoints were AGI incidence and 28-day mortality, the secondary endpoints were organ dysfunction and septic shock incidence. : From February 10 to March 10 2020, 83 critically ill patients of 1314 patients with COVID-19 were enrolled. Seventy-two (86.7%) patients had AGI during hospital stay, of them, 30 had AGI grade I, 35 had AGI grade II, 5 had AGI grade III, and 2 had AGI grade IV. The incidence of AGI grade II and above was 50.6%. As of March 16, 40 (48.2%) patients died within 28 days of admission, the median hospital stay was 12.0 days, ranging from 3 days to 27 days. Multiple organ dysfunction syndrome developed in 58 (69.9%) patients, septic shock in 16 (19.3%) patients. Patients with worse AGI grades had worse clinical variables, higher septic shock incidence and 28-day mortality. Sequential organ failure assessment scores (SOFA) (95% CI, 1.374-2.860; P <0.001), white blood cell (WBC) counts (95% CI, 1.037-1.379; P =0.014), duration of mechanical ventilation (MV) (95% CI, 1.020-1.340; P =0.025) were risk factors for the development of AGI grade II and above. Non-survivors were accompanied by higher incidence of AGI grade III to IV than survivors (17.5% vs. 0.0%, P =0.004). The AGI incidence was 86.7%, and hospital mortality was 48.2% in critically ill patients with COVID-19. SOFA scores, WBC counts, and duration of MV were risk factors for the development of AGI grade II and above. Patients with worse AGI grades had worse clinical severity variables, higher septic shock incidence and 28-day mortality. '},\n"," {'paper_id': '0f8261c63cdeeb5c2bf9294eba2954e1ca71b244',\n","  'section': 'Abstract',\n","  'text': 'The COVID-19 outbreak containment strategies in China based on non-pharmaceutical interventions (NPIs) appear to be effective. Quantitative research is still needed however to assess the efficacy of different candidate NPIs and their timings to guide ongoing and future responses to epidemics of this emerging disease across the World. We built a travel network-based susceptible-exposed-infectious-removed (SEIR) model to simulate the outbreak across cities in mainland China. We used epidemiological parameters estimated for the early stage of outbreak in Wuhan to parameterise the transmission before NPIs were implemented. To quantify the relative effect of various NPIs, daily changes of delay from illness onset to the first reported case in each county were used as a proxy for the improvement of case identification and isolation across the outbreak. Historical and near-real time human movement data, obtained from Baidu location-based service, were used to derive the intensity of travel restrictions and contact reductions across China. The model and outputs were validated using daily reported case numbers, with a series of sensitivity analyses conducted. We estimated that there were a total of 114,325 COVID-19 cases (interquartile range [IQR] 76,776 -164,576) in mainland China as of February 29, 2020, and these were highly correlated (p<0.001, R 2 =0.86) with reported incidence. Without NPIs, the number of COVID-19 cases would likely have shown a 67-fold increase (IQR: 44 -94), with the effectiveness of different interventions varying. The early detection and isolation of cases was estimated to prevent more infections than travel restrictions and contact reductions, but integrated NPIs would achieve the strongest and most rapid effect. If NPIs could have been conducted one week, two weeks, or three weeks earlier in China, cases could have been reduced by 66%, 86%, and 95%, respectively, together with significantly reducing the number of affected areas. However, if NPIs were conducted one week, two weeks, or three weeks later, the number of cases . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': '0fa18c47543c3ce69a3c9e4d77958c4d3691b1ed',\n","  'section': 'Abstract',\n","  'text': 'Sodium-bile acid cotransporter, also denominated sodium-taurocholate cotransporting polypeptide (NTCP) is an integral membrane protein with multiple hydrophobic transmembrane domains. The third extracellular domain of NTCP presents a stretch of nine aminoacids (KGIVISLVL) that is characterized by pronounced hydrophobicity and serves as receptor for a protein, preS1, showing the hydrophobic epta-peptide sequence NPLGFFP. Vitamin D-binding protein macrophage activating factor (DBP-MAF) is a multifunctional protein that is characterized by two hydrophobic regions able to bind fatty acids and vitamin D, respectively. Here we demonstrate that NTCP and DBP-MAF show significant sequence similarities as far as hydrophobic stretches of aminoacids are concerned. Alignment of the sequence of seven aminoacids preceding the 157-KGIVISLVL-165 stretch of NTCP shows four aminoacids that are identical to those of the corresponding sequence of DBP-MAF, and two that are conserved substitutions. In addition, in the sequence of DBP-MAF that is aligned with the sequence YKGIVISLVL of NTCP, there are two contiguous negatively charged aminoacids (ED) and, in the preceding epta-peptide sequence, there are three negatively charged aminoacids (D-ED), whereas in the corresponding sequence of NTCP there are only two (D--D) that are not contiguous. This concentration of negatively charged aminoacids may be involved in binding of protein inserts characterized by high density of positively charges residues. The alternating hydrophobic and electrostatic interactions described in this paper may help elucidating the biological roles of these proteins as far as protein-protein interactions are concerned. '},\n"," {'paper_id': '0fbb18050e29ca78191625d42576b1c574027377',\n","  'section': 'Abstract',\n","  'text': 'The ability to detect an infectious agent in a widespread epidemic is crucial to the success of quarantine efforts in addition to sensitive and accurate screening of potential cases of infection from patients in a clinical setting. Enabling testing outside of sophisticated laboratories broadens the scope of control and surveillance efforts, but also requires robust and simple methods that can be used without expensive instrumentation. Here we report a method to identify SARS-CoV-2 (COVID-19) virus RNA from purified RNA or cell lysis using loop-mediated isothermal amplification (LAMP) using a visual, colorimetric detection. This test was additionally verified using RNA samples purified from respiratory swabs collected from COVID-19 patients in Wuhan, China with equivalent performance to a commercial RT-qPCR test while requiring only heating and visual inspection. This simple and sensitive method provides an opportunity to facilitate virus detection in the field without a requirement for complex diagnostic infrastructure. . CC-BY 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': '0fd300aefb704c20f32152b97b6194015f1c74e7',\n","  'section': 'Abstract',\n","  'text': 'The outbreak of the novel coronavirus disease, COVID-19, originating from Wuhan, China in early December, has infected more than 70,000 people in China and other countries and has caused more than 2,000 deaths. As the disease continues to spread, the biomedical society urgently began identifying effective approaches to prevent further outbreaks. Through rigorous epidemiological analysis, we characterized the fast transmission of COVID-19 with a basic reproductive number 5.6 and proved a sole zoonotic source to originate in Wuhan. No changes in transmission have been noted across generations. By evaluating different control strategies through predictive modeling and Monte carlo simulations, a comprehensive quarantine in hospitals and quarantine stations has been found to be the most effective approach. Government action to immediately enforce this quarantine is highly recommended. '},\n"," {'paper_id': '0ff4ad5359ee2df3568605385a8616d8da66bf2c',\n","  'section': 'Abstract',\n","  'text': 'A novel coronavirus (SARS-CoV-2) has recently emerged as a global threat. As the epidemic progresses, many disease modelers have focused on estimating the basic reproductive number R 0 -the average number of secondary cases caused by a primary case in an otherwise susceptible population. The modeling approaches and resulting estimates of R 0 vary widely, despite relying on similar data sources. Here, we present a novel statistical framework for comparing and combining different estimates of R 0 across a wide range of models by decomposing the basic reproductive number into three key quantities: the exponential growth rate r, the mean generation intervalḠ, and the generation-interval dispersion κ. We then apply our framework to early estimates of R 0 for the SARS-CoV-2 outbreak. We show that many early R 0 estimates are overly confident. Our results emphasize the importance of propagating uncertainties in all components of R 0 , including the shape of the generation-interval distribution, in efforts to estimate R 0 at the outset of an epidemic. '},\n"," {'paper_id': '103dbbec77e480d9253910ae29263b3aef443b2f',\n","  'section': 'Abstract',\n","  'text': 'With the rapid development of mobile Internet in China, the information of the epidemic is full-time and holographic, and the role of information diffusion in epidemic control is increasingly prominent. At the same time, the publicity of all kinds of big data also provides the possibility to explore the impact of media information diffusion on disease transmission. This paper explores the mechanism of the influence of information diffusion on the spread of the novel coronavirus, develops a model of the interaction between information diffusion and disease transmission based on the SIR model, and empirically tests the role and mechanism of information diffusion in the spread of COCID-19 by using econometric method. The result shows that there was a significant negative correlation between the information diffusion and the spread of the novel coronavirus; The result of robust test shows that the spread of both epidemic information and protection information hindered the further spread of the epidemic. '},\n"," {'paper_id': '10424da2599f0258bfaf71a3186b999ab2d747d0',\n","  'section': 'Abstract',\n","  'text': \"Many mammalian carnivore species have been reduced to small, isolated populations by habitat destruction, fragmentation, poaching, and human conflict. Their limited genetic variability and increased exposure to domestic animals such as dogs place them at risk of further losses from infectious diseases. In India, domestic and feral dogs are associated with villages in and around protected areas, and may serve as reservoirs of pathogens to the carnivores within. India's Kanha Tiger Reserve (KTR) is home to a number of threatened and endangered mammalian carnivores including tiger (Panthera tigris), leopard (Panthera pardus), wolf (Canis lupus), and dhole (Cuon alpinus). It also has more than 150 villages with associated dog populations. We found that dog populations ranged from 14 to 45/village (3.7 to 23.7/km 2 ), and did not vary with village area, human population size, or distance from the KTR's core area, though they all increased between summer 2014 and winter 2015, primarily through reproduction. No dog tested positive for rabies but seroprevalence levels to three other generalist viral pathogens were high in summer (N=67) and decreased somewhat by winter (N=168): canine parvovirus (83.6% to 68.4%), canine distemper virus (50.7% to 30.4%) and canine adenovirus (41.8% to 30.9%). The declines in seroprevalence were primarily due to new recruitments by birth and these were not yet exposed to the viruses. Wild carnivores frequently entered the villages, as shown by tracks, scats, kills and other indicators, and the dogs are known to leave the villages so that encounters between dogs and wild carnivores may be common. We conclude that there is a large population of unvaccinated dogs in and around Kanha Tiger Reserve, with high levels of seroprevalence to pathogens with broad host ranges and these dogs which interact with wild carnivores, therefore posing a high risk of disease spillover to the wild carnivores. \"},\n"," {'paper_id': '10525ac89e46be4cb9cb9fd1131d28411a902047',\n","  'section': 'Abstract',\n","  'text': 'In our increasingly interconnected world, it is crucial to understand the risk of an outbreak originating in one country or region and spreading to the rest of the world. Digital disease surveillance tools such as ProMED and HealthMap have the potential to serve as important early warning systems as well as complement the field surveillance during an ongoing outbreak. Here we present a flexible statistical model that uses data produced from digital surveillance tools (ProMED and HealthMap) to forecast short term incidence trends in a spatially explicit manner. The model was applied to data collected by ProMED and HealthMap during the 2013-2016 West African Ebola epidemic. The model was able to predict each instance of international spread 1 to 4 weeks in advance. Our study highlights the potential and limitations of using publicly available digital surveillance data for assessing outbreak dynamics in real-time. '},\n"," {'paper_id': '105f9cfdb912d315c4d08957c26512b083145134',\n","  'section': 'Abstract',\n","  'text': 'Recently a novel coronavirus (SARS-CoV-2) emerged from Wuhan, China and has infected more than 571000 people leading to more than 26000 deaths. Since SARS-CoV-2 genome sequences show similarity with those of SARS, we sought to analyze all the available SARS-CoV-2 genomes based on the insights obtained from SARS genome specifically focusing on non-coding RNAs. Here, results are presented from the dual approach i.e identifying host encoded miRNAs that might regulate viral pathogenesis as well as identifying viral encoded miRNAs that might regulate host cell signaling pathways and aid in viral pathogenesis. Analysis utilizing first approach resulted in the identification of 10 host encoded miRNAs that could target the genome of both the viruses (SARS-CoV-2 and SARS reference genome). Interestingly our analysis revealed that there is significantly higher number of host miRNAs that could target SARS-CoV-2 genome as compared to the SARS reference genome. Results from second approach involving SARS-CoV-2 and SARS reference genome identified a set of virus encoded miRNAs which might regulate host signaling pathways. Our analysis further identified a similar \"GA\" rich motif in SARS-CoV-2 genome that was shown to play a vital role in lung pathogenesis during severe SARS infections. Hence, we successfully identified human and virus encoded miRNAs that might regulate pathogenesis of both these coronaviruses and the fact that more number of host miRNAs could target SARS-CoV-2 genomes possibly reveal as to why this virus follows mild pathogenesis in healthy individuals. We identified non-coding sequences in SARS-CoV-2 genomes that were earlier reported to contribute towards SARS pathology. The study provides insights into the overlapping sequences among these viruses for their effective inhibition as well as identifying new drug targets that could be used for development of new antivirals. : bioRxiv preprint the genomes of SARS-CoV-2 and SARS for identification of any common virus encoded miRNA. Contrary to our expectation, we did not find any virus encoded miRNA which was common to forward and reverse genomes of SARS and SARS- CoV-2 (fig 1e) . Recently, deep sequencing based approaches were used to identify SARS encoded non-coding RNAs which were then shown to play a critical role in SARS induced lung pathology [5]. The authors successfully identified a set of non-coding RNAs that were produced during SARS infections. The authors further characterized a \"GA\" rich region that was enriched in non-coding RNAs produced during SARS infection. Hence, we aimed to analyse SARS-CoV-2 genomic sequences for the identification of similar non-coding RNAs with a rationale that presence of the author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '106f10eaae4fb67aca5b2bd737ce2bf1c27fa243',\n","  'section': 'Abstract',\n","  'text': 'Most emerging pathogens can infect multiple species, underscoring the importance of 33 understanding the ecological and evolutionary factors that allow some hosts to harbor greater 34 infection prevalence and share pathogens with other species. However, our understanding of 35 pathogen jumps is primarily based around viruses, despite bacteria accounting for the greatest 36 proportion of zoonoses. Because bacterial pathogens in bats (Order: Chiroptera) can have 37 conservation and human health consequences, studies that examine the ecological and 38 evolutionary drivers of bacterial prevalence and barriers to pathogen sharing are crucially 39 needed. We here studied hemotropic Mycoplasma spp. (i.e., hemoplasmas) across a species-rich 40 bat community in Belize over two years. Across 469 bats spanning 33 species, half of individuals 41 and two-thirds of species were hemoplasma positive. Infection prevalence was higher for males 42 and for species with larger body mass and colony sizes. Hemoplasmas displayed high genetic 43 diversity (21 novel genotypes) and strong host specificity. Evolutionary patterns supported co-44 divergence of bats and bacterial genotypes alongside phylogenetically constrained host shifts. 45 Bat species centrality to the network of shared hemoplasma genotypes was phylogenetically 46 clustered and unrelated to prevalence, further suggesting rare-but detectable-bacterial sharing 47 between species. Our study highlights the importance of using fine phylogenetic scales when 48 assessing host specificity and suggests phylogenetic similarity may play a key role in host shifts 49 for not only viruses but also bacteria. Such work more broadly contributes to increasing efforts to 50 understand cross-species transmission and epidemiological consequences of bacterial pathogens. 51 '},\n"," {'paper_id': '107f0d5a47f48f85d6645cac0e7008edc4d9d8cd',\n","  'section': 'Abstract',\n","  'text': 'Public health concerns about recent viral epidemics have motivated researchers to seek transdisciplinary understanding of infection in wildlife hosts. With its deep history devoted to explaining the abundance and distribution of organisms, ecology can augment current methods for studying viral dynamics. However, datasets allowing ecological explorations of viral communities are lacking, and common methods for delineating viral operational taxonomic units (OTUs), or \"species\", are subjective. Here, we comprehensively sampled 1,086 bats from two Puerto Rican caves and tested them for infection with herpesviruses. Using percent identity of nucleotides and a machine learning algorithm, we categorized herpesviruses into 41 OTUs, representing approximately 80% of all herpesviruses in the host community. Although 13 OTUs were detected in multiple host species, OTUs generally exhibited host specificity by infecting a core host species at a significantly higher prevalence than in all other species combined. Only two OTUs showed significantly different prevalence between host sexes. This work is the first exploration of viral community ecology in a community of wildlife hosts. Lipkin WI (2013) A strategy to estimate unknown viral diversity in mammals. mBio 4: e00598-13. '},\n"," {'paper_id': '10aa307dc5239e819a19dd696b7f3e5bd34a37b1',\n","  'section': 'Abstract',\n","  'text': 'COVID-19 has caused a major epidemic worldwide, however, much is yet to be known 39 about the epidemiology and evolution of the virus. One reason is that the challenges 40 underneath sequencing HCoV-19 directly from clinical samples have not been com-41 pletely tackled. Here we illustrate the application of amplicon and hybrid capture (cap-42 ture)-based sequencing, as well as ultra-high-throughput metatranscriptomic (meta) 43 sequencing in retrieving complete genomes, inter-individual and intra-individual var-44 iations of HCoV-19 from clinical samples covering a range of sample types and viral 45 load. We also examine and compare the bias, sensitivity, accuracy, and other char-46 acteristics of these approaches in a comprehensive manner. This is, to date, the first 47 '},\n"," {'paper_id': '10abb3e977aa476a39bce4f1feda37c2e1250106',\n","  'section': 'Abstract',\n","  'text': '26 Previous findings of Middle East Respiratory Syndrome coronavirus (MERS-CoV)-27 related viruses in bats, and the ability of Tylonycteris-BatCoV HKU4 spike protein to utilize 28 MERS-CoV receptor, human dipeptidyl peptidase 4 hDPP4, suggest a bat ancestral origin of 29 MERS-CoV. We developed 12 primary bat cell lines from seven bat species, including 30 Tylonycteris pachypus, Pipistrellus abramus and Rhinolophus sinicus (hosts of Tylonycteris-31 BatCoV HKU4, Pipistrellus-BatCoV HKU5 and SARS-related-CoV respectively), and tested 32 their susceptibilities to MERS-CoVs, SARS-CoV and human coronavirus 229E (HCoV-229E). 33 Five cell lines, including P. abramus and R. sinicus but not T. pachypus cells, were susceptible 34 to human MERS-CoV EMC/2012. However, three tested camel MERS-CoV strains showed 35 different infectivities, with only two strains capable of infecting three and one cell lines 36 respectively. SARS-CoV can only replicate in R. sinicus cells, while HCoV-229E cannot 37 replicate in any bat cells. Bat dipeptidyl peptidase 4 (DPP4) sequences were closely related to 38 those of human and non-human primates but distinct from dromedary DPP4 sequence. Critical 39 residues for binding to MERS-CoV spike protein were mostly conserved in bat DPP4. DPP4 40 was expressed in the five bat cells susceptible to MERS-CoV, with significantly higher mRNA 41 expression levels than those in non-susceptible cells (P=0.0174), supporting that DPP4 42 expression is critical for MERS-CoV infection in bats. However, overexpression of T. pachypus 43 DPP4 failed to confer MERS-CoV susceptibility in T. pachypus cells, suggesting other cellular 44 factors in determining viral replication. The broad cellular tropism of MERS-CoV should 45 prompt further exploration of host diversity of related viruses to identify its ancestral origin. 46 47 All rights reserved. No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/326538 doi: bioRxiv preprint Existing evidence suggests that MERS-CoV may be originated from bats. In particular, 49 Tylonycteris-BatCoV HKU4 spike protein was shown to utilize MERS-CoV receptor, human 50 dipeptidyl peptidase 4 (DPP4). To better understand the potential infectivities of MERS-CoV in 51 bats, we developed 12 primary bat cell lines from seven bat species, including Tylonycteris 52 pachypus and Rhinolophus sinicus (host of Tylonycteris-BatCoV HKU4 and SARS-related-53 CoV respectively). MERS-CoV demonstrated much broader cellular tropism than SARS-CoV 54 and HCoV-229E, being able to infect five cell lines, including R. sinicus but not T. pachypus 55 cells. The close phylogenetic relationship between bat and human DPP4 genes supported the 56 ability of MERS-CoV to infect bat cells, while DPP4 expression appeared critical for MERS-57 CoV replication. However, overexpression of T. pachypus DPP4 failed to confer MERS-CoV 58 susceptibility in T. pachypus cells. The broad cellular tropism of MERS-CoV may reflect the 59 host diversity of MERS-CoV-related viruses and offers insights into its ancestral origin. 60 61 All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '110b4b089bd757bd54587c2ab3262a374462c098',\n","  'section': 'Abstract',\n","  'text': 'As the initial step in carbohydrate catabolism in cells, the substrate-specific 19 transporters via active transport and facilitated diffusion play a decisive role in passage of 20 sugars through the plasma membrane into the cytoplasm. The SecY complex (SecYEG) 21 in bacteria forms a membrane channel responsible for protein translocation. This work 22 30 31 Keywords: channel permeabilization/ free diffusion / plasma membrane/ sugar transport 32 33 All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '111b9a6e91c938696fcdb4cb128b8ae739dbe11c',\n","  'section': 'Abstract',\n","  'text': \"Background: As of March 2, 2020, SARS-CoV-2 has infected more than 80174 people and caused 2915 deaths in China. This virus rapidly spreads to 56 countries worldwide. Thus, in order to effectively block its transmission, it is urgent to uncover all the possible transmission routes of SARS-CoV-2. Methods: From January 28 to February 18, 2020, 35 female patients diagnosed with COVID-19 in Tongji Hospital were included in this descriptive study. The gynecologic history, clinical characteristics, laboratory findings and chest computed tomography (CT) of all patients were recorded in detail. To examine whether there is sexual transmission through vaginal from female to her partner, we employed realtime polymerase chain reaction testing (RT-PCR) to detect SARS-CoV-2 in vaginal environment (including vaginal discharge, cervical or vaginal residual exfoliated cells) and anal swab samples, and inquired recent sexual behaviors from the patients. Findings: The age range of the 35 patients with COVID-19 was 37-88 years. Over 50% patients infected with SARS-CoV-2 had chronic diseases. We tested the vaginal environment and anal swabs from the 35 female patients with COVID-19 and found that only an anal swab sample from one patient was positive for SARS-CoV-2. All the samples from vaginal environment were negative for SARS-CoV-2. The infection rate of the patients' sexual partner was 42·9%. Additionally, two female patients admitted having sex with their partners during a possible infection incubation period, while one patient's partner was uninfected and the other patient's partner was diagnosed with COVID-19 (after the diagnosis of the female patient). Conclusion: No positive RT-PCR result was found in the vaginal environment perhaps due to the lack of ACE2 expression, which is the receptor of SARS-CoV-2, in the vagina and cervix tissues (human protein atlas). The results from this study show no evidence of transmission of SARS-CoV-2 through vaginal sex from female to her partner. However, the risk of infection of non vaginal sex and other intimate contacts during vaginal sex should not be ignored. \"},\n"," {'paper_id': '112e05c31018c816d0bd9ee64336eba79ee786ac',\n","  'section': 'Abstract',\n","  'text': 'The dynamics of coronavirus disease 2019 (COVID-19) transmissibility after symptom onset remains unknown. We conducted a prospective case-ascertained study on laboratory-confirmed COVID-19 cases and their contacts. Secondary clinical attack rate (considering symptomatic cases only) was analyzed for different exposure windows after symptom onset of index cases and for different exposure settings. Thirty-two confirmed patients were enrolled and 12 paired data (index-secondary cases) were identified among the 1,043 contacts. The secondary clinical attack rate was 0.9% (95% CI 0.5-1.7%). The attack rate was higher among those whose exposure to index cases started within five days of symptom onset (2.4%, 95% CI 1.1-4.5%) than those who were exposed later (zero case from 605 close contacts, 95% CI 0-0.61%). The attack rate was also higher among household contacts (13.6%, 95% CI 4.7-29.5%) and nonhousehold family contacts (8.5%, 95% CI 2.4-20.3%) than that in healthcare or other settings. The higher secondary clinical attack rate for contacts near symptom onset remained when the analysis was restricted to household and family contacts. There was a trend of increasing attack rate with the age of contacts (p for trend < 0.001). High transmissibility of COVID-19 near symptom onset suggests that finding and isolating symptomatic patients alone may not suffice to contain the epidemic, and more generalized social distancing measures are required. Rapid reduction of transmissibility over time implies that prolonged hospitalization of mild cases might not be necessary in large epidemics. '},\n"," {'paper_id': '114ed64f52f503d9d2e2ba1fd1ee62b0a168cd84',\n","  'section': 'Abstract',\n","  'text': 'China, respiratory manifestations of the disease have been observed. However, as a fatal comorbidity, acute myocardial injury (AMI) in COVID-19 patients has not been previously investigated in detail. We investigated the clinical characteristics of COVID-19 patients with AMI and determined the risk factors for AMI in them. We analyzed data from 53 consecutive laboratory-confirmed and hospitalized COVID-19 patients (28 men, 25 women; age, 19-81 years). We collected information on epidemiological and demographic characteristics, clinical features, routine laboratory tests (including cardiac injury biomarkers), echocardiography, electrocardiography, imaging findings, management methods, and clinical outcomes. Results: Cardiac complications were found in 42 of the 53 (79.25%) patients: tachycardia (n=15), electrocardiography abnormities (n=11), diastolic dysfunction (n=20), elevated myocardial enzymes (n=30), and AMI (n=6). All the six AMI patients were aged >60 years; five of them had two or more underlying comorbidities (hypertension, diabetes, cardiovascular diseases, and chronic obstructive pulmonary disease). Novel coronavirus pneumonia (NCP) severity was higher in the AMI patients than in patients with non-definite AMI (p<0.001). All the AMI patients required care in intensive care unit; of them, three died, two remain hospitalized. Multivariate analyses showed that C-reactive protein (CRP) levels, NCP severity, and underlying comorbidities were the risk factors for cardiac abnormalities in COVID-19 patients. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '115fad0dccbe00455572ee7f336b6c5b139ef4d3',\n","  'section': 'Abstract',\n","  'text': 'One of the reasons for the fast spread of SARS-CoV-2 is the lack of accuracy in detection tools in the clinical field. Molecular techniques, such as quantitative real-time RT-PCR and nucleic acid sequencing methods, are widely used to identify pathogens. For this particular virus, however, they have an overall unsatisfying detection rate, due to its relatively recent emergence and still not completely understood features. In addition, SARS-CoV-2 is remarkably similar to other Coronaviruses, and it can present with other respiratory infections, making identification even harder. To tackle this issue, we propose an assisted detection test, combining molecular testing with deep learning. The proposed approach employs a state-of-the-art deep convolutional neural network, able to automatically create features starting from the genome sequence of the virus. Experiments on data from the Novel Coronavirus Resource (2019nCoVR) show that the proposed approach is able to correctly classify SARS-CoV-2, distinguishing it from other coronavirus strains, such as MERS-CoV, HCoV-NL63, HCoV-OC43, HCoV-229E, HCoV-HKU1, and SARS-CoV regardless of missing information and errors in sequencing (noise). From a dataset of 553 complete genome non-repeated sequences that vary from 1,260 to 31,029 bps in length, the proposed approach classifies the different coronaviruses with an average ac-: bioRxiv preprint curacy of 98.75% in a 10-fold cross-validation, identifying SARS-CoV-2 with an AUC of 98%, specificity of 0.9939 and sensitivity of 1.00 in a binary classification. Then, using the same basis, we classify SARS-CoV-2 from 384 complete viral genome sequences with human host, that contain the gene ORF1ab from the NCBI with a 10-fold accuracy of 98.17% , a specificity of 0.9797 and sensitivity of 1.00. Furthermore, an in-depth analysis of the results allow us to identify base pairs sequences that are unique to SARS-CoV-2 and do not appear in other virus strains, that could then be used as a base for designing new primers and examined by experts to extract further insights. These preliminary results seem encouraging enough to identify deep learning as a promising research venue to develop assisted detection tests for SARS-CoV-2. At this end the interaction between viromics and deep learning, will hopefully help to solve global infection problems. In addition, we offer our code and processed data to be used for diagnostic purposes by medical doctors, virologists and scientists involved in solving the SARS-CoV-2 pandemic. As more data become available we will update our system. '},\n"," {'paper_id': '11875dfda6cd04d5032611edf0e340582ceb620f',\n","  'section': 'Abstract',\n","  'text': 'An outbreak of new SARS-like viral in Wuhan, China has been named 2019-nCoV. The current state of the epidemic is increasingly serious, and there has been the urgent necessity to develop an effective new drug. In previous studies, it was found that the conformation change in CTD1 was the region where SARS-CoV bound to human ACE2. Although there are mutations of the 2019-nCoV, the binding energy of ACE2 remains high. The surface glycoprotein of 2019-nCoV was coincident with the CTD1 region of the S-protein by comparing the I-TASSER prediction model with the actual SARS model, which suggests that 2019-nCoV may bind to the ACE2 receptor through conformational changes. Furthermore, site prediction on the surface glycoprotein of 2019-nCoV suggests some core amino acid area may be a novel drug target against 2019-nCoV. '},\n"," {'paper_id': '11bb388ea0dbb352e25fcbc8614bfd6e62831e8b',\n","  'section': 'Abstract',\n","  'text': 'A recent outbreak of novel coronavirus (SARS-CoV-2), the causative agent of COVID-19, has spread rapidly all over the world. Human immunodeficiency virus (HIV) is another deadly virus and causes acquired immunodeficiency syndrome (AIDS). Rapid and early detection of these viruses will facilitate early intervention and reduce disease transmission risk. Here, we present an All-In-One Dual CRISPR-Cas12a (termed \"AIOD-CRISPR\") assay method for simple, rapid, ultrasensitive, one-pot, and visual detection of coronavirus SARS-CoV-2 and HIV virus. In our AIOD CRISPR assay, a pair of crRNAs was introduced to initiate dual CRISPR-Cas12a detection and improve detection sensitivity. The AIOD-CRISPR assay system was successfully utilized to detect nucleic acids (DNA and RNA) of SARS-CoV-2 and HIV with a sensitivity of few copies. Also, it was evaluated by detecting HIV-1 RNA extracted from human plasma samples, achieving a comparable sensitivity with real-time RT-PCR method. Thus, our method has a great potential for developing next-generation point-of-care molecular diagnostics. '},\n"," {'paper_id': '11c04553ff7a7996ee5dd75446e20aad20714fce',\n","  'section': 'Abstract',\n","  'text': 'Lack of evidence on SARS-CoV-2 transmission dynamics has led to shifting isolation guidelines between airborne and droplet isolation precautions. During the initial isolation of 13 individuals confirmed positive with COVID-19 infection, air and surface samples were collected in eleven isolation rooms to examine viral shedding from isolated individuals. While all individuals were confirmed positive for SARS-CoV-2, symptoms and viral shedding to the 15 environment varied considerably. Many commonly used items, toilet facilities, and air samples had evidence of viral contamination, indicating that SARS-CoV-2 is shed to the environment as expired particles, during toileting, and through contact with fomites. Disease spread through both direct (droplet and person-to-person) as well as indirect contact (contaminated objects and airborne transmission) are indicated, supporting the use of airborne isolation precautions. 20 '},\n"," {'paper_id': '11f96d05db6854ef95312aa3a4736724ce1f02d6',\n","  'section': 'Abstract',\n","  'text': 'We evaluated if interventions aimed at air travellers can delay establishment of a SARS-CoV-2 outbreak in a previously unaffected country. Methods: For countries with no sustained SARS-CoV-2 transmission and with no shared border with affected regions we simulated arriving infected air travellers. We assessed the effectiveness of syndromic screening at departure and/or arrival & traveller sensitisation to the COVID-2019-like symptoms with the aim to trigger rapid self-isolation and reporting on symptom onset to enable contact tracing. We assumed that syndromic screening would reduce the number of infected arrivals and that traveller sensitisation reduce the average number of secondary cases. We report the minimal expected delay achievable in 50% (75% & 97.5%) of simulations. In the simulations we account for uncertainty in the number of secondary cases in the absence of air traveller targeted interventions and the arrival times of infected cases and also present sensitivity analyses on arrival rates of infected travellers and the effectiveness of traveller sensitisation. Results: Under baseline assumptions exit and entry screening combined with traveller sensitisation can delay a local SARS-CoV-2 outbreak by at least 83 (75% of simulations: at least 36, 97.5% 8) days while there is no more than 1 infected traveller per week. The benefit of entry screening is small if exit screening is effective: the combination of only exit screening and traveller sensitisation can delay an outbreak by at least 76 (75%: 33, 97.5%: 7) days. With increasing rates of infected travellers, less effective sensitisation or without screening these delays shrink rapidly to a week or less. Conclusion: Syndromic screening and traveller sensitisation in combination could delay outbreaks in yet unaffected countries and support local containment efforts, but only if infected traveller numbers are very low. . CC-BY 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': '1218f278a4f8d83dac14b23c8f698062812ef9d5',\n","  'section': 'Abstract',\n","  'text': 'A novel coronavirus (SARS-CoV-2) first detected in Wuhan, China, has spread rapidly since December 2019, causing more than 80,000 confirmed infections and 2,700 fatalities (as of Feb 27, 2020). Imported cases and transmission clusters of various sizes have been reported globally suggesting a pandemic is likely. Here, we explore how seasonal variation in transmissibility could modulate a SARS-CoV-2 pandemic. Data from routine diagnostics show a strong and consistent seasonal variation of the four endemic coronaviruses (229E, HKU1, NL63, OC43) and we parameterize our model for SARS-CoV-2 using these data. The model allows for many subpopulations of different size with variable parameters. Simulations of different scenarios show that plausible parameters result in a small peak in early 2020 in temperate regions of the Northern Hemisphere and a larger peak in winter 2020/2021. Variation in transmission and migration rates can result in substantial variation in prevalence between regions. While the uncertainty in parameters is large, the scenarios we explore show that transient reductions in the incidence rate might be due to a combination of seasonal variation and infection control efforts but do not necessarily mean the epidemic is contained. Seasonal forcing on SARS-CoV-2 should thus be taken into account in the further monitoring of the global transmission. The likely aggregated effect of seasonal variation, infection control measures, and transmission rate variation is a prolonged pandemic wave with lower prevalence at any given time, thereby providing a window of opportunity for better preparation of health care systems. '},\n"," {'paper_id': '123db7cdec9fc63aa09782ea9f4267603be8f74d',\n","  'section': 'Abstract',\n","  'text': 'Advanced biomaterials provide versatile ways to spatially and temporally control immune cell activity, potentially enhancing their therapeutic potency and safety. Precise cell modulation demands multi-modal display of functional proteins with controlled densities on biomaterials. Here, we develop an artificial immune cell engager (AICE) platform -biodegradable particles onto which multiple proteins are densely loaded with ratiometric control via short nucleic acid tethers. We demonstrate the impact of AICE with varying ratios of anti-CD3 and anti-CD28 antibodies on ex vivo expansion of human primary T cells. We also show that AICE can be used to control the activity of engineered T cells in vivo. AICE injected intratumorally can provide a local priming signal for systemically administered AND-gate chimeric antigen receptor T cells, driving local tumor clearance while sparing uninjected tumors that model potentially cross-reactive healthy tissues. This modularly functionalized biomaterial thus provides a flexible platform to achieve sophisticated control over cell-based immunotherapies. Functionalized biomaterials can work synergistically with natural or engineered immune cells for cancer immunotherapy 1-15 . Adoptive cell therapy (ACT) using chimeric antigen receptor (CAR) engineered T cells has shown success and clinical approval for the treatment of B cell cancers 16 . However, for CAR T cells to fulfill their promising potential, particularly for targeting solid tumors 17, 18 , important challenges must be overcome to improve both efficacy 17-20 and safety 21-24 . For engineered anti-tumor T cells to achieve durable tumor remission in patients, a demanding manufacturing process is required to ensure the quantity and quality of the cell product for therapeutic use 1, 25-27 . To increase the tumor targeting specificity and All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '125677351ae5741aaf5aa34ba088e4328584450d',\n","  'section': 'Abstract',\n","  'text': 'Zoonotic coronavirus (CoV) infections, like those responsible for the current SARS-CoV-2 '},\n"," {'paper_id': '12920263b2846c61d6f2b6105189367a8ff1bc39',\n","  'section': 'Abstract',\n","  'text': \"Reproductive success in the eight founder strains of the Collaborative Cross (CC) was measured using a diallel-mating scheme. Over a 48-month period we generated 4,448 litters, and 24,782 weaned pups were used across 16 different published experiments. We identified factors that affect the average litter size in a cross by estimating the overall contribution of parent-of-origin, heterosis, inbred, and epistatic effects using a Bayesian zero-truncated overdispersed Poisson mixed model. The phenotypic variance of litter size has a substantial contribution (79%) from unexplained and environmental sources, but no detectable effect of seasonality. Most of the explained variance was due to additive effects (9.2%) and parental sex (maternal vs paternal strain; 5.8%), with epistasis accounting for 3.4%. Within the parental effects, the effect of the dam's strain explained more than the sire's strain (13.2% vs. 1.8%), and the dam's strain effects account for 74.2% of total variation explained. Dams from strains C57BL/6J and NOD/ShiLtJ increased the expected litter size by a mean of 1.66 and 1.79 pups, whereas dams from strains WSB/EiJ, PWK/PhJ, and CAST/EiJ reduced expected litter size by a mean of 1.51, 0.81, and 0.90 pups. Finally, there was no strong evidence for strain-specific effects on sex ratio distortion. Overall, these results demonstrate that strains vary substantially in their reproductive ability depending on their genetic background and that litter size is largely determined by dam.strain rather than sire.strain effects, as expected. This analysis adds to our understanding of factors that influence litter size in mammals, and also helps to explain breeding successes and failures in the extinct lines and surviving CC strains. KEYWORDS F1 cross additive heritability fertility BayesDiallel zero-truncated Poisson variance projection \"},\n"," {'paper_id': '12c8ef5614d9a35bed3ff95a3b1dc971842514fe',\n","  'section': 'Abstract',\n","  'text': \"RNA viruses are the only known RNA-protein (RNP) entities capable of autonomous replication (albeit within a permissive environment). A 33.5-kb nidovirus has been considered close to the upper size limit for such entities; conversely, the minimal cellular DNA genome is ~200 kb. This large difference presents a daunting gap for the transition from primordial RNP to contemporary DNA-RNP-based life. Whether or not RNA viruses represent transitional steps on the road to DNA-based life, studies of larger RNA viruses advance our understanding of size constraints on RNP entities. For example, emergence of the largest previously known RNA genomes (20-34 kb in positive-stranded nidoviruses, including coronaviruses) is associated with a proofreading exoribonuclease encoded in the nidoviral open reading frame 1b (ORF1b). However, apparent constraints on the size of ORF1b, which encodes this and other key replicative enzymes, have been hypothesized to limit further expansion of viral RNA genomes. Here, we characterize a novel nidovirus (planarian secretory cell nidovirus; PSCNV) whose disproportionately large ORF1b-like region, and overall 41.1 kb genome, substantially extend the presumed limits on RNA genome size. This genome encodes a predicted 13,556-aa polyprotein in an unconventional single ORF, yet retains canonical nidoviral genome organization and expression, and key replicative domains. Our evolutionary analysis suggests that PSCNV diverged early from multi-ORF nidoviruses, and subsequently acquired additional genes, including those typical of large DNA viruses or hosts. PSCNV's greatly expanded genome, proteomic complexity, and unique featuresimpressive in themselvesattest to the likelihood of still-larger RNA genomes awaiting discovery. author/funder. All rights reserved. No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/299776 doi: bioRxiv preprint RNA viruses are the only known RNA-protein (RNP) entities capable of autonomous replication. The upper genome size for such entities was assumed to be <35 kb; conversely, the minimal cellular DNA genome is ~200 kb. This large difference presents a daunting gap for the proposed evolution of contemporary DNA-RNP-based life from primordial RNP entities. Here, we describe a nidovirus from planarians, whose 41.1 kb genome is 23% larger than the largest known of RNA virus. The planarian secretory cell nidovirus has broken apparent constraints on the size of the genomic subregion that encodes core replication machinery, and has acquired genes not previously observed in RNA viruses. This virus challenges and advances our understanding of the limits to RNA genome size. author/funder. All rights reserved. No reuse allowed without permission. \"},\n"," {'paper_id': '12fac9aedb1a09a3922a3c084ce4723708e463d6',\n","  'section': 'Abstract',\n","  'text': 'Currently, a novel coronavirus 2019-nCoV causes an outbreak of viral pneumonia in Wuhan, China. Little is known about its epidemiological characteristics. Using the travel history and symptom onset of 88 confirmed cases that were detected outside Wuhan, we estimate the mean incubation period to be 6.4 (5.6 -7.7, 95% CI) days, ranging from 2.1 to 11.1 days (2.5 th to 97.5 th percentile). These values help to inform case definitions for 2019-nCoV and appropriate durations for quarantine. '},\n"," {'paper_id': '132837acbb324a3845909b6482b90045b25519ca',\n","  'section': 'Abstract',\n","  'text': 'On January 23, 2020, China imposed a quarantine on the city of Wuhan to contain the SARS-CoV-2 outbreak. Regardless of this measure the new infection has spread to several countries around the world. Here, we developed a method to study the dissemination of this infection by the airline routes and we give estimations of the time of arrival of the outbreaks to the different cities. In this work we show an analysis of the dispersion of this infection to other cities by airlines based on the classic model the Kermack and McKendrick complemented with diffusion on a graph composed of nodes which represent the cities and edges which represent the airline routes. We do several numerical simulations to estimate the date of arrival to different cities starting the infection at Wuhan, China and to show the robustness of the estimation respect to changes in the epidemiological parameters and to changes on the graph. we use Mexico City as an example. In this case, our estimate of the arrival time is between March 20 and March 30, 2020. This analysis is limited to the analysis of dispersion by airlines, so this estimate should be taken as an overestimate since the infection can arrive by other means. This model estimates the arrival of the infectious outbreak to Mexico between March 20 and March 30. This estimation gives a time period to implement and strengthen preventive measures aimed at the general population, as well as to strengthen hospital infrastructure and training of human resources in health. '},\n"," {'paper_id': '136ee862ad53b606c8f2bea917b9705f5079e88e',\n","  'section': 'Abstract',\n","  'text': 'We developed a computational tool to assess the risks of novel coronavirus outbreaks outside of China. We estimate the dependence of the risk of a major outbreak in a country from imported cases on key parameters such as: (i) the evolution of the cumulative number of cases in mainland China outside the closed areas; (ii) the connectivity of the destination country with China, including baseline travel frequencies, the effect of travel restrictions, and the efficacy of entry screening at destination; and (iii) the efficacy of control measures in the destination country (expressed by the local reproduction number R loc ). We found that in countries with low connectivity to China but with relatively high R loc , the most beneficial control measure to reduce the risk of outbreaks is a further reduction in their importation number either by entry screening or travel restrictions. Countries with high connectivity but low R loc benefit the most from policies that further reduce R loc . Countries in the middle should consider a combination of such policies. Risk assessments were illustrated for selected groups of countries from America, Asia, and Europe. We investigated how their risks depend on those parameters, and how the risk is increasing in time as the number of cases in China is growing. '},\n"," {'paper_id': '138d4617f4f4984bec9cf2c7540fed533d9e4d6e',\n","  'section': 'Abstract',\n","  'text': 'Health-care and public health professionals rely on accurate, real-time monitoring of infectious diseases for outbreak preparedness and response. Early detection of outbreaks is improved by systems that are pathogen-specific. We describe a system, FilmArray ® Trend, for rapid disease 45 reporting that is syndrome-based but pathogen-specific. Results from a multiplex molecular diagnostic test are sent directly to a cloud database. www.syndromictrends.com presents these data in near real-time. Trend preserves patient privacy by removing or obfuscating patient identifiers. We summarize the respiratory pathogen results, for 20 organisms from 344,000 patient samples acquired as standard of care testing over the last four years from 20 clinical 50 laboratories in the United States. The majority of pathogens show influenza-like seasonality, rhinovirus has fall and spring peaks and adenovirus and bacterial pathogens show constant detection over the year. Interestingly, the rate of pathogen co-detections, on average 7.7%, matches predictions based on the relative abundance of organisms present. 55 All rights reserved. No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/157156 doi: bioRxiv preprint 65 Newer efforts at syndrome-based surveillance [10-12] include BioSense (extraction of symptomatic data from electronic health records [13]), Google Flu (tracking of Internet search queries [14] but recently discontinued [15]), and Flu Near You (voluntary reporting [16]). Additionally, numerous next generation, syndromic surveillance systems; e.g., pharmacy sales 70 records [17, 18], Twitter conversations [19, 20], and Wikipedia hits [21, 22], have come online in the past five years. However, these systems cannot report the specific pathogen causing an increase in a particular set of symptoms. Finally, there are more localized efforts such as GermWatch in Utah [23, 24] and the Electronic Clinical Laboratory Reporting System (ECLRS) in New York [25, 26] that draw from hospital (HIS) and laboratory (LIS) information systems. 75 This disparity in technologies and data collection methods results in incomplete surveillance. Comprehensive and uniform diagnostic test data will aid in the identification of potential outbreaks. A combination of broad respiratory pathogen testing and an internal electronic surveillance system enabled the rapid dissemination of data across the largest health care system 80 in New York, the North Shore-LIJ Health System (now Northwell Health) during the influenza A H1N1-2009 pandemic in the New York City area. Pathogen-specific molecular testing permitted rapid a) notification to state epidemiologists; b) tracking of the virus so that health care resources could be managed effectively; and c) evaluation of influenza diagnostics [27, 28] . Today, with the threat of emerging pathogens such as Middle East Respiratory Syndrome 85 All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '139b9a993204af8852fbdb685a455ee770eb481a',\n","  'section': 'Abstract',\n","  'text': 'Manuscript word count: Key points 63; Abstract 349; Full text 2924 All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '13bdc1078ca5091bfd346a4ecc5dacb6566480d9',\n","  'section': 'Abstract',\n","  'text': 'The rigidity and flexibility of two recently reported crystal structures (PDB entries 6Y2E and 6LU7) of a protease from the SARS-CoV-2 virus, the infectious agent of the COVID-19 respiratory disease, has been investigated using pebble-game rigidity analysis, elastic network model normal mode analysis, and all-atom geometric simulations. This computational investigation of the viral protease follows protocols that have been effective in studying other homodimeric enzymes. The protease is predicted to display flexible motions in vivo which directly affect the geometry of a known inhibitor binding site and which open new potential binding sites elsewhere in the structure. A database of generated PDB files representing natural flexible variations on the crystal structures has been produced and made available for download from an institutional data archive. This information may inform structure-based drug design and fragment screening efforts aimed at identifying specific antiviral therapies for the treatment of COVID-19. Recent studies has established that the motions explored by geometric simulation correspond well to the essential dynamics extracted from MD trajectories [4] ; that global flexible motions can couple to . '},\n"," {'paper_id': '13bf9fe0df944c0ee0debb0d445527ad4418cc19',\n","  'section': 'Abstract',\n","  'text': 'word count: 224/250 18 Text word count: 5136/5000 19 All rights reserved. No reuse allowed without permission. Sialic acids (Sia) are widely displayed on the surfaces of cells and tissues. Sia come in 21 a variety of chemically modified forms, including those with acetyl modifications at the 22 C7, C8, and C9 positions. Here, we analyzed the distribution and amounts of these 23 acetyl modifications in different human and canine cells. As Sia or their variant forms 24 are receptors for influenza A and influenza C viruses, we examined the effects of these 25 modifications on virus infections. We confirmed that 9-O-acetyl and 7,9-O-acetyl 26 modified Sia are widely but variably expressed across cell lines from both humans and 27 canines. While they were expressed on the cell surface of canine MDCK cell lines, they 28 were located primarily within the Golgi compartment of human HEK-293 and A549 cells. 29 The O-acetyl modified Sia were expressed at low levels of 1-2% of total Sia in these cell 30 lines. We knocked out and over-expressed the sialate O-acetyltransferase gene 31 (CasD1), and knocked out the sialate O-acetylesterase gene (SIAE) using 32 CRISPR/Cas9 editing. Knocking out CasD1 removed 7,9-O-and 9-O-acetyl Sia 33 expression, confirming previous reports. However, over-expression of CasD1 and 34 knockout of SIAE gave only modest increases in 9-O-acetyl levels in cells and no 35 change in 7,9-O-acetyl levels, indicating that there are complex regulations of these 36 modifications. These modifications were essential for influenza C infection, but had no 37 obvious effect on influenza A infection. 38 '},\n"," {'paper_id': '14083b334f48819d6a42a2ed917adfa200c59df4',\n","  'section': 'Abstract',\n","  'text': 'Since the first report of COVID-19 from Wuhan China, the virus has rapidly spread across the globe now presently reported in 177 countries with positive cases crossing 400 thousand and rising. In the current study, prediction is made for highly infected countries by a simple and novel method using only cumulative positive cases reported. The rate of infection per week ( ) coefficient delineated three phases for the current COVID-19 pandemic. All the countries under study have passed Phase 1 and are currently in Phase 2 except for South Korea which is in Phase 3. Early detection with rapid and large-scale testing helps in controlling the COVID-19 pandemic. Staying in Phase 2 for longer period would lead to increase in COVID-19 positive cases. '},\n"," {'paper_id': '14220553c9160c06bb99f59d59e6a8ecf3595226',\n","  'section': 'Abstract',\n","  'text': \"COVID-19 is leading to the implementation of social distancing policies around the world and in the United States, including school closures. The evidence that mandatory school closures reduce cases and ultimately mortality mostly comes from experience with influenza or from models that do not include the impact of school closure on the healthcare labor supply or the role of the healthcare labor force in reducing the per infection mortality from the pathogen. There is considerable uncertainty of the incremental effect of school closures on transmission and lives saved from school closures. The likely, but uncertain, benefits from school closure need to be weighed against uncertain, and seldom quantified, costs of healthcare worker absenteeism associated with additional child care obligations. We analyze data from the US Current Population Survey to measure the potential child care obligations for US healthcare workers that will need to be addressed if school closures are employed as a social distancing measure. We account for the occupation within the healthcare sector, state, and household structure to identify the segments of the healthcare labor force that are most exposed to child care obligations from school closures. We use these estimates to identify the critical level for the importance of healthcare labor supply in increasing a patient's COVID-19 survival probability that would undo the benefits of school closures and ultimately increase cumulative mortality. The US healthcare sector has some of the highest child care obligations in the United States. 29% of healthcare provider households must provide care for children 3-12. Assuming non-working adults or a sibling 13 years old or older can provide child care, leaves 15% of healthcare provider households in need of childcare during a school closure, while 7% of healthcare households are single-parent households. We document the substantial variation within the healthcare system. For example, 35% of medical assistants and 31% of nursing, psychiatric, and home health aide households have child care obligations, while only 24% of emergency medical personnel have childcare obligations. Child care obligations can vary between states by over 10 percentage points. A 15% decline in the healthcare labor force, combined with reasonable parameters for COVID-19 such as a 15% case reduction from school closings and 2% baseline mortality rate implies that a 15% loss in the healthcare labor force must decrease the survival probability per percent healthcare worker lost by 17.6% for a school closure to increase cumulative mortality. This means that the per infection mortality rate cannot increase from 2% to 2.35% when the healthcare workforce declines by 15%; otherwise, school closures will lead to a greater number of deaths than they prevent. For school closures to unambiguously provide a net reduction in COVID-19 mortality with these parameters, the school closures must reduce cases by over 25%. School closures come with many tradeoffs. Setting aside economic costs, school closures implemented to reduce COVID-19 spread create unintended childcare obligations, which are particularly large in healthcare occupations. Detailed data are provided to help public health officials make informed decisions about the tradeoffs associated with closing schools. The results suggest that it is unclear if the potential All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. \"},\n"," {'paper_id': '14783283da3482bf2aab4e6100fe4ea5c29d8db7',\n","  'section': 'Abstract',\n","  'text': 'Since December 2019, more than 79,000 people have been diagnosed with infection of the Corona Virus Disease 2019 . A large number of medical staff were dispersed for Wuhan city and Hubei province to aid COVID-19 control. Psychological stress, especially vicarious traumatization (VT) caused by the COVID-19 pandemic, should not be ignored. To address this concern, the study employed a total of 214 general public (GP) and 526 nurses to evaluate VT scores via a mobile app-based questionnaire. Results showed that the VT scores slightly increased across periods of aiding COVID-19 control, although no statistical difference was noted (P = 0.083). However, the study found lower scores for VT in nurses [median = 69; interquartile range (IQR) = 56-85] than those of the GP (median = 75.5; IQR = 62-88.3) (P = 0.017). In addition, the VT scores for front-line nurses (FLNs; median = 64; IQR = 52-75), including scores for physiological and psychological responses, were significantly lower than those of non-front-line nurses (nFLNs; median = 75.5; IQR = 63-92) (P < 0.001). Interestingly, the VT scores of the GP were significantly higher than those of the FLNs (P < 0.001). However, no statistical difference was observed compared with those of nFLNs (P > 0.05). Importantly, nFLNs are more likely to suffer from VT, which might be related to two factors, namely, gender [odds ratio (OR) = 3.1717; 95% confidence interval (CI) = 4.247-18.808; P = 0.002] and fertility [OR = 2.072; 95%CI = 0.626-24.533; P = 0.039]. Therefore, increased attention should be paid to the psychological problems of the medical staff, especially nFLNs, and GP under the situation of the spread and control of COVID-19. Early strategies that aim . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint . https://doi.org/10. 1101 /2020 to prevent and treat VT in medical staff and GP are extremely necessary. '},\n"," {'paper_id': '14b60a25791d523218d16573129f3c7e317b7f37',\n","  'section': 'Abstract',\n","  'text': 'COVID-19 has become a pandemic. The influence of meteorological factors on the transmission and spread of COVID-19 if of interest. This study sought to examine the associations of daily average temperature (AT) and relative humidity (ARH) with the daily count of COVID-19 cases in 30 Chinese Additive Model (GAM) was fitted to quantify the province-specific associations between meteorological variables and the daily cases of COVID-19 during the study periods. In the model, the 14-day exponential moving averages (EMAs) of AT and ARH, and their interaction were included with time trend and health-seeking behavior adjusted. Their spatial distributions were visualized. AT and ARH showed significantly negative associations with COVID-19 with a significant interaction between them (0.04, 95% confidence interval: 0.004-0.07) in Hubei. Every 1°C increase in the AT led to a decrease in the daily confirmed cases by 36% to 57% when ARH was in the range from 67% to 85.5%. Every 1% increase in ARH led to a decrease in the daily confirmed cases by 11% to 22% when AT was in the range from 5.04°C to 8.2°C. However, these associations were not consistent throughout Mainland China. '},\n"," {'paper_id': '151878038cf26f6f2b2ec8a391e6c4d425211ca5',\n","  'section': 'Abstract',\n","  'text': \"As the outbreak of coronavirus disease 2019 progresses, prognostic markers for early identification of high-risk individuals are an urgent medical need. Italy has the highest rate of SARS-CoV-2 infection, the highest number of deaths, and the highest mortality rate among large countries. Worldwide, a more severe course of COVID-19 is associated with older age, comorbidities, and male sex. Hence, we searched for possible genetic components of the peculiar severity of COVID-19 among Italians, by looking at expression levels and variants in ACE2 and TMPRSS2 genes, which are crucial for viral infection. Exome and SNP array data from a large Italian cohort representative of the country's population were used to compare the burden of rare variants and the frequency of polymorphisms with European and East Asian populations. Moreover, we looked into gene expression databases to check for sexunbalanced expression. \"},\n"," {'paper_id': '158e546658e42ab0a89d174bc9facb437a034df6',\n","  'section': 'Abstract',\n","  'text': 'Cases from the ongoing outbreak of atypical pneumonia caused by the 2019 novel coronavirus (2019-nCoV) exported from mainland China can lead to self-sustained outbreaks in other populations. '},\n"," {'paper_id': '159a2b9cfa1b6755a09de95d76dad5490e6a2bb9',\n","  'section': 'Abstract',\n","  'text': 'When people are forced to be isolated from one another, do they crave social interactions in the same way a hungry person craves food? To address this question, we used functional magnetic resonance imaging (fMRI) to measure neural responses in participants (n=40) evoked by food and social cues after ten hours of mandated fasting or total social isolation. After isolation, people felt lonely and craved social interaction. Midbrain regions showed increased activation to food cues after fasting and to social cues after isolation; these responses were correlated with self-reported craving. Neural patterns in response to food cues when participants were hungry generalized to social cues after isolation. Our results support the intuitive idea that acute isolation causes social craving, similar to hunger. : bioRxiv preprint homology to human loneliness has been disputed(14), and it is not possible to assess whether a mouse subjectively feels lonely. In sum, the intuitive idea that unmet social needs evoke a motivation to seek social interaction, analogous to the way hunger motivates people to seek food, has never been directly tested in humans. Would acute social isolation evoke a \"social craving\" response in dopaminergic midbrain regions in humans? To address this question, we had to first address three methodological challenges. First, could we experimentally induce the experience of social isolation in human participants? To differentiate between pre-existing individual differences and the direct effects of isolation, we needed to induce isolation experimentally in a within-subject design, similar to the experimental studies of rodents(43, 46). However, human loneliness is not a simple product of objective isolation: people can be alone without feeling lonely, or feel lonely even in a crowd(47). Moreover, experimentally induced isolation would necessarily be brief, relative to the human lifespan, and for ethical reasons, human participants (unlike rodents) would be able to predict when the isolation would end. In all, the first challenge of this research was to develop an experimental induction of objective isolation that created the subjective experience of unmet social needs in human participants. To address this challenge, we had socially-connected healthy human adults spend ten hours (9 am to 7 pm) alone, with no social interaction and no other social stimulation (e.g. social media, email, fiction). We used self-report questionnaires to assess people\\'s resulting subjective experience of loneliness and social craving. Second, could we measure neural responses in the relevant dopaminergic midbrain regions? In primates, aversive motivation is represented in the substantia nigra pars compacta and ventral tegmental area (SN/VTA)(23), and SN/VTA is activated by craving for food and for drugs of addiction(28-30). We therefore hypothesized that acute isolation in humans might produce a social craving response in SN/VTA. Neuroimaging of the SN/VTA poses a technical challenge, though, because it is a small structure, adjacent to the sphenoid sinus (a large, air-filled cavity located anterior to the brainstem) and therefore prone to distortions and signal loss (19, 20) . To address this challenge, we optimized MRI image acquisition parameters, and used a newly developed atlas(48) to identify SN/VTA in individual participants\\' brains. Third, could we measure signals associated with craving? Although fMRI cannot directly measure DA levels, blood oxygenation dependent (BOLD) signals recorded in midbrain are correlated with DA '},\n"," {'paper_id': '159ba7c964dc99d71f35411ce8733f6160b141bd',\n","  'section': 'Abstract',\n","  'text': 'Nasopharygneal swabs (NPS) are the collection modality recommended by the Centers for 25 Disease Control and Prevention (CDC) for reverse-transcription polymerase chain reaction (RT-PCR) 26 testing for SARS-CoV2. NPS gather both extracellular material and human respiratory epithelial cells and, 27 when used with RT-PCR, have reliable sensitivity for detection of viral infection. However, at our 28 institution, we identified a 1.7% re-order rate within 7-days for NPS for respiratory pathogen RT-PCR, 29 which we hypothesize may be due to low confidence in adequate sample collection. We sought to 30 identify an inexpensive and accessible strategy for benchside quality assurance of NPS adequacy by 31 observing microscopic content of viral transport media. For 801 NPS samples collected in November 32 2019, we air-dried and safranin-stained aliquots of viral transport media on glass slides. We then 33 counted morphologically distinct ciliated columnar epithelial cells (CCEs). 19% of samples negative by 34 RT-PCR for respiratory pathogens had no CCEs, while just 6% of positive samples exhibited the same. 35 '},\n"," {'paper_id': '15ad534bb2ec7bc4b0ee83ddac11a1b837d743c6',\n","  'section': 'Abstract',\n","  'text': 'Background: To explore the cellular immunity and cytokines status of NCP patients and to predict 36 the correlation between the cellular immunity levels, cytokines and the severity of patients. 37 Methods: 123 NCP patients were divided into mild and severe groups. Peripheral blood was 38 collected, lymphocyte subsets and cytokines were detected. Correlation analysis was performed on 39 the lymphocyte subsets and cytokines, and the differences between the indexes of the two groups 40 were analyzed. 41 Results: 102 mild and 21 severe patients were included. Lymphocyte subsets were reduced in two 42 groups. The proportion of CD8 + T reduction in the mild and severe group was 28.43% and 61.9%, 43 respectively; The proportion of B cell reduction was 25.49% and 28.57%; The proportion of NK 44 cell reduction was 34.31% and 47.62%; The detection value of IL-6 was 0 in 55.88% of the mild 45 group, mild group has a significantly lower proportion of patients with IL-6 higher than normal than 46 severe group; There was no significant linear correlation between the lymphocyte subsets and 47 cytokines, while significant differences were noticed between the two groups in CD4 + T, CD8 + T, 48 IL-6 and IL-10. 49 Conclusions: Low levels of CD4+T and CD8+T are common in severe NCP. IL-6 and IL-10 levels 50 were higher in severe patients. T cell subsets and cytokines can be used as one of the basis for 51 predicting the transition from mild to severe. Large number of samples are still needed to confirm 52 the \"warning value\" of CD4 + T, CD8 + T IL-6 and IL-10. 53 '},\n"," {'paper_id': '15ba463df9c6f382b8bff4561b5c84617d1f7159',\n","  'section': 'Abstract',\n","  'text': 'Environmental factors, including seasonal climatic variability, can strongly impact on spatio-temporal patterns of infectious disease outbreaks. We assessed the effects of temperature and humidity on the global patterns of Covid-19 early outbreak dynamics during January-March 2020. Climatic variables were the best drivers of global variation of confirmed Covid-19 cases growth rates. Growth rates peaked in temperate regions of the Northern Hemisphere with mean 5 temperature of ~5°C and humidity of ~0.6-1.0 kPa during the outbreak month, while they decreased in warmer and colder regions. The strong relationship between local climate and Covid-19 growth rates suggests the possibility of seasonal variation in the spatial pattern of outbreaks, with temperate regions of the Southern Hemisphere becoming at particular risk of severe outbreaks during the next months. 10 '},\n"," {'paper_id': '15bad8368522a05f32dee771e5a8259f0e9cbbfd',\n","  'section': 'Abstract',\n","  'text': 'The COVID-19 disease has plagued over 110 countries and has resulted in over 4,000 deaths within 10 weeks. We compare the interaction between the human ACE2 receptor and the SARS-CoV-2 spike protein with that of other pathogenic coronaviruses using molecular dynamics simulations. SARS-CoV, SARS-CoV-2, and HCoV-NL63 recognize ACE2 as the natural receptor but present a distinct binding interface to ACE2 and a different network of residue-residue contacts. SARS-CoV and SARS-CoV-2 have comparable binding affinities achieved by balancing energetics and dynamics. The SARS-CoV-2-ACE2 complex contains a higher number of contacts, a larger interface area, and decreased interface residue fluctuations relative to SARS-CoV. These findings expose an exceptional evolutionary exploration exerted by coronaviruses toward host recognition. We postulate that the versatility of cell receptor binding strategies has immediate implications on therapeutic strategies. One Sentence Summary: Molecular dynamics simulations reveal a temporal dimension of coronaviruses interactions with the host receptor. '},\n"," {'paper_id': '15e5dd0eb126bfe384c25ba551317eb9ba2ad1ef',\n","  'section': 'Abstract',\n","  'text': 'Background: COVID-19 outbreak is spreading globally. Although the risk of infection in '},\n"," {'paper_id': '160ecaaa5a766c289fc2f6b5499f0dfe7aab971c',\n","  'section': 'Abstract',\n","  'text': 'Before the lock-down of Wuhan/Hubei/China, on January 23 rd 2020, a large number of individuals infected by COVID-19 moved from the epicenter Wuhan and the Hubei province due to the Spring Festival, resulting in an epidemic in the other provinces including the Shaanxi province. The epidemic scale in Shaanxi was comparatively small and with half of cases being imported from the epicenter. Based on the complete epidemic data including the symptom onset time and transmission chains, we calculate the control reproduction number (1.48-1.69) in Xi\\'an. We could also compute the time transition, for each imported or local case, from the latent, to infected, to hospitalized compartment, as well as the effective reproduction number. This calculation enables us to revise our early deterministic transmission model to a stochastic discrete epidemic model with case importation and parameterize it. Our model-based analyses reveal that the newly generated infections decay to zero quickly; the cumulative number of case-driven quarantined individuals via contact tracing stabilize at a manageable level, indicating that the intervention strategies implemented in the Shaanxi province have been effective. Risk analyses, important for the consideration of \"resumption of work\", show that a large second outbreak is expected if the level of case importation remains at the same level as between January 10 th and February 4 th 2020. However, if the case importation decreases by 30%, 60% and 90%, the second outbreak if happening will be of small-scale assuming contact tracing and quarantine/isolation remain as effective as before. Finally, we consider the effects of intermittent inflow with a Poisson distribution on the likelihood of multiple outbreaks. We believe the developed methodology and stochastic model provide an important model framework for the evaluation of revising travel restriction rules in the consideration of resuming social-economic activities while managing the disease control with potential case importation. '},\n"," {'paper_id': '164b0678afc42f1923aa100b624ca969a53fb3b2',\n","  'section': 'Abstract',\n","  'text': 'Confirmed infection cases in mainland China were analyzed using the data up to January 28, 2020 (first 13 days of reliable confirmed cases). For the first period the cumulative number of cases followed an exponential function. However, from January 28, we discerned a downward deviation from the exponential growth. This slower-than-exponential growth was also confirmed by a steady decline of the effective reproduction number. A backtrend analysis suggested the original basic reproduction number R0 to be about 2.4 to 2.5. As data become available, we subsequently analyzed them during three consecutive periods obtaining a sequence of model predictions. All available data up were processed the same way. We used a simple logistic growth model that fitted very well with all data. Using this model and the three sets of data, we estimated maximum cases as about 21,000, 28,000 and 35,000 cases refining these predictions in near-real time. With slightly different approach (linearization in time) the estimate of maximum cases was even higher (about 65,000). Although the estimates of maximum cases increase as more data were reported all models show reaching a peak in mid-February in contrast to the unconfined exponential growth. These predictions do not account for any possible other secondary sources of infection. '},\n"," {'paper_id': '169886ed560d25fc250346b0eb02b9b5fa73e5f9',\n","  'section': 'Abstract',\n","  'text': 'COVID-19, caused by SARS-CoV-2, has recently affected over 300,000 people and killed more than 10,000. The manner in which the key immune cell subsets change and their states during the course of COVID-19 remain unclear. Here, we applied single-cell technology to comprehensively characterize transcriptional changes in peripheral blood mononuclear cells during the recovery stage of COVID-19. Compared with healthy controls, in patients in the early recovery stage (ERS) of COVID-19, T cells decreased remarkably, whereas monocytes increased. A detailed analysis of the monocytes revealed that there was an increased ratio of classical CD14 ++ monocytes with high inflammatory gene expression as well as a greater abundance of CD14 ++ IL1B + monocytes in the ERS. CD4 + and CD8 + T cells decreased significantly and expressed high levels of inflammatory genes in the ERS. Among the B cells, the plasma cells increased remarkably, whereas the naïve B cells decreased. Our study identified several novel B cell-receptor (BCR) changes, such as IGHV3-23 and IGHV3-7, and confirmed isotypes (IGHV3-15, IGHV3-30, and IGKV3-11) previously used for virus vaccine development. The strongest pairing frequencies, IGHV3-23-IGHJ4, indicated a monoclonal state associated with SARS-CoV-2 specificity. Furthermore, integrated analysis predicted that IL-1β and M-CSF may be novel candidate target genes for inflammatory storm and that TNFSF13, IL-18, IL-2 and IL-4 may be beneficial for the recovery of COVID-19 patients. Our study provides the first evidence of an inflammatory immune signature in the ERS, suggesting that COVID-19 patients are still vulnerable after hospital discharge. Our identification of novel BCR signaling may lead to the development of vaccines and antibodies for the treatment of COVID-19. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. -The immune response was sustained for more than 7 days in the early recovery stage of COVID-19, suggesting that COVID-19 patients are still vulnerable after hospital discharge. -Single-cell analysis revealed a predominant subset of CD14 ++ IL1β + monocytes in patients in the ERS of COVID-19. -Newly identified virus-specific B cell-receptor changes, such as IGHV3-23, IGHV3-7, IGHV3-15, IGHV3-30, and IGKV3-11, could be helpful in the development of vaccines and antibodies against SARS-CoV-2. -IL-1β and M-CSF were discovered as novel mediators of inflammatory cytokine storm, and TNFSF13, IL-2, IL-4, and IL-18 may be beneficial for recovery. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '16f0732b9fee4779a3eb38280b344f866692d346',\n","  'section': 'Abstract',\n","  'text': \"Background: COVID-19 is an emerging disease caused by the SARS-CoV-2 virus; no specific medication has been identified to date. We aimed to investigate the administered medications and intervention times for patients who completely recovered from COVID-19. Methods: This single-center, retrospective, and observational study included 55 patients with COVID-19 who were transferred to Shenyang Sixth People's Hospital between January 20 and March 15, 2020. Demographic information, symptoms, laboratory indicators, treatment processes, and clinical outcomes were collected. Administered drugs and intervention times were compared in 47 and eight patients with mild and severe symptoms, respectively. Findings: All 55 patients recovered. Fifty-three patients (96·36%) received antiviral therapy, including 45 in the mild group (median treatment: 14 days; 17 received umifenovir) and all eight severe-group patients (median treatment: 17·5 days; four received lopinavir/ritonavir). Twenty-nine patients (52·72%) were administered antibiotics, including 21 in the mild group (median treatment: 13·5 days; 15 received moxifloxacin) and all eight in the severe group (median treatment: 9 days; two received linezolid). Moreover, seven patients (12·72%) were treated with glucocorticoids and nine (16·36%) with immunomodulators. \"},\n"," {'paper_id': '16f1fab1289587610e879c3d39ec74c12d42c651',\n","  'section': 'Abstract',\n","  'text': 'To explore longitudinal change patterns of key laboratory tests in patients with COVID-19, and to identify independent prognostic factors by examining the associations between laboratory findings and outcomes of patients. The multicenter study prospectively included 59 patients with COVID-19 treated at Jilin province from January 21, 2020 to May 5, 2020. Laboratory tests were included haematological, biochemical, and immunological tests. Laboratory findings, the characteristics of epidemiological and demographic data were extracted from electronic medical records. Eosinopenia was shown in 52.6% cases at onset, and the average value of eosinophil continued to significantly increase thereafter. Lymphopenia was found in 40.4% cases at onset, and the average value of lymphocyte was slowly elevated after day 5. Thrombocytopenia was shown in 12.3% cases at onset, and the average value of mean platelet volume was decreased sharply after day 7. The values of aspartate aminotransferase, lactate dehydrogenase, creatine kinase, creatinine kinase-muscle/brain activity, and cardiac troponin I, serum cardiac markers, were beyond the upper limit of RI from 6.1% to 30.6% at onset. '},\n"," {'paper_id': '16fc3ea80fcf22241771cba8952982d0faf82a66',\n","  'section': 'Abstract',\n","  'text': 'Relevant pandemic-spread scenario simulations provide guiding principles for containment and mitigation policy developments. Here we devise a simple model to predict the effectiveness of different mitigation strategies. The model consists of a set of simple differential equations considering the population size, reported and unreported infections, reported and unreported recoveries and the number of Covid-19-inflicted deaths. For simplification, we assume that Covid-19 survivors are immune (e.g. mutations are not considered) and that the virus can only be passed on by persons with undetected infections. While the latter assumption is a simplification (it is neglected that e.g. hospital staff may be infected by detected patients with symptoms), it was introduced here to keep the model as simple as possible. Moreover, the current version of the model does not account for age-dependent differences in the death rates, but considers higher mortality rates due to temporary shortage of intensive care units. Some of the model parameters have been fitted to the reported cases outside of China 1 from January 22 to March 12 of the 2020 Covid-19 pandemic. The other parameters were chosen in a plausible range to the best of our knowledge. We compared infection rates, the total number of people getting infected and the number of deaths in six different scenarios. Social distancing or increased testing can contain or drastically reduce the infections and the predicted number of deaths when compared to a situation without mitigation. We find that mass-testing alone and subsequent isolation of detected cases can be an effective mitigation strategy, alone and in combination with social distancing. However, unless one assumes that the virus can be globally defeated by reducing the number of infected persons to zero, testing must be upheld, albeit at reduced intensity, to prevent subsequent waves of infection. The model suggests that testing strategies can be equally effective as social distancing, though at much lower economical costs. We discuss how our mathematical model may help to devise an optimal mix of mitigation strategies against the Covid-19 pandemic. The website corona-lab.ch provides an interactive simulation tool based on the presented model. The Covid-19 pandemic is a serious threat that can be mitigated by different strategies. Social isolation is currently practiced in most countries in Europe, at least until alternative mitigation strategies can be implemented or therapies and vaccines become available. We have formulated a simple mathematical model to estimate how well social isolation or improved identification of Covid-19 infected people can slow down virus spread and reduce the death toll. '},\n"," {'paper_id': '16ffae6b21685dfcd71c62398ba65adefc9b5d44',\n","  'section': 'Abstract',\n","  'text': 'Objective: To test the hypothesis that knowledge of COVID-19 influences participation in different behaviors including self-reports of purchasing more goods than usual, attending large gatherings, and using medical masks. Methods: Cross-sectional online survey of 1,034 U.S. residents age 18+ conducted on March 17, 2020. Results: For every point increase in knowledge, the odds of participation in purchasing more goods (OR=0.88, 95% CI:0.81-0.95), attending large gatherings (OR=0.87, 95%CI: 0.81-0.93), and using medical masks (OR=0.56, 95% CI:0.50-0.62) decreased by 12%, 13%, and 44%, respectively. Gen X and Millennial participants had 56% to 76% higher odds, respectively, of increased purchasing behavior, compared to Baby Boomers. Results suggest politicization of response recommendations. Democrats had 30% lower odds of attending large gatherings (OR=0.70, 95% CI:0.50-0.97), and 48% lower odds of using medical masks (OR=0.52, 95% CI:0.34-0.78), compared to Republicans. '},\n"," {'paper_id': '17307ab6eda64e1538da204c02114dd36a9b7fd6',\n","  'section': 'Abstract',\n","  'text': 'The outbreak of the 2019 Novel Coronavirus (SARS-CoV-2) rapidly spread from Wuhan, China to more than 150 countries, areas or territories, causing staggering number of infections and deaths. A systematic profiling of the immune vulnerability landscape of SARS-CoV-2, which can bring critical insights into the immune clearance mechanism, peptide vaccine development, and antiviral antibody development, is lacking. In this study, we investigated the potential of the SARS-CoV-2 viral proteins to induce class I and II MHC presentation and to form linear antibody epitopes. We created an online database to broadly share the predictions as a resource for the research community. Using this resource, we showed that genetic variations in SARS-CoV-2, though still few for the moment, already follow the pattern of mutations in related coronaviruses, and could alter the immune vulnerability landscape of this virus. Importantly, we discovered evidence that SARS-CoV-2, along with related coronaviruses, used mutations to evade attack from the human immune system. Overall, we present an immunological resource for SARS-CoV-2 that could promote both therapeutic development and mechanistic research. : bioRxiv preprint Chinese and European populations. (c) The B cell epitope profiles of the SARS-CoV-2. The Y axis shows the predicted B cell epitope score. Only showing residues with predicted epitope score >0.6. (d) BLASTing the motif binding domain of the SARS-CoV-2 S protein and the SARS-CoV S protein. (e) The T cell and B cell epitope profiles of SARS-CoV. (f) The T cell and B cell epitope profiles of MERS-CoV. Fig. 2 Genetic drifts in SARS-CoV-2 are influenced by host immune pressure. (a-c) The relative mutational rate profiles of the three coronaviruses: (a) SARS-CoV-2, (b) SARS-CoV, and (c) MERS-CoV. The relative mutational rates are calculated by the percentage of nucleotides (nts) from all strains of each virus (n=114 for SARS-CoV-2, 19 for SARS-CoV-, and 519 for MERS-CoV) in each 60-nt bin that are different from the reference genome. The semi-transparent boxes mark the regions of high mutational rates due to artefacts of incomplete sequencing. These regions are shielded from calculations of gain and loss of immunogenicity. (d-f) The \"lost\" immunogenicity and \"gained\" immunogenicity due to mutations in each SARS-CoV-2 isolate compared with the reference genome. (d) CD8 + T epitopes, (e) CD4 + T epitopes, and (f) B cell epitopes. The red line indicates the total number of predicted B cell epitope-encoding amino acids in the reference sequence. (g-i) The same immunogenicity change analyses as in (d-f), but for SARS-CoV. (j-l) The same analysis for MERS-CoV. Sup. Fig. 1 The T cell epitope profiles of the European population. (a) SARS-CoV-2, (b) SARS-CoV, and (c) MERS-CoV. Sup. Fig. 2 The variation of T cell epitope profiles for SARS-CoV-2, SARS-CoV and MERS-CoV across populations. The heatmap represents the number of immunogenic binding epitopes across the binned genomes (500bp) of (a) SARS-CoV-2, (b) SARS-CoV and (c) MERS-CoV for the major HLA-A alleles shown as examples (allele frequency larger than 1%) in the European American (EA) and Hongkong Chinese (HK) populations. These major alleles are colored in black, blue or red if they are common to both EA and HK population, unique to EA population, or unique to HK population, respectively. On the right, the band of strength represents the cumulative number of immunogenic peptides, and the bands of EA and HK represent the HLA allele frequency of EA and HK populations, respectively. Fig. 3 Part of the multiple alignment results of all the SARS-CoV-2 strains. Fig. 4 B cell epitope-encoding amino acids in contiguous stretches of >10 epitope-encoding amino acids. (a) SARS-CoV-2, (b) SARS-CoV, and (c) MERS-CoV. Red lines denote the total numbers of epitope-encoding amino acids of the reference sequences. Sup. Sup. Fig. 5 The trend of immunogenicity to SARS-CoV-2 over time. (a) \"Gained/lost\" T cell immunogenicity (class I). (b) \"Gained/lost\" T cell immunogenicity (class II). (c) Changes in B cell immunogenicity (all epitope-encoding residues). (d) Changes in B cell immunogenicity (epitope-encoding residues in stretches of >10 epitope-encoding residues). The strains were grouped according to their collection time of '},\n"," {'paper_id': '17643a57b92abf0d6635d308949e78de2ecc7f66',\n","  'section': 'Abstract',\n","  'text': \"The WHO has declared SARS-CoV-2 outbreak a public health emergency of international concern. However, to date, there was hardly any study in characterizing the immune responses, especially adaptive immune responses to SARS-CoV-2 infection. In this study, we collected blood from COVID-19 patients who have recently become virus-free and therefore were discharged, and analyzed their SARS-CoV-2-specific antibody and T cell responses. We observed SARS-CoV-2-specific humoral and cellular immunity in the patients. Both were detected in newly discharged patients, suggesting both participate in immune-mediated protection to viral infection. However, follow-up patients (2 weeks post discharge) exhibited high titers of IgG antibodies, but with low levels of virus-specific T cells, suggesting that they may enter a quiescent state. Our work has thus provided a basis for further analysis of protective immunity to SARS-CoV-2, and understanding the pathogenesis of COVID-19, especially in the severe cases. It has also implications in designing an effective vaccine to protect and treat SARS-CoV-2 infection. : medRxiv preprint PBMCs were washed with PBS plus 2% FBS (Gibco, Grand Island, NY), and then Fc blocking reagent (Meltenyi Biotec, Inc., Auburn, CA) was added followed by a wash with PBS plus 2% FBS. Cells were then incubated for 30 min on ice with anti-CD45 (H130) (BioLegend), anti-CD3 (OKT3) (BioLegend), anti-CD8 (SK1) (BD), anti-CD56 (HCD56) (BioLegend), anti-All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. : medRxiv preprint 9 CD38 (HIT2) (BioLegend) and live/dead fixable aqua dye (eF660, eBioscience), washed twice with PBS plus 2% FBS and then stored at 4 °C until acquired by FACS Verse (BD Biosciences, San Jose, CA). Data were analyzed using FlowJo software (Version 10.0.8, Tree Star Inc., Ashland, Or). Prism 7 software is used for statistical analysis. Student's t test was performed for twogroup analysis. Pearson's correlation coefficients were calculated. P values less than 0.05 were considered to be statistically significant. \"},\n"," {'paper_id': '179df1e769292dd113cef1b54b0b43213e6b5c97',\n","  'section': 'Abstract',\n","  'text': 'COVID-19, a novel coronavirus outbreak starting in China, is now a rapidly developing public health emergency of international concern. The clinical spectrum of COVID-19 disease is varied, and identifying factors associated with severe disease has been described as an urgent research priority. It has been noted that elderly patients with pre-existing comorbidities are more vulnerable to more severe disease. However, the specific symptoms and comorbidities that most strongly predict disease severity are unclear. We performed a systematic review and meta-analysis to identify the symptoms and comorbidities predictive of COVID-19 severity. This study was prospectively registered on PROSPERO. A literature search was performed in three databases (MEDLINE, EMBASE and Global Health) for studies indexed up to 5 th March 2020. Two reviewers independently screened the literature and both also completed data extraction. Quality appraisal of studies was performed using the STROBE checklist. Random effects meta-analysis was performed for selected symptoms and comorbidities to identify those most associated with severe COVID-19 infection or ICU admission. Of the 2259 studies identified, 42 were selected after title and abstract analysis, and 7 studies (including 1813 COVID-19 patients) were chosen for inclusion. The ICU group were older (62.4 years) compared to the non-ICU group (46 years), with a significantly higher proportion of males (67.2% vs. 57.1%, p=0.04). Dyspnoea was the only significant symptom predictive for both severe disease (pOR 3.70, 95% CI 1.83 -7.46) and ICU admission (pOR 6.55, 95% CI 4.28-10.0). Notwithstanding the low prevalence of COPD in severe disease and ICUadmitted groups (4.5% and 9.7%, respectively), COPD was the most strongly predictive comorbidity for both severe disease (pOR 6.42, 95% CI 2. 44 -16.9) and ICU admission (pOR 17.8, 95% CI 6.56 -48.2). Cardiovascular disease and All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint (which was not peer-reviewed) is . https://doi.org/10. 1101 3 hypertension were also strongly predictive for both severe disease and ICU admission. Those with CVD and hypertension were 4.4 (95% CI 2.64 -7.47) and 3.7 (95% CI 2.22 -5.99) times more likely to have an ICU admission respectively, compared to patients without the comorbidity. Dyspnoea was the only symptom strongly predictive for both severe disease and ICU admission, and could be useful in guiding clinical management decisions early in the course of illness. When looking at ICU-admitted patients, who represent the more severe end of the spectrum of clinical severity, COPD patients are particularly vulnerable, and those with cardiovascular disease and hypertension are also at a high-risk of severe illness. To aid clinical assessment, risk stratification, efficient resource allocation, and targeted public health interventions, future research must aim to further define those at high-risk of severe illness with COVID-19. '},\n"," {'paper_id': '17bc8a3fc60e75b5c789089db53fc8c609737fd3',\n","  'section': 'Abstract',\n","  'text': 'Background: GenBank contains over 3 million viral sequences. The National Center for Biotechnology Information (NCBI) previously made available a tool for validating and annotating influenza virus sequences that is used to check submissions to GenBank. Before this project, there was no analogous tool in use for non-influenza viral sequence submissions. Results: We developed a system called VADR (Viral Annotation DefineR) that validates and annotates viral sequences in GenBank submissions. The annotation system is based on the analysis of the input nucleotide sequence using models built from curated RefSeqs. Hidden Markov models are used to classify sequences by determining the RefSeq they are most similar to, and feature annotation from the RefSeq is mapped based on a nucleotide alignment of the full sequence to a covariance model. Predicted proteins encoded by the sequence are validated with nucleotide-to-protein alignments using BLAST. The system identifies 43 types of \"alerts\" that (unlike the previous BLAST-based system) provide deterministic and rigorous feedback to researchers who submit sequences with unexpected characteristics '},\n"," {'paper_id': '17de28ebcf5e84031aec3acd10e45e62bc14f594',\n","  'section': 'Abstract',\n","  'text': 'The severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has progressed to a pandemic associated with substantial morbidity and mortality. The WHO and the United States Center for Disease Control and Prevention (CDC) have issued interim clinical guidance for management of patients with confirmed coronavirus disease , but there is limited data on the virologic and clinical characteristics for prognosis of severe COVID-19. A total of 50 patients with severe COVID-19 were divided into good and poor recovery groups. The dynamic viral shedding and serological characteristics of SARS-CoV-2 were explored. The risk factors associated with poor recovery and lung lesion resolutions were identified. In addition, the potential relationships among the viral shedding, the pro-inflammatory response, and lung lesion evolutions were characterized. A total of 58% of the patients had poor recovery and were more likely to have a prolonged interval of viral shedding. The longest viral shedding was 57 days after symptom onset. Older age, hyperlipemia, hypoproteinemia, corticosteroid therapy, consolidation on chest computed-tomography (CT), and prolonged SARS-CoV-2 IgM positive were all associated with poor recovery. Additionally, the odds of impaired lung lesion resolutions were higher in patients with hypoproteinemia, hyperlipemia, and elevated levels of IL-4 and ferritin. Finally, viral shedding and proinflammatory responses were closely correlated with lung lesion evolutions on chest CT. : medRxiv preprint Patients with severe COVID-19 have prolonged SARS-CoV-2 infection and delayed intermittent viral shedding. Older age, hyperlipemia, hypoproteinemia, corticosteroid usage, and prolonged SARS-CoV-2 IgM positive might be utilized as predicative factors for the patients with poor recovery. '},\n"," {'paper_id': '17e5527defb9efc3bcd2e23b0d491cbc9be93670',\n","  'section': 'Abstract',\n","  'text': 'In the current COVID-19 crisis, the US and many countries in the world are suffering acute shortages of modern ventilators to care for desperately ill patients. Since modern ICU ventilators are powerful devices that can deliver very high gas flow rates and pressures, multiple physicians have attempted to ventilate more than one patient on a single ventilatorso-called \"vent splitting\". Early applications of this approach have utilized simple concatenations of ventilator tubing and T-pieces, to provide flow to more than one patient. Additional approaches using custom flow splitters -sometimes made using 3D printing technologies -have also advanced into the clinic with FDA approval. However, heretofore there has been less progress made on controlling individual ventilatory pressures for patients with severe lung disease. Given the inherent variability and instability of lung compliance amongst patients with COVID-19, there remains an important need to provide a means of extending ventilator usefulness to more than one patient, but in a way that provides more tailored pressures that can be titrated over time. In this descriptive report, we provide the basis for a ventilator circuit that can support two patients with individualized peak inspiratory and end-expiratory pressures. The circuit is comprised of exclusively \"off the shelf\" materials and is inexpensive to produce. The circuit can be used with typical ICU ventilators, and with anesthesia ventilators used in operating rooms. Inspiratory and end-expiratory pressures for each patient can be titrated over time, without changes for one patient affecting the ventilation parameters of the other patient. Using in-line spirometry, individual tidal volumes can be measured for each patient. This Pressure-Regulated Ventilator Splitting (PReVentS) Yale University protocol operates under a pressure-control ventilatory mode, and may function optimally when patients are not triggering breaths from the ventilator. This method has been tested thus far only in the laboratory with mock lungs, and has not yet been deployed in animals or in patients. However, given the novelty and potential utility of this approach, we deemed it appropriate to provide this information to the broader critical care community at the present time. In coming days and weeks, we will continue to characterize and refine this approach, using large animal models and proof-of-principle human studies. '},\n"," {'paper_id': '180c88b68816bbb6eb57aa515a3aec4020178729',\n","  'section': 'Abstract',\n","  'text': '20 E-probe Diagnostic for Nucleic acid Analysis (EDNA) is a bioinformatic tool originally 21 developed to detect plant pathogens in metagenomic databases. However, 22 enhancements made to EDNA increased its capacity to conduct hypothesis directed 23 detection of specific gene targets present in transcriptomic databases. To target specific 24 pathogenicity factors used by the pathogen to infect its host or other targets of interest, 25 e-probes need to be developed for transcripts related to that function. In this study, 26 EDNA transcriptomics (EDNAtran) was developed to detect the expression of genes 27 related to aflatoxin production at the transcriptomic level. E-probes were designed from 28 genes up-regulated during A. flavus aflatoxin production. EDNAtran detected gene 29 transcripts related to aflatoxin production in a transcriptomic database from corn, where 30 aflatoxin was produced. The results were significantly different from e-probes being 31 used in the transcriptomic database where aflatoxin was not produced (atoxigenic AF36 32 strain and toxigenic AF70 in Potato Dextrose Broth). 33 34 Maize [1], peanuts [2], tree nuts, dried spices [3] and cottonseed [4] are crops that can 35 be infected during the pre-harvest, post-harvest and/or storage period with Aspergillus 36 flavus Link. This fungus produces polyketide secondary metabolites named aflatoxins. 37 '},\n"," {'paper_id': '181b7b57851e6f58a601b68e613d10c10616f774',\n","  'section': 'Abstract',\n","  'text': 'The beginning of 2020 has seen the emergence of COVID-19 outbreak caused by a novel coronavirus, Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2). There is an imminent need to better understand this new virus and to develop ways to control its spread. In this study, we sought to gain insights for vaccine design against SARS-CoV-2 by considering the high genetic similarity between SARS-CoV-2 and SARS-CoV, which caused the outbreak in 2003, and leveraging existing immunological studies of SARS-CoV. By screening the experimentally-determined SARS-CoV-derived B cell and T cell epitopes in the immunogenic structural proteins of SARS-CoV, we identified a set of B cell and T cell epitopes derived from the spike (S) and nucleocapsid (N) proteins that map identically to SARS-CoV-2 proteins. As no mutation has been observed in these identified epitopes among the available SARS-CoV-2 sequences (as of 9 February 2020), immune targeting of these epitopes may potentially offer protection against this novel virus. For the T cell epitopes, we performed a population coverage analysis of the associated MHC alleles and proposed a set of epitopes that is estimated to provide broad coverage globally, as well as in China. Our findings provide a screened set of epitopes that can help guide experimental efforts towards the development of vaccines against SARS-CoV-2. : bioRxiv preprint B cell responses in recovered patients with severe acute respiratory syndrome: A six-year follow-up study. '},\n"," {'paper_id': '189e553cea69b8eca43a7c51399bf9a327538c97',\n","  'section': 'Abstract',\n","  'text': 'The rapid spread of COVID-19 is a global public health challenge. To prevent the escalation of its transmission, China locked down one-third of its cities and strictly restricted human mobility and economic activities. Using timely and comprehensive air quality data in China, we show that these counter-COVID-19 measures led to remarkable improvement in air quality. Within weeks, the Air Quality Index and PM2.5 concentrations were brought down by 25%. The effects are larger in colder, richer, and more industrialized cities. We estimate that such improvement would avert 24,000 to 36,000 premature deaths from air pollution on a monthly basis. : medRxiv preprint Notes: Each column summarizes the mean values and standard deviations of each variable at a weekly level. More information about the dataset is described in Appendix 1. '},\n"," {'paper_id': '189ebc4f63c934960c6627e9b4138955ffe12f1d',\n","  'section': 'Abstract',\n","  'text': 'CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': '18d2115a15ed7d4f2be6ffe83165bfe3859b3462',\n","  'section': 'Abstract',\n","  'text': 'The ongoing SARS-CoV-2 outbreak has killed over twenty-one thousand and sickened over four 20 hundred thousand people worldwide, posing a great challenge to global public health. A sensitive and 21 accurate diagnosis method will substantially help to control disease expansion. Here, we developed a 22 chemiluminescence-immunoassay method based on the recombinant nucleocapsid antigen and the 23 magnetic beads for diagnosis of SARS-CoV-2 infections and surveillance of antibody changing pattern. . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . : medRxiv preprint 36 serological testing tool for the diagnosis of SARS-CoV-2 infections in the community. 37  '},\n"," {'paper_id': '18dc6a67045d2687d2b5c11c85797f42824ed243',\n","  'section': 'Abstract',\n","  'text': 'Background: The current outbreak caused by novel coronavirus (2019-nCoV) in China has become a worldwide concern. As of 28 January 2020, there were 4631 confirmed cases and 106 deaths, and 11 countries or regions were affected. We downloaded the genomes of 2019-nCoVs and similar isolates from the Global Initiative on Sharing Avian Influenza Database (GISAID and nucleotide database of the National Center for Biotechnology Information (NCBI). Lasergene 7.0 and MEGA 6.0 softwares were used to calculate genetic distances of the sequences, to construct phylogenetic trees, and to align amino acid sequences. Bayesian coalescent phylogenetic analysis, implemented in the BEAST software package, was used to calculate the molecular clock related characteristics such as the nucleotide substitution rate and the most recent common ancestor (tMRCA) of 2019-nCoVs. Results: An isolate numbered EPI_ISL_403928 showed different phylogenetic trees and genetic distances of the whole length genome, the coding sequences (CDS) of ployprotein (P), spike protein (S), and nucleoprotein (N) from other 2019-nCoVs. There are 22, 4, 2 variations in P, S, and N at the level of amino acid residues. The nucleotide substitution rates from high to low are 1·05 × 10 -2 (nucleotide substitutions/site/year, with 95% HPD interval being 6. '},\n"," {'paper_id': '197fdd3ee5d64808d5079bf839cfcc6b6e03ac75',\n","  'section': 'Abstract',\n","  'text': 'Coronavirus COVID-19 spreads through the population mostly based on social contact. To gauge the potential for widespread contagion, to cope with associated uncertainty and to inform its mitigation, more accurate and robust modelling is centrally important for policy making. We provide a flexible modelling approach that increases the accuracy with which insights can be made. We use this to analyse different scenarios relevant to the COVID-19 situation in the UK. We present a stochastic model that captures the inherently probabilistic nature of contagion between population members. The computational nature of our model means that spatial constraints (e.g., communities and regions), the susceptibility of different age groups and other factors such as medical pre-histories can be incorporated with ease. We analyse different possible scenarios of the COVID-19 situation in the UK. Our model is robust to small changes in the parameters and is flexible in being able to deal with different scenarios. This approach goes beyond the convention of representing the spread of an epidemic through a fixed cycle of susceptibility, infection and recovery (SIR). It is important to emphasise that standard SIR-type models, unlike our model, are not flexible enough and are also not stochastic and hence should be used with extreme caution. Our model allows both heterogeneity and inherent uncertainty to be incorporated. Due to the scarcity of verified data, we draw insights by calibrating our model using parameters from other relevant sources, including agreement on average (mean field) with parameters in SIR-based models. We use the model to assess parameter sensitivity for a number of key variables that characterise the COVID-19 epidemic. We also test several control parameters with respect to their influence on the severity of the outbreak. Our analysis shows that due to inclusion of spatial heterogeneity in the population and the asynchronous timing of the epidemic across different areas, the severity of the epidemic might be lower than expected from other models. We find that one of the most crucial control parameters that may significantly reduce the severity of the epidemic is the degree of separation of vulnerable people and people aged 70 years and over, but note also that isolation of other groups has an effect on the severity of the epidemic. It is important to remember that models are there to advise and not to replace reality, and that any action should be coordinated and approved by public health experts with experience in dealing with epidemics. The computational approach makes it possible for further extensive scenario-based analysis to be undertaken. This and a comprehensive study of sensitivity of the model to different parameters defining COVID-19 and its development will be the subject of our forthcoming paper. In that paper, we shall also extend the model where we will consider different probabilistic scenarios for infected people with mild and severe cases. '},\n"," {'paper_id': '1983cbc32412a9a7b1601b6778605f9e98ab69e0',\n","  'section': 'Abstract',\n","  'text': 'The mechanisms underlying multicellular development in the animal stem lineage may 25 be reconstructed through the study of choanoflagellates, the closest living relatives of 26 animals. To determine the genetic underpinnings of multicellularity in the emerging 27 model choanoflagellate S. rosetta, we performed a screen for mutants with defects in 28 multicellular rosette development. In two of the mutants, Jumble and Couscous, single 29 cells failed to develop into orderly rosettes but instead aggregated promiscuously into 30 amorphous clumps of cells. Both mutants mapped to lesions in genes encoding 31 glycosyltransferases and the mutations perturbed glycosylation patterns in the 32 extracellular matrix (ECM). In animals, glycosyltransferases transfer activated sugars to 33 donor molecules and thereby sculpt the polysaccharide-rich ECM, regulate integrin and 34 cadherin activity, and, when disrupted, contribute to tumorigenesis. The finding that 35 glycosyltransferases promote proper rosette development and prevent cell aggregation 36 in S. rosetta suggests a pre-metazoan role for glycosyltransferases in regulating 37 development and preventing abnormal tumor-like multicellularity. '},\n"," {'paper_id': '19a08cc1423d6006b2251bdcfd142f88db7002e7',\n","  'section': 'Abstract',\n","  'text': 'The coronavirus disease 2019 has become a global pandemic currently. Many drugs showed potential for COVID-19 therapy. However, genetic factors which can lead to different drug efficiency and toxicity among populations are still undisclosed in COVID-19 therapy. We selected 67 potential drugs for COVID-19 therapy (DCTs) from clinical guideline and clinical trials databases. 313 pharmaco-genes related to these therapeutic drugs were included. Variation information in 125,748 exomes were collected for racial differences analyses. The expression level of pharmaco-genes in single cell resolution was evaluated from single-cell RNA sequencing (scRNA-seq) data of 17 healthy adults. Results: Pharmacogenes, including CYP3A4, ABCB1, SLCO1B1, ALB, CYP3A5, were involved in the process of more than multi DCTs. 224 potential drug-drug interactions (DDIs) of DCTs were predicted, while 112 of them have been reported. Racial discrepancy of common nonsynonymous mutations was found in pharmacogenes including: VDR, ITPA, G6PD, CYP3A4 and ABCB1 which related to DCTs including ribavirin, α -interferon, chloroquine and lopinavir. Moreover, ACE2, the target of 2019-nCoV, was only found in parts of lung cells, which makes drugs like chloroquine that prevent virus binding to ACE2 more specific than other targeted drugs such as camostat mesylate. '},\n"," {'paper_id': '19bc9648d12dc6b7ca757f82dcd182c03111691e',\n","  'section': 'Abstract',\n","  'text': 'Clinical characteristics of novel coronavirus disease (COVID-19) have been described in numerous studies but yielded varying results. We aimed to conduct a systematic review on scientific literatures and to synthesize critical data on clinical traits of COVID-19 from its initial outbreak to pandemic. Systematic searches were conducted to identify retrospective observational study that contained clinical characteristics on COVID-19 through multiple databases. Two reviewers independently evaluated eligible publications. Data on clinical characteristics of COVID-19 were extracted and analyzed. RESULTS -two retrospective studies demonstrating the clinical characteristics of COVID-19 were included. A total of 3470 COVID-19 patients were synthesized to the final analysis in an unbiased manner. The most common symptom was fever (2878 [83.0%]), and 63.4% of the patients presented fever as onset symptom. There were 2528 [88.2%] of 2866 cases had abnormal lung findings on chest CT scan. Laboratory findings showed that 1498 [62.8%] of 2387 cases had lymphopenia, and 1354 [64.8%] of 2091 cases had an increased level of C-reactive protein (CRP). A total of 185 [11.5%] patients were admitted to intensive care unit (ICU) while the overall case fatality rate (CFR) was 3.7%. Compared to patients admitted outside of Hubei, China, those from Hubei had a significant higher ICU admission rate (21.9% vs. 2.5%, p<0.001). Also, CFR attributed to COVID-19 was significantly higher in Hubei than that of non-Hubei admissions (10.4% vs. 0.6%, p<0.001). This large patient-based systematic review presents a more precise profiling of the COVID-19 from its outbreak to current pandemic. Dynamic evolvements of COVID-19 are needed to be characterized in future studies. '},\n"," {'paper_id': '19e096b2ffdf2d39fa0e28a263bc20a3323bcb5f',\n","  'section': 'Abstract',\n","  'text': 'Risk indicators viral load (ORF1ab Ct), lymphocyte percentage (LYM%), C-reactive protein (CRP), interleukin-6 (IL-6), procalcitonin (PCT) and lactic acid (LA) in COVID-19 patients have been proposed in recent studies. However, the predictive effects of those indicators on disease classification and prognosis remains largely unknown. We dynamically measured those reported indicators in 132 cases of COVID-19 patients including the moderate-cured (moderated and cured), severe-cured (severe and cured) and critically ill (died). Our data showed that CRP, PCT, IL-6, LYM%, lactic acid and viral load could predict prognosis and guide classification of COVID-19 patients in different degrees. CRP, IL-6 and LYM% were more effective than other three factors in predicting prognosis. For disease classification, CRP and LYM% were sensitive in identifying the types between critically ill and severe (or moderate). Notably, among the investigated factors, LYM% was the only one that could distinguish between the severe and moderate types. Collectively, we concluded that LYM% was the most sensitive and reliable predictor for disease typing and prognosis. During the COVID-19 pandemic, the precise classification and prognosis prediction are critical for saving the insufficient medical resources, stratified treatment and improving the survival rate of critically ill patients. We recommend that LYM% be used independently or in combination with other indicators in the management of COVID-19. '},\n"," {'paper_id': '1a12cc7d49d8521bec5d447ebb413cbcf5aca8f4',\n","  'section': 'Abstract',\n","  'text': 'Objective: This paper studied the relationship between personality traits and mental health conditions of medical personnel to provide a basis and reference for the implementation of targeted education on mental health. A self-report inventory, the Symptom Checklist-90 (SCL-90), was used to investigate the mental health status of 548 medical personnel dealing with the new coronavirus pneumonia in eight provinces and cities of China. The overall mean SCL-90 score and mean values of factors (somatization, obsessive-compulsive, anxiety, phobic anxiety, and psychoticism) of the medical personnel were significantly higher than in the norm group (p < 0.05), while their average interpersonal sensitivity score was significantly lower (p < 0.01). In addition, personal factors affecting the mental health status of medical personnel were identified. ( all p < 0.05). The overall mental health status of medical personnel responding to new coronavirus pneumonia is generally higher than that of the norm group in China. The results of this study should contribute to measures to alleviate the psychological pressures on medical personnel dealing with the new coronavirus epidemic in China. [Keywords] medical personnel; new coronavirus pneumonia; SCL-90 inventory; mental health; influencing factors All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '1a48450da54865731ffe01c0e289a63065b3b1fd',\n","  'section': 'Abstract',\n","  'text': 'With evidence of sustained transmission in more than 190 countries, coronavirus disease 2019 has been declared a global pandemic. As such, data are urgently needed about risk factors associated with clinical outcomes. A retrospective chart review of 323 hospitalized patients with COVID-19 in Wuhan was conducted. Patients were classified into three disease severity groups (non-severe, severe, and critical), based on their initial clinical presentation. Clinical outcomes were designated as favorable and unfavorable, based on disease progression and response to treatments. Logistic regression models were performed to identify factors associated with clinical outcomes, and logrank test was conducted for the association with clinical progression. Current standard treatments did not show significant improvement on patient outcomes in the study. By univariate logistic regression model, 27 risk factors were significantly associated with clinical outcomes. Further, multivariate regression indicated that age over 65 years, smoking, critical disease status, diabetes, high hypersensitive troponin I (>0.04 pg/mL), leukocytosis (>10 x 10 9 /L) and neutrophilia (>75 x 10 9 /L) predicted unfavorable clinical outcomes. By contrast, the use of hypnotics was significantly associated with favorable outcomes. Survival analysis also confirmed that patients receiving hypnotics had significantly better survival. To our knowledge, this is the first indication that hypnotics could be an effective ancillary treatment for COVID-19. We also found that novel risk factors, such as higher hypersensitive . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': '1a791c5c6543aca9391b77ae58b79d7ddf3c1f3c',\n","  'section': 'Abstract',\n","  'text': \"Sialic acids (Sia) are the primary receptors for influenza viruses, and are widely displayed on cell surfaces and in secreted mucus. Sia may be present in variant forms that include O-acetyl modifications at C4, C7, C8, and C9 positions, and N-acetyl or N-glycolyl at C5. They can also vary in their linkages, including α2-3 or α2-6-linkages. Here, we analyzed the distribution of modified Sia in cells and tissues of wild-type mice, or in mice lacking cytidine 5'-30 monophosphate-N-acetylneuraminic acid hydroxylase (CMAH) enzyme that synthesizes Nglycolyl modifications (Neu5Gc). We also examined the variation of Sia forms on erythrocytes and saliva from different animals. To determine the effect of Sia modifications on influenza A virus (IAV) infection, we tested for effects on hemagglutinin (HA) binding and neuraminidase (NA) cleavage. We confirmed that 9-O-acetyl, 7,9-O-acetyl, 4-O-acetyl, and Neu5Gc 35 modifications are widely but variably expressed in mouse tissues, with the highest levels detected in the respiratory and gastrointestinal tracts. Secreted mucins in saliva and surface proteins of erythrocytes showed a great degree of variability in display of modified Sia between different species. IAV HA from different virus strains showed consistently reduced binding to both Neu5Gc and O-acetyl modified Sia; however, while IAV NA were inhibited by Neu5Gc and 40 O-acetyl modifications, there was significant variability between NA types. The modifications of Sia in mucus may therefore have potent effects on the functions of IAV, and may affect both pathogens and the normal flora of different mucosal sites. (241 words) IMPORTANCE 45 Sialic acids (Sia) are involved in many different cellular functions and are receptors for many pathogens. Sia come in many chemically modified forms but we lack a clear understanding of how they alter the interactions with microbes. Here we examine the expression of modified Sia in mouse tissues, on secreted mucus in saliva, and on erythrocytes, including those from IAV host species and animals used in IAV research. These Sia forms varied considerably between \"},\n"," {'paper_id': '1a9c229f4f866db4f9c23a77b8c1275b52407c64',\n","  'section': 'Abstract',\n","  'text': 'Early in planta detection of Xanthomonas axonopodis pv. punicae in pomegranate using 2 enhanced loop-mediated isothermal amplification assay 3 4 Abstract 33 Bacterial blight in pomegranate caused by Xanthomonas axonopodis pv. punicae (Xap) is an 34 increasing threat for pomegranate cultivation in India. To prevent the economic losses, it is pivotal 35 to detect the infection in latent stages rather than in later stages. We have developed an enhanced 36 method termed as loop-mediated isothermal amplification (LAMP) technique to evaluate for the 37 latent detection of Xap in pomegranate using six set of specific primers. Three DNA intercalating 38 dyes were used, such as Ethidium bromide, hydroxynaphthol blue (HNB) and SYBR Green which 39 resulted in visualising the positivity for LAMP assay. The reaction time and temperature were to 40 be 65°C from 30 min onwards, for the dyes and its sensitivity was observed up to 10 -7 ng in the 41 LAMP assay. For field applicability, LAMP assay detected Xap on 7 th day post infection while the 42 PCR amplified Xap after 11 th day post infection. Finally, the specificity of LAMP assay was 43 validated to be positive with ten Xap isolates for its accuracy and 29 non-Xap bacterial isolates 44 showed negative results. Moreover, this method could be used as a better alternative to PCR based 45 methods, for early detection of the pathogens. 46 47 50 Pomegranate (Punica granatum L.) is an important fruit crop widely cultivated from tropical to 51 temperate conditions [1]. India ranks first in the world for maximum area under pomegranate 52 cultivation (0.125 million hectares) and Maharashtra, Karnataka and Andhra Pradesh are the major 53 pomegranate growing states [2]. The bacterial blight caused by Xanthomonas axonopodis pv. 54 punicae (Xap) in pomegranate was first reported in India in 1952 and presently it has a serious 55 outbreak in all major pomegranate-growing states resulting in a major economic loss. Prominent 56 small water-soaked and dark irregular spots appearing on the leaf are the typical symptoms induced 57 by this pathogen. The pathogen infect the fruit, stem, branches causing cracking symptoms and 58 ultimately leads to yield loss up to 80 percent [3]. . CC-BY 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/328674 doi: bioRxiv preprint 59 Globally, it is estimated that a minimum 30-40% of the crop cultivated gets destroyed 60 annually due to pests and diseases [4]. In India, bacterial blight alone reduces the yield between 61 60 to 80 percent above global yield loss in pomegranate. To avoid yield loss, it is very important 62 and necessary to develop an efficiently sensitive pathogen detection method within incubation time 63 of the pathogen (i.e. time between the host infection and expression of the disease symptoms) 64 [5,6]. 65 The most common methods employed for detecting the plant pathogens include 66 immunological techniques like ELISA (enzyme-linked immunosorbent assay) [7] and molecular 67 methods such as PCR (polymerase chain reaction) [8] and Southern blotting [9]. The methods for 68 early detection of the plant pathogens in the field has to be designed in such a way that it has to be 69 competently very sensitive, specific, cost-effective and detectable with a minimal equipment 70 facilities [10]. For instance, the PCR based technique has its own drawbacks which require high 71 quality and quantity of DNA and amplification is complex that requires a thermal cycler for 72 changing the reaction temperature and requires relatively long time for the amplification process 73 to complete. Further, it also requires agarose gel electrophoresis and gel documentation to confirm 74 the amplification [11]. To overcome these limitations, researchers have shown interest in 75 developing an isothermal amplification technique termed as Loop-mediated isothermal 76 amplification (LAMP) [12]. LAMP is an auto-cycling strand displacement DNA synthesis 77 technique, which is carried out at a constant temperature and does not require a thermal-cycler. 78 The amplification is completed within 30-60 minutes at a constant temperature of 60-65º C, which 79 can be carried out in a simple water bath or heating block [13]. Moreover, LAMP is a cost effective 80 and it does not require a thermal cycler and electrophoretic unit, as the result can be inferred as 81 positive or negative with the colour change or turbidity observed by naked eye without spending 82 much time [14]. 83 The favorable target for bacterial pathogen diagnostic includes mainly, 16S ribosomal 84 RNA (rRNA) gene which is been widely used to identify the bacterial species among different 85 bacteria [15-17]. This diversity is unique for species and their sequences are presented inside their 86 nine (V1 -V9) hypervariable regions which constitutes useful targets [18]. These regions are 87 highly conserved stretches in bacteria, which facilitates the identification through PCR In principle, the LAMP assay recognizes six specifically designed primers that recognize 90 eight distinct sequences from the target bacterial DNA [20]. It consists of two outer primers (F3 91 and B3), two inner primers (FIP and BIP), a loop-forward (LF) and a loop-backward (LB). These 92 six primers ensure a high specificity for target DNA amplification. Moreover, these primers enable 93 generation of a stem-loop DNA for subsequent complex LAMP amplification, which includes self-94 priming reaction. A mixture of DNA, differing in the number of loops and length of the stem-loop 95 is produced as a final product [21]. By considering the advantages of LAMP assay, which is widely 96 used to detect broad range of microorganisms[22,23]. The present study was aimed to detect the 97 bacterial blight of pomegranate in early stages with minimum pathogen DNA template by 98 optimizing several DNA intercalating dyes with lower temperature in a shorter time with less 99 technical skills. 100 MATERIALS AND METHODS Source of bacterial blight material used for the study 102 Ten Xanthomonas axonopodis pv. punicae (Xap) isolates collected from four different regions of 103 Karnataka, India were used in this study (Fig 1) . Additionally, we have also isolated 29 non-104 Xanthomonas bacteria from pomegranate Table 1 . The pure culture Xap isolates were maintained 105 in a Nutrient Broth for 72 hours and the DNA was extracted using crude method. The quality and 106 quantity of the DNA were determined using Nanodrop (Model -DS-11 FX +, DeNovix, USA). '},\n"," {'paper_id': '1ad1d4b84aea4ceaf05c62a1ad04e7150f7f4684',\n","  'section': 'Abstract',\n","  'text': 'Clinical assessment of possible infection with SARS-CoV-2, the novel coronavirus responsible for the outbreak of COVID-19 respiratory illness, has been a major activity of infectious diseases services in the UK and elsewhere since the first report of cases in December 2019. We report our case series of 68 patients, reviewed by Infectious Diseases Consultants at a Regional Infectious Diseases Unit in the UK. We prospectively evaluated our service between the 29 th Jan 2020 and 24 th Feb 2020. Demographic, clinical, epidemiological and laboratory data were collected. We have compared clinical features and subsequent diagnosis between well patients not requiring admission for clinical reasons or antimicrobials with those assessed as needing either admission or antimicrobial treatment. A, RSV, non SARS/MERS coronaviruses, rhinovirus/enterovirus. 9/68 were treated with antimicrobials, 15/68 were admitted to a negative pressure room of whom 5/68 were admitted solely due to an inability to isolate at home. Patients requiring either admission on clinical grounds or antimicrobials (14/68) were similar to those not requiring admission or antimicrobials, with modestly more fever and shortness of breath in the clinically admitted / antimicrobial group. The most commonly prescribed antimicrobials were doxycycline, moxifloxacin and oseltamivir. The majority of patients had mild illness which did not require a clinical intervention to manage. This finding supports a community testing approach supported by clinicians to review the proportion of more unwell patients. '},\n"," {'paper_id': '1b172e179f17f34d95751998032355427afa78e8',\n","  'section': 'Abstract',\n","  'text': 'Background The rapid global spread of coronavirus disease (COVID-19) is unprecedented. The outbreak has quickly spread to more than 100 countries reporting over '},\n"," {'paper_id': '1b4ab28a9391d516e1753874c73d2d244104f825',\n","  'section': 'Abstract',\n","  'text': 'The spread of the novel coronavirus (SARS-CoV-2) has triggered a global emergency, that demands urgent solutions for detection and therapy to prevent escalating health, social and economic impacts. The spike protein (S) of this virus enables binding to the human receptor ACE2, and hence presents a prime target for vaccines preventing viral entry into host cells 1 . The S proteins from SARS-CoV-1 and SARS-CoV-2 are similar 2 , but structural differences in the receptor binding domain (RBD) preclude the use of SARS-CoV-1-specific neutralizing antibodies to inhibit SARS-CoV-2 3 . Here we used comparative pangenomic analysis of all sequenced Betacoronaviruses to reveal that, among all core gene clusters present in these viruses, the envelope protein E shows a variant shared by SARS and SARS-Cov2 with two completely-conserved key functional features, an ion-channel and a PDZ-binding Motif (PBM). These features trigger a cytokine storm that activates the inflammasome, leading to increased edema in lungs causing the acute respiratory distress syndrome (ARDS) 4-6 , the leading cause of death in SARS-CoV-1 and SARS-CoV-2 infection 7,8 . However, three drugs approved for human use may inhibit SARS-CoV-1 and SARS-CoV-2 Protein E, either acting upon the ion channel (Amantadine and Hexamethylene amiloride 9,10 ) or the PBM (SB203580 5 ), thereby potentially increasing the survival of the host, as already demonstrated for SARS-CoV-1in animal models. Hence, blocking the SARS protein E inhibits development of ARDS in vivo. Given that our results demonstrate that the protein E subcluster for the SARS clade is quasi-identical for the key functional regions of SARS-CoV-1 and SARS-CoV-2, we conclude that use of approved drugs shown to act as SARS E protein inhibitors can help prevent further casualties from COVID-2019 while vaccines and other preventive measures are being developed. '},\n"," {'paper_id': '1b531d266b4933bf8040739ed199914655b075d6',\n","  'section': 'Abstract',\n","  'text': 'The outbreak of pneumonia caused by a novel coronavirus (2019-nCoV) in Wuhan 17 City of China obtained global concern, the population outflow from Wuhan has 18 contributed to spatial expansion in other parts of China. We examined the effects of 19 population outflow from Wuhan on the 2019-nCoV transmission in other provinces 20 and cities of China, as well as the impacts of the city closure in Wuhan. We observed 21 a significantly positive association between population movement and the number of 22 cases. Further analysis revealed that if the city closure policy was implemented two 23 days earlier, 1420 (95% CI: 1059, 1833) cases could be prevented, and if two days 24 later, 1462 (95% CI: 1090, 1886) more cases would be possible. Our findings suggest 25 that population movement might be one important trigger of the 2019-nCoV infection 26 transmission in China, and the policy of city closure is effective to prevent the 27 epidemic. 28 '},\n"," {'paper_id': '1b9fed2824e97db8cb35912549dd2e9cde7b6c18',\n","  'section': 'Abstract',\n","  'text': 'The cutting-edge technology vaccinomics is the combination of two topics immunogenetics and immunogenomics with the knowledge of systems biology and immune profiling for designing vaccine against infectious disease. In our present study, an epitope-based peptide vaccine against nonstructural protein 4 of beta coronavirus, using a combination of B cell and T cell epitope predictions, followed by molecular docking methods are performed. Here, protein sequences of homologous nonstructural protein 4 of beta coronavirus are collected and conserved regions present in them are investigated via phylogenetic study to determine the most immunogenic part of protein. From the identified region of the target protein, the peptide sequence IRNTTNPSAR from the region ranging from 38-47 and the sequence PTDTYTSVYLGKFRG from the positions of 76-90 are considered as the most potential B cell and T cell epitopes respectively. Furthermore, this predicted T cell epitopes PTDTYTSVY and PTDTYTSVYLGKFRG interacted with MHC allelic proteins HLA-A*01:01 and HLA-DRB5*01:01 respectively with the low IC 50 values. These epitopes are perfectly fitted into the epitope binding grooves of alpha helix of MHC I molecule and MHC II molecule with binding energy scores -725.0 Kcal/mole and -786.0 Kcal/mole respectively, showing stability in MHC molecules binding. This MHC restricted epitope PTDTYTSVY also showed a good conservancy of 50.16% in world population coverage. This MHC I HLA-A*01:01 allele is present among 58.87% of Chinese population also. Therefore, the epitopes IRNTTNPSAR and PTDTYTSVYLGKFRG may be considered as potential peptides for peptide-based vaccine for coronavirus after further experimental study. '},\n"," {'paper_id': '1bc752b7e38185b9a02b1817161fcdf33d942d67',\n","  'section': 'Abstract',\n","  'text': '. Reaching a swift, reliable diagnosis of COVID-19 in the emergency departments is imperative to direct patients to proper care and to prevent disease dissemination. COVID-19 diagnosis is based on the identification of viral RNA through RT-PCR from oral-nasopharyngeal swabs, which however presents suboptimal sensitivity and may require several hours in overstressed laboratories. These drawbacks have called for an additional, complementary first line approach. CT is the gold standard method for the detection of interstitial pneumonia, a hallmark feature of COVID-19, often present in the asymptomatic stage of the disease. Here, we show that CT scan presents a sensitivity of 95.48% (std.err=0.35%), vastly outperforming RT-PCR. Additionally, as diagnostic accuracy is influenced by disease prevalence, we argue that predictive values provide a more precise measure of CT reliability in the current pandemics. We generated a model showing that CT scan is endowed with a high negative predictive value (> 90%) and positive predictive value (69 -84%), for the range of prevalence seen in countries with rampant dissemination. We conclude that CT is an expedite and reliable diagnostic tool to support first line triage of suspect COVID-19 patients in areas where the diffusion of the virus is widespread. '},\n"," {'paper_id': '1bd626c6e2bf333c14247fa3e4781bf8764caf3c',\n","  'section': 'Abstract',\n","  'text': 'The COVID-19 pandemic has led to unprecedented strain on intensive care unit (ICU) admission in parts of the world. Strategies to create surge ICU capacity require complex local and national service reconfiguration and reduction or cancellation of elective activity. These measures have an inevitable lag-time before additional capacity comes on-line. An accurate short-range forecast would be helpful in guiding such di cult, costly, and ethically challenging decisions. At the time this work began, cases in England were starting to increase. If this represents a true spread in disease then ICU demand could increase rapidly. Here we present a short-range forecast based on published real-time COVID-19 case data from the seven National Health Service (NHS) commissioning regions in England (East of England, London, Midlands, North East and Yorkshire, North West, South East and South West). We use a Monte Carlo approach to model the likely impact of current diagnoses on regional ICU capacity over a 14-day horizon under the assumption that the increase in cases represents the start of an exponential growth in infections. Our model is designed to be parsimonious and based on available epidemiological data from the literature at the moment. On the basis of the modelling assumptions made, ICU occupancy is likely to increase dramatically in the days following the time of modelling. If the current exponential growth continues, case numbers will be comparable to current ICU bed numbers within weeks. Despite variable growth in absolute patients, all commissioning regions are forecast to be heavily burdened under the assumptions used. Whilst, like any forecast model, there remain uncertainties both in terms of model specification and robust epidemiological data in this early prospective phase, it would seem that surge capacity will be required in the very near future. Our findings should be interpreted with caution, but we hope that our model will help policy decision makers with their preparations. The uncertainties in the data highlight the urgent need for ongoing real-time surveillance to allow forecasts to be constantly updated using high quality local patient-facing data as it emerges. '},\n"," {'paper_id': '1c06378f629bfa8870c89b3abfdecfc19db9b459',\n","  'section': 'Abstract',\n","  'text': \"Artificial nanoparticles accumulate a protein corona layer in biological fluids, which significantly influences their bioactivity. As nanosized obligate intracellular parasites, viruses share many biophysical properties with artificial nanoparticles in extracellular environments and here we show that respiratory syncytial virus (RSV) and herpes simplex virus 1 (HSV-1) accumulate a rich and distinctive protein corona in different biological fluids. Moreover, we show that corona pre-coating differentially affects viral infectivity and immune cell activation. Additionally, we demonstrate that viruses bind amyloidogenic peptides in their corona and catalyze amyloid formation via surface-assisted heterogeneous nucleation. Importantly, we show that HSV-1 catalyzes the aggregation of the amyloid beta peptide (Aβ42), a major constituent of amyloid plaques in Alzheimer's disease, in-vitro and in animal models. Our results highlight the viral protein corona as an acquired structural layer that is critical for viral-host interactions and illustrate a mechanistic convergence between viral and amyloid pathologies. doi: bioRxiv preprint HSV-1 is an example of another virus with high prevalence, infecting nearly 70% of the human population 17 . HSV-1 is a double-stranded DNA virus consisting of an icosahedral nucleocapsid surrounded by tegument and envelope with virion sizes ranging from 155 to 240 nm 18 . HSV-1 is a neurotropic virus that infects peripheral . CC-BY-NC-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/246785 doi: bioRxiv preprint 4 sensory neurons and establishes latency 19 . Latent HSV-1 is occasionally reactivated causing peripheral pathology and under certain circumstances it can migrate into the central nervous system causing herpes simplex encephalitis; the most common cause of sporadic fatal viral encephalitis 19 . In the context of the current work we focused on the presumptive role of HSV-1 in the pathology of AD. A number of risk factors have been associated with AD, including the E4 allele of the apolipoprotein E (Apo-E), diabetes, vascular pathology, neuroinflammation and infections 20 . Several recent studies have supported the theory of a significant role of HSV-1 in the disease 21 . HSV-1 DNA was found to be localized within amyloid plaques in AD patients and HSV-1 infection has been shown to promote neurotoxic Aβ accumulation in human neural cells and to the formation of Aβ deposits in the brains of infected mice 22,23 . Moreover, the presence of anti-HSV IgM antibodies, which indicate HSV reactivation, is correlated with a high risk of AD and antiherpetic treatment is correlated with a reduced risk of developing dementia 24,25 . Despite these correlations, the mechanism by which viruses induce amyloid aggregation, the major pathological hallmark of AD, is not known. In the present study, we demonstrated that upon encountering different biological fluids, RSV accumulated extensive and distinct protein coronae compared to HSV-1 and synthetic liposomes. The various coronae were dependent on the biological fluid and exerted markedly different effects on RSV infectivity and capacity to activate monocyte-derived dendritic cells (moDCs). Moreover, upon interaction with an amyloidogenic peptide derived from IAPP, RSV accelerated the process of amyloid aggregation via surface-assisted heterogenous nucleation. This amyloid catalysis was also demonstrated for HSV-1 and the Aβ42 peptide in-vitro and in an AD animal model. Our findings highlight the importance of viral protein corona interactions for viral pathogenesis and provide a direct mechanistic link between viral and amyloid pathologies. Results Based on the extensive literature describing the significant role of corona factors in synthetic nanoparticle functionality, we used established techniques to answer questions regarding RSV pathogenicity 26 . Using proteomics, we assessed the RSV protein corona profiles in adult human plasma (HP), juvenile (6 months old infants tested RSV negative at the time of sample collection) human plasma (jHP), human bronchoalveolar lavage fluid (BALF) from healthy adults, rhesus macaque plasma (MP), and fetal bovine serum (FBS). These biological fluids represent different microenvironments encountered by the virus in terms of tissue tropism (HP vs. BALF), zoonosis (MP) and culturing conditions (FBS). The biological fluids were screened for antibodies against RSV using ELISA, and both adult HP and BALF contained high levels of anti-RSV IgG antibodies, unlike jHP, MP and FBS (Supplementary Fig. 1 ). Viral stocks were produced in serum-free conditions to prevent initial contamination with FBS proteins. Virions produced under serum-free conditions were incubated with 10% v/v solutions of different biological fluids. Controls included non-infected cell medium representing the background cellular secretome, and synthetic lipid vesicles of a size comparable to RSV (200 nm) with positively or negatively charged surfaces. In addition, we compared the coronae of RSV to HSV-1 to probe for differences between different viruses of relatively similar size. After incubation for 1 h at 37 ˚C, the virions were re-harvested by centrifugation and washed twice before performing mass spectrometry-based proteomic analyses. Six replicates from two independent experiments. Statistics were calculated using GraphPad Prism 7 software. The detection of specific anti-RSV IgG antibodies in biological fluids was performed using Human Anti-Respiratory syncytial virus IgG ELISA Kit (ab108765, abcam®, Sweden), according to manufacturer's protocol. All biological fluids (FBS, HP, MP, BALF) were diluted to a protein concentration of 0.3 mg/ml before performing the assay and results were compared to the positive, cutoff, and negative controls provided by the kit manufacturer. HEp-2 cells were seeded in maintenance medium until they reached 50-60% confluency. On the day of infection, the cells were washed twice with warm PBS and medium was changed to VP-SFM with gentamycin. Before adding to the cells, sucrose cushion-concentrated RSV stocks were pre-incubated with different biological fluids (FBS, HP, MP, BALF) at a final protein concentration of 0.3 mg/ml in VP-SFM with 10 mM sodium citrate for 1 h at 37 ˚C. The corona-precoated viruses were then added to the cells in serum-free conditions (diluted 10x) at a MOI of 1. 24h post-infection, the medium was changed back to growth medium and cells were visualized with fluorescence microscopy 72h post-infection. After visualization, the cells were washed and stained with LIVE/DEAD™ Fixable Far Red Dead Cell Stain Kit (ThermoFisher, USA) for 30 min at 4 ˚C in Dulbecco's phosphate-buffered saline (DPBS no calcium, no magnesium, ThermoFisher), then fixed and washed for flow cytometry using Cytofix/Cytoperm™ (BD, USA) according to manufacturer's protocol. Cells were then scraped and resuspended in DPBS, and the data was acquired using MACSQuant® Analyser 10 flowcytometer (Miltenyi Biotec, Sweden). The data was analyzed by FlowJo software (TreeStar) by excluding the dead cells stained with far-red fluorescent dye and subsequent calculation of GFP positive cells within the viable cell population. For experiments with palivizumab, cells were treated with different concentrations of the antibody before infection. Human monocytes were negatively selected from buffy coats using the RosetteSep Monocyte Enrichment Kit (1mL/10mL buffy coat; StemCell Technologies) and differentiated into moDC, using GM-CSF (250ng/mL; PeproTech) and IL-4 (6.5ng/mL; R&D Systems) for 6 days in 37°C, 5% CO2 at a density of 5x10 5 cells/mL in RPMI 1640 completed with 10% FCS, 1mM sodium pyruvate, 10mM HEPES, 2mM L-glutamine, and 1% and penicillin/streptomycin (ThermoFisher, USA). Immature moDC were exposed to RSV pre-incubated with different biological fluids for 4h in serum free media, washed and then incubated in serum containing medium for 72 h before analyses of CD86 and RSV-GFP expression. Dead cells were excluded using Live/Dead fixable near-IR dead cell stain kit (ThermoFisher). Flow cytometry sample data were acquired on a Fortessa (BD Biosciences) and the analysis was performed in FlowJo software (TreeStar). NNFGAIL, GNNQQNY, Aβ42, Aβ40 were synthesized and purified by Pepscan (The Netherlands) and the final molecular weight was verified by MS. 10 µl of dimethyl sulfoxide (DMSO) (ThermoFisher, USA) were added to . \"},\n"," {'paper_id': '1c0b25c8e3c89243fd8797e2e964e12689bc8556',\n","  'section': 'Abstract',\n","  'text': 'As the predominant etiological agent of the common cold, human rhinovirus (HRV) is 19 the leading cause of human infectious disease. Early studies showed monovalent formalin-20 inactivated HRV vaccine can be protective, and virus-neutralizing antibodies (nAb) correlated 21 with protection. However, co-circulation of many HRV types discouraged further vaccine 22 efforts. We approached this problem straightforwardly. We tested the hypothesis that increasing 23 virus input titers in polyvalent inactivated HRV vaccine will result in broad nAb responses. Here, 24 we show that serum nAb against many rhinovirus types can be induced by polyvalent, 25 inactivated HRVs plus alhydrogel (alum) adjuvant. Using formulations up to 25-valent in mice 26 and 50-valent in rhesus macaques, HRV vaccine immunogenicity was related to sufficient 27 quantity of input antigens, and valency was not a major factor for potency or breadth of the 28 response. We for the first time generated a vaccine capable of inducing nAb responses to 29 numerous and diverse HRV types. 30 31 38 '},\n"," {'paper_id': '1c1f6785cf5735262d360b33346dd533f6e6b363',\n","  'section': 'Abstract',\n","  'text': 'We sought to estimate the ascertainment ratio of SARS-CoV-2 infection in Wuhan, China, using a modified Bayesian SEIR model with publicly reported case data. We estimated it at 0.465% (95%CI: 0.464-0.466%), implying that the outbreak in Wuhan was abated by depletion of susceptibles, rather than public health action alone. This suggests a high-transmissibility/low-severity profile for the current pandemic and raises doubt about whether suppression, rather than mitigation, is a feasible goal. '},\n"," {'paper_id': '1c66ec8950cbd1e0f0d4bc4a43ba9c79411cc5fa',\n","  'section': 'Abstract',\n","  'text': 'Viruses are one of the major causes of various acute and chronic infectious diseases and thus a major contributor to the global burden of disease. Several studies have shown how viruses have evolved to hijack basic cellular pathways and evade innate immune response by modulating key host factors and signalling pathways. A collective view of these multiple studies could advance our understanding of viral evasion mechanisms and provide new therapeutic perspectives for the treatment of viral diseases. Here, we performed an integrative meta-analysis to elucidate the 17 different host-virus interactomes. Network and bioinformatics analyses showed how viruses with small genomes efficiently achieve the maximal effect by targeting multifunctional and highly connected host proteins with a high occurrence of disordered regions. We also identified the core cellular process subnetworks that are targeted by all the viruses. Integration with functional RNA interference (RNAi) datasets showed that a large proportion of the targets are required for viral replication. Furthermore, we performed an interactome-informed drug re-purposing screen and identified novel activities for broad-spectrum antiviral agents against hepatitis C virus and human metapneumovirus. Altogether, these orthogonal datasets could serve as a platform for hypothesis generation and follow-up studies to broaden our understanding of the viral evasion landscape. author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '1d2947d5c0addcbbcfa71d87ea074a8b6ff2a973',\n","  'section': 'Abstract',\n","  'text': 'An epidemic can be characterized by its speed (i.e., the exponential growth rate r) and strength (i.e., the reproductive number R). Disease modelers have historically placed much more emphasis on strength, in part because the effectiveness of an intervention strategy is typically evaluated on this scale. Here, we develop a mathematical framework for this classic, strength-based paradigm and show that there is a corresponding speed-based paradigm which can provide complementary insights. In particular, we note that r = 0 is a threshold for disease spread, just like R = 1, and show that we can measure the speed and strength of an intervention on the same scale as the speed and strength of an epidemic, respectively. We argue that, just as the strength-based paradigm provides the clearest insight into certain questions, the speed-based paradigm provides the clearest view in other cases. As an example, we show that evaluating the prospects of \"test-and-treat\" interventions against the human immunodeficiency virus (HIV) can be done more clearly on the speed than strength scale, given uncertainty in the proportion of HIV spread that happens early in the course of infection. We suggest that disease modelers should avoid over-emphasizing the reproductive number at the expense of the exponential growth rate, but instead look at these as complementary measures. '},\n"," {'paper_id': '1d2e66528cfb3f754157404a225d551f6cdab1ae',\n","  'section': 'Abstract',\n","  'text': 'Public health preparedness for coronavirus disease 2019 is challenging in the absence of setting-specific epidemiological data. Here we describe the epidemiology of seasonal human coronaviruses (sCoVs) and other cocirculating viruses in the West of Scotland, UK. We analyzed routine diagnostic data for >70,000 episodes of respiratory illness tested molecularly for multiple respiratory viruses between 2005 and 2017. Statistical associations with patient age and sex differed between CoV-229E, CoV-OC43 and CoV-NL63. Furthermore, the timing and magnitude of sCoV outbreaks did not occur concurrently and coinfections were not reported. With respect to other cocirculating respiratory viruses, we found evidence of positive, rather than negative, interactions with sCoVs. These findings highlight the importance of considering cocirculating viruses in the differential diagnosis of COVID-19. Further work is needed to establish the occurrence/degree of cross-protective immunity conferred across sCoVs and with COVID-19, as well as the role of viral coinfection in COVID-19 disease severity. '},\n"," {'paper_id': '1d882d90109fb4496088342cf2bcaa4fec40cf9f',\n","  'section': 'Abstract',\n","  'text': \"Bayesian inference methods for individual-level geo-located epidemiological data have enabled reconstruction of transmission trees and quantification of disease spread in space and time, while accounting for uncertainty in missing data. However, these methods have rarely been applied to endemic diseases or ones in which asymptomatic infection plays a role, for which novel estimation methods are required. Here, we develop such methods to analyse longitudinal incidence data on visceral leishmaniasis (VL), and its sequela, postkala-azar dermal leishmaniasis (PKDL), in a highly endemic community in Bangladesh. Incorporating recent data on infectiousness of VL and PKDL, we show that while VL cases drive transmission when incidence is high, the contribution of PKDL increases significantly as VL incidence declines (reaching 55% in this setting). Transmission is highly focal: >85% of mean distances from inferred infectors to their secondary VL cases were <300m, and estimated average times from infector onset to secondary case infection were <4 months for 90% of VL infectors, but up to 2.75yrs for PKDL infectors. Estimated numbers of secondary VL cases per VL and PKDL case varied from 0-6 and were strongly correlated with the infector's duration of symptoms. Counterfactual simulations suggest that prevention of PKDL could have reduced VL incidence by up to a quarter. These results highlight the need for prompt detection and treatment of PKDL to achieve VL elimination in the Indian subcontinent and provide quantitative estimates to guide spatiotemporally-targeted interventions against VL. visceral leishmaniasis | post-kala-azar dermal leishmaniasis | spatiotemporal transmission | transmission tree | Bayesian inference S patiotemporal heterogeneity in incidence is a hallmark of 1 infectious diseases. Insight into this heterogeneity has in-2 creased considerably in recent years due to greater availability 3 of geo-located individual-level epidemiological data and the 4 development of sophisticated statistical inference methods for 5 partially observed transmission processes (1-6). These meth-6 ods have been developed for epidemics, in which the immune 7 status of the population is known, and for diseases with a 8 short time course that are relatively easily diagnosed, such 9 as measles, influenza, and foot-and-mouth disease (3, 4, 7). Here, we extend these methods to a slowly progressing endemic 11 disease of humans in which asymptomatic infection plays an 12 important role. 13 We analyse detailed longitudinal individual-level data on 14 incidence of visceral leishmaniasis (VL), and its sequela, post-15 kala-azar dermal leishmaniasis (PKDL), in a highly endemic 16 community in Fulbaria, Bangladesh (8). VL, also known as 17 kala-azar, is a lethal sandfly-borne parasitic disease targeted 18 for elimination as a public health problem (<1 case/10,000 19 people/year at subdistrict/district level depending on the 20 country) in the Indian subcontinent (ISC) by 2020 (9). PKDL 21 is a non-lethal skin condition that occurs after treatment 22 for VL in 5-20% of cases in the ISC, and less frequently in 23 individuals who report no history of prior VL (8, 10). It is 24 characterised by skin lesions of di ering severity and parasite 25 load, ranging from macules and papules (least severe, lowest 26 load) to nodules (most severe, highest load) (11). We estimate 27 the relative contributions of di erent disease states (VL, PKDL 28 and asymptomatic infection) to transmission, and quantify the 29 rate of spread of infection around infected individuals in space 30 and time by reconstructing transmission trees. Our analysis 31 provides insight into the spatiotemporal spread of visceral 32 leishmaniasis as well as quantitative estimates that can guide 33 the targeting of interventions, such as active case detection 34 and indoor residual spraying (IRS) of insecticide, around VL 35 Although methods for analysing individual-level geo-located disease data have existed for some time, they have rarely been used to analyse endemic human diseases. Here we apply such methods to nearly a decade's worth of uniquely detailed epidemiological data on incidence of the deadly vector-borne disease visceral leishmaniasis (VL) and its secondary condition, post-kala-azar dermal leishmaniasis (PKDL), to quantify the spread of infection around cases in space and time by inferring who infected whom, and estimate the relative contribution of different infection states to transmission. Our findings highlight the key role long diagnosis delays and PKDL play in maintaining VL transmission. This detailed characterisation of the spatiotemporal transmission of VL will help inform targeting of interventions around VL and PKDL cases. : medRxiv preprint 131 sion (over and above that from being at zero distance from 132 an infectious individual) and with di erent pre-symptomatic 133 and asymptomatic relative infectiousness. Models with addi-134 tional within-household transmission fitted the data signifi-135 cantly better than those without according to the Deviance 136 Information Criterion (DIC) (32), and for these models fit 137 improved with increasing pre-symptomatic and asymptomatic 138 infectiousness, such that the best-fitting model had additional 139 within-household transmission and assumed 2% relative infec-140 tiousness of pre-symptomatic and asymptomatic individuals. 141 Here we present the results from this model (see SI Text for 142 results from the other models). 143 Parameter Estimates. We estimated the transmission model 144 parameters and unobserved data using the Markov Chain 145 Monte Carlo (MCMC) algorithm described in Materials and 146 Methods and SI Text. The posterior distributions obtained 147 for the model parameters are shown in Fig. S6 and the corre-148 sponding posterior modes and 95% credible intervals (CI) are \"},\n"," {'paper_id': '1dbcf165dd450ed080bcea6594ad70dc53c7ba2e',\n","  'section': 'Abstract',\n","  'text': 'While lymphocytopenia is a common characteristic of patients infected by the novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), the mechanisms responsible for this depletion are unclear. Through careful inspection of the spleens and lymph nodes (LNs) from six cases with postmortem examinations, we observed that SARS-CoV-2 could directly infect secondary lymphoid organs to induce cell death. Immunohistochemistry demonstrated ACE2 (angiotensin-converting enzyme 2), the potential receptor of SARS-CoV-2, expresses on tissue-resident CD169 + macrophages in spleens and LNs. Immunofluorescent staining confirmed that viral nucleocaspid protein (NP) can be found in ACE2 + cells, CD169 + macrophages, but not in CD3 + T cells or B220 + B cells in spleens and LNs. SARS-CoV-2 infection induces severe tissue damage including lymph follicle depletion, splenic nodule atrophy, histiocyte hyperplasia and lymphocyte reductions. Moreover, in situ TUNEL staining illustrated that viral infection leads to severe lymphocyte apoptosis, which might be mediated by viral antigens inducing Fas upregulation. Furthermore, SARS-CoV-2 also triggers macrophages to produce IL-6, a proinflammatory cytokine that directly promotes lymphocyte necrosis. Collectively, these results demonstrate that SARS-CoV-2 directly neutralizes human spleens and LNs through infecting tissue-resident CD169 + macrophages. '},\n"," {'paper_id': '1dd898b5ca1ae70ec0e3cad89fc87a165002a99e',\n","  'section': 'Abstract',\n","  'text': 'In 2013, U.S. swine producers were confronted with the disruptive emergence of porcine epidemic diarrhoea (PED). Movement of animals among farms is hypothesised to have played a role in the spread of PED among farms. Via this or other mechanisms, the rate of spread may also depend on the geographic density of farms and climate. To evaluate such effects on a large scale, we analyse state-level counts of outbreaks with variables describing the distribution of farm sizes and types, aggregate flows of animals among farms, and an index of climate. Our first main finding is that it is possible for a correlation analysis to be sensitive to transmission model parameters. This finding is based on a global sensitivity analysis of correlations on simulated data that included a biased and noisy observation model based on the available PED data. Our second main finding is that flows are significantly associated with the reports of PED outbreaks. This finding is based on correlations of pairwise relationships and regression modeling of total and weekly outbreak counts. These findings illustrate how variation in population structure may be employed along with observational data to improve understanding of disease spread. together produce 95 percent of the U.S. pig crop. 5 Farms experiencing outbreaks have suffered 90 percent and higher losses of . CC-BY 4.0 International license is made available under a '},\n"," {'paper_id': '1df9ce60e358f6cef0be3f6e15fe2a9c6f93f39c',\n","  'section': 'Abstract',\n","  'text': '11 '},\n"," {'paper_id': '1dfcd4615a8763e053bd2fdb845f09fab68109f1',\n","  'section': 'Abstract',\n","  'text': 'While the ecology of infectious disease is a rich field with decades worth of empirical evidence and theory, there are aspects that remain relatively under-examined. One example is the importance of the free-living survival stage of certain pathogens, and especially is cases where they are transmitted indirectly between hosts through an environmental reservoir intermediate. In this study, we develop an integrated, broadly applicable mathematical method to examine diseases fitting this description-the waterborne, abiotic and other indirectly transmitted (W.A.I.T.) infection framework. To demonstrate its utility, we construct realistic models of two very different epidemic scenarios: cholera in a densely populated setting with limited access to clean drinking water and hepatitis C virus in an urban setting of injection-drug users. Using these two exemplars, we find that the W.A.I.T. model fortifies the centrality of reservoir dynamics in the \"sit and wait\" infection strategy, and provides a way to simulate a diverse set of intervention strategies. '},\n"," {'paper_id': '1e45cd5feb7928bbc8bcd524d9c6b7adeae08a6b',\n","  'section': 'Abstract',\n","  'text': \"ZG and YL made substantial contributions to the study concept and design. LW was in charge of the manuscript draft. XL, CH, SY and DL took responsibility for obtaining written consent from patients, obtaining ethical approval, collecting samples, and confirming data accuracy. LW and ZG participated in drafting the manuscript, and revising it on the basis of reviewers' comments. LW made substantial contributions to data acquisition, analysis, and interpretation. We declare no competing interests. With the permission of the corresponding authors, we can provide participant data without names and identifiers, but not the study protocol, statistical analysis plan, or informed consent form. Data can be provided after the Article is published. Once the data can be made public, the research team will provide an email address for communication. The corresponding authors have the right to decide whether to share the data or not based on the research objectives and plan provided. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. \"},\n"," {'paper_id': '1e48223ccda9ec1fa91dc5f2348e40c217c513e7',\n","  'section': 'Abstract',\n","  'text': 'A strategy for preventing further spread of the ongoing COVID-19 epidemic is to detect infections and isolate infected individuals without the need of extensive biospecimen testing. Objectives: Here we investigate the prevalence of loss of smell and taste among COVID-19 diagnosed individuals and we identify the combination of symptoms, besides loss of smell and taste, most likely to correspond to a positive COVID-19 diagnosis in non-severe cases. Design: Community survey. including loss of smell and taste; 579 were positive and 1123 negative. In this subset, we find that loss of smell and taste were present in 59% of COVID-19 positive individuals compared to 18% of those negative to the test, yielding an odds ratio (OR) of COVID-19 diagnosis of OR[95%CI]=6.59[5.25; 8.27], P= 1.90x10 -59 . We also find that a combination of loss of smell and taste, fever, persistent cough, fatigue, diarrhoea, abdominal pain and loss of appetite is predictive of COVID-19 positive test with sensitivity 0.54[0.44; 0.63], specificity 0.86[0.80; 0.90], ROC-AUC 0.77[0.72; 0.82] in the test set, and cross-validation ROC-AUC 0.75[0.72; 0.77]. '},\n"," {'paper_id': '1e9cafd54968d935da52bdcae678df1e6ec2b650',\n","  'section': 'Abstract',\n","  'text': 'A new coronavirus, SARS-CoV-2, has recently emerged to cause a human pandemic. Whereas molecular diagnostic tests were rapidly developed, serologic assays are still lacking, yet urgently needed. Validated serologic assays are important for contact tracing, identifying the viral reservoir and epidemiological studies. Here, we developed serological assays for the detection of SARS-CoV-2 neutralizing, spike-and nucleocapsid-specific antibodies. Using serum samples from patients with PCR-confirmed infections of SARS-CoV-2, other coronaviruses, or other respiratory pathogenic infections, we validated and tested various antigens in different in-house and commercial ELISAs. We demonstrate that most PCRconfirmed SARS-CoV-2 infected individuals seroconverted, as revealed by sensitive and specific in-house ELISAs. We found that commercial S1 IgG or IgA ELISAs were of lower specificity while sensitivity varied between the two, with IgA showing higher sensitivity. Overall, the validated assays described here can be instrumental for the detection of SARS-CoV-2-specific antibodies for diagnostic, seroepidemiological and vaccine evaluation studies. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '1eaa329f608055620a57e6273e9d1c409de1e9ee',\n","  'section': 'Abstract',\n","  'text': 'ATP-dependent chromatin remodelling proteins represent a diverse family of proteins that share ATPase domains that are adapted to regulate protein-DNA interactions. Here we present structures of the yeast Chd1 protein engaged with nucleosomes in the presence of the transition state mimic ADP-beryllium fluoride. The path of DNA strands through the ATPase domains indicates the presence of contacts conserved with single strand translocases and additional contacts with both strands that are unique to Snf2 related proteins. The structure provides connectivity between rearrangement of ATPase lobes to a closed, nucleotide bound state and the sensing of linker DNA. Two turns of linker DNA are prised off the surface of the histone octamer as a result of Chd1 binding, and both the histone H3 tail and ubiquitin conjugated to lysine 120 are re-orientated towards the unravelled DNA. This indicates how changes to nucleosome structure can alter the way in which histone epitopes are presented. '},\n"," {'paper_id': '1eb2a0cf3757558a49f6ad91294a832b04ca6353',\n","  'section': 'Abstract',\n","  'text': 'Background Since Dec 2019, SARS-CoV-2 has caused about fifty thousand patients and over two thousand deaths in Wuhan, China. We reported characteristics of patients with COVID-19 during epidemic ongoing outbreak in Wuhan. Methods Data of COVID-19 patients with clinical outcome in a designated hospital in Wuhan, were retrospectively collected from electronic medical records. Characteristics were compared between patients who died or recovered, and between patients with different disease severity. Results By Feb 25, 2020, 403 patients were enrolled, 100 died and 303 recovered. Most of non-survivors tended to be males, old aged, or with chronic diseases. Duration from illness onset to admission was 9 (7-12) days. Patients with severe or critical illness had more days from onset to admission compared to those with ordinary illness. Lymphopenia, anemia, hypoproteinemia, and abnormal serum sodium were presented in 52.6%, 54.6%, 69.8%, and 21.8% cases, respectively. Patients who died or with severe/critical illness showed increased white blood cell and neutrophil count, serum total bilirubin, creatinine, hypersensitive troponin I, D-dimer, procalcitonin, and C-reactive protein, and decreased red blood cell, lymphocyte, platelet count, and serum albumin on admission compared to those who recovered or with ordinary illness. Complications of acute organ injury and secondary infection were common in patients with COVID-19, especially in non-survivors. Conclusions Multiple homeostasis disturbances were common in patients with severe or critical illness at admission. Early support should be provided, especially for old men with chronic disease, which is vital to control disease progression and reduce mortality of COVID-19 during epidemic ongoing outbreak. '},\n"," {'paper_id': '1eb535ddf2778323e8af948d6e79865118276183',\n","  'section': 'Abstract',\n","  'text': 'Background It has been reported that several cases recovered from COVID-19 tested positive for SARS-CoV-2 after discharge (re-detectable positive, RP), however the clinical characteristics, significance and potential cause of RP patients remained elusive. Methods A total of 262 COVID-19 patients were discharged from January 23 to February 25, 2020, and were enrolled for analysis of their clinical parameters. The RP and non-RP (NRP) patients were grouped according to the disease severity during their hospitalization period. The clinical characterization at re-admission to the hospital was analyzed. SARS-CoV-2 RNA and plasma antibody levels were detected using high-sensitive detection methods. Findings Up to March 10, 2020, all of patients were followed up for at least 14 days, and 38/262 of RP patients (14.5%) were present. The RP patients were characterized by being less than 14-years old and having mild and moderate conditions as compared to NRP patients, while no severe patients became RP. Retrospectively, the RP patients displayed fewer symptoms, more sustained remission of CT imaging and earlier RNA negative-conversion but similar plasma antibody levels during their hospitalization period as compared to those NRP patients. When re-admitted to the hospital, these RP patients showed no obvious clinical symptoms or disease progression indicated by normal or improving CT imaging and inflammatory cytokine levels. All 21 close contacts of RP patients were tested negative for SARS-CoV-2 RNA, and no suspicious clinical symptoms were reported. However, 18/24 of RNA-negative samples detected by the commercial kit were tested to be positive for virus RNA using a hyper-sensitive method, suggesting the carrier status of virus possibly existed in patients recovered from COVID-19. Interpretation Our results showed that young and mild COVID-19 patients seem to be RP patients after discharge, who show no obviously clinical symptoms and disease progression upon re-admission. More sensitive RNA All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '1ee4cc235ce07b2d08d989e9001e2f4bd9af3138',\n","  'section': 'Abstract',\n","  'text': 'Since December 2019, Coronavirus Disease 2019 (COVID-19) emerged in Wuhan city and rapidly spread throughout China. The mortality of novel coronavirus pneumonia (NCP) in severe and critical cases is very high. Facing this kind of public health emergency, high efficient administrative emergency responsive mode in designated hospital is needed. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '1ef4b7e4a7fad1570e093426bb5aec602b0ae237',\n","  'section': 'Abstract',\n","  'text': 'The ongoing coronavirus disease 2019 pandemic is of global concern and has recently emerged in the US. In this paper, we construct a stochastic variant of the SEIR model to make a quasi-worst-case scenario prediction of the COVID-19 outbreak in the US West and East Coasts. The model is then fitted to current data and implemented using Runge-Kutta methods. Our computation results predict that the number of new cases would peak around mid-April 2000 and begin to abate by July, and that the number of cases of COVID-19 might be significantly mitigated by having greater numbers of functional testing kits available for screening. The model also showed how small changes in variables can make large differences in outcomes and highlights the importance of healthcare preparedness during pandemics. '},\n"," {'paper_id': '1f632ba80bc3f4c8a48b5d436dd843c3c48bbcf2',\n","  'section': 'Abstract',\n","  'text': 'Motivation: DNA metabarcoding is a com- '},\n"," {'paper_id': '1f65cad83a480573c71ab26c4a544757ff66067d',\n","  'section': 'Abstract',\n","  'text': 'Covid-19 (coronavirus disease 2019) is a rapidly spreading pandemic in many countries. The total confirmed cases of Covid-19 are exponentially increasing and many countries are fighting Covid-19 with all strategies. However, there is still lacking consensus for effective strategies. Here, I demonstrate the time dependence of the doubling time in the Covid-19 exponential growths. Tracking the time-dependent doubling time tells how well Covid-19 prevention works, giving an index for successful fighting. '},\n"," {'paper_id': '1f783a5e029f80516169b19fe6e0dacc1e171f87',\n","  'section': 'Abstract',\n","  'text': 'Influenza is a major global public health threat as a result of its highly pathogenic variants, large 27 zoonotic reservoir, and pandemic potential. Metagenomic viral sequencing offers the potential of 28 a diagnostic test for influenza which also provides insights on transmission, evolution and drug 29 resistance, and simultaneously detects other viruses. We therefore set out to apply Oxford 30 Nanopore Technology to metagenomic sequencing of respiratory samples. We generated 31 influenza reads down to a limit of detection of 10 2 -10 3 genome copies/ml in pooled samples, 32 observing a strong relationship between the viral titre and the proportion of influenza reads (p = 33 4.7x10 -5 ). Applying our methods to clinical throat swabs, we generated influenza reads for 27/27 34 samples with high-to-mid viral titres (Cycle threshold (Ct) values <30) and 6/13 samples with low 35 viral titres (Ct values 30-40). No false positive reads were generated from 10 influenza-negative 36 samples. Thus Nanopore sequencing operated with 83% sensitivity (95% CI 67-93%) and 100% 37 specificity (95% CI 69-100%) compared to the current diagnostic standard. Coverage of full 38 length virus was dependent on sample composition, being negatively influenced by increased 39 host and bacterial reads. However, at high influenza titres, we were able to reconstruct >99% 40 complete sequence for all eight gene segments. We also detected Human Coronavirus and 41 generated a near complete Human Metapneumovirus genome from clinical samples. While 42 further optimisation is required to improve sensitivity, this approach shows promise for the 43 Nanopore platform to be used in the diagnosis and genetic analysis of influenza and other 44 respiratory viruses. 45 46 '},\n"," {'paper_id': '1f80f32ab88715153f9690b096115e53f2fd6121',\n","  'section': 'Abstract',\n","  'text': 'medRxiv preprint 3 generally longer incubation and serial interval of less severe cases suggests a high risk of longterm epidemic in the absence of appropriate control measures. '},\n"," {'paper_id': '1f9d3f9a1a0e8db6a086e0a2b5ba50cf9f235dae',\n","  'section': 'Abstract',\n","  'text': 'A pattern in which nucleotide transitions are favored several-fold over transversions is common in molecular evolution. When this pattern occurs among amino acid replacements, explanations often invoke an effect of selection, on the grounds that transitions are more conservative in their effects on proteins. However, the underlying hypothesis of conservative transitions has never been tested directly. Here we assess support for this hypothesis using direct evidence: the fitness effects of mutations in actual proteins, measured via individual or paired growth experiments. We assembled data from 8 published studies, ranging in size from 24 to 757 single-nucleotide mutations that change an amino acid. Every study has the statistical power to reveal significant effects of amino acid exchangeability, and most studies have the power to discern a binary conservative-vs-radical distinction. However, only one study suggests that transitions are significantly more conservative than transversions. In the combined set of 1239 replacements, the chance that a transition is more conservative than a transversion is 53 % (95 % confidence interval, 50 % to 56 %), compared to the null expectation of 50 %. We show that this effect is not large compared to that of most biochemical factors, and is not large enough to explain the several-fold bias observed in evolution. In short, available data have the power to verify the \"conservative transitions\" hypothesis if true, but suggest instead that selection on proteins plays at best a minor role in the observed bias. . CC-BY 4.0 International license is made available under a '},\n"," {'paper_id': '1fb909712eea684a4fae4f0b419ca6bc7ba8e396',\n","  'section': 'Abstract',\n","  'text': 'For decades, cationic polymer nanoparticles have been investigated for nucleic acid delivery. Despite promising in vitro transfection results, most formulations have failed to translate into the clinic due to significant in vivo toxicity -especially when delivered intravenously. To address this significant problem, we investigated the detailed mechanisms that govern the complex in vivo systemic toxicity response to common polymeric nanoparticles. We determined that the toxicity response is material dependent. For branched polyethylenimine (bPEI) nanoparticles -toxicity is a function of multiple pathophysiological responses -triggering of innate immune sensors, induction of hepatic toxicity, and significant alteration of hematological properties. In contrast, for chitosan-based nanoparticles -systemic toxicity is primarily driven through innate immune activation. We further identified that modification of primary amines to secondary and tertiary amines using the small molecule imidazole-aceticacid (IAA) ameliorates in vivo toxicity from both nanocarriers by different, material-specific mechanisms related to Toll-like receptor 4 activation (for bPEI) and complement activation driven neutrophil infiltration (for chitosan), respectively. Our results provide a detailed roadmap for evaluating in vivo toxicity of nanocarriers and identifies potential opportunities to reduce toxicity for eventual clinical translation. '},\n"," {'paper_id': '1fe8e5f5709aa0e7fb3971ce721c27b879020e6e',\n","  'section': 'Abstract',\n","  'text': 'Background Since December 2019, a cluster of coronavirus disease 2019 (COVID-19) occurred in Wuhan, Hubei Province, China and spread rapidly from China to other countries. In-hospital mortality are high in severe cases and cardiac injury characterized by elevated cardiac troponin are common among them. The mechanism of cardiac injury and the relationship between cardiac injury and in-hospital mortality remained unclear. Studies focused on cardiac injury in COVID-19 patients are scarce. Objectives To investigate the association between cardiac injury and in-hospital mortality of patients with confirmed or suspected COVID-19. Methods Demographic, clinical, treatment, and laboratory data of consecutive confirmed or suspected COVID-19 patients admitted in Wuhan No.1 Hospital from 25 th December, 2019 to 15 th February, 2020 were extracted from electronic medical records and were retrospectively reviewed and analyzed. Univariate and multivariate Cox regression analysis were used to explore the risk factors associated with in-hospital death. A total of 110 patients with confirmed (n=80) or suspected (n=30) COVID-19 were screened and 48 patients (female 31.3%, mean age 70.58±13.38 year old) among them with high-sensitivity cardiac troponin I (hs-cTnI) test within 48 hours after admission were included, of whom 17 (17/48, 35.4%) died in hospital while 31 (31/48, 64.6%) were discharged or transferred to other hospital. High-sensitivity cardiac troponin I was elevated in 13 (13/48, 27.1%) patents. Multivariate Cox regression analysis showed pulse oximetry of oxygen saturation (SpO2) on admission (HR 0.704, 95% CI 0.546-0.909, per 1% decrease, p=0.007), elevated hs-cTnI (HR 10.902, 95% 1.279-92.927, p=0.029) and elevated d-dimer (HR 1.103, 95%CI 1.034-1.176, per 1mg/L increase, p=0.003) on admission were independently associated with in-hospital mortality. '},\n"," {'paper_id': '20154fa12ca525582cd1b42019927e8f7816fb51',\n","  'section': 'Abstract',\n","  'text': 'The use of metabolomics profiling to understand metabolism under different 20 physiological states has increased in recent years, which created the need for robust analytical 21 platforms. Here, we present a validated method for targeted and semi-quantitative analysis of 102 22 polar metabolites that covers major metabolic pathways from 24 classes in a single 17.5-min assay. The method has been optimized for a wide range of biological matrices from various organisms, 24 and involves automated sample preparation, and data processing using in-house developed R 25 package. To ensure reliability, the method was validated for accuracy, precision, selectivity, 26 specificity, linearity, recovery, and stability according to European Medicines Agency guidelines. We demonstrated excellent repeatability of the retention times (CV<4%), calibration curves 28 (R 2 ≥0.980) in their respective wide dynamic concentration ranges (CV<3%), and concentrations 29 (CV<25%) of quality control samples interspersed within 25 batches analyzed over a period of one- year. The robustness was demonstrated through high correlation between metabolite 31 concentrations measured using our method and NIST reference values (R 2 =0.967), including cross-32 platform comparability against the BIOCRATES AbsoluteIDQp180 kit (R 2 =0.975) and NMR analyses 33 (R 2 =0.884). We have shown that our method can be successfully applied in many biomedical 34 research fields and clinical trials, including epidemiological studies for biomarker discovery. In 35 summary, a thorough validation demonstrated that our method is reproducible, robust, reliable, 36 and suitable for metabolomics studies. '},\n"," {'paper_id': '207dcb6e3cd43cdea91d29a15be1ee34068bd54e',\n","  'section': 'Abstract',\n","  'text': 'The potential infectiousness of asymptomatic COVID-19 cases together with a substantial fraction of asymptomatic infections among all infections, have been highlighted in clinical studies. We conducted statistical modeling analysis to derive the delay-adjusted asymptomatic proportion of the positive COVID-19 infections onboard the Princess Cruises ship along with the timeline of infections. We estimated the asymptomatic proportion at 17.9% (95% CrI: 15.5%-20.2%), with most of the infections occurring before the start of the 2-week quarantine. Word count: Main: '},\n"," {'paper_id': '20ad63c451320613d188d79179a9a65949da50f7',\n","  'section': 'Abstract',\n","  'text': 'Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is the causative agent of the Coronavirus disease 2019 (COVID-19) which is currently negatively affecting the population and disrupting the global economy. SARS-CoV-2 belongs to the +RNA virus family that utilize single-stranded positive-sense RNA molecules as genomes. SARS-CoV-2, like other coronaviruses, has an unusually large genome for a +RNA virus that encodes four structural proteinsthe matrix (M), small envelope (E), spike (S) and nucleocapsid phosphoprotein (N) -and sixteen nonstructural proteins (nsp1-16) that together ensure replication of the virus in the host cell. The nucleocapsid phosphoprotein N is essential for linking the viral genome to the viral membrane. Its N-terminal RNA binding domain (N-NTD) captures the RNA genome while the Cterminal domain anchors the ribonucleoprotein complex to the viral membrane via its interaction with the M protein. Here, we characterized the structure of the N-NTD and its interaction with RNA using NMR spectroscopy. We observed a positively charged canyon on the surface of the N-NTD lined with arginine residues suggesting a putative RNA binding site. Next, we performed an NMR titration experiment using an RNA duplex. The observed changes in positions of signals in the N-NTD NMR spectra allowed us to construct a model of the N-NTD in complex with RNA. '},\n"," {'paper_id': '20c09ed5eeacb1e7cc346d6cee13e46a5284490d',\n","  'section': 'Abstract',\n","  'text': 'has rapidly spread to almost all parts of world. In coronaviruses, the receptor binding domain (RBD) in the distal part of S1 subunit of SARS-CoV-2 spike protein can directly bind to angiotensin converting enzyme 2 (ACE2). RBD promote viral entry into the host cells and is an important therapeutic target. In this study, we discovered that theaflavin showed the lower idock score (idock score: −7.95 kcal/mol). To confirm the result, we discovered that theaflavin showed FullFitness score of -991.21 kcal/mol and estimated ΔG of -8.53 kcal/mol for the most favorable interaction with contact area of SARS-CoV-2 RBD by SwissDock service. Regarding contact modes, hydrophobic interactions contribute significantly in binding and additional hydrogen bonds were formed between theaflavin and Arg454, Phe456, Asn460, Cys480, Gln493, Asn501 and Val503 of SARS-CoV-2 RBD, near the direct contact area with ACE2. Our results suggest that theaflavin could be the candidate of SARS-CoV-2 entry inhibitor for further study. '},\n"," {'paper_id': '20f2a638e2c9a1917b7c001c371335da3d288505',\n","  'section': 'Abstract',\n","  'text': 'The recent reclassification of the Riboviria, and the introduction of multiple new taxonomic 10 categories including both subfamilies and subgenera for coronaviruses (family Coronaviridae, 11 subfamily Orthocoronavirinae) represents a major shift in how official classifications are used to 12 designate specific viral lineages. While the newly defined subgenera provide much-needed 13 standardisation for commonly cited viruses of public health importance, no method has been 14 proposed for the assignment of subgenus based on partial sequence data, or for sequences that are 15 divergent from the designated holotype reference genomes. Here, we describe the genetic variation 16 of a partial region of the coronavirus RNA-dependent RNA polymerase (RdRp), which is one of the 17 most used partial sequence loci for both detection and classification of coronaviruses in molecular 18 epidemiology. We infer Bayesian phylogenies from more than 7000 publicly available coronavirus 19 sequences and examine clade groupings relative to all subgenus holotype sequences. Our 20 phylogenetic analyses are largely coherent with genome-scale analyses based on designated 21 holotype members for each subgenus. Distance measures between sequences form discrete clusters 22 between taxa, offering logical threshold boundaries that can attribute subgenus or indicate 23 sequences that are likely to belong to unclassified subgenera both accurately and robustly. We thus 24 propose that partial RdRp sequence data of coronaviruses is sufficient for the attribution of 25 subgenus-level taxonomic classifications and we supply the R package, \"MyCoV\", which provides a 26 method for attributing subgenus and assessing the reliability of the attribution. 27 author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '21100aba41a4bfb48d7dc37f1bf5dbb38bf3867a',\n","  'section': 'Abstract',\n","  'text': 'Human Coronaviruses (HCoVs), including severe acute respiratory syndrome coronavirus (SARS-CoV), Middle east respiratory syndrome coronavirus (MERS-CoV), and 2019 novel coronavirus (2019-nCoV), lead global epidemics with high morbidity and mortality. However, there are currently no effective drugs targeting 2019-nCoV. Drug repurposing, represented as an effective drug discovery strategy from existing drugs, could shorten the time and reduce the cost compared to de novo drug discovery. In this study, we present an integrative, antiviral drug repurposing methodology implementing a systems pharmacology-based network medicine platform, quantifying the interplay between the HCoV-host interactome and drug targets in the human protein-protein interaction network. Phylogenetic analyses of 15 HCoV whole genomes reveal that 2019-nCoV has the highest nucleotide sequence identity with SARS-CoV (79.7%) among the six other known pathogenic HCoVs. Specifically, the envelope and nucleocapsid proteins of 2019-nCoV are two evolutionarily conserved regions, having the sequence identities of 96% and 89.6%, respectively, compared to SARS-CoV. Using network proximity analyses of drug targets and known HCoV-host interactions in the human protein-protein interactome, we computationally identified 135 putative repurposable drugs for the potential prevention and treatment of HCoVs. In addition, we prioritized 16 potential anti-HCoV repurposable drugs (including melatonin, mercaptopurine, and sirolimus) that were further validated by enrichment analyses of drug-gene signatures and HCoV-induced transcriptomics data in human cell lines. Finally, we showcased three potential drug combinations (including sirolimus plus dactinomycin, mercaptopurine plus melatonin, and toremifene plus emodin) captured by . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': '212653c35d23a66ab384645e2df82a63db28d1ea',\n","  'section': 'Abstract',\n","  'text': 'Detection of SARS-CoV-2 infections to date has relied on RT-PCR testing. However, a 10 failure to identify early cases imported to a country, bottlenecks in RT-PCR testing, and the existence of infections which are asymptomatic, sub-clinical, or with an alternative presentation than the standard cough and fever have resulted in an under-counting of the true prevalence of SARS-CoV-2. Here, we show how publicly available CDC influenza-like illness (ILI) outpatient surveillance data can be repurposed to estimate the detection rate of symp-15 tomatic SARS-CoV-2 infections. We find a surge of non-influenza ILI above the seasonal average and show that this surge is correlated with COVID case counts across states. By quantifying the number of excess ILI patients in March relative to previous years and comparing excess ILI to confirmed COVID case counts, we estimate the symptomatic case detection rate of SARS-CoV-2 in the US to be 1/100 to 1/1000. This corresponds to approximately 20 10 million presumed symptomatic SARS-CoV-2 patients across the US during the week starting on March 15, 2020. Combining excess ILI counts with the date of onset of community transmission in the US, we also show that the early epidemic in the US was unlikely to be doubling slower than every three days. Together these results suggest a conceptual model for the COVID epidemic in the US in which rapid spread across the US are combined with 25 a large population of infected patients with presumably mild-to-moderate clinical symptoms. We emphasize the importance of testing these findings with seroprevalence data, and discuss the broader potential to repurpose outpatient time series for early detection and understanding of emerging infectious diseases. Here, we quantify background levels of non-influenza ILI over the past 10 years and identify a recent surge of non-influenza ILI starting the first week of March, 2020. This surge of excess 1 . CC-BY 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': '214ef8154bf31571fcb97fd44b8403df7e208e80',\n","  'section': 'Abstract',\n","  'text': 'Early detection of COVID-19 based on chest CT will enable timely treatment of patients and help control the spread of the disease. With rapid spreading of COVID-19 in many countries, however, CT volumes of suspicious patients are increasing at a speed much faster than the availability of human experts. Here, we propose an artificial intelligence (AI) system for fast COVID-19 diagnosis with an accuracy comparable to experienced radiologists. A large dataset was constructed by collecting 970 CT volumes of 496 patients with confirmed COVID-19 and 260 negative cases from three hospitals in Wuhan, China, and 1,125 negative cases from two publicly available chest CT datasets. Trained using only 312 cases, our diagnosis system, which is based on deep convolutional neural network, is able to achieve an accuracy of 94.98%, an area under the receiver operating characteristic curve (AUC) of 97.91%, a sensitivity of 94.06%, and a specificity of 95.47% on an independent external verification dataset of 1,255 cases. In a reader study involving five radiologists, only one radiologist is slightly more accurate than the AI system. The AI system is two orders of magnitude faster than radiologists and the code is available at https : //github.com '},\n"," {'paper_id': '21c4d6c08d861b792900c32c5a0df1c1d407a005',\n","  'section': 'Abstract',\n","  'text': 'Guided by the principle of primum non nocere (first do no harm), we report a cautionary note on the potential fatal toxicity of chloroquine (CQ) or hydroxychloroquine (HCQ) in combination with anti-diabetic drug metformin. We observed that the combination of CQ or HCQ and metformin, which were used in our studies as potential anti-cancer drugs, killed 30-40% of mice. While our observations in mice may not translate to toxicity in humans, the reports that CQ or HCQ has anti-COVID-19 activity, the use of CQ resulting in toxicity and at least one death, and the recent Emergency Use Authorization (EUA) for CQ and HCQ by the US Food and Drug Administration (FDA) prompted our report. Here we report the lethality of CQ or HCQ in combination with metformin as a warning of its potential serious clinical toxicity. We hope that our report will be helpful to stimulate pharmacovigilance and monitoring of adverse drug reactions with the use of CQ or HCQ, particularly in combination with metformin. '},\n"," {'paper_id': '22050361f4804b78eff485fb99a7dfa42a0438ec',\n","  'section': 'Abstract',\n","  'text': 'Coronavirus pandemic of 2019-2020 has already affected over a million people and caused over 50,000 deaths worldwide (as on April 3, 2020). Roughly half of the world population has been asked to work from home and practice social distancing as the search for a vaccine continues. Though government interventions such as lockdown and social distancing are theoretically useful, its debatable whether such interventions are effective in flattening the curve, which is ceasing or reducing the growth of infection in control populations. In this article, I present a simple power law model that enables a comparison of countries in time windows of 14 days since first coronavirus related death is reported in that country. It therefore provides means to access the efficacy of above interventions. '},\n"," {'paper_id': '224a389bd5dbeab45b8a25965298016fbc44d669',\n","  'section': 'Abstract',\n","  'text': 'Primary human airway epithelial cell (hAEC) cultures represent a universal platform to 20 propagate respiratory viruses and characterize their host interactions in authentic target cells. To 21 further elucidate specific interactions between human respiratory viruses and important host factors 22 in airway epithelium, it is important to make hAEC cultures amenable to genetic modification. 23 However, the short and finite lifespan of primary cells in cell culture creates a bottleneck for the 24 genetic modification of these cultures. In the current study, we show that the incorporation of the 25 Rho-associated protein kinase (ROCK) inhibitor (Y-27632) during cell propagation extends the life 26 span of primary human cells in vitro and thereby facilitates the incorporation of lentivirus-based 27 expression systems. Using fluorescent reporters for FACS-based sorting, we generated 28 homogenously fluorescent hAEC cultures that differentiate normally after lentiviral transduction. As 29 proof-of-principle, we demonstrate that host gene expression can be modulated post-differentiation 30 via inducible short hairpin (sh)RNA-mediated knockdown. Importantly, functional characterization of 31 these transgenic hAEC cultures with exogenous poly(I:C), as a proxy for virus infection, 32 demonstrates that such modifications do not influence the host innate immune response. Moreover, 33 the propagation kinetics of both human coronavirus 229E (HCoV-229E) and human respiratory 34 syncytial virus (RSV) were not affected. Combined, these results validate our newly established 35 protocol for the genetic modification of hAEC cultures thereby unlocking a unique potential for 36 detailed molecular characterization of virus -host interactions in human respiratory epithelium. 37 '},\n"," {'paper_id': '225f36e98f690b835d8cf0d8794afa9e89306038',\n","  'section': 'Abstract',\n","  'text': 'In this paper are presented predictions on the evolution in time of the number of positive cases in Italy of the Covid-19 pandemic based on official data and on the use of a function of the type of a Gauss Error Function as a Cumulative Distribution Function (CDF). We have analyzed the available data for China and Italy. The evolution in time of the number of cumulative diagnosed positive cases of Covid-19 in China very well approximates a distribution of the type of the Error Function, that is, the integral of a normal, Gaussian distribution. We have then used such a function to study the potential evolution in time of the number of positive cases in Italy by performing a number of fits of the official data so far available. We then found a statistical prediction for the day in which the peak of the number of daily positive cases in Italy occurs, corresponding to the flex of the fit, i.e., to the change in sign of its second derivative (that is the change from acceleration to deceleration) as well as of the day in which a substantial attenuation of such number of daily cases is reached. We have then performed 150 Monte Carlo simulations in the attempt to have a more robust prediction of the day of the above-mentioned peak and of the day of the substantial decrease of the number of daily positive cases. Although, official data have been used, these predictions are obtained with a heuristic approach, since those predictions are based on statistical approach and do not take into account either a number of relevant issues (such as medical, social distancing, virologic, epidemiological, etc.) or models of contamination diffusion. All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '227002d1833cfefb0e75033df14a87c7ca8f4d16',\n","  'section': 'Abstract',\n","  'text': 'As new cases of COVID-19 are being confirmed pressure is mounting to increase understanding of the factors underlying the spread the disease. Using data on local transmissions until the 23rd of March 2020, we develop an ensemble of 200 ecological niche models to project monthly variation in climate suitability for spread of SARS-CoV-2 throughout a typical climatological year. Although cases of COVID-19 are reported all over the world, most outbreaks display a pattern of clustering in relatively cool and dry areas. The predecessor SARS-CoV-1 was linked to similar climate conditions. Should the spread of SARS CoV-2 continue to follow current trends, asynchronous seasonal global outbreaks could be expected. According to the models, temperate warm and cold climates are more favorable to spread of the virus, whereas arid and tropical climates are less favorable. However, model uncertainties are still high across much of sub-Saharan Africa, Latin America and South East Asia. While models of epidemic spread utilize human demography and mobility as predictors, climate can also help constrain the virus. This is because the environment can mediate human-to-human transmission of SARS-CoV-2, and unsuitable climates can cause the virus to destabilize quickly, hence reducing its capacity to become epidemic. '},\n"," {'paper_id': '229fae391499a24af01bc6b50204afe09c050f8c',\n","  'section': 'Abstract',\n","  'text': 'for useful comments and suggestions. All remaining errors are our own. We quantify the causal impact of human mobility restrictions, particularly the lockdown of the city of Wuhan on January 23, 2020, on the containment and delay of the spread of the Novel Coronavirus (2019-nCoV). We employ a set of differencein-differences (DID) estimations to disentangle the lockdown effect on human mobility reductions from other confounding effects including panic effect, virus effect, and the Spring Festival effect. We find that the lockdown of Wuhan reduced inflow into Wuhan by 76.64%, outflows from Wuhan by 56.35%, and within-Wuhan movements by 54.15%. We also estimate the dynamic effects of up to 22 lagged population inflows from Wuhan and other Hubei cities, the epicenter of the 2019-nCoV outbreak, on the destination cities\\' new infection cases. We find, using simulations with these estimates, that the lockdown of the city of Wuhan on January 23, 2020 contributed significantly to reducing the total infection cases outside of Wuhan, even with the social distancing measures later imposed by other cities. We find that the COVID-19 cases would be 64.81% higher in the 347 Chinese cities outside Hubei province, and 52.64% higher in the 16 non-Wuhan cities inside Hubei, in the counterfactual world in which the city of Wuhan were not locked down from January 23, 2020. We also find that there were substantial undocumented infection cases in the early days of the 2019-nCoV outbreak in Wuhan and other cities of Hubei province, but over time, the gap between the officially reported cases and our estimated \"actual\" cases narrows significantly. We also find evidence that enhanced social distancing policies in the 63 Chinese cities outside Hubei province are effective in reducing the impact of population inflows from the epicenter cities in Hubei province on the spread of 2019-nCoV virus in the destination cities elsewhere. 3 With a population of over 11 million, Wuhan is the largest city in Hubei, the most populous city in Central China, and the seventh most populous city in China. The city is also a major transportation hub, with dozens of railways, roads and expressways passing through and connecting to other major cities, and home to 82 colleges and more than 1 million college students. '},\n"," {'paper_id': '22a4d26bde67d878869443ea26140f3d8239bed0',\n","  'section': 'Abstract',\n","  'text': 'Heterogeneity in disease transmission is widespread and, when not accounted for, can 12 produce unpredictable outbreaks of infectious disease. Despite this, precisely how 13 different sources of variation in host traits drive heterogeneity in disease transmission is 14 poorly understood. Here we dissected the sources of variation in pathogen transmission 15 using Drosophila melanogaster and Drosophila C Virus as a host-pathogen model 16 system. We found that infected lifespan, viral growth, virus shedding, and viral load at 17 death were all significantly influenced by fly genetic background, sex and female mating 18 status. To understand how variation in each of these traits may generate heterogeneity 19 in disease transmission, we estimated individual transmission potential by integrating 20 data on virus shedding and lifespan alongside previously collected data on social 21 aggregation. We found that ~15% of between-individual heterogeneity in disease 22 transmission was explained by a significant interaction between genetic and sex-specific 23 variation. We also characterised the amount of variation in viral load, virus shedding, and 24 lifespan following infection that could be explained by genetic background and sex. 25 Amongst the determinants of individual variation in disease transmission these sources 26 of host variation play roles of varying importance, with genetic background generally 27 playing the largest role. Our results highlight the importance of characterising sources of 28 variation in multiple host traits when studying disease transmission at the individual-level. 29 30 '},\n"," {'paper_id': '22a64ef0a092051de373de119d918deb5278cbae',\n","  'section': 'Abstract',\n","  'text': 'for systematic evaluation of the logistic model to predict epidemic growth. We sequentially estimated sets of model parameters (maximum number of cases K, growth rate r, and half-time t0) and the epidemic \"end time\" t95 (defined as the time when the number of cases, predicted or actual, reached 95% of the maximum). The estimates of these parameters were done for sequences of reported cases growing daily (back-casting for China and forecasting for the US). In both countries, the estimates of K grew very much in time during the exponential and nearly exponential phases making longer term forecasting not reliable. For the US, the current estimate of the maximum number of cases K is about 265,000 but it is very likely that it will grow in the future. However, running estimates of the \"end time\" t95 were in a much smaller interval for China (60 -70 days vs. the actual value of 67). For the US, the values estimated from the data sequences going back two weeks from now range from 70 to 80 days. If the behavior of the US epidemic is similar to the previous Chinese development, the number of reported cases could reach a maximum around April (2020) \"A mathematical model for simulating the transmission of Wuhan novel Coronavirus.\" bioRxiv, '},\n"," {'paper_id': '22ab9f53a5e0eca57b7a0d13e0773f9522022605',\n","  'section': 'Abstract',\n","  'text': 'A group of biologists, ARTIC Network, has proposed a multiplexed PCR primer set for whole-genome analysis of the novel coronavirus, SARS-CoV-2, soon after the start of COVID-19 epidemics was realized. The primer set was adapted by many researchers worldwide and has already contributed to the high-quality and prompt genome epidemiology '},\n"," {'paper_id': '22d12f509369047b9dcbb438042c93c5c044999d',\n","  'section': 'Abstract',\n","  'text': \"Recombinant protein production is a widely used technique in the biotechnology and biomedical industries, yet only a quarter of target proteins are soluble and can therefore be purified. We have discovered that global structural flexibility, which can be modeled by normalised B-factors, accurately predicts the solubility of 12,216 recombinant proteins expressed in Escherichia coli . We have optimised B-factors, and derived a new set of values for solubility scoring that further improves prediction accuracy. We call this new predictor the 'Solubility-Weighted Index' (SWI). Importantly, SWI outperforms many existing protein solubility prediction tools. Furthermore, we have developed 'SoDoPE' (Soluble Domain for Protein Expression), a web interface that allows users to choose a protein region of interest for predicting and maximising both protein expression and solubility. The SoDoPE web server and source code are freely available at https://tisigner.com/sodope and https://github.com/Gardner-BinfLab/TISIGNER-ReactJS , respectively. \"},\n"," {'paper_id': '22d9cbf4299bbbe5ec15c0c9950d2690bf0fd0eb',\n","  'section': 'Abstract',\n","  'text': 'Anthropization of natural habitats including climate change along with overpopulation and global travel have been contributing to emerging infectious diseases outbreaks. The recent COVID-19 outbreak in Wuhan, highlights such threats to human health, social stability and global trade and economy. We used species distribution modelling and environmental data from satellite imagery to model Blueprint Priority Diseases occurrences. We constructed classical regression and Support Vector Machine models based on environmental predictor variables such as landscape, tree cover loss, climatic covariates. Models were evaluated and a weighed mean was used to map the predictive risk of disease emergence. We mapped the predictive risk for filovirus, Nipah, Rift Valley Fever and coronavirus diseases. Elevation, tree cover loss and climatic covariates were found to significant factors influencing disease emergence. We also showed the relevance of disease biogeography and in the identification potential hotspots for Disease X in regions in Uganda and China. '},\n"," {'paper_id': '22f3be8bbf6d792549b7a6088df7440e425cb34f',\n","  'section': 'Abstract',\n","  'text': 'The new coronavirus (2019-nCoV) outbreak from December 2019 in Wuhan, Hubei, China, has been declared a global public health emergency. Angiotensin I converting enzyme 2 (ACE2), is the host receptor by 2019-nCov to infect human cells. Although ACE2 is reported to be expressed in lung, liver, stomach, ileum, kidney and colon, its expressing levels are rather low, especially in the lung. 2019-nCoV may use co-receptors/auxiliary proteins as ACE2 partner to facilitate the virus entry. To identify the potential candidates, we explored the single cell gene expression atlas including 119 cell types of 13 human tissues and analyzed the single cell coexpression spectrum of 51 reported RNA virus receptors and 400 other membrane proteins. Consistent with other recent reports, we confirmed that ACE2 was mainly expressed in lung AT2, liver cholangiocyte, colon colonocytes, esophagus keratinocytes, ileum ECs, rectum ECs, stomach epithelial cells, and kidney proximal tubules. Intriguingly, we found that the candidate co-receptors, manifesting the most similar expression patterns with ACE2 across 13 human tissues, are all peptidases, including ANPEP, DPP4 and ENPEP. Among them, ANPEP and DPP4 are the known receptors for human CoVs, suggesting ENPEP as another potential receptor for human CoVs. We also conducted \"CellPhoneDB\" analysis to understand the cell crosstalk between CoV-targets and their surrounding cells across different tissues. We found that macrophages frequently communicate with the CoVs targets through chemokine and phagocytosis signaling, highlighting the importance of tissue macrophages in immune defense and immune pathogenesis. '},\n"," {'paper_id': '230a43b9bd4bc48bf22b55fbf32c61d408db5de8',\n","  'section': 'Abstract',\n","  'text': 'BACKGROUND: Since its identification on the 7 th of January 2020, SARS-CoV-2 has spread to more than 180 countries worldwide, causing >11,000 deaths. At present, viral disease and transmission amongst children is incompletely understood. Specifically, there is concern that children could be an important source of SARS-CoV-2 in household transmission clusters. '},\n"," {'paper_id': '231512da8edf0a87ec6e9c99712b286e60712347',\n","  'section': 'Abstract',\n","  'text': 'COVID-19 is an emerging infectious disease and was recently declared as a pandemic by WHO. Currently, there is no vaccine or therapeutic available for this disease. Drug repositioning represents the only feasible option to address this global challenge and a panel of 48 FDAapproved drugs that have been pre-selected by an assay of SARS-CoV was screened to identify potential antiviral drug candidates against SARS-CoV-2 infection. We found a total of 24 drugs which exhibited antiviral efficacy (0.1 µM < IC 50 < 10 µM) against SARS-CoV-2. In particular, two FDA-approved drugs -niclosamide and ciclesonide -were notable in some respects. These drugs will be tested in an appropriate animal model for their antiviral activities. In near future, these already FDA-approved drugs could be further developed following clinical trials in order to provide additional therapeutic options for patients with COVID-19. author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '233978852b45429db49d2aabe1c9c9bfe73b1884',\n","  'section': 'Abstract',\n","  'text': 'medRxiv preprint COVID-19 has been impacting on the whole world critically and constantly Since December 2019. We have independently developed a novel statistical time delay dynamic model on the basis of the distribution models from CCDC. Based only on the numbers of confirmed cases in different regions in China, the model can clearly reveal that the containment of the epidemic highly depends on early and effective isolation. We apply the model on the epidemic in Japan and conclude that there could be a rapid outbreak in Japan if no effective quarantine measures are carried out immediately. '},\n"," {'paper_id': '23b618de7564c9acf9d35c75ed18d7d15adac6e9',\n","  'section': 'Abstract',\n","  'text': 'These authors contributed equally to this work. ⇤ The sudden outbreak of novel coronavirus 2019 increased the diagnostic burden of radiologists. In the time of an epidemic crisis, we hoped artificial intelligence (AI) to help reduce physician workload in regions with the outbreak, and improve the diagnosis accuracy for physicians before they could acquire enough experience with the new disease. Here, we present our experience in building and deploying an AI system that automatically analyzes CT images to detect COVID-19 pneumonia features. Different from conventional medical AI, we were dealing with an epidemic crisis. Working in an interdisciplinary team of over 30 people with medical and / or AI background, geographically distributed in Beijing and Wuhan, we were able to overcome a series of challenges in this particular situation and deploy the system in four weeks. Using 1,136 training cases (723 positives for COVID-19) from five hospitals, we were able to achieve a sensitivity of 0.974 and specificity of 0.922 on the test dataset, which included a variety of pulmonary diseases. Besides, the system automatically highlighted all lesion regions for faster examination. As of today, we have deployed the system 2 All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '23e7355b5e4e0209f64c9d8d5772092a53b72686',\n","  'section': 'Abstract',\n","  'text': 'Aims: Studies have indicated that chloroquine (CQ) shows antagonism against COVID-19 in vitro. '},\n"," {'paper_id': '23f1bab7d94bd0564e2319fddf75c1c7987ce75f',\n","  'section': 'Abstract',\n","  'text': 'We study an epidemic model with nonlinear incidence rate, describing the saturated mass action as well as the psychological effect of certain serious diseases on the community. Firstly, the existence and local stability of disease-free and endemic equilibria are investigated. Then, we prove the occurrence of backward bifurcation, saddle-node bifurcation, Hopf bifurcation and cusp type Bogdanov-Takens bifurcation of codimension 3. Finally, numerical simulations, including one limit cycle, two limit cycles, unstable homoclinic loop and many other phase portraits are presented. These results show that the psychological effect of diseases and the behavior change of the susceptible individuals may affect the final spread level of an epidemic. doi: bioRxiv preprint then ∆ > 0 if and only if R 0 > R * , ∆ = 0 if and only if R 0 = R * and ∆ < 0 if and only if R 0 < R * . It is obvious that R * < 1 and we can obtain the following theorem. 40 = q 20 = q 21 = q 22 = q 30 = q 31 = q 40 = 0. '},\n"," {'paper_id': '24bc718affd4fd658af41193218609b30d5ae88e',\n","  'section': 'Abstract',\n","  'text': 'COVID-19 threatens to overwhelm hospital facilities throughout the United States. We created an interactive, quantitative model that forecasts demand for COVID-19 related hospitalization based on county-level population characteristics, data from the literature on COVID-19, and data from online repositories. Using this information as well as user inputs, the model estimates a time series of demand for intensive care beds and acute care beds as well as the availability of those beds. The online model is designed to be intuitive and interactive so that local leaders with limited technical or epidemiological expertise may make decisions based on a variety of scenarios. This complements high-level models designed for public consumption and technically sophisticated models designed for use by epidemiologists. The model is actively being used by several academic medical centers and policy makers, and we believe that broader access will continue to aid community and hospital leaders in their response to COVID-19. '},\n"," {'paper_id': '24e17488d399c436305c819953beae2961214771',\n","  'section': 'Abstract',\n","  'text': 'Background: SARS-CoV-2-caused coronavirus disease (COVID-19) is posing a large casualty. The features of COVID-19 patients with and without pneumonia, SARS-CoV-2 transmissibility in asymptomatic carriers, and factors predicting disease progression remain unknown. Methods: We collected information on clinical characteristics, exposure history, and laboratory examinations of all laboratory-confirmed COVID-19 patients admitted to PLA General Hospital. Cox regression analysis was applied to identify prognostic factors. The last follow-up was February 18, 2020. Results: We characterized 55 consecutive COVID-19 patients. The mean incubation was 8.42 (95% confidence interval [CI], 6.55-10.29) days. The mean SARS-CoV-2-positive duration from first positive test to conversion was 9.71 (95%CI, 8.21-11.22) days. COVID-19 course was approximately 2 weeks. Asymptomatic carriers might transmit SARS-CoV-2. Compared to patients without pneumonia, those with pneumonia were 15 years older and had a higher rate of hypertension, higher frequencies of having a fever and cough, and higher levels of interleukin-6 (14.61 vs. 8.06pg/mL, P=0.040), B lymphocyte proportion (13.0% vs.10.0%, P=0.024), low account (<190/µL) of CD8 + T cells (33.3% vs. 0, P=0.019). Multivariate Cox regression analysis indicated that circulating interleukin-6 and lactate independently predicted COVID-19 progression, with a hazard ratio (95%CI) of 1.052 (1.000-1.107) and 1.082 (1.013-1.155), respectively. During disease course, T lymphocytes were . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': '24ff82a421f1a33db3ebaa22652bd62e0cb77c57',\n","  'section': 'Abstract',\n","  'text': 'Backgrounds: An ongoing outbreak of a novel coronavirus (2019-nCoV) pneumonia hit a major city of China, Wuhan, December 2019 and subsequently reached other provinces/regions of China and countries. We present estimates of the basic reproduction number, R0, of 2019-nCoV in the early phase of the outbreak. Accounting for the impact of the variations in disease reporting rate, we modelled the epidemic curve of 2019-nCoV cases time series, in mainland China from January 10 to January 24, 2020, through the exponential growth. With the estimated intrinsic growth rate (γ), we estimated R0 by using the serial intervals (SI) of two other well-known coronavirus diseases, MERS and SARS, as approximations for the true unknown SI. The early outbreak data largely follows the exponential growth. We estimated that the mean R0 ranges from 2.24 (95%CI: 1.96-2.55) to 3.58 (95%CI: 2.89-4.39) associated with 8-fold to 2-fold increase in the reporting rate. We demonstrated that changes in reporting rate substantially affect estimates of R0. '},\n"," {'paper_id': '2536b83acf84368d7c13be81fe07aa0575115da7',\n","  'section': 'Abstract',\n","  'text': 'The outbreak of novel coronavirus has the potential for global spread, infecting large numbers in all countries. In this case, estimating the country-specific basic reproductive ratio is a vital first step in public-health planning. The basic reproductive ratio (R0) is determined by both the nature of pathogen and the network of contacts through which the disease can spread -with this network determined by socio-demographics including age-structure and household composition. Here we focus on the age-structured transmission within the population, using data from China to inform agedependent susceptibility and synthetic age-mixing matrices to inform the contact network. This allows us to determine the country-specific basic reproductive ratio as a multiplicative scaling of the value from China. We predict that R0 will be highest across Eastern Europe and Japan, and lowest across Africa, Central America and South-Western Asia. This pattern is largely driven by the ratio of children to older adults in each country and the observed propensity of clinical cases in the elderly. '},\n"," {'paper_id': '25c7476e1d441463f442ce3f5c157ee699ddc70a',\n","  'section': 'Abstract',\n","  'text': 'The burgeoning epidemic caused by novel coronavirus 2019 (2019-nCoV) is currently a global concern. Angiotensin-converting enzyme-2 (ACE2) is a receptor of 2019-nCoV spike 1 protein (S1) and mediates viral entry into host cells. Despite the abundance of ACE2 in small intestine, few digestive symptoms are observed in patients infected by 2019-nCoV. Herein, we investigated the interactions between ACE2 and human defensins (HDs) specifically secreted by intestinal Paneth cells. The lectin-like HD5, rather than HD6, bound ACE2 with a high affinity of 39.3 nM and weakened the subsequent recruitment of 2019-nCoV S1. The author/funder. All rights reserved. No reuse allowed without permission. : bioRxiv preprint cloak of HD5 on the ligand-binding domain of ACE2 was confirmed by molecular dynamic simulation. A remarkable dose-dependent preventive effect of HD5 on 2019-nCoV S1 binding to intestinal epithelial cells was further evidenced by in vitro experiments. Our findings unmasked the innate defense function of lectin-like intestinal defensin against 2019-nCoV, which may provide new insights into the prevention and treatment of 2019-nCoV infection. (2018YQYLY004). We have no competing financial interests to declare. References The species Severe acute respiratory syndrome-related coronavirus: classifying 2019-nCoV and naming it SARS-CoV-2. Nature microbiology, '},\n"," {'paper_id': '2634ab4b2f262ef28c5cdb34759ff3cae6c6187f',\n","  'section': 'Abstract',\n","  'text': 'SARS-CoV-2 emerged by the end of 2019 to rapidly spread in 2020. At present, it is of utmost 31 importance to understand the virus biology and to rapidly assess the potential of existing drugs 32 and develop new active compounds. While some animal models for such studies are under 33 development, most of the research is carried out in the Vero E6 cells. Here, we propose fully 34 differentiated human airway epithelium cultures as a model for studies on the SARS-CoV-2. 35 Further, we also provide basic characteristics of the system. 36 37 '},\n"," {'paper_id': '264a816e7099246c13a5dad3a581a474d85d50bd',\n","  'section': 'Abstract',\n","  'text': 'medRxiv preprint BACKGROUND SARS-CoV-2 has caused a series of COVID-19 globally. SARS-CoV-2 binds angiotensin I converting enzyme 2 (ACE2) of renin-angiotensin system (RAS) and causes prevalent hypokalemia The patients with COVID-19 were classified into severe hypokalemia, hypokalemia, and normokalemia group. The study aimed to determine the relationship between hypokalemia and clinical features, the underlying causes and clinical implications of hypokalemia. RESULTS Feb 15, 2020, 175 patients with COVID-19 (92 women and 83 men; median age, 46 [IQR, 34-54] years) were admitted to hospital in Wenzhou, China, consisting 39 severe hypokalemia-, 69 hypokalemia-, and 67 normokalemia patients. Gastrointestinal symptoms were not associated with hypokalemia among 108 hypokalemia patients (P>0.05). Body temperature, CK, CK-MB, LDH, and CRP were significantly associated with the severity of hypokalemia (P<0.01). 93% of severe and critically ill patients had hypokalemia which was most common among elevated CK, CK-MB, LDH, and CRP. Urine K + loss was the primary cause of hypokalemia. 1 severe hypokalemia patients was given 3 g/day, adding up to an average of 34 (SD=4) g potassium during hospital stay. The exciting finding was that patients responded well to K + supplements when they were inclined to recovery. '},\n"," {'paper_id': '265508c0e1c727e1c467f7b87305a7190162ea52',\n","  'section': 'Abstract',\n","  'text': 'China has led a rapid global spread around the world. For controlling COVID-19 outbreaks, many countries have implemented two non-pharmaceutical interventions: suppression like immediate lockdowns in cities at epicentre of outbreak; or mitigation that slows down but not stopping epidemic for reducing peak healthcare demand. Both interventions have apparent pros and cons; the effectiveness of any one intervention in isolation is limited. We aimed to conduct a feasibility study for robustly estimating the number and distribution of infections, growth of deaths, peaks and lengths of COVID-19 breakouts by taking multiple pharmaceutical interventions in London and the UK, accounting for reduction of healthcare demand. impacted on the number and distribution of infections, growth of deaths over time during January 2020, and April 2020. We combined the calibrated model with data on the cases of COVID-19 in London, the UK (non-London) and the UK during February 2020 and March 2020 to estimate the number and distribution of infections, growth of deaths, and healthcare demand by using multiple interventions. Findings We estimated given that multiple interventions with an intensity range from 3 to 15, one optimal strategy was to take suppression with intensity 3 in London from 23 rd March for 100 days, and 3 weeks rolling intervention with intensity between 3 and 5 in non-London regions. In this scenario, the total infections and deaths in the UK were limited to 2.43 million and 33.8 thousand; the peak time of healthcare demand was due to the 65 th day (April 11 th ), where it needs hospital beds for 25.3 thousand severe and critical cases. If we took a simultaneous 3 weeks rolling intervention with intensity between 3 and 5 in all regions of the UK, the total infections and deaths increased slightly to 2.69 million and 37 thousand; the peak time of healthcare kept the same at the 65 th day, where it needs equivalent hospital beds for severe and critical cases of 25.3 thousand. But if we released high band of rolling intervention intensity to 6 or 8 and simultaneously implemented them in all regions of the UK, the COVID-19 outbreak would not end in 1 year and distribute a multi-modal mode, where the total infections and deaths in the UK possibly reached to 16.2 million and 257 thousand. Interpretation: Our results show that taking rolling intervention is probably an optimal strategy to effectively and efficiently control COVID-19 outbreaks in the UK. As large difference of population density and social distancing between London and non-London regions in the UK, it is more appropriate to implement consistent suppression in London for 100 days and rolling intervention in other regions. This strategy would potentially reduce the overall infections and deaths, and delay and reduce peak healthcare demand. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '26cb6703ca72bf97887abbc29a40b1bd9d7890f4',\n","  'section': 'Abstract',\n","  'text': 'Fibrolamellar carcinoma (FLC) is a rare, therapeutically intractable liver cancer that disproportionately affects youth. Although FLC tumors exhibit a distinct gene expression profile, the causative transcriptional mechanisms remain unclear. Here we used chromatin run-on sequencing to discover approximately 7,000 enhancers and 141 enhancer hotspots activated in FLC relative to non-malignant liver. Detailed bioinformatic analyses revealed aberrant ERK/MEK signaling and candidate master transcriptional regulators. We also defined the genes most strongly associated with aberrant FLC enhancer activity, including CA12 and SLC16A14. Treatment of FLC cell models with inhibitors of CA12 or SLC16A14 independently reduced cell viability and/or significantly enhanced the effect of MEK inhibitor cobimetinib. These findings highlight new molecular targets for drug development as well as novel drug combination approaches. '},\n"," {'paper_id': '26cc3f38f54d8bb124682d382d7b5ffa4df0a257',\n","  'section': 'Abstract',\n","  'text': 'Qing Mao (Phone +86 135 9418 0020; Abstract: An excessive immune response contributes to SARS-CoV, MERS-CoV and SARS-CoV-2 pathogenesis and lethality, but the mechanism remains unclear. In this study, the N proteins of SARS-CoV, MERS-CoV and SARS-CoV-2 were found to bind to MASP-2, the key serine protease in the lectin pathway of complement activation, resulting in aberrant complement activation and aggravated inflammatory lung injury. Either blocking the N protein:MASP-2 5 interaction or suppressing complement activation can significantly alleviate N protein-induced complement hyper-activation and lung injury in vitro and in vivo. Complement hyper-activation was also observed in COVID-19 patients, and a promising suppressive effect was observed when the deteriorating patients were treated with anti-C5a monoclonal antibody. Complement suppression may represent a common therapeutic approach for pneumonia induced by these 10 highly pathogenic coronaviruses. Short Title: SARS-CoV N over-activates complement by MASP-2 One Sentence Summary: The lectin pathway of complement activation is a promising target for 15 the treatment of highly pathogenic coronavirus induced pneumonia. All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. : medRxiv preprint Previously, we demonstrated that the SARS-CoV N protein interacts with a number of host proteins, including MAP19 (21), an alternative splicing product of MASP-2. Thus, in this 40 study, the interactions of the SARS-CoV, MERS-CoV and SARS-CoV-2 N proteins with MASP-2 were intensively investigated, and the pathological effects of these interactions on host immunity and inflammation were also elucidated, which provides a mechanism support for immunomodulation-based therapy against current COVID-19 epidemic. 45 All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '26e75d3c815aae7fd9b094c3e5c74d3f7132ca13',\n","  'section': 'Abstract',\n","  'text': 'BACKGROUND Person to person spread of COIVD-19 in the UK has now been confirmed. There are limited case series reporting the impact on women affected by coronaviruses (CoV) during pregnancy. In women affected by SARS and MERS, the case fatality rate appeared higher in women affected in pregnancy compared with non-pregnant women. We conducted a rapid, review to guide management of women affected by COVID -19 during pregnancy and developed interim practice guidance with the RCOG and RCPCH to inform maternity and neonatal service planning METHODS Searches were conducted in PubMed and MedRxiv to identify primary case reports, case series, observational studies or randomised-controlled trial describing women affected by coronavirus in pregnancy and on neonates. Data was extracted from relevant papers and the review was drafted with representatives of the RCPCH and RCOG who also provided expert consensus on areas where data were lacking RESULTS From 9964 results on PubMed and 600 on MedRxiv, 18 relevant studies (case reports and case series) were identified. There was inconsistent reporting of maternal, perinatal and neonatal outcomes across case reports and series concerning COVID-19, SARS, MERS and other coronaviruses. From reports of 19 women to date affected by COVID-19 in pregnancy, delivering 20 babies, 3 (16%) were asymptomatic, 1 (5%) was admitted to ICU and no maternal deaths have been reported. Deliveries were 17 by caesarean section, 2 by vaginal delivery, 8 (42%) delivered pre-term. There was one neonatal death, in 15 babies who were tested there was no evidence of vertical transmission. CONCLUSIONS Morbidity and mortality from COVID-19 appears less marked than for SARS and MERS, acknowledging the limited number of cases reported to date. Pre-term delivery affected 42% of women hospitalised with COVID-19, which may put considerable pressure on neonatal services if the UK reasonable worse-case scenario of 80% of the population affected is realised. There has been no evidence of vertical transmission to date. The RCOG and RCPCH have provided interim guidance to help maternity and neonatal services plan their response to COVID-19. BACKGROUND The common human coronaviruses; 1. 229E (alpha coronavirus), 2. NL63 (alpha coronavirus), 3. OC43 (beta coronavirus) and 4. HKU1 (beta coronavirus) cause the common cold. Three human coronaviruses cause more severe, acute illnesses; MERS-CoV causes Middle East Respiratory Syndrome, MERS, SARS-CoV causes severe acute respiratory syndrome, or SARS and SARS-CoV-2 coronavirus that causes COVID-19. There are limited case series reporting the impact on women affected by coronaviruses (CoV) during pregnancy. In women affected by SARS and MERS, the case fatality rate appeared higher in women affected in pregnancy compared with non-pregnant women. '},\n"," {'paper_id': '26f1c42474966b8daede2912e06175bd8b864f8e',\n","  'section': 'Abstract',\n","  'text': 'The development of molecular syndrome-based kits for the diagnosis of respiratory infections 18 offers rapid and sensitive detection of common respiratory pathogens and will have a 19 significant impact on the care of patients. In this study, we present the results obtained after 20 the introduction of the FilmArray respiratory panel in a field Point of care (POC) for the 21 diagnosis of virus and bacteria responsible of respiratory tract infections in Senegal rural area. 22 36 37 '},\n"," {'paper_id': '26f55674badfa21611298e1271eaa2cafdf0ce22',\n","  'section': 'Abstract',\n","  'text': 'Background: SARS-CoV-2 is the cause of the COVID-19 that has been declared a global pandemic by the WHO in 2020. The COVID-19 treatment guidelines vary in each country, and yet there is no approved therapeutic for COVID-19. were included. The search terms included combinations of: COVID, SARS-COV-2, glucocorticoids, convalescent plasma, antiviral, antibacterial. There were no restrictions on the type of study design eligible for inclusion. As of March 26, 2020, of the initial manuscripts identified (n=449) articles. Forty-one studies were included, of which clinical trials (n=3), (case reports n=7), case series (n=10), retrospective (n=11) and prospective (n=10) observational studies. Thirtysix studies were conducted in China (88%). The most common mentioned and reported medicine in this systematic review was corticosteroids (n=25), followed by Lopinavir (n=21) and oseltamivir (n=16). '},\n"," {'paper_id': '26faf026cf5c77945422b8a4664ae8200781e017',\n","  'section': 'Abstract',\n","  'text': 'Managing healthcare demand and capacity is especially difficult in the context of the COVID-19 pandemic, where limited intensive care resources can be overwhelmed by a large number of cases requiring admission in a short space of time. If patients are unable to access this specialist resource, then death is a likely outcome. The aim of this study is to estimate the extent to which such capacity-dependent deaths can be mitigated through demand-side initiatives involving nonpharmaceutical interventions and supply-side measures to increase surge capacity or reduce length of stay. A stochastic discrete event simulation model is developed to represent the key dynamics of the intensive care admissions process for COVID-19 patients. Model inputs are aligned to levers available to planners with key outputs including duration of time at maximum capacity (to inform workforce requirements), peak daily deaths (for mortuary planning), and total deaths (as an ultimate marker of intervention efficacy). The model -freely available -is applied to the COVID-19 response at a large hospital in England for which the effect of a number of possible interventions are simulated. Results. Capacity-dependent deaths are closely associated with both the nature and effectiveness of non-pharmaceutical interventions and availability of intensive care beds. For the hospital considered, results suggest that capacity-dependent deaths can be reduced five-fold through a combination of isolation policies, a doubling of bed capacity, and 25% reduced length of stay. Without treatment or vaccination there is little that can be done to reduce deaths occurring when patients have otherwise been treated in the most appropriate hospital setting. Healthcare planners should therefore focus on minimising the capacity-dependent deaths that are within their influence. '},\n"," {'paper_id': '272aa208a9e8ae2f2e0b4b2a74ec6ef0c53014bb',\n","  'section': 'Abstract',\n","  'text': 'A compounded method -exploiting the searching capabilities of an operation research algorithm and the power of bootstrap techniques -is presented. The resulting algorithm has been successfully tested to predict the turning point reached by the epidemic curve followed by the CoViD-19 virus in Italy. Futures lines of research, which include the generalization of the method to a broad set of distribution, will be finally given. '},\n"," {'paper_id': '2757708c4f72260abefb9d74302d868165cbc1a4',\n","  'section': 'Abstract',\n","  'text': 'bioRxiv preprint 42 findings also provide an application of human organoids in investigating the 43 tropism and pathogenesis of SARS-CoV-2, which would facilitate novel drug 44 discovery. '},\n"," {'paper_id': '278e2c52630f1dbbda6a79db15a2260510586150',\n","  'section': 'Abstract',\n","  'text': 'The simultaneous consideration of sequence alignment and RNA secondary structure, or structural alignment, is known to help predict more accurate secondary structures of homologs. However, the consideration is heavy and can be done only roughly to decompose structural alignments. '},\n"," {'paper_id': '2793131bcb54871487ad7183e6b19211b6d5b64f',\n","  'section': 'Abstract',\n","  'text': 'word count: 248 Manuscript word count: 3103 Abstract Background and Aim: Since late 2019, an unknown-origin pneumonia outbreak detected in Wuhan city, Hubei Province, China. We aimed to build a model to qualitatively and quantitatively assess publications of research of COVID-19 from 2019 to 2020. Materials and Methods: Data were obtained from the Web of Science (WOS), PubMed, and Scopus Core Collection on March 02, 2020, and updated on March 10. We conducted a qualitative and quantitative analysis of publication outputs, journals, authors, institutions, countries, cited references, keywords, and terms according to bibliometric methods using VOS viewer c software packages. Results: Initially, we identified 227 papers, of which after an exclusion process, 92 studies were selected for statistical analyses. China accounted for the highest proportion of published research (44 papers, 40.48%), followed by the United States (21 papers, 19.32%), and Canada (7 papers, 6.44%). Adjusted by gross domestic product (GDP), ranked first, with 0.003 articles per billion GDP. In total, the top 10 journals published 47 articles, which accounted for 51.08% of all publications in this Feld. A total of 6 studies (05.52%) were supported by National Natural Science Foundation of China. Chinese Academy of Sciences ranked second 2, 2.76%). Bibliometric and visualized mapping may quantitatively monitor research performance in science and present predictions. The subject of this study was the fast *Abstract All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. Abstract 3 '},\n"," {'paper_id': '28750ae423bceec7befb82f2b857f9d2ce989e90',\n","  'section': 'Abstract',\n","  'text': 'Reporting delays in disease surveillance impair the ability to assess the current dynamic of an epidemic. In continuously updated epidemic curves, case numbers for the most recent epidemic week or day usually appear to be lower than the previous, suggesting a decline of the epidemic. In reality, the epidemic curve may still be on the rise, because reporting delay prevents the most recent cases to appear in the case count. In context of the COVID-19 epidemic and for countries planning large international gatherings, such as the Summer Olympic Games in Japan 2020, the ability to assess the actual stage of an epidemic is of outmost importance. We applied now-casting onto COVID-19 data provided by the nCoV-2019 Data Working Group to evaluate the true count of cases, by taking into account reporting delays occurring between date of symptom onset and date of confirmation. We calculated a decrease of reporting delay, from a median delay of ten days in calendar week four 2020 to six days in calendar week eight, resulting in an overall mean of 4.3 days. The confidence intervals of the now-casting indicated an increase of cases in the last reporting days, while case country in that same time period suggested a decline. As a specific use case this tool may be of particular value for the challenging risk assessment and risk communication in the context of the Summer Olympic Games in Japan 2020 and similar situations elsewhere. '},\n"," {'paper_id': '28884786bfd579216f563c278e1730290dd4ccb8',\n","  'section': 'Abstract',\n","  'text': \"SARS-CoV-2, a member of the coronavirus family, has caused a global public health emergency. 1 Based on our analysis of hepatitis C virus and coronavirus replication, and the molecular structures and activities of viral inhibitors, we previously reasoned that the FDA-approved heptatitis C drug EPCLUSA (Sofosbuvir/Velpatasvir) should inhibit coronaviruses, including SARS-CoV-2. 2 Here, using model polymerase extension experiments, we demonstrate that the activated triphosphate form of Sofosbuvir is incorporated by low-fidelity polymerases and SARS-CoV RNA-dependent RNA polymerase (RdRp), and blocks further incorporation by these polymerases; the activated triphosphate form of Sofosbuvir is not incorporated by a host-like high-fidelity DNA polymerase. Using the same molecular insight, we selected two other anti-viral agents, Alovudine and AZT (an FDA approved HIV/AIDS drug) for evaluation as inhibitors of SARS-CoV RdRp. We demonstrate the ability of two HIV reverse transcriptase inhibitors, 3'fluoro-3'-deoxythymidine triphosphate and 3'-azido-3'-deoxythymidine triphosphate (the active triphosphate forms of Alovudine and AZT), to be incorporated by SARS-CoV RdRp where they also terminate further polymerase extension. Given the 98% amino acid similarity of the SARS-CoV and SARS-CoV-2 RdRps, we expect these nucleotide analogues would also inhibit the SARS-CoV-2 polymerase. These results offer guidance to further modify these nucleotide analogues to generate more potent broad-spectrum anticoronavirus agents. The recent appearance of a new coronaviral infection, COVID-19, in Wuhan, China, and its worldwide spread, has made international headlines. Already more than 3,000 deaths have been ascribed to this virus, and COVID-19 has reached near pandemic status. The virus has been isolated from the lower respiratory tracts of patients with pneumonia, sequenced and visualized by electron microscopy. 1 The virus, designated SARS-CoV-2, is a new member of the subgenus Sarbecovirus, in the Orthocoronavirinae subfamily, but is distinct from MERS-CoV and SARS-CoV. 1 The coronaviruses are single strand RNA viruses, sharing properties with other single-stranded RNA viruses such as hepatitis C virus (HCV), West Nile virus, Marburg virus, HIV virus, Ebola virus, dengue virus, and rhinoviruses. In particular, coronaviruses and HCV are both positive-sense single-strand RNA viruses, 3,4 and thus have a similar replication mechanism requiring a RNA-dependent RNA polymerase (RdRp). The coronavirus life cycle has been described. 3 Briefly, the virus enters the cell by endocytosis, is uncoated, and ORF1a and ORF1b of the positive strand RNA is translated to produce nonstructural protein precursors, including a cysteine protease and a serine protease; these further cleave the precursors to form mature, functional helicase and RNA-dependent RNA polymerase. A replication-transcription complex is then . CC-BY-NC-ND 4.0 : bioRxiv preprint sequence of the primer and template used for this extension reaction is shown at the top of the figure. Polymerase extension reactions were performed by incubating UTP with pre-assembled SARS-CoV polymerase (nsp12, nsp7 and nsp8), the indicated RNA template and primer, and the appropriate reaction buffer, followed by detection of reaction products by MALDI-TOF MS. The detailed procedure is shown in the methods. The accuracy for m/z determination is ± 10 Da. \"},\n"," {'paper_id': '290aeb17cb7f2686bf926870f6bf06f9d853306c',\n","  'section': 'Abstract',\n","  'text': 'There is little information about the coronavirus disease 2019 during pregnancy. This study aimed to determine the clinical features and the maternal and neonatal outcomes of pregnant women with Covid-19. In this retrospective analysis from five hospitals, we included pregnant women with Covid-19 from January 1 to February 20, 2020. The primary composite endpoints were admission to an intensive care unit (ICU), the use of mechanical ventilation, or death. Secondary endpoints included the clinical severity of Covid-19, neonatal mortality, admission to neonatal intensive care unit (NICU), and the incidence of acute respiratory distress syndrome (ARDS) of pregnant women and newborns. Thirty-three pregnant women with Covid-19 and 28 newborns were identified. One (3%) pregnant woman needed the use of mechanical ventilation. No pregnant women admitted to the ICU. There were no moralities among pregnant women or newborns. The percentages of pregnant women with mild, moderate, and severe symptoms were 13 (39.4%),19(57.6%), and 1(3%). One (3.6%) newborn developed ARDS and was admitted to the NICU. The rate of perinatal transmission of SARS-CoV-2 was 3.6%. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. : medRxiv preprint This report suggests that pregnant women are not at increased risk for severe illness or mortality with Covid-19 compared with the general population. The SARS-CoV-2 infection during pregnancy might not be associated with as adverse obstetrical and neonatal outcomes that are seen with the severe acute respiratory syndrome coronavirus (SARS-CoV) and Middle East respiratory syndrome coronavirus (MERS-CoV) infection during pregnancy. (Funded by the National Key Research and Development Program.) All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '2913d91f13fd59c698f68ba63008d8e0550c0607',\n","  'section': 'Abstract',\n","  'text': 'As of February 11, 2020, all prefecture-level cities in mainland China have reported confirmed cases of 2019 novel coronavirus (2019-nCoV), but the city-level epidemical dynamics is unknown. The aim of this study is to model the current dynamics of 2019-nCoV at city level and predict the trend in the next 30 days under three possible scenarios in mainland China. We developed a spatially explicit epidemic model to consider the unique characteristics of the virus transmission in individual cities. Our model considered that the rate of virus transmission among local residents is different from those with Wuhan travel history due to the self-isolation policy. We introduced a decay rate to quantify the effort of each city to gradually control the disease spreading. We used mobile phone data to obtain the number of individuals in each city who have travel history to Wuhan. This city-level model was '},\n"," {'paper_id': '292a277ed4963d0148fa3f4f2be6146401f526ae',\n","  'section': 'Abstract',\n","  'text': 'Article Summary Line: The 21 day lockdown in India to slow the COVID 19 epidemic in the 22 country has still not shown significant results, but the earlier interventions done are showing 23 their effect now. Running title: In-silico analysis of SARS-CoV-2 epidemic in India Abstract : After SARS-CoV-2 set foot in India, the Indian Government took a number of steps to limit the 37 spread of the disease in the country. This study involves assessing how the disease affected the 38 population in the initial days of the epidemic. Data was collected from government controlled 39 and crowdsourced websites and then put through analysis and calculations. With a study on age 40 and sex parameters of 413 patients, the median age of the affected individuals was found out to 41 be 36 years (IQR 25-54 years) with 20-39 years males being the most affected group. The 42 number of affected males (66.34%) was more than that of the females(33.66%).Using SIR 43 model, the range of contact rate(β) of India was calculated and the role of public health 44 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': '295036c0448244642eb6533de571def6cd8437e5',\n","  'section': 'Abstract',\n","  'text': 'Mutation rates can evolve through genetic drift, indirect selection due to genetic hitchhiking, or 28 direct selection on the physicochemical cost of high fidelity. However, for many systems, it has 29 been difficult to disentangle the relative impact of these forces empirically. In RNA viruses, an 30 observed correlation between mutation rate and virulence has led many to argue that their 31 extremely high mutation rates are advantageous, because they may allow for increased 32 adaptability. This argument has profound implications, as it suggests that pathogenesis in many 33 viral infections depends on rare or de novo mutations. Here we present data for an alternative 34 model whereby RNA viruses evolve high mutation rates as a byproduct of selection for 35 increased replicative speed. We find that a poliovirus antimutator, 3D G64S , has a significant 36 replication defect and that wild type and 3D G64S populations have similar adaptability in two 37 distinct cellular environments. Experimental evolution of 3D G64S under r-selection led to 38 reversion and compensation of the fidelity phenotype. Mice infected with 3D G64S exhibited 39 delayed morbidity at doses well above the LD 50 , consistent with attenuation by slower growth as 40 opposed to reduced mutational supply. Furthermore, compensation of the 3D G64S growth defect 41 restored virulence, while compensation of the fidelity phenotype did not. Our data are consistent 42 with the kinetic proofreading model for biosynthetic reactions and suggest that speed is more 43 important than accuracy. In contrast to what has been suggested for many RNA viruses, we find 44 that within host spread is associated with viral replicative speed and not standing genetic 45 diversity. 46 47 3 mutation rates. Since most mutations are deleterious and mutation rates are tuned by natural 53 selection, we asked why hasn\\'t the virus evolved to have a lower mutation rate? We used 54 experimental evolution and a murine infection model to show that RNA virus mutation rates may 55 actually be too high and are not necessarily adaptive. Rather, our data indicate that viral 56 mutation rates are driven higher as a result of selection for viruses with faster replication kinetics. We suggest that viruses have high mutation rates, not because they facilitate adaption, but 58 because it is hard to be both fast and accurate. 59 60 Introduction 61 Mutation is the ultimate source of genetic variation, and mutation rates can have a significant 62 impact on evolutionary rate [1-3]. The intraspecific variability in mutation rate in many viruses 63 and bacteria indicates that mutation rates have been optimized by natural selection [4-13]. Given that most mutations are deleterious, the burden of excess mutational load will select 65 against strains with abnormally high mutation rates [14][15][16][17]. This principle led to Sturtevant to 66 ask, \"Why does the mutation rate not evolve to zero?\" [18,19]. A large body of theoretical and experimental work suggests that the selective pressure for 69 higher mutation rates is due to either the physicochemical cost of maintaining a lower one or a 70 selective advantage from an increased supply of beneficial mutations [20-23]. Many have 71 argued for the adaptive benefit of high mutation rates in pathogenic microbes, which often exist 72 in dynamic environments and are subject to host immune pressure [7,24,25]. However, direct 73 selection of a variant with a higher mutation rate will only occur if it has been advantageous in 74 the past, and in many cases, it has been difficult to separate the causes of a higher mutation 75 rate from its consequences [19,26]. 76 77 . CC-BY 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/309880 doi: bioRxiv preprint 4 RNA viruses are ideal systems for studying the selective forces that act on mutation rates. While 78 interspecific mutation rates range from 10 -4 to 10 -6 errors per nucleotide copied [4], studies of 79 antimutators and hypermutators suggests that fidelity can only vary by several fold within a 80 species [27]. The severe burden of mutational load exerts a strong downward pressure on 81 mutation rates and hypermutator strains are attenuated in vivo [9,28-31]. Given the short 82 generation times and remarkable fecundity of many RNA viruses, a small kinetic cost to higher 83 fidelity should result in strong selection against antimutators [32,33]. However, the observed 84 attenuation of antimutator RNA viruses in vivo has led many to argue for the adaptive benefit of 85 high mutation rates, as genetic diversity provides a rich substrate for a virus\\' evolution in the 86 face of varying intrahost environments [7,10,34-38]. This concept is central to viral quasispecies 87 theory, which generally proposes a link between genetic diversity and viral fitness [24,25]. Here, we define the selective forces that shape viral mutation rates by studying an antimutator 90 variant. The 3D G64S mutant of poliovirus was selected after serial passage in ribavirin, an RNA 91 virus mutagen. The RNA dependent RNA polymerase (RdRp, 3D) of this variant contains a 92 single glycine to serine substitution [5-7]. The basal mutation rate of 3D G64S is reported to be 93 ~20-30% that of wild type virus (WT). While the 3D G64S mutant is attenuated in poliovirus 94 receptor transgenic mice, the relative importance of replicative speed and fidelity to this 95 phenotype is not clear [7,36]. Biochemical assays of 3D G64S suggest a physicochemical cost of 96 high fidelity, but as in other systems, its contribution to overall fitness remains unquantified 97 [6,19,39]. 99 Results We measured the relative fitness of 3D G64S by direct competition over serial passage by RT-101 qPCR (Fig. 1A) . Here, the fitness of 3D G64S is 0.78 ± 0.01 (n=3 replicates) relative to WT. This is 102 a moderate fitness defect, falling in the 64 th percentile in a dataset of 8970 fitness values 103 . CC-BY 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/309880 doi: bioRxiv preprint 5 obtained for point mutants of poliovirus under similar conditions [16] (e.g. HeLa cells, moi 0.1, 8 104 hour infection cycle, and 6 passages; Fig. 1B ). We also measured the relative growth properties 105 of WT and 3D G64S using a plaque size assay, which measures the growth, burst size, and 106 spread of individual viruses in the absence of competition [40][41][42]. The distribution of clonal 107 plaque sizes was significantly different (p<0.005, unpaired t-test with Welch correction, n=272 108 WT and n=220 3D G64S plaques), and consistent with a moderate fitness defect in 3D G64S (Fig.   109 1C). In contrast to prior work, we were able to detect a significant replication defect for 3D G64S by 110 one step growth curve, but only with rigorous synchronization, more frequent time points, and 111 larger numbers of replicates (Fig. 1D ). This replication defect was not specific to HeLa, as we 112 observed a similar lag for 3D G64S in a 3T3 cell line that we derived from mouse embryonic 113 fibroblasts from poliovirus receptor transgenic mice (PVR-3T3, Fig. 1E ). These data 114 demonstrate that the fitness defect of 3D G64S is largely attributable to its slower replicative 115 kinetics and is consistent with biochemical assays on purified RdRp [6,39]. The reduced mutation rate and replicative fitness of 3D G64S suggest a trade-off between speed 118 and fidelity in RNA virus replication. Here, the fitness gain from increased replicative speed is 119 offset by a reduction in fitness due to increased mutational load. We derived a quantitative 120 model of this trade-off (see SI Model 1) by measuring the replicative fitness (Fig. 1F ) and 121 mutation rate (Fig. 1G, Table S1 ) of WT and 3D G64S under exposure to an exogenous mutagen, 122 ribavirin [43]. Wild type and 3D G64S had equal fitness at approximately 150µM ribavirin. Based 123 on these data, our model indicates that WT incurs a fitness cost of 0.137 from mutational load 124 alone. Therefore, any fitness benefit of the high baseline mutation rates in WT would 125 presumably need to offset this cost. In 3D G64S , the cost of mutational load is reduced to 0.037. If viral RdRp are constrained by a speed-fidelity trade-off, selection for increased replicative 128 speed (r-selection) will increase mutation rate. We subjected the 3D G64S-1nt point mutant '},\n"," {'paper_id': '297f36eeb998b4e8729a0cbdf958671e9909edc1',\n","  'section': 'Abstract',\n","  'text': \"SARS-CoV-2, a member of the coronavirus family, is responsible for the current COVID-19 pandemic. Based on our analysis of hepatitis C virus and coronavirus replication, and the molecular structures and activities of viral inhibitors, we previously demonstrated that three nucleotide analogues inhibit the SARS-CoV RNA-dependent RNA polymerase (RdRp). Here, using polymerase extension experiments, we have demonstrated that the active triphosphate form of Sofosbuvir (a key component of the FDA approved hepatitis C drug EPCLUSA), is incorporated by SARS-CoV-2 RdRp, and blocks further incorporation. Using the same molecular insight, we selected the active triphosphate forms of three other anti-viral agents, Alovudine, AZT (an FDA approved HIV/AIDS drug) and Tenofovir alafenamide (TAF, an FDA approved drug for HIV and hepatitis B) for evaluation as inhibitors of SARS-CoV-2 RdRp. We demonstrated the ability of these three viral polymerase inhibitors, 3'-fluoro-3'-deoxythymidine triphosphate, 3'-azido-3'deoxythymidine triphosphate and Tenofovir diphosphate (the active triphosphate forms of Alovudine, AZT and TAF, respectively) to be incorporated by SARS-CoV-2 RdRp, where they also terminate further polymerase extension. These results offer a strong molecular basis for these nucleotide analogues to be evaluated as potential therapeutics for COVID-19. \"},\n"," {'paper_id': '2989bf36068372cfedef82f58f4777e60ccbcfaa',\n","  'section': 'Abstract',\n","  'text': 'The ongoing outbreak of COVID-19 has spread rapidly and sparked global concern. While the transmission of SARS-CoV-2 through human respiratory droplets and contact with infected persons is clear, the aerosol transmission of SARS-CoV-2 has been little studied. Methods: Thirty-five aerosol samples of three different types (total suspended particle, size segregated and deposition aerosol) were collected in Patient Areas (PAA) and Medical Staff Areas (MSA) of Renmin Hospital of Wuhan University (Renmin) and Wuchang Fangcang Field Hospital (Fangcang), and Public Areas (PUA) in Wuhan, China during COVID-19 outbreak. A robust droplet digital polymerase chain reaction (ddPCR) method was employed to quantitate the viral SARS-CoV-2 RNA genome and determine aerosol RNA concentration. The ICU, CCU and general patient rooms inside Renmin, patient hall inside Fangcang had undetectable or low airborne SARS-CoV-2 concentration but deposition samples inside ICU and air sample in Fangcang patient toilet tested positive. The airborne SARS-CoV-2 in Fangcang MSA had bimodal distribution with higher concentration than those in Renmin during the outbreak but turned negative after patients number reduced and rigorous sanitization implemented. PUA had undetectable airborne SARS-CoV-2 concentration but obviously increased with accumulating crowd flow. Room ventilation, open space, proper use and disinfection of toilet can effectively limit aerosol transmission of SARS-CoV-2. Gathering of crowds with asymptomatic carriers is a potential source of airborne SARS-CoV-2. The virus aerosol deposition on protective apparel or floor surface and their subsequent resuspension is a potential transmission pathway and effective sanitization is critical in minimizing aerosol transmission of SARS-CoV-2. '},\n"," {'paper_id': '29bc34aad802ab6b35074f23c631540a84b16d0d',\n","  'section': 'Abstract',\n","  'text': 'Background The 2019 novel coronavirus infected pneumonia (COVID-19) represents a significant public health threat. The COVID-19 emerged in December 2009 in Wuhan, China and rapidly spread to other regions and countries. The variation in transmission patterns and disease spread in regard to time or among different locations, partially reflecting the public health intervention effects, remains to be quantified. As most transmissibility-related epidemic parameters are unknown, we sought, with minimal assumptions, to estimate real-time transmissibility and forecast new cases using dynamic modelling. Methods Using the cases reported from the National Health Commission of China and transportation data, including the total number of travelling hours through railway, airplane, and car outbound from Wuhan, we have built a time-series model to estimate real-time underlying transmission rates of newly generated cases sequentially from in Wuhan, Hubei province and other 28 provinces in China. We quantified the instantaneous transmission rate and relative reproduction number (R t ) of COVID-19, and evaluated whether public health intervention affected the case transmissibility in each province. Based on the current estimates, we have predicted the trend of disease spread with a high level of certainty. Findings We estimated that R t declined from the range of 4 to 5 towards '},\n"," {'paper_id': '2a21fdd15e07c89c88e8c2f6c6ab5692568876ec',\n","  'section': 'Abstract',\n","  'text': 'During the current COVID-19 pandemic, testing kit and RNA extraction kit availability has become a major limiting factor in the ability to determine patient disease status and accurately quantify prevalence. Current testing strategies rely on individual tests of cases matching restrictive diagnostic criteria to detect SARS-CoV-2 RNA, limiting testing of asymptomatic and mild cases. Testing these individuals is one effective way to understand and reduce the spread of COVID-19. Here, we develop a pooled testing strategy to identify these low-risk individuals. Drawing on the well-studied group testing literature, modeling suggests practical changes to testing protocols which can reduce test costs and stretch a limited test kit supply. When most tests are negative, pooling reduces the total number of tests up to four-fold at 2% prevalence and eight-fold at 0.5% prevalence. At current SARS-CoV-2 prevalence, randomized group testing optimized per country could double the number of tested individuals from 1.85M to 3.7M using only 671k more tests. This strategy is well-suited to supplement testing for asymptomatic and mild cases who would otherwise go untested, and enable them to adopt behavioral changes to slow the spread of COVID-19. '},\n"," {'paper_id': '2ae710bf5f60c23c489e4c61245a35a4f85178cb',\n","  'section': 'Abstract',\n","  'text': 'The detection of serum antibodies to the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is emerging as a new tool for the coronavirus disease-2019 diagnosis. Since many coronaviruses are sensitive to heat, heating inactivation of samples at 56 ℃ prior to testing is considered a possible method to reduce the risk of transmission, but the effect of heating on the measurement of SARS-CoV-2 antibodies is still unclear. By comparing the levels of SARS-CoV-2 antibodies before and after heat inactivation of serum at 56 ℃ for 30 minutes using a quantitative fluorescence immunochromatographic assay, we shown that heat inactivation significantly interferes with the levels of antibodies to SARS-CoV-2. The IgM levels of all the 34 serum samples (100%) from COVID-19 patients decreased by an average level of 53.56%. The IgG levels were decreased in 22 of 34 samples (64.71%) by an average level of 49.54%. Similar changes can also be observed in the non-COVID-19 diseases group (n=9). Of note, 44.12% of the detected IgM levels were dropped below the cut-off value after heating, suggesting heat inactivation can lead to false-negative results of these samples. Our results indicate . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . https: //doi.org/10.1101//doi.org/10. /2020 that heat inactivation of serum at 56 ℃ for 30 minutes interferes with the immunoanalysis of antibodies to SARS-CoV-2. Heat inactivation prior to immunoanalysis is not recommended and the possibility of false-negative results should be considered if the sample was pre-inactivated by heating. '},\n"," {'paper_id': '2af91bbb625d983d5ad3aadb9e5f1d82268a0f1f',\n","  'section': 'Abstract',\n","  'text': 'The ongoing outbreak of coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) started in the end of 2019 in China has triggered a global public health crisis. Previous studies have shown that SARS-CoV-2 infects cells by binding angiotensin-converting enzyme 2 (ACE2), which is the same as SARS-CoV. The expression and distribution of ACE2 in the pancreas are unknown. At the same time, the injury of pancreas after SARS-CoV-2 infection has not been concerned. Here, we collected public datasets (bulk RNA-seq and single-cell RNA-seq) to indicate the expression and the distribution of ACE2 in pancreas (in both exocrine glands and islets). And further, clinical data including mild and severe patients with COVID-19 demonstrated there existed mild pancreatitis. In the 67 severe cases, 11 patients (16.41%) showed elevated levels of both amylase and lipase, and 5 patients (7.46%) showed imaging alterations. Only one patient (1.85%) showed elevated levels of both amylase and lipase in 54 mild cases, without imaging changes. Our study revealed the phenomenon and possible cause of mild pancreatic injury in patients with COVID-19. This suggests that pancreatitis after SARS-CoV-2 infection should also be paid attention in clinical work. '},\n"," {'paper_id': '2afff5fe89a3e3ec270a41c2b929dfeec68d7d50',\n","  'section': 'Abstract',\n","  'text': 'The exported cases of 2019 novel coronavirus infection that were confirmed outside of China provide an opportunity to estimate the cumulative incidence and confirmed case fatality risk (cCFR) in mainland China. Knowledge of the cCFR is critical to characterize the severity and understand the pandemic potential of COVID-19 in the early stage of the epidemic. Using the exponential growth rate of the incidence, the present study statistically estimated the cCFR and the basic reproduction number-the average number of secondary cases generated by a single primary case in a naïve population. We modeled epidemic growth either from a single index case with illness onset on 8 December, 2019 (Scenario 1), or using the growth rate fitted along with the other parameters (Scenario 2) based on data from 20 exported cases reported by 24 January, 2020. The cumulative incidence in China by 24 January was estimated at 6924 cases (95% CI: 4885, 9211) and 19,289 cases (95% CI: 10,901, 30,158), respectively. The latest estimated values of the cCFR were 5.3% (95% CI: 3.5%, 7.5%) for Scenario 1 and 8.4% (95% CI: 5.3%, 12.3%) for Scenario 2. The basic reproduction number was estimated to be 2.1 (95% CI: 2.0, 2.2) and 3.2 (95% CI: 2.7, 3.7) for Scenarios 1 and 2, respectively. Based on these results, we argued that the current COVID-19 epidemic has a substantial potential for causing a pandemic. The proposed approach provides insights in early risk assessment using publicly available data. '},\n"," {'paper_id': '2b90a9a3d33a1babe12796a19a4be5ddf8348b39',\n","  'section': 'Abstract',\n","  'text': 'The 2019 novel coronavirus, SARS-CoV-2, is an emerging pathogen of critical significance to international public health. Knowledge of the interplay between molecular-scale virus-receptor interactions, single-cell viral replication, intracellular-scale viral transport, and emergent tissue-scale viral propagation is limited. Moreover, little is known about immune system-virus-tissue interactions and how these can result in low-level (asymptomatic) infections in some cases and acute respiratory distress syndrome (ARDS) in others, particularly with respect to presentation in different age groups or pre-existing inflammatory risk factors like diabetes. A critical question for treatment and protection is why it appears that the severity of infection may correlate with the initial level of virus exposure. Given the nonlinear interactions within and among each of these processes, multiscale simulation models can shed light on the emergent dynamics that lead to divergent outcomes, identify actionable \"choke points\" for pharmacologic interactions, screen potential therapies, and identify potential biomarkers that differentiate response dynamics. Given the complexity of the problem and the acute need for an actionable model to guide therapy discovery and optimization, we introduce a prototype of a multiscale model of SARS-CoV-2 dynamics in lung and intestinal tissue that will be iteratively refined. The first prototype model was built and shared internationally as open source code and interactive, cloud-hosted executables in under 12 hours. In a sustained community effort, this model will integrate data and expertise across virology, immunology, mathematical biology, quantitative systems physiology, cloud and high performance computing, and other domains to accelerate our response to this critical threat to international health. '},\n"," {'paper_id': '2bdc5655187ad0c6bb1e7a1820c142985d8a690b',\n","  'section': 'Abstract',\n","  'text': 'Background: The novel coronavirus disease 2019 (COVID-19) outbreak is spreading 15 rapidly throughout China and the world. Hence, early surveillance and public health 16 emergency disposal are considered crucial to curb this emerging infectious disease. '},\n"," {'paper_id': '2c006d09b6fccc527bf5ee3de0f165b018c39e73',\n","  'section': 'Abstract',\n","  'text': 'The fast and untraceable virus mutations take lives of thousands of people before the immune system can produce the inhibitory antibody. Recent outbreak of novel coronavirus infected and killed thousands of people in the world. Rapid methods in finding peptides or antibody sequences that can inhibit the viral epitopes of COVID-19 will save the life of thousands. In this paper, we devised a machine learning (ML) model to predict the possible inhibitory synthetic antibodies for Corona virus. We collected 1933 virus-antibody sequences and their clinical patient neutralization response and trained an ML model to predict the antibody response. Using graph featurization with variety of ML methods, we screened thousands of hypothetical antibody sequences and found 8 stable antibodies that potentially inhibit COVID-19. We combined bioinformatics, structural biology, and Molecular Dynamics (MD) simulations to verify the stability of the candidate antibodies that can inhibit the Corona virus. '},\n"," {'paper_id': '2c17a581c10ece6c5b5dec88f228f1f43d3fcc14',\n","  'section': 'Abstract',\n","  'text': 'There is interest in the use of chloroquine/hydroxychloroquine (CQ/HCQ) and azithromycin (AZT) in COVID-19 therapy. Employing cystic fibrosis respiratory epithelial cells, here we show that drugs AZT and ciprofloxacin (CPX) act as acidotropic lipophilic weak bases and confer in vitro effects on intracellular organelles similar to the effects of CQ. These seemingly disparate FDA-approved antimicrobials display a common property of modulating pH of endosomes and trans-Golgi network. We believe this may in part help understand the potentially beneficial effects of CQ/HCQ and AZT in COVID-19, and that the present considerations of HCQ and AZT for clinical trials should be extended to CPX. '},\n"," {'paper_id': '2c18d80cfc792faa3d6e35c72d4b6b58ee93fdad',\n","  'section': 'Abstract',\n","  'text': \"We have developed a generalizable 'smart molecular diagnostic' capable of accurate point-ofcare (POC) detection of variable nucleic acid targets. Our one-pot isothermal assay relies on multiplex execution of four loop-mediated isothermal amplification reactions, with primers that are degenerate and redundant, thereby increasing the breadth of targets while reducing the probability of amplification failure. An easy-to-read visual answer is computed directly by a multi-input Boolean OR gate signal transducer that uses degenerate strand exchange probes to assess any combination of amplicons. We demonstrate our platform by using the same assay to detect divergent Asian and African lineages of the evolving Zika virus (ZIKV), while maintaining selectivity against non-target viruses. Direct analysis of biological specimens proved possible, with 20 virions / µl being directly detected in human saliva within 90 minutes, and crudely macerated ZIKV-infected Aedes aegypti mosquitoes being identified with 100% specificity and sensitivity. The ease-of-use with minimal instrumentation, broad programmability, and built-in fail-safe reliability make our smart molecular diagnostic attractive for POC use. While point-of-care (POC) diagnostic assays can be performed at or near the site of sample acquisition, they have for the most part been considered to be relatively simplistic tests that provide relatively little information to a clinician or public health worker. Familiar examples include electrochemical sensors for glucose, 1 or rapid immunoassays for metabolites such as human chorionic gonadotropin (the canonical pregnancy test), 2 and pathogens, either directly (influenza viruses) or via immune responses (antibodies against HIV-1/2). 3 The range of conditions and pathogens that could be tested for could likely be greatly expanded by developing POC diagnostics for nucleic acids 3 that would have greater sensitivity and accuracy. However, the current gold standard for molecular diagnostics, the quantitative polymerase chain reaction (qPCR), requires significant technical expertise and expensive and cumbersome equipment. Even portable instruments, such the Cepheid GeneXpert Omni, cost several thousand dollars and rely on expensive qPCR cartridge consumables for individual tests. \"},\n"," {'paper_id': '2cc809ed4d5c3640ee2351fdb1876e8ff6c01b51',\n","  'section': 'Abstract',\n","  'text': 'To assess the viability of isolation and contact tracing to control onwards transmission from imported cases of 2019-nCoV. We developed a stochastic transmission model, parameterised to the 2019-nCoV outbreak. We used the model to quantify the potential effectiveness of contact tracing and isolation of cases at controlling a 2019 nCoV-like pathogen. We considered scenarios that varied in: the number of initial cases; the basic reproduction number R0; the delay from symptom onset to isolation; the probability contacts were traced; the proportion of transmission that occurred before symptom onset, and the proportion of subclinical infections. We assumed isolation prevented all further transmission in the model. Outbreaks were deemed controlled if transmission ended within 12 weeks or before 5000 cases in total. We measured the success of controlling outbreaks using isolation and contact tracing, and quantified the weekly maximum number of cases traced to measure feasibility of public health effort. Findings: While simulated outbreaks starting with only 5 initial cases, R0 of 1.5 and little transmission before symptom onset could be controlled even with low contact tracing probability, the prospects of controlling an outbreak dramatically dropped with the number of initial cases, with higher R0, and with more transmission before symptom onset. Across different initial numbers of cases, the majority of scenarios with an R0 of 1.5 were controllable with under 50% of contacts successfully traced. For R0 of 2.5 and 3.5, more than 70% and 90% of contacts respectively had to be traced to control the majority of outbreaks. The delay between symptom onset and isolation played the largest role in determining whether an outbreak was controllable for lower values of R0. For higher values of R0 and a large initial number of cases, contact tracing and isolation was only potentially feasible when less than 1% of transmission occurred before symptom onset. Interpretation: We found that in most scenarios contact tracing and case isolation alone is unlikely to control a new outbreak of 2019-nCov within three months. The probability of control decreases with longer delays from symptom onset to isolation, fewer cases ascertained by contact tracing, and increasing transmission before symptoms. This model can be modified to reflect updated transmission characteristics and more specific definitions of outbreak control to assess the potential success of local response efforts. '},\n"," {'paper_id': '2cd8b9a41c271e30c064212f8e113204d3e6e29e',\n","  'section': 'Abstract',\n","  'text': 'Introduction: Outbreaks of infectious diseases trigger an increase in scientific research and output. Early in outbreaks, evidence is scarce, but it accumulates rapidly. We are continuously facing new disease outbreaks, including the new coronavirus (SARS-nCoV-2) in December 2019.The objective of this study was to describe the accumulation of evidence during the 2013-2016 Zika virus (ZIKV) outbreak in the Pacific and the Americas related to aetiological causal questions about congenital abnormalities and Guillain-Barré syndrome. Methods: We hypothesised that the temporal sequence would follow a pre-specified order, according to study design. We assessed 1) how long it takes before findings from a specific study design appear, 2) how publication of preprints could reduce the time to publication and 3) how time to publication evolves over time. Results: We included 346 publications published between March 6, 2014 and January 1, 2019. In the 2013--2016 ZIKV outbreak, case reports, case series and basic research studies were published first. Case-control and cohort studies appeared between 400-700 days after ZIKV was first detected in the region of the study origin. Delay due to the publication process were lowest at the beginning of the outbreak. Only 4.6% of the publications was available as preprints. Discussion: The accumulation of evidence over time in new causal problems generally followed a hierarchy. Preprints reduced the delay to initial publication. Our methods can be applied to new emerging infectious diseases. '},\n"," {'paper_id': '2d02192c3d251c2b561d37274350660277658a74',\n","  'section': 'Abstract',\n","  'text': 'To extend the frontier of genome editing and enable the radical redesign of mammalian genomes, we developed a set of dead-Cas9 base editor (dBE) variants that allow editing at tens of thousands of loci per cell by overcoming the cell death associated with DNA double-strand breaks (DSBs) and single-strand breaks (SSBs). We used a set of gRNAs targeting repetitive elementsranging in target copy number from about 31 to 124,000 per cell. dBEs enabled survival after 5 large-scale base editing, allowing targeted mutations at up to ~13,200 and ~2610 loci in 293T and human induced pluripotent stem cells (hiPSCs), respectively, three orders of magnitude greater than previously recorded. These dBEs can overcome current on-target mutation and toxicity barriers that prevent cell survival after large-scale genome engineering. One Sentence Summary: Base editing with reduced DNA nicking allows for the simultaneous 10 '},\n"," {'paper_id': '2d0285c90976dfa43d58eca6b7d1d7ef29994fc2',\n","  'section': 'Abstract',\n","  'text': 'The human coronavirus HCoV-19 infection can cause acute respiratory distress syndrome 47 (ARDS), hypercoagulability, hypertension, extrapulmonary multiorgan dysfunction. Effective 48 antiviral and anti-coagulation agents with safe clinical profiles are urgently needed to improve the 49 overall prognosis. We screened an FDA approved drug library and found that an anticoagulant agent 50 dipyridamole (DIP) suppressed HCoV-19 replication at an EC50 of 100 nM in vitro. It also elicited 51 potent type I interferon responses and ameliorated lung pathology in a viral pneumonia model. In 52 analysis of twelve HCoV-19 infected patients with prophylactic anti-coagulation therapy, we found 53 that DIP supplementation was associated with significantly increased platelet and lymphocyte counts 54 and decreased D-dimer levels in comparison to control patients. Two weeks after initiation of DIP 55 treatment, 3 of the 6 severe cases (60%) and all 4 of the mild cases (100%) were discharged from the 56 hospital. One critically ill patient with extremely high levels of D-dimer and lymphopenia at the time 57 of receiving DIP passed away. All other patients were in clinical remission. In summary, HCoV-19 58 infected patients could potentially benefit from DIP adjunctive therapy by reducing viral replication, 59 suppressing hypercoagulability and enhancing immune recovery. Larger scale clinical trials of DIP 60 are needed to validate these therapeutic effects. 61 62 '},\n"," {'paper_id': '2da0a7ba851e53d21dba05042a397eafcdab093d',\n","  'section': 'Abstract',\n","  'text': \"With an ongoing threat posed by circulating zoonotic strains, new strategies are required to 8 prepare for the next emergent coronavirus (CoV). Previously, groups had targeted conserved 9 coronavirus proteins as a strategy to generate live-attenuated vaccine strains against current 10 and future CoVs. With this in mind, we explored whether manipulation of CoV NSP16, a 11 conserved 2'O methyltransferase (MTase), could provide a broad attenuation platform against 12 future emergent strains. Using the SARS-CoV mouse model, a NSP16 mutant vaccine was 13 evaluated for protection from heterologous challenge, efficacy in the aging host, and potential 14 for reversion to pathogenesis. Despite some success, concerns for virulence in the aged and 15 potential for reversion makes targeting NSP16 alone an untenable approach. However, 16 combining a 2'O MTase mutation with a previously described CoV fidelity mutant produced a 17 vaccine strain capable of protection from heterologous virus challenge, efficacy in aged mice, 18 and no evidence for reversion. Together, the results indicate that targeting the CoV 2'O MTase 19 in parallel with other conserved attenuating mutations may provide a platform strategy for 20 rapidly generating live-attenuated coronavirus vaccines. . CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/309591 doi: bioRxiv preprint 31 attenuation as a major component of future live-attenuated coronavirus vaccines. . CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/309591 doi: bioRxiv preprint 77 However, the broad conservation of NSP16 coupled with robust viral yields permitted its use in 78 combination with another attenuating mutation in NSP14, exonuclease (ExoN) activity. Targeting both conserved coronavirus activities produced a stable, attenuated virus capable of 80 protection from heterologous challenge, efficacy in aged mice, and absence of reversion in 81 immunocompromised models. Together, the results indicate that targeting coronavirus NSP16 82 may be a critical component of a future live attenuated coronavirus vaccine approach. 83 84 . CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/309591 doi: bioRxiv preprint 107 SARS-CoV induce severe weight loss, fail to recover, and several succumb to infection (Fig.   108   1B) . Notably, while the SARS NSP16 mutant induces weight loss, mice recover from infection 109 diverging from wild-type infection 3 days post infection. Examining viral replication, the NSP16 110 . CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/309591 doi: bioRxiv preprint 130 (SHC014-MA15). Our results indicate that the attenuated 2'O MTase SARS-CoV mutant is 131 capable of inducing robust protection from the zoonotic group 2B spike virus in terms of 132 disease. Weight loss data, viral lung titer, and hemorrhage score indicated little to no disease in 133 mice vaccinated with dNSP16 compared to mock vaccinated mice ( Fig. 2A-C) . Importantly, while not equivalent to neutralization of wild-type SARS-CoV, sera from dNSP16-vaccinated 135 mice were able to effectively block the SHC014 chimeric virus, likely contributing to protection \"},\n"," {'paper_id': '2dc42a5795f324d28993c1786fa007c6a7eb4fc5',\n","  'section': 'Abstract',\n","  'text': 'About one month after the COVID-19 epidemic peaked in Mainland China and SARS-CoV-2 migrated to Europe and then the U.S., the epidemiological data begin to provide important insights into the risks associated with the disease and the effectiveness of intervention strategies such as travel restrictions and social distancing. Respiratory diseases, including the 2003 SARS epidemic, remain only about two months in any given population, although peak incidence and lethality can vary. The epidemiological data suggest that at least two strains of the 2020 SARS-CoV-2 virus have evolved during its migration from Mainland China to Europe. South Korea, Iran, Italy, and Italy\\'s neighbors were hit by the more dangerous \"SKII\" variant. While the epidemic in continental Asia is about to end, and in Europe about to level off, the more recent epidemic in the younger US population is still increasing, albeit not exponentially anymore. The peak level will likely depend on which of the strains has entered the U.S. first. The same models that help us to understand the epidemic also help us to choose prevention strategies. Containment of high-risk people, like the elderly, and reducing disease severity, either by vaccination or by early treatment of complications, is the best strategy against a respiratory virus disease. Social distancing or \"lockdowns\" can be effective during the month following the peak incidence in infections, when the exponential increase of cases ends. Earlier containment of low-risk people merely prolongs the time the virus needs to circulate until the incidence is high enough to initiate \"herd immunity\". Later containment is not helpful, unless to prevent a rebound if containment started too early. '},\n"," {'paper_id': '2dc6766fe253b1b832d8a3074404fb5daae4762f',\n","  'section': 'Abstract',\n","  'text': 'The bacterial bipartite transferrin receptor is an iron acquisition system that is required for 12 survival by several key human and animal pathogens. It consists of the TonB-dependent 13 transporter Transferrin binding protein A (TbpA) and the surface lipoprotein Transferrin binding 14 protein B (TbpB). Curiously, the Tbps are only found in host specific pathogens, and are 15 themselves host specific, meaning that they will bind to the transferrin of their host species, but 16 not to those of other animal species. While this phenomenon has long been established, neither 17 the steps in the evolutionary process that led to this exquisite adaptation for the host, nor the 18 steps that could alter it, are known. We sought to gain insight into these processes by studying 19 Tbp specificity in Histophilus somni, a major pathogen of cattle. A past study showed that whole 20 cells of H. somni specifically bind bovine transferrin, but not transferrin from sheep and goats, 21 two bovids whose transferrins share 93% amino acid sequence identity with bovine transferrin. 22 To our surprise, we found that H. somni can use sheep and goat transferrins as iron sources for 23 growth, and that HsTbpB, but not HsTbpA, has detectable affinity for sheep and goat 24 transferrins. Furthermore, a third transferrin binding protein, HsTbpA2, also showed affinity for 25 sheep and goat transferrins. Our results show that H. somni TbpB and TbpA2 act to broaden the 26 host transferrin recognition range of H. somni. 27 '},\n"," {'paper_id': '2e3c753682fba0d9c7a0462051a1287b9599ac92',\n","  'section': 'Abstract',\n","  'text': 'are Co-corresponding authors Word Counts: 2574 words Abstract Background Previous studies had described the differences in clinical characteristics between ICU and non-ICU patients. However, seldom study focused on confirmed and unconfirmed groups. Our aim was to compare clinical and imaging characteristics of COVID-19 patients outside Hubei province between confirmed and unconfirmed group. '},\n"," {'paper_id': '2e41aee6651cf4cab4edf5aa67cc401c2bef9d67',\n","  'section': 'Abstract',\n","  'text': 'To compare the sex differences in the clinical findings among patients with severe coronavirus disease 2019 . We retrospectively collected data of 47 patients diagnosed as severe type of COVID-19 from February 8 to 22, 2020, including demographics, illness history, physical examination, laboratory test, management, and compared differences between men and women. : Of the 47 patients, 28 (59.6%) were men. The median age was 62 years, and 30 (63.8%) had comorbidities. The initial symptoms were mainly fever (34 [72.3%]), cough (36 [76.6%]), myalgia (5 [10.6%]) and fatigue (7 [14.9%]). Procalcitonin level was higher in men than in women (0.08 vs. 0.04ng/ml, p=0.002). N-terminal-pro brain natriuretic peptide increased in 16 (57.1%) men and 5 (26.3%) women (p=0.037). Five men (17.9%) had detected positive influenza A antibody, but no women. During 2-week admission, 5 (17.9%) men and 1 (5.3%) woman were reclassified into the critical type due to deterioration. Mortality was 3.6% in men and 0 in women respectively. Four (21.1%) women and one man (3.6%) recovered and discharged from hospital. Conclusion: Sex differences may exist in COVID-19 patients of severe type. Men are likely to have more complicated clinical condition and worse in-hospital outcomes as compared to women. : medRxiv preprint 0.508 Neutrophil count, ×10 9 /L 3.1 (2.1-5.2) 4.1 (2.0-6.9) 2.9 (2.5-3.4) 0.353 Lymphocyte count, ×10 9 /L 1.3 (0.7-1.9) 1.2 (0.7-2.0) 1.3 (0.9-1.6) 0 869 ≥1.0, No. (%) 30 (63.8) 15 (53.6) 15 (78.9) 0.076 Platelet count, ×10 9 /L 230.0 (157.0-302.5) 218.0 (151.3-320.8) 230.0 (164.0-304.0) 0.696 ＞300.0, No. (%) 14 (29.8) 8 (28.6) 6 (31.6) 0.825 Procalcitonin, ng/ml 0.06 (0.03-0.10) 0.08 (0.05-0.19) 0.04 (0.03-0.07) 0.002 increased (＞0.05), No. (%) 25 (53.2) 19 (67.9) 6 (31.6) 0.014 All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '2e56268497fa58710ae197a849a617f2aeac1b38',\n","  'section': 'Abstract',\n","  'text': 'Objective: To compare the difference of SARS-CoV-2 IgG antibody between male and female COVID-19 patients and figure out a possible explanation for different outcome between male and female patients. Methods: A total number of 331 patients confirmed SARS-CoV-2 infection were enrolled. The plasma of these patients were collected during hospitalization and were detected for SARS-CoV-2 IgG antibody. Afterwards, the difference of IgG antibody between male and female patients was analyzed. '},\n"," {'paper_id': '2e7c17caecfe67b64f9b59df404f0761df96987f',\n","  'section': 'Abstract',\n","  'text': \"Health planners use forecasts of key metrics associated with influenza-like-illness (ILI); near-term weekly incidence, week of season onset, week of peak, and intensity of peak. Here, we describe our participation in a weekly prospective ILI forecasting challenge for the United States for the 2016-17 season and subsequent evaluation of our performance. We implemented a metapopulation model framework with 32 model variants. Variants differed from each other in their assumptions about: the force-of-infection (FOI); use of uninformative priors; the use of discounted historical data for not-yet-observed time points; and the treatment of regions as either independent or coupled. Individual model variants were chosen subjectively as the basis for our weekly forecasts; however, a subset of coupled models were only available part way through the season. Most frequently, during the 2016-17 season, we chose; FOI variants with both school vacations and humidity terms; uninformative priors; the inclusion of discounted historical data for not-yet-observed time points; and coupled regions (when available). Our near-term weekly forecasts substantially over-estimated incidence early in the season when coupled models were not available. However, our forecast accuracy improved in absolute terms and relative to other teams once coupled solutions were available. In retrospective analysis, we found that the 2016-17 season was not typical: on average, coupled models performed better when fit without historically augmented data. Also, we tested a simple ensemble model for the 2016-17 season and found that it underperformed our subjective choice for all forecast targets. In this study, we were able to improve accuracy during a prospective forecasting exercise by coupling dynamics between regions. Although reduction of forecast subjectivity should be a long-term goal, some degree of human intervention is likely to improve forecast accuracy in the medium-term in parallel with the systematic consideration of more sophisticated ensemble approaches. It is estimated that there are between 3 and 5 million worldwide annual seasonal cases 1 of severe influenza illness, and between 290 000 and 650 000 respiratory deaths [1]. 2 Influenza-like-illness (ILI) describes a set of symptoms and is a practical way for 3 health-care workers to easily estimate likely influenza cases. The Centers for Disease 4 Control (CDC) collects and disseminates ILI information, and has, for the last several 5 years, run a forecasting challenge (the CDC Flu Challenge) for modelers to predict 6 near-term weekly incidence, week of season onset, week of peak, and intensity of peak. 7 We have developed a modeling framework that accounts for a range of mechanisms 8 thought to be important for influenza transmission, such as climatic conditions, school 9 vacations, and coupling between different regions. In this study we describe our forecast 10 procedure for the 2016-17 season and highlight which features of our models resulted in 11 better or worse forecasts. Most notably, we found that when the dynamics of different 12 regions are coupled together, the forecast accuracy improves. We also found that the 13 most accurate forecasts required some level of forecaster interaction, that is, the 14 procedure could not be completely automated without a reduction in accuracy. PLOS 2/43 . CC-BY 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/309021 doi: bioRxiv preprint Infectious pathogens with short generation times pose public health challenges because 17 they generate substantial near-term uncertainty in the risk of disease. This uncertainty 18 is most acute and shared globally during the initial stages of emergence of novel human 19 pathogens such as SARS [2], pandemic influenza [3], or the Zika virus [4] . However, at 20 national and sub-national levels, uncertainty arises frequently for epidemic pathogens 21 such as seasonal influenza, dengue, RSV and rotavirus; causing problems both for 22 health planners and at-risk individuals who may consider changing their behavior to 23 mitigate their risk during peak periods. 24 Seasonal influenza affects populations in all global regions and is forecast annually in 25 temperate populations, either implicitly or explicitly [5]. Peak demand for both 26 outpatient and inpatient care is driven by peak incidence of influenza in many years [6]. 27 Therefore, the efficient provision of elective procedures and other non-seasonal health 28 care can be improved by accurate forecasts of seasonal influenza. Implicitly, most 29 temperate health systems use knowledge of historical scenarios with which to plan for 30 their influenza season. The current situation is then assessed against the deviation from 31 the historical averages and worst-cases as observed in their own surveillance system. The United States Centers for Disease Control (CDC) has sought to formalize regional 33 and national forecasts by introducing an annual competition [7] . Each week, 34 participating teams submit weekly estimates of incidence for the next four weeks, season 35 onset, and timing and intensity of the peak. Methods used by teams include purely 36 statistical models, [8-10] mechanistic models [11, 12] machine learning and hybrid 37 approaches [13-15]. Expert-opinion surveys have also been used and performed well. 38 Some teams augment their forecasts of the official ILI data with the use of potentially 39 faster or less-noisy datasets such as google flu trends [16]. 40 Here we describe a our mechanistic-model-supported participation in the 2016-17 CDC 41 influenza forecasting challenge, as an example of a disease forecasting process. We 42 emphasize a subjective human component of this process and also describe a 43 retrospective evaluation of the models for the previous six seasons. All the models 44 PLOS 3/43 . CC-BY 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/309021 doi: bioRxiv preprint described are implemented in the R package Dynamics of Interacting Community 45 Epidemics (DICE, https://github.com/predsci/DICE). 46 Methods 47 Data 48 The CDC Influenza-like-illness Surveillance Network (ILINet) Human and Health 49 Services (HHS) region and national data were downloaded from the CDC-hosted web 50 application FluView [17] and used to create a historic database of ILI cases. Fig S1 51 shows which states are grouped into each HHS region, along with the population of each 52 region. Because we require an absolute number of cases per week, the CDC ILINet data 53 is converted from percent ILI cases per patient to ILI cases. We estimate the absolute 54 number of weekly ILI cases by dividing the weighted percent of ILI cases in the CDC 55 data by 100 and multiplying it by the total weekly number of patients. We assume two 56 outpatient visits per person per year so that the total weekly number of patients is 57 estimated as: (total regional population)x(2 outpatient visits per person per year)x(1 58 year/52 weeks). 59 The estimate of two outpatient visits per year is based on two studies. In 2006 60 Schappert and Burt [18] studied the National Ambulatory Medical Care Surveys 61 (NAMCS) and the National Hospital Ambulatory Medical Care Surveys (NHAMCS) 62 and calculated an ambulatory rate of 3.8 visits per capita-year. The 2011 NAMCS [19] 63 and NHAMCS surveys [20] estimated ambulatory visit rates of 3.32 visits per capita per 64 year for physician's offices, 0.43 for hospital emergency departments and 0.33 hospital 65 Outpatient Departments. We sum these rates to get an outpatient visit per capita-year 66 of 4.08. We further estimate from the surveys that only half of these outpatient clinics 67 are sites that report to ILINet, and hence we rounded to our two outpatient visits per 68 year estimate. 69 Specific humidity (SH) is measured in units of kg per kg and is defined as the ratio of 70 PLOS 4/43 . CC-BY 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/309021 doi: bioRxiv preprint water vapor mass to total moist air mass. Two other measurements of humidity are 71 absolute humidity and relative humidity. SH is included in DICE as a potential modifier 72 of transmissibility for this time period and uses Phase-2 of the North American Land 73 Data Assimilation System (NLDAS-2) data base provided by NASA [21-23]. The 74 NLDAS-2 data base provides hourly specific humidity (measured 2-meters above the 75 ground) for the continental US at a spatial grid of 0.125 • which we average to daily and 76 weekly values. The weekly data is then spatially-averaged for the states and CDC 77 regions. 78 School vacation schedules were collected for the 2014-15 and 2015-16 academic years for 79 every state. For each state, a school district was identified to represent each of the three 80 largest cities. Vacation schedules were then collected directly from the district websites. 81 These three school vacation schedules were first processed to a weekly schedule with a 82 value of 0 indicating class was in session all five weekdays and a 1 indicating five 83 vacation days. Next, the representative state schedule was produced by averaging the 84 three weekly district schedules. Region schedules are obtained by a population-weighted 85 average of the state schedules. Similarly, the national schedule is generated by a 86 population-weighted average of the regions. 87 For the 2016-17 season we determine start and end times as well as spring and fall 88 breaks from the previous years schedules. Thanksgiving and winter vacation timing was 89 taken from the calendar where the winter break is assumed to be the last two calendar 90 weeks of the year. Based on the proportion of schools closed and number of days closed, 91 p(t) is assigned a value in the range [0, 1]. For example in week t i , if all schools are 92 closed for the entire week then we define the proportion of open schools p(t i ) = 1. 93 However, if all schools have Monday and Tuesday off (missing 2 of 5 days), then 94 p(t i ) = 0.4. Similarly, if 3 of 10 schools have spring break (entire week off), but the 95 other 7 schools have a full week of class then p(t i ) = 0.3. If all schools have a full week 96 of class then p(t i ) = 0. \"},\n"," {'paper_id': '2ea102f58147dab02e4dea90eb90dbc67149f678',\n","  'section': 'Abstract',\n","  'text': 'The 2019 novel coronavirus (2019-nCoV or SARS-CoV-2) has spread more rapidly than any other betacoronavirus including SARS-CoV and MERS-CoV. However, the mechanisms responsible for infection and molecular evolution of this virus remained unclear. 5 Methods: We collected and analyzed 120 genomic sequences of 2019-nCoV including 11 novel genomes from patients in China. Through comprehensive analysis of the available genome sequences of 2019-nCoV strains, we have tracked multiple inheritable SNPs and determined the evolution of 2019-nCoV relative to other coronaviruses. 10 Results: Systematic analysis of 120 genomic sequences of 2019-nCoV revealed cocirculation of two genetic subgroups with distinct SNPs markers, which can be used to trace the 2019-nCoV spreading pathways to different regions and countries. Although 2019-nCoV, human and bat SARS-CoV share high homologous in overall genome structures, they evolved into two distinct groups with different receptor entry specificities 15 through potential recombination in the receptor binding regions. In addition, 2019-nCoV has a unique four amino acid insertion between S1 and S2 domains of the spike protein, which created a potential furin or TMPRSS2 cleavage site. Our studies provided comprehensive insights into the evolution and spread of the 2019-nCoV. Our results provided evidence suggesting that 2019-nCoV 20 may increase its infectivity through the receptor binding domain recombination and a cleavage site insertion. author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '2eac7de1c407d9eab13a6d538142cd21a814ee7b',\n","  'section': 'Abstract',\n","  'text': \"Rapid production and publication of pathogen genome sequences during emerging disease outbreaks provide crucial public health information. In resource-limited settings, especially near an outbreak epicenter, conventional deep sequencing or bioinformatics are often challenging. Here we successfully used metagenomic next generation sequencing on an iSeq100 Illumina platform paired with an open-source bioinformatics pipeline to quickly characterize Cambodia's first case of COVID- \"},\n"," {'paper_id': '2ed28b3c39080ffc7b75faa527b8b74b6253b45c',\n","  'section': 'Abstract',\n","  'text': 'Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is already responsible for far more deaths than previous pathogenic coronaviruses (CoVs) from 2002 and 2012. The identification of clinically approved drugs to be repurposed to combat 2019 CoV disease (COVID-19) would allow the rapid implementation of potentially lifesaving procedures. The major protease (Mpro) of SARS-CoV-2 is considered a promising target, based on previous results from related CoVs with lopinavir (LPV), an HIV protease inhibitor. However, limited evidence exists for other clinically approved antiretroviral protease inhibitors, such as atazanavir (ATV). ATV is of high interest because of its bioavailability within the respiratory tract. Our results show that ATV could dock in the active site of SARS-CoV-2 Mpro, with greater strength than LPV. ATV blocked Mpro activity. We confirmed that ATV inhibits SARS-CoV-2 replication, alone or in combination with ritonavir (RTV) in Vero cells, human pulmonary epithelial cell line and primary monocytes, impairing virus-induced enhancement of IL-6 and '},\n"," {'paper_id': '2f516463f332a70a46e5157c6ce1030d0abc994d',\n","  'section': 'Abstract',\n","  'text': 'Testing strategies for to maximize number of people tested is urgently needed. Recently, it has been demonstrated that RT-PCR has the sensitivity to detect one positive case in a mixed sample 32 cases [9] . In this paper we propose non-adaptive and adaptive group testing strategies based on generalized binary splitting (GBS) [3] where we restrict the group test to the largest group that can be used. The method starts by choosing a group from the population to be tested, performing a test on the combined sample from the entire group and progressively splitting the group further into subgroups. Compared to individual testing at 4% prevalence we save 74% at 1% we save 91% and at 1% we save 97% of tests. We analyze the number of times each sample is used and show the method is still ecient if we resort to testing a case individually if the sample is running low. In addition we recommend clinical screening to lter out individuals with symptoms and show this leaves us with a population with lower prevalence. Our approach is particularly applicable to vulnerable conned populations such as nursing homes, prisons, military ships and cruise ships. '},\n"," {'paper_id': '2f6a5d5e69b50ca5f7b14edcd7b189f595e138a6',\n","  'section': 'Abstract',\n","  'text': 'We present a data-driven machine learning analysis of COVID-19 from its early infection dynamics, with the goal of extracting actionable public health insights, including predictions for the number of new infections over time. We focus on the transmission dynamics in the USA starting from the first confirmed infection on January 21 2020. We find that COVID-19 has a strong infectious force if left unchecked, with a doubling time of under 3 days. However it is not particularly virulent. Our methods may be of general interest. '},\n"," {'paper_id': '2f6efc4edc0c9fffd3acc810efd8616e7772b28a',\n","  'section': 'Abstract',\n","  'text': 'The appearance of the novel betacoronavirus SARS-CoV-2 represents a major threat to human health, and its diffusion around the world is causing dramatic consequences. The knowledge of the 3D structures of SARS-CoV-2 proteins can facilitate the development of therapeutic and diagnostic molecules. Specifically, comparative analyses of the structures of SARS-CoV-2 proteins and homologous proteins from previously characterized viruses, such as SARS-CoV, can reveal the common and/or distinctive traits that underlie the mechanisms of recognition of cell receptors and of molecules of the immune system. Herein, we apply our recently developed energy-based methods for the prediction of antibodybinding epitopes and protein-protein interaction regions to the Receptor Binding Domain (RBD) of the Spike proteins from SARS-CoV-2 and SARS-CoV. Our analysis focusses only on the study of the structure of RBDs in isolation, without making use of any previous knowledge of binding properties. Importantly, our results highlight structural and sequence differences among the regions that are predicted to be immunoreactive and bind/elicit antibodies. These results provide a rational basis to the observation that several SARS-CoV RDB-specific monoclonal antibodies fail to appreciably bind the SARS-CoV-2 counterpart. Furthermore, we correctly identify the region of SARS-CoV-2 RBD that is engaged by the cell receptor ACE2 during viral entry into host cells. The data, sequences and structures we present here can be useful for the development of novel therapeutic and diagnostic interventions. '},\n"," {'paper_id': '2fe18190f7b54e589dccf1d3bbfd73e87bfd2ab9',\n","  'section': 'Abstract',\n","  'text': '12 The current classification of animal viruses primarily relates to the virus molecular world, the 13 genomic architecture and the corresponding host-cell infection cycle. This virus centered 14 perspective does not make allowance for the precept that virus fitness hinges on the virus 15 transmission success. Virus transmission reflects the infection-shedding-transmission 16 dynamics and, with it, the organ system involvement and other, macroscopic dimensions of 17 the host environment. This study examines the transmission ecology of the world main 18 livestock viruses, 36 in total, belonging to eleven different families, and a mix of RNA, DNA 19 and retroviruses. Viruses are virtually ranked in an outer-to inner-body fashion, based on 20 the shifting organ system involvement and associated infection-shedding-transmission 21 dynamics. As a next step, this ranking is disentangled with the aim to contrast two main host 22 45 the virus. Two opposing virus evolution pathways emerge, respectively for generalist type, 46 outer-body and for specialist, inner-body viruses. '},\n"," {'paper_id': '2fe6b550f737baa47a5f2c8ab64cc3d9271c308a',\n","  'section': 'Abstract',\n","  'text': 'Our society is currently experiencing an unprecedented challenge, managing and containing an outbreak of a new coronavirus disease known as COVID-19. While China -where the outbreak started -seems to have been able to contain the growth of the epidemic, different outbreaks are nowadays being detected in multiple countries. Much is currently unknown about the natural history of the disease, such as a possible asymptomatic spreading or the role of age in both the susceptibility and mortality of the disease. Nonetheless, authorities have to take action and implement contention measures, even if not everything is known. To facilitate this task, we have studied the effect of different containment strategies that can be put into effect. Our work specifically refers to the situation in Spain as of February 28th, 2020, where a few dozens of cases have been detected. We implemented an SEIR-metapopulation model that allows tracing explicitly the spatial spread of the disease through data-driven stochastic simulations. Our results are in line with the most recent recommendations from the World Health Organization, namely, that the best strategy is the early detection and isolation of individuals with symptoms, followed by interventions and public recommendations aimed at reducing the transmissibility of the disease, which although not efficacious for disease eradication, would produce as a second order effect a delay of several days in the raise of the number of infected cases. . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': '2fef425f6ba67b76292a3adbd2a1385c0114e1c7',\n","  'section': 'Abstract',\n","  'text': 'Finding vaccine or specific antiviral treatment for global pandemic of virus diseases (such as the ongoing COVID-19) requires rapid analysis, annotation and evaluation of metagenomic libraries to enable a quick and efficient screening of nucleotide sequences. Traditional sequence alignment methods are not suitable and there is a need for fast alignment-free techniques for sequence analysis. Information theory and data compression algorithms provide a rich set of mathematical and computational tools to capture essential patterns in biological sequences. In 2013, our research group (Nagaraj et al., Eur. Phys. J. Special Topics 222(3-4), 2013) has proposed a novel measure known as Effort-To-Compress (ETC) based on the notion of compression-complexity to capture the information content of sequences. In this study, we propose a compression-complexity based distance measure for automatic identification of SARS coronavirus strains from a set of viruses using only short fragments of nucleotide sequences. We also demonstrate that our proposed method can correctly distinguish SARS-CoV-2 from SARS-CoV-1 viruses by analyzing very short segments of nucleotide sequences. This work could be extended further to enable medical practitioners in automatically identifying and characterizing SARS coronavirus strain in a fast and efficient fashion using short and/or incomplete segments of nucleotide sequences. Potentially, the need for sequence assembly can be circumvented. '},\n"," {'paper_id': '2ffc83b88bc84615273e97d8109e5439ce4dce60',\n","  'section': 'Abstract',\n","  'text': 'Estimating temporal changes in a target population from phylogenetic or count data is an important problem in ecology and epidemiology. Reliable estimates can provide key insights into the climatic and biological drivers influencing the diversity or structure of that population and evidence hypotheses concerning its future growth or decline. In infectious disease applications, the individuals infected across an epidemic form the target population. The renewal model estimates the effective reproduction number, R, of the epidemic from counts of its observed cases. The skyline model infers the effective population size, N, underlying a phylogeny of sequences sampled from that epidemic. Practically, R measures ongoing epidemic growth while N informs on historical caseload. While both models solve distinct problems, the reliability of their estimates depends on p-dimensional piecewise-constant functions. If p is misspecified, the model might underfit significant changes or overfit noise and promote a spurious understanding of the epidemic, which might misguide intervention policies or misinform forecasts. Surprisingly, no transparent yet principled approach for optimising p exists. Usually, p is heuristically set, or obscurely controlled via complex algorithms. We present a computable and interpretable p-selection method based on the minimum description length (MDL) formalism of information theory. Unlike many standard model selection techniques, MDL accounts for the additional statistical complexity induced by how parameters interact. As a result, our method optimises p so that R and N estimates properly adapt to the available data. It also outperforms comparable Akaike and Bayesian information criteria on several classification problems. Our approach requires some knowledge of the parameter space and exposes the similarities between renewal and skyline models. '},\n"," {'paper_id': '300183d862ef809c734535bb79928efa4cdcd208',\n","  'section': 'Abstract',\n","  'text': 'The COVID-19 infections in Pakistan are spreading at an exponential rate and a point may soon be reached where rigorous prevention measures would need to be adopted. Mathematical models can help define the scale of an epidemic and the rate at which an infection can spread in a community. We used an SIR model to predict the magnitude of the COVID-19 epidemic in Pakistan and compared the numbers with the reported cases on the national database. Our results indicate that Pakistan could hit peak number of infectious cases on May 26th, 2020 and by June 24th, 2020, 90% of the population will have become infected with the virus if policy interventions seeking to curb this infection are not adopted aggressively. Since the emergence of COVID-19, several mathematical models have been developed to simulate the rate of infection spread, infections per day and the resolution of the epidemic. The SIR model refers to the number of susceptible, infected and recovered cases during an epidemic at any given time. The model assumes that susceptible cases (S), infected cases (I) and . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': '300ff0def92740f7e1142036f9e6ae30845e13d2',\n","  'section': 'Abstract',\n","  'text': \"Motivated by the rapid spread of COVID-19 all across the globe, we have performed simulations of a system dynamic epidemic spread model in different possible situations. The simulation, not only captures the model dynamic of the spread of the virus, but also, takes care of population and mobility data. The model is calibrated based on epidemic data and events specifically of Sultanate of Oman, which can easily be generalized. The simulation results are quite disturbing, indicating that, during a process of stringent social distancing and testing strategies, a small perturbation can lead to quite undesirable outcomes. The simulation results, although consistent in expected outcomes across changing parameters' values, also indicate a substantial mismatch with real numbers. An analysis of what can be the reason of this mismatch is also performed. Within these contradictions, for Oman, regarding the eradication of epidemic, the future is not extremely alarming. \"},\n"," {'paper_id': '3028628066ec2401f3981f4e70c5b1acd4cef573',\n","  'section': 'Abstract',\n","  'text': 'Background: As the COVID-19 epidemic is spreading, incoming data allows us to quantify values of key variables that determine the transmission and the effort required to control the epidemic. We determine the incubation period and serial interval distribution for transmission clusters in Singapore and in Tianjin. We infer the basic reproduction number and identify the extent of pre-symptomatic transmission. Methods: We collected outbreak information from Singapore and Tianjin, China, reported from Jan.19-Feb.26 and Jan.21-Feb.27, respectively. We estimated incubation periods and serial intervals in both populations. Results: The mean incubation period was 7.1 (6.13, 8.25) days for Singapore and 9 (7.92, 10.2) days for Tianjin. Both datasets had shorter incubation periods for earlier-occurring cases. The mean serial interval was 4.56 (2.69, 6.42) days for Singapore and 4.22 (3.43, 5.01) for Tianjin. We inferred that early in the outbreaks, infection was transmitted on average 2.55 and 2.89 days before symptom onset (Singapore, Tianjin). The estimated basic reproduction number for Singapore was 1.97 (1.45, 2.48) secondary cases per infective; for Tianjin it was 1.87 (1.65, 2.09) secondary cases per infective. Conclusions: Estimated serial intervals are shorter than incubation periods in both Singapore and Tianjin, suggesting that pre-symptomatic transmission is occurring. Shorter serial intervals lead to lower estimates of R0, which suggest that half of all secondary infections should be prevented to control spread. '},\n"," {'paper_id': '3034d4ed2359a4706ff9f5096b1945eaa3b8bd1f',\n","  'section': 'Abstract',\n","  'text': 'The overall objective of this rapid overview of reviews (overview hereafter) was to identify evidence from systematic reviews (SRs) for infection control and prevention practices for adults aged 60 years and older in long-term care settings. Methods: Comprehensive searches in MEDLINE, EMBASE, the Cochrane Library, biorxiv.org/medrxiv.org, clinicaltrials.gov and the Global Infectious Disease Epidemiology Network (GIDEON) were carried out in early March 2020. Title/abstract and full-text screening, data abstraction, and quality appraisal (AMSTAR 2) were carried out by single reviewers. '},\n"," {'paper_id': '30457ee3ce0001f33938fbc246b4ce4eacd74f5d',\n","  'section': 'Abstract',\n","  'text': \"The condent high-throughput identication of small molecules remains one of the most challenging tasks in mass spectrometry-based metabolomics. SIRIUS has become a powerful tool for the interpretation of tandem mass spectra, and shows outstanding performance for identifying the molecular formula of a query compound, being the rst step of structure identication. Nevertheless, the identication of both molecular formulas for large compounds above 500 Daltons and novel molecular formulas remains highly challenging. Here, we present ZODIAC, a network-based algorithm for the de novo estimation of molecular formulas. ZODIAC reranks SIRIUS' molecular formula candidates, combining fragmentation tree computation with Bayesian statistics using Gibbs sampling. Through careful algorithm engineering, ZODIAC's Gibbs sampling is very swift in practice. ZODIAC decreases incorrect annotations 16.2-fold on a challenging plant extract dataset with most compounds above 700 Dalton; we then show improvements on four additional, diverse datasets. Our analysis led to the discovery of compounds with novel molecular formulas such as C 24 H 47 BrNO 8 P which, as of today, is not present in any publicly available molecular structure databases. Metabolomics characterizes metabolites with high-throughput techniques. In the past decade, liquid chromatography coupled with tandem mass spectrometry (LC-MS/MS) became widely adopted by the metabolomics community as a powerful and sensitive analytical platform, but yet only a fraction of the detected compounds can be eectively annotated, even partially. In untargeted LC-MS/MS experiments, thousands of metabolites can be detected and fragmented from a single biological sample. The measured fragmentation spectra (MS/MS spectra) are then used to examine the metabolite's structure. But condent high-throughput annotation of those MS/MS spectra remains highly challenging; this prevents the building of comprehensive knowledge in elds where metabolomics has become central, such as biomedical research, natural products drug discovery 1 , environmental science, and food science. Tandem mass spectrometry data is often searched against spectral libraries 2 ; unfortunately, spectral libraries are of limited size, in particular when it comes to biomolecules 3 . Although public spectral libraries are being expanded, only a fraction of acquired MS/MS spectra can be annotated by spectral library searches 1,46 . Computational methods have been developed that do not search in spectral libraries but rather in molecular structure databases 7 . Among these methods, CSI:FingerID 8 has repeatedly shown the . doi: bioRxiv preprint 2 Ludwig, Nothias, Dührkop, et al. best performance 912 . One reason for CSI:FingerID's improved performance is the integration of SIRIUS 10 , deducing the molecular formula of each query as the rst step of its analysis. Other tools lter candidates using the query precursor mass, reducing molecular formula annotation to a byproduct. This worsens identication rates 11 and can result in severe hidden prior problems 13,14 . Identifying the molecular formula is also the very rst step in structural elucidation using Nuclear Magnetic Resonance (NMR) or X-ray crystallography, guiding data interpretation based on atoms and unsaturation degree. The condent annotation of molecular formulas from mass spectrometry data is far from trivial, especially if executed de novo (without a structure database): Here, the number of candidate molecular formula grows rapidly with the compound size and elements beyond CHNOPS. To counter this growth, one can use heuristic constraints 15 or use only molecular formulas from some structure database 16,17 . Restricting the search space will improve the performance of a method in evaluation, but will prevent the discovery of novel molecular formulas in application. Arguably the best-performing computational method for molecular formula annotation is SIRIUS 4 10 , which combines isotope pattern matching 15,1823 and MS/MS fragmentation tree computation 22,2426 . SIRIUS reaches best-of-class performance without ltering or meta-scores 24 . But even SIRIUS has problems annotating molecular formula for compounds above 500 Da: Böcker & Dührkop 24 found that the percentage of correctly identied molecular formulas dropped substantially for larger masses. An alternative approach to annotate molecular formulas for a complete LC-MS run uses Gibbs sampling and Bayesian statistics, utilizing co-occurrence of molecular formulas diering by a predened set of biotransformations 2730 . Implicitly, these approaches try to identify molecular structures (or their isomers) from a restricted structure database, and cannot annotate novel molecular formulas. Network visualization approaches which connect compounds by hypothetical biotransformations and common chemical functional groups have been demonstrated to ease manual molecular formula annotation 31 . Independently, network-based methods were developed for structural elucidation and dereplication 1,32,33 . All of these approaches are based on the fact that compounds in an LC-MS run usually co-occur in a network of derivatives. We present ZODIAC (ZODIAC: Organic compound Determination by Integral Assignment of elemental Compositions) for condent, database-independent molecular formula annotation in LC-MS/MS data. ZODIAC takes advantage of the fact that an organism produces related metabolites that are derived from multiple, but limited, biosynthetic pathways. ZODIAC builds upon SIRIUS and uses, say, the top 50 molecular formula annotations from SIRIUS as candidates for one compound. ZODIAC then reranks molecular formula candidates using Bayesian statistics. Prior probabilities are derived from fragmentation tree similarity, which supports reciprocal plausibility within an LC-MS/MS dataset. On the theoretical side, we establish that nding an optimal solution to the resulting computational problem is non-deterministic polynomial time (NP)-hard; to this end, we resort to Gibbs sampling. Using extensive algorithm engineering, Gibbs sampling running times were reduced to a practical level. To boost robustness, ZODIAC can integrate spectral library search hits. We show that ZODIAC improves molecular formula annotation on a diverse set of biological samples. Furthermore, ZODIAC scores allow us to rank molecular formula annotations by condence. ZODIAC is not limited to molecular formulas from some structure database, allowing us to discover novel molecular formulas not present in any structural databases. ZODIAC was evaluated on ve diverse datasets representing samples from plants, human plasma, marine microalgae and mice fecal sample, see Supplementary Table 1, 5 and 6 and Supplementary Fig. 6. Input mzML/mzXML les were processed with OpenMS 34 and low-quality MS/MS spectra . \"},\n"," {'paper_id': '30814fcc87b8f3e5d4f7431d75b9505b049021e0',\n","  'section': 'Abstract',\n","  'text': 'The 2015-2016 emergence of Zika virus (ZIKV) in the Americas, and recognition that ZIKV 15 infection during pregnancy can result in birth defects, revealed a need for small animal models to 16 study ZIKV pathogenic mechanisms and evaluate candidate vaccines and antivirals. Mice would 17 be an attractive system for such studies, but ZIKV replicates poorly in laboratory mice because it 18 fails to antagonize murine STAT2 and STING. To address this, most ZIKV pathogenesis studies 19 have used mice with impaired interferon signaling (e.g. Ifnar1 -/or treatment with IFNAR1-blocking 20 antibodies). However, using mice with severe defects in innate antiviral signaling confounds 21 studies of viral pathogenic mechanisms. Collaborative Cross (CC) mice have proven to be a 22 valuable system for developing new mouse pathogenesis models for viral infections that are not 23 well modeled in conventional laboratory mouse lines. To test whether CC mice could provide an 24 immune-competent model for ZIKV pathogenesis, we infected CC lines with ZIKV and assessed 25 weight loss, viremia, and production of neutralizing antibodies. We tested 21 CC lines (CC001, 26 All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '30b5c7b8faf95265f67ae59f4686eaf9b2772893',\n","  'section': 'Abstract',\n","  'text': '14 Traveller screening is being used to limit further global spread of 2019 novel coronavirus (nCoV) 15 following its recent emergence. Here, we project the impact of different travel screening 16 programs given remaining uncertainty around the values of key nCoV life history and 17 epidemiological parameters. Even under best-case assumptions, we estimate that screening will 18 miss more than half of infected travellers. Breaking down the factors leading to screening 19 successes and failures, we find that most cases missed by screening are fundamentally 20 undetectable, because they have not yet developed symptoms and are unaware they were 21 exposed. These findings emphasize the need for measures to track travellers who become ill 22 after being missed by a travel screening program. We make our model available for interactive 23 use so stakeholders can explore scenarios of interest using the most up-to-date information. We 24 hope these findings contribute to evidence-based policy to combat the spread of nCoV, and to 25 prospective planning to mitigate future emerging pathogens. '},\n"," {'paper_id': '30c67218bbd1a5ffe68dab7fe7dfc5bc662e05a2',\n","  'section': 'Abstract',\n","  'text': 'Aims: Studies have indicated that chloroquine (CQ) shows antagonism against COVID-19 in vitro. '},\n"," {'paper_id': '311a15d92b87b2bc41f00afed881b95e93c22fcc',\n","  'section': 'Abstract',\n","  'text': 'doi: bioRxiv preprint 65 required for the antiviral activity of both IFITM3 and IFITM1 [9]. The subcellular localization of 66 IFITMs is a key determinant of their antiviral profile. When expressed singly, IFITM3 and IFITM2 67 preferentially localize to early and late endosomes and lysosomes, and restrict viruses that enter via 68 these endo-lysosomal compartments. In contrast, IFITM1 primarily localizes at the cell surface and 69 can restrict viruses that enter through the plasma membrane [11-14]. Indeed, mutants of IFITM3 70 that lack an N-terminal endocytic sorting motif 20 YEML 23 localize to the plasma membrane and lose 71 their ability to inhibit influenza A virus (IAV), alphavirus and coronavirus infection by endosomal 72 routes [14-18]. 73 74 Studies focusing on IFITM3 restriction of IAV and Semliki Forest Virus (SFV) indicate that virus 75 internalization is unaffected by IFITM3 expression and, for SFV at least, the viral envelope 76 . CC-BY-NC-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/737841 doi: bioRxiv preprint 109 knockout mice show enhanced morbidity after infection with IAV, alphaviruses and flaviviruses [33-110 . '},\n"," {'paper_id': '315197247b230bab03786c197dc1fc795164c371',\n","  'section': 'Abstract',\n","  'text': 'Accumulating evidence has shown that long noncoding RNAs (lncRNAs) are involved in 25 several biological processes, including immune responses. However, the role of lncRNAs in 26 antiviral innate immune responses remains largely unexplored. Here, we identify an 27 uncharacterized human lncRNA from influenza A virus (IAV) patients, antivirus and activate 28 neutrophil (AVAN), that is significantly up-regulated upon virus infection. Mechanistically, 29 nuclear lncRNA-AVAN positively regulates the transcription of forkhead box O3A (FOXO3a) 30 by associating with its promoter and inducing chromatin remodeling to promote neutrophil 31 chemotaxis. Furthermore, we also found that cytoplasmic lncRNA-AVAN directly binds 32 tripartite motif containing 25 (TRIM25) and enhances the association of TRIM25 and 33 Retinoic acid inducible gene-1 proteins (RIG-I) and the ubiquitylation of RIG-I, thereby 34 promoting TRIM25-and RIG-I-mediated antiviral innate immune signaling. More 35 importantly，we enforced the expression of AVAN in transgenic mice and found that it 36 significantly alleviated IAV virulence and virus production. Collectively, these findings 37 highlight the potential clinical implications of lncRNA-AVAN as a key positive regulator of 38 the antiviral innate immune response and a promising target for developing broad antiviral 39 therapeutics. 40 41 '},\n"," {'paper_id': '31710abfafa1ad587d8a91fa45fa621c432fef9c',\n","  'section': 'Abstract',\n","  'text': 'In December 2019, an outbreak of atypical pneumonia (Coronavirus disease 2019 -COVID-19) associated with a novel coronavirus (SARS-CoV-2) was reported in Wuhan city, Hubei province, China. The outbreak was traced to a seafood wholesale market and human to human transmission was confirmed. The rapid spread and the death toll of the new epidemic warrants immediate intervention. The intra-host genomic variability of SARS-CoV-2 plays a pivotal role in the development of effective antiviral agents and vaccines, but also in the design of accurate diagnostics. We analyzed NGS data derived from clinical samples of three Chinese patients infected with SARS-CoV-2, in order to identify small-and large-scale intra-host variations in the viral genome. We identified tens of low-or higher-frequency single nucleotide variations (SNVs) with variable density across the viral genome, affecting 7 out of 10 protein-coding viral genes. The majority of these SNVs corresponded to missense changes. The annotation of the identified SNVs but also of all currently circulating strain variations revealed colocalization of intra-host but also strain specific SNVs with primers and probes currently used in molecular diagnostics assays. Moreover, we de-novo assembled the viral genome, in order to isolate and validate intra-host structural variations and recombination breakpoints. The bioinformatics analysis disclosed genomic rearrangements over poly-A / poly-U regions located in ORF1ab and spike (S) gene, including a potential recombination hot-spot within S gene. Our results highlight the intra-host genomic diversity and plasticity of SARS-CoV-2, pointing out genomic regions that are prone to alterations. The isolated SNVs and genomic rearrangements, reflect the intra-patient capacity of the polymorphic quasispecies, which may arise rapidly during the outbreak, allowing immunological escape of the virus, offering resistance to anti-viral drugs and affecting the sensitivity of the molecular diagnostics assays. author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '317215c463463ddb39fa7564f67239dc484b86a9',\n","  'section': 'Abstract',\n","  'text': 'International Concern following a cluster of associated neurological disorders and neonatal malformations. In 2017, Zika cases declined, but future incidence in LAC remains uncertain due to gaps in our understanding, considerable variation in surveillance and a lack of a comprehensive collation of data from affected countries. Methods Our analysis combines information on confirmed and suspected Zika cases across LAC countries and a spatio-temporal dynamic transmission model for ZIKV infection to determine key transmission parameters and projected incidence in 91 major cities within 35 countries. Seasonality was determined by spatio-temporal estimates of Aedes aegypti vector capacity. We used country and state-level data from 2015 to mid-2017 to infer key model parameters, country-specific disease reporting rates, and the 2018 projected incidence. A 10-fold cross-validation approach was used to validate parameter estimates to out-of-sample epidemic trajectories. There was limited transmission in 2015, but in 2016 and 2017 there was sufficient opportunity for wide-spread ZIKV transmission in most cities, resulting in the depletion of susceptible individuals. We predict that the highest number of cases in 2018 within some Brazilian States (Sao Paulo and Rio de Janeiro), Colombia and French Guiana, but the estimated number of cases were no more than a few hundred. Model estimates of the timing of the peak in incidence were correlated (p<0.05) with the reported peak in incidence. The reporting rate varied across countries, with lower reporting rates for those with only confirmed cases compared to those who reported both confirmed and suspected cases. The findings suggest that the ZIKV epidemic is by and large over, with incidence projected to be low in most cities in LAC in 2018. Local low levels of transmission are probable but the estimated rate of infection suggests that most cities have a population with high levels of herd immunity. '},\n"," {'paper_id': '317943dec06b88c1c62ef0ecd832f0d644b1d417',\n","  'section': 'Abstract',\n","  'text': 'The novel coronavirus , first detected in Wuhan, China in December 2019, has spread to 28 countries/regions with over 43,000 confirmed cases. Much about this outbreak is still unknown. At this early stage of the epidemic, it is important to investigate alternative sources of information to understand its dynamics and spread. With updated real time domestic traffic, this study aims to integrate recent evidence of international evacuees extracted from Wuhan between Jan. 29 and Feb. 2, 2020 to infer the dynamics of the COVD-19 outbreak in Wuhan. In addition, a modified SEIR model was used to evaluate the empirical support for the presence of asymptomatic transmissions. Based on the data examined, this study found little evidence for the presence of asymptomatic transmissions. However, it is still too early to rule out its presence conclusively due to sample size and other limitations. The updated basic reproductive number was found to be 2.12 on average with a 95% credible interval of [2.04, 2.18]. It is smaller than previous estimates probably because the new estimate factors in the social and non-pharmaceutical mitigation implemented in Wuhan through the evacuee dataset. Detailed predictions of infected individuals exported both domestically and internationally were produced. The estimated case confirmation rate has been low but has increased steadily to 23.37% on average. The findings of this study depend on the validity of the underlying assumptions, and continuing work is needed, especially in monitoring the current infection status of Wuhan residents. '},\n"," {'paper_id': '31816b72a378debeb42e76ff7ff2af69727cd049',\n","  'section': 'Abstract',\n","  'text': 'BACKGROUND Nucleic acid test and antibody assay have been employed in the diagnosis for SARS-CoV-2 infection, but the use of viral antigen for diagnosis has not been successfully developed. Theoretically, viral antigen is the specific marker of the virus and precedes antibody appearance within the infected population. There is a clear need of detection of viral antigen for rapid and early diagnosis. METHODS We included a cohort of 239 participants with suspected SARS-CoV-2 infection from 7 centers for the study. We measured nucleocapsid protein in nasopharyngeal swab samples in parallel with the nucleic acid test. Nucleic acid test was taken as the reference standard, and statistical evaluation was taken in blind. We detected nucleocapsid protein in 20 urine samples in another center, employing nasopharyngeal swab nucleic acid test as reference standard. We developed a fluorescence immunochromatographic assay for detecting nucleocapsid protein of SARS-CoV-2 in nasopharyngeal swab sample and urine within 10 minutes. 100% of nucleocapsid protein positive and negative participants accord with nucleic acid test for same samples. Further, earliest participant after 3 days of fever can be identified by the method. In an additional preliminary study, we detected nucleocapsid protein in urine in 73.6% of diagnosed COVID-19 patients. CONCLUSIONS Those findings indicate that nucleocapsid protein assay is an accurate, rapid, early and simple method for diagnosis of COVID-19. Appearance of nucleocapsid protein in urine coincides our finding of the SARS-CoV-2 invading kidney and might be of diagnostic value. '},\n"," {'paper_id': '31e67053158e68f513214a7a33046b5cc5f26249',\n","  'section': 'Abstract',\n","  'text': '25 Nutritional stress, and especially a dearth of pollen, is considered an important factor associated 26 with honey bee colony losses. We used pollen-restricted colonies as a model to study the 27 nutritional stress conditions experienced in colonies within intensively cultivated agricultural 28 areas. This model was complemented by the establishment of an experimental design, which 29 allowed us to uncouple the effect of nutrition, behavior and age in colonies of similar size and 30 demography. We used this system to determine the effect of pollen restriction on workers\\' 31 behavioral development. Then, we analyzed the effect of nutritional stress, behavior and age on 32 the expression of key physiological genes involved in the regulation of division of labor. Finally, 33 we analyzed the effects of these variables on the expression of immune genes and the titers of 34 honey bee viruses. Our results show that pollen restriction led to an increased number of 35 precocious foragers and this behavioral transition was associated with important changes in the 36 expression of nutritionally regulated physiological genes, immunity and viral titers. Vitellogenin 37 (vg) and major royal jelly protein 1 (mrjp1) were the most predictive markers of nutrition and 38 behavior. The expression of immune genes was primarily affected by behavior, with higher levels 39 in foragers. Deformed wing virus (DWV) titers were significantly affected by behavior and 40 nutritional status, with higher titer in foragers and increased levels associated with pollen 41 ingestion. Correlation analyses support the predominant effect of behavior on immunity and 42 susceptibility to viral infection, revealing that both immune genes and DWV exhibited strong 43 negative correlations with genes associated with nursing, but positive correlations with genes 44 associated with foraging. Our results provide valuable insights into the physiological mechanisms 45 by which nutritional stress induce precocious foraging and increased susceptibility to viral 46 infections. 48 Approximately 35-40% of the world\\'s crop production comes from plant species that depend on 49 animal pollination [1] which is carried out primarily by honey bees [2]. However, populations of 50 honey bees have experienced a severe decline during recent years worldwide [3, 4]. Possible 51 causes for colony losses have been proposed, including the effects of pesticides [5], nutritional 52 stress [6-8], the parasitic mite Varroa destructor [9, 10] and synergistic interactions between 53 Varroa and honey bee viruses [11]. However, none of these independent factors is consistently 54 associated with colony losses to suggest a single causal agent [3]. Thus, colony losses are likely 55 the result of the combination of several underlying factors [12, 13]. 56 57 There are several lines of evidence indicating that nutritional stress is an important contributing 58 factor for colony losses. First, there is a positive relationship between the area of uncultivated 59 forage land surrounding an apiary and annual colony survival [6, 8] suggesting that habitat loss 60 play a significant role in honeybee colony losses. Second, at individual levels, honey bees from 61 colonies kept in areas of high cultivation exhibited decreased levels of physiological biomarkers 62 associated with nutrition compared with bees kept in areas of low cultivation [7, 14], 63 demonstrating that availability of floral resources affects the nutritional state of honey bees. 64 65 Habitat loss associated with increased use of monocultures reduces both the quality (diversity of 66 plant sources) and the quantity of the pollen collected by the bees [15]. Simultaneous flowering in 67 monocultures provide pollen for a restricted period, and hives surrounded by monocultures 68 presumably experience pollen dearth before and after this period. Pollen is the main source of 69 proteins and lipids for honey bees [16, 17] and pollen from different plants differ in amino acid 70 content [18, 19] and fatty acids [20]. Thus, the consumption of pollen from diverse plant sources 71 increases the probability of obtaining the set of nutritional components (e.g., essential amino 4 72 acids and fatty acids) required for balanced nutrition. 73 74 Plasticity is a key feature of honey bee behavioral development [21], which is important for 75 understanding the consequences of nutritional stress on colony health and fitness. In the honey 76 bee, as in other eusocial insects, behavioral division of labor in workers is initially age-dependent 77 [21, 22]. Under typical spring and summer conditions in temperate climates, honey bee lifespan is 78 approximately one month. During the first two to three weeks of adult life, workers perform tasks 79 in the hive such as brood care (\"nursing\"); while in the last one to two weeks of life, workers 80 forage for food outside the colony [23, 24]. However, honey bee behavioral development is 81 flexible and can be accelerated or delayed according to colony demography and as an adaptive 82 response to different challenges [21]. In particular, the timing of nurse to forager transition is 83 normally accelerated in response to diverse types of stressors, including diseases [25] and 84 malnutrition [26]. 85 86 Interactions among key elements of the extended insulin/insulin-like growth signaling/target of 87 rapamycin (IIS/TOR) pathway including vitellogenin (Vg), juvenile hormone (JH) and insulin-88 like peptides (ILPs), regulate important physiological processes in honey bees, including 89 behavioral division of labor [27-31]. Vg is a yolk protein highly expressed in the fat bodies [29], 90 secreted into the hemolymph and then imported by developing oocytes [32]. JH, one of the major 91 insect hormones, is synthesized in the corpora allata in a final reaction catabolized by the enzyme 92 methyl farnesoate epoxidase (MFE) [33]. There are two genes encoding insulin genes (ilp1 and 93 ilp2) in the honey bee genome [34], which are expressed in several tissues and organs including 94 the brain and fat bodies [27, 29, 30]. In workers, negative interactions among vg with JH and 95 insulin like peptides are associated with task performance: while vg levels are higher in nurses 96 compared with foragers [29, 35-37], JH titers [21, 36, 38, 39] brain ilp1 and fat body ilp2 mRNA 5 97 levels follow an opposite pattern [27, 29, 30]. Experimental manipulations showed that these 98 correlations reveal causative positive and negative interactions among these three regulatory 99 elements. Thus, JH treatments inhibit vg expression [35, 40], but induce higher brain ilp1 levels 100 [29] and precocious foraging [21, 41]. Consistently, experimental reduction of vg by RNA 101 interference also result in higher levels of JH [27] and precocious foraging [28]. 102 103 In recent years, the proposal that nutrition is an important factor regulating the division of labor 104 among social insects has received increased support [26, 30, 42-46]. Pollen consumption is low in 105 young bees, largest in about 9-day-old nurses and then declines to minimal amounts in foragers 106 [47], which rely primarily on honey intake. Cage experiments showed that 6-day-old workers 107 eating pollen and sugar have higher abdominal vg and low brain ilp1 levels, while same-age bees 108 fed with only sugar had the inverse pattern [30]; resembling the expression profiles observed in 109 colony-reared nurses and foragers, respectively [29]. The effect of pollen on behavioral 110 development could result from its protein content. In support of this view, it has been shown that 111 increased levels of hemolymph amino acids, induced by protein injection, resulted in high vg 112 mRNA levels and delayed foraging [27]. Additionally, lipid metabolism also has been involved 113 in nutritional regulation of division of labor. Lipid content closely correlates with task 114 performance, being higher in nurses compared to foragers [43]. Feeding of an inhibitor of fatty 115 acid synthesis (TOFA) induced precocious foraging [48], further supporting a role of lipids on 116 the regulation of division of labor. However, it remains to be tested whether supplementary lipids 117 (e.g., essential fatty acids) have an effect on the nutritional regulation of this process. 118 119 Since genes that regulate behavioral division of labor are nutritionally regulated, they could be 120 expected to be ideal biomarkers of both nutritional and behavioral state in cage as well as colony-121 level studies. Progressive evidence is revealing that behavioral and nutritional states influence 6 122 honey bee immunity and susceptibility to pathogens. Thus, the use of reliable markers for these 123 traits can also facilitate effective disease diagnosis. Indeed, the initial use of vg as a predictive 124 marker of colony losses [49] and colony nutritional state [14] provides a promising starting point 125 toward this direction. 126 127 Although insects lack an acquired immunity, they have developed an efficient innate immune 128 system against a broad variety of pathogens [50]. Innate immunity has been traditionally divided 129 into humoral and cellular responses, but there are complex interactions and no clearly defined 130 boundaries between both types of immune responses [51]. Four main gene pathways mediate 131 insect innate immunity: Toll, IMD, JNK and JAK/ STAT, which include elements implicated in 132 both the humoral and cellular immune responses and belong to gene families involved in 133 signaling, pathogen recognition and effector functions [52, 53]. 134 135 Humoral immune responses are mainly associated with antimicrobial effectors belonging to the 136 Toll pathway such as abaecin [54], defensin1 (def1) [55] and hymenoptaecin (hym) [56]. On the 137 other hand, cellular immunity is mediated by haemocytes and involves responses including 138 phagocytosis, nodulation and encapsulation [57] While phagocytosis is mediated by pathogen 139 recognition proteins (e.g., peptidoglycan recognition proteins L and S) [58] and transmembrane 140 proteins such as eater [59], nodulation and encapsulation are often accompanied by melanization, 141 a process catalyzed by the pro-phenoloxidase (PPO) and pro-phenoloxidase activator (PPOact) 142 enzymes [52, 60, 61]. 143 144 Different studies have revealed interactions among nutrition, immunity and physiological changes 145 associated with behavioral development. First, nutrition affects the expression of immune genes 146 [62-64] and susceptibility to different pathogens [65-68]. Second, infections of pathogens and 8 171 almost 3X higher when compared with P-UR colonies (100 and 35, respectively; P= <0.0001). 172 Analysis of nectar and pollen foragers in each nutritional group showed higher numbers of both 173 types of foragers in P-R compared with P-UR colonies (nectar: P-R=53, P-UR-24, P= <0.0001; 174 pollen: P-R=47, P-UR=11, P= <0.0001). While no significant differences were found between the 175 number of nectar and pollen foragers in P-R colonies (P=0.52), the difference between both types 176 of foragers in P-UR colonies was highly significant (P=0.0027) with a lower number of new 177 pollen foragers than nectar foragers under this condition ( Figure 1A and Figures S2 and S3 ). 178 179 2. Brood production. 180 Two days after the end of the behavioral observations, and one day after the last collection of 3-181 week-old workers, newly-emerged bees were counted on frames from P-R and P-UR colonies 182 until the last bees emerged after 16 days. The cumulative number of newly-emerged bees from P-183 R colonies was significantly lower compared to P-UR colonies (2573 and 3881, respectively. 184 P=0.014) (Figure 1B). 185 186 Figure 1. Effects of pollen restriction on foraging and brood production. Values were pooled 187 from eight pollen restricted (P-R) and eight pollen un-restricted colonies (P-U). The upper panel 188 (A) shows the effect of nutrition on the average number of new foragers per day. The y-axis 189 indicates the number of nectar foragers, pollen foragers and total foragers observed per day (sum 190 of 3 periods of 15 min each). Nutritional groups plotted in the x-axis are as follow: pollen-191 restricted colonies (R), pollen-unrestricted colonies (UR). The lower panel (B) compares the total 192 number of bees born (y-axis) from colonies in both nutritional conditions (x-axis). Error bars 193 represent SE across colonies. Significant differences in the total number of foragers between R 194 and UR colonies are represented with black stars. Differences in pollen or nectar foragers 195 between nutritional conditions are represented with blue stars. 196 197 198 199 200 9 201 3. Behavioral, nutritional and age analyses of gene expression and virus titers 202 Genes associated with physiological activity 203 204 Vg expression was significantly affected by behavior, with higher levels in nurses in all 205 nutritional and age groups (P= <0.0001 for all behavioral comparisons). Expression levels in 206 nurses were on average 3.75X higher compared with foragers. (Figure 2A) . Nutritional 207 treatments also had a significant effect on vg expression, with higher levels in nurses from 208 colonies with unrestricted access to pollen (P-UR) compared with nurses from colonies with 209 restricted access to pollen (P-R) in both age groups (2-week-old: P= <0.0001; 3-week-old: 210 P=0.0075). Foragers also showed higher levels of vg in P-UR colonies compared with P-R 211 colonies, but only at the younger group (P=0.0003). Lastly, age comparisons showed a 212 significant increase in 3-week-old workers in all the nutritional and behavioral comparisons with 213 the exception of foragers from P-UR colonies (Figure 2A and Table S2 ). 215 Mrjp1 expression levels were strongly affected by behavior with higher levels in nurses compared 216 with foragers in all the nutritional and age comparisons. On average, mrjp1 levels in nurses were 217 5X higher compared with foragers. On the other hand, nutritional comparisons showed that pollen 218 consumption induced a significant increase of mrjp1 in younger nurses (P=0.009) and foragers 219 (7.2e-6) compared with workers from P-R colonies. No nutritional differences were observed in 220 older 3-week-old workers. Age comparisons showed a tendency toward higher levels in older 221 foragers, which become significant in all the comparisons with the exception of foragers from P-222 UR colonies. It is interesting to note, that our whole-body analyses reveal that mrjp1 mRNA 223 levels were substantially higher compared with vg, being on average over 500X higher ( Figure   224 2A). 225 10 226 Mfe expression showed behaviorally related changes in 2-week-old workers, with higher levels in 227 nurses compared with foragers (P-R= 0.015, P-UR=0.016). In contrast, no significant behavioral 228 differences were observed in 3-week-old workers. On the other hand, nutritional comparisons 229 showed significantly higher mfe expression associated with pollen intake exclusively in 3-week-230 old workers (nurses: P=0.035, foragers: P=0.014). Lastly, age comparisons revealed a trend 231 toward down-regulation of mfe expression in older workers. However, significant differences 232 were only found in foragers from P-R colonies (P=0.002). ( Figure 2B ). 234 The expression of ilp1 revealed complex interactions in our behavioral and nutritional analyses. 235 The behavioral analysis showed significantly higher levels in younger foragers from P-UR 236 colonies (P=0.033) and in older nurses from P-R colonies (P=0.034). On the other hand, 237 nutritional comparisons exhibited higher levels in 2-week-old foragers from P-UR colonies and 238 3-week-old nurses from P-R colonies (2-week-old: nurses P=0.61, foragers P=0.014; 3-week-old 239 nurses P=0.01, foragers P=0.30). In contrast with behavior and nutritional analyses, age 240 comparisons revealed a significant down-regulation in the expression of ilp1 in most of the 241 nutritional and behavioral groups of older workers (P-R nurses: P=0.058, P-R foragers: P= 242 <0.0001, P-UR nurses: P= 0.0001, P-UR foragers: P= 0.0005) ( Figure 2B ). 244 Expression of ilp2 showed behavioral differences only in younger workers, with higher levels in 245 foragers both from P-R (P=0.0001) and P-UR (P=0.006) colonies. Nutritional comparisons 246 showed a trend toward higher ilp2 levels with pollen consumption in 2-week-old workers, but the 247 differences were marginally significant (P-R=0.08, P-UR=0.08) (Table S2 ). In contrast, higher 248 expression of ilp2 was associated with pollen restriction in both 3-week-old nurses (P=0.0001) 249 and foragers (P=0.037). Age comparisons revealed higher ilp2 expression in workers from 250 colonies experiencing pollen restriction (nurses P= <0.0001, foragers P=0.001). Additionally, our '},\n"," {'paper_id': '31fb22d877d6efa55e7c8261a59a1f0d35962829',\n","  'section': 'Abstract',\n","  'text': 'To mitigate and slow the spread of COVID-19, many countries have adopted unprecedented physical distancing policies, including the UK. We evaluate whether these measures might be sufficient to control the epidemic by estimating their impact on the reproduction number (R0, the average number of secondary cases generated per case). We asked a representative sample of UK adults about their contact patterns on the previous day. The questionnaire documents the age and location of contacts and as well as a measure of their intimacy (whether physical contact was made or not). In addition, we asked about adherence to different physical distancing measures. The first surveys were sent on Tuesday 24th March, one day after a \"lockdown\" was implemented across the UK. We compared measured contact patterns during the \"lockdown\" to patterns of social contact made during a non-epidemic period. By comparing these, we estimated the change in reproduction number as a consequence of the physical distancing measures imposed. We used a meta-analysis of published estimates to inform our estimates of the reproduction number before interventions were put in place. We found a 73% reduction in the average daily number of contacts observed per participant (from 10.2 to 2.9). This would be sufficient to reduce R0 from 2.6 prior to lockdown to 0.62 (95% confidence interval [CI] 0.37 -0.89) after the lockdown, based on all types of contact and 0.37 (95% CI = 0.22 -0.53) for physical contacts only. The physical distancing measures adopted by the UK public have substantially reduced contact levels and will likely lead to a substantial impact and a decline in cases in the coming weeks. However, this projected decline in incidence will not occur immediately as there are significant delays between infection, the onset of symptomatic disease and hospitalisation, as well as further delays to these events being reported. Tracking behavioural change can give a more . CC-BY-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': '3298d34e643c7a157f4226fde679f71194e417b5',\n","  'section': 'Abstract',\n","  'text': 'Background: the COVID-19 has been successfully contained in China but is spreading all over the world. We use phenomenological models to dissect the development of the epidemics in China and the impact of the drastic control measures both at the aggregate level and within each province. We use the experience from China to analyze the calibration results on Japan, South Korea, Iran, Italy and Europe, and make future scenario projections. : we calibrate the logistic growth model, the generalized logistic growth model, the generalized growth model and the generalized Richards model to the reported number of infected cases from Jan. 19 to March 10 for the whole of China, 29 provinces in China, four severely affected countries and Europe as a whole. The different models provide upper and lower bounds of our scenario predictions. Results: We quantitatively document four phases of the outbreak in China with a detailed analysis on the heterogenous situations across provinces. Based on Chinese experience, we identify a high risk in Japan with estimated total confirmed cases as of March 25 being 1574 (95% CI: [880, 2372]), and 5669 (95% CI: [988, 11340]) by June. For South Korea, we expect the number of infected cases to approach the ceiling, 7928 (95% CI: [6341, 9754]), in 20 days. We estimate 0.15% (95% CI: [0.03%, 0.30%]) of Italian population to be infected in a positive scenario. We would expect 114867 people infected in Europe in 10 days, in a negative but probable scenario, corresponding to 0.015% European population. '},\n"," {'paper_id': '329dfdd2fbe4b3c510c5329219bfd33764ab1710',\n","  'section': 'Abstract',\n","  'text': \"population. We report that domestication of the model yeast S. cerevisiae reprogrammed its life cycle entirely. We tracked growth, gamete formation and cell survival across many environments for nearly 1000 genome sequenced isolates and found a remarkable dichotomy between domesticated and wild yeasts. Wild yeasts near uniformly trigger meiosis and sporulate when encountering nutrient depletions, whereas domestication relaxed selection on sexual reproduction and favoured survival as quiescent cells. Domestication also systematically enhanced fermentative over respiratory traits while decreasing stress tolerance. We show that this yeast domestication syndrome was driven by aneuploidies and gene function losses that emerged independently in multiple domesticated lineages during the specie's recent evolutionary history. We found domestication to be the most dramatic event in budding yeast evolution, raising questions on how much domestication has distorted our understanding of this key model species. \"},\n"," {'paper_id': '32c2e402c3c88c3024a4dec3d78ba67c24f89ca6',\n","  'section': 'Abstract',\n","  'text': \"Bats host virulent zoonotic viruses without experiencing disease. A mechanistic understanding of 25 the impact of bats' virus hosting capacities, including uniquely constitutive immune pathways, 26 on cellular-scale viral dynamics is needed to elucidate zoonotic emergence. We carried out virus 27 infectivity assays on bat cell lines expressing induced and constitutive immune phenotypes, then 28 developed a theoretical model of our in vitro system, which we fit to empirical data. Best fit 29 models recapitulated expected immune phenotypes for representative cell lines, supporting 30 robust antiviral defenses in bat cells that correlated with higher estimates for within-host viral 31 propagation rates. In general, heightened immune responses limit pathogen-induced cellular 32 morbidity, which can facilitate the establishment of rapidly-propagating persistent infections 33 within-host. Rapidly-replicating viruses that have evolved with bat immune systems will likely 34 cause enhanced virulence following emergence into secondary hosts with immune systems that 35 diverge from those unique to bats. 36 37 38 39 40 41 42 43 44 45 46 . CC-BY 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/696195 doi: bioRxiv preprint Recent research advances are beginning to shed light on the molecular mechanisms by 58 which bats avoid pathology from these otherwise virulent pathogens (Brook and Dobson 2015). 59 Bats leverage a suite of species-specific mechanisms to limit viral load, which include host 60 receptor sequence incompatibilities for some bat-virus combinations (Ng et al. 2015; Igarashi et 61 al. 2020) and constitutive expression of the antiviral cytokine, IFN-, for others (Zhou et al. 62 2016). Typically, the presence of viral RNA or DNA in the cytoplasm of mammalian cells will 63 induce secretion of type I interferon proteins (IFN-and IFN-), which promote expression and 64 translation of interferon-stimulated genes (ISGs) in neighboring cells and render them effectively 65 antiviral (Stetson and Medzhitov 2006). In some bat cells, the transcriptomic blueprints for this 66 IFN response are expressed constitutively, even in the absence of stimulation by viral RNA or 67 DNA (Zhou et al. 2016). In non-flying mammals, constitutive IFN expression would likely elicit 68 widespread inflammation and concomitant immunopathology upon viral infection, but bats 69 \"},\n"," {'paper_id': '3319392ca8bd2b8f2e2d00bccdb80deaa8a77c2e',\n","  'section': 'Abstract',\n","  'text': 'The ongoing outbreak of viral pneumonia in China and beyond is associated with a novel 28 coronavirus, provisionally termed 2019-nCoV. This outbreak has been tentatively associated 29 with a seafood market in Wuhan, China, where the sale of wild animals may be the source of 30 zoonotic infection. Although bats are likely reservoir hosts for 2019-nCoV, the identity of 31 any intermediate host facilitating transfer to humans is unknown. Here, we report the 32 identification of 2019-nCoV related coronaviruses in pangolins (Manis javanica) seized in 33 anti-smuggling operations in southern China. Metagenomic sequencing identified pangolin 34 associated CoVs that belong to two sub-lineages of 2019-nCoV related coronaviruses, 35 including one very closely related to 2019-nCoV in the receptor-binding domain. The 36 discovery of multiple lineages of pangolin coronavirus and their similarity to 2019-nCoV 37 suggests that pangolins should be considered as possible intermediate hosts for this novel 38 human virus and should be removed from wet markets to prevent zoonotic transmission. '},\n"," {'paper_id': '332c2d84e64da921eae3b8bad2bd9d3df129ea01',\n","  'section': 'Abstract',\n","  'text': 'COVID-19 is caused by SARS-CoV-2, and has become a global pandemic. There is no highly effective medicine or vaccine, most of the patients were recovered by their own immune response, especially the virus specific IgG and IgM responses. However, the IgG/ IgM responses is barely known. To enable the global understanding of SARS-CoV-2 specific IgG/ IgM responses, a SARS-CoV-2 proteome microarray with 18 out of the 28 predicted proteins was constructed. The microarray was applied to profile the IgG/ IgM responses with 29 convalescent sera. The results suggest that at the convalescent phase 100% of patients had IgG/ IgM responses to SARS-CoV-2, especially to protein N, S1 but not S2. S1 purified from mammalian cell demonstrated the highest performance to differentiate COVID-19 patients from controls. Besides protein N and S1, significant antibody responses to ORF9b and NSP5 were also identified. Indepth analysis showed that the level of S1 IgG positively correlate to age and the level of LDH (lactate dehydrogenase), especially for women, while the level of S1 IgG negatively correlate to Ly% (Lymphocyte percentage). This study presents the first whole picture of the SARS-CoV-2 specific IgG/ IgM responses, and provides insights to develop precise immuno-diagnostics, effective treatment and vaccine. '},\n"," {'paper_id': '3350501191fd08b8b2566ad42fa0b293998be07c',\n","  'section': 'Abstract',\n","  'text': 'The recent pandemic created due to Novel Coronavirus (nCOV-2019) from Wuhan, China demanding a large scale of a general health emergency. This demands novel research on the vaccine to fight against this pandemic situation, re-purposing of the existing drugs, phylogenetic analysis to identify the origin and determine the similarity with other known viruses, etc. The very preliminary task from the research community is to analyze the wide verities of existing related research articles, which is very much time-consuming in such situations where each minute counts for saving hundreds of human lives. The entire manual processing is even lower down the efficiency in mining the information. We have developed a complete automatic literature mining system that delivers efficient and fast mining from existing biomedical literature databases. With the help of modern-day deep learning algorithms, our system also delivers a summarization of important research articles that provides ease and fast comprehension of critical research articles. The system is currently scanning nearly 1,46,115,136 English words from 29,315 research articles in not greater than 1.5 seconds with multiple search keywords. Our research article presents the criticality of literature mining, especially in pandemic situations with the implementation and online deployment of the system. '},\n"," {'paper_id': '33565294e6bc67fb7ee14dcae6cfdb08148f4ea5',\n","  'section': 'Abstract',\n","  'text': 'Abstract: 25 26 Macroscopic descriptions of populations commonly assume that encounters between 27 individuals are well mixed; i.e., each individual has an equal chance of coming into contact with 28 any other individual. Relaxing this assumption can be challenging though, due to the difficulty 29 of acquiring detailed knowledge about the non-random nature of encounters. Here, we fitted a 30 mathematical model of dengue virus transmission to spatial time series data from Pakistan and 31 compared maximum-likelihood estimates of \"mixing parameters\" when disaggregating data 32 across an urban-rural gradient. We show that dynamics across this gradient are subject not 33 only to differing transmission intensities but also to differing strengths of nonlinearity due to 34 differences in mixing. We furthermore show that neglecting spatial variation in mixing can lead 35 to substantial underestimates of the level of effort needed to control a pathogen with vaccines 36 or other control efforts. We complement this analysis with relevant contemporary environmental 37 drivers of dengue. 38 39 '},\n"," {'paper_id': '3379f2ad661859b90114bf78804f5c11ef84e3d7',\n","  'section': 'Abstract',\n","  'text': 'The pandemic cased by coronavirus COVID-19 is of great concern. A detailed scientific analysis of this phenomenon is still to come, but now it is urgently needed to evaluate the disease dynamics in order to organize the appropriate quarantine activities, to estimate the required number of places in hospitals, the level of individual protection, the rate of isolation of infected persons, etc. South Korea has achieved the stabilization of the number of cases at rather low level. The epidemic dynamics there can be compared with its development in other countries to make some preliminary, but very important conclusions. Here we provide a simple method of data comparison that can be useful for both governmental organizations and anyone. '},\n"," {'paper_id': '33a6aa86bb6fe30251d7f2c1f64c5b7a94c391a0',\n","  'section': 'Abstract',\n","  'text': 'medRxiv preprint SARS-CoV-2 infection. Patients with active or latent TB were more susceptible to SARS-CoV-2, and COVID-19 symptom development and progression were more rapid and severe. Meaning: Tuberculosis status should be assessed carefully at patient admission and management and therapeutic strategies adjusted accordingly to prevent rapid development of severe COVID-19 complications. Importance: Risk factors associated with COVID-19, the viral pneumonia originating in Wuhan, China, in Dec 2019, require clarification so that medical resources can be prioritized for those at highest risk of severe COVID-19 complications. Infection with M. tuberculosis (MTB), the pathogen that causes TB and latently infects ~25% of the global population, may be a risk factor for SARS-CoV-2 infection and severe COVID-19 pneumonia. Objective: To determine if latent or active TB increase susceptibility to SARS-COV-19 infection and disease severity, and lead to more rapid development of COVID-19 pneumonia. '},\n"," {'paper_id': '33bd85ea9ca61c4d0674c43ae5b901b271b46f1e',\n","  'section': 'Abstract',\n","  'text': 'We give an update to the original paper posted on 2/17/20 -now (as of 3/1/20) the China deaths are rapidly decreasing, and we find an exponential decline to the power law similar to the that predicted by the network model of Vazquez [2006]. At the same time, we see non-China deaths increasing rapidly, and similar to the early behavior of the China statistics. Thus, we see three stages of the spread of the disease in terms of number of deaths: exponential growth, power-law behavior, and then exponential decline in the daily rate. (Original abstract) The novel coronavirus (COVID-19) continues to grow rapidly in China and is spreading in other parts of the world. The classic epidemiological approach in studying this growth is to quantify a reproduction number and infection time, and this is the approach followed by many studies on the epidemiology of this disease. However, this assumption leads to exponential growth, and while the growth rate is high, it is not following exponential behavior. One approach that is being used is to simply keep adjusting the reproduction number to match the dynamics. Other approaches use rate equations such as the SEIR and logistical models. Here we show that the current growth closely follows power-law kinetics, indicative of an underlying fractal or small-world network of connections between susceptible and infected individuals. Positive deviations from this growth law might indicate either a failure of the current containment efforts while negative deviations might indicate the beginnings of the end of the pandemic. We cannot predict the ultimate extent of the pandemic but can get an estimate of the growth of the disease. '},\n"," {'paper_id': '33eff3ff8721d2abd9204e3aabd504e167f3f01d',\n","  'section': 'Abstract',\n","  'text': 'Background: To control COVID-19 outbreak in Japan, sports and entertainment events were canceled in Japan for two weeks from 26 February to 11 March. It has been designated as voluntary event cancellation (VEC). Object: This study predicts the effectiveness of VEC enduring and after its implementation. Method: We applied a simple susceptible-infected-recovery model to data of patients with symptoms in Japan during 14 January to VEC introduction and after VEC introduction to 8 March. We adjusted the reporting delay in the latest few days. Results: Results suggest that the basic reproduction number, R 0, before VEC introduced as 2.50 with a 95% confidence interval (CI) was [2.43, 2.55] and the effective reproduction number, R v, after VEC introduced as 1. 88; its 95% CI was [1.68,2.02]. Discussion and Conclusion: Results demonstrated that VEC can reduce COVID-19 infectiousness by 35%, but R 0 remains higher than one. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '342100efd557117568d971aa59f162620f3b9fcb',\n","  'section': 'Abstract',\n","  'text': 'Following the recent report of a furin-like cleavage site unique to SARS-CoV-2, we characterized 23 additional amino acid residues, positioned within the crown of the spike (S) glycoprotein, under 24 diversifying selection along the ancestral lineage of currently circulating pandemic strains, and 25 affecting enhanced ability to infect host cells. Molecular dynamic simulations revealed long-26 range covariant movements of SARS-CoV-2 S glycoprotein monomer in pre-fusion conformation 27 with furin cleavage, and residue T333, under purifying selection, hinge of such movement. 28 Evolutionary analysis revealed that the current human lineage likely emerged after a 29 recombination event between ancestors in bat and pangolin. Recombination impacted residues 30 . CC-BY-NC-ND 4.0 : bioRxiv preprint 109 movement between CTD1 (residues between red lines in Figure 1f) and the region 828-852 110 (highlighted by cyan lines in Figure 1 f), encompassing a portion of the fusion peptide (Suppl. 111 Movie 3, in which residues 828-852 are highlighted in cyan). This co-variant movement between 112 CTD1 and the fusion peptide region is indicative of long range coordinated movements, functional 113 to the protein rearrangement during the fusion step (Figure 1f, Suppl. Movie 3). A correlated '},\n"," {'paper_id': '343ac900b17776cfd86c294269adf66f94b7b197',\n","  'section': 'Abstract',\n","  'text': \"The legal behaviors in using medical masks in public have been finally promulgated by the Vietnamese Government after 47 days since the WHO declared the Public Health Emergency of International Concern (PHEIC) due to the COVID-19 pandemic. From a sample of 345 Vietnamese respondents aged from 15 to 47 years, this brief note found that the risk perception of COVID-19 danger significantly increases the likelihood of wearing the medical masks. In addition, there is a weak evidence about the differences in age under the COVID-19 outbreaks. More noticeably, those who use masks before COVID-19 pandemic tend to maintain their behaviors. Our results offer the insightful into Vietnamese citizens' responses in terms of using medical masks; even the uses of this method are still controversial. Our results are robust by performing Exploratory Factor Analysis for five features and further regressions. \"},\n"," {'paper_id': '3461d71f6890f7e5ba53bf168be3945cdb16d901',\n","  'section': 'Abstract',\n","  'text': 'Recombination is a process that unlinks neighbouring loci allowing for independent evolutionary trajectories within genomes of many organisms. If not properly accounted for, recombination can compromise many evolutionary analyses. In addition, when dealing with organisms that are not obligately sexually reproducing, recombination gives insight into the rate at which distinct genetic lineages come into contact. Since June, 2012, Middle East respiratory syndrome coronavirus (MERS-CoV) has caused 1106 laboratory-confirmed infections, with 421 MERS-CoV associated deaths as of April 16, 2015. Although bats are considered as the likely ultimate source of zoonotic betacoronaviruses, dromedary camels have been consistently implicated as the source of current human infections in the Middle East. In this paper we use phylogenetic methods and simulations to show that MERS-CoV genome has likely undergone numerous recombinations recently. Recombination in MERS-CoV implies frequent co-infection with distinct lineages of MERS-CoV, probably in camels given the current understanding of MERS-CoV epidemiology. We use the Genetic Algorithm for Recombination Detection (GARD) method (Kosakovsky Pond et al., 2006) , as implemented in the software package HyPhy (Pond et al., 2005) , to look for alternative tree topologies in sequence data. Briefly, the method compares a model where a single tree is derived from the whole alignment and alternative models where breakpoints are introduced into the alignment and phylogenetic trees are derived independently 4 . CC-BY-NC 4.0 International license is made available under a '},\n"," {'paper_id': '348db141e734a5d46bf9cea528a53c996f92e92f',\n","  'section': 'Abstract',\n","  'text': 'Purpose: COVID-19 has become global threaten. CT acts as an important method of 60 diagnosis. However, human-based interpretation of CT imaging is time consuming. 61 More than that, substantial inter-observer-variation cannot be ignored. We aim at 62 developing a diagnostic tool for artificial intelligence (AI)-based classification of CT 63 images for recognizing COVID-19 and other common infectious diseases of the lung. 64 Experimental Design: In this study, images were retrospectively collected and 65 prospectively analyzed using machine learning. CT scan images of the lung that show 66 or do not show COVID-19 were used to train and validate a classification framework 67 based on convolutional neural network. Five conditions including COVID-19 68 pneumonia, non-COVID-19 viral pneumonia, bacterial pneumonia, pulmonary 69 tuberculosis, and normal lung were evaluated. Training and validation set of images 70 were collected from Wuhan Jin Yin-Tan Hospital whereas test set of images were 71 collected from Zhongshan Hospital Xiamen University and the fifth Hospital of 72 Wuhan. 73 Results: Accuracy, sensitivity, and specificity of the AI framework were reported. For 74 test dataset, accuracies for recognizing normal lung, COVID-19 pneumonia, 75 non-COVID-19 viral pneumonia, bacterial pneumonia, and pulmonary tuberculosis 76 were 99.4%, 98.8%, 98.5%, 98.3%, and 98.6%, respectively. For the test dataset, 77 accuracy, sensitivity, specificity, PPV, and NPV of recognizing COVID-19 were 78 98.8%, 98.2%, 98.9%, 94.5%, and 99.7%, respectively. 79 All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '3493838bbb857a10f5f964dd57ba913bf9a26ce3',\n","  'section': 'Abstract',\n","  'text': 'The COVID-19 global pandemic is an unprecedented health emergency. Insufficient access to testing has hampered effective public health interventions and patient care management in a number of countries. Furthermore, the availability of regulatory-cleared reagents has challenged widespread implementation of testing. We rapidly developed a qRT-PCR SARS-CoV-2 detection assay using a 384-well format and tested its analytic performance across multiple nucleic acid extraction kits. Our data shows robust analytic accuracy on residual clinical biospecimens. Limit of detection sensitivity and specificity was confirmed with currently available commercial reagents. Our methods and results provide valuable information for other high-complexity laboratories seeking to develop effective, local, laboratory-developed procedures with high-throughput capability to detect SARS-CoV-2. '},\n"," {'paper_id': '34ae2188056cf9334b2303715748493ae10e42f5',\n","  'section': 'Abstract',\n","  'text': 'Fast testing can help mitigate the coronavirus disease 2019 pandemic. Despite their accuracy for single sample analysis, infectious diseases diagnostic tools, like RT-PCR, require substantial resources to test large populations. We develop a scalable approach for determining the viral status of pooled patient samples. Our approach converts group testing to a linear inverse problem, where false positives and negatives are interpreted as generated by a noisy communication channel, and a message passing algorithm estimates the illness status of patients. Numerical results reveal that our approach estimates patient illness using fewer pooled measurements than existing noisy group testing algorithms. Our approach can easily be extended to various applications, including where false negatives must be minimized. Finally, in a Utopian world we would have collaborated with RT-PCR experts; it is difficult to form such connections during a pandemic. We welcome new collaborators to reach out and help improve this work! Index Terms-Approximate '},\n"," {'paper_id': '35604e3a5d38d6794543372e4dd9ccf1d8d9cd2a',\n","  'section': 'Abstract',\n","  'text': 'The emergence and outbreak of SARS-CoV-2, the causative agent of COVID-19, has rapidly become a global concern and has highlighted the need for fast, sensitive, and specific tools to surveil circulating viruses. Here we provide assay designs and experimental resources, for use with CRISPR-based nucleic acid detection, that could be valuable for ongoing surveillance. We provide assay designs for detection of 67 viral species and subspecies, including: SARS-CoV-2, phylogenetically-related viruses, and viruses with similar clinical presentation. The designs are outputs of algorithms that we are developing for rapidly designing nucleic acid detection assays that are comprehensive across genomic diversity and predicted to be highly sensitive and specific. Of our design set, we experimentally screened 4 SARS-CoV-2 designs with a CRISPR-Cas13 detection system and then extensively tested the highest-performing SARS-CoV-2 assay. We demonstrate the sensitivity and speed of this assay using synthetic targets with fluorescent and lateral flow detection. Moreover, our provided protocol can be extended for testing the other 66 provided designs. Assay designs are available at https://adapt.sabetilab.org/ . '},\n"," {'paper_id': '3599c933334227fd95014d36baa4b87e41ffd1e1',\n","  'section': 'Abstract',\n","  'text': 'Background: The first case of COVID-19 atypical pneumonia was reported in Wuhan, China on December 1, 2019. Since then, at least 33 other countries have been affected and there is a possibility of a global outbreak. A tremendous amount of effort has been made to understand its transmission dynamics; however, the temporal and spatial transmission heterogeneity and changing epidemiology have been mostly ignored. The epidemic mechanism of COVID-19 remains largely unclear. Methods: Epidemiological data on COVID-19 in China and daily population movement data from Wuhan to other cities were obtained and analyzed. To describe the transmission dynamics of COVID-19 at different spatio-temporal scales, we used a three-stage continuous-time Susceptible-Exposed-Infectious-Recovered (SEIR) meta-population model based on the characteristics and transmission dynamics of each stage: 1) local epidemic from December 1, 2019 to January 9, 2020; 2) long-distance spread due to the Spring Festival travel rush from January 10 to 22, 2020; and 3) intra-provincial transmission from January 23, 2020 when travel restrictions were imposed. Together with the basic reproduction number (ܴ ) for mathematical modelling, we also considered the variation in infectivity and introduced the controlled reproduction number (ܴ ) by assuming that exposed individuals to be infectious; we then simulated the future spread of COVID across Wuhan and all the provinces in mainland China. In addition, we built a novel source tracing algorithm to infer the initial exposed number of individuals in Wuhan on January 10, 2020, to estimate the number of infections early during this epidemic. The spatial patterns of disease spread were heterogeneous. The estimated controlled reproduction number (ܴ ) in the neighboring provinces of Hubei province were relatively large, and the nationwide reproduction number − except for Hubei − ranged from 0.98 to 2.74 with an average of 1.79 (95% CI 1.77−1.80). Infectivity was significantly greater for exposed than infectious individuals, and exposed individuals were predicted to have become the major source of infection after January 23. For the . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': '35ac99bfeb665ce0a2ae361dba1161e69eb8b7d5',\n","  'section': 'Abstract',\n","  'text': 'Natural selection acting on synonymous mutations in protein-coding genes influences genome composition and evolution. In viruses, introducing synonymous mutations in genes encoding structural proteins can drastically reduce viral growth, providing a means to generate potent, live attenuated vaccine candidates. However, an improved understanding of what compositional features are under selection and how combinations of synonymous mutations affect viral growth is needed to predictably attenuate viruses and make them resistant to reversion. We systematically recoded all non-overlapping genes of the bacteriophage ΦX174 with codons rarely used in its E. coli host. The fitness of recombinant viruses decreases as additional deoptimizing mutations are made to the genome, although not always linearly, and not consistently across genes. Combining deoptimizing mutations may reduce viral fitness more or less than expected from the effect size of the constituent mutations and we point out difficulties in untangling correlated compositional features. We test our model by optimizing the same genes and find that the relationship between codon usage and fitness does not hold for optimization, suggesting that wild-type ΦX174 is at a fitness optimum. This work highlights the need to better understand how selection acts on patterns of synonymous codon usage across the genome and provides a convenient system to investigate the genetic determinants of virulence. : bioRxiv preprint host-phage relationships that are too short-lived for selection to fine tune codon usage in the phage, or inadequate understanding of the features under selection. Empirically developed (e.g. serial passage viral adaptation) vaccines have saved millions of lives over the last century, yet methodological improvements make rationally designed, recombinant vaccines attractive because they can be rapidly produced and specifically engineered for safety and effectiveness (46-50) . One proposed method of generating recombinant vaccines involves making many synonymous, attenuating changes to viral genomes, i.e. \"deoptimizing\" the viral genes (51) . The recombinant vaccine can be made by either editing the genome of the wild-type virus or by generating a viral genome entirely from synthesized nucleic acids. Synonymous deoptimization offers a potentially efficient and effective way of making vaccines: the protein sequences of recoded vaccines are identical to their target viruses, they replicate in their host to provide prolonged exposure to the antigen, and the introduction of many synonymous changes presumably assures evolutionary robustness, preventing the evolution of virulence by reversion. Poliovirus serves as a very good example for the synonymous recoding strategy. Development of a robust, live-attenuated poliovirus vaccine is desired because in some areas on Earth wild poliovirus and the emergent vaccine-derived polioviruses (cVDPVs) continues to cause concern over the resurgence of poliomyelitis (52-55) . A synthetic poliovirus was assembled in 2002 (56) , codon deoptimized in 2006 (57, 58) , codon pair deoptimized in 2008 (59) , and dinucleotide deoptimized in 2009 (11) . In all cases, attenuated viruses were produced by recoding the P1/capsid region of the genome. In vitro , these viruses replicate slower and produce lower viral titers than wild-type virus. In vivo the codon pair deoptimized strain protected mice against challenge by wild poliovirus (59) . While the mechanism of attenuation is not yet fully elucidated, reduced protein expression of the deoptimized genes is observed (11, 57-59) . These deoptimized poliovirus constructs are genetically stable and remain non-virulent for up to 25 passages in cell culture (57, 59) . The apparent success of building poliovirus vaccine candidates using synonymous recoding led to similar attempts to develop vaccines for influenza, adeno-associated, human immunodeficiency, papilloma, chikungunya, respiratory syncytial, simian immunodeficiency, '},\n"," {'paper_id': '35d36c0a736ae7ca642a8badd24bf4d9be3ca613',\n","  'section': 'Abstract',\n","  'text': 'To identify safe and effective medical countermeasures (e.g., antivirals/antibodies) to address the current outbreak of a novel coronavirus Methods: Comprehensive literature searches were developed by an experienced librarian for MEDLINE, EMBASE, the Cochrane Library, and biorxiv.org/medrxiv.org; additional searches for ongoing trials and unpublished studies were conducted in clinicaltrials.gov and the Global Infectious Diseases and Epidemiology Network (GIDEON). Title/abstract and full-text screening, data abstraction, and risk of bias appraisal were carried out by single reviewers. Results: 54 studies were included in the review: three controlled trials, 10 cohort studies, seven retrospective medical record/database studies, and 34 case reports or series. These studies included patients with severe acute respiratory syndrome (SARs, n=33), middle east respiratory syndrome (MERS, n=16), COVID-19 (n=3), and unspecified coronavirus (n=2). The most common treatment was ribavirin (n=41), followed by oseltamivir (n=10) and the combination of lopinavir/ritonavir (n=7). Additional therapies included broad spectrum antibiotics (n=30), steroids (n=39) or various interferons (n=12). No eligible studies examining monoclonal antibodies for COVID-19 were identified. One trial found that ribavirin prophylactic treatment statistically significantly reduced risk of MERS infection in people who had been exposed to the virus. Of the 21 studies reporting rates of ICU admission in hospitalized SARS or MERS patients, none reported statistically significant results in favour of or against antiviral therapies. Of the 40 studies reporting mortality rates in hospitalized SARS or MERS patients, one cohort study (MERS) and one retrospective study (SARS) found a statistically significant increase in the mortality rate for patients treated with ribavirin. Eighteen studies reported potential drugrelated adverse effects including gastrointestinal symptoms, anemia, and altered liver function in patients receiving ribavirin. '},\n"," {'paper_id': '3638c712297a30a3ab860eb46808ecf6684693d8',\n","  'section': 'Abstract',\n","  'text': \"Background. Italy currently constitutes the epicenter of the novel coronavirus disease (COVID-19) pandemic, having surpassed China's death toll. The \"},\n"," {'paper_id': '363bfe1af3cb9b684a80849c0be69e1e2478c421',\n","  'section': 'Abstract',\n","  'text': 'The presence of national policies for universal BCG vaccination is associated with flattened growth curves for confirmed cases of COVID-19 infection and resulting deaths in the first 30-day period of country-wise outbreaks. Prior work suggests that BCG vaccination reduces the risk of different infectious diseases. BCG vaccination may thus serve as a protective factor against COVID-19. Here, we drew on day-byday reports of both confirmed cases and deaths and analyzed growth curves in countries that mandate BCG policies versus countries that do not. Linear mixed models revealed that the presence of mandated BCG policies was associated with a significant flattening of the exponential increase in both confirmed cases and deaths during the first 30-day period of country-wise outbreaks. This effect held after controlling for median age, gross domestic product per capita, population density, population size, geographic region, net migration rate, and various cultural dimensions (e.g., individualism and the tightness vs. looseness of social norms). Our analysis suggests that mandated BCG vaccination can be effective in the fight against COVID-19. '},\n"," {'paper_id': '36521caf90f471c9da1a4e84f8562440d73ead9a',\n","  'section': 'Abstract',\n","  'text': 'The 2019 novel Coronavirus (COVID-19) emerged in Wuhan, China in December 2019 and has been spreading rapidly in China. Decisions about its pandemic threat and the appropriate level of public health response depend heavily on estimates of its basic reproduction number and assessments of interventions conducted in the early stages of the epidemic. '},\n"," {'paper_id': '368fb114dfd565dedb635f62b9062956b49cd475',\n","  'section': 'Abstract',\n","  'text': \"Bacterial biofilms represent an essential part of Earth's ecosystem that can cause multiple ecological, technological and health problems. The environmental resilience and sophisticated organization of biofilms are enabled by the extracellular matrix that creates a protective network of biomolecules around the bacterial community. Current anti-biofilm agents can interfere with extracellular matrix production but, being based on small molecules, are degraded by bacteria and rapidly diffuse away from biofilms. Both factors severely reduce their efficacy, while their toxicity to higher organisms create additional barriers to their practicality. In this paper we report on the ability of graphene quantum dots to effectively disperse mature Staphylococcus aureus biofilms, interfering with the self-assembly of amyloid fibers -a key structural component of the extracellular matrix. Mimicking peptide-binding biomolecules, graphene quantum dots form supramolecular complexes with phenol soluble modulins, the peptide monomers of amyloid fibers. Experimental and computational results show that graphene quantum dots efficiently dock near the N-terminus of the peptide and change the secondary structure of phenol soluble modulins, which disrupts their fibrillation and represents a novel strategy for mitigation of bacterial communities. \"},\n"," {'paper_id': '36a5f6d55d7c5f67d4344e36da0a72856ad3dda0',\n","  'section': 'Abstract',\n","  'text': '124 Main text including references and figure captions: 3,945 All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The novel coronavirus (2019-nCoV) is a recently emerged human pathogen that has spread widely since January 2020. Initially, the basic reproductive number, R0, was estimated to be 2.2 to 2.7. Here we provide a new estimate of this quantity. We collected extensive individual case reports and estimated key epidemiology parameters, including the incubation period. Integrating these estimates and high-resolution real-time human travel and infection data with mathematical models, we estimated that the number of infected individuals during early epidemic double every 2.4 days, and the R0 value is likely to be between 4.7 and 6.6. We further show that quarantine and contact tracing of symptomatic individuals alone may not be effective and early, strong control measures are needed to stop transmission of the virus. '},\n"," {'paper_id': '36e602b7081e7bb8a1cfeaa2fb92b0c43abd4fef',\n","  'section': 'Abstract',\n","  'text': \"Sequence analyses of RNA virus genomes remain challenging due to the exceptional genetic plasticity of these viruses. Because of high mutation and recombination rates, genome replication by viral RNA-dependent RNA polymerases leads to populations of closely related viruses, so-called 'quasispecies'. Standard (short-read) sequencing technologies are ill-suited to reconstruct large numbers of full-length haplotypes of (i) RNA virus genomes and (ii) subgenome-length (sg) RNAs comprised of noncontiguous genome regions. Here, we used a full-length, direct RNA sequencing (DRS) approach based on nanopores to characterize viral RNAs produced in cells infected with a human coronavirus. Using DRS, we were able to map the longest (∼26 kb) contiguous read to the viral reference genome. By combining Illumina and nanopore sequencing, we reconstructed a highly accurate consensus sequence of the human coronavirus (HCoV) 229E genome (27.3 kb). Furthermore, using long reads that did not require an assembly step, we were able to identify, in infected cells, diverse and novel HCoV-229E sg RNAs that remain to be characterized. Also, the DRS approach, which circumvents reverse transcription and amplification of RNA, allowed us to detect methylation sites in viral RNAs. Our work paves the way for haplotype-based analyses of viral quasispecies by demonstrating the feasibility of intra-sample haplotype separation. Even though several technical challenges remain to be addressed to exploit the potential of the nanopore technology fully, our work illustrates that direct RNA sequencing may significantly advance genomic studies of complex virus populations, including predictions on long-range interactions in individual full-length viral RNA haplotypes. \"},\n"," {'paper_id': '36fa6c04eaf8e66dd3bc6f6767d369a55feab7b4',\n","  'section': 'Abstract',\n","  'text': 'The number of COVID-19 cases is continuously increasing in different countries (as of March 2020) including the Philippines. It is estimated that the basic reproductive number of COVID-19 is around 1.5 to 4. The basic reproductive number characterizes the average number of persons that a primary case can directly infect in a population full of susceptible individuals. However, there can be superspreaders that can infect more than this estimated basic reproductive number. In this study, we formulate a conceptual mathematical model on the transmission dynamics of COVID-19 between the frontliners and the general public. We assume that the general public has a reproductive number between 1.5 to 4, and frontliners (e.g. healthcare workers, customer service and retail personnel, food service crews, and transport or delivery workers) have a higher reproduction number. Our simulations show that both the frontliners and the general public should be protected or resilient against the disease. Protecting only the frontliners will not result in flattening the epidemic curve. Protecting only the general public may flatten the epidemic curve but the infection risk faced by the frontliners is still high, which may eventually affect their work. Our simple model does not consider all factors involved in COVID-19 transmission in a community, but the insights from our model results remind us of the importance of community effort in controlling the transmission of the disease. All in all, the take-home message is that everyone in the community, whether a frontliner or not, should be protected or should implement preventive measures to avoid being infected. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '373b3b78e296f0046af3506452c6934b71084b58',\n","  'section': 'Abstract',\n","  'text': 'The outbreak of COVID-19 has laid unprecedented psychological stress on health workers (HWs). We aimed to assess the immediate psychological impact on HWs at Tongji Hospital in Wuhan, China. '},\n"," {'paper_id': '374a5295716649a80dca87097ad9ac902d344db6',\n","  'section': 'Abstract',\n","  'text': 'Databases of ICMJE-accepted clinical trial registry platform were searched on February 14, 2020. Randomized controlled trials (RCTs) and non-RCTs of COVID-19 were considered. Conditions of patients include common type, severe type or critical type. Interventions include traditional Chinese medicine (TCM) and Western medicine. We excluded trials that for discharged patients, psychological intervention and complications of COVID-19. The general information and outcomes, outcome measurement instruments and measurement times were extracted. The results were analysed by descriptive analysis. 19 registry platforms were searched. A total of 97 protocols were included from 160 protocols. For protocols of TCM clinical trials, 76 outcomes from 16 outcome domains were reported, and almost half (34/76, 44.74%) of outcomes were reported only once; the most frequently reported outcome was time of SARS-CoV-2 RNA turns to negative. 27 (27/76, 35.53%) outcomes were provided one or more outcome measurement instruments. 10 outcomes were provided one or more measurement time frame. For protocols of western medicine clinical trials, 126 outcomes from 17 outcome domains were reported; almost half (62/126, 49.21%) of outcomes were reported only once; the most frequently reported outcome was proportion of patients with negative SARS-CoV-2. 27 outcomes were provided one or more outcome measurement instruments. 40 (40/126, 31.75%) outcomes were provided one or more measurement time frame. Outcome reporting in protocols of clinical trials of COVID-19 is inconsistent. Thus, developing a core outcome set is necessary. '},\n"," {'paper_id': '379d7eb6a7394d8b20aacdafc2167fbe881b7c2a',\n","  'section': 'Abstract',\n","  'text': 'As of March 16, 2020, over 185,000 across the world, Italy became the red hotspot for the COVID-19 pandemic after China. With over 35,000 cases and 2900 deaths reported in the month of March in Italy, it is necessary to stimulate epidemic trend to understand the behavior of COVID-19 in Italy. By S.E.I.R. simulation, we estimated the most representative epidemic parameters occurred from March 1 to 14, 2020, thus being able to evaluate the consistency of the containment rules and identify possible Sars-Cov-2 local mutations. Our estimations are based on some assumptions and limitations exited. '},\n"," {'paper_id': '37aae10b06b0f03d5d473148abab4ea74fe80be1',\n","  'section': 'Abstract',\n","  'text': 'COVID-19 has spread with unequal efficiency in various parts of the world. In '},\n"," {'paper_id': '37afeaa38d66b8097cc7637bf62ac7d142e6b362',\n","  'section': 'Abstract',\n","  'text': 'To have epidemic potential, a pathogen must be able to spread in human populations, but of human-infective RNA viruses only a minority can do so. We investigated the evolution of human transmissibility through parallel analyses of 1755 virus genome sequences from 39 RNA virus genera. We identified 57 lineages containing human-transmissible species and estimated that at least 74% of these lineages have evolved directly from non-human viruses in other mammals or birds, a public health threat recently designated \"Disease X\". Humantransmissible viruses rarely evolve from virus lineages that can infect but not transmit between humans. This result cautions against focussing surveillance and mitigation efforts narrowly on currently known human-infective virus lineages and supports calls for a better understanding of RNA virus diversity in non-human hosts. author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '380a4d07d22910246c6f189fd2da93b270b96981',\n","  'section': 'Abstract',\n","  'text': 'Recently developed quantitative redox proteomic studies enable the direct identification of redox-sensing cysteine residues that regulate the functional behavior of target proteins in response to changing levels of reactive oxygen species (ROS). At the molecular level, redox regulation can directly modify the active sites of enzymes, although a growing number of examples indicate the importance of an additional underlying mechanism that involves conditionally disordered proteins. These proteins alter their functional behavior by undergoing a disorder-to-order transition in response to changing redox conditions. However, the extent to which this mechanism is used in various proteomes is currently unknown. Here, we use a recently developed sequence-based prediction tool incorporated into the IUPred2A web server to estimate redox-sensitive conditionally disordered regions on a large scale. We show that redox-sensitive conditional disorder is fairly widespread in various proteomes and that its presence strongly correlates with the expansion of specific domains in multicellular organisms that largely rely on extra stability provided by disulfide bonds or zinc ion binding. The analyses of yeast redox proteomes and human disease data further underlie the significance of this phenomenon in the regulation of a wide range of biological processes, as well as its biomedical importance. '},\n"," {'paper_id': '3843689b55794bbb8b950d7180899674ddc7e933',\n","  'section': 'Abstract',\n","  'text': 'word count: 100/100) 38 We modeled the viral dynamics of 13 untreated patients infected with SARS-CoV-2 to infer 39 viral growth parameters and predict the effects of antiviral treatments. In order to reduce peak 40 viral load by more than 2 logs, drug efficacy needs to be greater than 80% if treatment is 41 administered after symptom onset; an efficacy of 50% could be sufficient if treatment is 42 initiated before symptom onset. Given their pharmacokinetic/pharmacodynamic properties, 43 current investigated drugs may be in a range of 20-70% efficacy. They may help control virus 44 if administered very early, but may not have a major effect in severe patients. 45 46 All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '38bab3f17b186bd8ee289e5d135bb7d500ef500b',\n","  'section': 'Abstract',\n","  'text': 'Pockets of susceptibility resulting from spatial or social heterogeneity in vaccine coverage can drive 16 measles outbreaks, as cases imported into such pockets are likely to cause further transmission and lead 17 to large transmission clusters. Characterising the dynamics of transmission is essential for identifying 18 which individuals and regions might be most at risk. 19 As data from detailed contact tracing investigations are not available in many settings, we combined 20 age, location, genotype, and onset date of cases in order to probabilistically reconstruct the importation 21 status and transmission clusters within a newly developed R package called o2geosocial. 22 We compared our inferred cluster size distributions to 737 transmission clusters identified through 23 detailed contact-tracing in the United States between 2001 and 2016. We were able to reconstruct the 24 importation status of the cases and found good agreement between the inferred and reference clusters. 25 The results were improved when the contact-tracing investigations were used to set the importation 26 status before running the model. 27 '},\n"," {'paper_id': '38c35aed007b69cf403a097e93b18ad261072568',\n","  'section': 'Abstract',\n","  'text': 'The scientific community is focussed on developing antiviral therapies to 25 mitigate the impacts of the ongoing novel coronavirus disease This will be facilitated by improved understanding of viral dynamics within infected 27 hosts. Here, using a mathematical model in combination with published viral load 28 data collected from the same specimen (throat / nasal swabs or nasopharyngeal / 29 sputum / tracheal aspirate), we compare within-host dynamics for patients infected in 30 the current outbreak with analogous dynamics for MERS-CoV and SARS-CoV 31 infections. Our quantitative analyses revealed that SARS-CoV-2 infection dynamics 32 are more severe than those for mild cases of MERS-CoV, but are similar to severe 33 cases, and that the viral dynamics of SARS-CoV infection are similar to those of 34 MERS-CoV in mild cases but not in severe case. Consequently, SARS-CoV-2 35 generates infection dynamics that are more severe than SARS-CoV. Furthermore, 36 we used our viral dynamics model to predict the effectiveness of unlicensed drugs 37 that have different methods of action. The effectiveness was measured by AUC of 38 viral load. Our results indicated that therapies that block de novo infections or virus 39 production are most likely to be effective if initiated before the peak viral load (which 40 occurs around three days after symptom onset on average), but therapies that 41 promote cytotoxicity are likely to have only limited effects. Our unique mathematical 42 approach provides insights into the pathogenesis of SARS-CoV-2 in humans, which 43 are useful for development of antiviral therapies. 44 Keywords: 45 SARS-CoV-2, MERS-CoV, SARS-CoV, mathematical model, antiviral therapy 46 All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '38e3d1d69791756c82b0e76c5ac72115a82f5a9d',\n","  'section': 'Abstract',\n","  'text': 'Objective: The recent pandemic of novel coronavirus disease 2019 (COVID-19) is increasingly causing severe acute respiratory syndrome (SARS) and significant mortality. We aim here to identify the risk factors associated with mortality of coronavirus infected persons using a supervised machine learning approach. Research Design and Methods: Clinical data of 1085 cases of COVID-19 from 13 th January to 28 th February, 2020 was obtained from Kaggle, an online community of Data scientists. 430 cases were selected for the final analysis. Random Forest classification algorithm was implemented on the dataset to identify the important predictors and their effects on mortality. Results: The Area under the ROC curve obtained during model validation on the test dataset was 0.97. Age was the most important variable in predicting mortality followed by the time gap between symptom onset and hospitalization. Conclusions: Patients aged beyond 62 years are at higher risk of fatality whereas hospitalization within 2 days of the onset of symptoms could reduce mortality in COVID-19 patients. '},\n"," {'paper_id': '38e796bbec2f90b1b802c14dc922102e96f6361e',\n","  'section': 'Abstract',\n","  'text': 'In this work, we use a within-host viral dynamic model to describe the SARS-CoV-2 kinetics in host. Chest radiograph score data are used to estimate the parameters of that model. Our result shows that the basic reproductive number of SARS-CoV-2 in host growth is around 3.79. Using the same method we also estimate the basic reproductive number of MERS virus is 8.16 which is higher than SARS-CoV-2. The PRCC method is used to analyze the sensitivities of model parameters and the drug effects on virus growth are also implemented to analyze the model. '},\n"," {'paper_id': '39037d14bf47bea9a4721c8f331da9661f18fef5',\n","  'section': 'Abstract',\n","  'text': 'In this work we construct a mathematical model for the transmission and spread of coronavirus disease 2019 or COVID-19. Our model features delay terms to account for (a) the time lapse or latency period between contracting the disease and displaying symptoms, and (b) the time lag in testing patients for the virus due to the limited numbers of testing facilities currently available. We find that the delay introduces a significant disparity between the actual and reported time-trajectories of cases in a particular region. Specifically, the reported case histories lag the actual histories by a few days. Hence, to minimize the spread of the disease, lockdowns and similarly drastic social isolation measures need to be imposed some time before the reported figures are approaching their peak values. We then account for the social reality that lockdowns can only be of a limited duration in view of practical considerations. We find that the most effective interval for imposing such a limited-time lockdown is one where the midpoint of the lockdown period coincides with the actual peak of the spread of the disease in the absence of the lockdown. We further show that the true effectivity of imposing a lockdown may be misrepresented and grossly underestimated by the reported case trajectories in the days following the action. '},\n"," {'paper_id': '395e8569a0e6fc129029cb0b8cda1f7381954976',\n","  'section': 'Abstract',\n","  'text': 'Models of host-microbe dynamics typically assume a single-host population infected by a single 21 pathogen. In reality, many hosts form multi-species aggregations and may be infected with an 22 assemblage of pathogens. We used a meta-transcriptomic approach to characterize the viromes of 23 nine avian species in the Anseriformes (ducks) and Charadriiformes (shorebirds). This revealed the 24 presence of 27 viral species, of which 24 were novel, including double-stranded RNA viruses 25 (Picobirnaviridae and Reoviridae), single-stranded RNA viruses (Astroviridae, Caliciviridae, 26  '},\n"," {'paper_id': '396c98994ac34692a5fa7f8502fb33cebc54445a',\n","  'section': 'Abstract',\n","  'text': 'The influence of the outbreak of coronavirus disease (COVID-19) on mental health was poorly understood. The present study aimed to exam sleep problems and posttraumatic stress symptoms (PTSS) in Chinese immediately after the massive outbreak of COVID-19. A total of 2027 Chinese participated in the present study. Wuhan-expose history, sleep quality and PTSS were measured with self-rating scales. Results showed that there were significant differences of PCL-5 and of sleep quality scores in different data-collection dates (ps<0.05). There were significant differences of PCL-5 scores (t=-2.93, p<0.05) and latency onset of sleep (χ2=9.77, p<0.05) between participants with and without Wuhan-expose history. The interaction effect of Wuhan exposure history× sleep quality significantly influenced PCL-5 (ps<0.05). These results indicate that keeping good sleep quality in individuals with high infectious risk is a way to prevent PTSS. '},\n"," {'paper_id': '39ab0e9d7a93e6a7483bf4851e69cd66d2653256',\n","  'section': 'Abstract',\n","  'text': 'Pandemic coronavirus disease 2019 is caused by the emerging severe 24 acute respiratory syndrome coronavirus 2 (SARS-CoV-2), for which there are no 25 efficacious vaccines or therapeutics that are urgently needed. We expressed three 26 versions of spike (S) proteins-receptor binding domain (RBD), S1 subunit and S 27 ectodomain-in insect cells. RBD appears monomer in solutions, whereas S1 and S 28 associate into homotrimer with substantial glycosylation. The three proteins confer 29 excellent antigenicity with six convalescent COVID-19 patient sera. Cryo-electron 30 microscopy (cryo-EM) analyses indicate that the SARS-CoV-2 S trimer dominate in a 31 unique conformation distinguished from the classic prefusion conformation of 32 coronaviruses by the upper S1 region at lower position ~15 Å proximal to viral 33 membrane. Such conformation is proposed as an early prefusion state for the SARS-34 CoV-2 spike that may broaden the knowledge of coronavirus and facilitate vaccine 35 development. 36 '},\n"," {'paper_id': '3a6544f6f247366d4c12167f7fba743e9d981c63',\n","  'section': 'Abstract',\n","  'text': 'Angiotensin-converting enzyme 2 (ACE2) has been suggested to be the cellular receptor for the new coronavirus (2019-nCoV) that is causing the coronavirus disease 2019 . Like other coronaviruses such as the SARS-CoV, the 2019-nCoV uses the receptor binding domain (RBD) of the surface spike glycoprotein (S protein) to engage ACE2. We most recently determined the structure of the full-length human ACE2 in complex with a neutral amino acid transporter B 0 AT1. Here we report the cryo-EM structure of the full-length human ACE2 bound to the RBD of the 2019-nCoV at an overall resolution of 2.9 Å in the presence of B 0 AT1. The local resolution at the ACE2-RBD interface is 3.5 Å, allowing analysis of the detailed interactions between the RBD and the receptor. Similar to that for the SARS-CoV, the RBD of the 2019-nCoV is recognized by the extracellular peptidase domain (PD) of ACE2 mainly through polar residues. Pairwise comparison reveals a number of variations that may determine the different affinities between ACE2 and the RBDs from these two related viruses. author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '3a81dbb6350d25798f401c332abcf1b49616ffef',\n","  'section': 'Abstract',\n","  'text': 'Development of antiviral therapeutics emphasizes minimization of the effective dose and maximization 16 of the toxic dose, first in cell culture and later in animal models. Long-term success of an antiviral 17 therapeutic is determined not only by its efficacy but also by the duration of time required for drug-18 resistance to evolve. We have developed a microfluidic device comprised of ~6000 wells, with each 19 well containing a microstructure to capture single cells. We have used this device to characterize 20 enterovirus inhibitors with distinct mechanisms of action. In contrast to population methods, single-cell 21 analysis reveals that each class of inhibitor interferes with the viral infection cycle in a manner that can 22 '},\n"," {'paper_id': '3afd5fba7dc182ddfa769c0d766134b525581005',\n","  'section': 'Abstract',\n","  'text': 'Cases with coronavirus disease 2019 (COVID-19) emigrated from Wuhan escalated the risk of spreading in other cities. This report focused on the outside-Wuhan patients to assess the transmission and clinical characteristics of this illness. Contact investigation was conducted on each patient who admitted to the assigned hospitals in Hunan Province (geographically adjacent to Wuhan) from Jan 22, 2020 to Feb 12, 2020. Demographic, clinical, laboratory and radiological characteristics, medication therapy and outcomes were collected and analyzed. Patients were confirmed by PCR test. Of the 104 patients, 48 (46.15%) were imported cases and 56 (53.85%) were indigenous cases; 93 (89.42%) had a definite contact history with infections. Family clusters were the major body of patients. Transmission along the chain of 3 \"generations\" was observed. Mean age was 43 (rang, 8-84) years (including 3 children) and 49 (47.12%) were male. Most patients had typical symptoms, 5 asymptomatic infections were found and 2 of them infected their relatives. The median incubation period was 6 (rang, 1-32) days, of 8 patients ranged from 18 to 32 days. Just 9 of 16 severe patients required ICU care. Until Feb 12, 2020, 40 (38.46%) discharged and 1 (0.96%) died. For the antiviral treatment, 80 (76.92%) patients received traditional Chinese medicine therapy. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '3b040b06c58315ec7204d5a4e9ee8c51d5a9d3f0',\n","  'section': 'Abstract',\n","  'text': 'An unexplained outbreak of feline diarrhea and vomiting, negative for common 25 enteric viral and bacterial pathogens, was subjected to viral metagenomics and PCR. We 26 characterized from fecal samples the genome of a novel chapparvovirus we named fechavirus 27 that was shed by 8/17 affected cats and different feline bocaviruses shed by 9/17 cats. Also 28 detected were nucleic acids from attenuated vaccine viruses, members of the normal feline 29 virome, viruses found in only one or two cases, and viruses likely derived from ingested food 30 products. Epidemiological investigation of disease signs, time of onset, and transfers of affected 31 cats between three facilities support a possible role for this new chapparvovirus in a highly 32 contagious feline diarrhea and vomiting disease. 33 34 35 author/funder. All rights reserved. No reuse allowed without permission. : bioRxiv preprint the Neighbor-Joining phylogenetic trees for both NS1 and VP1 protein sequences were generated 132 using bootstrap method under Jones-Taylor-Thornton (JTT) model. 133 134 135 author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '3b181a741d4beafda3ba3a8fc68239493d49f6aa',\n","  'section': 'Abstract',\n","  'text': 'Backgrounds: In December 2019, a novel coronavirus (COVID-19) pneumonia hit Wuhan, Hubei Province, China and spread to the rest of China and overseas. The emergence of this virus coincided with the Spring Festival Travel Rush in China. It is possible to estimate total number of cases of COVID-19 in Wuhan, by 23 January 2020, given the cases reported in other cities and population flow data between cities. Methods: We built a model to estimate the total number of cases in Wuhan by 23 January 2020, based on the number of cases detected outside Wuhan city in China, with the assumption that if the same screening effort used in other cities applied in Wuhan. We employed population flow data from different sources between Wuhan and other cities/regions by 23 January 2020. The number of total cases was determined by the maximum log likelihood estimation. Findings: From overall cities/regions data, we predicted 1326 (95% CI: 1177, 1484), 1151 (95% CI: 1018, 1292) and 5277 (95% CI: 4732, 5859) as total cases in Wuhan by 23 January 2020, based on different source of data from Changjiang Daily newspaper, Tencent, and Baidu. From separate cities/regions data, we estimated 1059 (95% CI: 918, 1209), 5214 (95% CI: 4659, 5808) as total cases in Wuhan in Wuhan by 23 January 2020, based on different sources of population flow data from Tencent and Baidu. Conclusion: Sources of population follow data and methods impact the estimates of local cases in Wuhan before city lock down. '},\n"," {'paper_id': '3b22eecad8a582436c52284a4db2198a98a94e18',\n","  'section': 'Abstract',\n","  'text': 'The host antiviral response involves the induction of interferons and proinflammatory cytokines, but also the activation of cell death pathways, including apoptosis, to limit viral replication and spreading. This host defense is strictly regulated to eliminate the infection while limiting tissue damage that is associated with virus pathogenesis. Post-translational modifications, most notably phosphorylation, are key regulators of the antiviral defense implying an important role of protein phosphatases. Here, we investigated the role of the dual-specificity phosphatase 1 (DUSP1) in the host defense against human respiratory syncytial virus (RSV), a pathogenic virus of the Pneumoviridae family, and Sendai virus (SeV), a model virus being developed as a vector for anti-RSV vaccine. We found that DUSP1 is upregulated before being subjected to proteasomal degradation. DUSP1 does not inhibit the antiviral response, but negatively regulates virus-induced JNK/p38 MAPK phosphorylation. Interaction with the JNK-interacting protein 1 scaffold protein prevents dephosphorylation of JNK by DUSP1, likely explaining that AP-1 activation and downstream cytokine production are protected from DUSP1 inhibition. Importantly, DUSP1 promotes SeV-induced apoptosis and suppresses cell migration in RSVinfected cells. Collectively, our data unveil a previously unrecognized selective role of DUSP1 in the regulation of tissue damage and repair during infections by RSV and SeV. . CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/163360 doi: bioRxiv preprint relies on a complex set of phosphorylations mainly mediated by the TANK-binding kinase 1 (TBK1)/ IκB kinase epsilon (IKKε) kinases 13-17 . NF-κB, mainly p65/p50, activation during SeV and RSV infections occurs through IκB kinase (IKK)-dependent phosphorylation of the NF-κB inhibitor protein IκBα and of the p65 subunit 18, 19 . The signaling cascade leading to AP-1 activation is more elusive, but ultimately results in the phosphorylation of the nuclear Activating Transcription Factor 2 (ATF-2) and c-Jun subunits by JNK and p38 Mitogen-Activated Protein Kinases (MAPKs) 10,20,21 . The activation of these transcription factors promotes the transcription of early antiviral genes, type I/III IFNs and proinflammatory cytokines and chemokines 12, 22, 23 . In response to IFNs, hundreds of interferon-stimulated genes (ISGs) are induced to limit virus replication through enhancement of virus detection and innate immune signaling, cytoskeleton remodelling, inhibition of protein translation, induction of apoptosis, amongst other antiviral functions 24-26 . These same PRRs have also been shown to activate IFN-independent cell death pathways, including apoptosis 8 . Post-translational modifications (PTMs), including phosphorylation, ubiquitylation and acetylation, of signal transduction proteins involved in the pathways engaged downstream of virus sensing are crucial to reach a fine-tuned regulation of the innate immune response 27 . In light of the very well documented importance of phosphorylation PTMs in the regulation of the antiviral response, the protein phosphatases negatively regulating the signaling events have started to be identified. The exact role of the Ser/Thr protein phosphatase 1 (PP1) in the antiviral response remains elusive as PP1α and PP1γ were found to dephosphorylate MDA-5 and RIG-I leading to their activation 28 , while they were also described to be responsible for the dephosphorylation of key C-terminal phosphoresidues of IRF3 leading to its inhibition 29 . Most . CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/163360 doi: bioRxiv preprint 15 Clement, J. F. et al. Phosphorylation of IRF-3 on Ser 339 generates a hyperactive form of IRF-3 through regulation of dimerization and CBP association. '},\n"," {'paper_id': '3b73c8abb7528c958cec0bae1890828af007b599',\n","  'section': 'Abstract',\n","  'text': 'Domain growth plays an important role in many biological systems, and so the inclusion of domain growth in models of these biological systems is important to understanding how these biological systems function. In this work we present methods to include the effects of domain growth on the evolution of spatial correlations in a continuum approximation of a lattice-based model of cell motility and proliferation. We show that, depending on the way in which domain growth is implemented, different steady-state densities are predicted for an agent population. Furthermore, we demonstrate that the way in which domain growth is implemented can result in the evolution of the agent density depending on the size of the domain. Continuum approximations that ignore spatial correlations cannot capture these behaviours, while those that account for spatial correlations do. These results will be of interest to researchers in developmental biology, as they suggest that the nature of domain growth can determine the characteristics of cell populations. * ross@maths.ox.ac.uk In this work we examine how domain growth affects the evolution of spatial correlations between agents in IBMs, and importantly, how the way in which domain growth is implemented affects spatial correlations differently. To do so we use an agent-based (our agents represent 2 author/funder. All rights reserved. No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/041509 doi: bioRxiv preprint N x (t) by N y (t), where N x (t) is the number of lattice sites in a row and N y (t) is the number of sites in a column. All simulations are performed with periodic boundary conditions, and throughout this work the lattice spacing, ∆, is equal to one. For notational convenience we shall now write N x (t) as N x , and N y (t) as N y . In the IBM each agent is initially assigned to a lattice site, from which it can move or proliferate into an adjacent site. If an agent attempts to move into a site that is already occupied, 3 author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '3bdbda7b9bad2d94bfc88950c9efd33c90fb94b6',\n","  'section': 'Abstract',\n","  'text': 'An outbreak of the novel coronavirus SARS-CoV-2, the causative agent of COVID-19 respiratory disease, has infected over 290,000 people since the end of 2019, killed over 12,000, and caused worldwide social and economic disruption 1,2 . There are currently no antiviral drugs with proven efficacy nor are there vaccines for its prevention. Unfortunately, the scientific community has little knowledge of the molecular details of SARS-CoV-2 infection. To illuminate this, we cloned, tagged and expressed 26 of the 29 viral proteins in human cells and identified the human proteins physically associated with each using affinitypurification mass spectrometry (AP-MS), which identified 332 high confidence SARS-CoV-2-human protein-protein interactions (PPIs). Among these, we identify 67 druggable human proteins or host factors targeted by 69 existing FDA-approved drugs, drugs in clinical trials and/or preclinical compounds, that we are currently evaluating for efficacy in live SARS-CoV-2 infection assays. The identification of host dependency factors mediating virus infection may provide key insights into effective molecular targets for developing broadly acting antiviral therapeutics against SARS-CoV-2 and other deadly coronavirus strains. '},\n"," {'paper_id': '3becfc2b54e64064a86e184ec56dcc0397ea8395',\n","  'section': 'Abstract',\n","  'text': 'A key challenge for estimating the epidemiological parameters of the COVID-19 outbreak in Wuhan is the discrepancy between the officially reported number of infections and the true number of infections. A common approach to tackling the challenge is to use the number of infections exported from Wuhan to infer the true number in the city. This approach can only provide a static estimate of the epidemiological parameters before Wuhan lockdown on January 23, 2020, because there are almost no exported cases thereafter. Here, we propose a method to dynamically estimate the epidemiological parameters of the COVID-19 outbreak in Wuhan by recovering true numbers of infections from day-to-day official numbers. Using the method, we provide a comprehensive retrospection on how the disease had progressed in Wuhan from January 19 to March 5, 2020. Particularly, we estimate that the outbreak sizes by January 23 and March 5 were 11,239 [95% CI 10,852-11,541] and 124,506 [95% CI 96,703-155,317], respectively. The effective reproduction number attained its maximum on January 24 (3.42 [95% CI 3.21-3.54]) and became less than 1 from February 7 (0.76 [95% CI 0.66-0.87]). We also estimate the effects of two major government interventions on the spread of COVID-19 in Wuhan. Acknowledgements: We thank Huiran Li for her help with literature review and data processing. '},\n"," {'paper_id': '3c12e50b9da4b1fde21d68040610b73b8c216bf3',\n","  'section': 'Abstract',\n","  'text': \"Coronaviruses express a multifunctional papain-like protease, termed PLP2. PLP2 acts as 17 a protease that cleaves the viral replicase polyprotein, and a deubiquitinating (DUB) enzyme which 18 removes ubiquitin moieties from ubiquitin-conjugated proteins. Previous in vitro studies 19 implicated PLP2 DUB activity as a negative regulator of the host interferon (IFN) response, but 20 the role of DUB activity during virus infection was unknown. Here, we used X-ray structure-21 guided mutagenesis and functional studies to identify amino acid substitutions within the 22 ubiquitin-binding surface of PLP2 that reduced DUB activity without affecting polyprotein 23 processing activity. We engineered a DUB mutation (Asp1772 to Ala) into a murine coronavirus 24 and evaluated the replication and pathogenesis of the DUB mutant virus (DUBmut) in cultured 25 macrophages and in mice. We found that the DUBmut virus replicates similarly as the wild-type 26 virus in cultured cells, but the DUBmut virus activates an IFN response at earlier times compared 27 to the wild-type virus infection in macrophages, consistent with DUB activity negatively 28 regulating the IFN response. We compared the pathogenesis of the DUBmut virus to the wild-type 29 virus and found that the DUBmut-infected mice had a statistically significant reduction (p<0.05) 30 in viral titer in livers and spleens at day 5 post-infection, albeit both wild-type and DUBmut virus 31 infections resulted in similar liver pathology. Overall, this study demonstrates that structure-guided 32 mutagenesis aids the identification of critical determinants of PLP2-ubiquitin complex, and that 33 PLP2 DUB activity plays a role as an interferon antagonist in coronavirus pathogenesis. 34 Importance Coronaviruses employ a genetic economy by encoding multifunctional proteins that 36 function in viral replication and also modify the host environment to disarm the innate immune 37 response. The coronavirus papain-like protease 2 (PLP2) domain possesses protease activity, 38 All rights reserved. No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/782409 doi: bioRxiv preprint which cleaves the viral replicase polyprotein, and also DUB activity (de-conjugating 39 ubiquitin/ubiquitin-like molecules from modified substrates) using identical catalytic residues. To 40 separate the DUB activity from the protease activity, we employed a structure-guided mutagenesis 41 approach and identified residues that are important for ubiquitin-binding. We found that mutating 42 the ubiquitin-binding residues results in a PLP2 that has reduced DUB activity but retains protease 43 activity. We engineered a recombinant murine coronavirus to express the DUB mutant and showed 44 that the DUB mutant virus activated an earlier type I interferon response in macrophages and 45 exhibited reduced pathogenesis in mice. The results of this study demonstrate that PLP2/DUB is 46 an interferon antagonist and a virulence trait of coronaviruses. 47 48 Coronaviruses (CoVs) are members of the order Nidovirales, which includes enveloped viruses 49 with large (~30kb), positive-sense single-stranded RNA genomes that yield a characteristic nested 50 set of subgenomic mRNAs during replication in the cytoplasm of infected cells (1, 2). The genome 51 organization for coronaviruses is highly conserved, with the 5'-most two-thirds of the genome 52 encoding the replicase polyprotein, followed by sequences encoding the canonical structural 53 proteins: spike, envelope, membrane, and nucleocapsid. Many CoVs contain accessory genes, 54 which are interspersed among the genes for the structural proteins. Although these accessory genes 55 are not necessarily required for virus replication and are, in general, not highly conserved within 56 the virus family, many encode proteins that regulate the host response (3). Interestingly, 57 coronavirus replicase proteins, which are highly conserved, can also act as antagonists to block or 58 delay the host innate immune response to infection (1, 4-8). That a slew of coronavirus-encoded 59 accessory and non-accessory proteins have been shown to shape the host antiviral response 60 suggests that viral-mediated subversion of host defenses is an important element of infection. Here, 61 All rights reserved. No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/782409 doi: bioRxiv preprint we focus on the viral protease/deubiquitinase (DUB) with the goal of assessing the role of DUB 62 activity in shaping the pathogenesis of mouse hepatitis virus (MHV), a model murine coronavirus. Coronavirus proteases are essential for viral replication as they are responsible for processing 64 the replicase polyprotein (2,(9)(10)(11). Murine coronavirus MHV encodes three proteases: two 65 papain-like proteases (PLP1 and PLP2) and one chymotrypsin-like protease (3CLpro, also termed 66 Mpro). MHV PLP2 is similar to the single papain-like protease (termed PLpro) of Severe Acute 67 Respiratory Syndrome Coronavirus (SARS-CoV) and Middle East Respiratory Syndrome 68 Coronavirus (MERS-CoV) (9, 12-16). We and others have revealed that PLP2 or PLpro of 69 multiple coronaviruses, including MHV, are multifunctional, not only capable of cleaving the viral 70 polyprotein but also possessing deubiquitinase (DUB) and deISGylating (de-conjugating ISG15 71 molecule from modified substrates) activities (9, 12, 13, 15-27). However, it has been challenging 72 to study the effects of a mutated DUB on virus replication and pathogenesis because the protease 73 and DUB activities share the same catalytic site, disruption of which is lethal for virus replication. 74 Therefore, it was necessary for us to identify residue(s) within the MHV protease/DUB-ubiquitin 75 binding surface that can be mutated to result in reduced DUB activity without affecting polyprotein 76 processing. In this report, we describe the X-ray structure-guided identification of such residues 77 and evaluate the replication, IFN antagonistic effect, and pathogenesis of a recombinant DUB 78 mutant MHV. 79 Results Structure-guided identification of PLP2 residues interacting with ubiquitin. To investigate 81 the role of viral deubiquitinating activity in coronavirus replication and pathogenesis, we first 82 needed to identify amino acid residues that, when mutated, result in reduced DUB activity while 83 All rights reserved. No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/782409 doi: bioRxiv preprint 5 preserving the enzyme's deISGylating and protease activities, the latter being necessary for viral 84 replication. X-ray structural studies of SARS-and MERS-CoV papain-like protease/DUBs co-85 crystalized with ubiquitin modified at the c-terminus with a covalent warhead (aldehyde, 3-bromo- 86 propylamine, or propargylamine) allowed for identification of residues that are important for direct 87 interaction with ubiquitin (17, 18, 20). Here, we took a slightly different approach and mutated the 88 catalytic cysteine (C1716) of MHV PLP2 to a serine residue and co-crystallized it with free 89 ubiquitin. The X-ray structure of MHV PLP2 (C1716S) in complex with ubiquitin was determined 90 to a resolution of 1.85 Å and an overall Rfree = 19.6% and Rwork = 15.8% (PDB: 5WFI). The overall 91 structure of the MHV PLP2-ubiquitin complex is similar to other PLP2/PLpro ubiquitin-bound 92 structures (Fig. 1A). Ubiquitin binds within the palm region and is gripped by the zinc-fingers 93 motif while the C-terminus extends into the active site. 94 Next, we aligned the primary amino acid sequence of the MHV PLP2 domain with the 95 SARS-CoV and MERS-CoV papain-like protease domains (Fig. 1E) . The sequence alignment and 96 X-ray structure of the MHV PLP2-ubiquitin complex were then analyzed in conjunction with the 97 previous structural and mutagenesis studies on SARS-and MERS-CoV to identify candidate 98 residues that could be mutated to render a loss of DUB activity in vitro. From this analysis, we 99 identified three residues (R1803, D1772, F1812) in MHV PLP2 that form direct interactions with 100 ubiquitin (Fig. 1B-D) . Two of the side chain guanidinium nitrogens of R1803 form direct hydrogen 101 bonds with the backbone carbonyl oxygen of A46 in ubiquitin (Fig. 1B) . The two side chain 102 carboxylate oxygens of D1772 in MHV PLP2 interact with ubiquitin by forming direct bonds with 103 each of the guanidinium nitrogens of R42 and with one of the guanidinium nitrogens of R72 (Fig.   104 1C). Finally, F1812 forms Van der Waals contacts with the side chains of I44 and V70 and the 105 delta-carbon of R42 of ubiquitin (Fig. 1D) . 106 All rights reserved. No reuse allowed without permission. To identify a mutant MHV PLP2 enzyme that 107 retains protease activity but exhibits reduced DUB and/or deISGylating activity, we performed 108 site-directed mutagenesis on each of the residues (R1803, D1772, F1812) by changing each to an 109 alanine to disrupt interactions with ubiquitin. Each mutant enzyme was expressed, purified, and 110 tested for its ability to hydrolyze three substrates: z-RLRGG-AMC, Ub-AMC, and ISG15-AMC. The activity of each mutant enzyme toward each substrate relative to the wild-type enzyme is 112 shown in Fig. 2A . All three mutants retained their ability to hydrolyze the peptide substrate but 113 each mutant had altered specificity toward Ub-AMC and ISG15-AMC substrates. Mutation of 114 F1812 resulted in a substantial decrease in hydrolysis of both Ub-AMC and ISG15-AMC (Class 115 I), whereas mutation of R1803 resulted in loss of activity only toward ISG15-AMC (Class II), and 116 mutation of D1772 resulted in loss of activity only toward Ub-AMC (Class III). 117 151 All rights reserved. No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/782409 doi: bioRxiv preprint glycerophosphate, 1mM sodium orthovanadate, 1 µg/mL leupeptin, and 1mM 334 phenylmethylsulfonyl fluoride). Whole cell lysates were separated by SDS-PAGE and transferred 335 to PVDF membrane in transfer buffer (0.025M Tris, 0.192M glycine, 20% methanol) for 1 hour 336 at 60 Volts at 4°C. Following this, the membrane was blocked using 5% dried skim milk in TBST 337 buffer (0.9% NaCl, 10mM Tris-HCl, pH7.5, 0.1% Tween 20) overnight at 4°C. The membrane 338 was incubated with either polyclonal rabbit anti-GFP antibody (A11122, Life Technologies) for 339 the protease assay, or mouse anti-flag (F3165, Sigma) for the DUB assay. The membrane was then 340 washed three times for 15 minutes in TBST buffer followed by incubation with either secondary 341 donkey anti-rabbit-HRP antibody (711-035-152, Jackson ImmunoResearch) or goat anti-mouse-342 HRP antibody (1010-05, SouthernBiotech). Then the membrane was washed three more times for 343 15 minutes in TBST buffer. Detection was performed using Western Lighting Chemiluminescence 344 Reagent Plus (PerkinElmer) and visualized using a FluoroChemE Imager (Protein Simple). The 345 expression of PLP2, β-actin, and calnexin were probed with mouse anti-V5 antibody (R960, 346 ThermoFisher), mouse anti-β-actin (A00702, Genscript), or mouse anti-calnexin antibody (2433S, 347 Cell Signaling), respectively. 348 Biosensor live cell assay 349 The protease activity of PLP2 was also assessed using a biosensor live cell assay as 350 described previously (14). Briefly, HEK293T cells in a 96 black-wall plate were transfected with 351 37.5 ng pGlo-RLKGG construct and 50 ng PLP2 expression plasmids. GloSensor (E1290, 352 Promega) reagent diluted in DMEM+10% FCS was added at 18 hpi. Plates were read using a 353 luminometer (Veritas) every hour over a course of 5 hours. 354 All rights reserved. No reuse allowed without permission. \"},\n"," {'paper_id': '3c14c4c3a176c86833794029e787ce460c9d89dd',\n","  'section': 'Abstract',\n","  'text': 'Although bats are increasingly becoming the focus of scientific studies due to their unique properties, these exceptional animals are still among the least studied mammals. Assembly quality and completeness of bat genomes vary a lot and especially non-coding RNA (ncRNA) annotations are incomplete or simply missing. Accordingly, standard bioinformatics pipelines for gene expression analysis often ignore ncRNAs such as microRNAs or long antisense RNAs. The main cause of this problem is the use of incomplete genome annotations. We present a complete screening for ncRNAs within 16 bat genomes. NcRNAs affect a remarkable variety of vital biological functions, including gene expression regulation, RNA processing, RNA interference and, as recently described, regulatory processes in viral infections. Within all investigated bat assemblies we annotated 667 ncRNA families including 162 snoRNAs and 193 miRNAs as well as rRNAs, tRNAs, several snRNAs and lncRNAs, and other structural ncRNA elements. We validated our ncRNA candidates by six RNA-Seq data sets and show significant expression patterns that have never been described before in a bat species on such a large scale. Our annotations will be usable as a resource (Electronic Supplement) for deeper studying of bat evolution, ncRNAs repertoire, gene expression and regulation, ecology, and important host-virus interactions. Supplementary information: is available at rna.uni-jena.de/supplements/bats, the Open Science Framework (doi.org/10.17605/OSF.IO/4CMDN), and GitHub (github.com/rnajena/bats_ncrna). '},\n"," {'paper_id': '3c15ba282c7261b3447e3a68a91e1a3b23b6fa3d',\n","  'section': 'Abstract',\n","  'text': 'Objectives: No data were available about in-flight transmission of SARS-CoV-2. Here, we report an in-flight transmission cluster of COVID-19 and describe the clinical characteristics of these patients. Methods: After a flight, laboratory-confirmed COVID-19 was reported in 12 patients. Ten patients were admitted to the designated hospital. Data were collected from 25 th January to 28 th February 2020. Clinical information was retrospectively collected. Results: All patients are passengers without flight attendants. The median age was 33 years, and 70% were females. None was admitted to intensive care unit, and no patients succumbed through 28 th February. The median incubation period was 3.0 days and from illness onset to hospital admission was 2 days. The most common symptom was fever. Two patients were asymptomatic and negative for chest CT scan throughout the disease courses. On admission, initial RT-PCR were positive in 9 patients, however initial chest CT were positive in only half patients. The median lung \"total severity score\" of chest CT was 6. Notably, \"Crazy-Paving\" pattern, pleural effusion, and ground-glass nodules were also seen. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '3c476baf5542bef88b8aad92bcca6786d1afa282',\n","  'section': 'Abstract',\n","  'text': 'Dengue virus (DENV) represents a public health and economic burden in affected countries. The availability of genomic data is key to understand viral evolution and dynamics, supporting improved control strategies. Currently, the use of High Throughput Sequencing (HTS) technologies, which can be applied both directly to patient samples (shotgun metagenomics) and to PCR amplified viral sequences (targeted metagenomics), is the most informative approach to monitor the viral dissemination and genetic diversity. Despite many advantages, these technologies require bioinformatics expertise and appropriate infrastructure for the analysis and interpretation of the resulting data. In addition, the many software solutions available can hamper reproducibility and comparison of results. Here we present DEN-IM, a one-stop, user-friendly, containerised and reproducible workflow for the analysis of DENV sequencing data, both from shotgun and targeted metagenomics approaches. It is able to infer DENV coding sequence (CDS), identify serotype and genotype, and generate a phylogenetic tree. It can easily be run on any UNIX-like system, from local machines to high-performance computing clusters, performing a comprehensive analysis without the requirement of extensive bioinformatics expertise. Using DEN-IM, we successfully analysed two DENV datasets. The first comprised 25 shotgun metagenomic sequencing samples of varying serotype and genotype, including a spiked sample containing the existing four serotypes. The second dataset consisted of 106 targeted metagenomics samples of DENV 3 genotype III where DEN-IM allowed detection of the intra-genotype diversity. The DEN-IM workflow, parameters and execution configuration files, and documentation are freely available at https://github.com/B-UMMI/DEN-IM. The Dengue virus (DENV), a single-stranded positive-sense RNA virus belonging to the Flavivirus 2 genus, is one of the most prevalent arboviruses and is mainly concentrated in tropical and subtropical 3 regions. Infection with DENV results in symptoms ranging from mild fever to haemorrhagic fever and 4 shock syndrome [1]. Transmission to humans occurs through the bite of Aedes mosquitoes namely 5 Aedes aegypti and Aedes albopictus [2] In 2010, it was predicted that the burden of dengue disease 6 reached 390 million cases per year worldwide [3]. The high morbidity and mortality of dengue makes 7 it the arbovirus with the highest clinical significance [4]. 8 The viral genome of ∼11,000 nucleotides, consists of a Coding Sequence (CDS) of approximately 9 10.2 Kb that is translated into a single polyprotein encoding three structural proteins (capsid -C, 10 premembrane -prM, envelope -E) and seven non-structural proteins (NS1, NS2A, NS2B, NS3, NS4A, 11 NS4B and NS5. Additionally, the genome contains two Non-Coding Regions (NCRs) at their 5\\' and 12 3\\' ends [5]. DENV can be classified into four serotypes (1, 2, 3 and 4), differing from each other by 25% to 14 40% at the amino acid level. They are further classified into genotypes that vary by up to ∼3% 15 at the amino acid level [2]. The DENV-1 serotype comprises five genotypes (I-V), DENV-2 groups 16 six (I-VI, also named American, Cosmopolitan, Asian-American, Asian II, Asian I and Sylvatic), 17 DENV-3 four (I-III and V), and DENV-4 also four (I-IV). 18 DENV is a significant public health challenge in countries where the infection is endemic due 19 to the high health and economic burden. Despite the emergence of novel therapies and ecological 20 strategies to control the mosquito vector, there are still important knowledge gaps in the virus biology 21 and its epidemiology [2]. The implementation of a surveillance system relying on High Throughput Sequencing (HTS) 23 technologies allows the simultaneous identification and surveillance of DENV cases. Due to the 24 high sensitivity of these technologies, previous studies showed that viral sequences can be directly 25 obtained from patient sera using a shotgun metagenomics approach [6]. Alternatively, HTS can be 26 used in a targeted metagenomics approach in which a PCR step is used to pre-amplify viral sequences 27 before sequencing. In recent years, HTS has been successfully used as a tool for identification of 28 DENV directly from clinical samples [6, 7]. This also allows the rapid identification of the serotype 29 and genotype important for disease management as the genotype may be associated with disease 30 outcome [8]. 31 Several initiatives aim to facilitate the identification of the DENV serotype and genotype from 32 HTS data. The Genome Detective project (https://www.genomedetective.com/) offers an online 33 Dengue Typing Tool (https://www.genomedetective.com/app/typingtool/dengue/) relying on 34 BLAST and phylogenetic methods in order to identify the closest serotype and genotype, but it 35 requires as input assembled genomes in FASTA format. Alternatively, the same project offers 36 a Genome Detective Typing Tool (https://www.genomedetective.com/app/typingtool/virus/) 37 identifying viruses present in a sample. We developed DEN-IM as a ready-to-use, one-stop, reproducible bioinformatic analysis workflow 39 for the processing and phylogenetic analysis of DENV using paired-end raw HTS data. DEN-IM is 40 implemented in Nextflow [9], a workflow manager software that uses Docker (https://www.docker. 41 com) containers with pre-installed software for all the workflow tools. The DEN-IM workflow, as well 42 as parameters and documentation, are available at https://github.com/B-UMMI/DEN-IM. 43 3 The DEN-IM Workflow 44 DEN-IM is a user-friendly automated workflow allowing the analysis of shotgun or targeted metage-45 nomics data for the identification, serotyping, genotyping, and phylogenetic analysis of DENV. It is 46 implemented in Nextflow, a workflow management system that allows the effortless deployment and 47 3 . CC-BY 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/628073 doi: bioRxiv preprint execution of complex distributed computational workflows. The workflow is composed of five blocks: 1. Quality Control and Trimming, 2. Retrieval of DENV 49 sequences, 3. Assembly, 4. in silico Typing, and 5. Phylogeny, described in more detail in Figure 1 . 50 DEN-IM accepts as input raw paired-end sequencing data (FASTQ files), and informs the user with 51 an interactive HTML report with information on the quality control, mapping, assembly typing and 52 phylogenetic analysis, as well as all the output files of the whole pipeline. 53 3.1 1. Quality Control and Trimming 54 The Quality Control (QC) and Trimming block starts with a process to verify the integrity of the 55 input data. If the sequencing files are corrupted, the execution of the analysis of that sample is 56 terminated. 57 The sequences are then processed by FastQC (https://www.bioinformatics.babraham.ac.uk/ 58 projects/fastqc/, version 0.11.7) to determine the quality of the individual base pairs of the 59 raw data files. The low quality bases and adapter sequences are trimmed by Trimmomatic [10] 60 (version 0.36). By default, Trimmomatic removes the Illumina Nextera, TruSeq2 and TruSeq3 adapter 61 sequences that are detected, and relies on Phred scores [11] and FastQC analysis data to determine 62 to which extend the 5\\' and 3\\' ends of the reads need to be trimmed. The crop and headcrop setting 63 is calculated per sample, based on the GC ratio of each half of the read. By default, the window size 64 is set to 5 nucleotides with a minimum average quality of 20. In addition, paired-end reads with a 65 read length shorter than 55 nucleotides after trimming are removed from further analyses. 66 Lastly, the low complexity sequences, containing over 50% of poly-A, poly-N or poly-T nucleotides, 67 are filtered out of the raw data using PrinSeq [12] (version 0.10.4). 68 3.2 2. Retrieval of DENV Sequences 69 In the second step, DENV sequences are selected from the sample using Bowtie2 [13] (version 70 2.2.9) and Samtools [14] (version 1.4.1). As a reference we provide the DENV mapping database, a 71 curated DENV database composed of 3830 complete DENV genomes (see Methods, DENV Reference 72 Database). A permissive approach is followed by allowing for mates to be kept in the sample even 73 when only one read maps to the database in order to keep as many DENV derived reads as possible. The output of this step is a set of processed reads of putative DENV origin. 75 3.3 3. Assembly 76 DEN-IM applies a two assembler approach to generate assemblies of the DENV CDS. To obtain a 77 high confidence assembly, the processed reads are first de novo assembled with SPAdes [15] (version 78 3.12.0). If the full CDS fails to be assembled into a single contig, the data is re-assembled with 79 MEGAHIT assembler [16] (version 1.1.3), a more permissive assembler developed to retrieve longer 80 sequences from metagenomics data. The resulting assemblies are corrected with Pilon [17] (version 81 1.22) after mapping the processed reads to the assemblies with Bowtie2 [13]. 82 If more than one complete CDS is present in a sample, each of the sequences will follow the rest 83 of the DEN-IM workflow independently. If no full CDS is assembled neither with SPAdes nor with 84 MEGAHIT, the processed reads are passed on to the next step for consensus generation by mapping, 85 effectively constituting DEN-IM\\'s two pronged approach using both assemblers and mapping. 86 3.4 4. Typing 87 For each DENV complete CDS, the serotype and genotype is determined with the Seq Typing tool 88 (https://github.com/B-UMMI/seq_typing, version 2.0) [18] using BLAST [19] and the custom 89 Typing database of DENV containing 161 complete sequences (see Methods, DENV Reference 90 Database). The BLAST results are first cleaned to get the best hit for each sequence in the database, 91 based on alignment length, similarity, e-value and number of gaps. The tool determines which 92 reference sequence is more closely related to the query based on the identity and length of the 93 sequence covered, returning the serotype and genotype of the reference sequence for the purpose of 94 classifying the query. If a complete CDS fails to be obtained through the assembly process, the processed reads are 96 mapped against the DENV typing database, with Bowtie2 [13], using Seq Typing tool, with similar 97 criteria for coverage and identity to those used with the BLAST approach. If a type is determined, 98 the consensus sequence obtained follows through to the next step in the workflow. Otherwise, the 99 sample is classified as Non-Typable and its process terminated. 100 3.5 5. Phylogeny 101 All DENV complete CDSs and consensus sequences analysed in a workflow execution are aligned 102 with MAFFT [20] (version 7.402), in auto mode and with orientation being automatically determined 103 and adjusted. With the resulting alignment, a Maximum Likelihood tree is inferred with RaXML [21] 104 (version 8.2.11), using as default the GTR-Γ substitution model and a 500 times bootstrap. 105 Optionally, the closest reference sequence in the DENV typing database to each analysed sample 106 can be retrieved and included in the phylogenetic analysis. 107 4 Workflow Execution 108 The DEN-IM workflow can be executed in any UNIX-based system, from local machines to high-109 performance computing clusters (HPC) facilities with Nextflow and a container engine installation, 110 such as Docker (https://www.docker.com/, Shifter [22] or Singularity [23]. 111 Due to its Nextflow implementation, DEN-IM allows for out-of-the-box high-level parallelization 112 and offers direct support for distributed computational environments. DEN-IM integrates Docker 113 containerised images for all the tools necessary for its execution, ensuring reproducibility and the 114 tracking of both software code and version, regardless of the operating system used. The Docker 115 images provided are also compatible with other container engines. 116 Users can customise the workflow execution either by using command line options or by modifying 117 a simple plain-text configuration file (params.config). The version of each of the tools used in 118 DEN-IM can be changed by providing new container tags in the appropriate configuration file 119 (containers.config). The resources for each process can also be changed (resources.config). To make 120 the execution of the workflow as simple as possible, a set of default parameters and directives is 121 provided. 122 The local installation of the DEN-IM workflow, including the docker containers with all the tools 123 needed and the curated DENV database, requires 15 Gigabytes of free disk space. The minimum 124 requirements to execute the workflow are at least 5 Gigabytes of memory and 4 CPUs, although 7 125 Gigabytes of memory is advised. The disk space required for execution depends greatly on the size 126 of the input data, but for the datasets used in this article DEN-IM generates approximately 20 Gb 127 data per Gb of input data. 128 5 Output and Report 129 The output files of all tools are stored in the \\'results\\' folder in the directory of the DEN-IM execution. 130 The execution log file for each component, as well as a log file for the execution of the workflow, are 131 also available. 132 DEN-IM creates an interactive HTML report (Figure S1), stored in the \\'pipeline results\\' directory, 133 containing all the information in the results divided into four sections: report overview, tables, charts 134 and phylogenetic tree. The report can be easily exchanged between collaborators by compressing 135 and sharing the \"pipeline report\" folder. 136 The report overview contains information about the number of samples in the analysis. It allows 137 for selection, filtering and highlighting of particular samples and tools in the workflow. 138 The table section contains the results and statistics the quality control, assembly, read mapping 139 and in silico typing results. The in silico typing table contains the results of the serotype and 140 genotype of each CDS analysed, as well as identity and coverage and GenBank ID for the closest 141 reference in the DENV typing database. doi: bioRxiv preprint 206 was generated. For the assembled CDSs, all but one were assembled with MEGAHIT after not 207 producing a full CDS with SPAdes. Moreover, pronounced variation on the size of the assembled 208 8 . CC-BY 4.0 doi: bioRxiv preprint As a positive control sample, the supernatant of a viral culture containing DENV-2 strain 16681 388 was used. The negative control sample consisted of DNA-and RNA-free water (Sigma-Aldrich, St. 389 Louis, MO, USA). 390 A spiked sample was produced consisting of a mixture of four 5 µl of cDNA isolated from clinical 391 samples including all DENV serotypes (DENV-1 to -4). The viral cDNA for these samples was not in 392 equal concentration and the viral copy number in the clinical samples was assessed by RT-PCR [35]. 393 The results were as follow: DENV-2 with 1070000 copies/µl, DENV-1 with 117830 copies/µl, DENV-3 394 with 44300 copies/µl and DENV-4 with 6600 copies/µl. The cDNA libraries were generated using either the NEBNext R RNA First and Second strand 396 modules and the Nextera XT DNA library preparation kit (NXT), or the TruSeq RNA V2 library 397 preparation kit (TS). The libraries were sequenced in MiSeq and NextSeq instruments using 300-cycles 398 13 . CC-BY 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/628073 doi: bioRxiv preprint v2 paired-end cartridges. The DEN-IM workflow was executed with the raw sequencing data using the default parameters 400 and resources in an HPC cluster with 300 Cores/600 Threads of Processing Power and 3 TB RAM 401 divided through 15 computational nodes, 9 with 254 GB Ram and 6 with 126GB RAM. 402 9.4 Targeted Metagenomics Sequencing Data 403 The accession numbers for the 106 DENV-3 amplicon sequencing paired-end short-read datasets 404 are available under BioProject PRJNA394021. The list of Run Accession IDs were obtained 405 with NCBI\\'s RunSelector and the raw data was downloaded with the GetSeqENA tool (https: 406 //github.com/B-UMMI/getSeqENA). The DEN-IM workflow was executed with the raw sequencing data with default parameters and 408 resources in the same HPC cluster as the shotgun metagenomics dataset. 409 10 Availability of source code and requirements 410 Lists the following: 411 • Project name: DEN-IM 412 • Project home page: https://github.com/B-UMMI/DEN-IM 413 • Operating system(s): UNIX-like systems. 414 • Programming language: Nextflow, Python, Bash 415 • Other requirements: Java version 8 or highest. Docker/Singularity/Shifter 416 • License: GNU GPL v3 417 • Documentation and tutorials: https://github.com/B-UMMI/DEN-IM/wiki 418 11 Availability of supporting data and materials 419 The 106 DENV-3 targeted metagenomics sequencing paired-end short-read datasets are available 420 under BioProject PRJNA394021. The 25 shotgun metagenomics dataset is available under Bioproject 421 PRJNA474413 The accession number for all the samples in the shotgun metagenomics dataset are 422 available in the supplemental material (Table S3) . 423 12 Declarations 424 12.1 List of abbreviations 425 Control 428 12.2 Ethical Approval (optional) 429 This study followed international standards for the ethical conduct of research involving human 430 subjects. Data and sample collection was carried out within the DENVEN and IDAMS (International 431 Research Consortium on Dengue Risk Assessment, Management and Surveillance) projects. The 432 study was approved by the Ethics Review Committee of the Biomedical Research Institute, Carabobo 433 University (Aval Bioetico #CBIIB(UC)-014 and CBIIB-(UC)-2013-1), Maracay, Venezuela; the '},\n"," {'paper_id': '3c6649c7f1f9c7e71a0ae8980ac71429d2bca398',\n","  'section': 'Abstract',\n","  'text': 'This work compares deaths for confirmed COVID-19 cases in China to eight other countries, South Korea. After implementing varying intensities and timing of social distancing measures, several appear to be converging onto the decline in the daily growth rate of deaths, or relative second derivative of total deaths, seen in China after the implementation an aggressive social distancing policy. By calculating future trajectories in these countries based on the observed Chinese fatality statistics, an estimate of the total deaths and maximum daily death rates over a defined period of time is made. Our lower bound estimate for the United Kingdom based on the real data approximates the lower bound estimate of the recent modelling study of Ferguson et al. [1]. These results suggest there may be a threshold of effective public health intervention. Our method of viewing the data may be helpful in monitoring the course of the epidemic, judging the effectiveness of implementation, and monitoring the relaxation of social distancing. . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': '3c70c99afc7a38df3c4807857856ea258d378429',\n","  'section': 'Abstract',\n","  'text': 'Pompe disease results from a defect in human acid α-glucosidase (GAA), a lysosomal enzyme 17 that cleaves terminal α1-4 and α1-6 glucose from glycogen. In Pompe disease (also known as 18 Glycogen Storage Disorder type II), the accumulation of undegraded glycogen in lysosomes 19 leads to cellular dysfunction, primarily in muscle and heart tissues. Pompe disease is an active 20 candidate of clinical research, with pharmacological chaperone therapy tested and enzyme 21 replacement therapy approved. Despite production of large amounts of recombinant GAA 22 annually, the structure of GAA has not been reported until now. Here, we describe the first 23 structure of GAA, at 1.7Å resolution. Three structures of GAA complexes reveal the molecular 24 basis for the hundreds of mutations that lead to Pompe disease and for pharmacological 25 chaperoning in the protein. The GAA structure reveals a surprising second sugar-binding site 26 '},\n"," {'paper_id': '3c77cb64a44e6582a425f583b7d64e332b5df5f9',\n","  'section': 'Abstract',\n","  'text': 'This paper presents a rapid evidence review into the clinical and psychological impacts of COVID-19 on perinatal women and their infants. Literature search revealed that there is very little formal evidence on the impact of COVID-19 on pregnant, labouring and postnatal women or their babies. The clinical evidence to date suggests that pregnant and childbearing women, and their babies are not at increased risk of either getting infected, or of having severe symptoms or consequences than the population as a whole. There is no evidence on the short-and longer-term psychological impacts of restrictive practices or social and personal constraints for childbearing women during COVID-19 in particular, or infection pandemics in general. The potential for adverse mental health consequences of the pandemic should be recognised as a critical public health concern, together with appropriate care and support to prevent and ameliorate any negative impacts. '},\n"," {'paper_id': '3c7968f094ff50819208287754b78b38a6399031',\n","  'section': 'Abstract',\n","  'text': 'SARS-CoV2 is a novel highly virulent pathogen which gains entry to human cells by binding with the cell surface receptor -angiotensin converting enzyme (ACE2). We contrasted the binding interactions between human ACE2 and coronavirus spike protein receptor binding domains (RBDs) from (i) SARS-CoV2, (ii) the related but less virulent OC43 (Singapore COVID-19 strain), and (iii) the 2002 epidemiccausing SARS-CoV1. We find that the RBDs of the spike protein of SARS-CoV2 are highly optimized to achieve the strongest possible binding interaction with ACE2 which is consistent with its enhanced pathogenicity. SARS-CoV2 RBDs form the most stable complex with ACE2 (64.91%, and 28.07% higher binding scores than SARS-CoV1 and OC43, respectively) while occupying the greatest number of residues in the ATR1 binding site. In fact, the spike protein RBDs from SARS-CoV2 out-compete the angiotensin 2 receptor type I (ATR1) which is the native binding partner of ACE2 by 64% in terms of binding affinity (quantified using the Rosetta scoring function). They accomplish this through strong electrostatic attachments with every fourth residue on the N-terminus alpha-helix (starting from Ser19 to Asn53) as the turn of the helix makes these residues solvent accessible. These results offer a computational explanation for the increased pathogenicity of SARS-CoV2 and allude to therapeutic modalities by identifying and rank-ordering the ACE2 residues involved in binding with the virus. '},\n"," {'paper_id': '3c94f582610453b6c1389c3a5d6ad57bcf0ba033',\n","  'section': 'Abstract',\n","  'text': \"confirmed and 1368 patients have died from the disease. However, the clinical characteristics of the dyed patients were still not clearly clarified. This study aims to summarize the clinical characteristics of death cases with COVID-19 and to identify critically ill patients of COVID-19 early and reduce their mortality. The clinical records, laboratory findings and radiologic assessments included chest X-ray or computed tomography were extracted from electronic medical records of 25 died patients with COVID-19 in Renmin Hospital of Wuhan University from Jan 14 to Feb 13, 2020. Two experienced clinicians reviewed and abstracted the data. The mean age of the dead was 71.48 ± 12.42 years, the average course of the disease was 10.56 ± 4.42 days, all patients eventually died of respiratory failure. All of those who died had underlying diseases, the most common of which was hypertension (16/25, 64%), followed by diabetes (10/25, 40%), heart diseases (8/25, 32%), kidney diseases (5/25, 20%), cerebral infarction (4/25, 16%), chronic obstructive pulmonary disease (COPD, 2/25, 8%), malignant tumors (2/25, 8%) and acute pancreatitis (1/25, 4%). The most common organ damage outside the lungs was the heart, followed by kidney and liver. In the patients' last examination before death, white blood cell and neutrophil counts were elevated in 17 patients (17/25, 68%) and 18 patients (18/25, 72%), lymphocyte counts were decreased in 22 patients (22/25, 88%). Most patients' PCT, CRP and SAA levels were elevated, the percentages were 90.5% (19/21), 85% (19/20) and 100% (21/21) respectively. The levels of the last test of neutrophils (15/16, 93.8%), PCT (11/11, 100%), CRP (11/13, 84.6%), cTnI (8/9, 88.9%), D-Dimer (11/12, 91.6%) and LDH (9/9, 100%) were increased as compared to the first test, while the levels of lymphocytes were decreased (14/16, 87.5%). Interpretation The age and underlying diseases (hypertension, diabetes, etc.) were the most All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. \"},\n"," {'paper_id': '3cda134b5b12d6c4f7e16658dded4e6e3db35c2a',\n","  'section': 'Abstract',\n","  'text': \"medRxiv preprint Question What are the risk factors for severe patients with corona virus disease 2019 ? Findings First in this review and meta-analysis, we found that elderly male patients with a high body mass index, high breathing rate and a combination of underlying diseases were more likely to develop into critically ill patients. second, compared with ordinary patients, severe patients had more significant symptom such as fever and dyspnea. Last, we also found that the laboratory test results of severe patients had more abnormal than non-severe patients. Meaning This review summaried the risk factors of severe COVID-19 patients and aim to provide a basis for early identification of severe patients by clinicians. Importance: With the increasing number of infections for COVID-19, the global health resources are deficient. At present, we don't have specific medicines or vaccines against novel coronavirus pneumonia (NCP) and our assessment of risk factors for patients with severe pneumonia was limited. In order to maximize the use of limited medical resources, we should distinguish between mild and severe patients as early as possible. Objective: To systematically review the evidence of risk factors for severe corona virus disease 2019 (COVID-19) patients. Evidence Review: We conducted a comprehensive search for primary literature in All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. : medRxiv preprint both Chinese and English electronic bibliographic data bases including China National Knowledge Infrastructure (CNKI), Wanfang, Weipu, Chinese Biomedicine Literature Database (CBM-SinoMed), MEDLINE (via PubMed), EMBASE, Cochrane Central Register, and Web of science. The American agency for health research and quality (AHRQ) tool were used for assessing risk of bias. Mata-analysis was undertaken using STATA version 15.0. Results: 20 articles (N=4062 participants) were eligible for this systematic review and meta-analysis. First in this review and meta-analysis, we found that elderly male patients with a high body mass index, high breathing rate and a combination of underlying diseases (such as hypertension, diabetes, cardiovascular disease, and chronic obstructive pulmonary disease) were more likely to develop into critically ill patients. second, compared with ordinary patients, severe patients had more significant symptom such as fever and dyspnea. Besides, the laboratory test results of severe patients had more abnormal than non-severe patients, such as the elevated levels of white-cell counts, liver enzymes, lactate dehydrogenase, creatine kinase, c-reactive protein and procalcitonin, etc, while the decreased levels of lymphocytes and albumin, etc. Interpretation: This is the first systematic review investigating the risk factors for severe corona virus disease 2019 (COVID-19) patients. The findings are presented and discussed by different clinical characteristics. Therefore, our review may provide guidance for clinical decision-making and optimizes resource allocation. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. \"},\n"," {'paper_id': '3d498994d143a4ed7cc381700a19f2790d46bb36',\n","  'section': 'Abstract',\n","  'text': 'In the analysis of genomic sequence data, so-called \"alignment free\" approaches are often 12 selected for their relative speed compared to alignment-based approaches, especially in the 13 application of distance comparisons and taxonomic classification 1,2,3,4 . These methods are 14 typically reliant on excising K-length substrings of the input sequence, called K-mers 5 . In 15 the context of machine learning, K-mer based feature vectors have been used in 16 applications ranging from amplicon sequencing classification to predictive modeling for 17 antimicrobial resistance genes 6,7,8 . This can be seen as an analogy of the \"bag-of-words\" 18 model successfully employed in natural language processing and computer vision for 19 document and image classification 9,10 . Feature extraction techniques from natural language 20 processing have previously been analogized to genomics data 11 ; however, the \"bag-of-21 words\" approach is brittle in the RNA virus space due to the high intersequence variance 22 and the exact matching requirement of K-mers. To reconcile the simplicity of \"bag-of-23 words\" methods with the complications presented by the intrinsic variance of RNA virus 24 space, a method to resolve the fragility of extracted K-mers in a way that faithfully reflects 25 an underlying biological phenomenon was devised. Our algorithm, Vorpal, allows the 26 construction of interpretable linear models with clustered, representative \\'degenerate\\' K-27 mers as the input vector and, through regularization, sparse predictors of binary 28 phenotypes as the output. Here, we demonstrate the utility of Vorpal by identifying 29 nucleotide-level genomic motif predictors for binary phenotypes in three separate RNA 30 virus clades; human pathogen vs. non-human pathogen in Orthocoronavirinae, 31 hemorrhagic fever causing vs. non-hemorrhagic fever causing in Ebolavirus, and human-32 host vs. non-human host in Influenza A. The capacity of this approach for in silico 33 identification of hypotheses which can be validated by direct experimentation, as well as 34 identification of genomic targets for preemptive biosurveillance of emerging viruses, is 35 discussed. The code is available for download at https://github.com/mriglobal/vorpal. 36 37 Feature Extraction Algorithm Overview 38 In the quasispecies model, the virus organism is represented by the \"cloud\" of genotypes that can 39 be maintained by the virus within the allowable fitness parameters 12 . In the method proposed 40 '},\n"," {'paper_id': '3d5093a8e079fb17262111e5ef94bf01a3007971',\n","  'section': 'Abstract',\n","  'text': 'However, prevalence data are scarce in migrant populations. This study aims at evaluating the prevalence of S. stercoralis at hospital level in migrant populations or long term travellers being attended in out-patient and in-patient units as part of a systematic screening implemented in 6 Spanish hospitals. A cross-sectional study was conducted and systematic screening for S. stercoralis infection using serological tests was offered to all eligible participants. The overall seroprevalence of S. stercoralis was 9.04% (95% confidence interval [95%CI] 7.76 -10.31). The seroprevalence of people with a risk of infection acquired in Africa and Latin America was 9.35% (95%CI 7.01-11.69), 9.22% (7.5-10.93), respectively. The number of individuals coming from Asian countries was significantly smaller and the overall prevalence in these countries was 2.9% (95%CI -0.3; -6.2). There was only one case (1/14 (7.14%) from an individual from East European countries. The seroprevalence in units attending potentially immunosuppressed patients was significantly lower (5.64%) compared with the seroprevalence in other units of the hospital (10.20%) or Tropical diseases units (13.33%) (p<0.001). Conclusions: We report a hospital-based systematic screening of strongyloidiasis with a seroprevalence of almost 10% in a mobile population coming from endemic areas suggesting the need of implementing strongyloidiasis screening in hospitalized patients coming from endemic areas, particularly if they are at risk of immunosuppression. Strongyloidiasis is an infection caused by the helminth Strongyloides stercoralis which is ubiquitous in tropical and subtropical areas. In the rest of the countries, it is also frequent in migrants coming from tropical and subtropical areas. The disease is more severe when an infected subject has an impaired immune system. Within this study we have evaluated the prevalence of this infection in people being attended in six Spanish hospitals. The prevalence was around 9%, being higher in Africa and Latin America compared with other regions. In addition, the prevalence in patients with an impaired immune system (immunosuppression) was lower compared with people non suffering immunosuppression. These results suggest that the prevalence of strongyloidiasis is quite high among migrants living in Spain and that a screening programme should be designed, particularly in immunosuppressed patients that are at more risk of suffering severe complications of the infection. '},\n"," {'paper_id': '3d690de86ca6e200f4f9cac61b004e47ad753aae',\n","  'section': 'Abstract',\n","  'text': \"Coronaviruses (CoV) are positive-sense RNA viruses that infect numerous mammalian and avian 23 species and are capable of causing severe and lethal disease in humans. CoVs encode several 24 innate immune antagonists that interact with the host innate immune response to facilitate 25 efficient viral replication. CoV non-structural protein 14 (nsp14) encodes 3'-to-5' 26 exoribonuclease activity (ExoN), which performs a proofreading function and is required for 27 high-fidelity replication. Outside of the order Nidovirales, arenaviruses are the only RNA viruses 28 that encode an ExoN, which functions to degrade dsRNA replication intermediates. In this study, 29 we tested the hypothesis that CoV ExoN may also function to antagonize the innate immune 30 response. We demonstrate that viruses lacking ExoN activity [ExoN(-)] are sensitive to cellular 31 pretreatment with interferon beta (IFN-β) in a dose-dependent manner. In addition, ExoN(-) virus 32 replication was attenuated in wild-type bone marrow-derived macrophages (BMMs) and partially 33 restored in interferon alpha/beta receptor deficient (IFNAR-/-) BMMs. ExoN(-) virus replication 34 did not result in IFN-β gene expression, and in the presence of an IFN-β-mediated antiviral state, 35 ExoN(-) viral RNA levels were not substantially reduced relative to untreated. However, ExoN(-36 ) virus generated from IFN-β pretreated cells had reduced specific infectivity and decreased 37 relative fitness, suggesting that ExoN(-) virus generated during an antiviral state is less viable to 38 establish a subsequent infection. Overall, our data suggest MHV ExoN activity is required for 39 resistance to the innate immune response and antiviral mechanisms affecting the viral RNA 40 sequence and/or an RNA modification act on viruses lacking ExoN activity. 41 42 43 All rights reserved. No reuse allowed without permission. \"},\n"," {'paper_id': '3da1a03cb675f463b0d8e30744aa33512241dd29',\n","  'section': 'Abstract',\n","  'text': 'Pneumonia patients of 2019-ncov in latent period are not easy to be effectively quarantined, but there is evidence that they have strong infectious ability. Here, the infectious ability of patients during the latent period is slightly less than that of the infected patients was assumed. We established a new SEIR propagation dynamics model, that considered the weak transmission ability of the incubation period, the variation of the incubation period length, and the government intervention measures to track and isolate comprehensively. Based on the raw epidemic data of China from January 23, 2020 to February 10, 2020, the dynamic parameters of the new present SEIR model are fitted. Through the Euler integration algorithm to solve the model, the effect of infectious ability of incubation patients on the theoretical estimation of the present SEIR model was analyzed, and the occurrence time of peak number in China was predicted. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '3ddc987158400d3867939d3594da56dd546d6ddd',\n","  'section': 'Abstract',\n","  'text': \"Transmission between hosts is a critical part of the viral lifecycle. Recent studies of viral transmission have 9 used genome sequence data to evaluate the number of particles transmitted between hosts, and the role 10 of selection as it operates during the transmission process. However, the interpretation of sequence data 11 describing transmission events is a challenging task. We here present a novel and comprehensive frame-12 work for using short-read sequence data to understand viral transmission events. Our model describes 13 transmission as an event involving whole viruses, rather than independent alleles. We demonstrate how 14 selection and noisy sequence data may each affect inferences of the population bottleneck, and identify 15 circumstances in which selection for increased viral transmission may or may not be identified. Applying 16 our model to data from a previous experimental transmission study, we show that our approach grants a 17 more quantitative insight into viral transmission, inferring that between 2 to 6 viruses initiated infection, 18 and allowing for a more informed interpretation of transmission events. While our model is here applied 19 to influenza transmission, the framework we present is highly generalisable to other systems. Our work 20 provides new opportunities for studying viral transmission. 21 Introduction Understanding viral transmission is a key task for viral epidemiology. The extent to which a virus is able 23 to transmit between hosts determines whether it is likely to cause sporadic, local outbreaks, or spread 24 to cause a global pandemic [1, 2]. In a transmission event, the transmission bottleneck, which specifies 25 the number of viral particles founding a new infection, influences the amount of genetic diversity that is 26 retained upon transmission, with important consequences for the evolutionary dynamics of the virus [3,4]. Recent studies have used genome sequencing approaches to study transmission bottlenecks in influenza 29 populations. In small animal studies, the use of neutral genetic markers has shown that the transmission 30 bottleneck is dependent upon the route of transmission, whether by contact or aerosol transmission [5,6]. 31 In natural human influenza populations, where modification of the virus is not possible, population genetic 32 methods have been used to analyse bottleneck sizes. Analyses of transmission have employed different 33 approaches, exploiting the observation or non-observation of variant alleles [7] or using changes in allele 34 frequencies to characterise the bottleneck under a model of genetic drift [8-11]. A recent publication 35 improved this latter model, incorporating the uncertainty imposed upon allele frequencies by the process 36 of within-host growth [12]. Two studies of within-household influenza transmission have provided strik-37 ingly different outcomes in the number of viruses involved in transmission, with estimates of 1-2 [13] and 38 100-200 [14] respectively. 39 40 Another focus of research has been the role of selection during a transmission event; this is important 41 in the context of the potential for new influenza strains to become transmissible between mammalian 42 hosts [15, 16]. Studies examining transmissibility have assessed the potential for different strains of in-43 fluenza to achieve droplet transmission between ferrets under laboratory conditions [17-20]; ferrets provide 44 . CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/302331 doi: bioRxiv preprint a useful, if imperfect, model for transmission between humans [21, 22]. The application of bioinformatic 45 techniques to data from these experiments has identified 'selective bottlenecks' in the experimental evolu-46 tion of these viruses [23, 24], whereby some genetic variants appear to be more transmissible than others. In these studies, selection has been considered in terms of the population diversity statistic π; changes 48 in π N /π S , the ratio between non-synonymous and synonymous diversity, have been used to evaluate 49 patterns of selection across different viral segments. 50 51 We here note the need for a greater clarity of thinking in the analysis of viral transmission events. For 52 example, analysis of genetic variants in viral populations shows that synonymous and non-synonymous 53 mutations both have fitness consequences for viruses [25, 26]; the use of synonymous variants as a neu-54 tral reference set may not hold. More fundamentally, in an event where the effective population size is 55 small, the influences of selection and genetic drift may be of similar magnitude [27]. However selection 56 is assessed, this implies a need to separate stochastic changes in a population from selection, especially 57 where a transmission bottleneck may include only a small number of viruses [5, 13, 28]. The attribution 58 of a change in diversity to the action of selection or the attribution of changes in allele frequencies to 59 genetic drift could both potentially be flawed. Given the increasing availability of sequence data, more 60 sophisticated tools for the analysis of viral transmission are required. 61 62 Here we note three challenges in the analysis of data from viral transmission events. Firstly, selection 63 can produce changes in a population equivalent to those arising through a neutral population bottle-64 neck [29] (Figure 1A), making it necessary to distinguish between the two scenarios. A broad literature 65 has considered the simultaneous inference of the magnitude of selection acting upon a variant along with 66 an effective population size [30-35]. However, such approaches rely on the observation of an allele fre-67 quency at more than two time points so as to distinguish a deterministic model of selection (with an 68 implied infinite effective population size) from a combined model of selection with genetic drift; such 69 approaches cannot be directly applied to the analysis of viral transmission. 70 71 Secondly, inferences of transmission events need to account for the haplotype structure of viral pop-72 ulations, whereby whole viruses, rather than sets of independent alleles, are transmitted (Figure 1B). 73 The low rate of homologous recombination in segments of the influenza virus [36, 37] implies that viral 74 evolution proceeds at the haplotype level [38]; competition occurs between collections of linked alleles, or 75 segments, rather than the individual alleles themselves. Under such circumstances, fitter variants do not 76 always increase in frequency within a population [39-41]. Calculations of genetic drift, which are often 77 derived from the evolution of independent variants [42], need to be adjusted to account for this more 78 complex dynamics. 79 80 Thirdly, noise in the measurement of a population may influence the inferred size of a transmission 81 bottleneck (Figure 1C). A broad range of studies have examined the effect of noise in variant calling 82 and genome sequence analysis [43-50]; more recently formulae have been proposed to measure the preci-83 sion with which allele frequencies can be defined given samples from a population [51-53]. Where small 84 changes in allele frequencies are used to assess a population bottleneck, it is important to separate the 85 effects of noise in the measurement of populations from genuine changes in a population. 86 87 We here describe a novel method for the inference of population bottlenecks which resolves the above 88 issues. Our approach correctly evaluates changes in a population even where the data describing that 89 change is affected by noise. It explicitly accounts for the haplotype structure of a population, utilising the 90 data present within short sequence reads. Further, where these factors can be discriminated, our method 91 distinguishes between the influences on the population of selection and the transmission bottleneck. 92 Studies of viral evolution have highlighted the potential for payoffs between within-host viral growth and 93 . CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/302331 doi: bioRxiv preprint transmissibility [54]; given sufficient data we can evaluate how selection operates upon each of these two 94 phenotypes. Our model extends previous population genetic work on bottleneck inference to provide a 95 generalised model for the analysis of data spanning viral transmission events. 96 Results 97 Model outline 98 In the recent literature, the term 'bottleneck' has been applied to describe a reduction in the genetic 99 diversity of a population (e.g. [55]), whether arising from selection or a numerical reduction in the size of 100 a population. Here, we define a 'bottleneck' more strictly as a neutral process whereby a finite number 101 of viral particles from one population found a subsequent generation of the population, either within the 102 same host, or across a transmission event from host to recipient. Selection then constitutes a modification 103 to this process whereby some viruses, because of their genotype, have a higher or lower probability of 104 making it through the bottleneck to found the next generation. We applied a population genetic method to make a joint inference of the bottleneck size and the 107 extent of selection acting during a transmission event. We consider a scenario in which a viral population 108 is transmitted from one host to another, with samples being collected before and after the transmission 109 event (Figure 2A ). In our model viruses are categorised as haplotypes, that is, according to the variants 110 they contain at a subset of positions in their genome. As such, the viral population is represented as 111 a vector of haplotype frequencies; the population before transmission is represented by the vector q B . 112 During transmission, a random sample of N T viruses are passed on to the second host to give the founder 113 population q F . Selection for transmissibility, whereby genetic variants cause some viruses to be more 114 transmissible than others, is described by the function S T . The potentially small size of the founder pop-115 ulation means that the population evolves within the host under the influence of genetic drift to create 116 the large post-transmission population q A ; this process is approximated in our model by a population 117 bottleneck of effective size N G . Selection acting for within-host growth may further alter the genetic 118 composition of the population; this effect is described by the function S G . Our method thus allows for a 119 discrimination to be made between selection for increased within-host replication and for increased viral 120 transmissibility. Observations of the population are collected before and after transmission via a noisy 121 sequencing process to give the datasets x A and x B . The extent of noise in the sampling and sequencing is 122 characterised by the parameter C; a value of C = 0 would indicate that samples contain no information, 123 while a large C corresponds to noiseless sampling, as in a binomial model. 124 125 To summarise our approach, we note that both the transmission and within-host growth events can 126 be represented as sampling processes, which may each be biased by the effect of selection. As such, given 127 an estimate of the noise inherent to the sequencing process, and externally-derived estimates for N G and 128 S G , we can calculate an approximate likelihood for the parameters N T and S T given the observations 129 x B and x A . Maximising this likelihood gives an estimate for the size of the transmission bottleneck 130 and the extent to which specific genetic variants within the pre-transmission population confer increased 131 transmissibility upon viruses. 132 133 In our model we discriminate between changes in a population arising from selection and those aris-134 ing due to the population bottleneck. This is achieved by considering regions of the genome between 135 which recombination or reassortment has removed linkage disequilibrium between alleles ( Figure 2B ). As 136 transmission involves whole viruses, the bottleneck N T is preserved between regions. Meanwhile, in the 137 absence of epistasis, selection acting upon one region of the virus does not influence the composition of the 138 population in other parts of the genome. As such, a cross-region calculation estimates both N T and the 139 . doi: bioRxiv preprint 157 involving the absolute number of reads of a genome position, which specifies the depth of noise-free reads 158 that would produce the same degree of certainty in an allele frequency. This statistic is dominated by the 159 smaller of C and the absolute read depth; for large C this statistic is close to the absolute read depth, 160 while as the number of reads describing a variant frequency becomes large, the effective depth tends to 161 C + 1. Inferences of bottleneck sizes showed a limit on the inferred bottleneck size governed by noise in se- \"},\n"," {'paper_id': '3deb6393bd43f440a69c84da9ed61986b1781d36',\n","  'section': 'Abstract',\n","  'text': 'The ongoing novel coronavirus pneumonia COVID-19 outbreak in Wuhan, China, has engendered 25 numerous cases of infection and death. COVID-19 diagnosis relies upon nucleic acid detection; 26 however, current recommended methods exhibit high false-negative rates, low sensitivity, and 27 cannot identify other respiratory virus infections, thereby resulting patient misdiagnosis and 28 impeding epidemic containment. Combining the advantages of target amplification and long-read, 29 real-time nanopore sequencing, we developed nanopore target sequencing (NTS) to detect SARS-30 '},\n"," {'paper_id': '3e5633a9a76a8c03e4ec57e4659872a63f045117',\n","  'section': 'Abstract',\n","  'text': 'The present outbreak of lower respiratory tract infections, including respiratory distress syndrome, is the third spillover, in only two decades, of an animal coronavirus to humans resulting in a major epidemic. Here, the Coronavirus Study Group (CSG) of the International Committee on Taxonomy of Viruses, which is responsible for developing the official classification of viruses and taxa naming (taxonomy) of the Coronaviridae family, assessed the novelty of the human pathogen tentatively named 2019-nCoV. Based on phylogeny, taxonomy and established practice, the CSG formally recognizes this virus as a sister to severe acute respiratory syndrome coronaviruses (SARS-CoVs) of the species Severe acute respiratory syndrome-related coronavirus and designates it as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). To facilitate communication, the CSG further proposes to use the following naming convention for individual isolates: SARS-CoV-2/Isolate/Host/Date/Location. The spectrum of clinical manifestations associated with SARS-CoV-2 infections in humans remains to be determined. The independent zoonotic transmission of SARS-CoV and SARS-CoV-2 highlights the need for studying the entire (virus) species to complement research focused on individual pathogenic viruses of immediate significance. This research will improve our understanding of virus-host interactions in an ever-changing environment and enhance our preparedness for future outbreaks. : bioRxiv preprint Defining novelty and the place of SARS-CoV-2 within the taxonomy of the Coronaviridae family During the 21 st century, researchers studying coronaviruses -a family of enveloped positivestranded RNA viruses of vertebrates 8 -were confronted several times with the question of coronavirus novelty, including two times when a severe or even life-threatening disease was introduced into humans from a zoonotic reservoir: this happened with severe acute respiratory '},\n"," {'paper_id': '3e6dda036f3d57c4b1b5b318225ce2a1fccb6ab7',\n","  'section': 'Abstract',\n","  'text': 'Coronavirus disease 2019 , caused by SARS-CoV-2, has rapidly spread throughout China, but the clinical characteristics of Tibetan patients living in the Qinghai-Tibetan plateau are unknown. We aimed to investigate the epidemiological, clinical, laboratory and radiological characteristics of these patients. We included 67 Tibetan patients with confirmed SARS-CoV-2 infection. The patients were divided into two groups based on the presence of clinical symptoms at admission, with 31 and 36 patients in the symptomatic and asymptomatic groups, respectively. The epidemiological, clinical, laboratory and radiological characteristics were extracted and All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint (which was not peer-reviewed) is . https://doi.org/10.1101/2020.03.09.20033126 doi: medRxiv preprint analysed. No patient had a history of exposure to COVID-19 patients from Wuhan or had travelled to Wuhan. The mean age of Tibetan patients was 39.3 years and 59% of the patients were male. Seven patients presented with fever on admission and lymphocytopenia was present in 20 patients. 47 patients had abnormal chest CTs at admission instead of stating that 20 were unchanged. Lactate dehydrogenase levels were increased in 31 patients. Seven patients progressed to severe COVID-19; however, after treatment, their condition was stable. No patients died. Of the 36 asymptomatic patients, the mean age was younger than the symptomatic group (34.4±17.3vs 44.9±18.1 years, P=0.02). Lymphocyte count and prealbumin levels were higher in the asymptomatic group than the group with clinical symptoms (1.6±0.5 vs 1.3±0.6 and 241.8±68.2 vs 191.9±60.3, respectively; P<0.05). Imported cases of COVID-19 in Tibetan patients were generally mild in this high-altitude area. Absence of fever or radiologic abnormalities on initial presentation were common. '},\n"," {'paper_id': '3ebbcdddb122f405061bb5e1ddcf94abe9258d58',\n","  'section': 'Abstract',\n","  'text': 'In December 2019, the first cases of a novel coronavirus infection were diagnosed in Wuhan, China. Due to international travel and human-to-human transmission, the virus spread rapidly inside and outside of China. Currently, there is no effective antiviral treatment for COVID-19, therefore research efforts are focused on the rapid development of vaccines and antiviral drugs. The SARS-CoV-2 M pro protease constitutes one of the most attractive antiviral drug targets. To address this emerging problem, we have synthesized a combinatorial library of fluorogenic substrates with glutamine in the P1 position. We used it to determine the substrate preferences of the SARS-CoV and SARS-CoV-2 proteases, using natural and a large panel of unnatural amino acids. The results of our work provide a structural framework for the design of inhibitors as antiviral agents or diagnostic tests. '},\n"," {'paper_id': '3ef0df1127178cb56f40a4e4fcb6a39b82df6730',\n","  'section': 'Abstract',\n","  'text': 'Background 35 The reasons for the large differences between countries in the sizes of their SARS 36 CoV-2 epidemics is unknown. Individual level studies have found that the use of face 37 masks was protective for the acquisition and transmission of a range of respiratory 38 viruses including SARS CoV-1. We hypothesized that population level usage of face 39 masks may be negatively associated SARS CoV-2 spread. 40 41 At a country level, linear regression was used to assess the association between 43 '},\n"," {'paper_id': '3f401643493c879fb1660830d890a694206d6a1b',\n","  'section': 'Abstract',\n","  'text': 'Following its emergence in Wuhan, '},\n"," {'paper_id': '3fac0b7a9917984764a77f7fcb327e820e770d1b',\n","  'section': 'Abstract',\n","  'text': 'Herein we describe the Archaea diversity in a shallow pond in the Cuatro Ciénegas Basin (CCB), 24 Northeast Mexico, with fluctuating hypersaline conditions containing elastic microbial mats that 25 can form small domes where their anoxic inside reminds us of the characteristics of the Archaean 26 Eon, rich in methane and sulfur gases; thus, we named this site the Archaean Domes (AD). These 27 domes only form after heavy rains that are rare in the Chihuahuan desert. CCB is a unique oasis 28 with hundreds of ponds, containing endemic species of animals, plants and highly diverse and 29 unique microbial communities, despite its very biased stoichiometry, due mostly to extreme low 30 phosphorus content (soils, water columns and sediments). This extreme oligotrophy has favored 31 survival of ancestral microorganisms. Whole metagenome sequencing approach was performed for 32 this unusual site in three different seasons to assess the extent of the Archaea biodiversity, with a 33 focus on extremophiles, since members of the Archaea had been underrepresented in different study 34 sites within the oasis. We found a highly diverse Archaea community compassing ~5% of the 35 metagenomes. The archaeal portion in all three metagenomes maintained its abundance and most of 36 the strains showed to form a resilient core during three seasonal samplings (2016-2017), despite 37 environmental fluctuations. However, relative abundances of all 230 archaeal OTUs (defined using 38 a 97% cutoff) were low enough (<0.1%) to be considered part of the rare biosphere. AD finding and 39 their description within CCB confirms that this particular pond is the most diverse for Archaea that 40 we are aware of and opens new paths for understanding the forces that once drove and keep shaping 41 microbial community assemblage. 42 43 44 45 . CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/766709 doi: bioRxiv preprint Archaea domain remains largely unexplored [14] along with a diverse metabolism [15, 16], 64 unveiling new processes and key features involving microbes and community structure. 65 Cuatro Cienegas Basin (CCB) is an endangered oasis within the northern zone of the 66 Chihuahuan Desert in Mexico, characterized by an extremely unbalanced nutrient stoichiometry of 67 the area (N:P = 159:1), similar to the conditions of the Precambrian sea [17-21]. Strikingly, despite 68 . CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/766709 doi: bioRxiv preprint this nutrient deficiency CCB is considered a biodiversity hotspot for macroorganisms [22] and one 69 of the most diverse sites for microorganisms in the world [23][24][25][26][27]; this microbial biodiversity is 70 mirrored by its extreme diversity in virus [28]. CCB biodiversity is believed to have evolved as a 71 result of a long time environmental stability of a deep aquifer [29], as suggested by the marine 72 affiliations in many of the studied bacterial genomes [25, 30], virus [31], and probably, Archaea. 73 These observations led CCB scientists to raise hypothesis and propose CCB as a model of \"lost 74 world\", where extreme conditions favored the survival of ancestral marine lineages that in some 75 cases persist exclusively in this area [32]. Nevertheless, Archaea in CCB has been poorly studied 76 since they were underrepresented in previous metagenomic studies [24, 28, 33, 34]. 77 During a late March 2016 field work, an atypical spring rain apparently dissolved the salty 78 crust of an unnoticed shallow pond, allowing the uniquely flexible and impermeable microbial mats 79 to arise from the ground building bubbles, or domes with a strong sulfur-like smell. Our hypothesis 80 was that gas production of methanogenic as well as other microorganisms associated to the sulfur 81 cycle were degassing creating a locally anaerobic atmosphere. Therefore, the anaerobic layer of the 82 microbial mats were creating the domes by raising the flexible and impermeable mat where 83 photosynthesis was evident given the purple, and green layers, creating an unique \"alien\" local 84 landscape ( Figure 1 ). Based on the macroscopic morphology of the microbial mats, this locality in 85 the Pozas Azules ProNatura ranch (a private ecological reserve) was named by us the \"Archean 86 Domes\" (AD thereafter) (Fig. 1a) . 87 These unusual structures in an small extremophilic and fluctuating pond, are by themselves a 88 singularity, within CCB unique nature and amazing microbial diversity that has fascinated scientists 89 with unexpected findings [19-21, 25, 27, 29, 30, 33, 35-43]. High salt concentration in dry 90 conditions, and the production of biogases along dome formation under wet conditions, suggested 91 . CC-BY-NC-ND 4.0 International license is made available under a '},\n"," {'paper_id': '3fb67d663ace1f95a9228d83bf11ff675d0fbff2',\n","  'section': 'Abstract',\n","  'text': 'In December 2019, a cluster of acute respiratory illness, now known as SARS-CoV-2 pneumonia, occurred in Wuhan, China. World Health Organization (WHO) declared the rapidly spreading coronavirus outbreak a pandemic on March 11, 2020, acknowledging what has seemed clear for some time -the virus will likely spread to all countries on the globe. As of February 11, 2020, the Chinese Center for Disease Control and Prevention (China CDC) has officially reported that there are 2.0% (889) asymptomatic cases, 2.3% (1,023) death cases, and 80.9% mild cases among 44,672 confirmed cases. 51.4% (22,981) were male and 48.6% (21,691) were female. Lymphopenia, in particular T lymphopenia, was common among patients with SARS-COV-2 in the observation. A notable drop in CD4 and CD8 lymphocyte counts occurred early in the course of the syndrome and was associated with adverse outcomes. The appearing a phenomenon of lymphocyte depletion (PLD) suggested severe adverse outcomes. The outcome observed: 60% had discharged and 20% had die. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '3fd7868a52df3ae502c64a8318f45eb38a483913',\n","  'section': 'Abstract',\n","  'text': 'The main target of COVID-19 is the lungs where it may cause pneumonia in severely ill patients. Chest X-ray is an important diagnostic test to assess the lung for the damaging effects of COVID-19. Many other microbial pathogens can also cause damage to lungs leading to pneumonia but there are certain radiological features which can favor the diagnosis of pneumonia caused by COVID-19. With the rising number of cases of COVID-19, it would be imperative to develop computer programs which may assist the health professionals in the prevailing scenario. Materials & Methods : A total of two hundred and seventy eight (278) images of chest X-rays have been assessed by applying ResNet-50 convolutional neural network architectures in the present study. The digital images were acquired from the public repositories provided by University of Montreal and National Institutes of Health. These digital images of Chest X-rays were divided into three groups labeled as normal, pneumonia and COVID-19. The third group contains digital images of chest X-rays of patients diagnosed with COVID-19 infection while the second group contains images of lung with pneumonia caused by other pathogens. Results : The radiological images included in the data set are 89 images of lungs with COVID-19 infection, 93 images of lungs without any radiological abnormality and 96 images of patient with pneumonia caused by other pathogens. In this data set, 80% of the images were employed for training, and 20% for testing. A pre-trained (on ImageNet data set) ResNet-50 architecture was used to diagnose the cases of COVID-19 infections on lung Xray images. The analysis of the data revealed that computer vision based program achieved diagnostic accuracy of 98.18 %, and F1-score of 98.19. Conclusion : The performance of convolutional neural network regarding the differentiation of pulmonary changes caused by COVID-19 from the other type of pneumonias on digital images of the chest X-rays is excellent and it may be an extremely useful adjunct tool for the health professionals. '},\n"," {'paper_id': '3fe2552d3d8e6c9ead2c41e30fbb69e917277f53',\n","  'section': 'Abstract',\n","  'text': 'Background: Since mid-December 2019, a cluster of pneumonia-like diseases caused by a novel coronavirus, now designated COVID-19 by the WHO, emerged in Wuhan city and rapidly spread throughout China. Here we identify the clinical characteristics of COVID-19 in a cohort of patients in Shanghai. Methods: Cases were confirmed by real-time RT-PCR and were analysed for demographic, clinical, laboratory and radiological features. Of 198 patients, the median duration from disease onset to hospital admission was 4 days. The mean age of the patients was 50.1 years, and 51.0% patients were male. The most common symptom was fever. Less than half of the patients presented with respiratory systems including cough, sputum production, itchy or sore throat, shortness of breath, and chest congestion. 5.6% patients had diarrhoea. On admission, T lymphocytes were decreased in 45.8% patients. Ground glass opacity was the most common radiological finding on chest computed tomography. 9.6% were admitted to the ICU because of the development of organ dysfunction. Compared with patients not treated in ICU, patients treated in the ICU were older, had longer waiting time to admission, fever over 38.5 o C, dyspnoea, reduced T lymphocytes, elevated neutrophils and organ failure. In this single centre cohort of COVID-19 patients, the most common symptom was fever, and the most common laboratory abnormality was decreased blood T cell counts. Older age, male, fever over 38.5 o C, symptoms of dyspnoea, and underlying comorbidity, were the risk factors most associated with severity of disease. Patients. We obtained epidemiological, demographic, clinical, laboratory and management data from the medical records of patients infected with SARS-Cov-2. On Jan 20, 2020, the first human case of in Shanghai was confirmed. Since then all hospitals in Shanghai have opened special fever clinics to screen suspected patients, and laboratory confirmed patients were then admitted to a single designated hospital in Shanghai (Shanghai Public Health Clinical Centre). Laboratory confirmation of COVID-19 was done by the Chinese Centre for Disease Control and Prevention. Throat-swab specimens from the upper respiratory tract were obtained from all patients at admission and maintained in viral transport medium. COVID-19 was confirmed by real-time RT-PCR using the same protocol as described previously 3 . Confirmed patients were hospitalized into negative pressure wards for further medical observation and treatment. We collected data from patients who were admitted from Jan. 20 up to Feb. 15. All the data collected from the included cases have been shared with the WHO. Data Collection. Epidemiological exposure data, patient characteristics, clinical symptoms, laboratory and imaging findings and medical history were extracted from electronic medical records and analysed by licensed physicians. Laboratory data were recorded in standardized form. Initial investigations included a complete blood count, routine urinalysis, blood gases, coagulation function, erythrocyte sedimentation rate (ESR), C-reactive protein (CRP) and serum biochemical testing (including renal and liver function, serum lactate, lactate dehydrogenase, and electrolytes). To characterize the effect of coronavirus on the immune system, immunologic factors including serum immunoglobulin, complements, cytokines, rheumatoid factor (RF) and T cell and leukocyte subpopulations were analysed by flow cytometry. Non-laboratory information, including patient characteristics, epidemiological and medical history, were obtained and recorded by licensed physicians from direct interviews of patients and their relatives. Two researchers independently reviewed the data collection forms to review the data extracted. Written informed All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '402041df2d9812dbcdae08153d0350eee170eea9',\n","  'section': 'Abstract',\n","  'text': 'Translational control of memory processes is a tightly regulated process where the coordinated interaction and 47 modulation of translation factors provides a permissive environment for protein synthesis during memory 48 formation. Existing methods used to block translation lack the spatiotemporal precision to investigate cell-49 specific contributions to consolidation of long-term memories. Here, we have developed a novel chemogenetic 50 mouse resource for cell type-specific and drug-inducible protein synthesis inhibition (ciPSI) that utilizes an 51 engineered version of the catalytic kinase domain of dsRNA-activated protein (PKR). ciPSI allows rapid and 52 reversible phosphorylation of eIF2α causing a block on general translation by 50% in vivo. Using this resource, 53 we discovered that temporally structured pan-neuronal protein synthesis is required for consolidation of long-54 term auditory threat memory. Targeted protein synthesis inhibition in CamK2α expressing glutamatergic 55 neurons in lateral amygdala (LA) impaired long-term memory, which was recovered with artificial chemogenetic 56 reactivation at the cost of stimulus generalization. Conversely, genetically reducing phosphorylation of eIF2α in 57 CamK2α positive neurons in LA enhanced memory strength, but was accompanied with reduced memory 58 fidelity and behavior inflexibility. Our findings provide evidence for a finely tuned translation program during 59 consolidation of long-term threat memories. Memory is the capacity of an organism to encode, store, and retrieve information, and often guides the 83 action of the organism towards a better survival outcome. Aversive life-threatening events often lead to long-84 term associative memories between the environment in which those events were experienced in and the 85 threat, such that a salient cue from the event when presented again can elicit species-specific defensive 86 behaviors. Pavlovian cued threat conditioning is a useful experimental paradigm for understanding the 87 biological substrates of an associative threat memory, in which a neutral cue (e.g. light or tone) co-presented 88 with an innately aversive stimulus (e.g. air puff or footshock) elicits a defensive response such as freezing by a 89 repeat presentation by itself 1 . Cued threat conditioning is particularly amenable to studying memory 90 consolidation process because one-trial training is sufficient to form a persistent long-term memory and a 91 unimodal cue can be used for memory retrieval. Long-term aversive memories are thought to recruit a 92 distributed network of neurons across the brain, including subnuclei within amygdala, hippocampus, thalamus, 93 as well as the neocortex, depending on the brain state, cue complexity, and the sensory pathways engaged 2, 3 . Lateral amygdala is the central sensory gateway for the amygdaloid complex and is crucially engaged both in 95 the processing and storage of the associative aversive memories 4, 5, 6 . Decades of studies have reported that consolidation of long-term memories requires one or more 98 waves of new protein synthesis after the memory is first encoded 7, 8, 9, 10 ; however, the use of pharmacological 99 protein synthesis inhibitors (PSI) in these studies is burdened with limitations. For instance, an extensively 100 used PSI, anisomycin, causes unwanted side effects that include superinduction of immediate early genes, 101 activation of stress signaling pathways, and catecholamine release 11, 12 . Pharmacological PSIs also do not 102 discriminate between molecularly heterogeneous cell populations and thus, do not allow manipulation of cell 103 autonomous protein synthesis. Genetic approaches to alter the translation machinery at the initiation step 104 provide less ambiguous evidence for the role of protein synthesis in memory processes 13, 14 . However 105 constitutive deletion of genes encoding translational control molecules lacks the temporal control needed to 106 convincingly interrogate the role of protein synthesis in memory consolidation. Past attempts at making 107 chemogenetic tools for blocking protein synthesis have had limited success 15 . A key regulatory step in mammalian protein synthesis is the assembly of active eIF2.GTP and initiator-110 methionyl tRNA into a ternary complex that binds the 40S ribosome. Various types of cellular stress engage 111 pathway-specific protein kinases that phosphorylate the α subunit of eIF2 on serine 51, which converts eIF2 112 from a substrate to a competitive inhibitor of eIF2B, a guanine exchange factor that promotes the conversion of 113 inactive eIF2.GDP to active eIF2.GTP. Thus, phosphorylation of eIF2α stops the recycling of the ternary 114 complex and inhibits general translation 16 . However, phosphorylation of eIF2α also leads to a paradoxical 115 increase in gene-specific translation of transcripts harboring upstream open reading frames (uORFs) by 116 overcoming the inhibitory effects of uORFs on reinitiation at the putative start codon as ternary complexes 117 decline 17 . Dephosphorylation of eIF2α occurs during L-LTP as well as during consolidation and reconsolidation '},\n"," {'paper_id': '4025115c646713cd5fc71cdf3c96cc235152d9b0',\n","  'section': 'Abstract',\n","  'text': 'Human infections with a novel coronavirus (SARS-CoV-2) were first identified via syndromic surveillance in December of 2019 in Wuhan China. Since identification, infections (coronavirus disease-2019; COVID-19) caused by this novel pathogen have spread globally, with more than 250,000 confirmed cases as of March 21, 2020. An open-label clinical trial has just concluded, suggesting improved resolution of viremia with use of two existing therapies: hydroxychloroquine (HCQ) as monotherapy, and in combination with azithromycin (HCQ-AZ). [1, 2]. The results of this important trial have major implications for global policy in the rapid scale-up and response to this pandemic. The authors present results with p-values for differences in proportions between the study arms, but their analysis is not able to provide effect size estimates. To address this gap, more modern analytical methods including survival models, have been applied to these data, and show modest to no impact of HCQ treatment, with more significant effects from the HCQ-AZ combination, potentially suggesting a role for co-infections in COVID-19 pathogenesis. The trial of Gautret and colleagues, with consideration of the effect sizes, and p-values from multiple models, does not provide sufficient evidence to support wide-scale rollout of HCQ monotherapy for the treatment of COVID-19; larger randomzied studies should be considered. However, these data do suggest further study of HCQ-AZ combination therapy should be prioritized as rapidly as possible. '},\n"," {'paper_id': '403d885a33958731c4607ea632453e25a89e2b57',\n","  'section': 'Abstract',\n","  'text': 'Bunyaviruses are significant human pathogens, causing diseases ranging from hemorrhagic 26 fevers to encephalitis. Among these viruses, La Crosse virus (LACV), a member of the California 27 serogroup, circulates in the eastern and midwestern United States. While LACV infection is often 28 asymptomatic, dozens of cases of encephalitis are reported yearly. Unfortunately, no antivirals 29 have been approved to treat LACV infection. Here, we developed a method to rapidly test potential 30 antivirals against LACV infection. From this screen, we identified several potential antiviral 31 molecules, including known antivirals. Additionally, we identified many novel antivirals that 32 exhibited antiviral activity without affecting cellular viability. Valinomycin, a potassium ionophore, 33 was among our top targets. We found that valinomycin exhibited potent anti-LACV activity in 34 multiple cell types in a dose-dependent manner. Valinomycin did not affect particle stability or 35 infectivity, suggesting that it may preclude virus replication by altering cellular potassium ions, a 36 known determinant of LACV entry. We extended these results to other ionophores and found that 37 the antiviral activity of valinomycin extended to other viral families including bunyaviruses (Rift 38 Valley fever virus, Keystone virus), enteroviruses (Coxsackievirus, rhinovirus), flavirivuses (Zika), 39 and coronaviruses (229E and MERS-CoV). In all viral infections, we observed significant 40 reductions in virus titer in valinomycin-treated cells. In sum, we demonstrate the importance of 41 potassium ions to virus infection, suggesting a potential therapeutic target to disrupt virus 42 replication. 43 44 Importance 45 No antivirals are approved for the treatment of bunyavirus infection. The ability to rapidly screen 46 compounds and identify novel antivirals is one means to accelerate drug discovery for viruses 47 with no approved treatments. We used this approach to screen hundreds of compounds against 48 La Crosse virus, an emerging bunyavirus that causes significant disease, including encephalitis. 49 We identified several known and previously unidentified antivirals. We focused on a potassium 50 ionophore, valinomycin, due to its promising in vitro antiviral activity. We demonstrate that 51 valinomycin, as well as a selection of other ionophores, exhibits activity against La Crosse virus 52 as well as several other distantly related bunyaviruses. We finally observe that valinomycin has 53 activity against a wide array of human viral pathogens, suggesting that disrupting potassium ion 54 homeostasis with valinomycin may be a potent host pathway to target to quell virus infection. '},\n"," {'paper_id': '40970d447ac85abe8d49caa901298963f7960587',\n","  'section': 'Abstract',\n","  'text': 'As the SARS-CoV-2 pandemic is rapidly progressing, the need for the development of an effective vaccine is critical. A promising approach for vaccine development is to generate, through codon pair deoptimization, an attenuated virus. This approach carries the advantage that it only requires limited knowledge specific to the virus in question, other than its genome sequence. Therefore, it is well suited for emerging viruses for which we may not have extensive data. We performed comprehensive in silico analyses of several features of SARS-CoV-2 genomic sequence (e.g., codon usage, codon pair usage, dinucleotide/junction dinucleotide usage, RNA structure around the frameshift region) in comparison with other members of the coronaviridae family of viruses, the overall human genome, and the transcriptome of specific human tissues such as lung, which are primarily targeted by the virus. Our analysis identified the spike (S) and nucleocapsid (N) proteins as promising targets for deoptimization and suggests a roadmap for SARS-CoV-2 vaccine development, which can be generalizable to other viruses. '},\n"," {'paper_id': '409c431edf54d022463f40f2d31a0dfab42f5324',\n","  'section': 'Abstract',\n","  'text': 'Recently the novel coronavirus (2019-nCov) pneumonia outbreak in China then the world, and the Number of infections and death continues to increases. Search Query performs well in forecasting the epidemics. It is still a question whether search engine data can forecast the drift and the inflexion in 2019-nCov pneumonia. Based on the Baidu Search Index, we propose three prediction models: composite Index, composite Index with filtering and suspected NCP(Novel Coronavirus Pneumonia). The result demonstrates that the predictive model of composite index with filtering performs the best while the model of suspected NCP has the highest forecast error. We further predict the out-of-the-set NCP confirmed cases and monitor that the next peak of new diagnoses will occur on February 16 th and 17 th . '},\n"," {'paper_id': '4151e9b4ea5395d73bf33309dbca26551cc7c657',\n","  'section': 'Abstract',\n","  'text': 'The ecological success of social insects is based on division of labour, not only between queens 20 and workers, but also among workers. Whether a worker tends the brood or forages is strongly 21 influenced by age, fertility and nutritional status, with brood carers being younger, more fecund 22 and corpulent. Here, we experimentally disentangle behaviour from age and fertility 23 in Temnothorax longispinosus ant workers and analyse how these parameters are linked to 24 whole-body gene expression. Our transcriptome analysis reveals four times more genes 25 associated with behaviour than with age and only few fertility-associated genes. Brood carers 26 exhibited an upregulation of genes involved in lipid biosynthesis, whereas foragers invested in 27 metabolism. Additional simulations revealed that the experimental disassociation of co-varying 28 factors reduces transcriptomic noise, potentially explaining discrepancies between 29 transcriptomic studies on worker behaviour in other social insects. Our study highlights the 30 influence of nutritional status on task choice in ant workers. 31 32 33 34 35 36 author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '41d412415c46baf70ca91ebf049c0352ead19f18',\n","  'section': 'Abstract',\n","  'text': 'Cylindrical protein scaffolds are thought to stabilize membrane tubules, preventing membrane fission. In contrast, Snead et al. find that when scaffold proteins assemble, bulky disordered domains within them become acutely concentrated, generating steric pressure that destabilizes tubules, driving fission. Cellular membranes are continuously remodeled. The crescent-shaped bin-amphiphysinrvs (BAR) domains remodel membranes in multiple cellular pathways. Based on studies of BAR domains in isolation, the current paradigm is that they polymerize into cylindrical scaffolds that stabilize lipid tubules, preventing membrane fission. But in nature BAR domains are often part of multi-domain proteins that contain large intrinsically-disordered regions. Using in vitro and live cell assays, here we show that full-length BAR domaincontaining proteins, rather than stabilizing membrane tubules, are instead surprisingly potent drivers of membrane fission. Specifically, when BAR scaffolds assemble at membrane surfaces, their bulky disordered domains become crowded, generating steric pressure that destabilizes lipid tubules. More broadly, we observe this behavior with BAR domains that have a range of curvatures. These data challenge the idea that cellular membranes adopt the curvature of BAR scaffolds, suggesting instead that the ability to concentrate disordered domains is the key requirement for membrane remodeling and fission by BAR domain-containing proteins. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '41fc7191573c2f01fdef9c8ebb072e5f54cbb598',\n","  'section': 'Abstract',\n","  'text': 'All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '421bcc82fdb461b1b265b0a5d12801f9c15579d6',\n","  'section': 'Abstract',\n","  'text': 'Since the beginning of the outbreak in China, ending 2019, the Novel Coronavirus has spread subsequently to the rest of the world causing an ongoing pandemic. The World Health Organisation (WHO) declared COVID-19 \"a public health emergency of international concern.\" Taking into consideration the lockdown and quarantine situation, a research team of doctors from the Hospital of Salamanca, decided to do an evaluation of the current emotional state on the general population with a webbased survey in English and in Spanish, which was considered a useful and fast method that could help us determine how people perceived stress and worry due to the COVID-19. Methods: The survey included a 22 items, gathering information in 3 sections: Sociodemographic data, the Perceived Stress Scale (PSS-10) by Cohen and additional queries that assessed the current worry and change of behaviours due to this pandemic. Results: The survey received 1091 respondents from 41 countries, from March 17 to the 1 st of April, 2020. The mean age of the respondents was 43.1 (14.2) years old, and more than two thirds were women. 21.1% were health personnel. The mean of the PSS-10 score was 17.4 (6.4). Significantly higher scores were observed among women, youth, students, and among those who expressed concern and those who perceived increased susceptibility to the COVID-19. In contrast, no significant differences were observed between the health professionals and the general population. A weak correlation was observed between mean relative volume RSV of the last 28 days and the number of cases reported (rho = 0.31, p <0.001) and deaths (rho = 0.28, p <0.001). Discussion: With these results the researchers describe an increase of affective symptoms due to the COVID-19. This pandemic is raising the anxiety levels. The findings of the study show the affective and cognitive alterations people are going through. This survey is the first attempt to measure the psychological consequences this pandemic is having, in order to be able to provide the support to confront this global issue, addressing the mental health care that will be needed. '},\n"," {'paper_id': '422787d1a8391db8a23e809e861e477df0753c99',\n","  'section': 'Abstract',\n","  'text': 'Background. Brazil detected community transmission of COVID-19 on March 13, 2020. In this study we identify which areas in the country are most vulnerable for COVID-19, both in terms of the risk of arrival of COVID-19 cases and the risk of sustained transmission. The micro-regions with higher social vulnerability are also identified. Methods. Probabilistic models were used to calculate the probability of COVID-19 spread from São Paulo and Rio de Janeiro, according to previous data available on human mobility in Brazil. We also perform a multivariate cluster analysis of socio-economic indices to identify areas with similar social vulnerability. Results. The results consist of a series of maps of effective distance, outbreak probability, hospital capacity and social vulnerability. They show areas in the North and Northeast with high risk of COVID-19 outbreak that are also highly vulnerable. '},\n"," {'paper_id': '424b818d38dae03881b9c7b8093d9a9e6b1031ff',\n","  'section': 'Abstract',\n","  'text': 'Two notable features have been identified in the SARS-CoV-2 genome: (1) the receptor binding domain of SARS-CoV-2; (2) a unique insertion of twelve nucleotide or four amino acids (PRRA) at the S1 and S2 boundary. For the first feature, the similar RBD identified in SARs-like virus from pangolin suggests the RBD in SARS-CoV-2 may already exist in animal host(s) before it transmitted into human. The left puzzle is the history and function of the insertion at S1/S2 boundary, which is uniquely identified in SARS-CoV-2. In this study, we identified two variants from the first Guangdong SARS-CoV-2 cell strain, with deletion mutations on polybasic cleavage site (PRRAR) and its flank sites. More extensive screening indicates the deletion at the flank sites of PRRAR could be detected in 3 of 68 clinical samples and half of 22 in vitro isolated viral strains. These data indicate (1) the deletion of QTQTN, at the flank of polybasic cleavage site, is likely benefit the SARS-CoV-2 replication or infection in vitro but under strong purification selection in vivo since it is rarely identified in clinical samples; (2) there could be a very efficient mechanism for deleting this region from viral genome as the variants losing 23585-23599 is commonly detected after two rounds of cell passage. The mechanistic explanation for this in vitro adaptation and in vivo purification processes (or reverse) that led to such genomic changes in SARS-CoV-2 requires further work. Nonetheless, this study has provided valuable clues to aid further investigation of spike protein function and virus evolution. The deletion mutation identified in vitro isolation should be also noted for current vaccine development. '},\n"," {'paper_id': '430088e2d738c4c20413082888b2666668c9105f',\n","  'section': 'Abstract',\n","  'text': 'Wuhan and rapidly spread throughout the world, critically ill patients have a high mortality rate. We aimed to assess the effect of continuous renal replacement therapy (CRRT) on all-cause mortality in patients with COVID-19 undergoing invasive mechanical ventilation. '},\n"," {'paper_id': '4346e63e692fcda7d3076d25df3e6f50b9c31407',\n","  'section': 'Abstract',\n","  'text': 'We study a novel multi-strain SIR epidemic model with selective immunity by vaccination. A newer strain is made to emerge in the population when a preexisting strain has reached equilbrium. We assume that this newer strain does not exhibit cross-immunity with the original strain, hence those who are vaccinated and recovered from the original strain become susceptible to the newer strain. Recent events involving the COVID-19 virus demonstrates that it is possible for a viral strain to emerge from a population at a time when the influenza virus, a well-known virus with a vaccine readily available for some of its strains, is active in a population. We solved for four different equilibrium points and investigated the conditions for existence and local stability. The reproduction number was also determined for the epidemiological model and found to be consistent with the local stability condition for the disease-free equilibrium. 4 susceptibility to diseases amongst others [1-3]. The drop in numbers in vaccination has 5 led to outbreaks of diseases such as mumps [4-6] that could have been prevented by 6 '},\n"," {'paper_id': '4380f0251c595d6cb551e643198d2dacd3c6746c',\n","  'section': 'Abstract',\n","  'text': 'Corona Virus Disease 2019 has spread rapidly to more than 70 countries and regions overseas and over 80000 cases have been infected, resulting in more than three thousand deaths. Rapid diagnosis of patients remains a bottleneck in containing the progress of the epidemic. We used automated chemiluminescent immunoassay to detect serum IgM and IgG antibodies to 2019-nCoV of 736 subjects. COVID-19 patients were becoming reactive(positive) for specific antibodies from 7-12 days after the onset of morbidity. Specific IgM and IgG increased with the progression of the disease. The areas under the ROC curves of IgM and IgG were 0.988 and 1.000, respectively. Specific antibody detection has good sensitivity and specificity. Detection of specific antibodies in patients with fever can be a good distinction between COVID-19 and other diseases, so as to be a complement to nucleic acid diagnosis to early diagnosis of suspected cases. All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '43d416dbb7adeae604a4d124de169309f8ecd3c2',\n","  'section': 'Abstract',\n","  'text': 'Objective: To optimize clinical laboratory diagnosis of COVID-19 from suspect cases by Likelihood Ratio of SARS-CoV-2 IgM and IgG antibody. Methods: By reinterpreting the data in the article \"Diagnostic Value of Combined Detection of Serum 2019 novel coronavirus IgM and IgG Antibodies in novel coronavirusin Infection\", the positive likelihood ratio of IgM and IgG antibody in diagnosis of COVID-19 (nucleic acid positive patients) was calculated, and the posterior probability of IgM and IgG antibodies and their tandem detection to diagnose was finally calculated. Results: The positive likelihood ratios of single IgM and IgG antibody were 18.50 and 12.65 respectively, and the posterior probabilities were 90.18% and 86.26% respectively. However, the posterior probability of the two antibodies tandem detection is 99.15%, which can give clinicians quantitative confidence in the diagnosis of COVID-19 from suspected cases. According to the results of this study, combining the advantages and disadvantages of nucleic acid detection and antibody detection, the clinical pathway for clinicians to diagnose COVID-19 is found. Conclusion: For suspected cases, IgM and IgG antibody tests should be firstly done at the same time. If the antibody tests are all positive, COVID-19 can be confirmed. If not, nucleic acid detection (one or more times) is performed, and in extreme cases, high-throughput viral genome sequencing is performed. '},\n"," {'paper_id': '43d4eadd45196bbbafecfb3c888c18274d917859',\n","  'section': 'Abstract',\n","  'text': 'Infectious diseases, especially when new and highly contagious, could be devastating producing epidemic outbreaks and pandemics. Predicting the outcomes of such events in relation to possible interventions is crucial for societal and healthcare planning and forecasting of resource needs. Deterministic and mechanistic models can capture the main known phenomena of epidemics while also allowing for a meaningful interpretation of results. In this work a deterministic mechanistic population balance model was developed. The model describes individuals in a population by infection stage and age group. The population is treated as in a close well mixed community with no migrations. Infection rates and clinical and epidemiological information govern the transitions between stages of the disease. The present model provides a steppingstone to build upon and its current low complexity retains accessibility to non experts and policy makers to comprehend the variables and phenomena at play. '},\n"," {'paper_id': '43f760b5cb9c3b430c6d0b31fa90d42e71a6429c',\n","  'section': 'Abstract',\n","  'text': 'n=50; Word limit = 50 21 We computed reproduction number of COVID-19 epidemic in Iran using two different methods. We 22 estimated R0 at 3.6 (95% CI, 3.2, 4.2) (generalized growth model) and at 3.58 (95% CI, 1.29, 8.46) 23 (estimated epidemic doubling time of 1.20 (95% CI, 1.05, 1.44) days) respectively. Immediate social 24 distancing measures are recommended. 25 26 '},\n"," {'paper_id': '444bf5fa32b194dccaa78fc04d31c64212ad1674',\n","  'section': 'Abstract',\n","  'text': 'Cryo-EM single-particle analysis has proven powerful in determining the structures of rigid macromolecules. However, many protein complexes are flexible and can change conformation and composition as a result of functionally-associated dynamics. Such dynamics are poorly captured by current analysis methods. Here, we present cryoDRGN, an algorithm that for the first time 42 eukaryotic ribosome, the assembling bacterial ribosome, and the pre-catalytic spliceosome 43 complex. In these machines, we discover new conformational states and observe dynamic 44 molecular motions. CryoDRGN is distributed as an open-source tool 22 that can be easily integrated 45 in existing pipelines and is freely available at cryodrgn. '},\n"," {'paper_id': '447c606a4b4aeb4c066160c4243666695258c80f',\n","  'section': 'Abstract',\n","  'text': \"Background: Increased connectivity via air travel can facilitate the geographic spread of infectious diseases. The number of travelers alone does not explain risk; passenger origin and destination will also influence risk of disease introduction and spread. We described trends in international air passenger numbers and connectivity between countries with different capacities to detect and respond to infectious disease threats. We used the Fragile States Index (FSI) as an annual measure of countrylevel resilience and capacity to respond to infectious disease events. Countries are categorized as: Sustainable, Stable, Warning, or Alert, in order of increasing fragility. We included data for 177 sovereign states for the years 2007 to 2016. Annual inbound and outbound international air passengers for each country were obtained for the same time period. We examined trends in FSI score, trends in worldwide air travel, and the association between a state's FSI score and air travel. Results: Among countries included in the FSI rankings, the total number of passengers increased from 0.791 billion to 1.28 billion between 2007 and 2016. Increasing fragility was associated with a decrease in travel volumes, with a 2.9% (95% CI: 2.3-3.5%) reduction in passengers per 1-unit increase in FSI score. Overall, travel between countries of different FSI categories either increased or remained stable. The greatest increase was observed for travel to Warning countries from Warning countries, with an annual increase of 8,967,623 passengers (95%CI: 6,546,494 to 11,388,753) over the study period. \"},\n"," {'paper_id': '4485b9462ebf52c0062692d7ab7e423c2ae05d9e',\n","  'section': 'Abstract',\n","  'text': 'word count: 242 (250 max) 24 Significance word count: 120 (120 max) 25 Manuscript character count: 48,559 (49,000 max) 26 All rights reserved. No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/246884 doi: bioRxiv preprint 45 mediators of NK cell activation. Altogether, these data suggest that non-neutralizing antibody 46 functions, including virion phagocytosis, likely played a role in the observed 50% vaccine-47 mediated protection against HCMV acquisition. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '44a440cc1c135c938d2216ea672f8ef4f9c01296',\n","  'section': 'Abstract',\n","  'text': \"Zika virus (ZIKV) is a member of the family Flaviviridae. In 2015, ZIKV triggered a large epidemic in Brazil and spread across Latin America. In November of that year, the Brazilian Ministry of Health reported a 20-fold increase in cases of neonatal microcephaly, which corresponds geographically and temporally to the ZIKV outbreak. ZIKV was isolated from the brain tissue of a fetus diagnosed with microcephaly, and recent studies in mice models revealed that ZIKV infection may cause brain defects by influencing brain cell developments. Unfortunately, the mechanisms by which ZIKV alters neurophysiological development remain unknown. MicroRNAs (miRNAs) are small noncoding RNAs that regulate post-transcriptional gene expression by translational repression. In order to gain insight into the possible role of ZIKV-mediated miRNA signaling dysfunction in brain-tissue development, we computationally predicted new miRNAs encoded by the ZIKV genome and their effective hybridization with transcripts from human genes previously shown to be involved in microcephalia. The results of these studies suggest a possible role of these miRNAs on the expression of human genes associated with this disease. Besides, a new ZIKV miRNA was predicted in the 3'stem loop (3' SL) of the 3'untranslated region (3'UTR) of the ZIKV genome, suggesting the role of the 3'UTR of flaviviruses as a source of miRNAs. February 1 st , 2016 [8]. ZIKV was detected by electron microscopy and RT-qPCR in brains and amniotic fluid of microcephalic fetuses, strengthening the causal link between ZIKV and increased incidence of microcephaly [9, 10] . Furthermore, recent studies show that ZIKV can infect human iPSC-derived neural progenitor cells (NPCs) in vitro, resulting in dysregulation of cell-cycle-related pathways and increased cell death [11] . The development of animal models permits to begin to understand the underlying pathology of ZIKV infection [12, 13] . Modeling ZIKV infection in mice revealed direct effects of ZIKV on neural precursor cells development, including proliferation, differentiation and cell death, which may link ZIKV with the development of microcephaly [14] . Notably, \"},\n"," {'paper_id': '44b6eea3629b4e13595dc49813ee39a0d20dc5f2',\n","  'section': 'Abstract',\n","  'text': 'The process of molecular adaptation following a cross-species virus transmission event is 9 currently poorly understood. Here, we identified 137 protein sites that experienced deceleration 10 in their rate of evolution along the HIV-1/SIV phylogeny, likely indicating gain-of-function and 11 consequent adaptation. The majority of such events occurred in parallel to cross-species 12 transmission events and varied between HIV-1 groups, indicating independent adaptation 13 strategies. The evolutionary rate decelerations we found were particularly prominent in 14 accessory proteins that counteract host antiviral restriction factors, suggesting that these factors 15 are a major barrier to viral adaptation to a new host. Surprisingly, we observed that the non-16 pandemic HIV-1 group O, derived from gorillas, exhibited more rate deceleration events than 17 the pandemic group M, derived from chimpanzees. We suggest that the species barrier is higher 18 when the genetic distance of the hosts increases. Our approach paves the way for subsequent 19 studies on cross-species transfers in other major pathogens. '},\n"," {'paper_id': '44c680c61bc9c00ce628b5a26b6fa6539a47ad92',\n","  'section': 'Abstract',\n","  'text': 'Management of high mortality risk due to significant progression requires prior assessment of time-to-progression. However, few related methods are available for COVID-19 pneumonia. We retrospectively enrolled 338 adult patients admitted to one hospital between Jan 11, 2020 to Feb 29, 2020. The final follow-up date was March 8, 2020. We compared characteristics between patients with severe and non-severe outcome, and used multivariate survival analyses to assess the risk of progression to severe conditions. A total of 76 (31.9%) patients progressed to severe conditions and 3 (0.9%) died. The mean time from hospital admission to severity onset is 3.7 days. Age, body mass index (BMI), fever symptom on admission, co-existing hypertension or diabetes are associated with severe progression. Compared to non-severe group, the severe group already demonstrated, at an early stage, abnormalities in biomarkers indicating organ function, inflammatory responses, blood oxygen and coagulation function. The cohort is characterized with increasing cumulative incidences of severe progression up to 10 days after admission. Competing risks survival model incorporating CT imaging and baseline information showed an improved performance for predicting severity onset (mean time-dependent AUC = 0.880). All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '44ee8f0bb55088bfc778fe6f7afe1f0fcea42010',\n","  'section': 'Abstract',\n","  'text': 'Coronavirus disease 2019 can be screened and diagnosed through the 20 detection of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by 21 real-time reverse transcription polymerase chain reaction. SARS-CoV-2 nucleic acid 22 39 SARS-CoV-2 NAATs. 40 41 '},\n"," {'paper_id': '453ebcf634bce39c00259fd65a0ffbf3c197c306',\n","  'section': 'Abstract',\n","  'text': 'Necessary screenings must be performed to control the spread of the Corona Virus in daily life and to make a preliminary diagnosis of suspicious cases. The long duration of pathological laboratory tests and the wrong test results led the researchers to focus on different fields. Fast and accurate diagnoses are essential for effective interventions with COVID-19. The information obtained by using X-ray and Computed Tomography (CT) images is vital in making clinical diagnoses. Therefore it was aimed to develop a machine learning method for the detection of viral epidemics by analyzing X-ray images. In this study, images belonging to 6 situations, including coronavirus images, are classified. Since the number of images in the dataset is deficient and unbalanced, it is more convenient to analyze these images with hand-crafted feature extraction methods. For this purpose, firstly, all the images in the dataset are extracted with the help of four feature extraction algorithms. These extracted features are combined in raw form. The unbalanced data problem is eliminated by producing feature vectors with the SMOTE algorithm. Finally, the feature vector is reduced in size by using a stacked auto-encoder and principal component analysis to remove interconnected features in the feature vector. According to the obtained results, it is seen that the proposed method has leveraging performance, especially in order to make the diagnosis of COVID-19 in a short time and effectively. '},\n"," {'paper_id': '45f184e736c4a857bc0e3df0d55962f743c1493a',\n","  'section': 'Abstract',\n","  'text': 'One central goal of genome biology is to understand how the usage of the genome differs between 12 organisms. Our knowledge of genome composition, needed for downstream inferences, is critically 13 dependent on gene annotations, yet problems associated with gene annotation and assembly errors are 14 usually ignored in comparative genomics. Here we analyze the genomes of 68 species across 12 animal 15 phyla and some single-cell eukaryotes for general trends in genome composition and transcription, taking 16 into account problems of gene annotation. We show that, regardless of genome size, the ratio of introns to 17 intergenic sequence is comparable across essentially all animals, with nearly all deviations dominated by 18 increased intergenic sequence. Genomes of model organisms have ratios much closer to 1:1, suggesting that 19 the majority of published genomes of non-model organisms are underannotated and consequently omit 20 substantial numbers of genes, with likely negative impact on evolutionary interpretations. Finally, our 21 results also indicate that most animals transcribe half or more of their genomes arguing against differences 22 in genome usage between animal groups, and also suggesting that the transcribed portion is more 23 dependent on genome size than previously thought. 24 33 34 As the number of genome projects has grown, massive amounts of data have become available to study 35 how organisms organize and use their genomes. Genome projects vary substantially in quality of assembly 36 and annotation [3, 4] . Unfortunately, the predicted genes are often taken for granted as being correct when 37 these are only hypotheses of gene structure [5]. For example, one study found that almost half of the genes 38 in the Rhesus monkey genome had a predictable annotation error when compared to the closest human 39 homolog [6]. This has profound implications for all downstream analyses, such as studying evolution of 40 1 . CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/068627 doi: bioRxiv preprint orthologous proteins [7] and phylogeny based on protein matrices or gene content [8, 9]. When considered 41 across all genes, systematic errors in genome assembly or annotation would severely skew bulk parameters 42 of a genome. While issues of assembly are often thought to be technical problems that are resolved before continuing, 45 all subsequent analyses are dependent upon accurate genome assembly and annotation. The absence of a 46 protein family in a particular organism is only meaningful if it is certain that it is absent from the genome 47 and not merely the annotation, therefore it is of utmost importance that all genes are properly represented. Yet for most genome projects of non-model organisms, there are limited methods to determine if the assem-49 bly and annotation are sufficient for downstream comparative analyses. Internal metrics can be used, such 50 as the fraction of raw genomic reads or ESTs that map back to the assembly, though this does not tell us if a 51 gene is believable in the context of other animals. Alternatively, counts of \"universal\" single-copy orthologs 52 have been proposed as a metric of genome completeness [10, 11] , though these genes only represent a small 53 subset of all genes (few hundred out of tens of thousands in most animals). 54 55 Identification of universal trends in genome organization and transcription may enable better quantita-56 tive metrics of genome completeness. Mechanistic models relating to evolution of gene content or coding 57 fractions tended to focus on bacteria or archaea because of the relative ease of annotation. In regards to 58 eukaroytes, some patterns in genome size have been discussed [12-14]. Additionally, a handful of studies 59 have analyzed genome size in connection to other parameters such as indels [15], transposon content [16-19], 60 average intron length [20, 21] or total intron length [18]. Despite these advances, none of these studies have 61 estimated the amount of the genome that is genic (exonic plus intronic, including non-coding) based on in-62 dependent examination of single genomes and without averaging over a whole kingdom. Additionally, none 63 of them have described a way to account for technical problems in assembly and annotation. 64 65 Here we examine basic trends of genome size and the relationship to annotation quality across animals 66 and some single-celled eukaryotes. We show that assembly and annotation errors are widespread and pre-67 dictable and that many genomes are likely to be missing many genes. We further show that re-annotation 68 of select species with publicly available tools and transcriptome data improves the annotation. Future users 69 may benefit if databases incorporate more recent data from transcriptome sequencing, and update annota-70 tion versions more frequently. Comparison of genomic composition across many animal groups indicated a 71 ratio of introns:intergenic approaching 1:1, suggesting this as a potential parameter to identify genome com-72 pleteness across metazoans, and potentially other eukaryotes. Finally, this implies that animals transcribe 73 at least half of their genomes whereby small, exon-rich genomes transcribe most of the genome and large 74 genomes transcribe approximately half of the genome. 75 76 Methods 77 Genomic data sources 78 Data sources and parameters are available in Supplemental Table 1. 79 80 Genomic scaffolds and annotations for Ciona intestinalis [22], Branchiostoma floridae [23], Trichoplax ad-81 herens [24], Capitella teleta [25], Lottia gigantea [25], Helobdella robusta [25], Saccoglossus kowalevskii [26], 82 Monosiga brevicollis [27], Emiliania huxleyi [28], and Volvox carteri [29] were downloaded from the JGI 83 genome portal. 84 85 Genome assemblies and annotations for Sphaeroforma arctica, Capsaspora owczarzaki [30] and Salpin-86 goeca rosetta [31] were downloaded from the Broad Institute. 87 88 GFF annotations v2.1 [32] for Amphimedon queenslandica were downloaded from the Amphimedon 89 Genome website (http://amphimedon.qcloud.qcif.edu.au/downloads.html), and v1 annotations [33] and as-90 semblies were downloaded from Ensembl. 91 2 . CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/068627 doi: bioRxiv preprint 92 For Nematostella vectensis, Nemve1 assembly and annotations [34] were downloaded from JGI, and the 93 transcriptome for comparative reannotation was downloaded from http://www.cnidariangenomes.org/ [35]. 94 95 Genome assembly, transcriptome assemblies from Cufflinks and Trinity, and GFF annotations for Mne-96 miopsis leidyi [8] were downloaded from the Mnemiopsis Genome Portal (http://research.nhgri.nih.gov/mnemiopsis/). 97 Assembly and annotations for Sycon ciliatum [36] were downloaded from COMPAGEN. Assembly and 98 annotation for Botryllus schlosseri [37] were downloaded from the Botryllus Schlloseri genome project 99 103 Database (ciliate.org). Assembly and annotation for Symbiodinium kawagutii [40] were downloaded from 104 the Dinoflagellate Resources page (web.malab.cn/symka new/index.jsp). 105 106 Assemblies and annotations for Symbiodinium minutum [41], Pinctada fucata [42], Acropora digitifera 107 [43], Lingula anatina [44], Ptychodera flava [26], and Octopus bimaculoides [45] were downloaded from the 108 OIST Marine Genomics Browser zon marinus [61], Callorhinchus milii [62], Crassostrea gigas [63], Dendroctonus ponderosae [64], Tribolium 116 castaneum [65], Bombyx mori [66], Limulus polyphemus [67] were downloaded from the NCBI Genome server. 117 118 Genome assemblies and annotations of Caenorhabditis elegans [68], Drosophila melanogaster, Strongy-119 locentrotus purpuratus [69], Daphnia pulex [70], Apis mellifera [71], Ixodes scapularis [72], Strigamia mar-120 itima [73] were downloaded from Ensembl. '},\n"," {'paper_id': '4602afcb8d95ebd9da583124384fd74299d20f5b',\n","  'section': 'Abstract',\n","  'text': 'Viruses possessing class I fusion proteins require proteolytic activation by host cell proteases to mediate 18 fusion with the host cell membrane. The mammalian SPINT2 gene encodes a protease inhibitor that 19 targets trypsin-like serine proteases. Here we show the protease inhibitor, SPINT2, restricts cleavage-20 activation efficiently for a range of influenza viruses and for human metapneumovirus (HMPV). SPINT2 21 treatment resulted in the cleavage and fusion inhibition of full-length influenza A/CA/04/09 (H1N1) HA, 22 '},\n"," {'paper_id': '46030e9b8451bf3b0c90833dbeacce286a263a09',\n","  'section': 'Abstract',\n","  'text': 'Objective: HIV infection and aging are both associated with neurodegeneration. However, whether the aging process alone or other factors associated with advanced age account for the progression of neurodegeneration in the aging HIV-positive (HIV+) population remains unclear. Methods: HIV+ (n=70) and HIV-negative (HIV-, n=34) participants underwent diffusion tensor imaging (DTI) and metrics of microstructural properties were extracted from regions of interest (ROIs). A support vector regression model was trained on two independent datasets of healthy adults across the adult life-span (n=765, Cam-CAN = 588; UiO = 177) to predict participant age from DTI metrics, and applied to the HIV dataset. Predicted brain age gap (BAG) was computed as the difference between predicted age and chronological age, and statistically compared between HIV groups. Regressions assessed the relationship between BAG and HIV severity/medical comorbidities. Finally, correlation analyses tested for associations between BAG and cognitive performance. Results: BAG was significantly higher in the HIV+ group than the HIV-group F (1, 103) = 12.408, p = 0.001). HIV RNA viral load was significantly associated with BAG, particularly in older HIV+ individuals (R 2 = 0.29, F(7, 70) = 2.66, p = 0.021). Further, BAG was negatively correlated with domain-level cognitive function (learning: r = -0.26, p = 0.008; memory: r = -0.21, p = 0.034). '},\n"," {'paper_id': '461a32146936c2db3491cfe5fe95fbabae4a3d27',\n","  'section': 'Abstract',\n","  'text': 'JSA: Conception, design, data acquisition, analysis, interpretation, drafting manuscript for intellectually important content, and approval of final version; TO: Data acquisition and approval of final version; AMA: Analysis, interpretation and approval of final version; MA: Analysis, interpretation and approval of final version; SM: Analysis, interpretation and approval of final version; AA: data acquisition, analysis and approval of final version; SQ: interpretation and approval of final version; SM: interpretation, and approval of final version.; JRH: Conception, design, analysis, interpretation, and approval of final version. '},\n"," {'paper_id': '462cbb326ccd8587cae7a3538c8c6712d9013698',\n","  'section': 'Abstract',\n","  'text': \"Lei, MD have contributed equally to this work. Word count: 3166 All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. Background: The clinical outcomes of COVID-19 patients in Hubei and other areas are different. We aim to investigate the epidemiological and clinical characteristics of patient with COVID-19 in Hunan which is adjacent to Hubei. In this double-center, observational study, we recruited all consecutive patients with laboratory confirmed COVID-19 from January 23 to February 14, 2020 in two designated hospitals in Hunan province, China. Epidemiological and clinical data from patients' electronic medical records were collected and compared between mild, moderate and severe/critical group in detail. Clinical outcomes were followed up to February 20, 2020. Findings: 291 patients with COVID-19 were categorized into mild group (10.0%), moderate group (72.8%) and severe/critical group (17.2%). The median age of all patients was 46 years (49.8% were male). 86.6% patients had an indirect exposure history. The proportion of patients that had been to Wuhan in severe/critical group (48.0% vs 17.2%, p=0.006) and moderate group (43.4% vs 17.2%, p=0.007) were higher than mild group. Fever (68.7%), cough (60.5%), and fatigue (31.6%) were common symptoms especially for severe and critical patients. Typical lung imaging finding were bilateral and unilateral ground glass opacity or consolidation. Leukopenia, lymphopenia and eosinopenia occurred in 36.1%, 22.7% and 50.2% patients respectively. Increased fibrinogen was detected in 45 of 58 (77.6%) patients with available results. 29 of 44 (65.9%) or 22 of 40 (55.0%) patients were positive in Mycoplasma pneumonia or Chlamydia pneumonia antibody test respectively. Compared with mild or moderate group, severe/critical group had a relative higher level of neutrophil, All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. \"},\n"," {'paper_id': '46634d03aa169aab9c372746ebaf3aaa65d7c7d9',\n","  'section': 'Abstract',\n","  'text': 'Background: A pandemic of coronavirus disease 2019 which have caused more 24 than 80 thousand persons infected globally is still ongoing. This study aims to calculate its 25 case fatality rate (CFR). 26 The method, termed as converged CFR calculation, was based on the formula of 27 dividing the number of known deaths by the number of confirmed cases T days before, where 28 T was an average time period from case confirmation to death. It was found that supposing a 29 T, if it was smaller (bigger) than the true T, calculated CFRs would gradually increase 30 (decrease) to infinitely near the true T with time went on. According to the law, the true T 31 value could be determined by trends of daily CFRs calculated with different assumed T values 32 (left of true T is decreasing, right is increasing). Then the CFR could be calculated. 33 '},\n"," {'paper_id': '46c496bc5e9bb4292796a387a601cd63f4cbec39',\n","  'section': 'Abstract',\n","  'text': \"Scientific conferences provide valuable opportunities for researchers across career stages and disciplines to present their latest work and to network with their peers. The advent of the internet has opened new possibilities for interaction, collaboration and networking, yet the uptake of tools enabling remote participation at scientific meetings has been slow. Academic conferences have become more international with high proliferation in the number of meetings and attendees, but the format and quality of their organization lags behind what is possible. As a result, the experience of attending conferences in many disciplines today is not appreciably different in many respects. A few conferences in some disciplines have recently implemented valuable changes for the community and have become more receptive to attendees with families. Many meetings could still be improved significantly in terms of diversity, inclusivity, promoting early career researcher (ECR) networking and career development, venue accessibility, and more importantly, reducing the meetings' carbon footprint. These issues are openly visible and to some extent discussed among researchers on social media. It is important to accelerate and mandate these efforts so that researchers in all disciplines, in particular ECRs, consistently benefit from scientific gatherings, for years to come. We examined the current state of over 200 national and international academic meetings in various disciplines for key features of inclusivity and sustainability and propose solutions to make conferences more modern, effective, equitable and intellectually productive for the research community and environmentally sustainable for our planet. \"},\n"," {'paper_id': '46c5f4a01496ab4fe89b1dfba7b7f94445766640',\n","  'section': 'Abstract',\n","  'text': \"The epidemic of a novel coronavirus illness becomes as a global threat. The aim of this study is first to find the best prediction models for daily confirmed cases in countries with high number of confirmed cases in the world and second to predict confirmed cases with these models in order to have more readiness in healthcare systems. : This study was conducted based on daily confirmed cases of COVID-19 that were collected from the official website of Johns Hopkins University from January 22 th , 2020 to March 1 th , 2020. Auto Regressive Integrated Moving Average (ARIMA) model was used to predict the trend of confirmed cases. Stata version 12 were used. Results: Parameters used for ARIMA were (2,1,0) for Mainland China, ARIMA (2,2,2) for Italy , ARIMA(1,0,0) for South Korea, ARIMA (2,3,0) for Iran, and ARIMA(3,1,0) for Thailand. Mainland China and Thailand had almost a stable trend. The trend of South Korea was decreasing and will become stable in near future. Iran and Italy had unstable trends. Conclusions: Mainland China and Thailand were successful in haltering COVID-19 epidemic. Investigating their protocol in this control like quarantine should be in the first line of other countries' program \"},\n"," {'paper_id': '46ecc5f1ac34e8f2befad14d000ad7d0cde539d8',\n","  'section': 'Abstract',\n","  'text': 'The 2013-2016 West Africa EBOV epidemic was the biggest EBOV outbreak to date. An analysis of virus-specific CD8+ T-cell immunity in 30 survivors showed that 26 of those individuals had a CD8+ response to at least one EBOV protein. The dominant response (25/26 subjects) was specific to the EBOV nucleocapsid protein (NP). It has been suggested that epitopes on the EBOV NP could form an important part of an effective T-cell vaccine for Ebola Zaire. We show that a 9-amino-acid peptide NP44-52 (YQVNNLEEI) located in a conserved region of EBOV NP provides protection against morbidity and mortality after mouse adapted EBOV challenge. A single vaccination in a C57BL/6 : bioRxiv preprint mouse using an adjuvanted microsphere peptide vaccine formulation containing NP44-52 is enough to confer immunity in mice. Our work suggests that a peptide vaccine based on CD8+ T-cell immunity in EBOV survivors is conceptually sound and feasible. Nucleocapsid proteins within SARS-CoV-2 contain multiple class I epitopes with predicted HLA restrictions consistent with broad population coverage. A similar approach to a CTL vaccine design may be possible for that virus. '},\n"," {'paper_id': '47236f3c7fe98fc3ab9a033df5d29ec778509a09',\n","  'section': 'Abstract',\n","  'text': 'Background RT-PCR test for identification of viral nucleic acid is the current standard diagnostic method for the diagnosis of COVID-19 disease but technical reasons limit the utilization of this assay onlarge scalescreenings. Method We verified in a consecutive series of 191 symptomatic patients the clinical information that new rapid serological colorimetric test qualitatively analyzing IgM/IgG expression can provide with respect to standard assay and with respect to clinical outcome of patients. ResultsRapid serological test showed a sensitivity of 30% and a specificity of 89% with respect to the standard assay but, interestingly, these performances improve after 8 days of symptoms appearance. After 10 days of symptoms the predictive value of rapid serological test is higher than that of standardassay. When the behaviour of the two immunoglobulins was evaluated with respect to time length of symptoms appaerance, no significant difference in immunoglobulins behaviour was shown. Conclusions The rapid serological test analyzed in the present study is candidate to provide information on immunoreaction of the subject to COVID-19 exposure. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '475b615fd202273ccfb8b46a8135703d9874d4dc',\n","  'section': 'Abstract',\n","  'text': 'Since Dec 2019, the COVID-19 epidemic has spread over the globe creating one of the greatest pandemics ever witnessed. This epidemic wave will only begin to roll back once a critical proportion of the population is immunised, either by mounting natural immunity following infection, or by vaccination. The latter option can minimise the cost in terms of human lives but it requires to wait until a safe and efficient vaccine is developed, a period estimated to last at least 18 months. In this work, we use optimal control theory to explore the best strategy to implement while waiting for the vaccine. We seek a solution minimizing deaths and costs due to the implementation of the control strategy itself. We find that such a solution leads to an increasing level of control with a maximum reached near the fourth month of the epidemics and a steady decrease until vaccine deployment. This strategy strongly outperforms others with constant or cycling allocations of the same amount of resources to control the outbreak. This work opens new perspectives to mitigate the effects of the ongoing COVID-19 pandemics, and be used as a proof-of-concept in using mathematical modelling techniques to enlighten decision making and public health management in the early times of an outbreak. '},\n"," {'paper_id': '478ba63e9a318f16e5f90909358c2b4dd68f1395',\n","  'section': 'Abstract',\n","  'text': \"Background and purpose: The worldwide pandemic of coronavirus disease 2019 greatly challenges public medical systems. With limited medical resources, the treatment priority is determined by the severity of patients. However, many mild outpatients quickly deteriorate into severe/critical stage. It is crucial to early identify them and give timely treatment for optimizing treatment strategy and reducing mortality. This study aims to establish an AI model to predict mild patients with potential malignant progression. Methods: A total of 133 consecutively mild COVID-19 patients at admission who was hospitalized in Wuhan Pulmonary Hospital from January 3 to February 13, 2020, were selected in this retrospective IRB-approved study. All mild patients were categorized into groups with or without malignant progression. The clinical and laboratory data at admission, the first CT, and the follow-up CT at the severe/critical stage of the two groups were compared. Both multivariate logistic regression and deep learning-based methods were used to build the prediction models, with their area under ROC curves (AUC) compared. : Multivariate logistic regression depicted 6 risk factors for malignant progression: age >55years (OR 5.334, 95%CI 1.8-15.803), comorbid with hypertension (OR 5.093, 95%CI 1.236-20.986), a decrease of albumin (OR 4.01, 95%CI 1.216-13.223), a decrease of lymphocyte (OR 3.459, 95%CI 1.067-11.209), the progressive consolidation from CT1 to CTsevere (OR 1.235, 95%CI 1.018-1.498), and elevated HCRP (OR 1.015, 95%CI 1.002-1.029); and one protective factor: the presence of fibrosis at CT1 (OR 0.656, 95%CI 0.473-0.91). By combining the clinical data and the temporal information of the CT data, our deep learning-based models achieved the best AUC of 0.954, which outperformed logistic regression (AUC: 0.893), Conclusions: Our deep learning-based methods can identify the mild patients who are easy to deteriorate into severe/critical cases efficiently and accurately, which undoubtedly helps to optimize the treatment strategy, reduce mortality, and relieve the medical pressure. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. : medRxiv preprint achieved great performance in cancer outcome prediction 14 , head CT scans detection 15 and antibiotic discovery 16 . Moreover, compared with the traditional multi-stage methods, the deep learning-based model could significantly improve the efficiency of patient stratification (Figure 1) , which is very important when dealing with tremendous patients. Therefore, the purposes of our study are: 1) to develop models to identify the mild patients who are easy to deteriorate into severe/critical cases using AI-based methods and logistic regression respectively; 2) to seek clinical and imaging factors for prediction of patients with potential malignant progression. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. : medRxiv preprint ≥ 30 breaths per min; 2) resting blood oxygen saturation ≤ 93%; 3) the ratio of arterial oxygen partial pressure to fraction of inspiration oxygen ＜ 300mmHg; 4) ICU patients with one or multiple organ failure, shock, or mechanical ventilation. The time point of reaching the criteria of severe/critical progression was recorded. All patients underwent serial chest CT exams on a dedicated CT scanner (SOMATOM go.NOW, Siemens Healthineers, Germany) with the following parameters: slice thickness 3mm, slice gap 0mm, 130kV, 50mAs. All serial CT images were reviewed by three radiologists (QC, CC, DN, 2 years' experience in All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. \"},\n"," {'paper_id': '47a7579cf81744a0a83b1b64cc9017fb600452cd',\n","  'section': 'Abstract',\n","  'text': 'We conducted a retrospective study among 417 confirmed COVID-19 cases from Jan 1 to Feb 28, 2020 in Shenzhen, the largest migrant city of China, to identify the epidemiological and clinical features in settings of high population mobility. We estimated the median incubation time to be 5.0 days. 342 (82.0%) cases were imported, 161 (38.6%) cases were identified by surveillance, and 247 (59.2%) cases were reported from cluster events. The main symptoms on admission were fever and dry cough. Most patients (91.4%) had mild or moderate illnesses. Age of 50 years or older, breathing problems, diarrhea, and longer time between the first medical visit and admission were associated with higher level of clinical severity. Surveillance-identified cases were much less likely to progress to severe illness. Although the COVID-19 epidemic has been contained in Shenzhen, close monitoring and risk assessments are imperative for prevention and control of COVID-19 in future. Line: We characterized epidemiological and clinical features of a large population-based sample of COVID-19 cases in the largest migrant city of China, and our findings could provide knowledge of SARS-CoV-2 transmission in the context of comprehensive containment and mitigation efforts in similar settings. Running Title: Epidemiological and Clinical features of COVID-19 '},\n"," {'paper_id': '47f65bff75b7752a38ccbb540fbd74955b3addea',\n","  'section': 'Abstract',\n","  'text': \"To differentiate different entities of appendicitis through studying cluster/outbreak, ascertain common setting of cluster/outbreak, and provide the epidemiological evidences of infectious etiology of appendicitis. Background: Differential diagnosis and management for perforated appendicitis and non-perforated appendicitis and the infectious etiology of appendicitis are current hot topics. Methods: Field investigation for Tibetan students were carried out and reports published in English and Chinese medical journals were reviewed. The literature review included 473 patients in 7 cluster/outbreaks of appendicitis in 6 provinces and autonomous regions. All the clusters/outbreaks occurred in group living units. We found two classic entities of appendicitis with natural history from nonperforated appendicitis to perforated appendicitis and two entities of non-perforated appendicitis. In classic entities, one may represent majority of sporadic patients and the other may represent partial sporadic patients with obvious gastrointestinal manifestation. In entities of non-perforated appendicitis, one was identical to features of sporadic nonperforated appendicitis and the other one is identical to the following Tibetan students and associated with Fusobacterium. The field investigation for 120 Tibetan students with appendicitis showed that the resected appendices exhibited diffuse or focal hemorrhages and infiltration by eosinophils and by lymphocytes. Most patients had normal body temperature, white blood cell count and neutrophil count. This is a new entity of appendicitis. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. : medRxiv preprint 3 The clusters/outbreaks of appendicitis showed the features of infectious disease in epidemiology. The entity of perforated appendicitis was not found. Studying cluster/outbreak is a good method to differentiate different entities of appendicitis and infectious etiology. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. : medRxiv preprint . Cluster/outbreak often results from common cause, therefore almost every patient in cluster/outbreak belongs to the same entity and appeared in certain stage of same entity. Connecting each stage, we can know full natural histories of different entities and accordingly demonstrate whether or not perforated appendicitis and non-perforated appendicitis are different entities. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. : medRxiv preprint METHODS Field investigation of cluster/outbreak was carried out according to Reingold AL's methods 35 and literature review was carried out to find the other cluster/outbreak. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. : medRxiv preprint of non-perforated and perforated appendicitis: Implications for pathophysiology and management. Ann Surg 2007; 245(6):886-892. 7. Andersson RE. The natural history and traditional management of appendicitis revisited:Spontaneous resolution and predominance of prehospital perforations imply that a correct diagnosis is more important than an early diagnosis. World J Surg 2007; 31(1):86-92. 8. Körner H1, Söndenaa K, Söreide JA. Incidence of acute non-perforated and perforated appendicitis: age-specific and sex-specific analysis. World J Surg. 1997; 21(3):313-317. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. : medRxiv preprint 32. Lead from MMWR. Investigation of a cluster of appendicitis outbreaks-Texas. JAMA 1987; 258(3):310-311. 33. Guo Y, Xiao SY, Yan H, Sun ND, Jiang MS, Liu DY. Cluster of acute hemorrhagic appendicitis among high school students in Wuhan, China. Am J Surg 2004; 188(2):115-121. 34. Swidsinski A, Loening-Baucke V, Biche-Ool S, et al. Mucosal invasion by All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. \"},\n"," {'paper_id': '47f99622879e0b7467b10fc44d5c7f5257a877cb',\n","  'section': 'Abstract',\n","  'text': 'The emergence of the betacoronavirus, SARS-CoV-2 that causes COVID-19, represents a significant threat to global human health. Vaccine development is focused on the principal target of the humoral immune response, the spike (S) glycoprotein, that mediates cell entry and membrane fusion. SARS-CoV-2 S gene encodes 22 N-linked glycan sequons per protomer, which likely play a role in immune evasion and occluding immunogenic protein epitopes. Here, using a site-specific mass spectrometric approach, we reveal the glycan structures on a recombinant SARS-CoV-2 S immunogen. This analysis enables mapping of the glycanprocessing states across the trimeric viral spike. We show how SARS-CoV-2 S glycans differ from typical host glycan processing, which may have implications in viral pathobiology and vaccine design. '},\n"," {'paper_id': '4840f8430105778e9a8973743e866adad6254186',\n","  'section': 'Abstract',\n","  'text': 'Background: False negative results of SARS-CoV-2 nucleic acid detection pose threats to COVID-19 patients and medical workers alike. Objective: To develop multivariate models to determine clinical characteristics that contribute to false negative results of SARS-CoV-2 nucleic acid detection, and use them to predict false negative results as well as time windows for testing positive. Design: Retrospective Cohort Study (Ethics number of Tongji Hospital Measurements: Clinical information on CT imaging reports, blood routine tests, and clinic symptoms were collected. A multivariate logistic regression was used to explain and predict false negative testing results of SARS-CoV-2 detection. A multivariate accelerated failure model was used to analyze and predict delayed time windows for testing positive. Results: Of the 1,324 outpatients who diagnosed of COVID-19, 633 patients tested positive in their first SARS-CoV-2 nucleic acid test (47.8%), with a mean age of 51 years (SD=14.9); the rest, which had a mean age of 47 years (SD=15.4), tested negative in the first test. \"Ground glass opacity\" in a CT imaging report was associated with a lower chance of false negatives (aOR, 0.56), and reduced the length of time window for testing positive by 26%. \"Consolidation\" was associated with a higher chance of false negatives (aOR, 1.57), and extended the length of time window for testing positive by 44%. In blood routine tests, basophils (aOR, 1.28) and eosinophils (aOR, 1.29) were associated with a higher chance of false negatives, and were found to extend the time window for testing positive by 23% and 41%, respectively. Age and gender also affected the significantly. Limitation: Data were generated in a large single-center study. Testing outcome and positive window of SARS-CoV-2 detection for COVID-19 patients were associated with CT imaging results, blood routine tests, and clinical symptoms. Taking into account relevant information in CT imaging reports, blood routine tests, and clinical symptoms helped reduce a false negative testing outcome. The predictive AFT model, what we believe to be one of the first statistical models for predicting time window of SARS-CoV-2 detection, could help clinicians improve the accuracy and efficiency of the diagnosis, and hence, optimizes the timing of nucleic acid detection and alleviates the shortage of nucleic acid detection kits around the world. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '48656efc59191537073975938f25f201524971af',\n","  'section': 'Abstract',\n","  'text': 'Background: Severe ill patients with 2019 novel coronavirus (2019-nCoV) infection progressed rapidly to acute respiratory failure. We aimed to select the most useful prognostic factor for severe illness incidence. Methods: The study prospectively included 61 patients with 2019-nCoV infection treated at Beijing Ditan Hospital from January 13, 2020 to January 31, 2020. Prognostic factor of severe illness was selected by the LASSO COX regression analyses, to predict the severe illness probability of 2019-CoV pneumonia. The predictive accuracy was evaluated by concordance index, calibration curve, decision curve and clinical impact curve. Results: The neutrophil-to-lymphocyte ratio (NLR) was identified as the independent risk factor for severe illness in patients with 2019-nCoV infection. The NLR had a cindex of 0.807 (95% confidence interval, 0.676-0.38), the calibration curves fitted well, All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '48b7966e4d83d9e9789c35151fd6d870dbc6551a',\n","  'section': 'Abstract',\n","  'text': 'Background: Coronavirus pandemic is currently a global public health emergency. At present, no pharmacological treatment is known to treat this condition, and there is a need to review the available treatments. Objective: While there have been studies to describe the role of chloroquine and hydroxychloroquine in various viral conditions, there is limited information about the use of them in COVID-19. This systematic review aims to summarize the available evidence regarding the role of chloroquine in treating coronavirus infection. The preferred reporting items for systematic reviews and meta-analyses (PRISMA) guidelines were used for this review. were pre-prints (not yet peer-reviewed) articles) in this systematic review. All the articles mentioned about the role of chloroquine and /or hydroxychloroquine in limiting the infection with SARS-CoV-2 (the virus causing COVID-19). Conclusions: There is theoretical, experimental, preclinical and clinical evidence of the effectiveness of chloroquine in patients affected with COVID-19. There is adequate evidence of drug safety from the long-time clinical use of chloroquine and hydroxychloroquine in other indications. More data from ongoing and future trials will add more insight into the role of chloroquine and hydroxychloroquine in COVID-19 infection. '},\n"," {'paper_id': '48f486ccea11b4ba01186e287124316c962857f7',\n","  'section': 'Abstract',\n","  'text': \"Introduction: the current worldwide outbreak of Coronavirus disease 2019 due to a novel coronavirus (SARS-CoV-2) is seriously threatening the public health. The number of infected patients is continuously increasing and the need for Intensive Care Unit admission ranges from 5 to 26%. The mortality is reported to be around 3.4% with higher values for the elderly and in patients with comorbidities. Moreover, this condition is challenging the healthcare system where the outbreak reached its highest value. To date there is still no available treatment for SARS-CoV-2. Clinical and preclinical evidence suggests that nitric oxide (NO) has a beneficial effect on the coronavirusmediated acute respiratory syndrome, and this can be related to its viricidal effect. The time from the symptoms' onset to the development of severe respiratory distress is relatively long. We hypothesize that high concentrations of inhaled NO administered during early phases of COVID-19 infection can prevent the progression of the disease. \"},\n"," {'paper_id': '48fe4c881f0e7546bc34e5c8ab6aa9f3235ffcad',\n","  'section': 'Abstract',\n","  'text': 'Fundamental aspects of SARS-CoV-2 biology remain to be described, having the 20 potential to provide insight to the response effort for this high-priority pathogen. Here we 21 describe the first native RNA sequence of SARS-CoV-2, detailing the coronaviral 22 transcriptome and epitranscriptome, and share these data publicly. A data-driven inference 23 of viral genetic features and evolutionary rate is also made. The rapid sharing of sequence 24 information throughout the SARS-CoV-2 pandemic represents an inflection point for public 25 health and genomic epidemiology, providing early insights into the biology and evolution of 26 this emerging pathogen. 27 28 29 '},\n"," {'paper_id': '490a0e0b43ae5d73e13a6b714626f15a3fef9fbf',\n","  'section': 'Abstract',\n","  'text': 'COVID-19 has spread to most countries in the world. Puzzlingly, the impact of the disease is different in different countries. These differences are attributed to differences in cultural norms, mitigation efforts, and health infrastructure. Here we propose that national differences in COVID-19 impact could be partially explained by the different national policies respect to Bacillus Calmette-Guérin (BCG) childhood vaccination. BCG vaccination has been reported to offer broad protection to respiratory infections. We compared large number of countries BCG vaccination policies with the morbidity and mortality for COVID-19. We found that countries without universal policies of BCG vaccination (Italy, Nederland, USA) have been more severely affected compared to countries with universal and long-standing BCG policies. Countries that have a late start of universal BCG policy (Iran, 1984) had high mortality, consistent with the idea that BCG protects the vaccinated elderly population. We also found that BCG vaccination also reduced the number of reported COVID-19 cases in a country. The combination of reduced morbidity and mortality makes BCG vaccination a potential new tool in the fight against COVID-19. '},\n"," {'paper_id': '4919d5781ff33b7e946a7b24bb0068c340709b7b',\n","  'section': 'Abstract',\n","  'text': 'Recent outbreaks of infectious pathogens such as Zika, Ebola, and COVID-19 have underscored the need for the dependable availability of vaccines against emerging infectious diseases (EIDs). The cost and risk of R&D programs and uniquely unpredictable demand for EID vaccines have discouraged vaccine developers, and government and nonprofit agencies have been unable to provide timely or sufficient incentives for their development and sustained supply. We analyze the economic returns of a portfolio of EID vaccine assets, and find that under realistic financing assumptions, the expected returns are significantly negative, implying that the private sector is unlikely to address this need without public-sector intervention. We have sized the financing deficit for this portfolio and propose several potential solutions, including price increases, enhanced public-private partnerships, and subscription models through which individuals would pay annual fees to obtain access to a portfolio of vaccines in the event of an outbreak. . CC-BY 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': '493aa8aa2e9122620b1dcaf102ab62e643ad5632',\n","  'section': 'Abstract',\n","  'text': 'Motivated by the rapid upsurge of COVID-19 cases in the United States beginning March 2020, we forecast the disease spread and assess the effectiveness of containment strategies by using an estalished network-driven epidemic dynamic model. Our model is initialized using the daily counts of active and confirmed COVID-19 cases across the US. Based on our model predictions for the March 14-16 timeframe, the national epidemic peak could be expected to arrive by early June, corresponding to a daily active count of ≈ 7% of the US population, if no containment plans are implemented. Epidemic peaks are expected to arrive in the states of Washington and New York by May 21 and 25, respectively. With a modest 25% reduction in COVID-19 transmissibility via community-level interventions, the epidemic progression could be delayed by up to 34 days. Wholesale interstate traffic restriction is ineffective in delaying the epidemic outbreak, but it does desynchronize the arrival of state-wise epidemic peaks, which could potentially alleviate the burden on limited available medical resources. In addition to forecasting the arrival timeline of the state-wise epidemic peaks, we attempt at informing the optimal timing necessary to enforce community-level interventions. Our findings underscore the pressing need for preparedness and timely interventions in states with a large fraction of the vulnerable uninsured and liquid-asset-poverty populations. '},\n"," {'paper_id': '4977a01aec602236d923e970569adab5bdc5865e',\n","  'section': 'Abstract',\n","  'text': 'Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) causes the infectious disease COVID-19, which was first reported in Wuhan, China in December, 2019. Despite the tremendous efforts to control the disease, COVID-19 has now spread to over 100 countries and caused a global pandemic. SARS-CoV-2 is thought to have originated in bats; however, the intermediate animal sources of the virus are completely unknown. Here, we investigated the susceptibility of ferrets and animals in close contact with humans to SARS-CoV-2. We found that SARS-CoV-2 replicates poorly in dogs, pigs, chickens, and ducks, but efficiently in ferrets and cats. We found that the virus transmits in cats via respiratory droplets. Our study provides important insights into the animal reservoirs of SARS-CoV-2 and animal management for COVID-19 control. '},\n"," {'paper_id': '49ac69f362c27acbc6de0c5cbb640267e7a1e797',\n","  'section': 'Abstract',\n","  'text': 'Pan and ZY Peng are cocorresponding authors. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. Rationale: In late December 2019, an outbreak of acute respiratory illness, now officially named as COVID-19, or coronavirus disease 2019, emerged in Wuhan, China, now spreading across the whole country and world. More data were needed to understand the clinical characteristics of the disease. To study the epidemiology, clinical features and outcomes of patients with COVID-19. Methods: we performed a single center, retrospective case series study in 221 patients with laboratory confirmed SARS-CoV-2 pneumonia at a university hospital. The median age was 55.0 years and 48.9% were male and only 8 (3.6%) patients had a history of exposure to the Huanan Seafood Market. Compared to the non-severe pneumonia patients, the median age of the severe patients was significantly older, and they were more likely to have chronic comorbidities. Most common symptoms in severe patients were high fever, anorexia and dyspnea. On admission, 33.0% patients showed leukopenia and 73.8% showed lymphopenia. In addition, the severe patients suffered a higher rate of co-infections with bacteria or fungus and they were more likely to developing complications. As of February 15, 2020, 19.0% patients had been discharged and 5.4% patients died. 80% of severe cases received ICU care, and 52.3% of them transferred to the general wards due to relieved symptoms, and the mortality rate of severe patients in ICU was 20.5%. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. : medRxiv preprint 2 Conclusions: The COVID-19 epidemic spreads rapidly by human-to-human transmission. Patients with elder age, chronic comorbidities, blood leukocyte/lymphocyte count, procalcitonin level, co-infection and severe complications might increase the risk of poor clinical outcomes. Abstract: 247 words '},\n"," {'paper_id': '4a1cdb80b4dbdc8ce0bede07d2ef5e47bf1274c9',\n","  'section': 'Abstract',\n","  'text': 'In this work, we analyze the epidemic data of cumulative infected cases collected from many countries as reported by WHO starting from January 21 st 2020 and up till March 21 st 2020. Our inspection is motivated by the renormalization group (RG) framework. Here we propose the RGinspired logistic function of the form αE(t) = a 1 + e −c(t−t 0 ) −n as an epidemic strength function with n being asymmetry in the modified logistic function. We perform the non-linear least-squares analysis with data from various countries. The uncertainty for model parameters is computed using the squared root of the corresponding diagonal components of the covariance matrix. We carefully divide countries under consideration into 2 categories based on the estimation of the inflection point: the maturing phase and the growth-dominated phase. We observe that long-term estimations of cumulative infected cases of countries in the maturing phase for both n = 1 and n = 1 are close to each other. We find from the value of root mean squared error (RMSE) that the RG-inspired logistic model with n = 1 is slightly preferable in this category. We also argue that n determines the characteristic of the epidemic at an early stage. However, in the second category, the estimated asymptotic number of cumulative infected cases contain rather large uncertainty. Therefore, in the growth-dominated phase, we focus on using n = 1 for countries in this phase. Some of them are in an early stage of an epidemic with an insufficient amount of data leading to a large uncertainty on parameter fits. In terms of the accuracy of the size estimation, the results do strongly depend on limitations on data collection and the epidemic phase for each country. All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '4a24fcf8d2a7c69aef247ea12a740145daf68407',\n","  'section': 'Abstract',\n","  'text': \"The ability to comprehensively characterize exposures and immune responses to viral infections will be critical to better understanding human health and disease. We previously described the VirScan system, a phage-display based technology for profiling antibody binding to a comprehensive library of peptides designed to represent the human virome. The previous VirScan analytical approach did not fully account for disproportionate representation of viruses in the library or for antibody cross-reactivity among sequences shared by related viruses. Here we present the 'AntiViral Antibody Response Deconvolution Algorithm' ('AVARDA'), a multi-module software package for analyzing VirScan datasets. AVARDA provides a probabilistic assessment of infection at species-level resolution by considering alignment of all library peptides to each other and to all human viruses. We employed AVARDA to analyze VirScan data from a cohort of encephalitis patients with either known viral infections or undiagnosed etiologies. By comparing acute and convalescent sera, AVARDA successfully confirmed or detected antibody responses to human herpesviruses 1, 3, 4, 5, and 6, thereby improving the rate of diagnosing viral encephalitis in this cohort by 62.5%. We further assessed AVARDA's utility in the setting of an epidemiological study, demonstrating its ability to determine infections acquired in a child followed prospectively from infancy. We consider ways in which AVARDA's conceptual framework may be further developed in the future and describe how its analyses may be extended beyond investigations of viral infection. AVARDA, in combination with VirScan and other pan-pathogen serological techniques, is likely to find broad utility in the epidemiology and diagnosis of infectious diseases. All rights reserved. No reuse allowed without permission. \"},\n"," {'paper_id': '4a2cf4eeb236cbaa2885c9940554afc57cb4c9d5',\n","  'section': 'Abstract',\n","  'text': '29 Background: Our understanding of the corona virus disease 2019 (COVID-19) continues to 30 evolve. However, there are many unknowns about its epidemiology. 31 Purpose: To synthesize the number of deaths from confirmed COVID-19 cases, incubation 32 period, as well as time from onset of COVID-19 symptoms to first medical visit, ICU admission, 33 recovery and death of COVID-19. Data Synthesis: Out of 1675 non-duplicate studies identified, 57 were included. Pooled mean 41 incubation period was 5.84 (99% CI: 4.83, 6.85) days. Pooled mean number of days from the 42 onset of COVID-19 symptoms to first clinical visit was 4.82 (95% CI: 3.48, 6.15), ICU admission 43 was 10.48 (95% CI: 9.80, 11.16), recovery was 17.76 (95% CI: 12.64, 22.87), and until death 44 was 15.93 (95% CI: 13.07, 18.79). Pooled probability of COVID-19-related death was 0.02 (95% 45 CI: 0.02, 0.03). 46 Limitations: Studies are observational and findings are mainly based on studies that recruited 47 patient from clinics and hospitals and so may be biased toward more severe cases. 48 We found that the incubation period and lag between the onset of symptoms and diagnosis 50 of COVID-19 is longer than other respiratory viral infections including MERS and SARS; 51 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': '4aa612e5029df4c2a1db5a3655144affc80731be',\n","  'section': 'Abstract',\n","  'text': 'Current research is an attempt to understand the CoVID-19 pandemic curve through statistical approach of probability density function with associated skewness and kurtosis measures, change point detection and polynomial fitting to estimate infected population along with 30 days projection. The pandemic curve has been explored for above average affected countries, six regions and global scale during 64 days of 22nd January to 24th March, 2020. The global cases infection as well as recovery rate curves remained in the ranged of 0 -9.89 and 0 -8.89%, respectively. The confirmed cases probability density curve is high positive skewed and leptokurtic with mean global infected daily population of 6620. The recovered cases showed bimodal positive skewed curve of leptokurtic type with daily recovery of 1708. The change point detection helped to understand the CoVID-19 curve in term of sudden change in term of mean or mean with variance. This pointed out disease curve is consist of three phases and last segment that varies in term of day lengths. The mean with variance based change detection is better in differentiating phases and associated segment length as compared to mean. Global infected population might rise in the range of 0.750 to 4.680 million by 24 th April 2020, depending upon the pandemic curve progress beyond 24th March, 2020. Expected most affected countries will be USA, and UK with at least infected population of over 0.100 million. Infected population polynomial projection errors remained in the range of -78.8 to 49.0%. '},\n"," {'paper_id': '4b56993a4d0f091e70ef5dec2e5a61c3043f752e',\n","  'section': 'Abstract',\n","  'text': 'Motivation: Accurately determining the atomic structure of proteins represents a fundamental problem in the field of structural bioinformatics. A solution would be significant as protein structure information could be utilized in the medical field, e.g. in the development of vaccines for new viruses. This paper focuses on predicting the protein structure based on 3D images of the proteins captured through cryogenic electron microscopes (cryo-EM). A fully automated computationally efficient protein structure prediction method would be particularly beneficial in the field of cryo-EM as the technology allows researchers to photograph multiple large protein complexes in a single study, which means that a fast prediction method could allow for a high throughput of derived protein structures. We present a deep learning approach, DeepTracer, for predicting locations of the backbone atoms, secondary structure elements, and the amino acid types. In order to connect the predicted amino acids into chains, we applied a modified traveling salesman algorithm. Results: We trained our deep learning model on experimental cryo-EM density maps and tested it on a set of 50 density maps. We found that our new approach predicted protein structures with an average RMSD value of 1.18 and a coverage of 87.5%. Furthermore, we detected secondary structure information for 87.2% of amino acids correctly. We also showed preliminarily that 25.2% of amino acid types could be predicted directly from the 3D cryo-EM density map, considering 20 different types in total. Finally, we noted that the prediction runtime of DeepTracer is significantly improved compared to other methods. It predicts a large protein complex structure of more than 30,000 amino acids in only 2 hours. Availability: The repository of this project will be published. There are two main approaches to predict protein structures: homology modeling and de novo structure prediction. They differ in such that homology modeling requires structural knowledge about a homologous protein for its prediction while de novo methods do not. This limits the '},\n"," {'paper_id': '4b57ff228e46255a0ba7114af86ff26af4152d9b',\n","  'section': 'Abstract',\n","  'text': 'in host-pathogen interactions. CASD1 has been described as a sialate-Oacetyltransferase and has been shown to be essential for 9-O-acetylation of sialic acid in some cell lines in vitro. In this study, we used knockout mice to confirm that CASD1 is indeed responsible for 9-O-acetylation of sialic acids in vivo. We observed a complete loss of 9-O-acetylation of sialic acids on the surface of myeloid, erythroid and CD4 + T cells in Casd1-deficient mice. Although 9-O-acetylation of sialic acids on multiple hematopoietic lineages was lost, there were no obvious defects in hematopoiesis. Interestingly, red blood cells from Casd1-deficient mice also lost reactivity to TER-119, a rat monoclonal antibody that is widely used to mark the murine erythroid lineage. The sialic acid glyco-epitope recognized by TER-119 on red blood cells was sensitive to the sialic acid O-acetyl esterase activity of the hemagglutinin esterase from bovine coronavirus but not to the corresponding enzyme from the influenza C virus. During erythrocyte development TER-119 + Ery-A and Ery-B cells could be stained by catalytically inactive bovine coronavirus hemaggutinin-esterase but not by the inactive influenza C hemagglutinin esterase, while TER-119 + Ery-C stage cells and mature erythrocytes were recognized by both virolectins. These results suggest that throughout murine erythrocyte development, cells of the erythroid lineage express a glycoconjugate bearing a modified 7,9-di-O-acetyl form of sialic acid, that is recognized specifically by the bovine coronavirus lectin and not by the influenza C hemagglutinin, and this modified sialic acid moiety is a component of the TER-119 epitope. As erythrocytes mature, the surface of Ery-C cells and mature erythrocytes also acquires a distinct CASD1dependent 9-O-acetyl sialic acid moiety that can be recognized by virolectins from both influenza C and bovine coronavirus that are specific for 9-O-acetyl sialic acid. esterases from Influenza C and bovine coronavirus, we demonstrate that the murine erythroid lineage-specific epitope recognized by the TER-119 monoclonal antibody is comprised of a 7,9-di-O-acetyl sialic acid glycoform that is completely lost in Casd1deficient mice. The 7,9-di-O-acetyl sialic acid glycoform can be detected early in erythroid differentiation from the Ery-A and Ery-B stages onward. We also observed that a distinct 9-O-acetyl sialic acid moiety also decorates cells of the erythroid lineage from the Ery C stage onward including in mature red blood cells. RESULTS Prior in vitro studies have suggested that CASD1 is a sialic acid 9-O-acetyl transferase. To assess the function of 9-O-acetyl sialic acids in vivo, we generated Casd1-deficient mice (Fig 1A) . The influenza C hemagglutinin-esterase (CHE) that is known to be specific for 9-O-acetyl sialic acid was expressed as an Fc-fusion protein (CHE-Fc) (Muchmore and Varki, 1987). Inactivation of the esterase domain of CHE-Fc with diisopropyl fluorophosphate (DFP) yields CHE-FcD which has been widely used as a specific probe of 9-O-acetyl sialic acid (Klein et al., 1994) . Using the CHE-FcD reagent, we evaluated the effect of Casd1 deficiency on sialic acid 9-O-acetylation in all hematopoietic lineages, focusing our analysis on the spleen, bone marrow and lymph nodes. Wild-type myeloid cells expressing CD11b exhibit robust levels of 9-O-acetyl sialic acid, but this carbohydrate modification was completely lost in splenic CD11b + myeloid cells from Casd1-deficient mice (Fig 1B) . CD11b + cells in the bone marrow also express high levels of the mucin-like glycoprotein CD43, suggesting that much of the 9-. CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/469254 doi: bioRxiv preprint Mouse embryonic stem cells (C57/BL6N background) carrying a knockout-first targeted allele of Casd1 (Casd1 tm1a(EUCOMM)Hmgu ) were obtained from the EUCOMM consortium. Casd1 knock-in mice were generated at the BWH Transgenic Core facility. The resulting chimeric mice were crossed into C57BL/6NTac (Taconic) and interbred to homozygosity. They were then crossed into (B6.C-Tg(CMV-cre)1Cgn/J mice, (stock # 006054, Jackson Laboratory) to obtain the Casd1 tm1b null allele, which lacks exon 2 of Casd1. Excision of the neomycin expression cassette (Fig 1A) was confirmed by PCR. The following primers were used for genotyping. The wild-type allele was specifically amplified using TCCTCCCTCACTGTTCCTTC and AGGTGGGGAGGAAAGACAGT (291bp) and the Casd1 tm1b allele could be selectively amplified using TCCTCCCTCACTGTTCCTTC and GACAGCCAGTCACACAGCTT (5540bp). This allele was interbred to homozygosity to yield Casd1 tm1b/tm1b mice (referred to as Casd1-KO mice in the accompanying figures). The Casd1-deficient mice were viable and fertile. The Casd1 knockout colony was maintained by homozygous mating. Each experiment was performed at least twice on three mice each. Single cell suspensions were obtained from the blood, spleen, lymph nodes, thymus or bone marrow and flow cytometry analysis was performed using the following monoclonal (7G6) and CD16/CD32 (2.4G2). Monoclonal antibody clones are indicated in . CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/469254 doi: bioRxiv preprint parentheses. All antibodies were obtained from Biolegend Inc. unless otherwise indicated. One representative flow cytometry plot is shown for each condition. Influenza C hemagglutinin esterase (CHE) was recombinantly expressed as an Fcfusion protein and the esterase activity of CHE was inactivated with di-isofluoropropyl phosphate to generate the CHE-FcD reagent as previously described (Muchmore and Varki, 1987). Bovine coronavirus hemagglutinin esterase (Mebus strain, Uniprot accession number, P15776.1) was cloned into the pFUSE-mFc2a vector (Invivogen) into the EcoRI and BglII sites. A catalytically dead variant of BHE-Fc (BHE-S40A-Fc) was generated by site-directed mutagenesis. The Fc-fusion proteins were expressed in 293T cells and affinity purified using protein G. The CHE-FcD probe recognizes 9-O-acetyl sialic acids, and the BHE-S40A-Fc virolectin can bind both 9-O-acetyl and 7,9-di-Oacetyl sialic acids (Langereis et al., 2015; Bakkers et al., 2016). Briefly, a two-step staining procedure with CHE-FcD as the primary and anti-human IgG1-PE (clone HP6017) as the secondary or with BHE-S40A-Fc as the primary and anti-mouse IgG2a-FITC (clone RMG2a-62) as the secondary antibody were used. To assess the effect of CHE or BHE on 9-O-acetyl sialic acids on murine erythrocytes, a 1:100 dilution of whole blood in PBS, was treated with 10μg/mL of CHE-Fc or BHE-Fc at 37°C for 1 hour. The cells were washed twice with PBS at room temperature and stained with CHE-FcD as described above. Analysis of competitive binding between TER-119 and BHE-S40A-Fc was performed as follows. A 1:100 dilution of whole blood in PBS was incubated with varying concentrations of BHE-S40A-Fc at 37°C for one hour followed by the addition of 2 µg/mL of labeled TER-119 antibody for 10 minutes in the . CC-BY-NC-ND 4.0 International license is made available under a '},\n"," {'paper_id': '4b8bdaf9670d7c24f952ca38bfe6c699152421b7',\n","  'section': 'Abstract',\n","  'text': 'Understanding interspecific viral transmission is key to understanding viral ecology and evolution, 12 disease spillover into humans, and the consequences of global change. Prior work has 13 demonstrated that macroecological factors drive viral sharing in some mammalian groups, 14 but analyses have never attempted to predict viral sharing in a pan-mammalian context. Here 15 we show that host phylogenetic similarity and geographic range overlap are strong, nonlinear 16 predictors of viral sharing among species across the entire mammal class. Using these traits, 17 we predict global viral sharing patterns across 4196 mammal species and show that our 18 simulated network successfully predicts viral sharing and reservoir host status using internal 19 validation and an external dataset. We predict high rates of mammalian viral sharing in the 20 tropics, particularly among rodents and bats, and that within-and between-order sharing 21 differs geographically and taxonomically. Our results emphasize the importance of 22 macroecological factors in shaping mammalian viral communities, and provide a robust, 23 general model to predict viral host range and guide pathogen surveillance and conservation 24 efforts. '},\n"," {'paper_id': '4bc43084ccdb8d3bb704ca577473169cd1a64f9c',\n","  'section': 'Abstract',\n","  'text': 'Background: In December 2019, an outbreak of novel coronavirus disease emerged in Wuhan, China and has swiftly spread to other parts of China and a number of oversea countries. Our aim is to evaluate the effectiveness of the evolution of interventions and self-protection measures, estimate the risk of partial lifting control measures and predict the epidemic trend of the virus in mainland China excluding Hubei province based on the published data and a novel mathematical model. A novel COVID-19 transmission dynamic model incorporating the intervention measures implemented in China is proposed. We parameterize the model by using the Markov Chain Monte Carlo (MCMC) method and estimate the control reproduction number R c , as well as the effective daily reproduction ratio R e (t), of the disease transmission in mainland China excluding Hubei province. The estimation outcomes indicate that the control reproduction number is 3.36 (95% CI 3.20-3.64) and R e (t) has dropped below 1 since January 31st, 2020, which implies that the containment strategies implemented by the Chinese government in mainland China excluding Hubei province are indeed effective and magnificently suppressed COVID-19 transmission. Moreover, our results show that relieving personal protection too early may lead to the spread of disease for a longer time and more people would be infected, and may even cause epidemic or outbreak again. By calculating the effective reproduction ratio, we prove that the contact rate should be kept at least less than 30% of the normal level by April, 2020. To ensure the epidemic ending rapidly, it is necessary to maintain the current integrated restrict interventions and self-protection measures, including travel restriction, quarantine of entry, contact tracing followed by quarantine and isolation and reduction of contact, like wearing masks, etc. People should be fully aware of the real-time epidemic situation and keep sufficient personal protection until April. If all the above conditions are met, the outbreak is expected to be ended by April in mainland China apart from Hubei province. '},\n"," {'paper_id': '4bd697fdded7ee65154e28def586a7e9b60e7078',\n","  'section': 'Abstract',\n","  'text': 'The COVID-19 outbreak has been a serious public health threat worldwide. We use individually documented case descriptions of COVID-19 from China (excluding Hubei Province) to estimate the distributions of the generation time, incubation period, and periods from symptom onset to isolation and to diagnosis. The recommended 14-day quarantine period may lead to a 6.7% failure for quarantine. We recommend a 22-day quarantine period. The mean generation time is 3.3 days and the mean incubation period is 7.2 days. It took 3.7 days to isolate and 6.6 days to diagnose a patient after his/her symptom onset. Patients may become infectious on average 3.9 days before showing major symptoms. This makes contact tracing and quarantine ineffective. The basic reproduction number is estimated to be 1.54 with contact tracing, quarantine and isolation, mostly driven by super spreaders. '},\n"," {'paper_id': '4c159330655513984fac5ab8e0c575512d33b514',\n","  'section': 'Abstract',\n","  'text': \"While traditional microbiological freshwater tests focus on the detection of specific bacterial indicator species including pathogens, direct tracing of all aquatic DNA through metagenomics poses a profound alternative. Yet, in situ metagenomic water surveys face substantial challenges in cost and logistics. Here we present a simple, fast, inexpensive and remotely accessible freshwater diagnostics workflow centred around the portable nanopore sequencing technology. Using defined compositions and spatiotemporal microbiota from surface water of an example river in Cambridge (UK), we provide optimised experimental and bioinformatics guidelines, including a benchmark with twelve taxonomic classification tools for nanopore sequences. We find that nanopore metagenomics depicts both, the hydrological core microbiome and fine temporal gradients in line with complementary physicochemical measurements. Using reference-based sequence clustering, these data feature relevant sewage signals and pathogen maps at species level resolution. We anticipate that this framework will gather momentum for new environmental monitoring initiatives using portable devices. Extended Data Figure 2: Bioinformatics consensus workflow. Essential data processing steps, from nanopore sequencing to spatiotemporal bacterial composition analysis (Methods). After full-length 16S rDNA sequencing with the MinION (R9.4 flow cell), local basecalling of the raw fast5 files was performed using Guppy 69 . Output fastq files were filtered for length and quality (Methods), and reads assigned to their location barcode using Porechop. We then used Minimap2 15 (k = 15) and the SILVA v.132 database 16 for taxonomic classifications. Rarefaction reduced each sample to the same number of reads (37,000), allowing for a robust comparison of bacterial composition across samples in various downstream analyses. Extended Data Figure 3: Impact of rarefaction on diversity estimation. (a) Example rarefaction curve for bacterial family classifications of the 'April-1' sample. The chosen cut-off preserves most (~90 %) of the original family taxon richness (vertical line). (b) Difference between original and rarefied family richness at 37,000 reads across all freshwater sequencing runs with quantitative sequencing outputs above the chosen cut-off. Boxplots feature 100 independent rarefactions per sample. Error bars represent Q1 -1.5*IQR (lower), and Q3 + 1.5*IQR (upper), respectively. (c) Diversity visualisation of the ten most abundant bacterial families across all samples with sequencing outputs >37,000 reads, through 400 'unordered bubbles'. Taxonomic proportions and colours are in accordance with Figure 2b. Shannon (H) and Simpson (D) indices for all samples indicate marginal differences between pairs of original and rarefied sets. Extended Data Figure 4: Principal component analysis of river bacterial family compositions. (a-b) PCA with two independent rarefaction sets to 37,000 reads in all freshwater sequencing samples. Numbers and coloured dots indicate locations for each time point. The first and second principal components (PC1 and PC2, combined variance: ~41 %) robustly capture outlier samples 'April-7' along PC1 and 'April-2', 'August-4' and 'April-8' along PC2. (c-d) Fractional loads of the ten bacterial families most strongly contributing to changes along PC1 (c) and along PC2 (d). Error bars represent standard deviation of these families to the respective PC across four independent rarefactions. Subsequent principal components (PC3 and PC4) are less outlier-driven and depict spatial and temporal metagenomic trends within the River Cam. \"},\n"," {'paper_id': '4c2d265c52f6bef1ab88d1a3fc984f44f5851b67',\n","  'section': 'Abstract',\n","  'text': 'Background: Concerns have been raised regarding the safety of Angiotensin Converting Enzyme Inhibitors (ACEIs) and Angiotensin Receptor Blockers (ARBs) in patients with COVID-19, based on the hypothesis that such medications may raise expression of ACE2, the receptor for SARS-CoV-2. We conducted a literature review of studies (n=12) in experimental animals and human subjects (n=11) and evaluated the evidence regarding the impact of administration of ACEIs and ARBs on ACE2 expression. We prioritized studies that assessed ACE2 protein expression data, measured directly or inferred from ACE2 activity assays. The findings in animals are inconsistent with respect to an increase in ACE2 expression in response to treatment with ACEIs or ARBs. Control/sham animals show little to no effect in the plurality of studies. Those studies that report increases in ACE2 expression tend to involve acute injury models and/or higher doses than typically administered to patients. Data from human studies overwhelmingly imply that administration of ACEIs/ARBs does not increase ACE2 expression. Conclusion: Available evidence, in particular, data from human studies, does not support the hypothesis that ACEI/ARB use increases ACE2 expression and the risk of complications from COVID-19. We conclude that patients being treated with ACEIs and ARBs should continue their use for approved indications. '},\n"," {'paper_id': '4c31b4667c88438ca8f01c976f8859c3c58a17f5',\n","  'section': 'Abstract',\n","  'text': 'Recently emerged coronavirus designated as SARS-CoV-2 (also known as 2019 novel coronavirus (2019-nCoV) or Wuhan coronavirus) is a causative agent of coronavirus disease 2019 , which is rapidly spreading throughout the world now. More than 9,00,000 cases of SARS-CoV-2 infection and more than 47,000 COVID-19-associated mortalities have been reported worldwide till the writing of this article, and these numbers are increasing every passing hour. World Health Organization (WHO) has declared the SARS-CoV-2 spread as a global public health emergency and admitted that the COVID-19 is a pandemic now. The multiple sequence alignment data correlated with the already published reports on the SARS-CoV-2 evolution and indicated that this virus is closely related to the bat Severe Acute Respiratory Syndrome-like coronavirus (bat SARS-like CoV) and the well-studied Human SARS coronavirus (SARS CoV). The disordered regions in viral proteins are associated with the viral infectivity and pathogenicity. Therefore, in this study, we have exploited a set of complementary computational approaches to examine the dark proteomes of SARS-CoV-2, bat SARS-like, and human SARS CoVs by analysing the prevalence of intrinsic disorder in their proteins. According to our findings, SARS-CoV-2 proteome contains very significant levels of structural order. In fact, except for Nucleocapsid, Nsp8, and ORF6, the vast majority of SARS-CoV-2 proteins are mostly ordered proteins containing author/funder. All rights reserved. No reuse allowed without permission. : bioRxiv preprint 2 less intrinsically disordered protein regions (IDPRs). However, IDPRs found in SARS-CoV-2 proteins are functionally important. For example, cleavage sites in its replicase 1ab polyprotein are found to be highly disordered, and almost all SARS-CoV-2 proteins were shown to contain molecular recognition features (MoRFs), which are intrinsic disorder-based protein-protein interaction sites that are commonly utilized by proteins for interaction with specific partners. The results of our extensive investigation of the dark side of the SARS-CoV-2 proteome will have important implications for the structural and non-structural biology of SARS or SARS-like coronaviruses. Significance: The infection caused by a novel coronavirus (SARS-CoV-2) that causes severe respiratory disease with pneumonia-like symptoms in humans is responsible for the current COVID-19 pandemic. No in-depth information on structures and functions of SARS-CoV-2 proteins is currently available in the public domain, and no effective anti-viral drugs and/or vaccines are designed for the treatment of this infection. Our study provides the first comparative analysis of the order-and disorder-based features of the SARS-CoV-2 proteome relative to human SARS and bat CoV that may be useful for structure-based drug discovery. '},\n"," {'paper_id': '4c35605124dd8011395240fac71e80b52ddbbe7e',\n","  'section': 'Abstract',\n","  'text': 'Infections by RNA viruses such as Influenza, HIV still pose a serious threat to human health despite extensive research on viral diseases. One challenge for producing effective prevention and treatment strategies is high intra-species genetic diversity. As different strains may have different biological properties, characterizing the genetic diversity is thus important to vaccine and drug design. Next-generation sequencing technology enables comprehensive characterization of both known and novel strains and has been widely adopted for sequencing viral populations. However, genome-scale reconstruction of haplotypes is still a challenging problem. In particular, haplotype assembly programs often produce contigs rather than full genomes. As a mutation in one gene can mask the phenotypic effects of a mutation at another locus, clustering these contigs into genome-scale haplotypes is still needed. Results: We developed a contig binning tool, VirBin, which clusters contigs into different groups so that each group represents a haplotype. Commonly used features based on sequence composition and contig coverage cannot effectively distinguish viral haplotypes because of their high sequence similarity and heterogeneous sequencing coverage for RNA viruses. VirBin applied prototype-based clustering to cluster regions that are more likely to contain mutations specific to a haplotype. The tool was tested on multiple simulated sequencing data with different haplotype abundance distributions and contig sizes, and also on mock quasispecies sequencing data. The benchmark results with other contig binning tools demonstrated the superior sensitivity and precision of VirBin in contig binning for viral haplotype reconstruction. Availability: https://github.com/chjiao/VirBin Contact: yannisun@cityu.edu.hk '},\n"," {'paper_id': '4c84d1153daa335f5ff6d61db665024b32ac8195',\n","  'section': 'Abstract',\n","  'text': 'Background In December 2019, novel coronavirus (SARS-CoV-2) infected pneumonia occurred in Wuhan, China. The number of cases has increased rapidly but information on the clinical characteristics of SARS-CoV-2 pneumonia without comorbidities compared to normal controls in Chinese Han population is limited. Our objective is to describe the epidemiological and clinical characteristics of SARS-CoV-2 pneumonia without comorbidities compared to normal controls in the Chinese Han population. Methods Retrospective, multi-center case series of the 69 consecutive hospitalized patients with confirmed SARS-CoV-2 pneumonia, The study population included 69 hospitalized patients with confirmed SARS-CoV-2 pneumonia without comorbidities and 14,117 normal controls. 50.7% patients were male and 49.3% were female; 1.5% patients were asymptomatic cases, 63.8% patients were mild cases, and 36.2% patients were severe or critical cases. Compared with mild patients (n=44), severe or critical patients (n=25) were significantly older (median age, 67 years [IQR, vs. 49 years [IQR, 36-60]; p<0.01). Fever was present in 98.6% of the patients. The second most common symptom was cough (62.3%), fatigue (58.0%), sputum (39.1%), and headache (33.3%). The median incubation period was 4 days (IQR, 2 to 7). Leukocyte count was 74.1% of normal controls and lymphocyte count was 45.9% of normal controls. The phenomenon of lymphocyte depletion (PLD) observed in severe or critical cases in 100%. Levels of lactate dehydrogenase, D-dimer, procalcitonin, and interleukin-6 were showed significant differences between mild and severe or critical cases. Chest computed tomographic scans showed bilateral patchy patterns (49.3%), local patchy shadowing (29.0%), and ground glass opacity (21.7%). 7.3% patients were diagnosed ARDS, 7.3% patients were diagnosed acute cardiac injury (troponin I >28 pg/mL) and 4.4% patients were diagnosed fungal infections or shock. 4.3% patients have been discharged; 1.5% patient had died; 1.5% patient had recovery. Conclusions In this multicenter case series of 69 patients without comorbidities, the full All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. : medRxiv preprint 3 spectrum of asymptomatic, mild, severe, and critical cases is described. 50.7% patients were male and 49.3% were female; 1.5% patients were asymptomatic cases, 63.8% patients were mild cases, and 36.2% patients were severe or critical cases. 4.3% patients have been discharged; 1.5% patient had died; 1.5% patient had recovery. Among the 25 patients with severe or critical disease, 12.0% patients were underwent non-invasive mechanical ventilation, 8.0% patients underwent invasive mechanical ventilation, and 4.0% patients died. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '4c96d9cfc6da83e41b0889579d1e8b25765bd355',\n","  'section': 'Abstract',\n","  'text': 'Wuhan, China. The number of cases has increased rapidly but information on the clinical characteristics of SARS-CoV-2 pneumonia compared to normal controls in Chinese Han population is limited. Our objective is to describe the clinical characteristics of SARS-CoV-2 pneumonia compared to normal controls in the Chinese Han population. In this case series of 752 patients, the full spectrum of cases is described. Fever was present in 86-90% of the patients. The second most common symptom was cough (49.1-51.0%), fatigue (25.2-27.1%), sputum (20.0-23.1%), and headache (9.8-11.1%). the mortality rate is 4.6% in Wuhan, 1.9% in Beijing, and 0.9% in Shanghai. Our findings showed that the levels of lymphocytes were 0.8 (IQR, 0.6-1.1) 10 9 /L in Wuhan, 1.0 (IQR, 0.7-1.4) 10 9 /L in Beijing, and 1.1 (IQR, 0.8-1.5) 10 9 /L in Shanghai before admission to hospitals, respectively, indicating that cellular immune function might relate to the mortality. Based on the reference ranges of normal Chinese Han population and the data of the critically ill patients we have observed, it is recommended that reference ranges of people at high risk of COVID-19 infection are CD3 + lymphocytes below 900 cells/mm 3 , CD4 + lymphocytes below 500 cells/mm 3 , and CD8 + lymphocytes below 300 cells/mm 3 . All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '4cc91ce442f6fd13456d4e51508605fbcf891ecd',\n","  'section': 'Abstract',\n","  'text': 'Background: A range of case fatality ratio (CFR) estimates for COVID-19 have been produced that differ substantially in magnitude. We used individual-case data from mainland China and cases detected outside mainland China to estimate the time between onset of symptoms and outcome (death or discharge from hospital). We next obtained age-stratified estimates of the CFR by relating the aggregate distribution of cases by dates of onset to the observed cumulative deaths in China, assuming a constant attack rate by age and adjusting for the demography of the population, and age-and location-based underascertainment. We additionally estimated the CFR from individual line-list data on 1,334 cases identified outside mainland China. We used data on the PCR prevalence in international residents repatriated from China at the end of January 2020 to obtain age-stratified estimates of the infection fatality ratio (IFR). Using data on age-stratified severity in a subset of 3,665 cases from China, we estimated the proportion of infections that will likely require hospitalisation. We estimate the mean duration from onset-of-symptoms to death to be 17.8 days (95% credible interval, crI 16.9-19.2 days) and from onset-of-symptoms to hospital discharge to be 22.6 days (95% crI 21.1-24.4 days). We estimate a crude CFR of 3.67% (95% crI 3.56%-3.80%) in cases from mainland China. Adjusting for demography and under-ascertainment of milder cases in Wuhan relative to the rest of China, we obtain a best estimate of the CFR in China of 1.38% (95% crI 1.23%-1.53%) with substantially higher values in older ages. Our estimate of the CFR from international cases stratified by age (under 60 / 60 and above) are consistent with these estimates from China. We obtain an overall IFR estimate for China of 0.66% (0.39%-1.33%), again with an increasing profile with age. Interpretation: These early estimates give an indication of the fatality ratio across the spectrum of COVID-19 disease and demonstrate a strong age-gradient in risk. '},\n"," {'paper_id': '4d1090e239baa1ea54af521f4b8bd8b5ab51fd35',\n","  'section': 'Abstract',\n","  'text': \"A discrete dynamic model for human epidemics was developed in present study. The model included major parameters as transmission strength and its decline parameters, mean incubation period, hospitalization time, non-hospitalization daily mortality, non-hospitalization daily recovery rate, and hospitalization proportion, etc. Sensitivity analysis of the model indicated the total cumulative cases significantly increased with initial transmission strength, hospitalization time. The total cumulative cases significantly decreased with transmission strength's decline and hospitalization proportion, and linearly decreased with non-hospitalization daily mortality and non-hospitalization daily recovery rate. In a certain range, the total cumulative cases significantly increased with mean incubation period. Sensitivity analysis demonstrated that dynamic change of transmission strength is one of the most important and controllable factors. In addition, reducing the delay for hospitalization is much effective in weakening disease epidemic. Non-hospitalization recovery rate is of importance for enhancing immunity to recover from the disease. \"},\n"," {'paper_id': '4d2600919d48235342c2d36933db873933a9aab8',\n","  'section': 'Abstract',\n","  'text': '300 words, Text 2737 words All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. \\uf0b7 Alongside age, pre-existing medical conditions are perceived negatively during triage assessments, particularly if rare, and/or theoretically expected to influence cardiorespiratory risk; \\uf0b7 Anaesthetists use cardiopulmonary exercise testing to categorise patients to higher and lower risk independently to diagnostic labels, but this is not feasible in acute settings; \\uf0b7 Pulmonary arteriovenous malformations are an exemplar of a condition where, due to expected or measured abnormalities (hypoxaemia-low PaO2 SpO2), poor physiological capacity might be predicted. \\uf0b7 Neither age nor baseline SpO2 predicted lower/higher risk categories by anaerobic threshold, but haemoglobin-dependent indices of oxygen delivery to the tissues were associated with higher risk, offering opportunities for improvement by attention to anaemia and aerobic conditioning; \\uf0b7 Baseline exercise tolerance may override age and diagnostic labels in triage settings: the 13point VSAQ Veterans Specific Activity Questionnaire (VSAQ) is suggested as a rapid screening tool for cardiorespiratory risk assessment. Abstract 1 2 BACKGROUND: Rapid triaging, as in the current COVID-19 pandemic, focuses on age and pre-existing 3 medical conditions. In contrast, preoperative assessments use cardiopulmonary exercise testing (CPET) 4 to categorise patients to higher and lower risk independent of diagnostic labels. Since CPET is not 5 feasible in population-based settings, our aims included evaluation of a triage/screening tool for 6 cardiorespiratory risk. 7 METHODS: CPET-derived anaerobic thresholds were evaluated retrospectively in 26 patients with 8 pulmonary arteriovenous malformations (AVMs) who represent a challenging group to risk-categorise. 9 Pulmonary AVM-induced hypoxaemia secondary to intrapulmonary right-to-left shunts, anaemia from 10 underlying hereditary haemorrhagic telangiectasia and metabolic equivalents derived from the 13-point 11 Veterans Specific Activity Questionnaire (VSAQ) were evaluated as part of routine clinical care. Pre-12 planned analyses evaluated associations and modelling of the anaerobic threshold and patient-specific 13 variables. 14 RESULTS: In the 26 patients (aged 21-77, median 57 years), anaerobic threshold ranged from 7.6-24.5 15 (median 12.35) ml.min -1 kg -1 and placed more than half of the patients (15, 57.7%) in the >11 ml.min -16 1 kg -1 category suggested as \"lower-risk\" for intra-abdominal surgeries. Neither age nor baseline SpO2 17 predicted anaerobic threshold, or lower/higher risk categories, either alone or in multivariate analyses, 18 despite baseline oxygen saturation (SpO2) ranging from 79 to 99 (median 92)%, haemoglobin from 108 19 to 183 (median 156)g.L -1 . However, lower haemoglobin, and particularly, arterial oxygen content and 20 oxygen pulse, were associated with increased cardiorespiratory risk: Modelling a haemoglobin increase 21 of 25g.L -1 placed a further 7/26 (26.9%) patients in a lower risk category. For patients completing the 22 VSAQ, derived metabolic equivalents were strongly associated with anaerobic threshold, enabling risk 23 evaluations through a simple questionnaire. CONCLUSIONS: Baseline exercise tolerance may override age and diagnostic labels in triage settings. These data support approaches to risk reduction by aerobic conditioning and attention to anaemia. The VSAQ is suggested as a rapid screening tool for cardiorespiratory risk assessment to implement during 27 triage/screening. All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. : medRxiv preprint 52 (VSAQ [5]). This is a simple 13-point scale of activities of increasing difficulty whereby the user 53 indicates which activity normally causes them to stop when performed for a period of time.[5] The 54 activities correspond to metabolic equivalents (METs), and numerous studies show a good correlation 55 with AT derived from cardiopulmonary exercise testing [6-8], mortality [9] and postoperative 56 complications [10]. All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. : medRxiv preprint applied the VSAQ to observational studies in patients with pulmonary AVMs and HHT, [17, 18] we 75 hypothesised that this could prove to be a useful risk categorising tool for triage purposes across wider 76 patient groups. 77 78 79 Methods 80 General patient evaluations 81 With ethical approvals (LREC 00/5792), patient indices derived as part of the clinical assessment 82 process in a pulmonary AVM service at a single centre were examined as described elsewhere 83 All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. : medRxiv preprint 128 ischaemic attack, or venous thromboemboli, one was in atrial fibrillation, one had well controlled 129 hypertension, one was hypercholesterolaemic, one was significantly depressed, and two had benign 130 prostatic hypertrophy. 132 CPET Demographics identify a low risk group As presented in Table 1 , based on the established anaerobic threshold delimiter of 11 ml.min -1 kg -1 , 134 more than half of the cohort with pulmonary AVMs (15/26, 57.7%) were categorised as pre-operative 135 \"low risk\", comprising 13 males and 2 females. The low risk group achieved a median 97% of their 136 predicted maximum work rate compared to the high risk group median of 68% predicted (Table 1) . Similarly, the median peak ̇o 2 in the low risk group was 160% of the median in the high risk group. All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. : medRxiv preprint 157 [95% CI 43.8, 61.5] years, versus higher risk mean 53.2 [95% CI 43.2, 63.3] years (Table 2). In keeping 158 All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. A different picture emerged when examining markers of oxygen delivery to the tissues, thus confirming 182 that the study did have sufficient power to be discerning: First, the higher risk groups had a trend towards lower haemoglobin, although there was some overlap 185 in confidence intervals. Categorised by an anaerobic threshold delimiter of 11 ml.min -1 kg -1 , the mean 186 [95% CI] values were 159 [14.9, 17.0] g.L -1 for the low risk group compared to 144 [13.1, 15.6] g.L -1 187 for the high risk group (Table 1, Figure 3) . By the upper/lower 50 th percentiles, the respective means 188 [95% CI] were 158 [146, 171] g.L -1 for the lower risk group compared to 147 [135, 159] g.L -1 for the 189 higher risk group (Table 2). 190 191 More strikingly, the higher risk groups had significantly lower arterial oxygen content (CaO2) 192 representing the oxygen content per unit volume of arterial blood, and calculated based on the oxygen 193 carriage of 1.34mls per gram of fully saturated haemoglobin: Categorised by an anaerobic threshold 194 delimiter of 11 ml.min -1 .kg -1 , the mean [95% CI] values were 19.6[18.4, 21.0] ml.dL -1 for the low risk 195 group, but 17.4 [15.8, 19.0] ml.dL -1 for the high risk group (Table 1, Figure 3). Categorised by the 196 upper/lower 50 th percentiles, the means [95% CI] were 19.6 [18.1, 21.0] ml.L -1 for the lower risk group 197 but only 17.8 [16.4, 19.3] ml.L -1 for the higher risk group (Table 2). 198 199 Similarly, the higher risk group had a significantly lower mean oxygen pulse, representing the amount 200 of oxygen extracted/ delivered per heart beat: Categorised by an anaerobic threshold delimiter of 11 201 ml.min -1 kg -1 , the mean [95% CI] values were 14.4 [11.0, 17.3] ml.beat -1 for the low risk group, but 9.7 202 [6.7, 12.6] ml.beat -1 for the high risk group (Table 1). Categorised by the upper/lower 50 th percentiles, 203 the mean [95% CI] values were 14.6[12.3, 16.9] ml.L -1 for the lower risk group but only 9.7 [7.2, 12.1] 204 ml.L -1 for the higher risk group (Table 2). All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '4d60493c2a4c1369445f11035a9f69f10a197468',\n","  'section': 'Abstract',\n","  'text': 'In the wake of COVID-19 disease, caused by the SARS-CoV-2 virus, we designed and developed a predictive model based on Artificial Intelligence (AI) and Machine Learning algorithms to determine the health risk and predict the mortality risk of patients with COVID-19. In this study, we used documented data of 117,000 patients world-wide with laboratory-confirmed COVID-19. This study proposes an AI model to help hospitals and medical facilities decide who needs to get attention first, who has higher priority to be hospitalized, triage patients when the system is overwhelmed by overcrowding, and eliminate delays in providing the necessary care. The results demonstrate 93% overall accuracy in predicting the mortality rate. We used several machine learning algorithms including Support Vector Machine (SVM), Artificial Neural Networks, Random Forest, Decision Tree, Logistic Regression, and K-Nearest Neighbor (KNN) to predict the mortality rate in patients with COVID-19. In this study, the most alarming symptoms and features were also identified. Finally, we used a separate dataset of COVID-19 patients to evaluate our developed model accuracy, and used confusion matrix to make an in-depth analysis of our classifiers and calculate the sensitivity and specificity of our model. '},\n"," {'paper_id': '4da8a87e614373d56070ed272487451266dce919',\n","  'section': 'Abstract',\n","  'text': 'Deep sequencing of clinical samples is now an established tool for the detection of infectious pathogens, with direct medical applications. The large amount of data generated provides an opportunity to detect species even at very low levels, provided that computational tools can effectively interpret potentially complex metagenomic mixtures. Data interpretation is complicated by the fact that short sequencing reads can match multiple organisms and by the lack of completeness of existing databases, in particular for viral pathogens. This interpretation problem can be formulated statistically as a mixture model, where the species of origin of each read is missing, but the complete knowledge of all species present in the mixture helps with the individual reads assignment. Several analytical tools have been proposed to approximately solve this computational problem. Here, we show that the use of parallel Monte Carlo Markov chains (MCMC) for the exploration of the species space enables the identification of the set of species most likely to contribute to the mixture. The added accuracy comes at a cost of increased computation time. Our approach is useful for solving complex mixtures involving several related species. We designed our method specifically for the analysis of deep transcriptome sequencing datasets and with a particular focus on viral pathogen detection, but the principles are applicable more generally to all types of metagenomics mixtures. The work is implemented as a user friendly R package, available from CRAN: http://cran.r-project.org/web/packages/metaMix . Pathoscope identified 47 species. Of these 42 are members of the metagenomic community. 42 are the exact same strain, while 3 are either the same species but different strain or same genus but different species. However it fails to detect 68 species that are actually present in the mixture. Tuning the parameter that . CC-BY-NC-ND 4.0 International license is made available under a '},\n"," {'paper_id': '4dbde66230e386fe4ba89b6dd17382527631964c',\n","  'section': 'Abstract',\n","  'text': 'SARS-CoV-2 emerged in China at the end of 2019 and has rapidly become a pandemic with over 400,000 recorded COVID-19 cases and greater than 19,000 recorded deaths by March 24 th , 2020 (www.WHO.org) (1). There are no FDA approved antivirals or vaccines for any coronavirus, including SARS-CoV-2 (2). Current treatments for COVID-19 are limited to supportive therapies and off-label use of FDA approved drugs (3). Rapid development and human testing of potential antivirals is greatly needed. A potentially quicker way to test compounds with antiviral activity is through drug re-purposing (2, 4). Numerous drugs are already approved for use in humans and subsequently there is a good understanding of their safety profiles and potential side effects, making them easier to test in COVID-19 patients. Here, we present data on 20 FDA approved drugs tested for antiviral activity against SARS-CoV-2 that we have previously found to inhibit SARS-CoV and MERS-CoV (4). We find that 17 of these . CC-BY-NC-ND 4.0 : bioRxiv preprint also inhibit SARS-CoV-2 at a range of IC50 values at non-cytotoxic concentrations. From these we specifically followed up with hydroxychloroquine sulfate and chloroquine phosphate. '},\n"," {'paper_id': '4de1480ef058af6f4a43f5fc13aab543cd8dd088',\n","  'section': 'Abstract',\n","  'text': 'Background The COVID-19 pandemic caused more than 800,000 infections and 40,000 deaths 31 by the end of March 2020. However, some of the basic epidemiological parameters, such as the 32 exponential epidemic growth rate, are not well estimated, partially because of confounding 33 factors in data collection during a rapidly growing outbreak, such as underreporting, delays in 34 case confirmation and changes in surveillance intensities. 35 36 Methods We developed an inference approach using a mathematical model to control for these 37 confounding factors. We fitted the model to both infection incidence and death count data 38 collected from eight European countries and the US. Public health implications of empirical 39 estimates were examined using simulations. 40 41 Findings In all countries, the early epidemic period was characterized by exponential growth 42 with rates between 0·19-0·29/day (epidemic doubling times between 2·4-3·7 days). However, 43 the proportion of cases that had been detected was low (less than 20% of cases detected) except 44 for Germany (23·1%; CI: 5-85%). With such high epidemic growth rates, moderate intervention 45 efforts will have little impact on the public health outcome; high levels of efforts to achieve 46 greater than 77-86% reduction in transmission are needed, no matter the goal is to slowdown the 47 growth to protect a large fraction of population from infection within 18 months or to reverse the 48 growth all together. '},\n"," {'paper_id': '4e2bc97a6164191ba3d53abfa49ccd973aa80326',\n","  'section': 'Abstract',\n","  'text': 'Conclusion: CT severity score was associated with inflammatory levels and higher NLR and CT severity score on admission were independent risk factors for short-term progression in patients with NCP outside Wuhan. Furthermore, early admission and surveillance by CT should be recommended to improve clinical outcomes. '},\n"," {'paper_id': '4e3f4e31c370281c2d721d6113265c25d9c566c5',\n","  'section': 'Abstract',\n","  'text': 'Human astrovirus nonstructural protein nsP1a/4, located at the C-terminal end of 17 nsP1a, is thought to be involved in the regulation of RNA replication and capsid 18 maturation;however, its rolesviral growth and virulence are not well understood. 19 We investigated the intracellular host proteins that interact with nsP1a and 20 explored the potential roles of the interaction in the pathogenesis of human 21 astrovirus infection. We screened 14 independent proteins with a cDNA library 22 a nonstructural protein located at the C-terminal end of the human astrovirus 38 nsP1a polyprotein, interacts with the host protein CD63. Over expression of 39 CD63 promoted viral RNA replication, whereas knockdown of CD63 decreased 40 virus RNA replication, indicating that CD63 plays a critical role in the human 41 astrovirus life cycle. 42 43 KEYWORDS: human astrovirus, HAstV, protein-protein interaction, nsP1a, 44 nsP1a/4, CD63, virus replication 45 46 author/funder. All rights reserved. No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/331835 doi: bioRxiv preprint Viruses are obligate intracellular pathogens thatutilize host-cell machinery to 47 infect cells, replicate themselves, and then exit the cells for another round of 48 infection. Human astroviruses (HAstVs) are a group of non-enveloped, 49 single-stranded, and positive-sense RNA viruses in the family Astroviridae. 50 HAstVs are causative agents of viral diarrhea in young children, 51 immunocompromised patients, and elderly individuals (1-5). The astrovirus 52 genome has three open reading frames (ORFs); ORF1a and ORF1b encode two 53 nonstructural polyproteins (nsPs), and ORF2 encodes the capsid protein (6). The 54 nsP1a polyprotein, transcribed from ORF1a, is involved in viral transcription 55 and replication. When HAstV infects a host cell, nsP1a is cleaved into at least 56 four products, named nsP1a/1, nsP1a/2, nsP1a/3 (protease), and nsP1a/4. The 57 nsP1a/4 protein is located at the C-terminal end of nsP1a.Several domains have 58 been identified in nsP1a/4, including two coiled-coil regions, a death domain 59 (DD), a nuclear localization signal (NLS), a putative viral genome-linked 60 protein(VPg), and a hypervariable region (HVR)(7-12). It was confirmed that 61 nsP1a/4 plays important roles in the activation of capsid maturation and the 62 regulation of RNA replication (13, 14). To better understand the roles of nsP1a 63 and nsP1a/4 in HAstV infection and replication, we characterized host proteins 64 that interact with nsP1a or nsP1a/4 by utilizing a yeast two-hybrid approach. We 65 identified at least 14 host proteins that could bind to nsP1a, one of which was 66 CD63. CD63, a member of the tetraspan transmembrane protein family, is a 68 author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '4e4797b808c48e3b9514b3f598ebcff6a598eddc',\n","  'section': 'Abstract',\n","  'text': 'The Coronavirus disease -19 (COVID19) that started in Wuhan, China in December 2019 has spread worldwide emerging as a global pandemic. The severe respiratory pneumonia caused by the novel SARS-CoV-2 has so far claimed more than 14,500 lives and has impacted human lives worldwide. Development of universal vaccines against the novel SARS-CoV-2 holds utmost urgency to control COVID19 pandemic that appears to be more severe than any of the previous outbreaks of severe acute respiratory syndrome (SARS) and Middle-East respiratory syndrome (MERS). However, as the novel SARS-CoV-2 displays high transmission rates, the underlying severity hidden in the SARS-CoV2 genomes is required to be fully understood. We studied the complete genomes of 95 strains of SARS-CoV-2 reported from different geographical regions worldwide to uncover the pattern of spread of the novel SARS-CoV-2 across the globe. We show that there is no direct transmission pattern of the virus among neighbouring countries suggesting that the outbreak is a result of travel of infected humans to different countries. We revealed unique single nucleotide polymorphisms (SNPs) in nsp13, nsp14, nsp15, nsp16 (present in ORF1b polyprotein region) and S-Protein within 10 viral isolates from USA. These viral proteins are involved in RNA replication and processing, indicating highly evolved strains of the novel SARS-CoV-2 circulating in the population of USA than in other countries. Furthermore, we found an isolate from USA (MT188341) to carry frameshift mutation between positions 2540 and 2570 of nsp16 which functions as mRNA cap-1 methyltransferase (2′-O-MTase). Thus, we reason that the replicative machinery of the novel SARS-CoV-2 is fast evolving to evade host challenges and survival. These mutations are needed to be considered, otherwise it will be difficult to develop effective treatment strategies. The two proteins also had dN/dS values approaching 1-ORF1ab polyprotein (dN/dS= 0.996, 0.575) and S protein (dN/dS= 0.88) and might confer selective advantage to the virus. Through '},\n"," {'paper_id': '4e4beb03faeb030e9e3b13d47b8b7a197f4715c8',\n","  'section': 'Abstract',\n","  'text': 'Background: Conflicting recommendations exist related to whether masks have a protective effect on the spread of respiratory viruses. Results: A total of 21 studies met our inclusion criteria. Meta-analyses suggest that mask use provided a significant protective effect (OR = 0.35 and 95% CI = 0.24-0.51). Use of masks by healthcare workers (HCWs) and non-healthcare workers (Non-HCWs) can reduce the risk of respiratory virus infection by 80% (OR = 0.20, 95% CI = 0.11-0.37) and 47% (OR = 0.53, 95% CI = 0.36-0.79). The protective effect of wearing masks in Asia (OR = 0.31) appeared to be higher than that of Western countries (OR = 0.45). Masks had a protective effect against influenza viruses (OR = 0.55), SARS (OR = 0.26), and SARS-CoV-2 (OR = 0.04). In the subgroups based on different study designs, protective effects of wearing mask were significant in cluster randomized trials, case-control studies and retrospective studies. Conclusions: This study adds additional evidence of the enhanced protective value of masks, '},\n"," {'paper_id': '4e74fa29e6eb9dfba40b7c3b2af788f93cf14564',\n","  'section': 'Abstract',\n","  'text': 'Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2 '},\n"," {'paper_id': '4e7cd4e923777d6caaa76bb2b93f6121bcd5b6a3',\n","  'section': 'Abstract',\n","  'text': 'In recent decades, many new flavi-like viruses have been discovered predominantly in different invertebrates and, as was recently shown, some of them may cause disease in humans. The Jingmen tick virus (JMTV) group holds a special place among flavi-like viruses because, in contrast to the \"classic\" flaviviruses and other flavi-like viruses, they have a segmented ssRNA(+) genome. Two segments of the JMTV genome have homology with regions of the flavivirus genome encoding polymerase and helicase-protease proteins. JMTVs have several open reading frames (ORF) in segments encoding glycoprotein(s) and capsid protein and these ORF are specific only to them. JMTVs greatly differ in virion size. We isolated three strains of Alongshan virus (ALSV), which is a representative of the JMTV group, from adult Ixodes persulcatus ticks collected in two geographically-separated Russian regions in the tick cell line IRE/CTVM19. One of the strains persisted in the IRE/CTVM19 cells without cytopathic effect for three years. Most virions purified from tick cells were spherical with a diameter of approximately 40.5 nm. In addition, we found smaller particles of approximately 13.1 nm in diameter. We obtained full genome sequences of all four segments of two of the isolated ALSV strains, and partial sequences of one segment from the third strain. Phylogenetic analysis on genome segment 2 of the JMTV group clustered our novel strains with other ALSV strains. We found evidence for the existence of a novel upstream ORF in the glycoprotein-coding segment of ALSV and other members of the JMTV group. Significance Statement We isolated three strains of Alongshan virus (ALSV) from adult Ixodes persulcatus ticks from two geographically separate areas of Russia in the Ixodes ricinus tick cell line IRE/CTVM19. One of the strains persisted in the IRE/CTVM19 cells without cytopathic effect for three years. Our study confirmed the value of tick cell lines in virus isolation and maintenance of persistent infection. The majority of virions of the ALSV strain Miass527 were enveloped spherical particles with a diameter of 40.5±3.7 nm. We found evidence for the existence of a novel upstream ORF in the glycoprotein-coding segment of ALSV and other members of the Jingmen tick virus group. '},\n"," {'paper_id': '4eb8b7fd0032816e4a29d65b06939266d6446624',\n","  'section': 'Abstract',\n","  'text': 'Odor memories are exceptionally robust and essential for animal survival. The olfactory (piriform) cortex has long been hypothesized to encode odor memories, yet the cellular substrates for olfactory learning and memory remain unknown. Here, using intersectional, cFos-based genetic manipulations (\"Fos-tagging\"), we show that olfactory fear conditioning activates sparse and distributed ensembles of neurons in mouse piriform cortex. We demonstrate that chemogenetic silencing of these Fostagged piriform ensembles selectively interferes with odor fear memory retrieval, but does not compromise basic odor detection and discrimination. Furthermore, chemogenetic reactivation of piriform neurons that were Fos-tagged during olfactory fear conditioning causes a decrease in exploratory behavior, mimicking odor-evoked fear memory recall. Together, our experiments identify odor-specific ensembles of piriform neurons as necessary and sufficient for odor fear memory recall. author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '4ed084719f40fac25989dbc99d08c2c609b33289',\n","  'section': 'Abstract',\n","  'text': 'G-quadruplexes are non-canonical nucleic acid structures that control transcription, replication, and recombination in organisms. G-quadruplexes are present in eukaryotes, prokaryotes, and viruses. In the latter, mounting evidence indicates their key biological activity. Since data on viruses are scattered, we here present a comprehensive analysis of putative G-quadruplexes in the genome of all known viruses that can infect humans. We show that the presence, distribution, and location of G-quadruplexes are features characteristic of each virus class and family. Our statistical analysis proves that their presence within the viral genome is orderly arranged, as indicated by the possibility to correctly assign up to two-thirds of viruses to their exact class based on the G-quadruplex classification. For each virus we provide: i) the list of all G-quadruplexes formed by GG-, GGG-and GGGG-islands present in the genome (positive and negative strands), ii) their position in the viral genome along with the known function of that region, iii) the degree of conservation among strains of each G-quadruplex in its genome context, iv) the statistical significance of G-quadruplex formation. This information is accessible from a database '},\n"," {'paper_id': '4f030bbfb35b1e3d40ef4b0e1f12ea8d82bc7900',\n","  'section': 'Abstract',\n","  'text': 'All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. Background: South Korea was among the first countries to report a case of the novel coronavirus (COVID-19) outside of China. As of 22 March, 2020, South Korea reported 8897 confirmed cases of and 104 deaths from COVID-19. We collected the number of laboratory-confirmed cases and deaths in South Korea from the World Health Organization (as of 21 March, 2020) and case distribution and fatality rates by age from the Korean Center for Disease Control and Prevention (as of 22 March, 2020). We estimated population-level mortality rates by fitting a negative binomial regression model with the number of deaths as the outcome and population by age as an offset. We then calculated the age-standardized death rate (ASDR) based on the current COVID-19 figures and for alternative scenarios of increased prevalence. The COVID-19 population-level mortality rate (per 100,000 person-years) increased with age: from 0.1 deaths among 30-39 year olds to 9.5 deaths among ≥80 year olds. The ASDR (per 100,000 person-years) was 0.8 deaths. The ASDR would increase to 52.0 deaths at a 1% prevalence (becoming the third leading cause of death) and 155.9 deaths at 3% prevalence (becoming the leading cause of death). Interpretation: Currently, the population-level mortality burden of COVID-19 in South Korea, as measured by the ASDR, was relatively low compared to other causes of death partly due to the low prevalence of COVID-19. If the prevalence increases from another outbreak, the mortality burden could increase substantially and surpass other leading causes. '},\n"," {'paper_id': '4f392ef1aa2234dda52ab8e6540f275322a20697',\n","  'section': 'Abstract',\n","  'text': 'Enveloped viruses gain entry into host cells by fusing with cellular membranes, a step required for virus replication. Coronaviruses, including the severe acute respiratory syndrome coronavirus (SARS-CoV), Middle East respiratory syndrome coronavirus (MERS-CoV), and infectious bronchitis virus (IBV), fuse at the plasma membrane or use receptor-mediated endocytosis and fuse with endosomes depending on the cell or tissue type. The virus Spike (S) protein mediates fusion with the host cell membrane. We have shown previously that an Abl kinase inhibitor, imatinib, significantly reduces SARS-CoV and MERS-CoV viral titers and prevents endosomal entry by HIV SARS S and MERS S pseudotyped virions. SARS-CoV and MERS-CoV are classified as BSL-3 viruses, which can make experimentation into the cellular mechanisms involved in infection more challenging. Here, we use IBV, a BSL-2 virus, as a model for studying the role of Abl kinase activity during coronavirus infection. We found that imatinib and two specific Abl kinase inhibitors, GNF2 and GNF5, reduce IBV titers by blocking the first round of virus infection. Additionally, all three drugs prevented IBV S-induced syncytia formation prior to the hemifusion step. Our results indicate that membrane fusion (both virus-cell and cell-cell) is blocked in the presence of Abl kinase inhibitors. Studying the effects of Abl kinase inhibitors on IBV will be useful in identifying host cell pathways required for coronavirus infection. This will provide insight into possible therapeutic targets to treat infections by current as well as newly emerging coronaviruses. '},\n"," {'paper_id': '4f5901d41427c10d4dafdc6d3cef2dd20bba871a',\n","  'section': 'Abstract',\n","  'text': 'For understanding the evolution of social behavior in microbes, mathematical theory can aid empirical research but is often only used as a qualitative heuristic. How to properly formulate social evolution theory has also been contentious. Here we evaluate kin and multilevel selection theory as tools for analyzing microbial data. We reanalyze published datasets that share a common exper-5 imental design and evaluate these theories in terms of data visualization, statistical performance, biological interpretation, and quantitative comparison across systems. We find that the canonical formulations of both kin and multilevel selection are almost always poor analytical tools because they use statistical regressions that are poorly specified for the strong selection and nonadditive fitness effects common in microbial systems. Analyzing both individual and group fitness outcomes 10 helps clarify the biology of selection. We also identify analytical practices in empirical research that suggest how theory might better handle the challenges of microbial data. A quantitative, data-driven approach thus shows how kin and multilevel selection theory both have substantial room for improvement as tools for understanding social evolution in all branches of life. '},\n"," {'paper_id': '4f5f812dec32f2da50e3b8bb7a63626f45933f6a',\n","  'section': 'Abstract',\n","  'text': 'With our successful experience of treating COVID-19 patients, we 22 retrospectively found that routine clinical features could reliably predict severe 23 pneumonia development, thus provide quick and affordable references for 24 physicians to save the otherwise fatal patients with the limited medical 25 resource. Abstract 28 Approximately 15-20% of COVID-19 patients will develop severe 29 pneumonia, about 10 % of which will die if not properly managed. Earlier 30 discrimination of the potential severe patients basing on routine clinical and 31 laboratory changes and commencement of prophylactical management will not 32 only save their lives but also mitigate the otherwise overwhelmed health care 33 burden. In this retrospective investigation, the clinical and laboratory features 34 were collected from 125 COVID-19 patients, who were classified into mild (93 35 cases) or severe (32 cases) groups according to their clinical outcomes after 3 36 to 7-days post-admission. The subsequent analysis with single-factor and 37 multivariate logistic regression methods indicated that 17 factors on admission 38 differed significantly between mild and severe groups, but that only comorbid 39 with underlying diseases, increased respiratory rate (>24/min), elevated C-40 reactive protein (CRP >10mg/liter), and lactate dehydrogenase 41 (LDH >250U/liter), were independently associated with the later disease 42 development. Finally, we evaluated their prognostic values with the receiver 43 operating characteristic curve (ROC) analysis and found that the above four 44 factors could not confidently predict the occurrence of severe pneumonia 45 individually, but that a combination of fast respiratory rate and elevated LDH 46 significantly increased the predictive confidence (AUC= 0.944, sensitivity= 47 329 Coronavirus-Infected Pneumonia '},\n"," {'paper_id': '4f8d24c531d2c334969e09e4b5aed66dcc925c4b',\n","  'section': 'Abstract',\n","  'text': 'Background: Radiologic characteristics of 2019 novel coronavirus (2019-nCoV) infected pneumonia (NCIP) which had not been fully understood are especially important for diagnosing and predicting prognosis. Purpose: To summarize the clinical and radiologic characteristics of NCIP cases and analyze the association of radiologic findings with mortality cases. We retrospective studied 27 consecutive patients who were confirmed NCIP, the clinical characteristics and CT image findings were collected, and the association of radiologic findings with mortality of patients was evaluated. Results: 27 patients included 12 men and 15 women, with median age of 60 years . 17 patients discharged in recovered condition and 10 patients died in hospital. The median age of mortality group was higher compared to survival group (68 (IQR 63-73) vs 55 (IQR 35-60), P = 0.003). The comorbidity rate in mortality group was significantly higher than in survival group (80% vs 29%, P = 0.018). The predominant CT characteristics consisted of ground glass opacity (67%), bilateral sides involved (86%), both peripheral and central distribution (74%), and lower zone involvement (96%). The median CT score of mortality group was higher compared to survival group (30 (IQR 7-13) vs 12 (IQR 11-43), P = 0.021), with more frequency of consolidation (40% vs 6%, P = 0.047) and air bronchogram (60% vs 12%, P = 0.025). An optimal cutoff value of a CT score of 24.5 had a sensitivity of 85.6% and a specificity of 84.5% for the prediction of mortality. Conclusions: 2019-nCoV was more likely to infect elderly people with chronic comorbidities. CT findings of NCIP were featured by predominant ground glass opacities mixed with consolidations, mainly peripheral or combined peripheral and central distributions, bilateral and lower lung zones being mostly involved. A simple CT scoring method was capable to predict mortality. '},\n"," {'paper_id': '4ffcd1d8e0f56c5a238bbd4040ec23410ef71a1b',\n","  'section': 'Abstract',\n","  'text': 'We report the first 7,755 patients with confirmed COVID-19 in Korea as of March 13, 2020. A total of 66 deaths were identified, resulting case fatality proportion of 0.9%. Older people, and those with coexisting medical conditions were at risk for fatal outcomes. The highest number of cases were from Daegu, followed by Gyeongbuk, with elevated age-stratified case fatality. This summary may help to understand the disease dynamics in the early phase of COVID-19 outbreak, therefore, to guide future public health measures. '},\n"," {'paper_id': '500c5ea0961b573ba5355d364ca94c00a0a26e4a',\n","  'section': 'Abstract',\n","  'text': '11 12 Keywords: 13 dengue virus; zika virus; yellow fever virus; flavivirus; NS1; caveolin-1; chaperone caveolin 14 complex; viral protein trafficking; unconventional secretion; mosquito cells 15 16 17 ABSTRACT 18 Dengue virus (DENV) is a mosquito-borne virus of the family Flaviviridae. The RNA viral genome 19 encodes for a polyprotein that is co-translationally processed into three structural proteins and . CC-BY 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/370932 doi: bioRxiv preprint 2 20 seven non-structural proteins. The non-structural protein 1 (NS1) is a multifunctional viral protein 21 actively secreted in vertebrate and mosquito cells during DENV infection. In mosquito cells, NS1 22 is secreted in a caveolin-1 (CAV-1) dependent manner by an unconventional pathway. The 23 caveolin chaperone complex (CCC) is a cytoplasmic complex formed by caveolin-1 and the 24 chaperones FKBP52, Cy40 and CyA which is responsible for cholesterol traffic inside the cell. 25 In this work, we demonstrate that in infected mosquito cells, DENV NS1 is secreted by an early 26 and unconventional route that bypasses the Golgi apparatus in close association with the CCC. 27 Treatment of mosquito cells with classic secretion inhibitors such as brefeldin A, golgicide A and 28 Fli-06 showed no effect on NS1 secretion, but significant reductions in recombinant luciferase 29 secretion and virion release. Silencing the expression of CAV1, FKBP52 with siRNAs or the 30 inhibition of CyA by cyclosporine A resulted in significant decrease in NS1 secretion without 31 affecting virion release. Using co-localization, co-inmunoprecipitation and proximity ligation 32 assays, NS1 was found to co-localize and interact with all the protein components of the CCC 33 in mosquito infected cells. In addition, CAV-1 and FKBP52 expression was found augmented in 34 DENV infected cells. Finally, the treatment of ZIKV infected mosquito cells with brefeldin A and 35 golgicide A showed no effect on NS1 secretion, while affecting virion release. ZIKV NS1 was 36 also found to co-localize with CAV-1 in infected mosquito cells. These results suggest that in 37 mosquito cells, ZIKV NS1 follows the same secretory pathway observed for DENV NS1. The 38 association of NS1 with the cholesterol transporter CCC agrees with the lipoprotein nature of 39 secreted hexameric NS1. 40 41 AUTHOR SUMMARY . CC-BY 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/370932 doi: bioRxiv preprint 3 42 Dengue protein NS1 is secreted in infected mosquito and vertebrate cells. In humans, secreted 43 NS1 have been associated with pathogenesis. In mosquito cells, NS1 follows an unconventional 44 secretion pathway that is dependent on Caveolin-1. This work shows that in mosquito cells, NS1 45 secretion is associated to the chaperone caveolin complex , a complex formed by caveolin-1 and 46 several chaperones, in charge of cholesterol transport within the cells. Reduction of the 47 expression or the activity of chaperone caveolin complex in mosquito infected cells, diminished 48 the secretion of NS1 without affecting virion release. Direct interaction between NS1 and the 49 chaperone caveolin complex proteins was demonstrated by several assays. Moreover, 50 increased expression of the caveolin-1 and co-chaperone FKBP52 during dengue infection was 51 found, presumably in response to the higher requirements of these proteins during dengue virus 52 infection. Results obtained with ZIKV infected mosquito cells suggest that also ZIKV NS1 is 53 released following an unconventional secretory route in association with the chaperone caveolin 54 complex. The functions of secreted NS1 within mosquito are unclear. However, giving the 55 importance of the soluble NS1 in the vertebrate host, manipulation of the NS1 secretory route 56 may prove a valuable strategy for dengue mosquito control and patient treatment. 57 58 59 4 64 infection and its public health burden are hard to estimate, but at least 100 countries are currently 65 endemic for dengue [2]. DENV has a single stranded RNA genome of positive polarity. The 66 genome of DENV is translated into a single polyprotein and encodes for three structural (C, E, 67 prM/M) and seven non-structural (NS) (NS1, NS2A, NS2B, NS3, NS4A, NS4B, NS5) proteins. 68 The polyprotein is then cleaved by host and viral proteases to release individual viral proteins 69 [3]. The viral genome replication process within the host cell is mainly driven by the NS proteins. 70 The NS1 protein act as an scaffolding protein that anchors the replication complex to the ER 71 membrane and interacts physically with NS4B [4]. The NS1 protein is a 352-amino-acid 72 polypeptide with a molecular weight of 46-55 kDa, depending on its glycosylation status. The 73 NS1 protein exists in multiple oligomeric forms and is found in different cellular locations: a cell 74 membrane-bound form in association with virus-induced intracellular vesicular compartments, 75 on the cell surface and as a soluble secreted hexameric lipoparticle [4]. The NS1 monomeric 76 form rapidly dimerizes in the endoplasmic reticulum (ER), then three dimeric forms of NS1 77 arrange to form an hexamer [5]. The hexameric form of NS1 shows an open barrel form filled 78 with lipids and cholesterol which resemble the lipid composition of the HDL particle [6]. 79 Recent studies have shown that the DENV NS1 protein was not only secreted from vertebrate 80 cells, but that was also efficiently secreted from mosquito cells lines [7,8]. The secretion of NS1 81 in vertebrate cells follows the classical Golgi-pathway [9]. However, NS1 secretion in infected 82 mosquito cells is associated to a caveolin-1 (CAV-1) dependent pathway and was found to be 83 Brefeldin A (BFA)-insensitive, suggesting a traffic route that bypasses the Golgi-complex [10]. 84 Caveolae are made up of interlocking heteropolymers of a family of small proteins (caveolins: 1-85 3) and a second family of accessory structural proteins (flotillins and three family of cavins). The 86 caveolar architecture is connected with unstructured cavin filaments by coiled-coil domains into . CC-BY 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/370932 doi: bioRxiv preprint 5 87 a polygonal net-like complex. This complex is believed to provide scaffolding for compartmented 88 cellular processes and participates in multiple cellular functions, including endocytosis, 89 transcytosis, membrane homeostasis, inflammation, and signal transduction [11]. Caveolin 1 90 (Cav-1), a 21-24-kDa scaffolding protein, is not only a key structural component of the caveolae 91 organelle but also plays an important in the transport of free-cholesterol inside the cell [12,13]. 92 The chaperone caveolin complex (CCC) is a cytosolic complex reported to transport cholesterol 93 synthetized de novo from the ER to cell membranes or other compartments within the cell. CCC 94 has been described as a complex of CAV-1, Cyclophilin A (CyA), FK506-binding protein 4 or 95 heat shock protein 56 (FKBP52), and Cyclophilin 40 or D (Cy40) [12,14]. Cyclophilin A (CyA), 96 an 18 kDa peptidylprolyl cis-trans isomerase, is a ubiquitous and multifunctional protein. In 97 addition to its role as a host-cell receptor for cyclosporine A, CyA has diverse functions in 98 inflammatory conditions and diseases [15,16]. The 52-kDa FK506-binding protein (FKBP52) an 99 immunophilin belonging to the FKBP family, is a known co-chaperone of the heat shock protein 100 90 (HSP90) and thus may play a role in the intracellular trafficking of hetero-oligomeric forms of 101 the steroid hormone receptors [17,18]. Cyclophilin 40 (Cy40), a member of a family of highly 102 homologous peptidylprolyl cis-trans isomerases (PPIases), is known to play a role in 103 mitochondrial permeability transition (MPT), being an integral constituent of the MPT pore [19]. 104 Given the CAV-1 dependent secretion of NS1 protein in mosquito cells and the lipoprotein nature 105 of the released hexameric form of NS1, it was found plausible to study the association of NS1 106 traffic to the novo cholesterol transport in DENV infected mosquito cells. In this work, data is 107 presented indicating that in infected mosquito cells, DENV NS1 enters the unconventional 108 secretory pathway very early after maturation in the ER and usurps the cholesterol transport 109 between the ER and the plasma membrane, mediated by the CCC, to reach the extracellular . CC-BY 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/370932 doi: bioRxiv preprint 6 110 space. In addition, data is presented suggesting that a similar pathway is used for the secretion 111 of Zika Virus NS1 protein in infected mosquito cells. 112 113 RESULTS 114 NS1 secretion is not affected by drugs that disrupt early steps of the classical secretion 115 pathway 116 Golgicide A (GCA) is a powerful inhibitor of the COPI vehicle transport from ER to Golgi [20]. 117 Thus, the cytotoxicity of GCA in the mosquito cell lines (C6/36, Aag2) and the vertebrate cell line 118 BHK-21, used for comparisons, was measured using the reduction of tetrazolium salts to 119 examine proliferation in cells treated with serial dilutions of GCA. No significant cytotoxicity was 120 observed under 30 \\uf06dM of GCA in any of the three cell types (Fig 1A) . Fli-06 is a novel drug which 121 inhibits the diffusion of ER synthetized proteins to the ER-exit sites [21,22]. Fli-06 cytotoxicity 122 was determined in C6/36 and Aag2 cells also using also tetrazolium salt reduction. No significant 123 cytotoxicity was observed under 100 \\uf06dM of Fli-06 in the mosquito cells lines (Fig 1B) . Thus, 124 concentration of 27 \\uf06dM and 100 \\uf06dM were used for GCA and Fli-06, respectively, since these 125 concentrations proved non-toxic, yet effective in causing Golgi disruption (Fig 1C) . Figures 1D   126 and 1E show that the GCA-treatment of DENV infected C6/36 and Aag2 cells did not cause any 127 change in NS1 secretion. In contrast, NS1 secretion was reduced over 70% in infected BHK-21 128 cell treated either with GCA or BFA ( Fig 1F) . As expected, virion release was reduced in 129 mosquito and vertebrate cells after GCA or BFA treatment (Fig 1G, 1H and 1I ). To further explore 130 the traffic route of NS1 in DENV infected mosquito cells, cells were also treated with Fli-06. 131 Treatment of infected C6/36 or Aag2 cells with non-toxic concentrations of Fli-06 shows no effect . CC-BY 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/370932 doi: bioRxiv preprint 7 132 in NS1 secretion, while significantly affecting virion release (Fig 1G, 1H) . Experiments with Fli-133 06 were not carried out in BHK-21 due to high drug toxicity. The results with GCA and Fli-06 134 indicate NS1 is not secreted by a classical secretory pathway and that newly synthetized DENV 135 NS1 do not reach the ER-exit sites and leaves the ER compartment very early after synthesis. 136 Since the release of mature virions, which are multimolecular complexes, may not be equivalent 137 to the release of a single protein, an additional control for protein trafficking through the classical 138 secretory pathway was included. To this aim, a vector, pAc5-mCherry-GLuc-neo, constitutively 139 expressing Gaussia Luciferase in C6/36 cell was constructed (S1 Fig). To evaluate the effect of 140 GCA treatment on GLuc secretion, cells were treated with GCA for 24 h and the activity of GLuc 141 present in the cell supernatants analyzed using a luminescence assay. A decrease in luciferase 142 activity, in relation to the control (DMSO), was detected upon treatment of C6/36 with GCA, and 143 in BHK-21 cells transfected with ptk-GLuc vector, run in parallel as control (Fig.1J). These results 144 indicated that the luciferase reporter is secreted in both mosquito and vertebrate cells following 145 a classical secretory route and reinforce the results indicating that in mosquito cells DENV NS1 146 is secreted by an unconventional secretory route that bypasses the Golgi (Fig 1K). 147 148 NS1 secretion is dependent on the CCC 149 The CCC is formed by the association of CAV-1 with the chaperones CyA, FKBP52, Cy40 150 [12,13]. The CCC is responsible for cholesterol transport inside the cell. Previous data by us [10] 151 indicated that NS1 secretion was dependent on CAV-1. Thus, to fully evaluate the participation 152 of the CCC in DENV NS1 secretion from mosquito cells, CyA was pharmacologically inhibited 153 by treatment with Cyclosporin A (CsA) and the expression of the chaperones FKBP52 and Cy40 . CC-BY 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/370932 doi: bioRxiv preprint 8 154 knocked down using siRNAs. Previously, the non-cytotoxic concentration for CsA in mosquito 155 and vertebrate cells were determined (Fig 2A) . DENV2 or DENV4-infected C6/36 or Aag2 cells 156 were treated with 9 \\uf06dM CsA for 24 hours; NS1 secretion was reduced in about 30% and 40% in 157 C6/36 and Aag2 cells, respectively (Fig 2B, 2E) . However, virion release in the treated mosquito 158 cells was not significantly affected (Fig 2C and 2F ). In contrast, the inhibition of CyA in BHK-21 159 cells did not have any effect in either NS1 or virion secretion (Figs 2H and 2I). To further 160 corroborate the participation of CyA in cholesterol and lipid traffic, the lipid droplet count in CsA 161 treated versus untreated cells was determined. In all cases, C6/36, Aag2 and BHK-21 cells 162 treated with CsA showed a diminished amount of lipid droplets per cell as compared with un-163 treated cells (Fig 2D, 2G and 2J). 164 The expression of FKBP52 in C6/36 cells was knock down by siRNA transfection. Optimal gene 165 knock down was achieved after 48h post transfection (Fig 3A) . The effect of FKPB52 reduced 166 expression in cholesterol and lipid metabolism was evaluated measuring the lipid droplet 167 counting per cell and a reduction of about 50% in the number of lipid droplets was observed after 168 the treatment (Fig 3B) . In cells treated with siRNA FKBP52, a significant reduction in NS1 of 169 about 20% secretion was observed ( Fig 3C) . Viral release was not affected by FKBP52-knock 170 down (Fig 3D) . Given that FKBP52 is a co-chaperone of HSP90 [17,23], we wanted to discard if 171 the observed inhibitory effect on NS1 secretion might be related to a reduction in HSP90 activity. 172 Thus, the effect of the HSP90 inhibitor geldanamycin (GA) in NS1 release from DENV infected 173 C6/36 cells was tested. As shown in Figure 3E , NS1 secretion was not affected after GA 174 treatment. The expected reduction in virus yield observed in GA treated cell (Fig 3F) show that 175 effective GA doses were used. These results indicate that the effect in NS1 secretion observed 176 after FKBP52 silencing is not related with HSP90 activity and was rather associated with the '},\n"," {'paper_id': '50688ca74af7fb4395c03e3cf7ec3bddc84e6165',\n","  'section': 'Abstract',\n","  'text': 'Bio-fluids contain various circulating cell-free RNA transcripts (ccfRNAs). The composition of 20 these ccfRNAs varies between bio-fluids and constitute tantalizing biomarker candidates for 21 several pathologies. ccfRNAs have also been demonstrated as mediators of cellular 22 communication, yet little is known about their function in physiological and developmental 23 settings and most works are limited to in-vitro studies. Here, we have developed iTAG-RNA, a 24 novel method for the unbiased tagging of RNA transcripts in mice in-vivo. We used this method 25 to isolate hepatocytes and kidney proximal epithelial cells-specific transcriptional response to a 26 dietary challenge without interfering with the tissue architecture, and to identify multiple 27 hepatocyte-secreted ccfRNAs in plasma. We also identified transfer of these hepatic derived 28 ccfRNAs to adipose tissue, where they likely serve as a buffering mechanism to maintain 29 cholesterol and lipid homeostasis. Our findings directly demonstrate in-vivo transfer of RNAs 30 between tissues and highlight its implications for endocrine signaling and homeostasis. Little is known about the biological function of circulating cell-free RNAs (ccfRNA). Found to be 34 associated with exosomes, lipoproteins, ribonucleoproteins and more, these transcripts can be 35 isolated and sequenced from multiple bio-fluids such as plasma, lymph, cerebral fluids, breast milk 36 and more [1, 2]. ccfRNAs are directly implicated in the development of several pathologies 37 including cancer and obesity [3][4][5] and are intensively studied as disease biomarkers [6, 7]. Despite 38 this, the role they play in physiological and developmental settings and in mediating cell-to-cell 39 communication remains largely unknown. In-vitro, a growing number of works demonstrate the 40 relevance of RNA based cellular communication [8][9][10][11], however in-vivo evidence is still limited. This discrepancy is partly due to the difficulties posed to tracking ccfRNAs from transcriptional 42 source to potential sites of action in-vivo. Indeed the tools available to study ccfRNAs in 43 physiological settings are limited and very few studies attempt to tackle this problem directly. One work found evidence to suggest that the majority of circulating miRNAs originate in adipose 45 tissue and that some of the adipose derived miRNAs may play a role in the regulation of liver 46 Fgf21 levels [12]. However, this work focuses on miRNA and does not directly demonstrate 47 transfer of RNAs between tissues nor directly identify adipose secreted RNAs. Transfer of miRNAs was also demonstrated between epithelial cells of the caput epididymis to 49 maturing spermatozoa, leading to a shift in sperm RNA content during its maturation [13]. This 50 study made use of 4-thiouracil-tagging (TU-tagging) [14] combined with SLAM-Seq [15] to 51 demonstrate loading of miRNAs transcribed in caput epididymis into maturing spermatozoa. TU-52 tagging entails cell-type specific expression of uracil phosphoribosyltransferase (UPRT) and 53 administration of 4-thiouracil, with the assumption that only cells expressing UPRT would 54 incorporate 4-thiouracil into transcribing RNA. Thio-RNA can then be purified and used for 55 downstream gene expression analyses, or alternatively combined with SLAM-Seq to identify 56 labeled transcripts. TU-tagging has proven useful in several additional systems [14, 16, 17], 57 however, given endogenous [18] and alternative [19] pathways for uracil incorporation, the 58 labeling specificity in this method remains unclear. In addition, as is demonstrated in Herzog et. 59 al [15] and by Sharma et. al [13], labeling with TU-tagging of PolI and PolIII transcripts is 60 inefficient, rendering tRNAs and ribosomal transcripts unlabeled. Indeed there are only a limited number of techniques enabling in-vivo targeted labeling of RNAs. 62 In addition to TU-tagging, 5-ethynylcytosine-tagging (EC-tagging) [20] is a new method, which 63 utilizes cell-type specific co-expression of cytosine deaminase (CD) with UPRT to achieve RNA 64 labeling with 5-ethynyluridine (5EU) following administration of 5-ethynylcytosine. Both TU and 65 EC tagging use cre-recombination to express the relevant enzymes in a tissue specific manner and 66 stochastic expression from the cre-promoter may lead to unwanted expression of the enzymes in 67 different tissues [21]. Finally, one recently developed method called Mime-seq allows for cell type 68 specific labeling of microRNA [22]. In this method, tissue specific expression of a plant derived 69 methyltransferase mediates a 3′-terminal 2′-O-methylation of microRNAs that, when combined 70 with a methylation dependent library construction, allows for sequencing of tissue specific 71 microRNAs. Mime-seq allows labeling of miRNAs alone leaving other RNA biotypes unlabeled. 72 Given the need for a technique that allows for a Cre-independent and unbiased labeling of total 73 RNA transcription in-vivo, we developed iTAG-RNA [For In-vivo Targeted Tagging of RNA]. 74 This method incorporates mouse genetics with a novel uridine analog and an established RNA 75 labeling chemistry to allow tagging of total RNA in target cells in-vivo. Using iTAG-RNA we are 76 able to identify transcriptional re-programming of hepatocytes in-vivo following an acute high fat 77 diet stress and to enrich for and identify hepatocyte derived plasma ccfRNAs. Moreover, we are 78 able to identify RNA-based liver-to-adipose RNA transfer. These liver derived ccfRNAs include 79 variable coding and non-coding RNAs such as miRNAs and tRNAs. Among the miRNAs 80 transferred from liver to adipose tissue we find mir-33, mir-10b and mir130a, which target major 81 regulators of cholesterol and lipid efflux and bio-synthesis such as Srebf1 [23], Abca1 [23, 24], 82 Ppara [25] and Pparg [26] respectively. 83 Our study demonstrates for the first time an unbiased technique that allows labelling, tracking and 84 quantification of variable types of ccfRNAs from their transcriptional source to downstream 85 tissues, in which they can potentially act to regulate expression of target genes. We demonstrate 86 RNA-based liver-to-adipose transfer of a myriad of RNA transcripts and their response to an 87 environmental challenge. The continued identification and characterization of RNA based 88 signaling in-vivo is imperative for the understanding of developmental, physiological and 89 pathological processes, and can aid in the future development of relevant disease biomarkers. 90 91 Results . CC-BY-NC-ND 4.0 International license is made available under a '},\n"," {'paper_id': '507f31c10039eee488b98373f0df204b06accb95',\n","  'section': 'Abstract',\n","  'text': 'I estimate plausible true positive (TP) rates for the number of COVID-19 tests per day, most relevant when the number of test is on the same order of magnitude as number of infected persons. I then modify a standard SEIR model to model current growth patterns and detection rates in South Korea and New York state. Although reducing transmission rates have the largest impact, increasing TP rates by ~10% in New York can have an impact equal to adding tens of thousands of new tests per day. Increasing both TP rates and tests per day together can have significant impacts and likely be more easily sustained than social distancing restrictions. '},\n"," {'paper_id': '50903a4e5afc3c8703ea7fc2c16bf42b8a8635fb',\n","  'section': 'Abstract',\n","  'text': 'Background: COVID-19 diagnosis is a critical problem, mainly due to the lack or delay in the test results. We aimed to obtain a model to predict SARS-CoV-2 infection in suspected patients reported to the Brazilian surveillance system. Methods: We analyzed suspected patients reported to the National Surveillance System that corresponded to the following case definition: patients with respiratory symptoms and fever, who traveled to regions with local or community transmission or who had close contact with a suspected or confirmed case. Based on variables routinely collected, we obtained a multiple model using logistic regression. The area under the receiver operating characteristic curve (AUC) and accuracy indicators were used for validation. Results: We described 1468 COVID-19 cases (confirmed by RT-PCR) and 4271 patients with other illnesses. With a data subset, including 80% of patients from Sao Paulo (SP) and Rio Janeiro (RJ), we obtained a function which reached an AUC of 95.54% (95% CI: 94.41% -96.67%) for the diagnosis of COVID-19 and accuracy of 90.1% (sensitivity 87.62% and specificity 92.02%). In a validation dataset including the other 20% of patients from SP and RJ, this model exhibited an AUC of 95.01% (92.51% -97.5%) and accuracy of 89.47% (sensitivity 87.32% and specificity 91.36%). Conclusion: We obtained a model suitable for the clinical diagnosis of COVID-19 based on routinely collected surveillance data. Applications of this tool include early identification for specific treatment and isolation, rational use of laboratory tests, and input for modeling epidemiological trends. '},\n"," {'paper_id': '50a217a2dacfe1364383ec8c681f64f2fd76dbe7',\n","  'section': 'Abstract',\n","  'text': \"Governments around the world must rapidly mobilize and make difficult policy decisions to mitigate the COVID-19 pandemic. Because deaths have been concentrated at older ages, we highlight the important role of demography, particularly how the age structure of a population may help explain differences in fatality rates across countries and how transmission unfolds. We examine the role of age structure in deaths thus far in Italy and South Korea and illustrate how the pandemic could unfold in populations with similar population sizes but different age structures, showing a dramatically higher burden of mortality in countries with older versus younger populations. This powerful interaction of demography and current age-specific mortality for COVID-19 suggests that social distancing and other policies to slow transmission should consider both the age composition of local and national contexts as well as the social connectedness of older and younger generations. We also call for countries to provide case and fatality data disaggregated by age and sex to improve real-time targeted nowcasting. Governments are rapidly mobilizing to minimize transmission of COVID-19 through social distancing and travel restrictions to reduce fatalities and the outstripping of healthcare capacity. It is becoming clear that the pandemic's progression and impact may be strongly related to the demographic composition of the population, specifically population age structure. Demographic science can provide new insights into how the pandemic may unfold and the intensity and type of measures needed to slow it down. Currently, COVID-19 mortality risk is highly concentrated at older ages, particularly those aged 80+. In China, case-fatality rate (CFR) estimates range from 0.4% (40-49 years), jumping to 14.8% (80+ years).(1) This is consistent with the data from Italy as of March 16, where the reported CFR is 11.8% for those 70-79, 18.8% for 80-89, and 21.6% for those >90. Thus far in Italy, only 1.5% of deaths have occurred in those under aged 60.(2) As testing becomes more widespread, these case fatality rates may come down due to an increase in the true denominator. Nonetheless, in South Korea where testing was broader and the health system not overtaxed, the current CFR for those 80+ is still an alarming 9.26%. (7) The Importance of Age Structure Population age structure may explain the remarkable variation in fatalities across countries and why countries such as Italy are especially vulnerable. The deluge of critical and fatal COVID-19 cases in Italy was unexpected given the health and wealth of the affected region. Italy is one of the oldest populations in the world with 23.3% its population over age 65, compared to 12% in China (3). Italy is also a country characterized by extensive intergenerational contacts which are supported by a high degree of residential proximity between adult children and their parents (4). Even when inter-generational families do not live together, daily contacts among non-co-resident parent-child pairs are frequent. Many Italians also often prefer to live close to their extended family and commute to work daily. \"},\n"," {'paper_id': '50aa3305f87a409bd1601bdec5d3cd404fe825a3',\n","  'section': 'Abstract',\n","  'text': 'BACKGROUND: Acute gastroenteritis (AGE) is a leading cause of morbidity and mortality in young children worldwide. Human Bocavirus (HBoV) is an emerging virus globally associated with diarrhea. The aim of this study was to demonstrate the prevalence of HBoV genotypes in children (≤5 years) from rural communities in South Africa (SA) suffering from AGE. A total of 141 fecal samples of children ≤5 years with acute gastroenteritis (AGE) were collected from rural Primary Health Care facilities in the Vhembe district of SA between June 2017 and July 2018. Clinical symptoms and demographic data were also recorded. A total of 102 (72%) were outpatients and 39 (28%) were hospitalized patients. Human Bocavirus (HBoV) genotypes were determined using Real-Time Multiplex PCR. DNA extracts of positive samples were confirmed by conventional PCR targeting the NS1 gene. Co-infection with other enteric viruses were determined in HBoV positive samples using Real-Time PCR. RESULTS: HBoV was detected in 8 (5.7%) children with AGE. Children were in the age group between 1-24 months. HBoV1 and HBoV3 genotypes were each detected in 3 (37.5%) stool samples and HBoV2 in 2 (25%) stool samples. Co-infection with other enteric viruses included Rotavirus (37.5%); Adenovirus (37.5%); Norovirus (25%) and Astrovirus (12.5%). CONCLUSION: HBoV infections could be seen as a potential emerging diarrheal pathogen in South Africa. Further studies are required to understand the role of HBoV infections in children and adults with acute gastroenteritis. '},\n"," {'paper_id': '50be51724c114a2dcfbc9618d94a0166822003de',\n","  'section': 'Abstract',\n","  'text': 'A novel coronavirus, SARS-CoV-2, was recently identified in patients with an acute respiratory syndrome, COVID-19. To compare its pathogenesis with that of previously emerging coronaviruses, we inoculated cynomolgus macaques with SARS-CoV-2 or MERS-CoV and compared with historical SARS-CoV infections. In SARS-CoV-2-infected macaques, virus was 5 excreted from nose and throat in absence of clinical signs, and detected in type I and II pneumocytes in foci of diffuse alveolar damage and mucous glands of the nasal cavity. In SARS-CoV-infection, lung lesions were typically more severe, while they were milder in MERS-CoV infection, where virus was detected mainly in type II pneumocytes. These data show that SARS-CoV-2 can cause a COVID-19-like disease, and suggest that the severity of SARS-CoV-2 infection 10 is intermediate between that of SARS-CoV and MERS-CoV. '},\n"," {'paper_id': '50ed5cabf8d74c2a700184f876ad15de2239adfa',\n","  'section': 'Abstract',\n","  'text': 'Severe fever with thrombocytopenia syndrome (SFTS) is an emerging infectious 24 disease localized to China, Japan, and Korea that is characterized by severe hemorrhage and a 25 high fatality rate. Currently, no specific vaccine or treatment has been approved for this 26 disease. To develop a therapeutic agent for SFTS, we isolated antibodies from a phage-27 displayed antibody library that was constructed from a patient who recovered from SFTS 28 virus (SFTSV) infection. One antibody, designated as Ab10, was reactive to the Gn envelope 29 glycoprotein of SFTSV and protected host cells and A129 mice from infection in both in 30 vitro and in vivo experiments. Notably, Ab10 protected 80% of mice, even when injected 5 31 days after inoculation with a lethal dose of SFTSV. Using cross-linker assisted mass 32 spectrometry and alanine scanning, we located the non-linear epitope of Ab10 on the Gn 33 glycoprotein domain II and an unstructured stem region, suggesting that Ab10 may inhibit a 34 conformational alteration that is critical for cell membrane fusion between the virus and host 35 cell. Ab10 reacted to recombinant Gn glycoprotein in Gangwon/Korea/2012, HB28, and SD4 36 strains. Additionally, based on its epitope, we predict that Ab10 binds the Gn glycoprotein in 37 247 of 272 reported SFTSV isolates previously reported. Together, these data suggest that 38 Ab10 has potential to be developed into a therapeutic agent that could protect against more 39 than 90% of reported SFTSV isolates. 40 41 '},\n"," {'paper_id': '510ecab01c66bc2a911abcf67451ec482ed18a05',\n","  'section': 'Abstract',\n","  'text': 'Background '},\n"," {'paper_id': '5129bb31949c66ebd544658c531004046d4102da',\n","  'section': 'Abstract',\n","  'text': 'Effective screening for patients is important to limit the epidemic. However, some defects make the currently applied diagnosis methods are still not very ideal for early warning of patients. We aimed to develop a diagnostic model that allows for the quick screening of highly suspected patients using easy-to-get variables. A total of 1,311 patients receiving severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) nucleicacid detection were included, whom with a positive result were classified into COVID-19 group. Multivariate logistic regression analyses were performed to construct the diagnostic model. Receiver operating characteristic (ROC) curve analysis were used for model validation. RESULTS After analysis, signs of pneumonia on CT, history of close contact, fever, neutrophil-to-lymphocyte ratio (NLR), Tmax and sex were included in the diagnostic model. Age and meaningful respiratory symptoms were enrolled into COVID-19 early warning score (COVID-19 EWS). The areas under the ROC curve (AUROC) indicated that both of the diagnostic model (training dataset 0.956 [95%CI 0.935-0.977, P < 0.001]; validation dataset 0.960 [95%CI 0.919-1.0, P < 0.001] ) and COVID-19 EWS (training dataset 0.956 [95%CI 0.934-0.978, P < 0.001] ; validate dataset 0.966 [95%CI 0.929-1, P < 0.001]) had good discrimination capacity. In addition, we also obtained the cut-off values of disease severity predictors, such as CT score, CD8 + T cell count, CD4 + T cell count, and so on. The new developed COVID-19 EWS was a considerable tool for early and relatively accurately warning of SARS-CoV-2 infected patients. '},\n"," {'paper_id': '514a0f8afdc838a2a8b384c79637a15217a4d921',\n","  'section': 'Abstract',\n","  'text': 'The ongoing historic outbreak of COVID-19 not only constitutes a global public health crisis, but also carries a 33 devastating social and economic impact. The disease is caused by a newly identified coronavirus, Severe Acute Respiratory Syndrome coronavirus 2 (SARS-CoV-2). There is an urgent need to identify antivirals to 35 curtail the COVID-19 pandemic. Herein, we report the remarkable sensitivity of SARS-CoV-2 to recombinant 36 human interferons α and β (IFNα/β). Treatment with IFN-α at a concentration of 50 international units (IU) per 37 milliliter drastically reduces viral titers by 3.4 log or over 4 log, respectively, in Vero cells. The EC50 of IFN-α 38 and IFN-β treatment is 1.35 IU/ml and 0.76 IU/ml, respectively, in Vero cells. These results suggest that SARS-39 CoV-2 is more sensitive than many other human pathogenic viruses, including SARS-CoV. Overall, our results 40 demonstrate the potent efficacy of human Type I IFN in suppressing SARS-CoV-2 infection, a finding which 41 could inform future treatment options for COVID-19. 42 43 '},\n"," {'paper_id': '515d0c8dd3d5e837775f19c7b2577f2d2e8e3b88',\n","  'section': 'Abstract',\n","  'text': 'These authors contributed equally + Correspondence regarding sample collection and medical interpretation should be set to M. H. (m_halberthal@rambam.health.gov.il), Y. G. (y_geffen@rambam.health.gov.il), or M. S-C (M_Szwarcwort@rambam.health.gov.il); correspondence regarding the experimental procedure and data analysis should be sent to R. K. (rkishony@technion.ac.il). The recent emergence of SARS-CoV-2 lead to a current pandemic of unprecedented levels. Though diagnostic tests are fundamental to the ability to detect and respond, many health systems are already experiencing shortages of reagents associated with this test. Here, testing a pooling approach for the standard RT-qPCR test, we find that a single positive sample can be detected even in pools of up to 32 samples, with an estimated false negative rate of 10%. Detection of positive samples diluted in even up to 64 samples may also be attainable, though may require additional amplification cycles. As it uses the standard protocols, reagents and equipment, this pooling method can be applied immediately in current clinical testing laboratories. We hope that such implementation of a pool test for COVID-19 would allow expanding current screening capacities thereby enabling the expansion of detection in the community, as well as in close integral groups, such as hospital departments, army units, or factory shifts. '},\n"," {'paper_id': '522da4df2c31909547ac81763ad28d318fdb88e6',\n","  'section': 'Abstract',\n","  'text': 'The outbreak of COVID-19, which is caused by SARS-CoV-2 virus, continues to spread 23 globally, but there is currently very little understanding of the epitopes on the virus. In 24 this study, we have determined the crystal structure of the receptor-binding domain 25 (RBD) of the SARS-CoV-2 spike (S) protein in complex with CR3022, a neutralizing 26 antibody previously isolated from a convalescent SARS patient. CR3022 targets a highly 27 conserved epitope that enables cross-reactive binding between SARS-CoV-2 and 28 SARS-CoV. Structural modeling further demonstrates that the binding site can only be 29 accessed when at least two RBDs on the trimeric S protein are in the \"up\" conformation. Overall, this study provides structural and molecular insight into the antigenicity of 31 SARS-CoV-2. 33 ONE SENTENCE SUMMARY Structural study of a cross-reactive SARS antibody reveals a conserved epitope on the 35 SARS-CoV-2 receptor-binding domain. '},\n"," {'paper_id': '5241581d168b90832f88103bd40c8db2d843a6ea',\n","  'section': 'Abstract',\n","  'text': 'The outbreaks of coronavirus disease 2019 caused by the novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) remain a huge threat to the public health worldwide. Clinical data is limited up to now regarding the risk factors in favor of severe conversion of non-severe case with COVID-19. Aims: This study analyzed a hospital staff data to figure out general clinical features of COVID-19 in terms of the association of cardiovascular manifestations (CVMs) with inhospital outcomes of COVID-19 cases. Methods: Retrospective, single-center case series of 41 consecutive hospitalized health staff with confirmed COVID-19 were collected at clinical, laboratory, radiological, treatment data and in-hospital adverse events were collected and analyzed. Final date of follow-up was March 3, 2020. A comparative study was applied between cases with CVMs and those without CVMs. Results: Of all, clinicians and clinical nurses accounted for 80.5%, while 87.8% of all had history of patient contact. The population was presented with a mean age of 39.1 ± 9.2 and less comorbidities than community population. The three most frequent symptoms of COVID-19 cases analyzed were fever (82.9%), myalgia or fatigue (80.5%) and cough (63.4%). While, the three most frequent initial symptoms were myalgia or fatigue (80.5%), fever (73.2%) and cough (41.5%). There were 95.1% cases featured as non-severe course of disease according to the official standard in China. Patients with CVMs and those without CVMs accounted for 58.5% and 41.5%, respectively. Compared with cases without CVMs, patients with CVMs were presented with lower baseline lymphocyte count (0.99 ± 0.43 and 1.55 ± 0.61, P<0.001), more who had at least once positive nucleic acid detection of throat swab during admission (50.0% and 11.8%, P=0.011), and more received oxygen support (79.2% and 23.5%, P<0.001). The rate of in-hospital adverse events was significantly higher in patients with CVMs group (75.0% and 23.5%, P=0.001). Multivariable logistic regression model indicated that, coexisting with CVMs in COVID-19 patients was not independently associated with in- '},\n"," {'paper_id': '52a8089242fbe484e537cd8ea5c0c9ec3bf32150',\n","  'section': 'Abstract',\n","  'text': \"One striking characteristic of certain herpesviruses is their ability to induce rapid and 10 widespread RNA decay in order to gain access to host resources. This phenotype is induced by 11 viral endoribonucleases, including SOX in KSHV, muSOX in MHV68, BGLF5 in EBV and vhs in 12 HSV-1. Here, we performed comparative RNA-seq upon expression of these herpesviral 13 endonucleases in order to characterize their effect on the host transcriptome. Consistent with 14 previous reports, we found that approximately two thirds of transcripts are downregulated in 15 cells expressing any of these viral endonucleases. Among transcripts spared from degradation, 16 we uncovered a cluster of transcripts that systematically escape degradation from all tested 17 endonucleases. Among these escapees, we identified C19ORF66 and reveal that like the 18 previously identified escapees, this transcript is protected from degradation by its 3'UTR. We 19 then show that C19ORF66, a known anti-viral protein, is a potent KSHV restriction factor, 20 suggesting that its ability to escape viral cleavage may be an important component of the host 21 response to viral infection. Collectively, our comparative approach is a powerful tool to pinpoint 22 key regulators of the viral-host interplay and led us to uncover a novel KSHV regulator. 23 24 25 26 All rights reserved. No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/506410 doi: bioRxiv preprint characterized by a defined sequence motif (19) rendering it difficult to identify new escaping 48 transcripts by traditional sequence search. Consequently, the host vs. viral endonuclease 49 dichotomy to only defining characteristic of this novel type of RNA element. 50 All rights reserved. No reuse allowed without permission. \"},\n"," {'paper_id': '52e88fad10a7a632c989627528d3411d99fb322d',\n","  'section': 'Abstract',\n","  'text': 'The pathogenic Middle East respiratory syndrome coronavirus (MERS-CoV), severe acute respiratory syndrome coronavirus (SARS-CoV-1) and COVID-19 coronavirus have all emerged into the human population with devastating consequences. These viruses make use of a large envelope protein called spike (S) to engage host cell receptors and catalyze membrane fusion. Because of the vital role that these S proteins play, they represent a vulnerable target for the development of therapeutics to combat these highly pathogenic coronaviruses. Here, we describe the isolation and characterization of single-domain antibodies (VHHs) from a llama immunized with prefusion-stabilized coronavirus spikes. These VHHs are capable of potently neutralizing MERS-CoV or SARS-CoV-1 S pseudotyped viruses. The crystal structures of these VHHs bound to their respective viral targets reveal two distinct epitopes, but both VHHs block receptor binding. We also show cross-reactivity between the SARS-CoV-1 S-directed VHH and SARS-CoV-2 S, and demonstrate that this cross-reactive VHH is capable of neutralizing SARS-CoV-2 S pseudotyped viruses as a bivalent human IgG Fc-fusion. These data provide a molecular basis for the neutralization of pathogenic betacoronaviruses by VHHs and suggest that these molecules may serve as useful therapeutics during coronavirus outbreaks. '},\n"," {'paper_id': '530f3ba1a2c044305d73123fd8d2246f63120bc3',\n","  'section': 'Abstract',\n","  'text': 'Objective: To investigate the impact of the COVID-19 outbreak on the medical advice seeking of Chinese children with chronic kidney disease (CKD). : An anonymous online questionnaire survey was conducted in 17 pediatric nephropathy diagnosis and treatment centers in China. The questions collected basic information on the patients and their parents and data on changes in the approach to medical treatment and their needs in the context of the outbreak etc. This is a Multicenter Cross-section Study. Results: A total of 735 valid questionnaires were collected. 555 patients (75.5%) and their parents said that the outbreak had a significant influence on their medical treatment: 264 patients (47.6%) said that it would be delayed by 2 to 4 weeks and 199 patients (35.9%) by 4 to 8 weeks. 510 patients (84.16%) hoped to get in touch with specialists through online consultation, and 528 patients (84.5%) hoped that online consultation could be implemented and that medication could be delivered to them.. A total of 458 patients (62.3%) said that their greatest concern was that the CKD would be aggravated or that they would experience a relapse; only 203 patients were infected by 2019-nCoV. A total of 313 patients (42.5%) experienced anxiety and thus required the intervention of psychologists. Conclusion: The COVID-19 outbreak has affected the medical treatment of children with CKD. Online consultation, medication delivery and psychological counselling are the greatest needs reported by patients and their families and could especially provide solutions for the management of low income children with CKD in remote rural areas in the context of the COVID-19 epidemic. The study can well explore the impact of COVID-19 outbreak on the medical treatment of CKD children in China and the needs of current patients. The study explored ways to meet the medical needs of CKD children in the context of a public health emergency of international concern, which provides a method support for all countries in the world experiencing COVID-19 outbreak. The study is an exploration of the coping strategies for CKD management in China. The limitations of this study is that it is quite simple and descriptive, and many studies need to be further carried out. '},\n"," {'paper_id': '532dc582abd41e833303742ca40976db63fe6fc1',\n","  'section': 'Abstract',\n","  'text': \"During lytic replication of Kaposi's sarcoma-associated herpesvirus (KSHV), the gene expression landscape of a cell is remodeled to evade the immune response and create an environment favorable to viral replication. A major driver of these gene expression changes is a virally encoded, messenger RNA (mRNA)-specific endonuclease termed SOX. SOX cleaves the majority of cytoplasmic mRNAs, but does so at specific internal sites loosely defined by a degenerate sequence motif. If and how RNA sequence directs SOX targeting remained unknown. To address these questions, we used recombinant, highly purified SOX endonuclease in a series of biochemical assays to reconstitute the cleavage reaction in vitro and gain significant insight into the biochemical mechanism of both SOX target recognition and endonucleolytic cleavage. Using this system, we determined that cut site specificity is preserved with purified SOX and a validated target RNA and thus does not require additional cellular cofactors. Furthermore, we showed that . CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/320929 doi: bioRxiv preprint SOX displays robust, sequence-specific RNA binding to residues proximal to the cleavage site, which must be presented in a particular structural context. The strength of SOX binding dictates cleavage efficiency, providing an explanation for the breadth of target RNA susceptibility observed in cells. Kaposi's sarcoma-associated herpesvirus (KSHV) is an oncogenic human virus that causes Kaposi's sarcoma, primary effusion lymphoma, and multicentric Castleman disease. During viral replication, KSHV expresses an enzyme called SOX that cuts and inactivates the majority of cellular messenger RNAs, preventing their translation into proteins. Some mRNAs are efficiently cleaved by SOX, while others are poorly cleaved, but the mechanistic basis underlying this selectivity has remained largely unknown. Here, we reveal that the efficiency of RNA cleavage is heavily impacted by RNA sequences proximal to the cleavage site, which serve as a SOX binding platform. This helps explain both the range of RNA cleavage efficiency observed in SOX-expressing cells as well as the sequence specificity underlying SOX targeting. \"},\n"," {'paper_id': '53326420d7eb607538a248c0da27d3b092f651e2',\n","  'section': 'Abstract',\n","  'text': '288 cases have been confirmed out of China from January 3 to February 13, 2020. We collected and synthesized all available information on these cases from official sources and media. We analyzed importations that were successfully isolated and those leading to onward transmission. We modeled their number over time, in relation to the origin of travel (Hubei province, other Chinese provinces, other countries) and interventions. We characterized importations timeline to assess the rapidity of isolation, and epidemiologically linked clusters to estimate the rate of detection. We found a rapid exponential growth of importations from Hubei, combined with a slower growth from the other areas. We predicted a rebound of importations from South East Asia in the upcoming weeks. Time from travel to detection has considerably decreased since the first importation, however 6 cases out of 10 were estimated to go undetected. Countries outside China should be prepared for the possible emergence of several undetected clusters of chains of local transmissions. '},\n"," {'paper_id': '53442eacc3f233078507fa37b78267399e8c1e3b',\n","  'section': 'Abstract',\n","  'text': 'Zika Virus (ZIKV) is a causative agent for poor pregnancy outcome and fetal developmental abnormalities, including microcephaly and eye defects. As a result, ZIKV is now a confirmed teratogen. Understanding host-pathogen interactions, specifically cellular perturbations caused by ZIKV, can provide novel therapeutic targets. In order to complete viral replication, viral pathogens control the host cellular machineries and regulate various factors, including long noncoding RNA (lncRNA) genes, at transcriptional levels. The role of lncRNA genes in the pathogenesis of ZIKV-mediated microcephaly and eye defects is currently unknown. To gain additional insights, we focused on profiling the differentially expressed lncRNA genes during ZIKV infection in mammalian cells. For this study, we employed a contemporary clinical Zika viral isolate, PRVABC59, of Asian genotype. We utilized an unbiased RNA sequencing approach to profile the lncRNA transcriptome in ZIKV infected Vero cells. We identified a total of 121 lncRNA genes that are differentially regulated at 48 hours post-infection. The majority of these genes are independently validated by reverse-transcription qPCR. A notable observation was that the lncRNAs, MALAT1 (Metastasis Associated Lung Adenocarcinoma Transcript 1) and NEAT1 (Nuclear Paraspeckle Assembly Transcript 1), are down-regulated upon Zika viral infection. MALAT1 and NEAT1 are known as nuclear localized RNAs that regulate gene expression and cell proliferation. Protein-lncRNA interaction maps revealed that MALAT1 and NEAT1 share common interacting partners and form a larger network comprising of 71 cellular factors. ZIKV-mediated dysregulation of these two regulatory lncRNAs can alter the expression of respective target genes and associated biological functions, an important one being cell division. In conclusion, this investigation is the first to provide insight into the biological connection of lncRNAs and ZIKV which can be further explored for developing antiviral therapy and understanding fetal developmental processes. '},\n"," {'paper_id': '537009c0181c3269f40753f75ddca4d17ae8abae',\n","  'section': 'Abstract',\n","  'text': 'The epidemic caused by the novel coronavirus in Wuhan at the end of 2019 has become an urgent public event of worldwide concern. However, due to the changing data of the epidemic, there is no scientific estimate of the cure rate and case fatality rate of the epidemic. This study proposes a method to estimate the cure rate and case fatality rate of COVID-19. The ratio of cumulative discharges on a given day to the sum of cumulative discharges on a given day and cumulative deaths before j days is used to estimate the cure rate. Moreover, the case fatality ratio can also be estimated. After simulation calculations, j is statistically appropriate when it is 8-10, and it is also clinically appropriate. When j is 9, based on the available data, it is inferred that the cure rate of this epidemic is about 93% and the case fatality rate is about 7%. This method of estimating the cure rate can be used to evaluate the effectiveness of treatment in different medical schemes and different regions, and has great value and significance for decision-making in the epidemic. improvement of medical conditions, and rich experience in treatment, patients with COVID-19 will be treated more effectively. Consequently, the CR will increase, and the CFR will decrease. The authors declare no conflict of interest. '},\n"," {'paper_id': '53726a84035f04167440dde803026c996d8794c0',\n","  'section': 'Abstract',\n","  'text': 'Background: Globally, the novel coronavirus, is a public health problem causing respiratory infections. Since the outbreak of severe acute and Middle East respiratory syndromes coronavirus was not reported to cause human infections. Now, it become an epidemic proportion with growing number of cases and deaths. Methods: A scoping review was conducted following the methodological framework. In this scoping review, 50 records published before 28 March, 2020 were included and discussed to better understand the current epidemiology, pathogenesis, immunological response, diagnosis, evasion mechanisms and suggested strategies to boost immune system, challenges, treatment, and future preventions of the virus. PubMed, BioRxiv, MedRxiv, Global Health and google scholars were searched comprehensively for articles, preprints, grey literatures, reports, websites, conference proceedings and expert information. Studies conducted in human and published in English language were included in the analysis. All the findings and statements of the review regarding the outbreak are based on published information as listed in citations. We identified 360 records, of which 50 studies met the inclusion criteria. We synthesized the data from the included records and dig out the deep insights of them and pooled into this review. The burden of the outbreak is worsening due to overcrowding, presence of asymptomatic carriers, scarcity of test kits, the immune escaping ability of the virus and lack of awareness. Due to the fast-spreading nature of COVID-19 the prevention and control strategies become challenging. It is imposing social, psychological, and socio-economic impacts. We recommended that following social distancing, isolation suspects, using personal protective equipment, health education and introducing hand-washing practices, avoiding contact with animals, vaccine development and treatment for controlling and prevention. '},\n"," {'paper_id': '538632fe0548a99c7db35480edcd74802237c1df',\n","  'section': 'Abstract',\n","  'text': 'COVID-19 coronavirus pandemic has virtually locked down the entire world of human population, and through its rapid and unstoppable spread COVID-19 has essentially compartmentalised the population merely into susceptible, exposed, infected and recovered classes. Adapting the classical epidemic modelling framework, two distinct routes of COVID-19 transmission are incorporated into a model: (a) direct person-to-person contact transmission, and (b) indirect airborne and fomites-driven transmission. The indirect non-contact transmission route needs to explored in models of COVID-19 spread, because evidences show that this route of transmission is entirely viable with hugely uncertain level of relative contribution. This theoretical study based on model simulations demonstrates the following: (1) Not incorporating indirect transmission route in the model leads to underestimation of the basic reproduction number, and hence will impact on the COVID-19 mitigation decisions; (2) Lockdown measures can suppress the primary infection peak, but will lead to a secondary peak whose relative strength and time of occurrence depend on the success and duration of the lockdown measures; (3) To make lockdown effective, a considerable level of reduction in both contact and non-contact transmission rates over a long period is required; (4) To bring down the infection cases below any hypothetical health-care capacity, reduction of non-contact transmission rate is key, and hence active measures should be taken to reduce non-contact transmission (e.g., extensive uses of areal and aerosol disinfectant in public spaces to improve contaminated surfaces and air); (5) Any premature withdrawal of lockdown following the sign of a brief retracement in the infection cases can backfire, and can lead to a quicker, sharper and higher secondary peak, due to reactivation of the two transmission routes. Based on these results, this study recommends that any exit policy from lockdown, should take into account the level of transmission reduction in both routes, the absolute scale of which will vary among countries depending on their health-service capacity, but should be computed using accurate time-series data on infection cases and transmission rates. '},\n"," {'paper_id': '53bfbed1d1353b6fcc48b2dbf37878bdc9eb5841',\n","  'section': 'Abstract',\n","  'text': \"Aim: Kidney impairment is observed in patients with . We aimed to demonstrate the effect of anti-COVID-19 agent remdesivir on renal fibrosis. Methods: Remdesivir and its active nucleoside metabolite GS-441524 were used to treat TGF-β stimulated renal fibroblasts (NRK-49F) and human renal epithelial cells (HK2). Cell viability was determined by CCK8 assay, and fibrotic markers were measured by Western blotting. Vehicle or remdesivir were given by intraperitoneal injection or renal injection through the left ureter in unilateral ureteral obstruction (UUO) mice. Serum and kidneys were harvested. The concentrations of remdesivir and GS-441524 were measured using LC-MS/MS. Renal and liver function were assessed. Renal fibrosis was evaluated by Masson's trichrome staining and Western blotting. Results: Remdesivir and GS-441524 inhibited cell proliferation and the expression of fibrotic markers (fibronectin, pSmad3, and aSMA) in NRK-49F and HK2 cells. Intraperitoneal injection or renal injection of remdesivir attenuated renal fibrosis of UUO kidneys. Renal and liver function were not changed in remdesivir treated UUO mice. Remdesivir can not be detected, but two remdesivir metabolites were detected after injection. Conclusion: Remdesivir inhibits renal fibrosis in obstructed kidneys. author/funder. All rights reserved. No reuse allowed without permission. \"},\n"," {'paper_id': '53d0f495085900746ae8492a12b1283c056e7308',\n","  'section': 'Abstract',\n","  'text': 'Please note: this is a working document and has not been submitted for journal publication. It is planned that a later version of this document will be submitted for peer-reviewed publication, but in the interests of sharing information during a rapidly changing epidemic landscape, we are making this early version available. We estimate the growth in demand for ICU beds in Chicago during the emerging COVID-19 epidemic, using state-of-the-art computer simulations calibrated for the SARS-CoV-2 virus. The questions we address are these: (1) Will the ICU capacity in Chicago be exceeded, and if so by how much? (2) Can strong mitigation strategies, such as lockdown or shelter in place order, prevent the overflow of capacity? (3) When should such strategies be implemented? Our answers are as follows: (1) The ICU capacity may be exceeded by a large amount, probably by a factor of ten. (2) Strong mitigation can avert this emergency situation potentially, but even that will not work if implemented too late. (3) If the strong mitigation precedes April 1 st , then the growth of COVID-19 can be controlled and the ICU capacity could be adequate. The earlier the strong mitigation is implemented, the greater the probability that it will be successful. After around April 1 2020, any strong mitigation will not avert the emergency situation. In Italy, the lockdown occurred too late and the number of deaths is still doubling every 2.3 days. It is difficult to be sure about the precise dates for this window of opportunity, due to the inherent uncertainties in computer simulation. But there is high confidence in the main conclusion that it exists and will soon be closed. Our conclusion is that, being fully cognizant of the societal trade-offs, there is a rapidly closing window of opportunity to avert a worst-case scenario in Chicago, but only with strong mitigation/lockdown implemented in the next week at the latest. If this window is missed, the epidemic will get worse and then strong mitigation/lockdown will be required after all, but it will be too late. '},\n"," {'paper_id': '53d4687229c3d2940589c42fa3038ddc8712808b',\n","  'section': 'Abstract',\n","  'text': 'The COVID-19 pandemic has shown a markedly low proportion of cases among children. Age disparities in observed cases could be explained by assortative mixing patterns and reactive school closures which decrease mixing between children, or by children exhibiting lower susceptibility to infection, or by children having a lower propensity to show clinical symptoms. We formally test these hypotheses by fitting an age-structured mathematical model to epidemic data from six countries, finding 1 . CC-BY 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint . https://doi.org/10.1101/2020.03.24.20043018 doi: medRxiv preprint / strong age dependence in the probability of developing clinical symptoms, rising from around 20% in under 10s to over 70% in older adults. We find that interventions aimed at halting transmission in children may have minimal effects on preventing cases depending on the relative transmissibility of subclinical infections. Our estimated age-specific clinical fraction has implications for the expected global burden of clinical cases because of demographic differences across settings. In younger populations, the expected clinical attack rate would be lower, although it is likely that comorbidities in low-income countries will affect disease severity. Without effective control measures, regions with older populations may see disproportionally more clinical cases, particularly in the later stages of the pandemic. '},\n"," {'paper_id': '540f321681ba936b7463b73e60fb389d8f1fbb57',\n","  'section': 'Abstract',\n","  'text': 'Background: Exponential-like infection growths leading to peaks (which could be the inflection points or turning points) are usually the hallmarks of infectious disease outbreaks including coronaviruses. To predict the inflection points, i.e., inflection time (T max ) & maximal infection number (I max ) of the novel coronavirus (COVID-19), we adopted a trial and error strategy and explored a series of approaches from simple logistic modeling (that has an asymptomatic line) to sophisticated tipping point detection techniques for detecting phase transitions but failed to obtain satisfactory results. Method: Inspired by its success in diversity-time relationship (DTR), we apply the PLEC (power law with exponential cutoff) model for detecting the inflection points of COVID-19 outbreaks. The model was previously used to extend the classic species-time relationship (STR) for general DTR (Ma 2018), and it has two \"secondary\" parameters (computed from its 3 parameters including power law scaling parameter w, taper-off parameter d to overwhelm virtually exponential growth ultimately, and a parameter c related to initial infections): one that was originally used for estimating the potential or \\'dark\\' biodiversity is proposed to estimate the maximal infection number (I max ) and another is proposed to determine the corresponding inflection time point (T max ). We successfully estimated the inflection points [I max , T max ] for most provinces (≈85%) in China with error rates <5% in both I max and T max . We also discussed the constraints and limitations of the proposed approach, including (i) sensitive to disruptive jumps, (ii) requiring sufficiently long datasets, and (iii) limited to unimodal outbreaks. : medRxiv preprint Ma, ZS (2018b) Diversity time-period and diversity-time-area relationships exemplified by the human microbiome. Scientific Reports, 8(1), 7214. https://doi. '},\n"," {'paper_id': '54122e9771f67d234bdcc67172a788bc96eec34e',\n","  'section': 'Abstract',\n","  'text': ' 18  '},\n"," {'paper_id': '541777e512aa3907a9db91cde5207a07eabde5ef',\n","  'section': 'Abstract',\n","  'text': '2 postnatal diet. 3 4 Running Title. Effect of diet on lipidome of perinatal vagina 5 6 Summary sentence. Vaginal swab lipidome profiles at 48 h reflect the fat composition of 7 neonatal diet during first two days postnatal. '},\n"," {'paper_id': '541e7d329fa466fe7b961263666386dddec75894',\n","  'section': 'Abstract',\n","  'text': 'Declaring the end of an outbreak is an important step in controlling infectious disease outbreaks. An objective estimation of the probability of cases arising in the future is important to reduce the risk of post-declaration flare-ups. We developed a simulationbased model to quantify that probability. We tested it on simulated Ebola Virus Disease (EVD) data and found this probability was most sensitive to the instantaneous reproduction number, the reporting rate, and the delay between symptom onset and recovery or death of the last detected case. For EVD, our results suggest that the current WHO criterion of 42 days since the outcome of the last detected case is too short and very sensitive to underreporting. The 90 days of enhanced surveillance period after the end-of-outbreak declaration is therefore crucial to capture potential flare-ups of cases. Hence, we suggest a shift to a preliminary end-of-outbreak declaration after 63 days from the symptom onset day of the last detected case. This should be followed by a 90-day enhanced surveillance, after which the official end-ofoutbreak can be declared. This corresponds to less than 5% probability of flare ups in most of the scenarios examined. Our quantitative framework could be adapted to define end-of-outbreak criteria for other infectious diseases. '},\n"," {'paper_id': '544f23d64a007c69bc637743a3ebe7809d9c9c69',\n","  'section': 'Abstract',\n","  'text': 'medRxiv preprint Interpretation Parameters that successfully reproduce the observed number of cases indicate that both R 0 and the prevalence of the virus might be underestimated. This is in concordance with the newest research on undocumented COVID-19 cases. Consequently, the actual mortality rate is putatively lower than estimated. Confirmation of the pandemic characteristic by further refinement of the model and screening tests is crucial for developing an effective strategy for the global epidemiological crisis. 1 Balcan D, Gonçalves B, Hu H, Ramasco JJ, Colizza V, Vespignani A. Modeling the spatial spread of infectious diseases: the GLobal Epidemic and Mobility computational model. J Comput Sci 2010; 1 : 132-45. 2 Van den Broeck W, Gioannini C, Gonçalves B, Quaggiotto M, Colizza V, Vespignani A. The GLEaMviz computational tool, a publicly available software to explore realistic epidemic spreading scenarios at the global scale. BMC Infect Dis 2011; 11 : 37. 3 Tizzoni M, Bajardi P, Poletto C, et al. Real-time numerical forecast of global epidemic spreading: case study of 2009 A/H1N1pdm. BMC Med 2012; '},\n"," {'paper_id': '5470929d340ca0fbb030ce8e00a885a0d730adbe',\n","  'section': 'Abstract',\n","  'text': 'medRxiv preprint SEIR-PO models generally show a better performance than SIR, SEIR and SEIR-AHQ models on the COVID-19 epidemics, whose success could be attributed to the inclusion of self-protection and quarantine, and a proper trade-off between model complexity and fitting accuracy. '},\n"," {'paper_id': '54ae20515fcef8417e255b19df1a5bf6f2ab40bb',\n","  'section': 'Abstract',\n","  'text': 'The novel coronavirus disease 2019 (COVID-19) outbreak on the Diamond Princess ship has caused over 634 cases as of February 20, 2020. We model the transmission process on the ship with a stochastic model and estimate the basic reproduction number at 2.2 (95%CI: 2.1−2.4). We estimate a large dispersion parameter than other coronaviruses, which implies that the virus is difficult to go extinction. The epidemic doubling time is at 4.6 days (95%CI: 3.0−9.3), and thus timely actions were crucial. The lesson learnt on the ship is generally applicable in other settings. '},\n"," {'paper_id': '54cfc3c68e1e4832fee5b3294e5673e37978ebdf',\n","  'section': 'Abstract',\n","  'text': \"Corona virus disease 2019 has rapidly become the most severe public health issue all over the world. Despite respiratory symptoms, hepatic injury has also been observed in clinical settings. This study aimed to investigate the risk factors involved with hepatic injury in the patients with COVID-19. Methods: A total of 85 hospitalized patients who were diagnosed with COVID-19 in Beijing You'an Hospital were retrospectively analyzed. According to liver function, they were divided into ALT normal group (n=52) and ALT elevation group (n=33). Clinical features and laboratory data were compared between the two groups. The independent risk factors for liver injury were analyzed. Results: There were 33 patients with hepatic injury in our study, accounting for 38.8% (33/85). The patients in ALT elevation group were older than those in ALT normal group. The levels of lactic acid, CRP, myoglobin, and neutrophils were significantly higher in ALT elevation group. The lymphocytes and albumin were significantly lower in ALT elevation group. The proportion of severe and critical patients in ALT elevation group was significantly higher. Multivariate logistic regression analysis showed CRP ≥ 20 mg/L and lymphocyte count< 1.1×10^9/L were independently related to hepatic injury. Lymphopenia and CRP may serve as the risk factors related to hepatic injury in patients with COVID-19, which might be related to inflammatory cytokine storm in liver injury. Early detection and timely treatment of hepatic injury in patients with COVID-19 are necessary. \"},\n"," {'paper_id': '54d2fd5689ac3255e8041c36b5fb333a3bd3f9b2',\n","  'section': 'Abstract',\n","  'text': \"The CoViD-19 outbreak has escalated to a pandemic in the last few weeks, with no signs of stopping. Pharmaceutical solutions based upon virologic studies, at this point, remain inconclusive. In contrast, this paper looks towards epidemiological models during this phase of viral growth, in particular, by providing a responsive, timely model of the R value based on the previous few days' results. Such an R value, although bearing less statistical precision due to limited sampling, could allow R to become a more effective, responsive standalone measure of infectious transmission. It demonstrates that the R value can be used as a dynamic, time-dependent indicator without the use of curve-fitting, and also estimates the most recent R-value of the \"},\n"," {'paper_id': '5507325ad63261970f22c625abcfa59f99e24efd',\n","  'section': 'Abstract',\n","  'text': 'RNA degradation by RNase L during 2-5A-mediated decay (2-5AMD) is a conserved mammalian stress response to viral and endogenous double-stranded RNA (dsRNA). 2-5AMD onsets rapidly and facilitates a switch of protein synthesis from homeostasis to production of interferons (IFNs). To understand the mechanism of this protein synthesis reprogramming, we examined 2-5AMD in human cells. 2-5AMD triggers polysome collapse characteristic of a translation initiation defect, but translation initiation complexes and ribosomes purified from the translation-arrested cells remain functional. Using spike-in RNA-seq we found that basal messenger RNAs (mRNAs) rapidly decay, while mRNAs encoding IFNs and IFN-stimulated genes evade 2-5AMD and accumulate. The IFN evasion results from the combined effect of better mRNA stability and positive feedback amplification in the IFN response. Therefore, 2-5AMD and transcription act in concert to revamp the cellular mRNA composition. The resulting preferential accumulation of innate immune mRNAs establishes \"prioritized\" synthesis of defense proteins. '},\n"," {'paper_id': '552548a4e9a576953acad9fa549ec4d796fe52c9',\n","  'section': 'Abstract',\n","  'text': \"Clavibacter is an agriculturally important genus comprising a single species, Clavibacter michiganensis, and multiple subspecies, including, C. michiganensis subsp. nebraskensis which causes Goss's wilt/blight of corn and accounts for high yield losses -listed among the five most significant diseases of corn in the United States of America. Our research objective was to develop a robust and rapid multiplex TaqMan real-time PCR (qPCR) to detect C. michiganensis in general and C. michiganensis subsp. nebraskensis with enhanced reliability and accuracy by adding non-complementary AT sequences to the 5' end of the forward and reverse primers. Comparative genomic analyses were performed to identify unique and conserved gene regions for primer and probe design. The unique genomic regions, ABC transporter ATP-binding protein CDS/ABC-transporter permease and MFS transporter were determined for specific detection of C. michiganensis and C. m. subsp. nebraskensis, respectively. The AT-rich sequences at the 5' position of the primers enhanced the reaction efficiency and sensitivity of rapid qPCR cycling; the reliability, accuracy and high efficiency of the developed assay was confirmed after testing with 59 strains from inclusivity and exclusivity panels -no false positives or false negatives were detected. The assays were also validated through naturally and artificially infected corn plant samples; all samples were detected for C. michiganensis and C. m. subsp. nebraskensis with 100% accuracy. The assay with 5' AT-rich sequences detected up to 10-and 100-fg of C. michiganensis and C. michiganensis subsp. nebraskensis genome targets, respectively. No adverse effect was observed when sensitivity assays were spiked with host genomic DNA. Addition of 5' AT rich sequences enhanced the qPCR reaction efficiency from 0.82 (M = -3.83) and 0.91 (M = -3.54) to 1.04 (with optimum slope value; M = -3.23) for both C. michiganensis and C. michiganensis subsp. nebraskensis, respectively; a increase of 10-fold sensitivity was also obtained with C. michiganensis primer set. The methodology proposed here can be used to optimize the reaction efficiency and to harmonize the diagnostic protocols which have prodigious applications in routine diagnostics, biosecurity and microbial forensics. Δ G value (kcal/mol) in plot calculated by mFold web server; d Selfcomplementary and 3' self-complementary scores (Any and 3', respectively) of the oligos calculated using Primer3 Plus. e Query cover and Identity (%) of the oligos were calculated using Blastn; no significant similarity was found for any of FLAP primers. The General Time Reversible with gamma distribution model (GTR + G) was used to construct the maximum likelihood tree. The different subspecies within C. michiganensis were grouped in three main clusters. C. m. subsp. tessellarius was grouped discretely in one clade. In contrast, the remaining eight subspecies were placed into two clades. In the first clade, C. m. subsp. nebraskensis, C. m. subsp. insidiosus, C. m. subsp. capsici, C. m. subsp. phaseoli and C. m. subsp. chiloensis were grouped. Interestingly, isolates of C. m. subsp. nebraskensis were placed . CC-BY 4.0 International license is made available under a \"},\n"," {'paper_id': '553ee978cf0dbcd9955699dbcb6f30b776d61e98',\n","  'section': 'Abstract',\n","  'text': 'There is an urgent necessity of effective medication against SARS CoV-2, which is producing the COVID-19 pandemic across the world. Its main protease (M pro ) represents an attractive pharmacological target due to its involvement in essential viral functions. The crystal structure of free M pro shows a large structural resemblance with the main protease of CoV-1. Here we report that CoV-2 M pro is however 300% more sensitive than CoV-1 M pro in transmitting tiny structural changes across the whole protein through long-range interactions. The largest sensitivity of M pro to structural perturbations is located exactly around the catalytic site Cyst-145, and coincides with the binding site of several of its inhibitors. These findings, based on a simplified representation of the protein as a residue network, may help in designing potent inhibitors of CoV-2 M pro . : bioRxiv preprint We turn now our attention to the analysis of the M pro of CoV-2 complexed with an inhibitor for which the crystal structure was resolved by Zhang et al. in two 9 author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '5579c8733222689f36b41d31b76700ab4f4dd96e',\n","  'section': 'Abstract',\n","  'text': \"Social distancing policies are critical but economically painful measures to flatten the curve against emergent infectious diseases. As the novel coronavirus that causes COVID-19 spread throughout the United States in early 2020, the federal government issued social distancing recommendations but left to the states the most difficult and consequential decisions restricting behavior, such as canceling events, closing schools and businesses, and issuing stay-at-home orders. We present an original dataset of state-level social distancing policy responses to the epidemic and explore how political partisanship, COVID-19 caseload, and policy diffusion explain the timing of governors' decisions to mandate social distancing. An event history analysis of five social distancing policies across all fifty states reveals the most important predictors are political: all else equal, Republican governors and governors from states with more Trump supporters were slower to adopt social distancing policies. These delays are likely to produce significant, on-going harm to public health. for helpful comments and suggestions, and Erika Steiskal for graphic design assistance. State social distancing data are available at http://covid19statepolicy.org/. \"},\n"," {'paper_id': '5585d267f9b505b068f9f72bff5f12c248472f0a',\n","  'section': 'Abstract',\n","  'text': \"Background: Since late December 2019, novel coronavirus-infected pneumonia (NCP) emerged in Wuhan, Hubei province, China. Meanwhile, NCP rapidly spread from China to other countries, and several countries' government rush to evacuate their citizens from Wuhan. We analyzed the infection rate of the evacuees and extrapolated the results in Wuhan's NCP incidence estimation. Methods: We collected the total number and confirmed cases of 2019-nCov infection in the evacuation of Korea, Japan, Germany, Singapore, and France and estimated the infection rate of the 2019 novel coronavirus (2019-nCov) among people who were evacuated from Wuhan with a meta-analysis. NCP incidence of Wuhan was indirectly estimated based on data of evacuation. \"},\n"," {'paper_id': '558d318e1655da9f5e9507ccf24fc773a0d377ff',\n","  'section': 'Abstract',\n","  'text': 'We utilized a cell-level approach to examine glycolytic pathways in the DLPFC of subjects with schizophrenia (n=16) and control (n=16) subjects and found decreased mRNA expression of glycolytic enzymes in pyramidal neurons, but not astrocytes. To replicate these novel bioenergetic findings, we probed independent datasets for bioenergetic targets and found similar abnormalities. Next, we used a novel strategy to build a schizophrenia bioenergetic profile by a tailored application of the Library of Integrated Network-Based Cellular Signatures data portal (iLINCS) and investigated connected cellular pathways, kinases, and transcription factors using Enrichr. Finally, with the goal of identifying drugs capable of \"reversing\" the bioenergetic schizophrenia signature, we performed a connectivity analysis with iLINCS and identified peroxisome proliferator-activated receptor (PPAR) agonists as promising therapeutic targets. We administered a PPAR agonist to the GluN1 knockdown model of schizophrenia and found it improved long-term memory. Taken together, our findings suggest that tailored bioinformatics approaches, coupled with the LINCS library of transcriptional signatures of chemical and genetic perturbagens may be employed to identify novel treatment strategies for schizophrenia and related diseases. '},\n"," {'paper_id': '55d15b44cadac3402124640369b8ef956bc017a8',\n","  'section': 'Abstract',\n","  'text': '2 sensitivity to health care worker collected nasopharyngeal swabs 3 4 ABSTRACT 24 '},\n"," {'paper_id': '55e0d20b6cc56c2cdc131e1dcc51f752575cf532',\n","  'section': 'Abstract',\n","  'text': 'the transcription machinery. These results mechanistically uncovered how the steps of pause-30! release and termination maintain transcriptional memory over mitosis. '},\n"," {'paper_id': '56a81431a148344381a1da80c877885f3f3d6d59',\n","  'section': 'Abstract',\n","  'text': 'Alternative splicing is a widespread phenomenon in metazoans by which single genes are able to produce multiple isoforms of the gene product. However, this has been poorly characterised in apicomplexans, a major phylum of some of the most important global parasites. Efforts have been hampered by atypical transcriptomic features, such as the high AT content of Plasmodium RNA, but also the limitations of short read sequencing in deciphering complex splicing events. In this study, we utilised the long read direct RNA sequencing platform developed by Oxford Nanopore Technologies (ONT) to survey the alternative splicing landscape of Toxoplasma gondii and Plasmodium falciparum. We find that while native RNA sequencing has a reduced throughput, it allows us to obtain full-length or near full-length transcripts with comparable quantification to Illumina sequencing. By comparing this data with available gene models, we find widespread alternative splicing, particular intron retention, in these parasites. Most of these transcripts contain premature stop codons, suggesting that in these parasites, alternative splicing represents a pathway to transcriptomic diversity, rather than expanding proteomic diversity. Moreover, alternative splicing rates are comparable between parasites, suggesting a shared splicing machinery, despite notable transcriptomic differences between the parasites. This work highlights a strategy in using long read sequencing to understand splicing events at the whole transcript level, and has implications in future interpretation of RNA-seq studies. '},\n"," {'paper_id': '57a86b3acd182c955877afd4792719a4e9a0ac32',\n","  'section': 'Abstract',\n","  'text': 'Background: So far, the psychological impact of COVID-19 epidemic among family members of Health care workers (HCWs) in China has been neglected. The present cross-sectional study aimed to investigate the mental health status and related factors of families of HCWs in Designated Hospitals in Ningbo, China. Method: Family members of HCWs working in five designated hospitals in Ningbo, China were recruited between February 10th and 20th, 2020. Information on demographic variables, the COVID-19-related events in the lives, knowledge of COVID-19 and the working status of family members (that is, HCWs) was collected using online self-administered questionnaires. The mental health status were assessed using the Chinese version of Patient Health Questionnare-9 (PHQ-9) and Chinese version of Generalized Anxiety Disorder-7 (GAD-7). Multivariable logistic regression analyses were conducted to identify the main factors associated with the mental health conditions. Results: A total of 822 participants completed questionnaires correctly. (response rate of 95.80% ). The overall prevalence of GAD and depressive symptoms were 33.73%, and 29.35%, respectively. More times (hours) to focus on the COVID-19 (Odd ratio (OR)=1.215, 95% confidence interval (CI):1.061-1.391), family members (that is, HCWs) directly contact with confirmed or suspected COVID-19 patients (OR=1.477, 95%CI:1.069-2.040) were risk factors for GAD, while higher participants self-reported safety score for protective equipment of HCWs (OR=0.807, 95%CI:0.700-0.930) was a protective factor. More times (hours) to focus on the COVID-19 (OR=1.215, 95%CI:1.061-1.391), longer average working times per week for family members (that is, HCWs) (OR=1.017, 95%CI:1.005-1.029), being parents and other next of kin of HCWs were risk factors for depressive symptoms (OR=3.526, respectively). In addition, compared with participants who were HCWs, participants who were enterprise workers and were more likely to develop depressive symptoms(OR=1.750, 95%CI:1.104-2.776), while who were government employees or institutions All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '57c5661bf42468bb2898c1e26aa18a0d02237bfc',\n","  'section': 'Abstract',\n","  'text': 'Rapid spread of SARS-CoV-2 in Wuhan prompted heightened surveillance in Shenzhen and elsewhere in China. The resulting data provide a rare opportunity to measure key metrics of disease course, transmission, and the impact of control. The Shenzhen CDC identified 391 SARS-CoV-2 cases from January 14 to February 12, 2020 and 1286 close contacts. We compare cases identified through symptomatic surveillance and contact tracing, and estimate the time from symptom onset to confirmation, isolation, and hospitalization. We estimate metrics of disease transmission and analyze factors influencing transmission risk. Cases were older than the general population (mean age 45) and balanced between males (187) and females (204). Ninety-one percent had mild or moderate clinical severity at initial assessment. Three have died, 225 have recovered (median time to recovery is 21 days). Cases were isolated on average 4.6 days after developing symptoms; contact tracing reduced this by 1.9 days. Household contacts and those travelling with a case where at higher risk of infection (ORs 6 and 7) than other close contacts. The household secondary attack rate was 15%, and children were as likely to be infected as adults. The observed reproductive number was 0.4, with a mean serial interval of 6.3 days. Our data on cases as well as their infected and uninfected close contacts provide key insights into SARS-CoV-2 epidemiology. This work shows that heightened surveillance and isolation, particularly contact tracing, reduces the time cases are infectious in the community, thereby reducing R. Its overall impact, however, is uncertain and highly dependent on the number of asymptomatic cases. We further show that children are at similar risk of infection as the general population, though less likely to have severe symptoms; hence should be considered in analyses of transmission and control. '},\n"," {'paper_id': '57e01ad2a4961cd5cc6a3733f5f8c013a8946f3c',\n","  'section': 'Abstract',\n","  'text': 'Background: In the beginning of January 2020, new unknown virus pneumonia cases started to emerge in local hospitals in Wuhan, China. This virus epidemic quickly became a public health emergency of international concern by the WHO. Enormous amount of medical supplies as well as healthcare personals from other provinces were mobilized to support Wuhan. This current work tent to help people understanding how infectious disease spread and the purpose and consequences of various efforts based on simulation model. Method: a simulation model was created using known parameters. R0 set to 3 and mean incubation time to be 7.5days. the epidemic was divided to 3 periods. Simulation would run 50 times to mimic different patient0 status. Personal activity index was used to mimic different level of control measures. 141427709 simulated patients were created. '},\n"," {'paper_id': '57eb12d93057a4bc1a043420327251b2327c9b39',\n","  'section': 'Abstract',\n","  'text': 'Background: As of May 27, 2018, 54 cases of Ebola virus disease (EVD) were reported in Équateur Province, Democratic Republic of Congo. We used reported case counts and time series from prior outbreaks to estimate the current outbreak size and duration with and without vaccine use. We modeled Ebola virus transmission using a stochastic branching process model with a negative binomial distribution, using both estimates of reproduction number R declining from supercritical to subcritical derived from past Ebola outbreaks, as well as a particle filtering method to generate a probabilistic projection of the future course of the outbreak conditioned on its reported trajectory to date; modeled using 0%, 44%, and 62% estimates of vaccination coverage. Additionally, we used the time series for 18 prior Ebola outbreaks from 1976 to 2016 to parameterize a regression model predicting the outbreak size from the number of observed cases from April 4 to May 27. With the stochastic transmission model, we projected a median outbreak size of 78 EVD cases (95% credible interval: 52, 125.4), 86 cases (95% credible interval: 53, 174.3), and 91 cases (95% credible interval: 52, 843.5), using 62%, 44%, and 0% estimates of vaccination coverage. With the regression All rights reserved. No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/331447 doi: bioRxiv preprint model, we estimated a median size of 85.0 cases (95% prediction interval: 53.5, 216.6). Conclusions: This outbreak has the potential to be the largest outbreak in DRC since 2007. Vaccines are projected to limit outbreak size and duration but are only part of prevention, control, and care strategies. '},\n"," {'paper_id': '584b43ae1d473b93ad7b6c95d681e8f356ba82cd',\n","  'section': 'Abstract',\n","  'text': 'Purpose of review: Preprint servers including arXiv and bioRxiv have disrupted the scientific communication landscape by providing rapid access to pre--peer reviewed research. MedRxiv is a recently launched free online repository for preprints in the health sciences. We sought to summarize potential benefits and risks to preprint server use, from both the researcher and end--user perspective, and evaluate the uptake of preprint servers in the nephrology community. We performed a rapid review of articles describing preprint servers and their use. We approached the 20 highest impact nephrology journals regarding their policy towards the use of preprint servers. We evaluated the average time from study completion to publication of impactful articles in nephrology. Finally, we evaluated the number of nephrology articles submitted to preprint servers. Findings: To date over 600 kidney--related articles have been uploaded to bioRxiv and medRxiv. The average time from study completion to publication was over 10 months. 16 of the top 20 nephrology journals currently accept research submitted to a preprint server. Transparency and collaboration, visibility and recognition, and rapid dissemination of results were identified as benefits of preprint servers. Concerns exist regarding the potential risk of non--peer reviewed medical research being publicly available. Limitations: Preprint servers remain a recent phenomenon in health sciences and their long-term impact on the medical literature remains to be seen. Implications: The quantity of research submitted to preprint servers is likely to continue to grow. The model for dissemination of research results will need to adapt to incorporate preprint servers. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '58869cc8053741f55e3db6e1d45e758a79e3fb3f',\n","  'section': 'Abstract',\n","  'text': 'Background: Whether cardiovascular disease (CVD) and its traditional risk factors predict severe coronavirus disease 2019 is uncertain, in part, because of potential confounding by age and sex. Methods: We performed a systematic review of studies that explored pre-existing CVD and its traditional risk factors as risk factors of severe COVID-19 (defined as death, acute respiratory distress syndrome, mechanical ventilation, or intensive care unit admission). We searched PubMed and Embase for papers in English with original data (≥10 cases of severe COVID-19). Using random-effects models, we pooled relative risk (RR) estimates and conducted metaregression analyses. Results: Of the 373 publications identified in our search, 15 papers met our inclusion criteria, with 51,845 COVID-19 patients including 9,066 severe cases. Older age was consistently associated with severe COVID-19 in all eight eligible studies, with RR >~5 in >60-65 vs. <50 years. Two studies showed no change in the RR of age after adjusting for covariate(s). In univariate analyses, factors significantly associated with severe COVID-19 were male sex (14 studies; pooled RR=1.70, [95%CI 1.52-1.89]), hypertension (10 studies; 2.74 [2.12-3.54]), diabetes (11 studies; 2.81 [2.01-3.93]), and CVD (9 studies; 3.58 [2.06-6.21]). RR for male sex was likely to be independent of age. Meta-regression analyses were suggestive of confounding by age for the other three factors. Only two studies reported multivariable analysis, with one showing non-significant association for CVD and the other demonstrating adjusted RR ~2 for hypertension and diabetes. No study explored renin-angiotensin system inhibitors as a risk factor for severe COVID-19. Conclusions: In addition to older age and male sex, hypertension, diabetes, and CVD were associated in univariate analyses with severe COVID-19. Although there is still uncertainty . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10. 1101 /2020 regarding the magnitude of potential confounding, these risk factors can be used to inform objective decisions on COVID-19 testing, clinical management, and workforce planning. '},\n"," {'paper_id': '58bc0f2712ea1ced1f886c9fdd17b8cf014e66cb',\n","  'section': 'Abstract',\n","  'text': 'Infectious disease interventions like contact precautions and vaccination have proven effective in disease 7 control and elimination. The priority given to interventions can depend strongly on how virulent the pathogen 8 is, and interventions may also depend partly for their success on social processes that respond adaptively 9 to disease dynamics. However, mathematical models of competition between pathogen strains with differing 10 natural history profiles typically assume that human behaviour is fixed. Here, our objective is to model the 11 influence of social behaviour on the competition between pathogen strains with differing virulence. We couple 12 a compartmental Susceptible-Infectious-Recovered model for a resident pathogen strain and a mutant strain 13 with higher virulence, with a differential equation of a population where individuals learn to adopt protective 14 behaviour from others according to the prevalence of infection of the two strains and the perceived severity of 15 the respective strains in the population. We perform invasion analysis, time series analysis and phase plane 16 analysis to show that perceived severities of pathogen strains and the efficacy of infection control against 17 them can greatly impact the invasion of more virulent strain. We demonstrate that adaptive social behaviour 18 enables invasion of the mutant strain under plausible epidemiological scenarios, even when the mutant strain 19 has a lower basic reproductive number than the resident strain. Surprisingly, in some situations, increasing 20 the perceived severity of the resident strain can facilitate invasion of the more virulent mutant strain. Our 21 results demonstrate that for certain applications, it may be necessary to include adaptive social behaviour 22 in models of the emergence of virulent pathogens, so that the models can better assist public health efforts 23 to control infectious diseases. 24 1 . CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/293936 doi: bioRxiv preprint 25 Modern approaches to developing a theory of the spread of infectious diseases can be traced to 1927 when 26 Kermack and McKendrick developed an integro-differential equation model now widely described as the SIR 27 (Susceptible-Infected-Recovered) model [1]. The model tracks changes in the number of individuals suscep-28 tible to an infection S(t), the number of infected individuals I(t), and (implicitly) the number of recovered 29 individuals R(t). Compartmental models such as the SIR model are useful for mechanistic modelling of 30 infection transmission in populations. They have since been further developed to study the evolution and 31 epidemiology of multiple species of pathogens in a population or different strains of the same species [2]. Some models focus on between-host competition while some others on within-host competition [3] . Bull 33 suggested in the 1990s that coupling inter-host and intra-host dynamics in models may be desirable [4] . Models linking between-host transmission dynamics to within-host pathogen growth and immune response 35 are now becoming commonplace [3, 5, 6, 7, 8]. One such approach is to link host viral load (which is a 36 necessary condition of virulence) to the between-host transmission rate. Compartmental models have also been used to study the phenomenon of pathogen virulence-the rate 38 at which a pathogen induces host mortality and/or reduces host fecundity [9, 10, 11, 12]. It was initially 39 believed that hosts and parasites co-evolved to a state of commensalism (whereby parasites benefit from 40 their host without harming them) [13, 14] but this hypothesis was later challenged [15]. In mathematical 41 models, virulence is often treated as a fixed model parameter expressing the excess mortality rate caused 42 by the pathogen. For instance, virulence has been assumed to depend on the intrinsic reproductive rate 43 of the parasite [16]. Other research expresses the transmission rate β and the recovery rate µ in terms 44 of a parameter ν that represents virulence [17]. When the impact of human behaviour is discussed in 45 such models, it is discussed in terms of hypothesized effects of human behaviour on the value of the fixed 46 parameter representing virulence. A Human Immunodeficiency Virus (HIV) virulence model by Massad et 47 al [18] shows that reducing the number of sexual partners could possibly drive HIV to be a more benign 48 pathogen. However, the model assumes that the number of sexual partners can simply be moved up or down 49 as a model parameter, whereas in reality the number of sexual partners in a population is the outcome of 50 a dynamic socio-epidemiological process that merits its own mechanistic modelling, and itself responds to 51 pathogen virulence. In general, these models do not treat human behaviour as a dynamic variable that can 52 evolve in response to transmission dynamics and influence the evolution of virulence. (A few exceptions 53 exist, including work that allows virulence to be a function of the number of infected hosts, thus capturing a 54 situation where the magnitude of the epidemics affects the ability of health care services to host patients [19].) 55 However, as human responses to both endemic and emerging infectious diseases show, human behaviour can 56 2 . CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/293936 doi: bioRxiv preprint have a significant influence on how infections get transmitted. For instance, an early and well-documented 57 example shows how the residents of the village of Eyam, England quarantined themselves to prevent the 58 spread of plague to neighbouring villages [20]. Individuals moved to less populated areas during the Spanish 59 Influenza pandemic in the early 20th century [21]. More recently, masks became widely used during the 60 outbreak of the Severe Acute Respiratory Syndrome (SARS) at the beginning of the 21st century [22], and 61 it has been shown pathogen virulence in Marek\\'s disease can evolve in response to how vaccines are used 62 [23]. Theoretical models of the interactions between human behaviour and the spread of infectious diseases 64 are increasingly studied [24, 25, 26, 27, 28, 29, 30]. For instance, Bagnoli et al [31] found that under 65 certain conditions, a disease can be driven extinct by reducing the fraction of the infected neighbours of an 66 individual. Zanette et al [32] showed that if susceptible individuals decide to break their links with infected 67 agents and reconnect at a later time, then the infection is suppressed. Gross [33] also shows that rewiring of 68 edges in a network (and thus social interaction) can greatly influence the spread of infectious diseases. Of 69 the compartmental models, we focus on those that have used concepts from evolutionary game theory such 70 as imitation dynamics [34] to describe the evolution of behaviour and its interplay with the epidemics. An 71 example of imitation dynamics concerns, as described in detail in [35], the effect of vaccination on the spread 72 of infectious diseases. Each individual in the population picks one strategy and adopts it: \"to vaccinate\" or 73 \"not to vaccinate\". The proportion of vaccinators is modelled using an ordinary differential equation and 74 is coupled with a standard SIR model. An important aspect of behavioural models is to couple them with 75 epidemiological processes such as transmission. This coupling creates a feedback loop between behaviour 76 and spread of the disease. 77 Given that adaptive social behaviour is important in many aspects of infection transmission, we hy-78 pothesize that adaptive social behaviour can also influence selection between pathogen strains with differing 79 virulence in ways that cannot be captured by assuming it to be represented by a fixed parameter. Our 80 objective in this paper was to explore how behaviour and virulence influence one another, in a coupled 81 behaviour-disease differential equation model. The model allows individuals who perceive an increase in 82 the prevalence of infection to increase their usage of practices that reduce transmission rates (such as social 83 distancing and hand-washing) and thereby boost population-level immunity. This approach can help us 84 understand the effects specific social dimensions, such as level of concern for a strain or the rate of social 85 learning, on the coupled dynamics of pathogen strain emergence and human behaviour in a situation where 86 virulence imposes evolutionary trade-offs and is strain-specific. Instead of considering long-term evolutionary 87 processes with repeated rounds of mutation and selection, we focus on the case of invasion of a single mutant 88 strain with a large phenotypical difference compared to the resident strain. In the next section, we describe 89 3 . CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/293936 doi: bioRxiv preprint 101 due to infection (virulence); µ is a birth rate and δ is the background death rate. All variables represent 102 the number of individuals with the given infection status (for instance, S is the number of susceptible 103 individuals). Since R does not appear in the other equations, we can omit R from the analysis. The system of differential equations in the presence of adaptive human behaviour couples the SI 1 I 2 R 105 epidemic spread with a differential equation for human behaviour (\"coupled model\"). Each individual in the 106 population can choose to accept or reject behaviours that reduce infection risk (e.g. washing hands, wearing 107 a mask, social distancing), and individuals imitate successful strategies observed in others. Let x represent 108 the proportion of individuals accepting preventive behaviour (we will call these \"protectors\"). Individuals 109 sample others in the population at rate κ, representing social learning. The choice is based on the perceived 110 severity ω 1 (resp. ω 2 ) from the resident (resp. mutant) strain, where ω 1 (resp. ω 2 ) can be quantified as the 111 probability that an infection by the resident (resp. mutant) strain results in a severe case of disease. The 112 more severe cases the population observes, the more attractive preventive behaviour becomes: we assume '},\n"," {'paper_id': '58fd342d5c70641bd4e72aa822f0e532c0127ca3',\n","  'section': 'Abstract',\n","  'text': 'The ongoing pandemic of the coronavirus disease 2019 is an infectious disease caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV2). We have performed an integrated sequence-based analysis of SARS-CoV2 genomes from different geographical locations in order to identify its unique features absent in SARS-CoV and other related coronavirus family genomes, conferring unique infection, facilitation of transmission, virulence and immunogenic features to the virus. The phylogeny of the genomes yields some interesting results. Systematic gene level mutational analysis of the genomes has enabled us to identify several unique features of the SARS-CoV2 genome, which includes a unique mutation in the spike surface glycoprotein (A930V (24351C>T)) in the Indian SARS-CoV2, absent in other strains studied here. We have also predicted the impact of the mutations in the spike glycoprotein function and stability, using computational approach. To gain further insights into host responses to viral infection, we predict that antiviral host-miRNAs may be controlling the viral pathogenesis. Our analysis reveals nine host miRNAs which can potentially target SARS-CoV2 genes. Interestingly, the nine miRNAs do not have targets in SARS and MERS genomes. Also, hsa-miR-27b is the only unique miRNA which has a target gene in the Indian SARS-CoV2 genome. We also predicted immune epitopes in the genomes '},\n"," {'paper_id': '5913eb54f45440b5b524bf9bea5fc8b47115e825',\n","  'section': 'Abstract',\n","  'text': 'The recent emergence of the pathogenic SARS-CoV-2 initiated a worldwide health crisis. The entry of the virus into cells is mediated by the binding of the viral Spike protein to the angiotensin-converting enzyme-2 (ACE2), followed by its priming by the TMPRSS2 serine protease, both present on the cellular membrane of the target cells. In the respiratory tract, these targets are ciliated cells. Interestingly, various reports indicate an association between SARS-CoV-2 infection and anosmia, suggesting an alteration not restricted to the respiratory tissue, but that might also include the olfactory sensory epithelium. We explored this possibility by generating RNA-seq libraries from human neuroepithelium, in which we found significant expression of ACE2 and TMPRSS2. To determine whether specific cell types of this chemosensory tissue may coexpress both of the virus entry genes, we analyzed a scRNA-seq dataset. We determined that sustentacular cells, which are in direct contact with the external world and maintain the integrity of olfactory sensory neurons, represents a prime candidate for SARS-CoV-2 infection via the nose, and possibly for SARS-CoV-2-induced anosmia. author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '594d23a412649df7dbfef5347a2a0c208ed4417d',\n","  'section': 'Abstract',\n","  'text': 'Background The outbreak of COVID-2019 is becoming a global public health '},\n"," {'paper_id': '598b8f280571ace476ad56a469e7959918fa7958',\n","  'section': 'Abstract',\n","  'text': '22 '},\n"," {'paper_id': '5998e7fb99b5c58f7563017ad679fbe2f9974a0c',\n","  'section': 'Abstract',\n","  'text': 'To determine the relative impact of the use of chloroquine and hydroxychloroquine on outcomes important to patients with COVID 19. This is the protocol of a living systematic review. We will follow a common protocol for multiple parallel systematic reviews, already published and submitted to PROSPERO (awaiting ID allocation). We will include randomised controlled trials evaluating the effect of chloroquine and hydroxychloroquine -as monotherapy or in combination with other drugs -versus placebo or no treatment in patients with COVID-19. Randomised trials evaluating chloroquine and hydroxychloroquine in infections caused by other coronaviruses, such as MERS-CoV and SARS-CoV, and non-randomised studies in COVID-19 will be searched in case no direct evidence from randomised trials is found, or if the direct evidence provides low-or very low-certainty for critical outcomes. Two reviewers will independently screen each study for eligibility, extract data, and assess the risk of bias. We will perform random-effects meta-analyses and use GRADE to assess the certainty of the evidence for each outcome. '},\n"," {'paper_id': '59b585747f7b8fe57d10f2c7504cd7cd38bd87c5',\n","  'section': 'Abstract',\n","  'text': 'We propose a novel numerical method able to determine efficiently and effectively the relationship of complementarity between portions of protein surfaces. This innovative and general procedure, based on the representation of the molecular iso-electron density surface in terms of 2D Zernike polynomials, allows the rapid and quantitative assessment of the geometrical shape complementarity between interacting proteins, that was unfeasible with previous methods. We first tested the method with a large dataset of known protein complexes obtaining an overall area under the ROC curve of 0.76 in the blind recognition of binding sites and then applied it to investigate the features of the interaction between the Spike protein of SARS-CoV-2 and human cellular receptors. Our results indicate that SARS-CoV-2 uses a dual strategy: its spike protein could also interact with sialic acid receptors of the cells in the upper airways, in addition to the known interaction with Angiotensin-converting enzyme 2. '},\n"," {'paper_id': '59eab95c43fdea01481fdbf9bae45dfe28ffc693',\n","  'section': 'Abstract',\n","  'text': \"In current severe global emergency situation of 2019-nCov outbreak, it is imperative to identify vulnerable and susceptible groups for effective protection and care. Recently, studies found that 2019-nCov and SARS-nCov share the same receptor, ACE2. In this study, we analyzed five large-scale bulk transcriptomic datasets of normal lung tissue and two single-cell transcriptomic datasets to investigate the disparities related to race, age, gender and smoking status in ACE2 gene expression and its distribution among cell types. We didn't find significant disparities in ACE2 gene expression between racial groups (Asian vs Caucasian), age groups (>60 vs <60) or gender groups (male vs female). However, we observed significantly higher ACE2 gene expression in former smoker's lung compared to non-smoker's lung. Also, we found higher ACE2 gene expression in Asian current smokers compared to non-smokers but not in Caucasian current smokers, which may indicate an existence of gene-smoking interaction. In addition, we found that ACE2 gene is expressed in specific cell types related to smoking history and location. In bronchial epithelium, ACE2 is actively expressed in goblet cells of current smokers and club cells of non-smokers. In alveoli, ACE2 is actively expressed in remodelled AT2 cells of former smokers. Together, this study indicates that smokers especially former smokers may be more susceptible to 2019-nCov and have infection paths different with non-smokers. Thus, smoking history may provide valuable information in identifying susceptible population and standardizing treatment regimen. \"},\n"," {'paper_id': '59ebacae3048d5db81d51968371b6350658e8b94',\n","  'section': 'Abstract',\n","  'text': \"Since the new coronavirus epidemic in China in December 2019, information and discussions about COVID-19 have spread rapidly on the Internet and have quickly become the focus of worldwide attention, especially on social media. and described the trend of public attention on COVID-19 epidemic-related topics. ROST CM6.0 (ROST Content Mining System Version 6.0) was used to analyze the collected text for word segmentation, word frequency, and sentiment analysis. We further described the hot topic keywords and sentiment trends of public attention. We used VOSviewer to implement a visual cluster analysis of hot keywords and build a social network of public opinion content. The study has four main findings. First, we analyzed the changing trend of the public's attention to the COVID-19 epidemic, which can be divided into three stages. Second, the hot topic keywords of public attention at each stage are . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10. 1101 /2020 slightly different. In addition, the emotional tendency of the public toward the COVID-19 epidemic-related hot topics has changed from negative to neutral, with negative emotions weakening and positive emotions increasing as a whole. Finally, we divided the COVID-19 topics with the most public concern into five categories: (1) new COVID-19 epidemics and their impact; (2) frontline reporting of the epidemic and prevention and control measures; (3) expert interpretation and discussion on the source of infection; (4) medical services on the frontline of the epidemic; and (5) focus on the global epidemic and the search for suspected cases. Conclusions: This is the first study of public attention on the COVID-19 epidemic using a Chinese social media platform (i.e., Sina Microblog). Our study found that social media (e.g., Sina Microblog) can be used to measure public attention to public health emergencies. During the epidemic of the novel coronavirus, a large amount of information about the COVID-19 epidemic was disseminated on Sina Microblog and received widespread public attention. We have learned about the hotspots of public concern regarding the COVID-19 epidemic. These findings can help the government and health departments better communicate with the public on health and translate public health needs into practice to create targeted measures to prevent and control the spread of COVID-19. \"},\n"," {'paper_id': '5a0089eb2fc6fc7eabc5f98b7216e1568967b7c7',\n","  'section': 'Abstract',\n","  'text': 'The overall objective of this rapid review was to identify infection protection and control recommendations from published clinical practice guidelines (CPGs) for adults aged 60 years and older in long-term care settings Methods: Comprehensive searches in MEDLINE, EMBASE, the Cochrane Library, and relevant CPG publishers/repositories were carried out in early March 2020. Title/abstract and full-text screening, data abstraction, and quality appraisal (AGREE-II) were carried out by single reviewers. '},\n"," {'paper_id': '5a51f0a5a8d59d0e6598256bbb2a411196bd2d72',\n","  'section': 'Abstract',\n","  'text': ' 33  '},\n"," {'paper_id': '5a5a7b6d40c4b752aabf69f69bbd7c71e2658e8e',\n","  'section': 'Abstract',\n","  'text': 'China reported zero locally-acquired malaria cases in 2017 and 2018. Understanding the spatio-9 temporal pattern underlying this decline, especially the relationship between locally-acquired and 10 imported cases, can inform efforts to maintain elimination and prevent re-emergence. This is 11 particularly pertinent in Yunnan province, where the potential for local transmission is highest. Using 12 a geo-located individual-level dataset of cases recorded in Yunnan province between 2011 and 2016, 13 we jointly estimate the case reproduction number, , and the number of unobserved sources of 14 infection. We use these estimates within spatio-temporal geostatistical models to map how 15 transmission varied over time and space, estimate the timeline to elimination and the risk of 16 resurgence. Our estimates suggest that, maintaining current intervention efforts, Yunnan is unlikely 17 to experience sustained local transmission up to 2020. However, even with a mean of 0.005 18 projected for the year 2019, locally-acquired cases are possible due to high levels of importation. '},\n"," {'paper_id': '5a843f4299b04090ff76451decfe9f0b025cfe48',\n","  'section': 'Abstract',\n","  'text': 'Metagenomic sequencing has the potential to transform microbial detection and characterization, but new tools are needed to improve its sensitivity. We developed CATCH (Compact Aggregation of Targets for Comprehensive Hybridization), a computational method to enhance nucleic acid capture for enrichment of diverse microbial taxa. CATCH designs compact probe sets that achieve full coverage of known sequence diversity and that scale well with this diversity. To illustrate applications of CATCH, we focused on capturing viral genomes. We designed, synthesized, and validated multiple probe sets, including one that targets whole genomes of the 356 viral species known to infect humans. Capture with these probe sets enriched unique viral content on average 18× and allowed us to assemble genomes that we could not otherwise recover, while accurately preserving within-sample diversity. We used this approach to recover genomes from the 2018 Lassa fever outbreak in Nigeria and to improve detection of viral infections in samples with unknown content. Together, this work demonstrates a path toward more sensitive, cost-effective metagenomic sequencing. '},\n"," {'paper_id': '5a9f31802c72069630e3cb55d014ffb8e8b362d3',\n","  'section': 'Abstract',\n","  'text': \"This paper presents a stochastic optimization model for allocating and sharing a critical resource in the case of a pandemic. The demand for different entities peaks at different times, and an initial inventory from a central agency is to be allocated. The entities (states) may share the critical resource with a different state under a risk-averse condition. The model is applied to study the allocation of ventilator inventory in the COVID-19 pandemic by the U.S. Department of Homeland Security, Federal Emergency Management Agency (FEMA) to different states in the US. Findings suggest that if less than 60% of the ventilator inventory is available for non-COVID-19 patients, FEMA's stockpile of 20,000 ventilators (as of 03/23/2020) would be nearly adequate to meet the projected needs. However, when more than 75% of the available ventilator inventory must be reserved for non-COVID-19 patients, various degrees of shortfall are expected. In an extreme case, where the demand is assumed to be concentrated in the top-most quartile of the forecast confidence interval, the total shortfall over the planning horizon (till 05/31/20) is about 28,500 ventilator days, with a peak shortfall of 2,700 ventilators on 04/12/20. The results also suggest that in the worse-than-average to severe demand scenario cases, NY requires between 7,600-9,200 additional ventilators for COVID-19 patients during its peak demand. However, between 400 to 2,000 of these ventilators can be given to a different state after the peak demand is subsided. \"},\n"," {'paper_id': '5aa0306fa361d31085f6df03d7e71307534b34c6',\n","  'section': 'Abstract',\n","  'text': 'The epidemic caused by SARS-CoV-2 was first reported in Wuhan, China, and now is spreading worldwide. The Chinese government responded to this epidemic with multiple public health policies including locking down the city of Wuhan, establishing multiple temporary hospitals, and prohibiting public gathering events. Here, we constructed a new real-time status dynamic model of SEIO (MH) to reveal the influence of national public health policies and to model the epidemic in Wuhan. '},\n"," {'paper_id': '5b11e063f09adbfdaf7fd515ecc6c09a83674263',\n","  'section': 'Abstract',\n","  'text': '14 '},\n"," {'paper_id': '5b1a2b3207e3f674b94f6f271b92c29f3a8a6754',\n","  'section': 'Abstract',\n","  'text': 'Objective: To investigate the correlation between clinical characteristics and cardiac injury of COVID-2019 pneumonia. '},\n"," {'paper_id': '5b25c63b89586e93a09b5e213a20cfd9b0418b60',\n","  'section': 'Abstract',\n","  'text': '166 countries/regions, including cases of human-to-human transmission around the world. The proportions of this epidemics is probably one of the largest challenges faced by our interconnected modern societies. According to the current epidemiological reports, the large basic reproduction number, R0 ∼ 2.3, number of secondary cases produced by an infected individual in a population of susceptible individuals, as well as an asymptomatic period (up to 14 days) in which infectious individuals are undetectable without further analysis, pave the way for a major crisis of the national health capacity systems. Recent scientific reports have pointed out that the detected cases of COVID19 at young ages is strikingly short and that lethality is concentrated at large ages. Here we adapt a Microscopic Markov Chain Approach (MMCA) metapopulation mobility model to capture the spread of COVID-19. We propose a model that stratifies the population by ages, and account for the different incidences of the disease at each strata. The model is used to predict the incidence of the epidemics in a spatial population through time, permitting investigation of control measures. The model is applied to the current epidemic in Spain, using the estimates of the epidemiological parameters and the mobility and demographic census data of the national institute of statistics (INE). The results indicate that the peak of incidence will happen in the first half of April 2020 in absence of mobility restrictions. These results can be refined with improved estimates of epidemiological parameters, and can be adapted to precise mobility restrictions at the level of municipalities. The current estimates largely compromises the Spanish health capacity system, in particular that for intensive care units, from the end of March. However, the model allows for the scrutiny of containment measures that can be used for health authorities to forecast with accuracy their impact in prevalence of COVID-19. Here we show by testing different epidemic containment scenarios that we urge to enforce total lockdown to avoid a massive collapse of the Spanish national health system. '},\n"," {'paper_id': '5b27a16123f46b5aaeda3171d22e075dcbd36ee6',\n","  'section': 'Abstract',\n","  'text': 'It has been recently estimated that one-third of the world population will be infected by Zika virus, but unfortunately so far there is no vaccine or medicine available. In particular, the special concern on the vaccine treatment to Zika and Dengue arising from antibody-dependent enhancement strongly emphasizes the irreplaceable role of its NS2B-NS3 protease (NS2B-NS3pro) as a target for anti-Zika drug discovery/design due to its absolutely-essential role in viral replication. Very recently we identified two small molecules inhibit Zika NS2B-NS3pro in non-competitive mode, with Ki values of 0.57 and 2.02 µM respective for p-Nitrophenyl-p-guanidino benzoate and qucertin. Here, by molecular docking, we show that although one is designed compound while another is a natural product, both molecules bind to the same pocket on the back of the substratebinding pocket of Zika NS2B-NS3pro. As the two inhibitors fundamentally differ from cn-716, the only known peptidomimetic boronic acid inhibitor in both structure scaffolds and inhibitory modes, our discovery might open up a new avenue for the future development of allosteric inhibitors, which is highly demanded to achieve therapeutic inhibition of flaviviral NS2B-NS3pro complexes. Furthermore, as qucertin is abundant in many vegetables and fruits such caper, lovage, tea and red onion, our results should benefit the public to immediately fight Zika virus. author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '5b53db319389827d3bb4940c2cc3d1e35289691b',\n","  'section': 'Abstract',\n","  'text': 'Objective: many potential factors contribute to the outbreak of COVID-19. The aim of this study was to explore the effects of various meteorological factors on the incidence of COVID-19. Taking Hubei province of China as an example, where COVID-19 was first reported and there were the most cases, we collected 53 days of cases up to March 10(total 67773 confirmed cases).COVID-19 confirmed cases were retrieved from the official website of Hubei Health Commission. Ten meteorological parameters were provided by China meteorological administration, including average pressure (hPa), average temperature ( ), maximum temperature, minimum temperature ( ), average water vapor pressure (hPa), average relative humidity (%),etc. Cross correlation analysis and linear regression were used to judge the relationship of . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.04.01.20050526 doi: medRxiv preprint meteorological factors and increment of COVID-19 confirmed cases. Under 95% CI, the increment of confirmed cases in Hubei were significantly correlated with four meteorological parameters of average pressure, average temperature, minimum temperature and average water vapor pressure (equivalent to absolute humidity).The average pressure was positively correlated with the increment ( r=+0.358,p=0.010).The negative correlations included average temperature (r=-0.306,p=0.029), minimum air temperature (r=-0.347,p=0.013), average water vapor pressure (r=-0.326,p=0.020). The linear regression results show if minimum temperature increases by 1 , the incremental confirmed cases in Hubei decreases by 72.470 units on average. The incidence of COVID-19 was significantly correlated with average pressure, average temperature, minimum temperature and average water vapor pressure. It is positively correlated with the average pressure and negatively correlated with the other three parameters. Compared with relative humidity, 2019-nCov is more sensitive to water vapor pressure. The reason why the epidemic situation in Hubei expanded rapidly is significantly related to the climate characteristics of low temperature and dryness of Hubei in winter. '},\n"," {'paper_id': '5b9a15ad62b0e85fcb5012e7e08e5183b4da786b',\n","  'section': 'Abstract',\n","  'text': \"25 26 27 INTRODUCTION: Epidemic forecasting and prediction tools have the potential to provide 28 actionable information in the midst of emerging epidemics. While numerous predictive studies 29 were published during the 2016-2017 Zika Virus (ZIKV) pandemic, it remains unknown how 30 timely, reproducible and actionable the information produced by these studies was. METHODS: 31 To improve the functional use of mathematical modeling in support of future infectious disease 32 outbreaks, we conducted a systematic review of all ZIKV prediction studies published during the 33 recent ZIKV pandemic using the PRISMA guidelines. Using MEDLINE, EMBASE and grey 34 literature review, we identified studies that forecasted, predicted or simulated ecological or 35 epidemiological phenomenon related to the Zika pandemic that were published as of March 01, 36 2017. Eligible studies underwent evaluation of objectives, data sources, methods, timeliness, 37 reproducibility, accessibility and clarity by independent reviewers. RESULTS: 2034 studies were 38 identified, of which n = 73 met eligibility criteria. Spatial spread, R 0 (basic reproductive number) 39 and epidemic dynamics were most commonly predicted, with few studies predicting Guillain-40 Barré Syndrome burden (4%), sexual transmission risk (4%) and intervention impact (4%). Most 41 studies specifically examined populations in the Americas (52%), with few African-specific 42 studies (4%). Case count (67%), vector (41%) and demographic data (37%) were the most 43 common data sources. Real-time internet data and pathogen genomic information were used in 44 7% and 0% of studies, respectively, and social science and behavioral data were typically absent 45 in modeling efforts. Deterministic models were favored over stochastic approaches. Forty 46 percent of studies made model data entirely available, 29% provided all relevant model code, 47 43% presented uncertainty in all predictions and 54% provided sufficient methodological detail 48 allowing complete reproducibility. Fifty-one percent of predictions were published after the 49 epidemic peak in the Americas. While the use of preprints improved the accessibility of ZIKV 50 predictions by a median 119 days sooner than journal publication dates, they were used in only 51 30% of studies. CONCLUSIONS: Many ZIKV predictions were published during the 2016-2017 52 pandemic. The accessibility, reproducibility, timeliness, and incorporation of uncertainty in 53 these published predictions varied and indicates that there is substantial room for 54 improvement. To enhance the utility of analytical tools for outbreak response, it is essential to 55 improve the sharing of model data, code, and preprints for future outbreaks, epidemics and 56 pandemics. 57 58 Author summary: Researchers published many studies which sought to predict and forecast 59 important features of Zika virus (ZIKV) infections and their spread during the 2016-2017 ZIKV 60 pandemic. We conducted a comprehensive review of such ZIKV prediction studies and 61 evaluated their aims, the data sources they used, which methods were used, how timely they 62 were published, and whether they provided sufficient information to be used or reproduced by 63 others. Of the 73 studies evaluated, we found that the accessibility, reproducibility, timeliness, 64 and incorporation of uncertainty in these published predictions varied and indicates that there 65 is substantial room for improvement. We identified that the release of study findings before 66 formal journal publication ('pre-prints') increased the timeliness of Zika prediction studies, but \"},\n"," {'paper_id': '5ba386b58c8d194e8cbe581eed95f7c64d05cd53',\n","  'section': 'Abstract',\n","  'text': \"The ability to rapidly assemble and prototype cellular circuits is vital for biological research and its applications in biotechnology and medicine. Current methods that permit the assembly of DNA circuits in mammalian cells are laborious, slow, expensive and mostly not permissive of rapid prototyping of constructs. Here we present the Mammalian ToolKit (MTK), a Golden Gate-based cloning toolkit for fast, reproducible and versatile assembly of large DNA vectors and their implementation in mammalian models. The MTK consists of a curated library of characterized, modular parts that can be easily mixed and matched to combinatorially assemble one transcriptional unit with different characteristics, or a hierarchy of transcriptional units weaved into complex circuits. MTK renders many cell engineering operations facile, as showcased by our ability to use the toolkit to generate single-integration landing pads, to create and deliver libraries of protein variants and sgRNAs, and to iterate through Cas9-based prototype circuits. As a biological proof of concept, we used the MTK to successfully design and rapidly construct in mammalian cells a challenging multicistronic circuit encoding the Ebola virus (EBOV) replication complex. This construct provides a non-infectious biosafety level 2 (BSL2) cellular assay for exploring the transcription and replication steps of the EBOV viral life cycle in its host. Its construction also demonstrates how the MTK can enable important and time sensitive applications such as the rapid testing of pharmacological inhibitors of emerging BSL4 viruses that pose a major threat to human health. and Extensible Mammalian Modular Assembly kit (EMMA) 3 based on Golden Gate assembly reactions were proposed. Both methods use sets of modular parts with given functionality, which can be assembled directionally into transcription units (TUs) by type-IIS restriction enzymes. While useful, all three methods have important shortcomings. GMAP requires sequencing in between all cloning steps, which makes the method slow and expensive. EMMA and mMoClo provide a method to hierarchically assemble DNA vectors that encode large circuits, but fall short on shareability, integration of Cas9 sgRNA cloning, a library of tested parts, and ease of use. While in yeast and plant systems, methods for the rapid, efficient and hierarchical assembly of genetic circuits from a large library of well described modular parts [8] [9] [10] [11] exist, we are unaware that such a cloning toolkit for mammalian systems is readily available. In this work, we adapted the modular cloning strategy (MoClo) 12 strategy to build mammalian expression vectors. The Mammalian ToolKit (MTK) is a library of over 300 parts including vetted promoters, 3'UTRs, fluorescent proteins, insulator and P2A elements that can be combined to build large genetic circuits in a rapid and efficient fashion. These vectors can be delivered to a wide range of cell types, through viral, recombinase and CRISPR/Cas9 methods. As a proof of concept, we built a hAAVS1 landing pad for single integration of genetic circuits, created and delivered libraries of protein variants and sgRNAs and rapidly compared multi-species Cas9-based genetic circuits. Finally, we used Ebola virus (EBOV) as a test case to address an emerging need to streamline the generation of BSL2 reagents during outbreaks of novel BSL4 viruses. Using the MTK, we rapidly built a non-infectious multicistronic ribonucleoprotein expression system for EBOV in a mammalian host. This system recapitulates the transcription and replication sub-lifecycle of the virus in a mammalian host, and can therefore be used to accelerate the study of viral biology and discovery of antiviral molecules. \"},\n"," {'paper_id': '5bb41c4446cd4096b2bd85f44b00a7475ad752da',\n","  'section': 'Abstract',\n","  'text': \"Background: The spread of an novel coronavirus (SARS-CoV-2, previously named 2019-nCoV) has already taken on pandemic proportions, affecting over 100 countries in a matter of weeks. Elucidating the diagnostic value of different methods, especially the auxiliary diagnosis value of antibodies assays for SARS-CoV-2 infection is helpful for improving the sensitivities of pathogenic-diagnosis, providing timely treatment, and differentiating the infected cases from the healthy, thus preventing further epidemics. Methods: Medical records from 38 patients with confirmed SARS-CoV-2 infection in the Second People's Hospital of Fuyang from January 22, 2020 to February 28, 2020 were collected and retrospectively analyzed. Specimens including throat swabs, sputum and serum were collected during the hospitalization period, viral RNAs and serum IgM-IgG antibodies to SARS-CoV-2 were measured respectively. The detectability of different methods as well as the auxiliary diagnosis value of antibodies test for SARS-CoV-2 infection were analyzed. Results: Among 38 patients, the total seropositive rate for IgM and IgG was 50.0% and 92.1%, respectively. Two patients remained seronegative throughout the course of illness. In the early phase of illness, the RNA test for sputum specimens possessed the highest detectability(92.3%), followed by the the RNA test for throat swabs (69.2%), and the antibodies assays presented lower positive rates(IgM, 23.0%, IgG, 53.8%). While, the sensitivity of antibodies assays overtook that of RNA test since day 8 after onset (IgM, 50.0%; IgG, 87.5%). Of note, the positive rate of throat swabs was only 13.0% for cases in later phase(≥15 d.a.o), and the sensitivities of IgM and IgG rose to 52.2% and 91.3%, respectively. Combined use of antibodies assay and qRT-PCR at the same time was able to improve the sensitivities of pathogenic-diagnosis, especially for the throat swabs group at the later stage of illness. Moreover, most of these cases with undetectable viral RNA in throat swabs specimens at the early stage of illness were able to be IgM/IgG seropositive after 7 days. The antibodies detection against SARS-CoV-2 offers vital clinical information for physicians, and could be used as an effective supplementary indicator . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not peer-reviewed) The copyright holder for this preprint . \"},\n"," {'paper_id': '5bb89950ec5a06e2b7f69b2a9c4213dda19b1ab0',\n","  'section': 'Abstract',\n","  'text': 'BACKGROUND The outbreak of the new coronavirus infection in Wuhan City, Hubei Province in December 2019, poses a huge threat to China and even global public health security. Respiratory droplets and contact transmission are the main routes of transmission of new coronaviruses. Compared with SARS and Ebola viruses, new coronavirus infections are infectious during the incubation period. Traditional SEIR (susceptibility-exposure-infection-Removal) There are some differences in conditions for the prediction of the epidemic trend of new coronavirus infection. The outbreak of the new coronavirus infection coincided with the Spring Festival before and after the Chinese Spring Festival.It is necessary to make appropriate optimization and amendments to the traditional model to meet the actual evolution of the epidemic situation. The traditional SEIR model assumes that the virus-infected person is not infectious during the incubation period and that the infected person did not take isolation measures during the illness. The transmission of the new coronavirus no longer meets the basic assumptions of the classical kinetic system. Therefore, this article first establishes a modified SEIR model. Predict and analyze the changing trend of the epidemic situation, then estimate the parameters involved in the infection dynamics model, and then use Matlab to simulate the established dynamic equations based on public data and analyze the results. Recommendations for universal prevention and control of infectious diseases. . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . : medRxiv preprint 2 Wuhan City took no action, assuming the average daily number of contacts per infected person k = 5, the number of infected persons will reach about 2,384,803 people; If wuhan adopts the measures of sealing the city on January 22, 2020, under the premise of k=2, the number of infected people decreases by 19,773 compared with that on January 23, and there is no significant change in the time when the number of infected people reaches the peak. Under the premise of k = 1, the number of infected persons was reduced by 14,330 compared with the closure on January 23, and the time to reach the peak of the number of infected persons was reduced by 2 days. If Wuhan City is closed for one day, the number of infected persons will increase from 106,145 to 130,626 under the premise of k = 2; the number of infected persons will increase from 74,369 to 92,010 under the premise of k = 1. Comparing the number of confirmed diagnoses actually notified by the department with the number of infected people obtained from the simulation of the model, it can be seen that the city closure measures adopted by the Wuhan Municipal Government on January 23 and the first-level response measures adopted by the country are effective for the epidemic Prevention and control play a vital role. Wearing a mask when going out and avoiding close contact with people can effectively reduce the infection rate. '},\n"," {'paper_id': '5bbc428de7fc2f10c24d8439edda26d84b319687',\n","  'section': 'Abstract',\n","  'text': 'medRxiv preprint Background: The 2019-nCoV outbreak in Wuhan, China has attracted world-wide attention. As of February 11, 2020, a total of 44730 cases of novel coronavirus-infected pneumonia associated with COVID-19 were confirmed by the National Health Commission of China. Methods: Three approaches, namely Poisson likelihood-based method (ML), exponential growth rate-based method (EGR) and stochastic Susceptible-Infected-Removed dynamic model-based method (SIR), were implemented to estimate the basic and controlled reproduction numbers. Results: A total of 71 chains of transmission together with dates of symptoms onset and 67 dates of infections were identified among 5405 confirmed cases outside Hubei as reported by February 2, 2020. Based on this information, we find the serial interval having an average of 4.41 days with a standard deviation of 3.17 days and the infectious period having an average of 10.91 days with a standard deviation of 3.95 days. The controlled reproduction number is declining. It is lower than one in most regions of China, but is still larger than one in Hubei Province. Sustained efforts are needed to further reduce the ܴ to below one in order to end the current epidemic. '},\n"," {'paper_id': '5c31897d01f3edc7f58a0f03fceec2373fcfdc3d',\n","  'section': 'Abstract',\n","  'text': 'Motivated by the rapid spread of a novel coronavirus (2019-nCoV) in Mainland China, we use a global metapopulation disease transmission model to project the impact of both domestic and international travel limitations on the national and international spread of the epidemic. The model is calibrated on the evidence of internationally imported cases before the implementation of the travel quarantine of Wuhan. By assuming a generation time of 7.5 days, the reproduction number is estimated to be 2.4 [90% CI 2.2-2.6]. The median estimate for number of cases before the travel ban implementation on January 23, 2020 is 58, 956 [90% CI 40,471] in Wuhan and 3,491 [90% CI 1,924 -7,360] in other locations in Mainland China. The model shows that as of January 23, most Chinese cities had already received a considerable number of infected cases, and the travel quarantine delays the overall epidemic progression by only 3 to 5 days. The travel quarantine has a more marked effect at the international scale, where we estimate the number of case importations to be reduced by 80% until the end of February. Modeling results also indicate that sustained 90% travel restrictions to and from Mainland China only modestly affect the epidemic trajectory unless combined with a 50% or higher reduction of transmission in the community. '},\n"," {'paper_id': '5c525cc04075b7c82e52b1e80d96bb394b1a0907',\n","  'section': 'Abstract',\n","  'text': 'Any plan for stopping the ongoing 2019-nCov epidemic must be based on a quantitative understanding of the proportion of the at-risk population that needs to be protected by effective control measures in order for transmission to decline sufficiently and quickly enough for the epidemic to end. Using an SEIR-type transmission model, we contrasted two alternate strategies by modeling the proportion of the population that needs to be protected from infection by onetime vaccination (assuming 100% effectiveness) or by testing with isolation and treatment of individuals within six, 24, or 48 hours of symptom onset. If R is currently 2.2, vaccination at the herd immunity coverage of 55% would drive R just below 1, but transmission could persist for years. Over 80% of coverage is required to end the epidemic in 6 months with population-wide vaccination. The epidemic could be ended in just under a year if testing with isolation and treatment reached 80% of symptomatically infected patients within 24 hours of symptom onset (assuming 10% asymptomatic transmission). The epidemic could be ended in six months if testing with isolation and treatment reached 90% of symptomatic patients. If 90% of symptomatic patients could be tested within six hours of symptoms appearing, the epidemic could be ended in under four months. '},\n"," {'paper_id': '5cb61edbbbed7a03b791fe1628e1d68577a9a980',\n","  'section': 'Abstract',\n","  'text': 'I employ a simple mathematical model of an epidemic process to evaluate how three basic quantities: the reproduction number (R), the number of infectious individuals (I), and total community size (N) affect strategies to control COVID-19. Numerical simulations show that strict suppression measures at the beginning of an epidemic can create low infectious numbers, which thereafter can be managed by mitigation measures over longer periods to flatten the epidemic curve. The stronger the suppression measure, the faster it achieves the low levels of exposed and infectious numbers that are conducive to subsequent management. Our results point to a two-step control strategy that begins with some level of confinement to reduce R below 1, followed by sufficient mitigation measures that manage the epidemic by maintaining R at approximately 1. '},\n"," {'paper_id': '5cd48b8ab4ef66ca582df97a293d22a5088fa568',\n","  'section': 'Abstract',\n","  'text': 'The current outbreak of Coronavirus Disease 2019 (COVID-19) by a novel 28 betacoronavirus severe acute respiratory syndrome coronavirus 2 (SARS-29 CoV-2) has aroused great public health concern. Coronavirus has a history of 30 causing epidemics in human and animals. In 2017 an outbreak in piglets by a 31 novel coronavirus was emerged designated as swine acute diarrhea 32 syndrome coronavirus (SADS-CoV) which is originated from the same genus 33 of horseshoe bats (Rhinolophus) as Severe Acute Respiratory Syndrome CoV 34 (SARS-CoV) having a broad species tropism. In addition to human cells, it 35 can also infect cell lines from diverse species. Coronavirus host range is 36 determined by its spike glycoprotein (S). Given the importance of S protein in 37 viral entry to cells and host immune responses, here we report the cryo-EM 38 structure of the SADS-CoV S in the prefusion conformation at a resolution of 39 3.55 Å. Our study reveals that SADS-CoV S structure takes an intra-subunit 40 quaternary packing mode where the NTD and CTD from the same subunit 41 pack together by facing each other. The comparison of NTD and CTD with 42 that of the other four genera suggests the evolutionary process of the SADS-43 CoV S. Moreover, SADS-CoV S has several characteristic structural features, 44 such as more compact architecture of S trimer, and masking of epitopes by 45 glycan shielding, which may facilitate viral immune evasion. These data 46 provide new insights into the evolutionary relationships of SADS-CoV S and 47 would extend our understanding of structural and functional diversity, which 48 . '},\n"," {'paper_id': '5cf2bec1530cef38cd0eea173fb90ba1a788e11f',\n","  'section': 'Abstract',\n","  'text': \"The basal ganglia and thalamus play an important role in cognition, procedural learning, eye movements, control of voluntary motor movements, emotional control, habit development, and are structures that are severely impacted by neurological disorders such as Alzheimer's disease, Parkinson's disease, or Tourette syndrome. To understand the structural connectivity of cortical and subcortical circuits in the healthy human brain could thus be of pivotal importance for detecting changes in this circuitry and to start early intervention, to assess the progress of movement rehabilitation, or the effectiveness of therapeutic approaches in neuropsychiatry. While conventional magnetic resonance imaging (MRI), positron emission tomography, or magnetoencephalography are able to provide detailed information about connectivity at the macro level, the sensitivity and specificity these imaging techniques put limits on the amount of detail one can obtain when measuring in vivo connectivity of human basal ganglia and thalamus. In contrast, the multiband diffusion echo planar imaging MRI sequence, which acquires multiple slices of the brain simultaneously, enables high resolution imaging of these brain structures with only short acquisition times at 3-Tesla field strength. Here, we introduce a novel protocol that allows us to generate comprehensive in vivo participant-specific probabilistic patterns and visualizations of the structural connections that exist within basal ganglia and thalamic nuclei. Moreover, we are able to map specific parcellations of these nuclei into sub-territories based on their connectivity with primary motor-, and somatosensory cortex. The detailed subcortical structural connectivity mapping introduced in this work could benefit early intervention and therapy methods for human movement rehabilitation and for treating neuropsychiatric disorders. author/funder. All rights reserved. No reuse allowed without permission. \"},\n"," {'paper_id': '5cfb44ecdfb61b2898b6cf7315b1e810874a9560',\n","  'section': 'Abstract',\n","  'text': 'The 2019-nCoV outbreak has raised concern of global spread. While person-to-person transmission within the Wuhan district has led to a large outbreak, the transmission potential outside of the region remains unclear. Here we present a simple approach for determining whether the upper limit of the confidence interval for the reproduction number exceeds one for transmission in the United States, which would allow endemic transmission. As of February 7, 2020, the number of cases in the United states support subcritical transmission, rather than ongoing transmission. However, this conclusion can change if pre-symptomatic cases resulting from human-to-human transmission have not yet been identified. The 2019-nCoV outbreak is a global health risk, especially considering the number of patients who have already died after becoming infected in Wuhan, China. Many initial characterizations of the virus have focused on the basic reproduction number, R , which reflects the average number of infections that each new infection causes in a susceptible population. Whenever R is greater than one, epidemic transition becomes possible and within the Wuhan province, R has been estimated as high as 4.1 1-3 . '},\n"," {'paper_id': '5d3612cd95331a2f0c43ef9d03ce583b5ff8c995',\n","  'section': 'Abstract',\n","  'text': 'The 2019-nCoV is reported to share the same entry (ACE2) as SARS-CoV according to the updated findings. Analyzing the distribution and expression level of the route of coronavirus may help reveal underlying mechanisms of viral susceptibility and post-infection modulation. In this study, we found that the expression of ACE2 in healthy populations and patients with underlying diseases was not significantly different, suggesting relatively similar susceptibility. Besides, based on the expression of ACE2 in smoking individuals, we inferred that long-term smoking might be a risk factor for 2019-nCoV. Analyzing the ACE2 in SARS-CoV infected cells suggested that ACE2 was more than just a receptor but also participated in post-infection regulation, including immune response, cytokine secretion, and viral genome replication. Moreover, we also constructed Protein-protein interaction (PPI) networks and identified hub genes in viral activity and cytokine secretion. Our findings may explain the clinical symptoms so far and help clinicians and researchers understand the pathogenesis and design therapeutic strategies for 2019-nCoV. : medRxiv preprint GO biological process annotation gmt file was downloaded from MSigDB (https://www.gsea-msigdb.org/gsea/msigdb/). GSEA was performed to analyze the possible biological processes related to ACE2 in healthy people using clusterProfiler (12) . The parameters were nPerm = 1000, minGSSize = 10, maxGSSize = 500, the biological processes with p-value < 0.05 were considered significant. GSVA was performed using GSVA R package. The immune infiltrating was quantified using the ssGSEA method in GSVA R package. The gene list for immune cells was derived from Bindea G et al.(13). All viral-related biological process and cytokine secretion-biological process proteins were extracted from the gmt file, Cytoscape v3.7.2 was used to construct the PPI network using BisoGenet application, the PPI sources include DIP, BIOGRID, HPRD, INTACT, MINT and BIND. And the nodes with topological importance in the interaction network were screened by calculating Degree Centrality (DC) with the Cytoscape plugin CytoNCA. Hub proteins were identified using Cytoscape plugin CytoHubba. '},\n"," {'paper_id': '5d768aca47f1452fd3e3d2c246926b8532ca1c87',\n","  'section': 'Abstract',\n","  'text': 'Xylella fastidiosa (Xf) is a quarantine plant pathogen bacterium originating from the Americas 15 and that has emerged in Europe in 2013. Xf can be detected directly on plant macerate using 16 molecular methods such as real-time PCR, which is a sensitive technique. However, some 17 plants may contain components that can act as PCR reaction inhibitors, which can lead to false 18 negative results or an underestimation of the bacterial concentration present in the analyzed 19 plant sample. Droplet digital PCR (ddPCR) is an innovative tool based on the partitioning of 20 the PCR reagents and the DNA sample into thousands of droplets, allowing the quantification 21 of the absolute number of target DNA molecules present in a reaction mixture, or an increase 22 of the detection sensitivity. In this study, a real-time PCR protocol, already used for Xf detection 23 in the framework of official surveys in the European Union, was transferred and optimized for 24 Xf detection using ddPCR. This new assay was evaluated and compared to the initial real-time 25 PCR on five plant matrices artificially inoculated and on naturally infected plants. In our 26 conditions, this new ddPCR enabled the detection of Xf on all artificially inoculated plant 27 macerates with a similar limit of detection, or a slight benefit for Quercus ilex. Moreover, 28 ddPCR improved diagnostic sensitivity as it enabled detection of Xf in samples of Polygala 29 myrtifolia or Q. ilex that were categorized as negative or close to the limit of detection using 30 the real-time PCR. Here, we report for the first time a ddPCR assay for the detection of the 31 bacterium Xf. 32 '},\n"," {'paper_id': '5d9e490190986584d799e2828c6fd9679e5ae159',\n","  'section': 'Abstract',\n","  'text': 'Bats are reservoirs for a number of highly pathogenic zoonotic viruses, yet they remain relatively asymptomatic during infection. Whether this viral resistance is due to a unique innate immune system is unknown. An evolutionarily conserved feature of vertebrate antiviral immunity is the interferon (IFN) response, which triggers cellular defenses through interferon-stimulated gene (ISG) expression. While bats encode an intact IFN system, global ISG expression patterns in bat cells are not well characterized. Here, we used RNA-Seq to assess the transcriptional response to IFNα in cells derived from the bat Pteropus alecto (black flying fox). We show induction of more than 100 transcripts, most of which are canonical ISGs observed in other species. Kinetic gene profiling revealed that P. alecto ISGs fall into two unique temporal subclusters with similar early induction kinetics but distinct late-phase declines. In contrast to bat ISGs, human ISGs generally remained elevated for longer periods following IFN treatment, suggesting host-based differences in gene regulatory mechanisms. Notably, we also identified a small group of non-canonical bat ISGs, including an enzymatically active RNASEL that plays a role in controlling viral infection. These studies provide insight into the innate immune response of an important viral reservoir and lay a foundation for studies into the immunological features that may underlie unique virus-host relationship in bats. '},\n"," {'paper_id': '5daceb492bad28a61fc43c4be83a164042d8c430',\n","  'section': 'Abstract',\n","  'text': 'Rotavirus (RV) replicates in round-shaped cytoplasmic viral factories although 37 how they assemble remains unknown. 38 During RV infection, NSP5 undergoes hyperphosphorylation, which is primed by 39 the phosphorylation of a single serine residue. The role of this post-translational 40 modification in the formation of viroplasms and its impact on the virus replication 41 remains obscure. Here we investigated the role of NSP5 during RV infection by 42 taking advantage of a modified fully tractable reverse genetics system. An NSP5 43 trans-complementing cell line was used to generate and characterise several 44 recombinant rotaviruses (rRVs) with mutations in NSP5. We demonstrate that a 45 rRV lacking NSP5, was completely unable to assemble viroplasms and to 46 replicate, confirming its pivotal role in rotavirus replication. 47 A number of mutants with impaired NSP5 phosphorylation were generated to 48 further interrogate the function of this post-translational modification in the 49 assembly of replication-competent viroplasms. We showed that the rRV mutant 50 strains exhibit impaired viral replication and the ability to assemble round-shaped 51 viroplasms in MA104 cells. Furthermore, we have investigated the mechanism of 52 NSP5 hyper-phosphorylation during RV infection using NSP5 phosphorylation-53 negative rRV strains, as well as MA104-derived stable transfectant cell lines 54 expressing either wt NSP5 or selected NSP5 deletion mutants. Our results 55 indicate that NSP5 hyper-phosphorylation is a crucial step for the assembly of 56 round-shaped viroplasms, highlighting the key role of the C-terminal tail of NSP5 57 in the formation of replication-competent viral factories. Such a complex NSP5 58 phosphorylation cascade may serve as a paradigm for the assembly of functional 59 viral factories in other RNA viruses. 60 61 62 63 64 65 66 . CC-BY-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/660217 doi: bioRxiv preprint 3 IMPORTANCE 67 Rotavirus (RV) double-stranded RNA genome is replicated and packaged into 68 virus progeny in cytoplasmic structures termed viroplasms. The non-structural 69 protein NSP5, which undergoes a complex hyperphosphorylation process during 70 RV infection, is required for the formation of these virus-induced organelles. 71 However, its roles in viroplasm formation and RV replication have never been 72 directly assessed due to the lack of a fully tractable reverse genetics (RG) 73 system for rotaviruses. Here we show a novel application of a recently developed 74 RG system by establishing a stable trans-complementing NSP5-producing cell 75 line required to rescue rotaviruses with mutations in NSP5. This approach 76 allowed us to provide the first direct evidence of the pivotal role of this protein 77 during RV replication. Furthermore, using recombinant RV mutants we shed light 78 on the molecular mechanism of NSP5 hyperphosphorylation during infection and 79 its involvement in the assembly and maturation of replication-competent 80 viroplasms. 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 '},\n"," {'paper_id': '5dc2c157ddcd97f36c94faab478c38f463f78ae3',\n","  'section': 'Abstract',\n","  'text': 'The SARS-CoV-2 epidemic has rapidly spread outside China with major outbreaks occurring in Italy, South Korea and Iran. Phylogenetic analyses of whole genome sequencing data identified a distinct SARS-CoV-2 clade linked to travellers returning from Iran to Australia and New Zealand. This study highlights potential viral diversity driving the epidemic in Iran, and underscores the power of rapid genome sequencing and public data sharing to improve the detection and management of emerging infectious diseases. '},\n"," {'paper_id': '5de22810c166459e1cdc4f9dcd8976ecdbcbead8',\n","  'section': 'Abstract',\n","  'text': \"Background: With the worldwide outbreak of COVID-19, an accurate model to predict how the coronavirus pandemic will evolve in individual countries becomes important and urgent. Our goal is to provide a prediction model to help policy makers in different countries address the epidemic outbreak and adjust the control policies to contain the spread of the severe acute respiratory syndrome coronavirus 2 (SARS-Cov-2) more effectively. Methods: Unlike the classic public health and virus propagation models, this new projection model takes both government intervention and public response into account to generate reliable projections of the outbreak 10 days to 2 weeks in advance. This method is an observation based projection similar than the classic Moore's Law in miroelectronics. The Moore's law is not based on any physics law and yet has anticipated the development of microelectronics for decades. This work is an empirical relation to decribe the evolution of epidemic to pandemic situations in different countries. The country was selected as an observation unit because the regulation and political decision is an national decision for numerous measures such as the implementation of social distancing, the quarantine of suspected cases, and the closing of borders to achieve territorial containment. China. Then the model was also validated by the South Korean data. With a reduction of cases calculated as reduction coefficient of the increase rate of daily cases Rc = 2% per day, we observed a very efficient policy with a strict systematic control in both China and South Korea. For the moment, the Canada, USA, Australia may have difficulties to limit the fast evolution of the epidemic. With a Rc<0.5%, it's particularly important for the USA to consider escalating the control measures because the affected cases can reach more than one million very soon. Interpertation: Due to the difference of national disciplines and historical culture, the national policy may be implemented and observed with different efficiency. The starting point where the government decided to apply total containment can also play a key role for the evolution of the pendemic situation. The model will allow each national government of the nations still affected by the pandemic to project the situation for the coming 10 to 14 days. It's very important for the deployment of national and international efforts to stop the pandemic situation. \"},\n"," {'paper_id': '5e78cd85d425e3bfef6fe7cfabbcfa1cf6ae1859',\n","  'section': 'Abstract',\n","  'text': 'Background: On December 2019, Novel coronavirus disease (COVID-19) was detected in Wuhan, China, and then spread around the world. There is little information about effects of COVID-19 on Pregnant women and newborns as a sensitive population. The current study is a systemic review and Meta-analysis to measure the risks and determine the presentations of COVID-19 in pregnant women and newborn. Methods: online data bases were searched on march 20. Heterogeneity of the included studies was assessed using the Cochran Q test and Higgins I 2 statistic and expressed as percentage. All data were analyzed with 95% confidence intervals. A total of 7 studies involving 50 participants with Positive test of COVID-19 were enrolled. Mean age of pregnant women was 30.57 years old and the Mean Gestational age was All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. : medRxiv preprint 36.9 weeks. Other variables such as Apgar score, birth weight, Sign and symptoms, Complications and Laboratory data were Analyzed. Our findings showed same clinical characteristics in pregnant women as in nonpregnant adults, with the main symptoms being cough and fever. No vertical transmission was seen and all patients delivered healthy neonates. Our findings would be of great help to the decision making process, regarding the management of pregnant women diagnosed with COVID-19. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '5ec284329dfa97be65ed5e598c40426f03499186',\n","  'section': 'Abstract',\n","  'text': 'Due to the recent diffusion of COVID-19 outbreak, the scientific community is making efforts in analysing models for understanding the present situation and predicting future scenarios. In this paper, we propose a Susceptible-Infected-Exposed-Recovered-Dead (SEIRD) differential model [Weitz J. S. and Dushoff J., Scientific reports, 2015] for the analysis and forecast of the COVID-19 spread in Italian regions, using the data from the Italian Protezione Civile from February 24th 2020. In this study, we investigate an adaptation of SEIRD that takes into account the actual policies of the Italian government, consisting of modelling the infection rate as a time-dependent function (SEIRD(rm)). Preliminary results on Lombardia and Emilia-Romagna regions confirm that SEIRD(rm) fits the data more accurately than the original SEIRD model with constant rate infection parameter. Moreover, the increased flexibility in the choice of the infection rate function makes it possible to better control the predictions due to the lockdown policy. 12 the outbreak containment. 13 We consider here deterministic compartmental models, based on a system of initial 14 value problems of Ordinary Differential Equations. This theory has been studied since 15 the beginning of the century by W.O. Kermack and A. G. MacKendrick [3] who 16 proposed the basic Susceptible-Infected-Removed (SIR) model. The SIR model and its 17 later modifications, such as Susceptible-Exposed-Infected-Removed (SEIR) [4] were later 18 introduced in the study of outbreaks diffusion. These models split the population into 19 '},\n"," {'paper_id': '5f06ba54c9208857d92abfd16488c44217f8d726',\n","  'section': 'Abstract',\n","  'text': 'Scintillators exhibit visible luminescence, called scintillation, when irradiated with X-rays. Given that X-rays penetrate through biological tissues, X-ray-induced scintillation would enable remote optogenetic control of neural functions at any depths in the brain. Here we show that a yellow-emitting inorganic scintillator, Ce-doped Gd3(Al,Ga)5O12 (Ce:GAGG), can effectively activate red-shifted excitatory and inhibitory opsins. Using these scintillator-opsin combinations, we successfully activated and inhibited midbrain dopamine neurons of freely moving mice by X-ray irradiation, producing bi-directional modulation of place preference behavior. The Ce:GAGG crystal was biocompatible and could be implanted for a long period without progressive neuroinflammatory responses. Neither brain injury nor behavioral dysfunction was acutely induced by radiation during the behavioral tests. Thus, X-ray-induced scintillation allows wireless control of cellular functions in living animals, expanding X-ray applications to functional studies of biology and medicine. '},\n"," {'paper_id': '5f2e586d719f24d65278de208759dab8213327f3',\n","  'section': 'Abstract',\n","  'text': \"ABSTRCT 29 Given the scale and rapid spread of severe acute respiratory syndrome coronavirus 2 30 (SARS-CoV-2, known as 2019-nCov) infection , the ongoing global 31 SARS-CoV-2 outbreak has become a huge public health issue. Rapid and precise 32 diagnostic methods are thus immediately needed for diagnosing COVID-19, 33 providing timely treatment and facilitating infection control. A one-step reverse 34 transcription loop-mediated isothermal amplification (RT-LAMP) coupled with 35 nanoparticles-based biosensor (NBS) assay (RT-LAMP-NBS) was successfully 36 established for rapidly and accurately diagnosing COVID-19. A simple equipment 37 (such as heating block) was required for maintaining a constant temperature (63°C) 38 for only 40 min. Using two designed LAMP primer sets, F1ab (opening reading frame 39 1a/b) and np (nucleoprotein) genes of SARS-CoV-2 were simultaneously amplified 40 and detected in a 'one-step' and 'single-tube' reaction, and the detection results were 41 easily interpreted by NBS. The sensitivity of SARS-CoV-2 RT-LAMP-NBS was 12 42 copies (each of detection target) per reaction, and no cross-reactivity was generated 43 from non-SARS-CoV-2 templates. Among clinically diagnosed COVID-19 patients, 44 the analytical sensitivity of SARS-CoV-2 was 100% (33/33) in the oropharynx swab 45 samples, and the assay's specificity was also 100% (96/96) when analyzed the clinical 46 samples collected from non-COVID-19 patients. The total diagnosis test from sample 47 collection to result interpretation only takes approximately 1 h. In sum, the 48 RT-LAMP-NBS is a promising tool for diagnosing the current SARS-CoV-2 49 infection in first line field, public health and clinical laboratories, especially for 50 resource-challenged regions. 51 \"},\n"," {'paper_id': '5f72fd41d7ba8d4d7de7aa1f83fc5c90ffa50b03',\n","  'section': 'Abstract',\n","  'text': 'The fourth outbreak of the Coronaviruses, known as the 2019-nCoV, has occurred in Wuhan city of Hubei province in China in December 2019. We propose a timevarying sparse vector autoregressive (VAR) model to retrospectively analyze and visualize the dyamic transmission routes of this outbreak in mainland China over January 31 -February 19, 2020. Our results demonstrate that the influential interprovince routes from Hubei have become unidentifiable since February 4, whereas the self-transmission in each province was accelerating over February 4-15. From February 16, all routes became less detectable, and no influential transmissions could be identified on February 18 and 19. Such evidence supports the effectiveness of government interventions, including the travel restrictions in Hubei. Implications of our results suggest that in addition to the origin of the outbreak, virus preventions are of crucial importance in provinces with the largest migrant workers percentages (e.g., Jiangxi, Henan and Anhui) to controlling the spread of 2019-nCoV. '},\n"," {'paper_id': '5f7e6772c35ba7d67891202d43e0a43917f90971',\n","  'section': 'Abstract',\n","  'text': 'Reproduction number is an epidemiologic indicator that reflects the contagiousness and transmissibility of infectious agents. This paper aims to estimate the reproduction number of in the early phase of COVID-19 outbreak in Vietnam. '},\n"," {'paper_id': '5fb2cce255dd56b944ae52cec664bd7f4f222c66',\n","  'section': 'Abstract',\n","  'text': 'There has been an outbreak of coronavirus disease in Wuhan city, Hubei province, China since December 2019. Cases have been exported to other parts of China and more than 20 countries. We provide estimates of the daily trend in the size of the epidemic in Wuhan based on detailed information of 10,940 confirmed cases outside Hubei province. '},\n"," {'paper_id': '5fee35a22d17fd777231c338a57ac4a93a670cc1',\n","  'section': 'Abstract',\n","  'text': 'The purpose of this article is to reach all those who find it difficult to become well informed about the repercussions of a lockdown strategy to tackle the COVID-19 pandemic and to spark discussion and thought. Here we use simple stochastic simulations to evaluate different approaches taken to tackle the crisis, along with the efficiency they will hold and the number of casualties they may incur. It is clear that the less strict the social distancing the more time it will take for life to return to normal, and the more lives will be at risk. This is shown through simulations formed by an open sourced code, which allows evaluation of the outcomes from different intervention scenarios or conditions. '},\n"," {'paper_id': '602bd8abdd8f0d464941d989322e843533cea15d',\n","  'section': 'Abstract',\n","  'text': 'The coronavirus SARS-CoV-2 has infected more than 600,000 people and has overwhelmed hospital systems around the world. However, the factors mediating fatal SARS-CoV-2 infections are poorly understood. Here, we show that cigarette smoke causes a dose-dependent upregulation of Angiotensin Converting Enzyme 2 (ACE2), the SARS-CoV-2 receptor, in rodent and human lungs. Using single-cell sequencing data, we demonstrate that ACE2 is expressed in a subset of epithelial cells that line the respiratory tract, including goblet cells, club cells, and alveolar type 2 cells. Chronic smoke exposure triggers a protective expansion of mucus-secreting goblet cells and a concomitant increase in ACE2 expression. In contrast, quitting smoking causes a decrease in lung ACE2 levels. Taken together, these results may partially explain why smokers are particularly likely to develop severe SARS-CoV-2 infections, and they suggest that quitting smoking could lessen coronavirus susceptibility. '},\n"," {'paper_id': '60f9bdb06e4d0d721e3436eedbe6891594d2de46',\n","  'section': 'Abstract',\n","  'text': 'There is big concern for estimating the lethality and the extent of undetected infections associated with the novel coronavirus SARS-CoV2 outbreak. While detailed epidemiological models are certainly needed, I suggest here an orthogonal approach based on a minimum number of parameters robustly fitted from the cumulative data easily accessible for all countries at the John Hopkins University database that became the worldwide reference for the pandemics. I show that, after few days from the beginning of the outbreak, the apparent death rate can be extrapolated to infinite time through regularized regression such as rescaled ridge regression. The variation from country to country of these extrapolated death rates appears to depend almost only (r 2 = 0.91) on the ratio between performed tests and detected cases even when the apparent instantaneous lethality rates are as different as 9% in Italy and 0.4% in Germany. Extrapolating to the limit of infinite number of tests, I obtain a death rate of 0.012 ± 0.012, in agreement with other estimates. The inverse relationship between the extrapolated death rate and the intensity tests allows estimating that more than 50% of cases were undetected in most countries, with more than 90% undetected cases in countries severely hit by the epidemics such as Italy. Finally, I propose to adopt the ratio between the cumulative number of recovered and deceased persons as an indicator that can anticipate the halting of the epidemics. '},\n"," {'paper_id': '610549c879c62efc42f2ad32f63c65b25d8931c7',\n","  'section': 'Abstract',\n","  'text': 'Background The outbreak of the recently emerged novel corona virus disease 2019 '},\n"," {'paper_id': '6115e9fbaed5e154ff92cb8d02387fec72251dab',\n","  'section': 'Abstract',\n","  'text': 'Severe acute respiratory syndrome (SARS-CoV-2) due to novel Coronavirus '},\n"," {'paper_id': '618cd102ec5051a05ca5131e21961c964152f15c',\n","  'section': 'Abstract',\n","  'text': 'Adenoviruses are double-strained DNA viruses found in a great number of vertebrates, 8 including humans. In order to understand their transmission dynamics, it is crucial, even from a 9 human health perspective, to investigate how host traits influence their prevalence. Bats are 10 important reservoirs for Adenoviruses, and here we use the results of recent screenings in 11 Western Europe to evaluate the association between characteristic traits of bat species and their 12 probability of hosting Adenoviruses, taking into account their phylogenetic relationships. Across 13 species, we found an important phylogenetic component in the presence of Adenoviruses and 14 mating strategy as the most determinant factor conditioning the prevalence of Adenoviruses 15 across bat species. Contrary to other more stable mating strategies (e.g. harems), swarming could 16 hinder transmission of Adenoviruses since this strategy implies that contacts between individuals 17 are too short. Alternatively, bat species with more promiscuous behavior may develop a stronger 18 immune system. Outstandingly high prevalence of Adenoviruses was reported for the Iberian 19 species Pipistrellus pygmaeus, P. kuhlii and Nyctalus lasiopterus and we found that in the latter, 20 males were more likely to be infected by Adenoviruses than females, due to the 21 immunosuppressing consequence of testosterone during the mating season. As a general trend 22 4 29 Author Summary 30 Adenoviruses are DNA viruses with a wide range of vertebrate hosts, including humans, 31 causing ocular, respiratory and gastrointestinal diseases. Here, we focus on the prevalence of 32 Adenoviruses in bats, which are known to be natural reservoir of many viruses, using the results 33 of recent screenings for prevalence of these viruses in 33 European bat species. Our aim is to 34 find association between Adenoviruses prevalence and biological and behavioral host traits, 35 considering the heterogeneity both between and within species in order to have a deeper 36 understanding of mechanisms of viral transmission. Our results highlight the importance of mating strategy: bats species using swarming as mating 38 strategy are less likely to be infected by Adenoviruses. Moreover, we found that locality of 39 capture can explain a higher prevalence of Adenovirus within species. However, no general 40 pattern has been found in the analysis at individual level, suggesting a strong species specificity 41 and complex viral transmission dynamics. 42 43 '},\n"," {'paper_id': '6197b0f53a6a883f99c2602a89309b8b2f27da92',\n","  'section': 'Abstract',\n","  'text': 'The early detection of SARS-CoV-2, the causative agent of (COVID-19) is now a critical task for the clinical practitioners. The COVID-19 spread is announced as pandemic outbreak between people worldwide by WHO since 11/ March/ 2020. In this consequence, it is top critical priority to become aware of the infected people so that prevention procedures can be processed to minimize the COVID-19 spread and to begin early medical health care of those infected persons. In this paper, the deep studying based totally methodology is usually recommended for the detection of COVID-19 infected patients using X-ray images. The help vector gadget classifies the corona affected X-ray images from others through usage of the deep features. The technique is useful for the clinical practitioners for early detection of COVID-19 infected patients. The suggested system of multi-level thresholding plus SVM presented high accuracy in classification of the infected lung with Covid-19. All images were of the same size and stored in JPEG format with 512 * 512 pixels. The average sensitivity, specificity, and accuracy of the lung classification using the proposed model results were 95.76%, 99.7%, and 97.48%, respectively. '},\n"," {'paper_id': '61dfcb1e49f66017e6ee12ec8a16644dff6ed280',\n","  'section': 'Abstract',\n","  'text': 'The aim of the present study was to assess whether SARS-CoV-2 can be detected in the expressed prostatic secretion (EPS) of patients with corona virus disease 2019 (COVID-19). Methods: 18 cases of COVID-19, and 5 suspected cases, were selected from three medical centers to detect the RNA expression of SARS-CoV-2 in their EPS with RT-PCR. Results were negative in all EPS samples for SARS-CoV-2 of suspected and confirmed patients. No SARS-CoV-2 was expressed in EPS of patients with COVID-19. '},\n"," {'paper_id': '620d69e419e3c8756de8bf96d86f6bf0de7ed919',\n","  'section': 'Abstract',\n","  'text': \"Background: Viral encephalitis is a dangerous compromise between the need to robustly clear pathogen from the brain and the need to protect neurons from bystander injury. Theiler's murine encephalomyelitis virus (TMEV) infection of C57Bl/6 mice is a model of viral encephalitis in which the compromise results in hippocampal damage and permanent neurological sequelae. We previously identified brain infiltrating inflammatory monocytes as the primary driver of this hippocampal pathology, but the mechanisms involved in recruiting these cells to the brain were unclear. Methods: Chemokine expression levels in the hippocampus were assessed by microarray, ELISA, RT-PCR, and immunofluorescence. Monocyte infiltration during acute TMEV infection was measured by flow cytometry. CCL2 levels were manipulated by immunodepletion and by specific removal from neurons in mice generated by crossing a line expressing the Cre recombinase behind the synapsin promoter to animals with floxed CCL2. Results: Inoculation of the brain with TMEV induced hippocampal production of the proinflammatory chemokine CCL2 that peaked at 6 hours postinfection, whereas inoculation with UV-inactivated TMEV did not elicit this response. Immunofluorescence revealed that hippocampal neurons expressed high levels of CCL2 at this timepoint. Genetic deletion of CCR2 and systemic immunodepletion of CCL2 abrogated or blunted the infiltration of inflammatory monocytes into the brain during acute infection. Specific genetic deletion of CCL2 from neurons reduced serum and hippocampal CCL2 levels and inhibited inflammatory monocyte infiltration into the brain. Conclusions: We conclude that intracranial inoculation with infectious TMEV rapidly induces the expression of CCL2 in neurons, and this cellular source is necessary for CCR2-dependent infiltration of inflammatory monocytes into the brain during the most acute stage of encephalitis. These findings highlight a unique role for neuronal production of chemokines in the initiation of leukocytic infiltration into the infected central nervous system. \"},\n"," {'paper_id': '622580ea34c3acc011faa7573bc549e34078edde',\n","  'section': 'Abstract',\n","  'text': 'The novel coronavirus disease was first reported in Wuhan, China in December 2019 and is now pandemic all over the world. Previous study has reported several COVID-19 cases with conjunctivitis. However, the complete profiling of COVID-19 related ocular symptoms and diseases are still missing. We aim to investigate the ocular manifestations and clinical characteristics of COVID-19 patients. Methods: A total of five hundred and thirty-four patients were recruited at Mobile Cabin Hospital and Tongji Hospital. We collected information on demographic characteristics, exposure history, ocular symptoms, systemic concomitant symptoms, eye drop medication, eye protections, radiologic findings, and SARS-CoV-2 detection in nasopharyngeal swabs by RT-PCR from questionnaires and electronic medical records. '},\n"," {'paper_id': '62298f90df017016d34e978befc29acbec201ccd',\n","  'section': 'Abstract',\n","  'text': 'For optimal exploitation of nanoparticles (NPs) in biomedicine, and to predict nanotoxicity, detailed knowledge on the cellular responses to cell-bound or internalized NPs is imperative. The outcome of NP-cell interaction is dictated by the type and magnitude of the NP insult and the cellular response. Here, we have systematically studied the impact of minor differences in NP composition on cellular stress responses and viability by using highly similar poly(alkylcyanoacrylate) (PACA) particles. Surprisingly, PACA particles differing only in their alkyl side chains; butyl (PBCA), ethylbutyl (PEBCA), or octyl (POCA), respectively, induced different stress responses and modes of cell death in human cell lines. POCA particles induced endoplasmic reticulum stress and apoptosis. In contrast, PBCA and PEBCA particles induced lipid peroxidation by depletion of the main cellular antioxidant glutathione (GSH), in a manner depending on the levels of the GSH precursor cystine, and transcription of the cystine transporter SLC7A11 regulated by ATF4 and Nrf2. Intriguingly, these particles activated the recently discovered cell death mechanism ferroptosis, which constitutes a promising alternative for targeting multidrug-resistant cancer stem-like cells. Of the two, PBCA was the strongest inducer. In summary, our findings highlight the cellular sensitivity to nanoparticle composition and have important implications for the choice of PACA monomer in therapeutical settings. author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '625b53389c239245a7d644ff55d132ee528668f6',\n","  'section': 'Abstract',\n","  'text': 'Objectives: To estimate the basic reproduction number (R 0 ) of COVID-19 in the early stage of the epidemic and predict the expected number of new cases in Shahroud, Northeast of Iran. The R 0 of COVID-19 was estimated using the serial interval distribution and the number of incidence cases. The serial interval was fit with a gamma distribution. The probable incidence and cumulative incidence in the next 30 days were predicted using the assumption that daily incidence follows a Poisson distribution determined by daily infectiousness. Data analysis was done using \"earlyR\" and \"projections\" packages in R software. The maximum-likelihood value of R 0 was 2.7 (95% confidence interval (CI): 2.1 to 3.4) for the COVID-19 epidemic in the early 14 days and decreased to 1.13 (95% CI: 1.03 to 1.25) by the end of the day 41. The expected average number of new cases in Shahroud is 9.0±3.8 case/day, which means an estimated total of 271 (95% CI: 178-383) new cases in the next 30 days. It is essential to reduce the R 0 to values below one. Therefore, we strongly recommend enforcing and continuing the current preventive measures, restricting travel, and providing screening tests for a larger proportion of the population. '},\n"," {'paper_id': '62b978569b93bcbafae6ae2a082dc70b8c6abdf0',\n","  'section': 'Abstract',\n","  'text': 'We use human mobility models, for which we are experts, and attach a virus infection dynamics to it, for which we are not experts but have taken it from the literature, including recent publications. This results in a virus spreading dynamics model. The results should be verified, but because of the current time pressure, we publish them in their current state. Recommendations for improvement are welcome. We come to the following conclusions: . CC-BY 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint . '},\n"," {'paper_id': '62d68fb65591e60de58891c42f5a1457b9654603',\n","  'section': 'Abstract',\n","  'text': 'The World Health Organization (WHO) has recently declared coronavirus disease 2019 (COVID-19) a public health emergency of global concern. Updated analysis of cases might help identify the characteristic and risk factors of the illness severity. We extracted data regarding 47 patients with confirmed COVID-19 from Renmin Hospital of Wuhan University between February 1 and February 18, 2020. The degree of severity of COVID-19 patients (severe vs. non-severe) was defined at the time of admission according to American Thoracic Society (ATS) guidelines for communityacquired pneumonia (CAP). The median age was 64.91 years, 26 cases (55.31%) were male of which, and 70.83% were severe cases. Severe patients had higher APACHE II (9.92 vs 4.74) and SOFA (3.0 vs 1.0) scores on admission, as well as the higher PSI (86.13 vs 61.39), Curb-65 (1.14 vs 0.48) and CT semiquantitative scores (5.0 vs 2.0) when compared with nonsevere patients. Among all univariable parameters, APACHE II, SOFA, lymphocytes, CRP, LDH, AST, cTnI, BNP, et al were significantly independent risk factors of COVID-19 severity. Among which, LDH was most positively related both with APACHE II (R = 0.682) and SOFA (R = 0.790) scores, as well as PSI (R = 0.465) and CT (R = 0.837) scores. To assess the diagnostic value of these selected parameters, LDH (0.9727) had maximum sensitivity (100.00%) and specificity (86.67%), with the cutoff value of 283. As a protective factor, lymphocyte counts less than 1.045 x 10 9 /L showed a good accuracy for identification of severe patients with AUC = 0.9845 (95%CI 0.959-1.01), the maximum specificity (91.30%) and sensitivity (95.24%). In addition, LDH was positively correlated with CRP, AST, BNP and cTnI, while negatively correlated with lymphocyte cells and its subsets, including CD3 + , CD4 + and CD8 + T cells (P < 0.01). '},\n"," {'paper_id': '63741136af88d75b21e06f0ebdc0008bdfc58b9d',\n","  'section': 'Abstract',\n","  'text': 'Protection of healthcare workforce is of paramount relevance for the care of infected and non-infected patients in the setting of a pandemic such as coronavirus disease 2019 . Healthcare workers are at increased risk to become infected because of contact to infected patients, infected co-workers and their community outside the hospital. The ideal organisational strategy to protect the healthcare workforce in a situation in which social distancing cannot be maintained at the workplace remains to be determined. In this study, we have mathematically modelled strategies for the employment of hospital workforce with the goal to simulate health and productivity of the workers. Therefore, deterministic models were extended to account for stochastic influences potentially occurring in rather small populations. The models were also designed to determine if desynchronization of medical teams by dichotomizing the workers may protect the workforce. Our studies model workforce productivity depending on the infection rate, the presence of reinfection and the efficiency of home office. As an application example, we apply our theory to the case of coronavirus disease 2019 . The results of the models reveal that a desynchronization strategy in which two medical teams work alternating for 7 days reduces the infection rate of the healthcare workforce. During the spread of worldwide pandemics, protecting and supporting caregivers to maintain the workforce in hospitals is a crucial and challenging task. Under such extraordinary situations, the efficiency of healthcare workforce is threatened by several factors including: 1) infected patients, 2) infected co-workers and 3) infected persons outside the hospital. This is of high importance in the current pandemic of coronavirus disease 2019 given that the virus may be transmitted by asymptomatic persons based on the long incubation period averaging at 5 days (1, 2). Unlike for most other professions, social distancing is typically not possible in medical teams where healthcare workers are required to work in close contact with patients and coworkers. Currently, little is known about which work organizational strategies in the hospital are most suitable protect the healthcare workforce. Potential strategies may include prolonging duty hours and thereby limiting interactions between coworkers or complete desynchronization of the workforce in which teams are dichotomized and each half of the team works for one week alternatively. . CC-BY 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': '63781701d3aac3b1640caae24b2c8c2a06c8e6ab',\n","  'section': 'Abstract',\n","  'text': \"Predictive models for the spatial spread of infectious diseases has received much attention in recent years as tools for the management of infectious diseas outbreaks. Prominently, various versions of the so-called gravity model, borrowed from transportation theory, have been used. However, the original literature suggests that the model has some potential misspecifications inasmuch as it fails to capture higher-order interactions among population centers. The fields of economics, geography and network sciences holds alternative formulations for the spatial coupling within and among conurbations. These includes Stouffer's rank model, Fotheringham's competing destinations model and the radiation model of Simini et al. Since the spread of infectious disease reflects mobility through the filter of age-specific susceptibility and infectivity and since, moreover, disease may alter spatial behavior, it is essential to confront with epidemiological data on spread. To study their relative merit we, accordingly, fit variants of these models to the uniquely detailed dataset of prevaccination measles in the 954 cities and towns of England and Wales over the years 1944-65 and compare them using a consistent likelihood framework. We find that while the gravity model is a reasonable first approximation, both Stouffer's rank model, an extended version of the radiation model and the Fotheringham competing destinations model provide significantly better fits, Stouffer's model being the best. Through a new method of spatially disaggregated likelihoods we identify areas of relatively poorer fit, and show that it is indeed in densely-populated conurbations that higher order spatial interactions are most important. Our main conclusion is that it is premature to narrow in on a single class of models for predicting spatial spread of infectious disease. The supplemental materials contain all code for reproducing the results and applying the methods to other data sets. The ability to predict how infectious disease will spread is of great importance in the 1 face of the numerous emergent and re-emergent pathogens that currently threatening 2 : bioRxiv preprint human well-being. We identified a variety of alternative models that predict human 3 mobility as as a function of population distribution across a landscape. These consider 4 some models that account for pair-wise interactions between population centers, as well 5 as some that allow for higher-order interactions. We trained the models using a 6 uniquely rich spatiotemporal data set on pre-vaccination measles in England and wales 7 (1944-65), which comprises more than a million records from 954 cities and towns. Likelihood rankings of the different models reveal strong evidence for higher-order 9 interactions in the form of competition among cities as destinations for travelers and, 10 thus, dilution of spatial transmission. The currently most commonly used so-called 11 'gravity' models were far from the best in capturing spatial disease dynamics. 12 42 movement patterns [13] including a recent incarnation as the 'radiation' model [14]. While these models have proved useful in their original applications, their use in 44 describing infectious disease spread is complicated by two considerations. First, spatial 45 interaction models predict bulk migration fluxes between population centers, but the 46 impact of these movements on the dynamics of infectious disease can depend strongly 47 not only on the magnitude, but also on the composition, of the migrant pool. In 48 particular, the extent to which migrants are more or less likely to be susceptible to 49 infection, or actually infected, than the general population can be critically important, 50 as can the age profile of the migrant pool, due to pronounced age-specific patterns of 51 behavior, susceptibility, and infectiousness [5, 7, 15] . Second, infectious disease dynamics 52 and Wales using a spatial hazards approach [18] . We considered four main classes of 268 models, including the gravity, competing destinations, Stouffer rank, and radiation 269 models, in addition to pure diffusion and mean field models which are interesting special 270 \"},\n"," {'paper_id': '63a6f3a7d293a64324634b67596b6e9a5c95ad87',\n","  'section': 'Abstract',\n","  'text': 'Many countries have applied lockdown to suppress COVID-19, with devastating economic consequences. Here we propose exit strategies from lockdown that suppress the epidemic and provide sustainable, albeit reduced, economic activity. We use mathematical models to show that a cyclic schedule of 4-day work and 10-day lockdown, or similar variants that can be adapted in response to epidemiological observations, can in certain conditions suppress the epidemic while providing part-time employment. The cycle reduces the reproduction number R by a combination of reduced exposure time and a resonance effect where those infected during work days reach peak infectiousness during lockdown days. Throughout, full epidemiological measures need to continue including hygiene, physical distancing and extensive testing and contact tracing. Adaptive work-lockdown cycles can provide epidemic control and offer predictability to many economic sectors. '},\n"," {'paper_id': '63b645b7d8cd2502bedca374d0418486838bf4a0',\n","  'section': 'Abstract',\n","  'text': 'Non-viral gene delivery is constrained by the dwell time that most synthetic nucleic acid nanocarriers spend inside endosomal compartments. In order to overcome this endosomalrelease bottleneck, methods are required that measure nanocarrier uptake kinetics and transfection efficiency simultaneously. Here, we employ live-cell imaging on single-cell arrays (LISCA) to study the delivery-time distribution of lipid-based mRNA complexes under varied serum conditions. By fitting a translation-maturation model to hundreds of individual eGFP reporter fluorescence time courses, the protein expression onset times and the expression rates after transfection are determined. Using this approach, we find that delivery timing and protein expression rates are not intrinsically correlated at the single-cell level, even though population-averaged values of both parameters conjointly change as a function of increasing external serum protein fraction. Lipofectamine mediated delivery showed decreased transfection efficiency and longer delivery times with increasing serum protein concentration. This is in contrast to ionizable lipid nanoparticles (LNPs) mediated transfer, which showed increased efficiency and faster uptake in the presence of serum. In conclusion, the interdependences of single-cell expression rates and onset timing provide additional clues on uptake and release mechanisms, which are useful for improving nucleic acid delivery. author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '6427e85b4473cc2a84635b08384e479c860270da',\n","  'section': 'Abstract',\n","  'text': 'Methods: A systematic review and pooled analysis was performed. Eligible studies were 23 identified from database and hand searches up to March 2, 2020. Data on clinical (including 24 laboratory and radiological) and epidemiological (including demographic) characteristics of 25 confirmed COVID-19 cases were extracted and combined by simple pooling. 26 27 Results: Of 644 studies identified, 69 studies (involving 48,926 patients) were included in the 28 analysis. The average age of the patients was 49.16 years. A total of 51.46% of the patients 29 were men and 52.32% were non-smokers. Hypertension (50.82%) and diabetes (20.89%) were 30 the most frequent comorbidities observed. The most common symptoms were fever (83.21%), 31 cough (61.74%), and myalgia or fatigue (30.22%). Altered levels of blood and biochemical 32 parameters were observed in a proportion of the patients. Most of the patients (78.50%) had 33 bilateral lung involvements, and 5.86% showed no CT findings indicative of viral pneumonia. 34 Acute respiratory distress syndrome (28.36%), acute cardiac injury (7.89%) and acute kidney 35 injury (7.60%) were the most common complications recorded. 36 37 Conclusions: Clinical and epidemiological characteristics of COVID-19 patients were mostly 38 heterogeneous and non-specific. This is the most comprehensive report of the characteristics of 39 '},\n"," {'paper_id': '6445688d3f59cb7e02f2a8b28450cdf118a8a373',\n","  'section': 'Abstract',\n","  'text': 'Social mixing patterns are crucial in driving transmission of infectious diseases and informing public health interventions to contain their spread. Age-specific social mixing is often inferred from surveys of self-recorded contacts which by design often have a very limited number of participants. In addition, such surveys are rare, so public health interventions are often evaluated by considering only one such study. Here we report detailed population contact patterns for '},\n"," {'paper_id': '64478db449af8e4a5645c17f548649a0159da0c6',\n","  'section': 'Abstract',\n","  'text': 'Emerging infectious diseases increasingly threaten wildlife populations. Most studies focus 27 on managing short-term epidemic properties, such as controlling early outbreaks. Predicting 28 long-term endemic characteristics with limited retrospective data is more challenging. We 29 used individual-based modelling informed by individual variation in pathogen load and 30 transmissibility to predict long-term impacts of a lethal, transmissible cancer on Tasmanian 31 devil (Sarcophilus harrisii) populations. For this, we employed Approximate Bayesian 32 Computation to identify model scenarios that best matched known epidemiological and 33 demographic system properties derived from ten years of data after disease emergence, 34 enabling us to forecast future system dynamics. We show that the dramatic devil population 35 declines observed thus far are likely attributable to transient dynamics. Only 21% of 36 matching scenarios led to devil extinction within 100 years following devil facial tumour 37 disease (DFTD) introduction, whereas DFTD faded out in 57% of simulations. In the 38 remaining 22% of simulations, disease and host coexisted for at least 100 years, usually with 39 long-period oscillations. Our findings show that pathogen extirpation or host-pathogen 40 coexistence are much more likely than the DFTD-induced devil extinction, with crucial 41 management ramifications. Accounting for individual-level disease progression and the long-42 term outcome of devil-DFTD interactions at the population-level, our findings suggest that 43 immediate management interventions are unlikely to be necessary to ensure the persistence of 44 Tasmanian devil populations. This is because strong population declines of devils after 45 disease emergence do not necessarily translate into long-term population declines at 46 equilibria. Our modelling approach is widely applicable to other host-pathogen systems to 47 predict disease impact beyond transient dynamics. '},\n"," {'paper_id': '6499f5c865c3db3bdfc1e969ea940a80d547a5bb',\n","  'section': 'Abstract',\n","  'text': 'Codon usage patterns of viruses reflect a series of evolutionary changes that enable viruses to shape their survival rates and fitness toward the external environment and, most importantly, their hosts. In the present study, we employed multiple codon usage analysis indices to determine genotype specific codon usage patterns of Zika virus (ZIKV) strains from the current outbreak and those reported previously. Several genotype specific and common codon usage traits were noted in ZIKV coding sequences, indicative of independent evolutionary origins from a common ancestor. The overall influence of natural selection was found to be more profound than that of mutation pressure and acting on specific set of viral genes belonging to ZIKV strains of Asian genotype from the recent outbreak. Furthermore, an interplay of codon adaptation and deoptimization have been observed in ZIKV genomes. The collective findings of codon analysis in association with the geographical data of Aedes populations in the Americas suggests that ZIKV have evolved a dynamic set of codon usage patterns in order to maintain a successful replication and transmission chain within multiple hosts and vectors. '},\n"," {'paper_id': '64b327001f8fa95b83dc23259a2ad617be12498c',\n","  'section': 'Abstract',\n","  'text': \"One unexplored aspect of HIV-1 genetic architecture is how codon choice influences 27 population diversity and evolvability. Here we compared the development of HIV-1 28 resistance to protease inhibitors (PIs) between wild-type (WT) virus and a synthetic 29 virus (MAX) carrying a codon-pair re-engineered protease sequence including 38 (13%) 30 synonymous mutations. WT and MAX viruses showed indistinguishable replication in 31 MT-4 cells or PBMCs. Both viruses were subjected to serial passages in MT-4 cells 32 with selective pressure from the PIs atazanavir (ATV) and darunavir (DRV). After 32 33 successive passages, both the WT and MAX viruses developed phenotypic resistance to 34 PIs (IC 50 14.6 ± 5.3 and 21.2 ± 9 nM for ATV, and 5. 9 ± 1.0 and 9.3 ± 1.9 for DRV, 35 respectively). Ultra-deep sequence clonal analysis revealed that both viruses harbored 36 previously described resistance mutations to ATV and DRV. However, the WT and 37 MAX virus proteases showed different resistance variant repertoires, with the G16E and 38 V77I substitutions observed only in WT, and the L33F, S37P, G48L, Q58E/K, and L89I 39 substitutions detected only in MAX. Remarkably, G48L and L89I are rarely found in 40 vivo in PI-treated patients. The MAX virus showed significantly higher nucleotide and 41 amino acid diversity of the propagated viruses with and without PIs (P < 0.0001), 42 suggesting higher selective pressure for change in this recoded virus. Our results 43 indicate that HIV-1 protease position in sequence space delineates the evolution of its 44 mutant spectra. Nevertheless, the investigated synonymously recoded variant showed 45 mutational robustness and evolvability similar to the WT virus. 46 47 IMPORTANCE 48 Large-scale synonymous recoding of virus genomes is a new tool for exploring various 49 aspects of virus biology. Synonymous virus genome recoding can be used to investigate 50 . CC-BY-NC-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/315366 doi: bioRxiv preprint 3 how a virus's position in sequence space defines its mutant spectrum, evolutionary 51 trajectory, and pathogenesis. In this study, we evaluated how synonymous recoding of 52 the human immunodeficiency virus type 1 (HIV-1) protease impacts the development of 53 protease inhibitor (PI) resistance. HIV-1 protease is a main target of current 54 antiretroviral therapies. Our present results demonstrate that the wild-type (WT) virus 55 and the virus with the recoded protease exhibited different patterns of resistance 56 mutations after PI treatment. Nevertheless, the developed PI resistance phenotype was 57 indistinguishable between the recoded virus and the WT virus, suggesting that the 58 synonymously recoded protease HIV-1 and the WT protease virus were equally robust 59 and evolvable. \"},\n"," {'paper_id': '64dc33172e117661b2b2dcd252a92df929c98a60',\n","  'section': 'Abstract',\n","  'text': \"Early warning signals (EWS) identify systems approaching a critical transition, where the system undergoes a sudden change in state. For example, monitoring changes in variance or autocorrelation offers a computationally inexpensive method which can be used in real-time to assess when an infectious disease transitions to elimination. EWS have a promising potential to not only be used to monitor infectious diseases, but also to inform control policies to aid disease elimination. Previously, potential EWS have been identified for prevalence data, however the prevalence of a disease is often not known directly. In this work we identify EWS for incidence data, the standard data type collected by the Centers for Disease Control and Prevention (CDC) or World Health Organization (WHO). We show, through several examples, that EWS calculated on simulated incidence time series data exhibit vastly different behaviours to those previously studied on prevalence data. In particular, the variance displays a decreasing trend on the approach to disease elimination, contrary to that expected from critical slowing down theory; this could lead to unreliable indicators of elimination when calculated on real-world data. We derive analytical predictions which can be generalised for many epidemiological systems, and we support our theory with simulated studies of disease incidence. Additionally, we explore EWS calculated on the rate of incidence over time, a property which can be extracted directly from incidence data. We find that although incidence might not exhibit typical critical slowing down properties before a critical transition, the rate of incidence does, presenting a promising new data type for the application of statistical indicators. The threat posed by infectious diseases has a huge impact on our global society. It is therefore critical to monitor infectious diseases as new data become available during control campaigns. One obstacle in observing disease emergence or elimination is understanding what influences noise in the data and how this fluctuates when near to zero cases. The standard data type collected is the number of new cases per day/month/year but mathematical modellers often focus on data such as the total number of infectious people, due to its analytical properties. We have developed a methodology to monitor the standard type of data to inform whether a disease is approaching emergence or disease elimination. We have shown computationally how : bioRxiv preprint 453 Model 1 & 2, exhibits a near perfect ability to identify the increasing trend. An AUC score of 0 demonstrates that the time series trend is instead decreasing and 455 as such it doesn't correspond to the predetermined prediction. A perfectly diagnosed 456 decreasing indicator when compared to the null model will result in zero sensitivity 457 under these conditions and an AUC score of 0. Fig. 3 highlights which indicators are in 458 some cases increasing (AUC close to one), decreasing (AUC close to zero) or are poor 459 \"},\n"," {'paper_id': '652b887cd2561765edc651debca32ae298660917',\n","  'section': 'Abstract',\n","  'text': 'The pandemic of the coronavirus disease 2019 has become a global public health crisis. COVID-19 is marked by its rapid progression from mild to severe conditions, particularly in the absence of adequate medical care. However, the physiological changes associated with COVID-19 are barely understood. In this study, we performed untargeted metabolomic and lipidomic analyses of plasma from a cohort of COVID-19 patients who had experienced different symptoms. We found the metabolite and lipid alterations exhibit apparent correlation with the course of disease in these COVID-19 patients, indicating that the development of COVID-19 affected patient metabolism. Moreover, many of the metabolite and lipid alterations, particularly ones associated with hepatic functions, have been found to align with the progress and severity of COVID-19. This work provides valuable knowledge about blood biomarkers associated with COVID-19 and potential therapeutic targets, and presents important resource for further studies of COVID-19 pathogenesis. : medRxiv preprint 4 critical illness, metabolic and lipid abnormalities are commonly observed, which are believed to contribute to physiology and pathology. Moreover, previous studies have demonstrated dramatic alterations of metabolome and lipidome in human plasma caused by various diseases including viral infections, such as Ebola virus disease (8, 9). Here, we performed the untargeted metabolomic and lipidomic profilings of plasma samples collected from a cohort of COVID-19 patients, including COVID-19 fatalities and survivors recovered from mild or severe symptoms. Our findings here show many of the metabolite and lipid alterations, particularly ones associated with hepatic functions, align with the progress and severity of the disease, which would provide valuable knowledge about blood biomarkers associated with COVID-19 as well as potential therapeutic targets, and shed light into the pathogenesis of COVID-19. '},\n"," {'paper_id': '653908266458cee18030948824399fefea36d8da',\n","  'section': 'Abstract',\n","  'text': 'In this letter we analyze the real-time infection data of COVID-19 epidemic for nine nations. We observe that till 27 March 2020, the number of infected individuals (I(t)) in USA, Spain, Germany, Iran, France, and India are growing exponentially. On the contrary, I(t) curves for China and South Korea exhibits power law behavior before flattening of the curve. The derivativeİ(t), which is also the daily infection count, is proportional to I(t) for the exponential regime, but not for the power law regime. These valuable indicators could be used for epidemic forecast. We also argue that long-term community transmission and/or the transmission by asymptomatic carriers traveling long distances may be inducing the power law growth of the epidemic. '},\n"," {'paper_id': '655ce1a9137ef415bf0c2b7876598da8fdbaaf78',\n","  'section': 'Abstract',\n","  'text': 'The virus that was first discovered in China Wuhan Province about 3 months ago (first cases were reported in Wuhan on December 31st 2019) has spread world wide. The six (6) top countries (excluding China) most affected so far include; USA, Italy, Spain, Germany, France and Iran. With Italy showing the highest death toll. In Uganda where it was discovered on 19/3/2020 with one (01) case has just in nine (9) days, increased to thirty (30) infected individuals. This model is a wake up call over the rate at which COVID-19 is likely to spread throughout the country. Thus it is a guide for policymakers and planners to benchmark on for solutions to this deadly virus. '},\n"," {'paper_id': '656ef218f7dcea5555a59cdc29d0f9ec8d5c2a82',\n","  'section': 'Abstract',\n","  'text': \"Coronaviruses encode a mix of highly conserved and novel genes as well as genetic elements 2 necessary for infection and pathogenesis, raising the possibility for common targets for 3 attenuation and therapeutic design. In this study, we focus on the highly conserved non-4 structural protein (NSP) 16, a viral 2'O methyl-transferase (MTase) that encodes critical 5 functions in immune modulation and infection. Using reverse genetics, we disrupted a key motif 6 in the conserved KDKE motif of MERS NSP16 (D130A) and evaluated the effect on viral 7 infection and pathogenesis. While the absence of 2'O MTase activity had only marginal impact 8 on propagation and replication in Vero cells, the MERS dNSP16 mutant demonstrated 9 significant attenuation relative to control both in primary human airway cultures and in vivo. 10 Further examination indicated the MERS dNSP16 mutant had a type I IFN based attenuation 11 and was partially restored in the absence of IFIT molecules. Importantly, the robust attenuation 12 permitted use of MERS dNSP16 as a live attenuated vaccine platform protecting from challenge 13 with a mouse adapted MERS-CoV strain. These studies demonstrate the importance of the 14 conserved 2'O MTase activity for CoV pathogenesis and highlight NSP16 as a conserved 15 universal target for rapid live attenuated vaccine design in an expanding CoV outbreak setting. 16 All rights reserved. No reuse allowed without permission. Significance 1 Coronavirus emergence in both human and livestock represents a significant threat to global 2 public health, as evidenced by the sudden emergence of SARS-CoV, MERS-CoV, PEDV and 3 swine delta coronavirus in the 21 st century. These studies describe an approach that effectively 4 targets the highly conserved 2'O methyl-transferase activity of coronaviruses for attenuation. 5 With clear understanding of the IFN/IFIT based mechanism, NSP16 mutants provide a suitable 6 target for a live attenuated vaccine platform as well as therapeutic development for both current 7 and future emergent CoV strains. Importantly, other approaches targeting other conserved pan-8 coronavirus functions have not yet proven effective against MERS-CoV, illustrating the broad 9 applicability of targeting viral 2'O MTase function across coronaviruses. \"},\n"," {'paper_id': '661003bddb5c847237ff4a1f34a823f7971e2090',\n","  'section': 'Abstract',\n","  'text': \"Many recombinant vector vaccines are capable of replication within the host. They consist of a 5 fully competent vector backbone engineered to express an antigen from a foreign transgene. From the 6 perspective of viral replication, the transgene is not only dispensable but may even be intrinsically 7 detrimental. Thus vaccine revertants that delete the transgene may evolve to dominate the within-8 host population and in doing so reduce the antigenicity of the vaccine. We apply mathematical and 9 computational models to study this process, including the dynamics of vaccine and revertant growth 10 plus the dynamics of innate and adaptive immunity. Although the selective basis of vaccine evolution is 11 easy to comprehend, the immunological consequences are not. One complication is that, despite possible 12 fitness differences between vaccine and revertant, the opportunity for vaccine evolution is limited by the 13 short period of growth before the viral population is cleared. Even less obvious, revertant per se does not 14 interfere with immunity to vaccine except as the revertant suppresses vaccine abundance; the magnitude 15 of this interference depends on mechanisms and timing of viral suppression. Adaptive immunity targeting 16 the foreign antigen is also a possible basis of vaccine inferiority, but it is not worsened by vaccine evolution. Overall, we find that within-host vaccine evolution can sometimes matter to the adaptive immune response 18 targeting the foreign antigen, but even when it does matter, simple principles of vaccine design and the 19 control of inoculum composition can largely mitigate the effects. 20 Author Summary 21 Recombinant vector vaccines are live replicating viruses that are engineered to carry extra genes derived from 22 a pathogen -and these produce proteins against which we want to generate immunity. These genes may 23 evolve to be lost during the course of replication within an individual, and there is a concern that this can 24 severely limit the vaccine's efficacy. The dynamics of this process are studied here with mathematical models. The potential for vaccine evolution is somewhat reduced by the short-term growth of the vaccine population 26 before it is suppressed by the immune response. Even when within-host evolution can be a problem, the 27 1 . CC-BY 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/545087 doi: bioRxiv preprint models show that increasing the vaccine inoculum size or ensuring that the inoculum is mostly pure vaccine 28 can largely avoid the loss of immunity arising from evolution. 29 \"},\n"," {'paper_id': '66bc90ad788ec702b6d3e930016b56747af81c7c',\n","  'section': 'Abstract',\n","  'text': \"The ongoing pandemic of the coronavirus disease 2019 started in China and devastated a vast majority of countries. In India, COVID-19 cases are steadily increasing since January 30, 2020, and the government-imposed lockdown across the country to curtail community transmission. COVID-19 forecasts have played an important role in capturing the probability of infection and the basic reproduction rate. In this study, we predicted some trajectories of trajectories associated with COVID-19 in the coming days in India using an Autoregression integrated moving average model (ARIMA) and Richard's model. By the end of April 2020, the incidence of new cases is predicted to be 5200 (95% CI: 4650 to 6002) through the ARIMA model versus be 6378 (95% CI: 4904 to 7851) Richard model. We estimated that there would be a total of 197 (95% CI: 118 to 277) deaths and drop down in the recovery rates will reach around 501 (95% CI: 245 to 758) by the end of April 2020. These estimates can help to strengthen the implementation of strategies to increase the health system capacity and enactment of social distancing measures all over India. \"},\n"," {'paper_id': '66f8cfe75dce9b250b555fe8d136cf006f55649a',\n","  'section': 'Abstract',\n","  'text': \"Various epidemic prevention and control measures aimed at reducing person-to-person contact has paid a certain cost while controlling the epidemic. So accurate evaluation of these measures helps to maximize the effectiveness of prevention and control while minimizing social costs. In this paper, we develop the model in Dirk Brockmann and Dirk Helbing (2013) to theoretically explain the impact mechanism of traffic control and social distancing measures on the spread of the epidemic, and empirically tests the effect of the two measures in China at the present stage using econometric approach. We found that both traffic control and social distancing measures have played a very good role in controlling the development of the epidemic. Nationally, social distancing measures are better than traffic control measures; the two measures are complementary and their combined action will play a better epidemic prevention effect; Traffic control and social distancing do not work everywhere. Traffic control only works in cities with higher GDP per capita and population size, while fails in cities with lower GDP per capita and population size. In cities with lower population size, social distancing becomes inoperative; the rapid and accurate transmission of information, a higher protection awareness of the public, and a stronger confidence of residents in epidemic prevention can promote the realization of the measure effects. The findings above verify the effectiveness and correctness of the measures implemented in China at present, at the same time, we propose that it is necessary to fully consider the respective characteristics of the two measures, cooperating and complementing each other; what's more, measures should be formulated according to the city's own situation, achieving precise epidemic prevention; Finally, we should increase the transparency of information, improve protection awareness of the public, guide emotions of the public in a proper way, enhancing public confidence. \"},\n"," {'paper_id': '670790eb6d158a57450705bcd87218bf479ed3be',\n","  'section': 'Abstract',\n","  'text': 'We estimated the fraction and timing of presymptomatic transmissions of COVID19 with mathematical models combining the available data of the incubation period and serial interval. We found that up to 79.7% transmissions could be presymptomatic among the imported cases in China outside Wuhan. The average timing of presymptomatic transmissions is 3.8 days (SD = 6.1) before the symptom onset, which is much earlier than previously assumed. All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '6710c917f9b40e2e593c6f05b5aad788ae1eadb7',\n","  'section': 'Abstract',\n","  'text': 'The COVID-19 epidemic in China has been effectively controlled. It is of great significance to study the law of cross-regional spread of the epidemic, for the prevention and control of the COVID-19 in the future in China and other countries or regions. In this study, the cross-regional connection intensity between cities was characterized based on the probability and the effective distance of the shortest path tree, and the empirical analysis was carried out based on the high-frequency data such as the cases of COVID 19 outbreaks. It is concluded that the higher the intensity of inter-city connection, the larger scale the cross-regional spread of the epidemic. '},\n"," {'paper_id': '671b44ae9bec89cacf7ae9e7b2a79649a9b66773',\n","  'section': 'Abstract',\n","  'text': 'On December 31, 2019, the World Health Organization was notified about a cluster of pneumonia of unknown aetiology in the city of Wuhan, China. Chinese authorities later identified a new coronavirus (2019-nCoV) as the causative agent of the outbreak. As of January 23, 2020, 655 cases have been confirmed in China and several other countries. Understanding the transmission characteristics and the potential for sustained human-to-human transmission of 2019-nCoV is critically important for coordinating current screening and containment strategies, and determining whether the outbreak constitutes a public health emergency of international concern (PHEIC). We performed stochastic simulations of early outbreak trajectories that are consistent with the epidemiological findings to date. We found the basic reproduction number, R 0 , to be around 2.2 (90% high density interval 1.4-3.8), indicating the potential for sustained human-to-human transmission. Transmission characteristics appear to be of a similar magnitude to severe acute respiratory syndrome-related coronavirus (SARS-CoV) and the 1918 pandemic influenza. These findings underline the importance of heightened screening, surveillance and control efforts, particularly at airports and other travel hubs, in order to prevent further international spread of 2019-nCoV. '},\n"," {'paper_id': '672865cb53271ddd7dd4a32a9c35801825b73fcb',\n","  'section': 'Abstract',\n","  'text': 'Background: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is a virus that causes severe respiratory illness in humans, which eventually results in the current outbreak of novel coronavirus disease around the world. The research community is interested to know what are the hotspots in coronavirus (CoV) research and how much is known about COVID-19. This study aimed to evaluate the characteristics of publications involving coronaviruses as well as COVID-19 by using a topic modeling analysis. : We extracted all abstracts and retained the most informative words from the COVID-19 Open Research Dataset, which contains all the 35,092 pieces of coronavirus related literature published up to March 20, 2020. Using Latent Dirichlet Allocation modeling, we trained an eighttopic model from the corpus. We then analyzed the semantic relationships between topics and compared the topic distribution between COVID-19 and other CoV infections. Results: Eight topics emerged overall: clinical characterization, pathogenesis research, therapeutics research, epidemiological study, virus transmission, vaccines research, virus diagnostics, and viral genomics. It was observed that COVID-19 research puts more emphasis on clinical characterization, epidemiological study, and virus transmission at present. In contrast, topics about diagnostics, therapeutics, vaccines, genomics and pathogenesis only accounted for less than 10% or even 4% of all the COVID-19 publications, much lower than those of other CoV infections. Conclusions: These results identified knowledge gaps in the area of COVID-19 and offered directions for future research. : medRxiv preprint 6 topic coherence and human understanding. The final model is visualized as a bubble graph in Fig. 2, with one bubble representing one topic and the size indicating its popularity. As shown in the figure, all the bubbles are completely separate from each other, suggesting the suitability of the model. Increasing the number of topics would make each individual topic more specific and might increase overlap between topics. Decreasing the number of topics would result in topics to be more high-level abstract. Figure 2. Visualization of the eight-topic model trained on the whole corpus. One bubble represents one topic. The topics are ordered based on prevalence in the corpus. Inset: the coherence scores of models with different numbers of topics. '},\n"," {'paper_id': '672ca9b6e50e26c49fc7be2ef3b4f685badf9db1',\n","  'section': 'Abstract',\n","  'text': 'This study integrates the daily intercity migration data with the classic Susceptible-Exposed-Infected-Removed (SEIR) model to construct a new model suitable for describing the dynamics of epidemic spreading of Coronavirus Disease 2019 in China. Daily intercity migration data for 367 cities in China are collected from Baidu Migration, a mobile-app based human migration tracking data system. Historical data of infected, recovered and death cases from official source are used for model fitting. The set of model parameters obtained from best data fitting using a constrained nonlinear optimization procedure is used for estimation of the dynamics of epidemic spreading in the coming weeks. Our results show that the number of infections in most cities in China will peak between mid February to early March 2020, with about 0.8%, less than 0.1% and less than 0.01% of the population eventually infected in Wuhan, Hubei Province and the rest of China, respectively. '},\n"," {'paper_id': '67a2ae408efcb6686a22ae27435c7e4fa10c4a21',\n","  'section': 'Abstract',\n","  'text': 'Introduction: More than 93,000 cases of coronavirus disease have been reported worldwide. We describe the epidemiology, clinical course, and virologic characteristics of the first 12 U.S. patients with COVID-19. We collected demographic, exposure, and clinical information from 12 patients confirmed by CDC during January 20-February 5, 2020 to have COVID-19. Respiratory, stool, serum, and urine specimens were submitted for SARS-CoV-2 rRT-PCR testing, virus culture, and whole genome sequencing. Results: Among the 12 patients, median age was 53 years (range: 21-68); 8 were male, 10 had traveled to China, and two were contacts of patients in this series. Commonly reported signs and symptoms at illness onset were fever (n=7) and cough (n=8). Seven patients were hospitalized with radiographic evidence of pneumonia and demonstrated clinical or laboratory signs of worsening during the second week of illness. Three were treated with the investigational antiviral remdesivir. All patients had SARS-CoV-2 RNA detected in respiratory specimens, typically for 2-3 weeks after illness onset, with lowest rRT-PCR Ct values often detected in the first week. SARS-CoV-2 RNA was detected after reported symptom resolution in seven patients. SARS-CoV-2 was cultured from respiratory specimens, and SARS-CoV-2 RNA was detected in stool from 7/10 patients. In 12 patients with mild to moderately severe illness, SARS-CoV-2 RNA and viable virus were detected early, and prolonged RNA detection suggests the window for diagnosis is long. Hospitalized patients showed signs of worsening in the second week after illness onset. for use under a CC0 license. '},\n"," {'paper_id': '67c136d7af3f6957a92bb35dd8b7afd018276a20',\n","  'section': 'Abstract',\n","  'text': 'evaluated the association between changes in the incidence of AKI and COVID-19 disease and clinical outcomes by using logistic regression models. Results: A total of 287 patients, 55 with AKI and 232 without AKI, were included in the analysis. Compared to patients without AKI, AKI patients were older, predominantly male, and were more likely to present with hypoxia and have pre-existing hypertension and cerebrovascular disease. Moreover, AKI patients had higher levels of white blood cells, D-dimer, aspartate aminotransferase, total bilirubin, creatine kinase, lactate dehydrogenase, procalcitonin, C-reactive protein, a higher prevalence of hyperkalemia, lower lymphocyte counts, and higher chest computed tomographic scores. The incidence of stage 1 AKI was 14.3%, and the incidence of stage 2 or 3 AKI was 4.9%. Patients with AKI had substantially higher mortality. Conclusions: AKI is an important complication of COVID-19. Older age, male, multiple pre-existing comorbidities, lymphopenia, increased infection indicators, elevated D-dimer, and impaired heart and liver functions were the risk factors of AKI. AKI patients who progressed to stages 2 or 3 AKI had a higher mortality rate. '},\n"," {'paper_id': '67c7ba39e9529c00c7d6f82b4bad1b8858636a9d',\n","  'section': 'Abstract',\n","  'text': 'The coronavirus disease pandemic has increased the necessity of immediate clinical decisions and effective usage of healthcare resources. Currently, the most validated diagnosis test for COVID-19 (RT-PCR) is in shortage in most developing countries, which may increase infection rates and delay important preventive measures. The objective of this study was to predict the risk of positive COVID-19 diagnosis with machine learning, using as predictors only results from emergency care admission exams. We collected data from 235 adult patients from the Hospital Israelita Albert Einstein in São Paulo, Brazil, from 17 to 30 of March, 2020, of which 102 (43%) received a positive diagnosis of COVID-19 from RT-PCR tests. Five machine learning algorithms (neural networks, random forests, gradient boosting trees, logistic regression and support vector machines) were trained on a random sample of 70% of the patients, and performance was tested on new unseen data (30%). The best predictive performance was obtained by the support vector machines algorithm (AUC: 0.85; Sensitivity: 0.68; Specificity: 0.85; Brier Score: 0.16). In conclusion, we found that targeted decisions for receiving COVID-19 tests using only routinely-collected data is a promising new area with the use of machine learning algorithms. '},\n"," {'paper_id': '68041db129253fd6ea54686d016952e8821dcfb8',\n","  'section': 'Abstract',\n","  'text': 'COVID-19, SARS, and MERS are featured by fibrinolytic dysfunction. To test the role of the fibrinolytic niche in the regeneration of alveolar epithelium, we compared the self-renewing capacity of alveolar epithelial type 2 (AT2) cells and its differentiation to AT1 cells between wild type (wt) and fibrinolytic niche deficient mice (Plau -/and Serpine1 Tg ). A significant reduction in both proliferation and differentiation of deficient AT2 cells was observed in vivo and in 3D organoid cultures. This decrease was mainly restored by uPA derived A6 peptide, a binding fragment to CD44 receptors. The proliferative and differential rate of CD44 + AT2 cells was greater than that of CD44controls. There was a reduction in transepithelial ion transport in deficient monolayers compared to wt cells. Moreover, we found a marked suppression in total AT2 cells and CD44 + subpopulation in lungs from brain dead patients with acute respiratory distress syndrome (ARDS) and a mouse model infected by influenza viruses. Thus, we demonstrate that the fibrinolytic niche can regulate AT2-mediated homeostasis and regeneration via a novel uPA-A6-CD44 + -ENaC cascade. Keywords: urokinase plasminogen activator, alveolar progenitor epithelial cells, alveologenesis, 3D organotypic culture, polarized monolayer, acute lung injury, alveolar epithelium author/funder. All rights reserved. No reuse allowed without permission. : bioRxiv preprint RESULTS To examine the effects of fibrinolytic niche on AT2 cells in normal and injured lungs, we infected wt and Plau -/mice with the PR8 H1N1 type A influenza virus intranasally24 , 25. The severity of injured regions was classified as mild, moderate, and severe based on the lung injury score26. Randomly selected fields were analyzed with the ImageJ software to count proSPC + AT2 cells (green) and the total cells for wt (Fig. 1A) and Plau -/mice (Fig. 1B) . Although AT1 cells were stained with anti-pdpn antibody, it was not possible to clearly distinguish from other non-epithelial cells, i.e., basal cells, leukocytes, endothelial cells, and fibroblasts due to considerable overlap. author/funder. All rights reserved. No reuse allowed without permission. : bioRxiv preprint 4 AT2 cells per field of wt lungs were approximately two-fold (12.6 ± 0.55%) that of Plau -/lung preparations (6.0 ± 0.35 %, n=26 from 8 different mice). Moreover, influenza-infected lungs displayed widespread alveolar collapse and increased thickness of the lung interstitial septa in a severity-dependent manner. AT2 cells per field were significantly reduced in both moderate and severe injury regions as compared to wt controls (Fig. 1A-B) . Similarly, this finding was observed in Plau -/mice ( Fig. 1B-C) . AT2 cells per field for Plau -/mice were markedly fewer than in wt animals (Fig. 1C. n=9 mice, 5 fields per mice). This is consistent with a reduction in the yield of total AT2 cells (1.7 ´ 10 6 cells for Plau -/mice vs 2.6 ´ 10 6 cells for wt mice, Fig. 1D) . We compared CD44 + AT2 cells between healthy and ARDS patients (n = 3 patients per group). As shown by fluorescenceactivated cell sorting (FACS) data, CD44 + AT2 cells were reduced in ARDS patients (Fig. 1E-G) . These results suggest that Plau -/mice may mimic impaired fibrinolytic niches in the lung of ARDS patients. To determine whether the Plau gene is required for re-alveolarization in vitro, we quantified spheroids formed by primary AT2 cells from both wt and Plau -/mice in parallel ( Fig. 2A) . Organoids were captured as 4´ DIC images from 4 to 12 days post seeding (Fig. S2) . Organoids with a diameter > 50 µm (Fig. 2B) and colony forming efficiency (CFE) (Fig. 2C) were significantly decreased (n = 12, P < 0.01) in Plau -/cultures compared with wt controls. Apparently, the organoids with a diameter ranging from 50 -200 μm resulted in a reduction in colonies and CFE (Fig. 2D) . The suppression in organoid formation associated with Plau -/cells contributed to the reduction of total surface area, a clinical parameter for epithelial repair and development (Fig. 2E) . The large organoids were filled with culture medium, having a smaller lumen with thicker walls for Plau -/cultures over wt controls (Fig. 2F-G) . These data provide direct evidence for the regulation of AT2-mediated re-alveologenesis by uPA. In addition, the organoids were further visualized with cell-permeable green fluorescent calcein to measure the cystic fibrosis transmembrane conductance regulator (CFTR) and ENaC activity (Fig. S3) . The surface area was not altered in 30 minutes in the presence of cAMP-elevating forskolin, CFTRinh, and amiloride. One potential explanation is that these organoids do not develop tight junctions intercellularly as well as polarized monolayers. author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '681c5099b2afb08fe5cb2f58d92e9c6e4873bec2',\n","  'section': 'Abstract',\n","  'text': 'Cellular membranes display an incredibly diverse range of shapes, both in the plasma membrane and at membrane bound organelles. These morphologies are intricately related to cellular functions, enabling and regulating fundamental membrane processes. However, the biophysical mechanisms at the origin of these complex geometries are not fully understood from the standpoint of membrane-protein coupling. In this work, we focused on a minimal model of helicoidal ramps representative of specialized endoplasmic reticulum compartments. Given a helicoidal membrane geometry, we asked what is the distribution of spontaneous curvature required to maintain this shape at mechanical equilibrium? Based on the Helfrich energy of elastic membranes with spontaneous curvature, we derived the shape equation for minimal surfaces, and applied it to helicoids. We showed the existence of switches in the sign of the spontaneous curvature associated with geometric variations of the membrane structures. Furthermore, for a prescribed gradient of spontaneous curvature along the exterior boundaries, we identified configurations of the helicoidal ramps that are confined between two infinitely large energy barriers. Overall our results suggest possible mechanisms for geometric control of helicoidal ramps in membrane organelles based on curvatureinducing proteins. '},\n"," {'paper_id': '683416e300a4f4075835852558855b07c342bd72',\n","  'section': 'Abstract',\n","  'text': 'Zika virus (ZIKV) is a global public health issue due to its association with severe 33 developmental disorders in infants and neurological disorders in adults. Because ZIKV uses 34 glycosylation of its envelope (E) protein to interact with host cell receptors to facilitate entry, 35 these interactions could also be important for designing therapeutics and vaccines. Due to a lack 36 of information about Asn-linked (N-glycans) on ZIKV E, we analyzed ZIKV E of various strains 37 derived from different cells. ZIKV E proteins are extensively modified with oligomannose-, 38 hybrid-and complex-N-glycans of a highly heterogeneous nature. Host cell-surface glycans 39 correlated strongly with the glycomic features of ZIKV E. Mechanistically, we discovered that 40 ZIKV N-glycans are important in viral pathogenesis, as mannose-specific C-type lectins DC-41 SIGN and L-SIGN mediate cell entry of ZIKV. Our findings represent the first detailed mapping 42 of N-glycans on ZIKV E of various strains and their functional significance. 43 44 All rights reserved. No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/564542 doi: bioRxiv preprint 45 Zika virus (ZIKV) is mainly transmitted to humans via infected mosquitoes, though other 46 transmission routes, such as through placenta and sexual intercourse can also occur 1,2 . Recently, 47 the considerable increase in ZIKV infection rates has raised urgent global urgent concerns, 48 especially after the outbreaks in Yap Islands (2007) 3,4 , French Polynesia (2013) 5,6 , Easter Island 49 (2014) 7 , and Brazil (2015) 8,9 . These ZIKV outbreaks were associated with a sharp increase in 50 cases of Guillain Barre Syndrome (GBS), an autoimmune disease featured by weakening and 51 paralysis of the limbs and face. In 2015, Zika spread to South and Central America, infecting 52 thousands of people in Brazil and Colombia, where it was associated with an increase in GBS 53 rates as well as a significant increase in severe fetal abnormalities including spontaneous 54 abortion, stillbirth, hydrocephaly, microcephaly, hydranencephaly/hydrops fetalis, and placental 55 insufficiency 10-14 . ZIKV is a small, enveloped positive-strand RNA virus 15 . The genomic RNA 56 of this mosquito-borne flavivirus contains a single open reading frame (ORF) that encodes a 57 polyprotein 16,17 . This polyprotein undergoes co-and post-translational processing to produce 58 three structural proteins, which are capsid (C), pre-membrane (prM) and envelope (E), in 59 addition to seven non-structural proteins (NS1, NS2A, NS2B, NS3, NS4A, NS4B, and NS5) 17,18 . 60 Of the three structural proteins, E is the major surface glycoprotein, containing three domains (I, 61 II, III) and two transmembrane helices 17-19 . The majority of flaviviruses E proteins are post-62 translationally modified by N-glycans at amino acid 153/154 within a highly conserved 63 glycosylation motif of N-X-T/S at positions 154-156 (where X is any amino acid except proline) 64 16 . This is a primary target of neutralizing antibodies and is required for virus entry 20 . 65 Previously, it was demonstrated that the N-glycosylation sites on the E protein of flaviviruses are 66 highly conserved (Supplementary Fig. 1) , playing a vital role in both infectivity and assembly 67 All rights reserved. No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/564542 doi: bioRxiv preprint 6 two ZIKV E proteins from African lineage strains, MR766 and IbH, displayed similar N-glycan 102 profiles across the six cell lines that were tested (Supplementary Fig. 4a, b) . When produced in 103 C6/36, Vero, THP-1, and SNB19 cells, these N-glycan profiles were heavily dominated by 104 sialylated glycans. For these four cell lines, bi-and tri-sialylated tri-antennary N-glycans (3241.8 105 m/z and 3603.0 m/z) were the most abundant found on ZIKV E, followed by bi-sialylated bi-106 antennary (2792.5 m/z) and tetra-sialylated tri-antennary (3964.2 m/z) N-glycans 107 (Supplementary Fig. 4a, b) . ZIKV E produced in SNB19 cells was also found to be decorated 108 with Man5 (1579.9 m/z), agalactosylated bisected (1907.1 m/z), core-fucosylated non-sialylated 109 bi-antennary (2244.3 m/z), and core-fucosylated mono-sialylated bi-antennary (2605.5 m/z) N-110 glycans (Supplementary Fig. 4a, b). When expressed in the LLC-MK2 and JEG-9 cell lines, the 111 N-glycan profiles of MR766 and IbH strains of E were markedly different. When produced in 112 LLC-MK2 cells, bi-sialylated bi-antennary (2792.5 m/z) and core-fucosylated agalactosylated 113 (1836.0 m/z) N-glycans were by far the two most abundant glycan species (Supplementary Fig. 114 4a, b). More complex N-glycans, with three or more sialic acids and additional lactosamine 115 repeats, were not identified in ZIKV E expressed in LLC-MK2 cells. Finally, the ZIKV E 116 produced in JEG-3 cells revealed a different N-glycosylation pattern between MR766 and IbH 117 strains. While the di-and tri-sialylated N-glycans (2792.5, 3241.8, and 3603.0 m/z) were the 118 most abundant for the MR766 ZIKV-E produced in JEG-3, the variety of N-glycans identified in 119 JEG-3 cells was greater, notably including Man 5 GlcNAc 2 (1579.9 m/z), Man 6 GlcNAc 2 (1784.0 120 m/z), Man 7 GlcNAc 2 (1988.1 m/z), Man 8 GlcNAc 2 (2192.2 m/z), asialo bi-antennary (2070.2 121 m/z), and core fucosylated asialo-bi-antennary (2244.3 m/z) N-glycans (Supplementary Fig.   122 4a). In contrast, the IbH strain produced in the JEG-3 cells displayed a more restricted N-glycan 123 profile, with the notable absence of sialylated N-glycans and with the core fucosylated asialo-bi-124 All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '685cdc49b5db13ec22f0eda615b026bf59cd23f4',\n","  'section': 'Abstract',\n","  'text': 'Viruses are known to have some of the highest and most diverse mutation rates found 12 in any biological replicator, with single-stranded (ss) RNA viruses evolving the fastest, 13 and double-stranded (ds) DNA viruses having rates approaching those of bacteria. As 14 mutation rates are tightly and negatively correlated with genome size, selection is a clear 15 driver of viral evolution. However, the role of intragenomic interactions as drivers of 16 viral evolution is still unclear. To understand how these two processes affect the long-17 term evolution of viruses infecting humans, we comprehensively analyzed ssRNA, ssDNA, 18 dsRNA, and dsDNA viruses, to find which virus types and which functions show evidence 19 for episodic diversifying selection and correlated evolution. We show that selection mostly 20 affects single stranded viruses, that correlated evolution is more prevalent in DNA viruses, 21 and that both processes, taken independently, mostly affect viral replication. However, 22 the genes that are jointly affected by both processes are involved in key aspects of their life 23 cycle, favoring viral stability over proliferation. We further show that both evolutionary 24 processes are intimately linked at the amino acid level, which suggests that it is the 25 joint action of selection and correlated evolution, and not just selection, that shapes the 26 evolutionary trajectories of viruses -and possibly of their epidemiological potential. 27 '},\n"," {'paper_id': '686135083f06748330ae6bcd283556b00ca6925c',\n","  'section': 'Abstract',\n","  'text': \"Objectives. Assess how people perceive the risks of coronavirus infection, whether people take preventive measures, and which (pre-outbreak) factors contribute to the perceived risks and measures taken, such as (pre-outbreak) respiratory problems, heart problems, diabetes, anxiety and depression symptoms, loneliness, age, gender, marital status and education level. Methods. Data were collected in the longitudinal LISS panel, based on a random sample of the Dutch population. The coronavirus survey started on March 2, and the data collection ended on March 17 2020. Data were linked with surveys on health and social integration conducted at the end of 2019 (N study sample =3,540). Results. About 15% perceived the risk of infection as high, and 11% the risk becoming ill when infected. Multivariate logistic regression analyses showed the following. Older age-groups perceived the risk for coronavirus infection as lower (all adjusted Odd Ratio's [aOR] ≤ .058). In total, 43.8% had taken preventive measures, especially females (aOR=1.47, 95% CI=1.27-1.69). Those with lower education levels less often used preventive measures (aOR=0.56, 95% CI=0.46-0.69). Those with preoutbreak respiratory problems (aOR=2.89, 95% CI=2.24-3.73), heart problems (aOR=2.17, 95% CI=1.49-3.16) and diabetes (aOR=3.10, 95% CI=2.02-4.74) perceived the risk becoming ill when infected as higher than others. However, respondents with pre-outbreak respiratory problems and diabetes did not more often take preventive measures. Vulnerable patients more often recognize that they are at risk becoming ill when infected by the coronavirus, but many do not take preventive measures. Interventions to stimulate the use of preventive measures should pay additional attention to physically vulnerable patients, men and those with lower education levels. \"},\n"," {'paper_id': '68a0699074f22efda1d55a5a52d31bcc7c0dbb7b',\n","  'section': 'Abstract',\n","  'text': 'To examine the impact of providing healthcare during or after health emergencies caused by viral epidemic outbreaks on healthcare workers´(HCWs) mental health, and to assess the available evidence base regarding interventions to reduce such impact. Design: Systematic rapid review and meta-analysis. Data sources: MEDLINE, Embase, and PsycINFO, searched up to 23 March 2020. We selected observational and experimental studies examining the impact on mental health of epidemic outbreaks on HCWs. One reviewer screened titles and abstracts, and two reviewers independently reviewed full texts. We extracted study characteristics, symptoms, prevalence of mental health problems, risk factors, mental health interventions, and its impact. We assessed risk of bias for each individual study and used GRADE to ascertain the certainty of the evidence. We conducted a narrative and tabulated synthesis of the results. We pooled data using random-effects meta-analyses to estimate the prevalence of specific mental health problems. We included 61 studies (56 examining impact on mental health and five about interventions to reduce such impact). Most were conducted in Asia (59%), in the hospital setting (79%), and examined the impact of the SARS epidemic (69%). The pooled prevalence was higher for anxiety (45%, 95% CI 21 to 69%; 6 studies, 3,373 participants), followed by depression (38%, 95% CI 15 to 60%; 7 studies, 3,636 participants), acute stress disorder (31%, 95% CI 0 to 82%, 3 studies , 2,587 participants), burnout (29%, 95% CI 25 to 32%; 3 studies; 1,168 participants), and post-traumatic stress disorder (19%, 95% CI 11 to 26%, 10 studies, 3,121 participants). Based on 37 studies, we identified factors associated with the likelihood of developing those problems, including sociodemographic (younger age and female gender), social (lack of social support, social rejection or isolation, stigmatization), and occupational (working in a high risk environment (frontline staff), specific occupational roles (e.g., nurse), and lower levels of specialised training, preparedness and job experience) factors. Five studies reported interventions for frontline HCW, two of which were educational and aimed to prevent mental health problems by increasing HCWs´ resilience. These interventions increased confidence in support and training, pandemic self-efficacy, and interpersonal problems solving (very low certainty). One multifaceted intervention implemented training and organisational changes) targeted at hospital nurses during the SARS epidemic, reporting improvements in . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': '68e509d42a7d3349550512d80eca4e2b2123613f',\n","  'section': 'Abstract',\n","  'text': 'A novel coronavirus (2019-nCov) was identified in Wuhan, Hubei Province, China in December of 2019. This new coronavirus has resulted in thousands of cases of lethal disease in China, with additional patients being identified in a rapidly growing number internationally. 2019-nCov was reported to share the same receptor, Angiotensin-converting enzyme 2 (ACE2), with SARS-Cov. Here based on the public database and the state-of-the-art single-cell RNA- '},\n"," {'paper_id': '691da9d5576aa959e22114a1ebb35548efd276a0',\n","  'section': 'Abstract',\n","  'text': \"The Eyring's rate process theory and free volume concept, two very popular theories in chemistry and physics fields, are employed to treat infectious disease transmissions. The susceptible individuals are assumed to move stochastically from one place to another. The virus particle transmission rate is assumed to obey the Eyring's rate process theory and also controlled by how much free volume available in a system. The transmission process is considered to be a sequential chemical reaction, and the concentrations or fractions of four epidemiological compartments, the susceptible, the exposed, the infected, and the removed, can be derived and calculated. The obtained equations show that the basic reproduction number, R 0 , is not a constant, dependent on the volume fraction of virus particles, virus particle size, and virus particle packing structure, the energy barrier associated with susceptible individuals, and environment temperature. The developed models are applied to treat coronavirus disease 2019 (Covid-19) transmission and make predictions on peak time, peak infected, and R 0 . Our work provides a simple and straightforward approach to estimate how infection diseases evolve and how many people may be infected. \"},\n"," {'paper_id': '691f54c001708f6684ae7837adc749fe09404e61',\n","  'section': 'Abstract',\n","  'text': 'During the decoding of a subset of mRNAs a proportion of ribosomes 8 productively shift to the -1 reading frame at a specific, slippage-prone site 1 . While the 9 great majority of occurrences of the \"Programmed -1 Ribosomal Frameshifting\" (-1 10 PRF) involve viruses 2 , other mobile elements or retroelements 1 , a dramatic instance of 11 utilized human cellular -1 frameshifting in a non-retroelement derived mRNA has been 12 reported. The mRNA for which -1 frameshifting was claimed due to a stimulatory 13 pseudoknot is that which encodes immune functioning C-C chemokine receptor 5 14 (CCR5), the HIV-1 co-receptor 3 . The publication ascribed the importance of the reporter 15 CCR5 mRNA frameshifting directing translating ribosomes to a premature termination 16 codon leading to mRNA destabilization via the nonsense-mediated decay pathway 17 (NMD), rather than creating a C-terminally unique peptide. The same lab estimates that 18 up to 10% of eukaryotic genes may be regulated by this frameshift-NMD mechanism 4 19 and point to the potential for novel therapeutic strategies aimed at treating disease by -1 20 PRF 5 . Efficient -1 frameshifting requires a slippery site together with an appropriately 23 positioned stimulatory RNA structure. The reported CCR5 slip site (U_UUA_AAA where 24 underscores set of the reading frame) has previously been shown to facilitate high 25 levels of -1 frameshifting in vitro when positioned 5\\' of a known stimulatory pseudoknot 6 . 26 However, frameshifting efficiencies in vivo could be substantially different and further, at 27 least as of 2016, there were no known virus frameshift sites with the reported CCR5 28 shift site 1 . 29 30 We challenge the reported results by showing that frameshifting at the relevant 31 CCR5 sequence cassette does not occur above background levels and that the claimed 32 pertinent sequence is not as distinctively conserved as previously presented. Lastly, we 33 demonstrate that previously published retroelement derived human frameshift signals 34 do indeed harbor frameshift activity. 35 36 Using BLAST, we first identified orthologues of human CCR5 in the NCBI RefSeq 37 database. Sequences were aligned and, on examining the frameshift region, we 38 observed substitutions in the slippery site sequence which would likely decrease or 39 eliminate frameshifting in related organisms. Indeed, none of the nucleotides in the 40 slippery site are conserved across vertebrates (Figure 1). The predicted frameshift site 41 in CCR5 was previously reported as most highly conserved in primates, except for 42 Lemur catta 3 . Primate CCR5 CDS sequences were aligned and nucleotide conservation 43 was measured at single nucleotide resolution (Figure 2A) and smoothed in a 10 44 . CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/513333 doi: bioRxiv preprint ( Figure 2B ) nucleotide sliding window. This analysis showed that the entire CDS, not 45 just the predicted frameshift site, is highly conserved among primates. Thus, the 46 observation that the frameshift site is mostly conserved in primates does not indicate 47 that the (putative) ability to frameshift is subject to purifying selection. 48 49 Dual-luciferase vectors in which the test cassette is sandwiched between and 50 fused to both Firefly and Renilla luciferase encoding sequences, have been widely used 51 for assaying translational recoding events (Figure 3A). The frameshifting efficiency is 52 calculated by dividing Firefly luciferase activity by Renilla luciferase activity. This ratio is 53 then normalized to an in-frame control vector in which Renilla and Firefly luciferases are 54 both situated in the zero-frame to account for differences in activity between the two 55 enzymes (a ratio of a ratio) (Figure 3B). 56 57 In nearly all cases the fused test encoded polypeptide has only a small or no 58 effect on the upstream encoded luciferase activity. However, in one case luciferase 59 activity was anomalously low (unpublished) and we developed a variant vector system 60 (pSGDluc 7 ) containing a StopGo sequence directly 3\\' of the first and immediately 5\\' of 61 the second luciferase encoding sequence. Subsequently we made another variant by 62 removing a potential splice site, pSGDlucV3.0 (deposited with Addgene). Thus in earlier 63 constructs, where the insert-encoded peptides were appended to the C-terminus of 64 Renilla luciferase, the Renilla luciferase activity could dropparticularly for the short 65 zero-frame encoded peptide, causing an artificial inflation of translation recoding rates. 66 . CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/513333 doi: bioRxiv preprint This problem has been overcome by inserting flanking StopGo sequences that result in 67 co-translational excision of the insert-encoded peptides, thus releasing independent 68 Renilla and Firefly proteins that are identical in sequence for both the test and in-frame 69 control constructs. The publication on CCR5 frameshifting utilized the very first iteration 70 of a dual luciferase vector that we developed in 1998 8 . 71 72 We recloned the same CCR5 frameshift sequence tested in the original article 73 into pSGDlucV3.0. For CCR5 we used negative controls comparable to those in the 74 original article, which include a slip site mutant (UUUAAAA \\uf0e0 GCGCGCG for Y.A.K., 75 UUUAAAA \\uf0e0 AUUCAAA for G.L.), a 5`-termination control (also known as a premature 76 termination codon control or PTC control) and a 3`-termination control (also known as 77 an out-of-frame stop codon control or OOF control). Lastly, we included measurements 78 from the empty vector alone (Figure 3C). All four of these controls are meant to 79 establish a baseline for background levels of luminescence 80 81 The dual luciferase assays were carried out in the same cell line used in the 82 original article, HeLa. Renilla luciferase, the reference luciferase, is stable for all the 83 constructs (Figure 4A). Similarly Firefly luciferase activities for the in-frame controls of 84 CCR5 and a HIV-1 control are consistent as well, indicating that when forced to express 85 Firefly luciferase, both constructs produce proteins with similar activity. The Firefly 86 luciferase activity for the WT HIV-1 frameshift construct is >40 fold higher than that of 87 WT CCR5 or any of its negative controls. Furthermore, the Firefly luciferase activity for 88 . CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/513333 doi: bioRxiv preprint CCR5 and all its negative controls are all comparable to the activity of the empty vector 89 control, which is the baseline for background levels of Firefly luciferase activity ( Figure   90 4B). After normalizing the luciferase activities for the test vectors to their respective in-91 frame control vectors, we find that HIV-1 shows frameshift rates that are comparable to 92 previously reported findings in HeLa cells 9 . However, CCR5 WT frameshifting is 93 ~0.15%, indistinguishable from its negative controls, indicating that CCR5 does not 94 frameshift above background at our limit of sensitivity and is certainly far less than the 95 previously reported level of 9-11% 3 . In fact, CCR5 SSM demonstrates slightly higher 96 rates of FS than the WT, indicating that translational recoding values this low are 97 indicative of background ( Figure 4C) . These experiments were repeated in HEK293T 98 cells to control for cell-line specific results and we found that the results followed 99 identical trends to those obtained in HeLa cells (Figure 4D-F) . To ensure that these results were repeatable, the cloning and assaying of CCR5 102 frameshifting was completed independently by a different laboratory (G.L., J.F.A.), 103 which used modestly different cloning and assaying strategies. When utilizing both 104 HeLa and HEK293T cells, we found that CCR5 frameshifted at an insignificant rate as 105 well (Figure 5A-F). 106 107 Retrotransposons and retroviruses share a common origin. Ribosomal 108 frameshifting has been well documented in viruses and thus it stands to reason that 109 retrotransposons and/or derivatives of retrotransposons in the human genome could 110 . CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/513333 doi: bioRxiv preprint frameshift. Three derivatives of retrotransposons in the human genome, PNMA3 (Ma3), 111 PNMA5 (Ma5), and PEG10, have been previously demonstrated to frameshift in-vivo 112 and in-vitro 1 . All three encode zero-frame proteins with homology to the viral protein gag 113 and their trans-frame protein have similarity to gag-pol. All three share a slippery site 114 common with some viruses in the Retroviridae family, GGGAAAC. These sequences 115 were assayed in both HeLa and HEK293T cells (Figure 6). Although the rates of -1 116 PRF were slightly different from previously reported, all three signals stimulated 117 frameshifting far above background. These results further confirm the fidelity of the 118 pSGDlucV3.0 vector for assaying -1 PRF, already seen for other translational recoding 119 events 7 . 120 121 In conclusion, using an improved reporter assay system, we find no evidence of -122 1 frameshifting above background levels on the CCR5 mRNA. Additionally, a lack of 123 statistically significant purifying selection acting on the slip site (a hallmark for 124 functionally relevant frameshift signals) supports the argument that CCR5 does not 125 contain a biologically important frameshift signal. There are several possible 126 explanations for the discrepancy between the reported CCR5 results and the results 127 presented here but the most obvious one is the standard decoding of a significant 128 percentage of mRNAs generated by cryptic splicing. Methods 133 Alignment of CCR5 slippery sites 134 The NCBI mRNA and RefSeq BLAST databases were searched with blastn using the 135 human CCR5 mRNA (accession NM_000579.3) as query on December 4th, 2018 using 136 default parameters, with the number of hits expanded from 100 to 500. The best 137 sequence per organism, as determined by \\'Max Score\\', was taken up until 70% query 138 cover. These sequences were aligned with MUSCLE. This alignment was trimmed to 139 only include the putative frameshift region of CCR5 that was generated by code written 140 in Python 3. The alignment image was generated using the box-shade server 141 (https://embnet.vital-it.ch/software/BOX_form.html). 142 143 CCR5 Primate CDS analysis 144 Of the sequences obtained in the ortholog search, only those belonging to the 145 taxonomic descriptor of \\'primate\\' (taxid 9443) were used. Lemur catta sequences were 146 ignored, since the original manuscript states that the CCR5 frameshift signal is not 147 conserved in Lemur catta. All CDS sequences were aligned with MUSCLE and then 148 either at single nucleotide resolution or a smoothed 10 nucleotide window plotted in 149 Python 3. doi: bioRxiv preprint pSGDlucV3.0 was generated by introducing silent mutations into the Renilla coding 153 sequence to disrupt two potential donor splice sites (TGGgtaagt). 154 For Figures 4 and 6 (Y.A.K.) the vector was digested with PspXI and Bgl-II. Overlapping 155 oligonucleotides for insert sequences were ordered from Sigma Aldrich 156 (Supplementary Table 1), resuspended at 100 \\uf06dM, diluted to 10 \\uf06dM, and then mixed 157 together at a 1:100 dilution. This 1:100 dilution of the forward and reverse 158 oligonucleotide was heated to 95 o C. 2 \\uf06dL of this mix was ligated into 40 ng of linearized 159 backbone vector using NEB T4 ligase. 160 For Figure 5 (G.L.) The vector was digested with PspXI and Bgl-II. Using forward and 161 reverse PCR primers (Supplementary Table 1), the CCR5 WT, SSM and IFC were all 162 amplified from genomic DNA with enzyme overhangs. These PCR products were 163 subsequently digested and ligated into pSGDlucV3.0. 164 165 Transfections and Dual Luciferase Assays 166 For Figures 4 and 6 (Y.A.K.) HeLa and HEK293T cells were reverse transfected with 167 100 ng of plasmid using 0.2 \\uf06dL Lipofectamine 2000 in a 96-well format. Transfection mix 168 was added to 30,000 cells and incubated for 1-2 minutes at room temperature, before 169 adding Fetal Calf Serum. The next day, media was removed from cells and 100 \\uf06dL of 170 fresh 1X passive lysis buffer (Promega) was added to each well. Samples were frozen For Figure 5 (G.L. ) HeLa and HEK293T cells were reverse transfected with 25 ng of 174 plasmid using 0.2 \\uf06dL Lipofectamine 2000 in a half-area 96-well format. Transfection mix 175 was added to 40,000 cells. The next day, transfected cells were lysed in 12.6 \\uf06dL of 1X 176 passive lysis buffer (Promega) and luciferase activity was measured by injection of 20 177 \\uf06dL of each reagent. 178 179 Background level readings (measured from water transfected cells) were subtracted '},\n"," {'paper_id': '6955c93d7ccc8424d38e8fed7346e7d1a52b9422',\n","  'section': 'Abstract',\n","  'text': 'We analyze the temporal evolution of accumulated hospitalization cases due to COVID-19 in Belgium. The increase of hospitalization cases is consistent with an initial exponential phase, and a subsequent power law growth. For the latter, we estimate a power law exponent of ≈ 2.2, which is consistent with growth kinetics of COVID-19 in China and indicative of the underlying small world network structure of the epidemic. Finally, we fit an SIR-X model to the experimental data and estimate the effect of containment policies in comparison to their effect in China. This model suggests that the base reproduction rate has been significantly reduced, but that the number of susceptible individuals that is isolated from infection is very small. Based on the SIR-X model fit, we analyze the COVID-19 mortality and the number of patients requiring ICU treatment over time. '},\n"," {'paper_id': '697ed1b5ce6fda0e877550967f762bd6f29d00b7',\n","  'section': 'Abstract',\n","  'text': \"Background and objective: Severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2)-induced coronavirus disease 2019 has been pandemic worldwide. Several reports observed a reduction of lymphocytes among COVID-19 patients. However, clinical significance of lymphocyte reduction in COVID-19 patients remains unclear. The objective of this study was to analyze the association between lymphocyte reduction at early stage and the prognosis of COVID-19 patients. All 192 hospitalized patients with COVID-19 were enrolled. Electronic medical records, including demographic data, clinical characteristics, comorbidities and exposure history, were collected. Biochemical indexes on admission and chest computed tomography (CT) were detected. Patient's prognosis was followed up. Results: On admission, 84 (43.8%) patients suffered from lymphocyte reduction among COVID-19 patients. The count and percentage of lymphocytes on admission were lower among more than seventy-year-old patients than those of younger patients. Multivariate logistic regression revealed that older age was a risk factor of lymphocyte reduction. Of interest, chest CT score, a key marker of lung injury, was increased among COVID-19 patients with lymphocyte reduction. By contrast, PaCO2, SpO2 and oxygenation index, several respiratory function markers, were decreased in COVID-19 patients with lymphocyte reduction. Moreover, TBIL and DBIL, two markers of hepatic injury, creatinine and urea nitrogen, two indices of renal function, and creatine kinase, AST and LDH, three myocardial enzymes, were elevated in COVID-19 patients with lymphocyte reduction. Among 84 COVID-19 patients with lymphocyte reduction, 32.1% died. Fatality rate was obviously higher in COVID-19 \"},\n"," {'paper_id': '698e0cf9549b15dd23a852c59f887acba2ee260d',\n","  'section': 'Abstract',\n","  'text': 'As COVID-19 is rapidly spreading across the globe, short-term modeling forecasts provide timecritical information for decisions on containment and mitigation strategies. A main challenge for short-term forecasts is the assessment of key epidemiological parameters and how they change as first governmental intervention measures are showing an effect. By combining an established epidemiological model with Bayesian inference, we analyze the time dependence of the effective growth rate of new infections. For the case of COVID-19 spreading in Germany, we detect change points in the effective growth rate which correlate well with the times of publicly announced interventions. Thereby, we can (a) quantify the effects of recent governmental measures to mitigating the disease spread, and (b) incorporate analogue change points to forecast future scenarios and case numbers. Our code is freely available and can be readily adapted to any country or region. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. for carefully and promptly reviewing our work internally. We thank the Priesemann group -especially Daniel Gonzalez Marx, Fabian Mikulasch, Lucas Rudelt & Andreas Schneider -for exciting discussions and for their valuable comments. We thank the colleagues of the Göttingen Campus, with whom we were discussing the project and the case forecast in the past weeks very intensively: '},\n"," {'paper_id': '69f473aee22a4ac8df15a54a6109b70b1ae9b7e7',\n","  'section': 'Abstract',\n","  'text': 'Protein therapeutics represent a significant and growing component of the modern pharmacopeia, but their potential to treat human disease is limited because most proteins fail to traffic across biological membranes. Recently, we discovered that cell-permeant miniature proteins (CPMPs) containing a precisely defined, penta-arginine motif traffic readily to the cytosol and nucleus with efficiencies that rival those of hydrocarbon-stapled peptides active in animals and man. Like many cell-penetrating peptides (CPPs), CPMPs enter the endocytic pathway; the difference is that CPMPs are released efficiently from endosomes while other CPPs are not. Here, we seek to understand how CPMPs traffic from endosomes into the cytosol and what factors contribute to the efficiency of endosomal release. First, using two complementary cell-based assays, we exclude endosomal rupture as the primary means of endosomal escape. Next, using a broad spectrum of techniques, including an RNA interference (RNAi) screen, fluorescence correlation spectroscopy (FCS), and confocal imaging, we identify VPS39-a gene encoding a subunit of the homotypic fusion and protein sorting (HOPS) complex-as a critical determinant in the trafficking of CPMPs and hydrocarbon-stapled peptides to the cytosol. Although CPMPs neither inhibit nor activate HOPS function, HOPS activity is essential to efficiently deliver CPMPs to the cytosol. Subsequent multi-color confocal imaging studies identify CPMPs within the endosomal lumen, particularly within the intraluminal vesicles (ILVs) of Rab7 + and Lamp1 + endosomes that are the products of HOPS-mediated fusion. These results suggest that CPMPs require HOPS to reach ILVs-an environment that serves as a prerequisite for efficient endosomal escape. 4 conclude that HOPS allows CPMPs to traffic into intraluminal vesicles (ILVs), a favorable environment for endosomal escape. The identification of ILVs as a portal for passing proteins into the cytosol will aid the development of next-generation biologics that overcome the limitations imposed by cellular membranes. RESULTS The simplest way for a CPMP to escape from an endosome is if the endosome ruptures, in part or in full (25). Although there has been limited work on the effects of certain CPPs on the integrity of large unilamellar vesicles (LUVs) in vitro (26), the concentration-dependent effects of CPMPs or more traditional cell-penetrating peptides (CPPs) on endosomal integrity in cultured cells have not been thoroughly evaluated. Thus, we began our analysis with two complementary assays that together detect both subtle and severe endosomal damage in cells treated with a CPMP or CPP. One assay exploits a set of eGFP-labeled galectins to fluorescently tag damaged endosomes to enable their visualization using confocal microscopy, while the other employs a fluorescently tagged version of the nonalysine (Lys9) peptide to quantify the extent of endosome rupture in cells treated with a CPMP or CPP. In both cases, the effects of the two most efficient CPMPs-aPP5.3 (1) and ZF5.3 (2)-were compared with those of prototypic members of three CPP families: the hydrocarbon-stapled peptide SAH-p53-8 (3) (22); Doctaarginine (D-Arg8, 4) (27), and a cyclic peptide containing both natural and unnatural amino acids, CPP12 (5) (26) (Figure 1 and Figure S1 ). SAH-p53-8 (3) is a hydrocarbon-stapled peptide that reaches the cell interior despite the absence of excess positive charge, D-octaarginine is the proteolytically stable enantiomer of the widely studied octapeptide L-Arg8 (27, 28), and CPP12 (5) is a cyclic peptide that reportedly reaches the cytosol with an efficiency that rivals aPP5.3 (26). A characteristic feature of endosomes that have been damaged by invading viruses (29, 30), bacteria-triggered autophagy (31-34), or endosomolytic nanoparticles (35) is the cytosolic display of β-galactosides linked to proteoglycans that are ordinarily found on the luminal side of endolysosomal compartments (36). Once displayed to the cytosol, β-galactosides recruit cytosolic galectin (Gal) proteins (37, 38), such as Gal1, 3, 4, 8, and 9, that share a conserved βgalactoside binding site ( Figure S2A ) (39, 40). In particular, Gal3 and Gal8 are recruited to damaged Rab7 + and Lamp1 + endosomes that form along the degradative branch of the endocytic pathway (33, 40). Previous work has shown that endosomal damage can be detected by monitoring the translocation of eGFP fusions of Gal3 or Gal8 from the cytosol to endosome surfaces (29-34). Fusions of Gal3 or Gal8 and eGFP have also been used to visualize late endosome damage upon siRNA release from endosomes (40), screen for small molecules that induce lysosomal rupture (41), visualize endosomal leakage induced by osmocytosis and . CC-BY-NC-ND 4.0 International license is made available under a '},\n"," {'paper_id': '6a8d26d75bd44c7666c9b7ed782c13a57f409122',\n","  'section': 'Abstract',\n","  'text': 'Respiratory syncytial virus (RSV) is the leading cause of severe respiratory disease in infants. Other than age at the time of infection, the causes and correlates of severe illness in infants lacking known risk factors are poorly defined. We recruited a cohort of confirmed RSV-infected infants and simultaneously assayed the presence of resident airway microbiota and the molecular status of their airways using a novel method. Rigorous statistical analyses identified a molecular airway gene expression signature of severe illness dominated by excessive chemokine expression. Global 16S rRNA sequencing confirmed an association between H. influenzae and clinical severity. Interestingly, adjusting for H. influenzae in our gene expression analysis revealed an association between severity and airway lymphocyte accumulation. Exploring the relationship between airway gene expression and the time of onset of clinical symptoms revealed a robust, acute activation of interferon (IFN) signaling, which was absent in subjects with severe illness. Finally, we explored the relationship between IFN activity, airway gene expression and productive RSV infection using a novel in vitro model of bona fide pediatric human airway epithelial cells. Interestingly, blocking IFN signaling, but not IFN ligand production, in these cells leads to increased viral infection. Our data reveal that acute airway interferon responses are physiologically relevant in the context of infant RSV infection and may be a target for therapeutic intervention. Additionally, the airway gene expression signature we define may be useful as a biomarker for efficacy of intervention responses. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '6aa2716c00d89f4b760a9167b15d6203303ae6b9',\n","  'section': 'Abstract',\n","  'text': 'Single particle cryo-electron microscopy (cryoEM) is becoming widely adopted as a tool for structural characterization of biomolecules at near-atomic resolution. Vitrification of the sample to obtain a dense distribution of particles within a single field of view remains a major bottleneck for the success of such experiments. Here, we describe a simple and cost-effective method to increase the density of frozen-hydrated particles on grids with holey carbon support films. It relies on performing multiple rounds of sample application and blotting prior to plunge freezing in liquid ethane. We show that this approach is generally applicable and significantly increases particle density for a range of samples, such as small protein complexes, viruses and filamentous assemblies. The method is versatile, easy to implement, minimizes sample requirements and can enable characterization of samples that would otherwise resist structural studies using single particle cryoEM. '},\n"," {'paper_id': '6ae20454d1a9f228864de24660c2460becbc8151',\n","  'section': 'Abstract',\n","  'text': 'Wuhan coronavirus, called 2019-nCoV, is a newly emerged virus that infected more than 9692 people and leads to more than 213 fatalities by January 30, 2020. Currently, there is no effective treatment for this epidemic. However, the viral protease of a coronavirus is well-known to be essential for its replication and thus is an effective drug target. Fortunately, the sequence identity of the 2019-nCoV protease and that of severe-acute respiratory syndrome virus (SARS-CoV) is as high as 96.1%. We show that the protease inhibitor binding sites of 2019-nCoV and SARS-CoV are almost identical, which means all potential anti-SARS-CoV chemotherapies are also potential 2019-nCoV drugs. Here, we report a family of potential 2019-nCoV drugs generated by a machine intelligence-based generative network complex (GNC). The potential effectiveness of treating 2019-nCoV by using some existing HIV drugs is also analyzed. '},\n"," {'paper_id': '6b039b72ca7301c604c743ce7ca3e2c4f5659068',\n","  'section': 'Abstract',\n","  'text': 'Background: All types of facioscapulohumeral muscular dystrophy (FSHD) are caused by the aberrant myogenic activation of the somatically silent DUX4 gene, which initiates a cascade of cellular events ultimately leading to FSHD pathophysiology. Therefore, FSHD is a dominant gain-of-function disease that is amenable to modeling by DUX4 overexpression. However, there is large variability in the patient population. Typically, progressive skeletal muscle weakness becomes noticeable in the second or third decade of life, yet there are many genetically FSHD individuals who develop symptoms much later in life or remain relatively asymptomatic throughout their lives. Conversely, in rare cases, FSHD may present clinically prior to 5-10 yrs of age, ultimately manifesting as a very severe early onset form of the disease. Thus, there is a need to control the timing and severity of pathology in FSHD-like models. Methods: We have recently described a line of conditional DUX4 transgenic mice, FLExDUX4, that develop a myopathy upon induction of human DUX4-fl expression in skeletal muscle. Here, we use the FLExDUX4 mouse crossed with the skeletal muscle-specific and tamoxifen inducible line ACTA1-MerCreMer to generate a highly versatile bi-transgenic mouse model with chronic, lowlevel DUX4-fl expression and mild pathology, that can be induced to develop more severe FSHD-like pathology in a dose-dependent response to tamoxifen. We identified conditions to reproducibly generate models exhibiting mild, moderate, or severe DUX4-dependent pathophysiology, and characterized their progression. Results: We assayed DUX4-fl mRNA and protein levels, fitness, strength, global gene expression, histopathology, and immune response, all of which are consistent with an FSHD-like myopathic phenotype. Importantly, we identified sex-specific and muscle-specific differences that should be considered when using these models for preclinical studies. Conclusions: The ACTA1-MCM;FLExDUX4 bi-transgenic mouse model expresses a chronic low level of DUX4-fl and has mild pathology and detectable muscle weakness. The onset and progression of moderate to severe pathology can be controlled via tamoxifen injection to provide consistent and readily screenable phenotypes for assessing therapies targeting DUX4-fl mRNA and protein. Thus, these FSHD-like mouse models can be used to study a range of DUX4-fl expression and pathology dependent upon investigator need, through controlled mosaic expression of DUX4. In addition to the complex genetic and epigenetic conditions that are required to develop clinical FSHD, the pathogenic mechanism is also unusual among neuromuscular diseases. While FSHD is an autosomal dominant gain-of-function disease, the pathogenic DUX4 gene is typically expressed in only a small fraction (<1%) of myocytes, ultimately leading to debilitating muscle pathology over time [8, 16, 26]. This may account in part for the more common adult onset of clinical symptoms in FSHD patients [27, 28] . In addition, it appears that FSHD pathology is caused by sporadic bursts of increased DUX4-fl expression in differentiated myocytes, which are epigenetically suppressed in healthy and asymptomatic subjects [9, 19, 29] . Since these cells are syncytial, the detrimental effects of aberrant DUX4-FL expression may be found throughout an FSHD myofiber despite expression initially being restricted to a small percentage of myonuclei at any one time [26]. Regardless, DUX4-fl expression in FSHD myocytes and skeletal muscle, even when bursting, is still extremely rare, highly variable, and difficult to detect [8, 16, 19] . Since DUX4-FL is a transcription factor not typically expressed in healthy muscle, its aberrant expression alters the mRNA profiles of numerous genes in myocytes [24, 30, 31] . Many of these DUX4 target genes, including germline specific genes, cleavage stage genes, and immune system regulators, are not normally expressed in healthy myogenic cells [22, 24, 30, 31 ]. In addition, DUX4-FL expression ultimately initiates a cascade of numerous potentially detrimental events including the disruption of proteostasis, nonsense-mediated decay, and mRNA splicing mechanisms [19, 32, 33] , altered myogenesis [34, 35] , and induction of apoptosis [19, [36] [37] [38] . These DUX4-mediated changes, either alone or in combination, lead to progressive muscle cell death and ultimately pathology [10, 18] . Thus, the DUX4-fl mRNA and protein are prime targets for therapeutic intervention, and animal models for FSHD should be based on DUX4-fl expression in adult skeletal muscle. We have previously reported the generation of the FLExDUX4 (FLExD) conditional DUX4-fl transgenic line of mice, which contains a DUX4 transgene engineered into the Rosa26 locus using the FLEx directional switch system [39, 40] to bypass the embryonic lethality from leaky embryonic expression of this transgene [41] . Upon Cre-mediated induction, the transgene recombines to express DUX4-fl under control of the Rosa26 gene promoter ( Figure 1A ). Importantly, since many elements within DUX4 are prime targets for sequence-based therapies [42] [43] [44] [45] , the DUX4 transgene maintains the exon/intron structure of the endogenous human gene, Male ACTA1-MCM mice were crossed with female Rosa26 NZG mice to assess induction of Cremediated recombination using our TMX dosage regimens. Control or bi-transgenic mice were treated with TMX, as above. Whole gastrocnemius muscles were fixed in LacZ fixative (0.2% '},\n"," {'paper_id': '6b3a290581a7a6354807bd0b2d9a9ff0030fa99c',\n","  'section': 'Abstract',\n","  'text': 'The need for accurate statistics has never been felt so deeply as the novel COVID-19 pathogen spreads around the world and quantifying its severity is a primary clinical and public health issue. In Italy, the magnitude and increasing trend of the case-fatality risk (CFR) is fueling the already high levels of public alarm. In this paper, we highlight that the widely used crude CFR is an inaccurate measure of the disease severity since the pandemic is still unfolding. With the goal to improve its comparability over time and across countries at this stage, we then propose a demographic adjustment of the CFR that addresses the bias arising from differential case ascertainment by age. '},\n"," {'paper_id': '6b8f0278197ebdc10baa346e8b18fa8e521f31cf',\n","  'section': 'Abstract',\n","  'text': 'Chest X-ray is the first imaging technique that plays an important role in the diagnosis of COVID-19 disease. Due to the high availability of large-scale annotated image datasets, great success has been achieved using convolutional neural networks (CNN s) for image recognition and classification. However, due to the limited availability of annotated medical images, the classification of medical images remains the biggest challenge in medical diagnosis. Thanks to transfer learning, an effective mechanism that can provide a promising solution by transferring knowledge from generic object recognition tasks to domain-specific tasks. In this paper, we validate and adopt our previously developed CNN, called Decompose, Transfer, and Compose (DeTraC ), for the classification of COVID-19 chest X-ray images. DeTraC can deal with any irregularities in the image dataset by investigating its class boundaries using a class decomposition mechanism. The experimental results showed the capability of DeTraC in the detection of COVID-19 cases from a comprehensive image dataset collected from several hospitals around the world. High accuracy of 95.12% (with a sensitivity of 97.91%, a specificity of 91.87%, and a precision of 93.36%) was achieved by DeTraC in the detection of COVID-19 X-ray images from normal, and severe acute respiratory syndrome cases. 12 disease within computed tomography (CT ). In [3], a modified version of ResNet-50 13 pre-trained network has been provided to classify CT images into three classes: healthy, 14 COVID-19 and bacterial pneumonia. Chest x-ray images (CXR) were used in [4] by a 15 CNN constructed based on various ImageNet pre-trained models to extract the high 16 '},\n"," {'paper_id': '6ba80f58d066d9fb759bac1191c72267d94afd39',\n","  'section': 'Abstract',\n","  'text': \"Background: To control the spread of Corona Virus Disease , 30 screening large numbers of suspected cases for appropriate quarantine and 31 treatment is a priority. Pathogenic laboratory testing is the diagnostic gold standard 32 but it is time consuming with significant false negative results. Fast and accurate 33 diagnostic methods are urgently needed to combat the disease. Based on 34 COVID-19 radiographical changes in CT images, we aimed to develop a deep 35 learning method that could extract COVID-19's graphical features in order to provide 36 a clinical diagnosis ahead of the pathogenic test, thus saving critical time for 37 disease control. 38 Methods and Findings: We collected 1,065CT images of pathogen-confirmed 39 COVID-19 cases (325 images) along with those previously diagnosed with typical 40 viral pneumonia (740 images). We modified the Inception transfer-learning model 41 to establish the algorithm, followed by internal and external validation. The internal 42 validation achieved a total accuracy of 89.5% with specificity of 0.88 and sensitivity 43 of 0.87. The external testing dataset showed a total accuracy of 79.3% with 44 specificity of 0.83 and sensitivity of 0.67. In addition, in 54 COVID-19 images that 45 first two nucleic acid test results were negative, 46 were predicted as COVID-19 46 positive by the algorithm, with the accuracy of 85.2%. 47 Conclusion: These results demonstrate the proof-of-principle for using 48 artificial intelligence to extract radiological features for timely and accurate 49 COVID-19 diagnosis. 50 \"},\n"," {'paper_id': '6c185e0c7cb01ce3de268d1ce3416c1b807a79c4',\n","  'section': 'Abstract',\n","  'text': \"1 The human betacoronaviruses HKU1 and OC43 (subgenus Embecovirus) arose from 2 separate zoonotic introductions, OC43 relatively recently and HKU1 apparently much longer 3 ago. Embecovirus particles are studded with two types of surface projections called S (for 4 spike) and HE (for haemagglutinin-esterase), with S mediating receptor-binding and 5 membrane fusion and HE acting as a receptor-destroying enzyme. Together, they promote 6 dynamic virion attachment to glycan-based receptors with 9-O-acetylated sialic acid as main 7 constituent. We recently showed that adaptation of HKU1 and OC43 to replication in the 8 human respiratory tract involved loss-of-function mutations in the lectin domain of HE. Here 9 we present the cryo-EM structure of the ~80 kDa, heavily glycosylated HKU1 HE at a global 10 resolution of 3.4 Å. Comparison to existing HE structures reveals a drastically truncated lectin 11 domain, incompatible with sialic acid binding, but with the structure and function of the HE 12 esterase domain left intact. Our cryo-EM structure, in combination with mass spectrometry 13 analysis, also describes the extent of glycosylation on the now redundant lectin domain, which 14 forms a putative glycan shield. The findings further our insight into the evolution and host 15 adaptation of human embecoviruses and also demonstrate the utility of cryo-EM for studying 16 small, heavily glycosylated proteins which are intractable to X-ray crystallography. : bioRxiv preprint 19 mammals and birds with a propensity to cross host species barriers 1 . Zoonotic CoV infections 20 pose an ever-looming threat to public health 2 . Indeed, this century alone saw the advent of 21 three novel respiratory CoVs highly lethal to humans. The 2002/2003 SARS-CoV variant was 22 quickly contained 3,4 , and MERS-CoV, natural to dromedary camels, does not spread efficiently 23 within the human population 5 . SARS-CoV-2, however, seems well on course to become 24 established in humans 6 . Four other respiratory coronaviruses of zoonotic origin did breach the 25 species barrier successfully and are maintained worldwide through continuous human-to-26 human transmission 7 . Studies of these latter human CoVs and their zoonotic ancestors are 27 instrumental in understanding the odds and risks of CoV cross-species transmission as well 28 as the requirements for adaptation to the human host. Human CoVs HKU1 and OC43 (subgenus Embecovirus, genus Betacoronavirus) are related 30 but distinct 8 . In immunocompetent individuals, these viruses are generally associated with 31 benign common colds, but may cause significant morbidity and even mortality in the frail 9,10 . They entered the human population separately: HKU1 several hundred years ago from a yet 33 unknown animal reservoir 11,12 , whereas OC43 entered far more recently (70-120 years ago), 34 apparently from a bovine coronavirus (BCoV) spill-over 13 . These embecoviruses use 9-O-35 acetylated sialoglycans (9-O-Ac-Sia) as receptors to which they attach in a highly dynamic 36 fashion with fast on and off rates via fusion spike protein S 14 . A second type of envelope 37 protein unique to embecoviruses, called the haemagglutinin-esterase (HE), serves as a 38 receptor-destroying enzyme 8 . CoV HEs are 40-50 kDa type I membrane glycoproteins that assemble into covalently-linked 40 homodimers, disulfide-bonded in the juxta-membrane region 15 . In HEs of animal 41 embecoviruses, an O-Ac-Sia-binding lectin domain (LD) appended to the esterase domain 42 (ED) upregulates sialate-O-acetylesterase activity towards multivalent ligands such as the 43 . CC-BY 4.0 : bioRxiv preprint densely clustered sialoglycans on mucins 15-17 . However, OC43 and HKU1, subject to 44 convergent evolution, lost HE lectin function 8 . Thus, the dynamics of virion-glycan interactions 45 were altered, and virion-mediated receptor destruction was restricted, apparently as an 46 adaptation to replication in the human respiratory tract. For OC43, the phylogenetic record 47 shows that after its introduction in humans, HE-mediated receptor binding was selected 48 against and ultimately lost completely through progressive accumulation of subtle single site 49 substitutions in LD. In HKU1, HE lectin function was also lost, but here the LD suffered several 50 deletions 8,18 . 51 For OC43, BCoV and murine embecoviruses 8,15-17 , HE crystal apo-and holostructures 52 revealed how HE LDs bind O-Ac-Sias and how this property was lost in OC43. HKU1 HE, however, proved refractory to crystallisation. Exhaustive attempts to solve its structure and 54 assess the structural consequences of the LD deletions did not meet with success, prompting 55 us to consider alternative methods. In the last decade, single-particle analysis by cryo-electron 56 microscopy (SPA cryo-EM) revolutionised the field of structural biology with methodological 57 advances now allowing the visualisation of large biological macromolecules at resolutions 58 which permit atomic modelling 19 . In the wake of these innovations, cryo-EM structures were 59 determined for several CoV S proteins, including those of OC43 and HKU1 to resolutions of 60 2.8 and 4 Å resolution, respectively 20,21 . These studies were greatly facilitated by the 61 considerable size (>400 kDa) and three-fold symmetry of the S homo-trimers. Because of the 62 low signal-to-noise ratio of cryo-EM images, SPA reconstruction of specimens smaller than 63 100 kDa remains problematic and only for a handful of such proteins, high-resolution (i.e. <3.5 64 Å) cryo-EM structures have been resolved 22-24 . Nevertheless, inspired by these reports, we 65 explored SPA to characterise the homo-dimeric HKU1 HE ectodomain, a complex with a 66 protein mass of ~80 kDa plus ~30 kDa of N-glycans, most of which is disordered. Here we 67 present the structure of HKU1 HE at a global resolution of 3.4 Å. The data reveal that, over 68 centuries of HKU1 circulation in humans, the variable loops that form the LD receptor-binding 69 site were trimmed back completely. The jelly roll core structure, on which these loops were 70 . CC-BY 4.0 : bioRxiv preprint grafted, remains intact, presumably to preserve structure-function of the ED SGNH hydrolase 71 into which it is embedded. Conceivably, the conservation of a functionally obsolete but 72 structurally essential domain in a viral envelope protein may pose an antigenic liability. The 73 data show that through acquisition of new N-linked glycosylation sites, the HKU1 HE LD 74 domain was provided with a putative glycan shield. The findings provide structural insight into 75 the evolution of multidomain proteins after partial loss of function and a predictive evolutionary 76 trajectory for OC43 HE. 77 RESULTS Structure determination of HKU1 HE by single-particle analysis There are two variants of the HKU1 HE protein, type A and type B, which share ~85% 80 sequence identity 25 . For structural analysis, the HKU1-A HE ectodomain was expressed as 81 an Fc fusion protein in HEK293T cells and purified by protein A affinity chromatography, 82 followed by on-the-bead thrombin cleavage to remove the Fc domain, and size exclusion 83 chromatography (Figure S1A). The protein backbone of the HKU1 HE ectodomain construct 84 has a molecular weight of ~40 kDa but runs as a smeared band between 50 and 75 kDa by 85 SDS-PAGE analysis (Figure S1B), indicating extensive and heterogeneous glycosylation. 86 Optimised cryo-EM grids, prepared with purified HKU1 HE, revealed monodisperse particles 87 in a variety of orientations (Figure 1A). Single-particle analysis produced 2D class averages 88 corresponding to different views of the HKU HE dimer (Figure 1B). Subsequently, a 3D 89 reconstruction of HKU1 HE was produced at a global resolution of 3.4 Å (Figure 1C, S2 and 90 S3A-B). Local resolution of the map ranges from 3.3-4.1 Å with the most well-resolved regions 91 located in the core of the molecule (Figure S3D). The quality of the map permitted modelling 92 of residues 15-346 (Figure 1D-E). Each protomer is stabilised by six disulfide bonds, the 93 presence of which are corroborated by the cryo-EM density (Figure 1F). Dimerisation of HKU1 94 HE is achieved by interaction of the vestigial LDs and the membrane-proximal domains (MP) 95 (Figure 2A-B). : bioRxiv preprint comparison of the membrane proximal domain and esterase domain of HKU1 to these related 100 proteins revealed a high-level of structural conservation, with the aligned Ca\\uf061 positions having 101 RMSD values of 0.81 Å and 0.76 Å for MHV-New-Jersey (NJ) strain, and 0.83 Å and 0.84 Å 102 for BCoV-Mebus strain. While the core fold of the esterase domain is similar between all three 103 proteins, the α2 helix of HKU1 more closely resembles that of BCoV 15 , and lacks the zinc 104 binding site found in MHV-NJ 17 forms a zinc binding site 17 . The ED catalytic site of HKU1 HE 105 is most similar to that of BCoV, OC43 and MHV-DVIM strain (Figure S4A-B), which all 106 recognise 9-O-Ac-Sia. This is consistent with functional data showing that HKU1 HE 107 possesses sialate-9-O-acetylesterase receptor destroying activity 8 . In contrast, MHV-NJ HE 108 recognises 4-O-Ac-Sia, thus rationalising the differences observed in the active site 109 architecture 17 (Figure S4C). 110 The lectin domain of HKU1 HE dramatically differs from those of MHV-NJ and BCoV. The LD 111 jelly roll core is essentially conserved, with RMSD values of 0.95 Å and 0.89 Å for BCoV and 112 MHV-NJ, respectively. However, whereas in BCoV HE and MHV extensive loops emanate 113 from this jelly roll core to form the LD carbohydrate-binding site (CBS), such loops are either 114 much shorter or absent in the HKU1 HE structure (Figure 2E-G). As compared to the BCoV 115 and MHV HEs, HKU1 HE suffered deletions in the β5-β6 loop, the β7-β8 loop, and β9-β10 116 loop (β11-β12 in BCoV) - see figure 2 and S5. The long β4-β5 loop has undergone a minor 117 deletion in HKU1 HE but is also displaced with respect to that of BCoV and MHV HE, reaching 118 a maximum distance of 10 Å between residues 144-148. In BCoV and MHV HE, each of these 119 loops participates in ligand/receptor binding. In addition, the metal binding site (MBS) which 120 stabilizes the β11-β12 CBS loop in MHV and BCoV (β13-β14) is absent in HKU1 (Figure 2E-121 G). Two of the three metal coordinating sidechains, present in BCoV HE (Figure S6A), are 122 partially conserved in HKU1, indicating that the MBS was likely present in the ancestral 123 protein. However, the critical D220 site in BCoV HE falls within the β7-β8 loop deletion of 124 HKU1 HE (Figure S6B). Thus, HKU1 has lost all essential components of the HE CBS and 125 the LD domain has been stripped back essentially to the jelly roll core. : bioRxiv preprint 129 composition of each of these N-linked glycosylation sites, we performed in-depth 130 glycoproteomics profiling of the same recombinant HEK293T cell derived material used for 131 cryo-EM. HKU1 HE was digested in parallel with trypsin, chymotrypsin and alpha lytic protease : bioRxiv preprint 157 Side-by-side comparison of HKU1, MHV-NJ, BCoV and OC43 reveals that HKU1 HE lectin 158 domain has a much flatter topography ( Figure  4A-D) . In addition to the dramatically altered : bioRxiv preprint cryo-EM map, the glycans on the LD are not visible beyond the first core GlcNAc. However, 174 mass spectrometry analysis confirms that these primarily contain complex glycans, comprising 175 between 9 to 16 saccharide units. To understand where these lesser-ordered regions are 176 situated, a difference map of the N-linked glycans was generated and a gaussian filter was 177 applied in order to highlight low resolution features. Interestingly, density for the disordered 178 portion of these glycans forms a crown of glycan density, which encircles the LD and covers 179 much of its surface ( Figure 5C ). Of note, density belonging to the N110 glycan of HKU1 180 overlaps with the former sialic acid binding site. Side-by-side comparison shows that the LD 181 of HKU1 is ~8 Å shorter than BCoV, with none of the CBS loop remnants protruding above 182 the glycan crown ( Figure  5D -F). While our mass spectrometry data reveals that the N168 site 183 is only 1.6 % occupied, we do observe difference density which extends tangentially from this 184 site. This putative N168 density extends over the entrance to the esterase domain active site 185 ( Figure  5F ). : bioRxiv preprint With respect to the glycans as a shield against humoral immunity, the HKU1 HE glycan crown 235 not only covers much of the LD, but also appears to extend outwards over the esterase active 236 site entrance. The entrance to the HE esterase domain active site is subject to sequence 237 variation in BCoV ( Figure  5A ), suggesting that this part of HE too is a vulnerable target for : bioRxiv preprint humoral immunity. In the context of a crowded viral envelope, with densely packed S and HE 239 proteins, the membrane distal glycan crown of HKU1 HE might help shield the regions below 240 without any potential loss of function resulting from direct glycosylation of the ED active site 241 entrance. Interestingly, OC43 has independently acquired two N-linked glycans in equivalent positions 243 to HKU1. Thus, our findings for HKU1 offer a glimpse into a possible future for OC43 HE. During circulation of OC43 in the last 70-120 years, the HE LD underwent structural changes 245 which were deleterious to 9-O-Ac-Sia binding, while remaining largely intact. Over a longer 246 timespan OC43 HE may well follow the fate of its HKU1 homologue, however, through similar 247 deletions of the surface-exposed loops in the LD. Conversely, the evolution of OC43 allows 248 informed speculation on the early steps in HKU1's adaptation to humans. From a general evolutionary perspective, our data offers insights into how viruses deal with a 250 structurally integral but enzymatically obsolete component of multifunctional proteins in the 251 face of relentless immune pressure. Interestingly, receptor binding site loop deletions have 252 also been reported for H7N2 and H9N2 avian influenza haemagglutinin proteins, and in the 253 latter case were demonstrated to facilitate immune escape 32,33 . In both cases, sialic acid 254 binding was retained but the receptor preference was altered. Also for the HIV-1 envelope 255 glycoprotein, susceptibility to neutralising antibodies is countered by modulating surface 256 exposure and glycosylation pattern of the variable regions 34 . Our findings reinforce the notion 257 that for embecoviruses, the HE, in addition to S, should be considered a viable candidate for 258 vaccine development. Moreover, the highly conserved HE esterase active site offers an 259 attractive target for the development of broad-spectrum antivirals against OC43 and HKU1 as 260 well as against novel human embecoviruses, should such emerge in the future. By 261 demonstrating that cryo-EM can be used to study CoV HEs at high-resolution, we expand the 262 structural biology toolkit for future research, which may focus on antibody or inhibitor 263 complexes. Methods 265 Protein expression and purification 266 The human codon-optimised sequence for HKU1 HE (Q5MQD1) was cloned into a pCD5-T-267 Fc expression plasmid 15 . The resulting construct encodes a chimeric protein comprising of the 268 HE ectodomain fused to the human IgG1 Fc domain, with the domains separated by a 269 thrombin cleavage site. The HE-Fc was produced by transient expression in HEK293T cells 270 and purified from cell culture supernatants by protein A affinity chromatography followed by 271 on-bead thrombin cleavage as described previously 15 . Beads were pelleted, and the HE 272 ectodomain in the supernatant was concentrated to a volume of ~150 µl and injected on a 273 Superdex200 increase column (GE Healthcare) pre-equilibrated in 10 mM Tris pH 8 and 274 50 mM NaCl. Monodisperse dimeric fractions were concentrated to a final concentration of 275 4.0 mg/ml (Figure S1A). Sample purity was assessed by SDS PAGE gel analysis (Figure 276 S1B). 277 Cryo-electron microscopy 278 Three μl of purified HKU1 HE (4 mg/ml) was dispensed on Quantifoil R1.2/1.3 200-mesh grids 279 (Quantifoil Micro Tools GmbH) that had been freshly glow discharged for 30 seconds at 20 280 mA using GloQube Glow Discharge system (Quorum Technologies). Grids were blotted for 281 five seconds using Whatman No. 1 filter paper and immediately plunge-frozen into liquid 282 ethane cooled by liquid nitrogen using a Vitrobot Mark IV plunger (Thermo Fisher Scientific) 283 equilibrated to ~95% relative humidity, 4°C. Movies of frozen-hydrated HKU1 HE were 284 collected using Titan Krios G4 Cryo-TEM (Thermo Fisher Scientific) operating at 300 keV and 285 equipped with a Falcon 4 Direct Electron Detector (Thermo Fisher Scientific). All cryo-EM data 286 were acquired using the EPU 2 software (Thermo Fisher Scientific). Microscope was aligned 287 to produce fringe-free imaging (FFI) allowing five acquisition areas within a hole and 288 aberration-free image shift (AFIS) was used to acquire images from up to 21 holes per single 289 stage move. Movies were collected in electron counting mode at 96,000x corresponding to a 290 calibrated pixel size of 0.805 Å/pix over a defocus range of -1.0 to -2.5 μm. 6,029 movies were : bioRxiv preprint collected using a dose rate of 5 e -/pix/sec for a total of 5.6 seconds (207 ms per fraction, 27 292 fractions), resulting in a total exposure of ~40 e -/Å 2 (1.5 e -/Å 2 /fraction). Image processing Collected movie stacks were manually inspected and then imported in Relion version 3.0.1. 35 Drift and gain correction were performed with MotionCor2 36 , and GCTF 37 was used to estimate 296 the contrast transfer function for each movie. Movies with a GCTF-estimated resolution of 10 Å 297 or worse were discarded. One thousand particles were picked manually and 2D classified. The resulting classes were then used as templates for autopicking in Relion, resulting in 299 936,260 particles. Fourier binned (2 x 2) particles were extracted in a 90 pixel box and 300 subjected to a round of 2D classification after which 565,194 particles were retained. Using 301 the 'molmap' command in UCSF chimera 38 , the HKU1 HE homology model was used to 302 generate a 40 Å resolution starting model for 3D classification. Particles selected from 2D 303 classification were subject to a round of 3D classification (with C2 symmetry). Particles 304 belonging to the best class (119,717 particles) were re-extracted unbinned in a 320 pixel box 305 to ensure delocalised high-resolution information was not excluded. Subsequent 3D auto-306 refinement (with C2 symmetry) and post-processing yielded a map with a resolution of 3.79 Å. Further sub-classification attempts did not lead to improvements in map quality or resolution. Per particle defocus estimation improved the resolution to 3.73 Å. Relion's Bayesian polishing 309 procedure was then performed on these particles, with all movie frames included, which 310 produced a 3.68 Å map 39 . Next, particles were assigned to their respective AFIS group and 311 subject to beam-tilt refinement which further improved the resolution to 3.48 Å. Data 312 processing was then continued in Relion version 3.1b 40 , where a final round of per-particle 313 defocus and per-micrograph astigmatism estimation was performed. This produced a 3.39 Å 314 resolution map, based on the gold-standard FSC = 0.143 criterion. A negative B-factor of 315 −122 Å 2 was applied during the final post-processing step. Local resolution estimations were 316 : bioRxiv preprint space refinement', carbohydrate module and 'sphere refinement' tools 42,43 . Iterative rounds of 322 manual fitting in Coot and real space refinement in Phenix 44 were carried out to improve non-323 ideal rotamers, bond angles and Ramachandran outliers. During real space refinement, 324 secondary structure and NCS restraints were imposed. Validation was carried out using 325 Molprobity (general/protein;; 45 ) and Privateer 46,47 . All data collection, image processing and 326 refinement information can be found in Table S1. 327 Analysis and visualisation 328 To generate a difference map of the N-linked glycans, cryo-EM density within 3 Å of a fitted 329 HKU1 HE dimer, encompassing only the protein component, was erased using the UCSF 330 chimera 'color zone' tool. The remaining density was then gaussian filtered using the 'volume 331 filter' tool in UCSF chimera 38 . Multiple sequence aligments of BCoV, OC43 and HKU1 HE 332 were generated using NCBI BLAST and subsequently plotted onto their respective structures 333 using UCSF Chimera. Alignment of the representative HKU1 genotype A and B sequences 334 (Figure S8) was generated using the EMBL-EBI Clustal Omega programme 48 . Surface 335 colouring of BCoV and HKU1 HE using the Kyte-Doolittle hydrophobicity scale was performed 336 in UCSF chimera. RMSD values were calculated using the 'MatchMaker' tool of UCSF 337 Chimera with default settings. Figures were generated using UCSF Chimera 38 , UCSF 338 ChimeraX 49 and PyMOL (The PyMOL Molecular Graphics System, Version 2.0, Schrödinger, 339 LLC). 340 Sample preparation for glycoproteomics analysis. 341 Nine μg of CoV-HKU1 HE were incubated in 100mM Tris pH 8.5, 2% sodium deoxycholate, 342 10mM tris(2-carboxyethyl)phosphine, and 40mM iodoacetamide at 95°C for ten minutes and 343 at 25°C for 30 minutes in the dark. Denatured, reduced and alkylated CoV-HKU1 HE (3 μg) : bioRxiv preprint was then diluted into fresh 50mM ammonium bicarbonate and incubated overnight at 37°C 345 either with 0.056 μg of trypsin (Promega), chymotrypsin (Sigma Aldrich) or alpha lytic protease 346 (Sigma). Formic acid was then added to a final concentration of 2% and the samples were 347 centrifuged at 14,000 rpm for 20 min at 4°C, followed by another round of centrifugation for 5 348 min to precipitate the sodium deoxycholate and collect the peptides from the supernatants. 349 Then, the CoV-HKU1 HE tryptic, chymotryptic and alpha lytic protease digests were desalted 350 using 30 µm Oasis HLB 96-well plate (Waters). The Oasis HLB sorbent was activated with 351 100% acetonitrile and subsequently equilibrated with 10% formic acid in water. Next, peptides 352 were bound to the sorbent, washed twice with 10% formic acid in water and eluted with 100 353 µL of 50% acetonitrile/5%formic acid (v/v). The eluted peptides were vacuum-dried and 354 resuspended in 100 µL of 2% formic acid in water. 355 Mass spectrometry 356 Six μl of resuspended peptides for the glycoform identification and two μl two-fold diluted 357 peptides for the glycoform quantification were run on an Orbitrap Fusion Tribrid (ThermoFisher 358 Scientific, Bremen) mass spectrometer coupled to nanospray UHPLC system Agilent 1290 359 (Agilent Technologies) in duplicates. A 90-min LC gradient from 0% to 35% acetonitrile was 360 used to separate peptides at a flow rate of 300 nl/min. A Poroshell 120 EC C18 (50 cm x 75 361 µm, 2.7 µm, Agilent Technologies) analytical column and a ReproSil-Pur C18 (2 cm x 100 µm, 362 3 µm, Dr. Maisch) trap column were used for the peptide separation. The data was acquired 363 in data-dependent mode. Orbitrap Fusion parameters for the full scan MS spectra were as 364 follows: an AGC target of 4 * 10^5 at 60 000 resolution, scan range 350-2000 m/z, Orbitrap 365 maximum injection time 50 ms. Ten most intense ions (2+ to 8+ ions) were subjected to 366 fragmentation with electron-transfer/higher energy collision dissociation ion fragmentation 367 scheme 50 . The supplemental higher energy collision dissociation energy was set at 30%. The 368 MS2 spectra were acquired at a resolution of 30 000 with an AGC target of 5 * 10^5, maximum 369 injection time 250 ms, scan range 120-4000 m/z and dynamic exclusion of 16 s. : bioRxiv preprint search windows for MS1/MS2, respectively. Up to three missed cleavages were permitted 374 using C-terminal cleavage at R/K for trypsin, F/Y/W/M/L for chymotrypsin and T/A/S/V for 375 alpha lytic protease. Carbamidomethylation of cysteine was set as fixed modification, 376 methionine oxidation as variable common 1, glycan modifications as variable common 2, 377 allowing up to max. 2 variable common parameters per one peptide. All N-linked glycan 378 databases from Byonic were merged into a single non-redundant list to be included in the 379 database search. All reported glycopeptides in the Byonic result files were manually inspected 380 for quality of fragment assignments. All glycopeptide identifications were merged into a single 381 non-redundant list per sequon. Glycans were classified based on HexNAc content as high-382 mannose (2 HexNAc), hybrid (3 HexNAc) or complex (> 3 HexNAc). Byonic search results 383 were exported to mzIdentML format to build a spectral library in Skyline 52 and extract peak 384 areas for individual glycoforms from MS1 scans. The full database of variable N-linked glycan 385 modifications from Byonic was manually added to the Skyline project file in XML format. Reported peak areas were pooled based on the number of HexNAc, Fuc or NeuAc residues 387 to distinguish high-mannose/hybrid/complex glycosylation, fucosylation and sialylation, 388 respectively. The quantified data was represented with GraphPad Prism 8 software. 389 Coordinates are deposited in the Protein Data Bank under accession code 6Y3Y. The 391 corresponding EM density maps (final unsharpened, sharpened, local resolution filtered, half 392 maps, N-glycan difference map, and mask) have been deposited to the Electron Microscopy 393 Data Bank under the accession EMD-10676. The raw LC-MS/MS files and glycoproteomics 394 analyses have been deposited to the ProteomeXchange Consortium via the PRIDE partner 395 repository with the dataset identifier PXD017545. All reagents and relevant data are available 396 from the authors upon request. \"},\n"," {'paper_id': '6c3e940e2848827af17dd23005a9dee8ac0cd349',\n","  'section': 'Abstract',\n","  'text': \"Since 2007, repeated outbreaks of Zika virus (ZIKV) has affected millions of people worldwide and created global health concern with major complications like microcephaly and Guillain Barre's syndrome. Generally, ZIKV transmits through mosquitoes (Aedes aegypti) like other flaviviruses, but reports show blood transfusion and sexual mode of ZIKV transmission which further makes the situation alarming. Till date, there is not a single Zika specific licensed drug or vaccine present in the market. However, in recent months, several antiviral molecules have been screened against viral and host proteins. Among those, (−)-Epigallocatechin-3-gallate (EGCG), a green tea polyphenol has shown great virucidal potential against flaviviruses including ZIKV. However, the mechanistic understanding of EGCG targeting viral proteins is not yet entirely deciphered except little is known about its interaction with viral envelope protein and viral protease. Since literature has shown significant inhibitory interactions of EGCG against various kinases and bacterial DNA gyrases; we designed our study to find inhibitory actions of EGCG against ZIKV NS3 helicase. NS3 helicase is playing a significant role in viral replication by unwinding RNA after hydrolyzing NTP. We employed molecular docking and simulation approach and found significant interactions at ATPase site and also at RNA binding site. Further, the enzymatic assay has shown significant inhibition of NTPase activity with an IC50 value of 295.7 nM and Ki of 0.387 ± 0.034 µM. Our study suggests the possibility that EGCG could be considered as prime backbone molecule for further broadspectrum and multitargeted inhibitor development against ZIKV and other flaviviruses. author/funder. All rights reserved. No reuse allowed without permission. \"},\n"," {'paper_id': '6c76fe66d27d9c9e1c47b943e497309cf58bd4e4',\n","  'section': 'Abstract',\n","  'text': 'Alimohamad Asghari, MD. Professor of otolaryngology, Skull base research center, The five senses institute, '},\n"," {'paper_id': '6c7c93c25caa6d94c70c891e2446a4a12e729365',\n","  'section': 'Abstract',\n","  'text': 'We firstly reported a case of acute myelitis in a SARS-CoV-2-infected patient. A 66-year-old man with COVID-19 was admitted with acute flaccid paralysis of bilateral lower limbs and urinary and bowel incontinence. All serum microbiological studies were negative except for SARS-CoV-2 nucleic acid testing. Clinical findings could be ascribed to a post-infectious acute myelitis. He was receiving treatment with ganciclovir, lopinavir/ritonavir, moxifloxacin, dexamethasone, human immunoglobulin and mecobalamin. With a diagnosis of post-infectious acute myelitis and comprehensive treatment, paralysis of bilateral lower extremities ameliorated. After two times negative novel coronavirus RNA nasopharyngeal swabs tests, he was discharged and transferred to a designated hospital for isolation treatment and rehabilitation therapy. '},\n"," {'paper_id': '6c91b00faa16142426820016ad3a7847fc77c8e4',\n","  'section': 'Abstract',\n","  'text': 'A newly identified coronavirus, 2019-nCoV, has been posing significant threats to public health since December 2019. ACE2, the host cell receptor for severe acute respiratory syndrome coronavirus (SARS), has recently been demonstrated in mediating 2019-nCoV infection. Interestingly, besides the respiratory system, substantial proportion of SARS and 2019-nCoV patients showed signs of various degrees of liver damage, the mechanism and implication of which have not yet been determined. Here, we performed an unbiased evaluation of cell type specific expression of ACE2 in healthy liver tissues using single cell RNA-seq data of two independent cohorts, and identified specific expression in cholangiocytes. The results indicated that virus might directly bind to ACE2 positive cholangiocytes but not necessarily hepatocytes. This finding suggested the liver abnormalities of SARS and 2019-nCoV patients may not be due to hepatocyte damage, but cholangiocyte dysfunction and other causes such as drug induced and systemic inflammatory response induced liver injury. Our findings indicate that special care of liver dysfunction should be installed in treating 2019-nCoV patients during the hospitalization and shortly after cure. : bioRxiv preprint rise of liver enzymes, implying liver function damage in patients infected with MERS-CoV [9]. An very recent epidemiologic study revealed that 43 out of 99 initial patients infected with 2019-nCoV had various degrees of liver function abnormality, with alanine aminotransferase (ALT) or aspartate aminotransferase (AST) above the normal range, and more important, one '},\n"," {'paper_id': '6c9306850b5da6412d800d19643c80d3dc3af2dc',\n","  'section': 'Abstract',\n","  'text': \"The pandemic state of COVID-19 caused by the SARS CoV-2 put the world in quarantine and is causing an unprecedented economic crisis. However, COVID-19 is spreading in different rates at different countries. Here, we tested the effect of three classes of predictors, i.e., socioeconomic, climatic and transport, on the rate of daily increase of COVID-19. We found that global connections, represented by countries' importance in the global air transportation network, is the main explanation for the growth rate of COVID-19 in different countries. Climate, geographic distance and socioeconomics did not affect this big picture analysis. Geographic distance and climate were significant barriers in the past but were surpassed by the human engine that allowed us to colonize almost every corner on Earth. Based on our global analysis, the global network of air transportation could lead to a worst-case scenario of synchronous global pandemic if board control measures in international airports were not taken and are not sustained during this pandemic. Despite all limitations of a global analysis, our results indicate that the current claims that the growth rate of COVID-19 may be lower in tropical countries should be taken very carefully, at risk to disturb well-established and effective policy of social isolation that may help to avoid higher mortality rates due to collapse of national health systems. This is the case of Brazil, a well-connected tropical country that presents the second highest increase rate of COVID-19 and might experience a serious case of human-induced disasters if decision makers take into consideration unsupported claims of the growth rate of COVID-19 might be lower in tropical countries. \"},\n"," {'paper_id': '6cb64c9a76fdb0f42aa7479705519d37ec098031',\n","  'section': 'Abstract',\n","  'text': 'The novel coronavirus (2019-nCOV) outbreak has been a serious concern around the globe. Since people are in tremor due to the massive spread of Coronavirus in the major parts of the world, it requires to predict the risk of this infectious disease. In this situation, we develop a model to measure the risk of infectious disease and predict the risk of 2019-nCOV transmission by using data of four countries-US, Australia, Canada and China. The model underlies that higher the population density, higher the risk of transmission of infectious disease from human to human. Besides, population size, case identification rate and travel of infected passengers in different regions are also incorporated in this model. '},\n"," {'paper_id': '6cbefcf4c0fbb8466db7d00b8f53e1f37a6bf8b5',\n","  'section': 'Abstract',\n","  'text': 'Objective: The emerging and recent 2014 Ebola Virus Disease (EVD) outbreaks rang the bell to call upon efforts from globe to assist resource-constrained countries to strengthen public health surveillance system for early response. Malawi adopted the Integrated Disease Surveillance and Response (IDSR) strategy to develop its national surveillance system since 2002 and revised its guideline to fulfill the International Health Regulation (IHR) requirements in 2014. This study aimed to understand the state of IDSR implementation and differences between guideline and practice for future disease surveillance system strengthening. Methods: This was a mixed-method observational study. Quantitative data were to analyze completeness and timeliness of surveillance system performance from national District Health Information System 2 (DHIS2). Qualitative data were collected through interviews with 29 frontline health service providers from the selected district and key informants of the IDSR system implementation and administration at district and national levels. Findings: The current IDSR system showed relatively good completeness (76.4%) but poor timeliness (41.5%) of total expected monthly reports nationwide and zero weekly reports. The challenges of IDSR implementation revealed through qualitative data included lack of supervision, inadequate resources for training and difficulty to implement weekly report due to overwhelming paperwork at frontline health services. Conclusions: The differences between IDSR technical guideline and actual practice were huge. The developing information technology infrastructure in Malawi and emerging mobile health (mHealth) technology can be opportunities for the country to overcome these challenges and improve surveillance system to have better timeliness for the outbreaks and unusual events detection. '},\n"," {'paper_id': '6cf87a546884756094da0d300e85a061c2cc43ea',\n","  'section': 'Abstract',\n","  'text': 'Background: The outbreak of novel coronavirus pneumonia in China began in December 2019. Studies on novel coronavirus disease (COVID-19) were less based on pediatric patients. This study aimed to reveal the clinical characteristics of COVID-19 in children. Method: This study retrospectively analyzed the clinical symptoms, laboratory results, chest CT, and treatment of children with laboratory-confirmed COVID-19(ie, with samples that were positive for 2019 novel coronavirus[2019-nCoV]) who were admitted to Result: Nine patients had no obvious clinical symptom. 11 patients developed fever. Other symptoms, including cough(in eleven of seventeen patients), rhinorrhea(in two), diarrhea(in two), vomiting(in two), were also observed. A small minority of patients had lymphocytopenia. Alanine transaminase or transaminase increased in three cases. According to chest CT scan, 11 patients showed unilateral pneumonia, 8 patients had no pulmonary infiltration. No serious complications such as acute respiratory syndrome and acute lung injury occurred in all patients. Conclusion: The clinical characteristics of 2019-nCoV infection in children were different from adult. The overall condition of children were mild and have a good prognosis. '},\n"," {'paper_id': '6d0127f985edbe088bc279865bef25a31f54a066',\n","  'section': 'Abstract',\n","  'text': \"The novel coronavirus SARS-CoV-2 (2019-nCoV) is a member of the family coronaviridae and contains a single-stranded RNA genome with positive-polarity. To reveal the evolution mechanism of SARS-CoV-2 genome, we performed comprehensive genomic analysis with newly sequenced SARS-CoV-2 strains and 20 closely related coronavirus strains. Among 98 nucleotide mutations at 93 sites of the genome among different SARS-CoV-2 strains, 58 of them caused amino acid change, indicating a result of neutral evolution. However, the ratio of nucleotide substitutions to amino acid substitutions of spike gene (9.07) between SARS-CoV-2 WIV04 and Bat-SARSr-CoV RaTG13 was extensively higher than those from comparisons between other coronaviruses (range 1.29 -4.81). The elevated synonymous mutations between SARS-CoV-2 and RaTG13, suggesting they underwent stronger purifying selection. Moreover, their nucleotide substitutions are enriched with T:C transition, which is consistent with the mutation signature caused by deactivity of RNA 3'-to-5' exoribonuclease (ExoN). The codon usage was similar between SARS-CoV-2 and other strains in beta-coronavirus lineage B, suggesting it had small impact on the mutation pattern. In comparison of SARS-CoV-2 WIV04 with Bat-SARSr-CoV RaTG13, the ratios of non-synonymous to synonymous substitution rates (dN/dS) was the lowest among all performed comparisons, reconfirming the evolution of SARS-CoV-2 under stringent selective pressure. Moreover, some sites of spike protein might be subjected to positive selection. Therefore, our results will help understanding the evolutionary mechanisms contribute to viral pathogenicity and its adaptation with hosts. \"},\n"," {'paper_id': '6d0bae20db2c120336ad5fea0440a90339b65c74',\n","  'section': 'Abstract',\n","  'text': \"Many viral infections can be prevented by immunizing with live, attenuated vaccines. Early methods of attenuation were hit-and-miss, and these are now much improved by genetic engineering. But even current attenuation methods operate on the principle of genetic harm, reducing the virus's ability to grow, which in turn limits the host immune response below that of infection by wild-type. We use mathematical models of the dynamics of virus and its control by innate and adaptive immunity to explore the tradeoff between attenuation of virus growth and the generation of immunity. Our analysis suggests that directed attenuation that disables key viral defenses against the host immune responses may attenuate viral growth without compromising, and potentially even enhancing the generation of immunity. We explore which immune evasion pathways should be attenuated and how attenuating multiple pathways could lead to robust attenuation of pathology with enhancement of immunity. \"},\n"," {'paper_id': '6d1b40bcdde63e22931509db389c6bf1949901dd',\n","  'section': 'Abstract',\n","  'text': 'Drugs and drug delivery systems have to traverse multiple biological barriers to achieve therapeutic efficacy. In diseases of mucosal-associated tissues such as cystic fibrosis (CF), successful delivery of gene and drug therapies remains a significant challenge due to an abnormally concentrated viscoelastic mucus, which prevents ~99% of all drugs and particles from penetrating the mucus barrier and the underlying epithelia for effective therapy, resulting in decreased survival. We used combinatorial peptide-presenting phage libraries and nextgeneration sequencing to identify hydrophilic, close to net-neutral charged peptides that penetrate the mucus barrier ex vivo in sputum from CF patients with ~600-fold better penetration than a positively charged control. After mucus penetration, nanoparticles conjugated with our selected peptides successfully translocated into lung epithelial cells derived from CF patients and demonstrated up to three-fold improved cell uptake compared to non-modified carboxylated-and gold standard PEGylated-nanoparticles. The selected peptides act as surface chemistries with synergistic functions to significantly improve the ability of drug delivery systems to overcome the human mucosal barriers and provide efficient cellular internalization. Our screening strategy provides a biologically-based discovery assay that directly addresses transport through mucus and cell barriers and has the potential to advance drug and gene delivery to multiple mucosal barriers. '},\n"," {'paper_id': '6dd88d9273b7892dd1b5cece675247592386ef50',\n","  'section': 'Abstract',\n","  'text': 'Background: As the novel coronavirus triggering COVID-19 has broken out in Wuhan, China and spread rapidly worldwide, it threatens the lives of thousands of people and poses a global threat on the economies of the entire world. However, infection with COVID-19 is currently rare in children. Objective: To discuss the latest findings and research focus on the basis of characteristics of children confirmed with COVID-19, and provide an insight into the future treatment and research direction. Methods: We searched the terms \"COVID-19 OR coronavirus OR SARS-CoV-2\" AND \"Pediatric OR children\" on PubMed, Embase, Cochrane library, NIH, CDC, and CNKI. The authors also reviewed the guidelines published on Chinese CDC and Chinese NHC. Results: We included 25 published literature references related to the epidemiology, clinical manifestation, accessary examination, treatment, and prognosis of pediatric patients with COVID-19. Conclusion: The numbers of children with COVID-19 pneumonia infection are small, and most of them come from family aggregation. Symptoms are mainly mild or even asymptomatic, which allow children to be a risk factor for transmission. Thus, strict epidemiological history screening is needed for early diagnosis and segregation. This holds especially for infants, who are more susceptible to infection than other age groups in pediatric age, but have most likely subtle and unspecific symptoms. They need to be paid more attention to. CT examination is a necessity for screening the suspected cases, because most of the pediatric patients are mild cases, and plain chest X-ray do not All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. : medRxiv preprint usually show the lesions or the detailed features. Therefore, early chest CT examination combined with pathogenic detection is a recommended clinical diagnosis scheme in children. The risk factors which may suggest severe or critical progress for children are: Fast respiratory rate and/or; lethargy and drowsiness mental state and/or; lactate progressively increasing and/or; imaging showed bilateral or multi lobed infiltration, pleural effusion or rapidly expending of lesions in a short period of time and/or; less than 3 months old or those who underly diseases. For those critical pediatric patients with positive SARS-CoV-2 diagnosis, polypnea may be the most common symptom. For treatment, the elevated PCT seen in children in contrast to adults suggests that the underlying coinfection/secondary infection may be more common in pediatric patients and appropriate antibacterial treatment should be considered. Once cytokine storm is found in these patients, anti-autoimmune or blood-purifying therapy should be given in time. Furthermore, effective isolation measures and appropriate psychological comfort need to be provided timely. '},\n"," {'paper_id': '6de59b8f8122ee30e23ef152639bb402264810f7',\n","  'section': 'Abstract',\n","  'text': 'Steroid compounds, which are expected to have dual functions in blocking host inflammation and MERS-CoV replication, were screened from a chemical library. Within this library, ciclesonide, an inhaled corticosteroid, suppressed human coronavirus replication in cultured cells, but did not suppress replication of respiratory syncytial virus or influenza virus. The effective concentration of ciclesonide to block SARS-CoV-2 (the cause of replication (EC 90 ) was 6.3 μ M. After the eleventh consecutive MERS-CoV passage in the presence of ciclesonide, a resistant mutation was generated, which resulted in an amino acid substitution (A25V) in nonstructural protein (NSP) 15, as identified using reverse genetics. A recombinant virus with the mutation was also resistant to ciclesonide suppression of viral replication. These observations suggest that the effect of ciclesonide was specific to coronavirus, suggesting this is a candidate drug for treatment of patients suffering MERS or COVID-19. author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '6e0a621ce934a48880db47ea2241cefdf5b43898',\n","  'section': 'Abstract',\n","  'text': 'cases is critical to support efforts to contain or reduce spread of viral infection in the community. Here we detected a significant association between domestic travel or travel to Europe and the identification of new cases in San Francisco, California, USA. These findings suggest that domestic and European travelers may need to be prioritized for evaluation of acute infection from COVID-19 in the setting of limited testing capacity. As of April 4 th , 2020, the COVID-19 pandemic caused by the novel SARS-CoV-2 '},\n"," {'paper_id': '6e152fc97c25c1da2b60c3062a97242427fe6b67',\n","  'section': 'Abstract',\n","  'text': '19 '},\n"," {'paper_id': '6ede7d40a9858d89341de780a3be9e8b43291151',\n","  'section': 'Abstract',\n","  'text': 'Currently, COVID-19 caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has been widely spread around the world; nevertheless, so far there exist no specific antiviral drugs for treatment of the disease, which poses great challenge to control and contain the virus. Here, we reported a research finding that SARS-CoV-2 invaded host cells via a novel route of CD147-spike protein (SP). SP bound to CD147, a receptor on the host cells, thereby mediating the viral invasion. Our further research confirmed this finding. First, in vitro antiviral tests indicated Meplazumab, an anti-CD147 humanized antibody, significantly inhibited the viruses from invading host cells, with an EC 50 of 24.86 μg/mL and IC 50 of 15.16 μg/mL. Second, we validated the interaction between CD147 and SP, with an affinity constant of 1.85×10 -7 M. Co-Immunoprecipitation and ELISA also confirmed the binding of the two proteins. Finally, the localization of CD147 and SP was observed in SARS-CoV-2 infected Vero E6 cells by immuno-electron microscope. Therefore, the discovery of the new route CD147-SP for SARS-CoV-2 invading host cells provides a critical target for development of specific antiviral drugs. '},\n"," {'paper_id': '6f04b753eea330cf3d704f36d34778ddb7927973',\n","  'section': 'Abstract',\n","  'text': \"Recently the World Health Organization has declared the outbreak of a severe acute respiratory syndrome coronavirus 2 (SARS-Cov-2) as a pandemic, and declared it as Public Health Emergency of International Concern. More than 683,536 positive cases and 32,139 deaths caused by novel corona virus 2019 (COVID-19) has affected 199 countries and territories. This pandemic can transform into an extremely destructive form if we still do not take it seriously. In the present study, we propose a generalized SEIR model of COVID-19 to study the behaviour of its transmission under different control strategies. In the model, all possible cases of human to human transmission are taken care and its reproduction number is formulated to analyse accurate transmission dynamics of the coronavirus outbreak. Optimal control theory is applied in the model to pretend the impact of various intervention strategies, including voluntary quarantine, isolation of infected individuals, improving an individual's immunity and hospitalisation. Also, effect of the control strategies on model is analysed graphically by simulating the model numerically. \"},\n"," {'paper_id': '6f53b9dff58818839c45410d11b58f0f3e327d8a',\n","  'section': 'Abstract',\n","  'text': 'Since late December 2019 a new epidemic outbreak has emerged from Whuhan, China. Rapidly the new coronavirus has spread worldwide. China '},\n"," {'paper_id': '6f86b074f8b9da74fbf32a1886f143cbd9aa5ca2',\n","  'section': 'Abstract',\n","  'text': 'medRxiv preprint Funding Information: Health Commission of Hubei Province scientific research project (No. WJ2019Q048 to M.Z.) We declare no conflict of interest. Abstract: Since SARS-CoV-2 infection was first identified in December 2019, it spread rapidly and a global pandemic of COVID-19 has occurred. ACE2, the receptor for entry into the target cells by SARS-CoV-2, was found to abundantly express in testes, including spermatogonia, Leydig and Sertoli cells. However, there is no clinical evidence about whether SARS-CoV-2 infection can affect male gonadal function so far. In this study, we compared the sex-related hormones between 81 reproductive-aged men with SARS-CoV-2 infection and 100 age-matched healthy men, and found that serum luteinizing hormone (LH) was significantly increased, but the ratio of testosterone (T) to LH and the ratio of follicle stimulating hormone (FSH) to LH were dramatically decreased in males with COVID-19. Besides, multivariable regression analysis indicated that c-reactive protein (CRP) level was significantly associated with serum T:LH ratio in COVID-19 patients. This study provides the first direct evidence about the influence of medical condition of COVID-19 on male sex hormones, alerting more attention to gonadal function evaluation among patients recovered from SARS-CoV-2 infection, especially the reproductive-aged men. '},\n"," {'paper_id': '6f8a531b4b84d4daaadcff9d7e03ee24274baf0b',\n","  'section': 'Abstract',\n","  'text': 'Background: Febrile illness is a major burden in African children, and non-malarial causes of 45 fever are uncertain. We built and employed IDseq, a cloud-based, open access, bioinformatics 46 platform and service to identify microbes from metagenomic next-generation sequencing of 47 tissue samples. In this pilot study, we evaluated blood, nasopharyngeal, and stool specimens 48 from 94 children (aged 2-54 months) with febrile illness admitted to Tororo District Hospital, 49 Uganda. 50 51 Results: The most common pathogens identified were Plasmodium falciparum (51.1% of 52 samples) and parvovirus B19 (4.4%) from blood; human rhinoviruses A and C (40%), 53 respiratory syncytial virus (10%), and human herpesvirus 5 (10%) from nasopharyngeal swabs; 54 and rotavirus A (50% of those with diarrhea) from stool. Among other potential pathogens, we 55 identified one novel orthobunyavirus, tentatively named Nyangole virus, from the blood of a 56 child diagnosed with malaria and pneumonia, and Bwamba orthobunyavirus in the nasopharynx 57 of a child with rash and sepsis. We also identified two novel human rhinovirus C species. 58 59 Conclusions: This exploratory pilot study demonstrates the utility of mNGS and the IDseq 60 platform for defining the molecular landscape of febrile infectious diseases in resource limited 61 areas. These methods, supported by a robust data analysis and sharing platform, offer a new tool 62 for the surveillance, diagnosis, and ultimately treatment and prevention of infectious diseases. 63 To address these challenges, we developed IDseq, a cloud-based open-source 90 bioinformatics platform and service for detection of microbial pathogens from metagenomic 91 next-generation sequencing (mNGS) data. IDseq requires minimal computational hardware and 92 is designed to enhance accessibility and build informatics capacity in resource limited regions. 93 Here, we leveraged IDseq and mNGS data to perform an exploratory proof-of-concept molecular 94 survey of febrile children in rural Uganda to characterize pathogens associated with fever, 95 including both well-recognized and novel causes of illness. 96 97 Analyses/Results 98 '},\n"," {'paper_id': '6fa1537d3cad42c332a707aa76b62a4d6d5e8736',\n","  'section': 'Abstract',\n","  'text': 'Background: Excessive monocyte/macrophage activation with the development of a cytokine storm and subsequent acute lung injury, leading to acute respiratory distress syndrome (ARDS) is a feared consequence of infection with COVID-19. The ability to recognize and potentially intervene early in those patients at greatest risk of developing this complication could be of great clinical utility. Methods: We performed detailed flow cytometric analysis of peripheral blood samples from 28 COVID-19 '},\n"," {'paper_id': '6fba5d79bdf83322d365179fa84ca02de2a16ebb',\n","  'section': 'Abstract',\n","  'text': 'Using a sensitive research-grade infrared camera, we find that common facial cosmetics and lotions mask skin temperature in assays of the human forehead. We test a family of 10 commonly-used cosmetic products and find that volatile liquids and creams lower thermal skin temperature by at least 2 C for up to 5-10 min and at least 1 C for up to 20 min, respectively. Powder and cream that contains brightening agent lower indefinitely the skin temperature sensed by infrared camera. With the qualification that these experiments were performed in a controlled laboratory setting rather than the mass crowd screening environment where infrared temperature sensing of humans sees widespread use, our tests suggest that for human subjects whose face was treated with certain cosmetics and lotions, infrared-based screening for elevated facial temperature (fever) can be unreliable. '},\n"," {'paper_id': '6fcf4019b0be24071210f97eda4baba53a6a79de',\n","  'section': 'Abstract',\n","  'text': 'A word count for text :2460 words All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '6fdd6c7b52765b183475c90fc17518cd256cda72',\n","  'section': 'Abstract',\n","  'text': 'Doubling Time (DT) is typically calculated for growth curves that show exponential growth, such as the cumulative number of COVID-19 cases day by day. DT represents the time it takes before the number of COVID-19 cases, in a certain country or area, doubles. Throughout the ongoing COVID-19 outbreak, DT values are continually changing. These changes are influenced by the measures that are recommended by the health authorities and implemented by governments. After the government-imposed shutdowns of Nordic Countries that were announced around the 12 th of March 2020, we followed the development of the DT in the region. Governments put in place measures never before experienced during peace time; working from home, closed schools and kindergartens, travel bans and social distancing. We conducted analyses to evaluate the effectiveness of these measures. Does it work? The initial set of results following the shutdown are encouraging, demonstrating a trend towards slower growth; however, this could be reversed if the measures that are in place now are abandoned too early. Premature optimism can be very costly. In this report we describe a method for monitoring the epidemic in real time and evaluating the effectiveness of the implemented measures. '},\n"," {'paper_id': '7009ba62167382fab0a64abc6545f248972ed028',\n","  'section': 'Abstract',\n","  'text': \"In the global effort to combat the COVID-19 pandemic, governments and public health agencies are striving to rapidly increase the volume and rate of diagnostic testing. The most common form of testing today employs Polymerase Chain Reaction in order to identify the presence of viral RNA in individual patient samples one by one. This process has become one of the most significant bottlenecks to increased testing, especially due to reported shortages in the chemical reagents needed in the PCR reaction. Recent technical advances have enabled High-Throughput PCR, in which multiple samples are pooled into one tube. Such methods can be highly efficient, saving large amounts of time and reagents. However, their efficiency is highly dependent on the frequency of positive samples, which varies significantly across regions and even within regions as testing criterion and conditions change. Here, we present two possible optimized pooling strategies for diagnostic SARS-CoV-2 testing on large scales, both addressing dynamic conditions. In the first, we employ a simple information-theoretic heuristic to derive a highly efficient re-pooling protocol: an estimate of the target frequency determines the initial pool size, and any subsequent pools found positive are re-pooled at half-size and tested again. In the range of very rare target (<0.05), this approach can reduce the number of necessary tests dramatically, for example, achieving a reduction by a factor of 50 for a target frequency of 0.001. The second method is a simpler approach of optimized one-time pooling followed by individual tests on positive pools. We show that this approach is just as efficient for moderate targetproduct frequencies (0.05<0.2), for example, achieving a two-fold in the number of when the frequency of positive samples is 0.07. These strategies require little investment, and they offer a significant reduction in the amount of materials, equipment and time needed to test large numbers of samples. We show that both these pooling strategies are roughly comparable to the absolute upperbound efficiency given by Shannon's source coding theorem. We compare our strategies to the naïve way of testing and to alternative matrix-pooling methods. Most importantly, we offer straightforward, practical pooling instructions for laboratories that perform large scale PCR assays to diagnose SARS-CoV-2 viral particles. These two pooling strategies may offer ways to alleviate the bottleneck currently preventing massive expansion of SARS-CoV-2 testing around the world. \"},\n"," {'paper_id': '70242f0f937a11c385935ef0242ab86b56dbcdad',\n","  'section': 'Abstract',\n","  'text': \"Background: COVID-19 is still becoming an increasing global threat to public health. More detailed and specific characteristics of COVID-19 are needed to better understand this disease. Additionally, durations of COVID-19, e.g., the average time from exposure to recovery, which is of great value in understanding this disease, has not been reported so far. Aims: To give the information on clinical characteristics and different durations of COVID-19 and to identify the potential risk factors for longer hospitalization duration. In this retrospective study, we enrolled 77 patients (mean age: 52±20 years; 44.2% males) with laboratory-confirmed COVID-19 admitted to Beijing YouAn Hospital during 21 st Jan and 8 th February 2020. Epidemiological, clinical and radiological data on admission were collected; complications and outcomes were followed up until 29 th February 2020. The study's endpoint was the discharge within two weeks. Cox proportional-hazards regression was performed to identify risk factors for longer hospitalization duration. Of 77 patients, there are 34 (44.2%) males, 24 (31.2%) with comorbidities, 22 (28.6%) lymphopenia, 20 (26.0%) categorized as severe patients, and 28 (36.4%) occurred complications. By the end of follow-up, 64 (83.1%) patients were discharged home after being tested negative for SARS-CoV-2 infections, 8 remained in hospital and 5 died. 36 (46.8%) patients were discharged within 14 days and thus reached the study endpoint, including 34 (59.6%) of 57 non-severe patients and 2 (10%) of 20 severe patients. The overall cumulative probability of the endpoint was 48.3%. Hospital length of stay and duration of exposure to discharge for 64 discharged patients were 13 (10-16.5) and 23 (18-24.5) days, respectively. Multivariable stepwise Cox regression model showed bilateral pneumonia on CT scan, shorter time from the illness onset to admission, severity of disease and lymphopenia were independently associated with longer hospitalized duration. Conclusions: COVID-19 has significantly shorter duration of disease and hospital length of stay than SARS. Bilateral pneumonia on CT scan, shorter period of illness onset to admission, lymphopenia, severity of disease are the risk factors for longer hospitalization duration of COVID-19. \"},\n"," {'paper_id': '70c7260fc0702bd4a554421ac68972c991313e56',\n","  'section': 'Abstract',\n","  'text': \"Using incipient lines of the Collaborative Cross (CC), a murine genetic reference population, we previously identified a quantitative trait loci (QTL) associated with low SARS-CoV titer. In this study, we integrated sequence information and RNA expression of genes within the QTL to identify mucin 4 (Muc4) as a high priority candidate for controlling SARS-CoV titer in the lung. To test this hypothesis, we infected Muc4 -/mice and found that female, but not male, Muc4 -/mice developed more weight loss and disease following infection with SARS-CoV. Female Muc4 -/mice also had more difficulty breathing despite reduced lung pathology; however, no change in viral titers was observed. Comparing across viral families, studies with chikungunya virus, a mosquito-borne arthralgic virus, suggests that Muc4's impact on viral pathogenesis may be widespread. Although not confirming the original titer QTL, our data identifies a role for Muc4 in the SARS-CoV disease and viral pathogenesis. Given the recent emergence of SARS-CoV-2, this work suggest that Muc4 expression plays a protective role in female mice not conserved in male mice following SARS-CoV infection. With the SARS-CoV-2 outbreak continuing, treatments that modulate or enhance Muc4 activity may provide an avenue for treatment and improved outcomes. In addition, the work highlights the importance of studying host factors including host genetics and biological sex as key parameters influencing infection and disease outcomes. \"},\n"," {'paper_id': '70cc2e5152d3dc4d44494124ff556c9bbe9e6f41',\n","  'section': 'Abstract',\n","  'text': '1 Background: A new virus broke out in Wuhan, Hubei, China, and was later named 2019 novel 2 coronavirus (2019-nCoV). The clinical characteristics of severe pneumonia caused by 2019-nCoV are 3 still not clear. 4 Objectives: The aim of this study was to explore the clinical characteristics and risk factors of the 5 severe pneumonia caused by the 2019-nCoV in Wuhan, China. 6 Method: The study included patients hospitalized at the central hospital of Wuhan who had been 7 diagnosed with a pneumonia caused by the novel coronavirus. Clinical features, chronic co-8 morbidities, demographic data, laboratory examinations, and chest computed tomography (CT) scans 9 were reviewed through electronic medical records. SPSS was used for data analysis to explore the 10 clinical characteristics and risk factors of the patients with the severe pneumonia. '},\n"," {'paper_id': '70e1f2a0ff48a5d24e75fc2920d757968b224d79',\n","  'section': 'Abstract',\n","  'text': '341 words; main text: 2964 words '},\n"," {'paper_id': '7106f766224a4fcb29ef781c8f27b34746b84a55',\n","  'section': 'Abstract',\n","  'text': 'The coronavirus disease 2019 has grown up to be a pandemic within a short span of time. To investigate transmission dynamics and then determine control methodology, we took epidemic in Wuhan as a study case. Unfortunately, to our best knowledge, the existing models are based on the common assumption that the total population follows a homogeneous spatial distribution, which is not the case for the prevalence occurred both in the community and in hospital due to the difference in the contact rate. To solve this problem, we propose a novel epidemic model called SEIR-HC, which is a novel epidemic model with two different social circles. Using the model alongside the exclusive optimization algorithm, the spread process of COVID-19 epidemic in Wuhan city is reproduced and then the propagation characteristics and unknown data are estimated. Furthermore, the control measures implemented in Wuhan are assessed and the control methodology of COVID-19 is discussed to provide guidance for limiting the epidemic spread. '},\n"," {'paper_id': '7116ddab89281e7641fb659019c90ee3715f5039',\n","  'section': 'Abstract',\n","  'text': 'Persistent bacterial bronchitis is a leading cause of chronic wet cough in young children. This 20 study aimed to characterise the respiratory bacterial microbiota of healthy children and to 21 assess the impact of the changes associated with the development of persistent bacterial 22 bronchitis. 23 Blind, protected brushings were obtained from 20 healthy controls and 24 children with 24 persistent bacterial bronchitis, with an additional directed sample obtained from persistent 25 bacterial bronchitis patients. DNA was extracted, quantified using a 16S rRNA gene 26 quantitative PCR assay prior to microbial community analysis by 16S rRNA gene 27 sequencing. 28 No significant difference in bacterial diversity or community composition (R 2 = 0.01, P = 29 0.36) was observed between paired blind and non-blind brushes, showing that blind 30 brushings are a valid means of accessing the airway microbiota. This has important 31 implications for collecting lower respiratory samples from healthy children. 32 A significant decrease in bacterial diversity (P < 0.001) and change in community 33 composition (R 2 = 0.08, P = 0.004) was observed between controls and patients. Bacterial 34 communities within patients with PBB were dominated by Proteobacteria, and indicator 35 species analysis showed that Haemophilus and Neisseria were significantly associated with 36 the patient group. In 15 (52.9%) cases the dominant organism by sequencing was not 37 identified by standard routine clinical culture. 38 The bacteria present in the lungs of patients with persistent bacterial bronchitis were less 39 diverse in terms of richness and evenness. The results validate the clinical diagnosis, and 40 suggest that more attention to bacterial communities in children with chronic cough may lead 41 to more rapid recognition of this condition with earlier treatment and reduction in disease 42 burden. 43 '},\n"," {'paper_id': '712f1a74a5083d690832eed5fd9ff16a04434d8d',\n","  'section': 'Abstract',\n","  'text': 'Novel Corona virus (COVID-19 or 2019-nCoV) is an emerging global health concern that requires a rapid diagnostic test. Quantitative reverse transcription PCR (qRT-PCR) is currently the standard for COVID-19 detection; however, Reverse Transcription Loop-Mediated Isothermal Amplification (RT-LAMP) may allow for faster and cheaper field based testing at point-of-risk. The objective of this study was to develop a rapid screening diagnostic test that could be completed in under 30 minutes. Simulated patient samples were generated by spiking serum, urine, saliva, oropharyngeal swabs, and nasopharyngeal swabs with a portion of the COVID-19 nucleic sequence. The samples were tested using RT-LAMP as well as by conventional qRT-PCR. Specificity of the RT-LAMP was evaluated by also testing against other related coronaviruses. RT-LAMP specifically detected COVID-19 in simulated patient samples. This test was performed in under 30 minutes. This approach could be used for monitoring of exposed individuals or potentially aid with screening efforts in the field and potential ports of entry. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '71b5efee7d4d7de8ed88da63c0fa12944a811dc4',\n","  'section': 'Abstract',\n","  'text': 'The coronavirus disease COVID-19, caused by emerging SARS-CoV-2, has posed serious 15 threats to global public health, economic and social stabilities, calling for the prompt 16 development of therapeutics and prophylactics. In this study, we firstly verified that 17 SARS-CoV-2 uses human ACE2 as a cell receptor and its spike (S) protein mediates high 18 membrane fusion activity. Comparing to that of SARS-CoV, the heptad repeat 1 (HR1) 19 sequence in the S2 fusion protein of SARS-CoV-2 possesses markedly increased α-helicity 20 and thermostability, as well as a higher binding affinity with its corresponding heptad repeat 2 21 (HR1) site. Then, we designed a HR2 sequence-based lipopeptide fusion inhibitor, termed 22 IPB02, which showed highly poent activities in inibibiting the SARS-CoV-2 S 23 protein-mediated cell-cell fusion and pseudovirus infection. IPB02 also inhibited the 24 SARS-CoV pseudovirus efficiently. Moreover, the strcuture and activity relationship (SAR) 25 of IPB02 were characterzized with a panel of truncated lipopeptides, revealing the amino acid 26 motifs critical for its binding and antiviral capacities. Therefore, the presented results have 27 provided important information for understanding the entry pathway of SARS-CoV-2 and the 28 design of antivirals that target the membrane fusion step. 29 30 '},\n"," {'paper_id': '729a9faa0dd40ae0392c7590418583096673a5b4',\n","  'section': 'Abstract',\n","  'text': 'Due to its excessively high capacity for human-to-human transmission, the 2019 novel coronavirus disease , first reported in Wuhan in China, spread rapidly to the entire nation and beyond, and has now been declared a global public health emergency. Understanding the transmission pattern of the virus and the efficacy of transmission control measures is crucial to ensuring regional and global disease control. Here we propose a simple model based on exponential infectious growth, but with a timevarying, largely damping, transmission rate. This model provides an excellent fit to the existing data from the 102 countries and regions which have reported cases for more than 6 days, and, we think, has largely captured the transmission patterns of the COVID-19 outbreak under a variety of intervention and control measures. We found that the damping rate, defined as the rate of the exponential decline in transmission rate, ranged from -0.125 to 0.513 d -1 globally (a negative damping rate represents acceleration in spread). The estimated peak time (when the fastest spread occurs) and the final number of infections were found to be greatly affected by the damping rate. Successful control measures, such as those implemented in China and South Korea, have resulted in a clear pattern of exponential damping in the viral spread (also shown during the 2003 outbreak of Severe Acute Respiratory Syndrome, SARS). The damping rate, therefore, could be used as an indicator for the efficacy of implemented control measures. Our model suggests that the COVID-19 outbreak is currently accelerating worldwide, especially rapidly in certain countries (e.g. USA and Australia) where exponential damping is yet to emerge. Consistent with the message from the World Health Organisation (WHO), we thus strongly suggest all countries to take active measures to contain this global pandemic. Slight increments in the damping rate from additional control efforts, especially in countries showing weak or no exponential damping in COVID-19 transmission, could lead to a radically more positive outcome in the fight to contain the pandemic. '},\n"," {'paper_id': '72b346dfe312aa5ce146236f988ca8e89e417652',\n","  'section': 'Abstract',\n","  'text': 'The international spread of the novel, pathogenic SARS-coronavirus 2 (SARS-CoV-2) poses a global challenge on both healthcare and society. A multitude of research efforts worldwide aim at characterizing the cellular factors involved in viral transmission in order to reveal therapeutic targets. For a full understanding of the susceptibility for SARS-CoV-2 infection, the cell type-specific expression of the host cell surface receptor is necessary. The key protein suggested to be involved in host cell entry is Angiotensin I converting enzyme 2 (ACE2), and its expression has been reported in various human organs, in some cases with inconsistent or contradictory results. Here, we aim to verify a reliable expression profile of ACE2 in all major human tissues and cell types. Based on stringently validated immunohistochemical analysis and highthroughput mRNA sequencing from several datasets, we found that ACE2 expression is mainly localized to microvilli of the intestinal tract and renal proximal tubules, gallbladder epithelium, testicular Sertoli cells and Leydig cells, glandular cells of seminal vesicle and cardiomyocytes. The expression in several other previously reported locations, including alveolar type II (AT2) cells, could not be confirmed. Furthermore, ACE2 expression was absent in the AT2 lung carcinoma cell line A549, often used as a model for viral replication studies. Our analysis suggests that the expression of ACE2 in the human respiratory system appears to be limited, and the expression of the receptor in lung or respiratory epithelia on the protein level is yet to be confirmed. This raises questions regarding the role of ACE2 for infection of human lungs and highlights the need to further explore the route of transmission during SARS-CoV-2 infection. '},\n"," {'paper_id': '72b41bbcaed1ccbe1e6a223f9e9b89a85ad5d4dd',\n","  'section': 'Abstract',\n","  'text': 'Novel coronavirus pneumonia (NCP) is an emerging, highly contagious community acquired pneumonia (CAP) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Highly efficient and accurate microbiological laboratory assay is essential to confirm the SARS-CoV-2 infection, rule out other pathogens that can cause CAP, and monitor secondary infections. Here, we enrolled and provide microbiological analysis for 129 suspected and 52 transferred confirmed NCP patients hospitalized in the First Affiliated '},\n"," {'paper_id': '72f80b98d15a0f4fd1e27bbddd82df2de8297bfa',\n","  'section': 'Abstract',\n","  'text': 'Data visualization is an important tool for exploring and communicating findings in medical research, and specially in epidemiological surveillance. The COVID19-Tracker app systematically produces daily updated data visualization and analysis of SARS-CoV-2 epidemic in Spain. It collects automatically daily data on COVID-19 diagnosed cases, intensive care unit admissions, and mortality, from February 24 th , 2020 onwards. Two applications have already been developed; 1) to analyze data trends and estimating short-term projections; and 2) To assess the effect of the lockdown on the trend of incident data. We are currently planning to improve the app by uploading shortly new applications for data visualization and analysis, which may help for a better understanding of the SARS-CoV-2 epidemic data in Spain. '},\n"," {'paper_id': '7326e5ee9bba1b74f32284948fac870c62f56969',\n","  'section': 'Abstract',\n","  'text': 'The novel coronavirus disease-2019 has been spreading around the world rapidly and declared as a pandemic by WHO. Here, we compared the ABO blood group distribution in 2,173 patients with COVID-19 confirmed by SARS-CoV-2 test from three hospitals in Wuhan and Shenzhen, China with that in normal people from the corresponding regions. The results showed that blood group A was associated with a higher risk for acquiring COVID-19 compared with non-A blood groups, whereas blood group O was associated with a lower risk for the infection compared with non-O blood groups. This is the first observation of an association between the ABO blood type and COVID-19. It should be emphasized, however, that this is an early study with limitations. It would be premature to use this study to guide clinical practice at this time, but it should encourage further investigation of the relationship between the ABO blood group and the COVID-19 susceptibility. . All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '7344c6de7ce2258e74ce131bed72e6ad465f0ce8',\n","  'section': 'Abstract',\n","  'text': 'In order to quantitatively characterize the epidemic of COVID-19, useful relations among parameters describing an epidemic in general are derived based on the Kermack-McKendrick model. The first relation is 1/τ grow =1/τ trans −1/τ inf , where τ grow is the time constant of the exponential growth of an epidemic, τ trans is the time for a pathogen to be transmitted from one patient to uninfected person, and the infectious time τ inf is the time during which the pathogen keeps its power of transmission. The second relation p(∞ ) ≈ 1− exp(− (R 0 − 1)/0.60) is the relation between p(∞ ), the final size of the disaster defined by the ratio of the total infected people to the population of the society, and the basic reproduction number, R 0 , which is the number of persons infected by the transmission of the pathogen from one infected person during the infectious time. The third relation 1/τ end = '},\n"," {'paper_id': '734779fad93249a2f6fede6afd10eeff7b37919b',\n","  'section': 'Abstract',\n","  'text': 'There is an urgent need to project how transmission of the novel betacoronavirus SARS-CoV-2 will unfold in coming years. These dynamics will depend on seasonality, the duration of immunity, and the strength of cross-immunity to/from the other human coronaviruses. Using data from the United States, we measured how these factors affect transmission of human betacoronaviruses HCoV-OC43 and HCoV-HKU1. We then built a mathematical model to simulate transmission of SARS-CoV-2 through the year 2025. We project that recurrent wintertime outbreaks of SARS-CoV-2 will probably occur after an initial pandemic wave. We summarize the full range of plausible transmission scenarios and identify key data still needed to distinguish between them, most importantly longitudinal serological studies to determine the duration of immunity to SARS-CoV-2. Main text. The ongoing SARS-CoV-2 epidemic has caused nearly 80,000 detected cases of COVID-19 illness and claimed over 2,500 lives as of 24 Feb 2020 (1). With sustained transmission reported in China, Japan, Iran, Italy, and South Korea (1), the outbreak is on the verge of becoming a pandemic. The required intensity, duration, and urgency of public health responses will depend on how the initial pandemic wave unfolds and on the subsequent transmission dynamics of SARS-CoV-2. One possibility is that SARS-CoV-2 will follow its closest genetic relative, SARS-CoV, and be eradicated by intensive public health measures after causing a brief but intense epidemic (2). Increasingly, public health authorities consider this scenario unlikely (3). Alternatively, the transmission of SARS-CoV-2 could resemble that of pandemic influenza by circulating seasonally after causing an initial global wave of infection (4). Such a scenario could reflect the previous emergence of known human coronaviruses from zoonotic origins e.g. human coronavirus (HCoV) OC43 (5). This paper identifies viral, environmental, and immunologic factors which in combination will determine which scenarios in fact play out, and identifies key data needed to distinguish between them. The transmission dynamics of the SARS-CoV-2 epidemic will depend on factors including the degree of seasonal variation in transmission strength, the duration of immunity, and the degree of cross-immunity between SARS-CoV-2 and other coronaviruses. SARS-CoV-2 belongs to the betacoronavirus genus, which includes the SARS coronavirus, MERS coronavirus, and two other human coronaviruses, HCoV-OC43 and HCoV-HKU1. The SARS and MERS coronaviruses cause severe illness with case fatality rates of 9 and 36% respectively, but the transmission of both has remained limited (6). HCoV-OC43 and HCoV-HKU1 infections may be asymptomatic or associated with mild to moderate upper respiratory tract illness; these HCoVs are considered the second most common cause of the common cold (6). While investigations into the spectrum of illness caused by SARS-CoV-2 are ongoing, recent evidence indicates the majority of cases experience mild illness with more limited occurrence of severe lower respiratory infection (7). Current COVID-19 case fatality rates in China are estimated at 2.9% within Hubei province, and 0.4% outside (8), suggesting lower severity than SARS and MERS but higher severity than HCoV-OC43 and HCoV-HKU1. In terms of transmission, the ability of SARS-CoV-2 to cause widespread infection is more reflective of HCoV-OC43 and HCoV-HKU1 than of its more clinically severe relatives. HCoV-OC43 and HCoV-HKU1 cause annual wintertime outbreaks of respiratory illness in temperate regions (9, 10), suggesting that wintertime climate and host behaviors may facilitate transmission, as is true for influenza (11) (12) (13) . Immunity to HCoV-OC43 and HCoV-HKU1 appears to wane appreciably within one year (14), while SARS infection can induce longer-lasting immunity (15). The . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': '7352452a89aa55529c073dfece3ea425bb91d7d2',\n","  'section': 'Abstract',\n","  'text': 'The SARS-CoV-2 (COVID-19) novel corona virus represents a significant health risk, particularly in older patients. Cancer is one of the leading causes of death in most rich countries, and delivering chemotherapy may be associated with increased risk in the presence of a pandemic infection. Estimating this risk is crucial in making decisions about balancing risks and benefits from administering chemotherapy. However, there are no specific data about chemotherapy risks per se. Here we develop a simple model to estimate the potential harms in patients undergoing chemotherapy during a COVID outbreak. We use age-related case fatality rates as a basis for estimating risk, and use previous data from risk of death during influenza outbreaks to estimate the additional risk associated with chemotherapy. We use data from randomised trials to estimate benefit across a range of curative and palliative settings, and address the balance of benefit against the risk of harm. We then use those data to estimate the impact on national chemotherapy delivery patterns. '},\n"," {'paper_id': '736b4793b11d01f4b55ea53eaf59f7325493b27b',\n","  'section': 'Abstract',\n","  'text': \"Objective: Health system resilience and resilience of a country include the capacity of health personnel, institutions, and populations to prepare for and effectively respond to crises. This study investigates the knowledge and attitudes of the public concerning Ebola Virus Disease in Trinidad and Tobago. Design and Methods: A cross sectional study whereby respondents (n = 920) were sampled from public places. Data were collected via interviewer administered questionnaires. Data were analysed using SPSS version 23. Results: The response rate was 67.6 % (622/920). The main age category of responders was the 20 to 30 year age category (40.5%); responders were mostly female (58.0 %). Regarding knowledge, there were significant differences among occupational categories (F = 2.811, df1 = 6, df2 = 571, p-value = 0.011). Tukey's HSD post hoc test revealed that the mean knowledge scores for professional and sales occupations differed significantly (p-value = 0.003). There was a significant association between being afraid to go for treatment and age (p-value = 0.001). Significant associations were also found between occupational grouping and education attainment with opinion about the preparedness of private medical facilities, likelihood to shun family members with Ebola, being afraid to go for treatment and preference for traditional medicine (p-value <0.05). This study highlights opportunities for community engagement to enhance health system resilience during outbreaks which would maximise national and global health security. All rights reserved. No reuse allowed without permission. \"},\n"," {'paper_id': '7395de48b2402e5e1561a93b99e8aae8728c6ff1',\n","  'section': 'Abstract',\n","  'text': \"TRIM25 is a novel RNA-binding protein and a member of the Tripartite Motif (TRIM) family of E3 ubiquitin ligases, which plays a pivotal role in the innate immune response. Almost nothing is known about its RNA-related roles in cell biology. Furthermore, its RNA-binding domain has not been characterized. Here, we reveal that RNA-binding activity of TRIM25 is mediated by its PRY/SPRY domain, which we postulate to be a novel RNA-binding domain. Using CLIP-seq and SILAC-based co-immunoprecipitation assays, we uncover TRIM25's endogenous RNA targets and protein binding partners. Finally, we show that the RNAbinding activity of TRIM25 is important for its ubiquitin ligase function. These results reveal new insights into the molecular roles and characteristics of RNA-binding E3 ubiquitin ligases and demonstrate that RNA could be an essential factor for their biological functions. All rights reserved. No reuse allowed without permission. Although TRIM25 has been generally well characterized, the functional importance of its RNA-binding activity remains unclear. Our recent study provided first clues: we have shown that the binding of TRIM25 to pre-let-7 leads to its more efficient Lin28a-mediated uridylation (Choudhury et al., 2014) . This finding raised the possibility that RNA processing may be spatially controlled through post-translational protein modifications by RNA-binding protein-modifying enzymes, such as TRIM25. Furthermore, TRIM25 could be using RNA as a scaffold to ubiquitinate its targets. Importantly, the transcriptome and proteome-wide targets of TRIM25 have not yet been identified. Finally, its RNA-binding domain has not been unequivocally described. Here, using data from a proteome-wide analysis of RNA-binding domains, RNA pull-downs and EMSA, we show that the PRY/SPRY domain of TRIM25 harbors its RNA-binding activity. Furthermore, TRIM25 CLIP-seq and quantitative SILAC-based coimmunoprecipitation assays reveal a comprehensive overview of interacting RNAs and proteins, respectively. Our CLIP-seq data identifies hundreds of coding and non-coding RNAs bound by TRIM25. We show that TRIM25 binds a distinct set of RNAs and prefers to bind sequences rich in G and C. We also demonstrate that TRIM25 associates with many RNA-binding proteins, thus suggesting it could play a role in regulating RNA metabolism. Crucially, we present evidence that the binding of TRIM25 to RNA is important for its ubiquitin ligase activity towards itself (autoubiquitination) and physiologically relevant target ZAP. Finally, by showing that the PRY/SPRY regions of other TRIM proteins rescue TRIM25's RNA-binding activity, we propose that TRIM25 represents a novel group of E3 ligases with RNA-binding activity that use PRY/SPRY as a novel RNA-binding domain. These results provide original insights into the RNA-mediated activities of RNA-binding E3 ubiquitin ligases. All rights reserved. No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/200410 doi: bioRxiv preprint RESULTS TRIM25 directly interacts with RNA through the PRY/SPRY domain. \"},\n"," {'paper_id': '73998a3f565b661816d48aa7c20fa5ae617864d5',\n","  'section': 'Abstract',\n","  'text': 'A newly identified novel coronavirus (2019-nCoV) has caused numerous acute respiratory syndrome cases in Wuhan China from December 2019 to Feb 2020. Its fast spreading to other provinces in China and overseas is very likely causing a pandemic. Since the novel coronavirus has been reported to be capable of endangering thousands of lives, it is extremely important to find out how the coronavirus is transmitted in human organs. Apart from fever and respiratory complications, gastrointestinal symptoms are observed in some patients with 2019-nCoV but the significance remains undetermined. The cell receptor angiotensin covering enzyme II (ACE2), which is the major receptor of SARS-nCoV, has been reported to be a cellular entry receptor of 2019-nCoV as well. Here, to more precisely explore the potential pathogen transmission route of the 2019-nCoV infections in the gastrointestinal tract, we analyzed the ACE2 RNA expression profile in the colon tissue of healthy adults and colorectal cancer patients of our cohort and other databases. The data indicates that ACE2 is mainly expressed in epithelial cells of the colon. The expression of ACE2 is gradually increased from healthy control, adenoma to colorectal cancer patients in our cohort as well as in the external Asian datasets. According to the expression profile of ACE2 in colon epithelial cells, we speculate adenoma and colorectal cancer patients are more likely to be infected with 2019-nCoV than healthy people. Our data may provide a theoretical basis for the classification and management of future 2019-nCoV susceptibility people in clinical application. '},\n"," {'paper_id': '73a954bbd21a08cf0a662b37f1f97989285969c4',\n","  'section': 'Abstract',\n","  'text': 'Classifying the mechanisms of antibiotic failure has led to the development of new treatment strategies for killing bacteria. Among the currently described mechanismswhich include resistance, persistence and tolerancewe propose defiance as a subclass of antibiotic failure specific to prodrugs. Using locked antimicrobial peptides (AMP) that are activated by bacterial 5 proteases as a prototypic prodrug, we observe that although treatment eliminates bacteria across the vast majority of environmental conditions (e.g., temperature, concentration of growth nutrients), bacteria spontaneously switch from susceptibility to defiance under conditions that alter the competing rates between bacterial proliferation and prodrug activation. To identify the determinants of this switch-like behavior, we model bacteria-prodrug dynamics as a multi-rate 10 feedback system and identify a dimensionless quantity we call the Bacterial Advantage Heuristic (BAH) that perfectly classifies bacteria as either defiant or susceptible across a broad range of treatment conditions. By recognizing that the bacterial switch from susceptibility to defiance behaves analogously to electronic transistors, we construct prodrug logic gates (e.g., AND, OR, NOT, etc.) to allow assembly of an integrated 3-bit multi-prodrug circuit that kills defiant bacteria 15 under all possible combinations of BAH values (i.e., 000, 001, …, 111) that represent a broad range of possible treatment conditions. Our study identifies a form of bacterial resistance specific to prodrugs that is described by a predictive dimensionless constant to reveal logic-based treatment strategies using multi-prodrug biological circuits. '},\n"," {'paper_id': '73c8af41cfdbf52c0dfba37727e3b94cb56b495e',\n","  'section': 'Abstract',\n","  'text': 'Wuhan Novel Coronavirus disease outbreak has become a global outbreak which has raised the concern of scientific community to design and discover a definitive cure against this deadly virus which has caused deaths of numerous infected people upon infection and spreading. To date, no antiviral therapy or vaccine is available which can effectively combat the infection caused by this virus. This study was conducted to design possible epitope-based subunit vaccines against the SARS-CoV-2 virus using the approaches of reverse vaccinology and immunoinformatics. Upon continual computational experimentation three possible vaccine constructs were designed and one vaccine construct was selected as the best vaccine based on molecular docking study which is supposed to effectively act against SARS-CoV-2. Later, molecular dynamics simulation and in silico codon adaptation experiments were carried out in order to check biological stability and find effective mass production strategy of the selected vaccine. Hopefully, this study will contribute to uphold the present efforts of the researches to secure a definitive treatment against this lethal virus. '},\n"," {'paper_id': '73d80c8f5780d70bd8d343188c56e898e91557b6',\n","  'section': 'Abstract',\n","  'text': \"Middle East respiratory syndrome coronavirus (MERS-CoV) is a major emerging zoonotic infectious disease. Since its first outbreak in 2012, the virus has repeatedly transmitted from camels to humans with 2,468 confirmed cases, causing 851 deaths. To date, there are no efficacious drugs and vaccines against : bioRxiv preprint 2 MERS-CoV, increasing its potential to cause a public health emergency. A critical step in the life cycle of MERS-CoV is the fusion with the host cell with its spike (S) protein as main determinant of viral entry. Proteolytic cleavage of S exposes its fusion peptide (FP), which initiates membrane fusion. Previous studies on the related severe acute respiratory syndrome coronavirus (SARS-CoV) FP have shown that calcium (Ca 2+ ) plays an important role for fusogenic activity via a Ca 2+ binding pocket with conserved glutamic acid (E) and aspartic acid (D) residues. SARS-CoV and MERS-CoV FP share a high sequence homology and here, we investigated whether Ca 2+ is required for MERS-CoV fusion by substituting E and D residues in the MERS-CoV FP with neutrally charged alanines. Upon verifying mutant cell surface expression and proteolytic cleavage, we tested the mutants ability to mediate infection of pseudoparticles (PPs) on host cells without and with Ca 2+ . Our results demonstrate that intracellular Ca 2+ enhances MERS-CoV WT PPs infection by approximately two-fold and that E891 is a crucial residue for Ca 2+ interaction. Electron spin resonance revealed that this enhancement could be attributed to Ca 2+ increasing MERS-CoV FP fusion-relevant membrane ordering. Intriguingly, isothermal calorimetry titration showed that MERS-CoV FP binds one Ca 2+ , as opposed to SARS-CoV FP which binds two. Our data suggests that there are significant differences in FP-Ca 2+ interactions of MERS-CoV and SARS-CoV FP despite their high sequence similarity and that the number of Ca 2+ ions interacting with the FP has implications on the fusion dynamics of the virus. 3 dromedary camels [3] [4] [5] . It is currently believed that disease outbreaks in humans result from interactions with dromedary camels. Although transmissions between humans is rare, this route was responsible for a widespread outbreak of MERS-CoV in 2015 in South Korea [6] . Moreover, it was demonstrated that MERS-like bat CoV could potentially infect human tissues in the absence of an intermediate host when the respective tissue expresses a protease that is able to prime the virus for fusion [7] . More recently, the World Health Organization (WHO) listed MERS-CoV as one of ten diseases that requires urgent accelerated research and development because of its potential to cause a public health emergency with few or no efficacious drugs and/or vaccines to counter them [8] . To address these needs, it is important to have a more nuanced understanding of MERS-CoV infection process to develop therapeutics to target crucial steps in MERS-CoV infection. The MERS-CoV spike (S) protein contains a receptor binding subunit (S1) and a membrane fusion subunit (S2), which facilitates binding to the host cell surface receptor, dipeptidyl peptidase 4 (DPP4), and fusion with the host cell membrane, respectively [10, 11] . To mediate fusion with the host cell membrane, S needs to be cleaved by either host cell surface proteases or endosomal proteases (such as transmembrane protease serine 2 (TMPRSS2), furin, or cathepsins) [12] [13] [14] [15] [16] . Cleavage can occur at two different sites: the S1 and S2 boundary (S1/S2), and adjacent to a fusion peptide (FP) within the S2 domain (S2') [17, 18] . It is suggested that both cleavage events act in concert to promote membrane fusion and viral infectivity. The first cleavage event takes place at the S1/S2 site, which then is believed to open up the S2' site for further proteolytic processing, resulting in the exposure of the FP. As a consequence, the FP inserts into the host membrane, mediating membrane fusion [19] . MERS-CoV viral fusion can occur at two different locations in the host cell: at the plasma membrane or at the endosomal membrane. Host cell surface proteases, such as TMPRSS2, can cleave MERS-CoV S for entry directly at the plasma membrane [13] . In the absence of cell surface proteases, MERS-CoV is endocytosed, \"},\n"," {'paper_id': '74339c753a0f2ad3b1e26afa83f5b3d801c4a338',\n","  'section': 'Abstract',\n","  'text': 'Background. We investigated a likely scenario of COVID-19 spreading in Brazil through the complex airport network of the country, for the 90 days after the first national occurrence of the disease. After the confirmation of the first imported cases, the lack of a proper airport entrance control resulted in the infection spreading in a manner directly proportional to the amount of flights reaching each city, following first occurrence of the virus coming from abroad. Methodology. We developed a SIR (Susceptible-Infected-Recovered) model divided in a metapopulation structure, where cities with airports were demes connected by the number of flights. Subsequently, we further explored the role of Manaus airport for a rapid entrance of the pandemic into indigenous territories situated in remote places of the Amazon region. Results. The expansion of the SARS-CoV-2 virus between cities was fast, directly proportional to the airport closeness centrality within the Brazilian air transportation network. There was a clear pattern in the expansion of the pandemic, with a stiff exponential expansion of cases for all cities. The more an airport showed closeness centrality, the greater was its vulnerability to SARS-CoV-2. We discussed the weak pandemic control performance of Brazil in comparison with other tropical, developing countries, namely India and Nigeria. Finally, we proposed measures for containing virus spreading taking into consideration the scenario of high poverty. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. : medRxiv preprint 105 106 + 1 = + 107 108 where the indexes t and t+1 represent the present time and the next time, respectively, 109 and N=S+I+R is the total constant population. In this work, we proposed two 110 modifications of the SIR model. The first one is related to the fact that we considered all 111 Brazilian cities that have an airport. Thus, we had S i , I i , and R i where i was a given city. '},\n"," {'paper_id': '7433cbab6bbad6768694e2c35ad3b0405d10d7ec',\n","  'section': 'Abstract',\n","  'text': 'We investigate the transmissibility of coronavirus for symptomatic and asymptomatic patients using the Ningbo Covid-19 data 1 . Through more in-depth and comprehensive statistical analysis, we conclude that there is no difference in the transmission rates of coronavirus between the symptomatic and asymptomatic patients, which is consistent with the original findings in Chen et al. 1 '},\n"," {'paper_id': '74b00f19c3af87d1081644f02490ba250f57b7ca',\n","  'section': 'Abstract',\n","  'text': 'Background: New endemic disease has been spread across Wuhan City, China on December 2019. Within few weeks, the World Health Organization (WHO) announced a novel coronavirus designated as coronavirus disease 2019 . In late January 2020, WHO declared the outbreak of a \"public-health emergency of international concern\" due to the rapid and increasing spread of the disease worldwide. Currently, there is no vaccine or approved treatment for this emerging infection; thus the objective of this study is to design a multi epitope peptide vaccine against COVID-19 using immunoinformatics approach. Method: Several techniques facilitating the combination of immunoinformatics approach and comparative genomic approach were used in order to determine the potential peptides for designing the T cell epitopes-based peptide vaccine using the envelope protein of 2019-nCoV as a target. Results: Extensive mutations, insertion and deletion were discovered with comparative sequencing in COVID-19 strain. Additionally, ten peptides binding to MHC class I and MHC class II were found to be promising candidates for vaccine design with adequate world population coverage of 88.5% and 99.99%, respectively. Conclusion: T cell epitopes-based peptide vaccine was designed for COVID-19 using envelope protein as an immunogenic target. Nevertheless, the proposed vaccine is rapidly needed to be validated clinically in order to ensure its safety, immunogenic profile and to help on stopping this epidemic before it leads to devastating global outbreaks. '},\n"," {'paper_id': '74bee8bf3229f4c28991a7231c02d911d24770e9',\n","  'section': 'Abstract',\n","  'text': 'The World Health Organization declared that COVID-19 outbreak constituted a Public Health Emergency of International Concern and the development of reliable laboratory diagnosis of SARS-CoV-2 became mandatory to identify, isolate and provide optimized care for patients early. RT-qPCR testing of respiratory secretions is routinely used to detect causative viruses in acute respiratory infection. RT-qPCR in-house protocols to detect the SARS-CoV-2 have been described. Validations of these protocols are considered a key knowledge gap for COVID-19, especially if executed in a high throughput format. Here, we investigate the analytical sensitivity and specificity of two interim RT-qPCR protocols for the qualitative detection of SARS-CoV-2 executed in a fully automated platform. Under our conditions, the N1 and RdRP (modified) showed the highest analytical sensitivity for their RNA targets. E assay, in its original concentration, was considered a tertiary confirmatory assay. Taken together, N1, RdRP (optimized) and E presented appropriated analytical sensibility and specificity in our automated RT-qPCR workflow for COVID-19 virus, E being at least 4-fold less sensitive than the others. This study highlights the importance of local validation of in-house assays before its availability to the population. The use of the synthetic RT-qPCR target to investigate novel assays diagnostic parameters in automated workflows is a quick, simple effective way to be prepared for upcoming threats. The proposed assay detected the fisrt SARS-CoV-2 infection in Brazilian Central-West. . CC-BY 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': '74fcbece5dd416519a5334b9cc2584cd11bcda5d',\n","  'section': 'Abstract',\n","  'text': 'Background: United States government scientists estimate that COVID-19 may kill between 100,000 and 240,000 Americans. The majority of the pre-existing conditions that increase the risk of death for COVID-19 are the same diseases that are affected by long-term exposure to air pollution. We investigate whether long-term average exposure to fine particulate matter (PM 2.5 ) increases the risk of COVID-19 deaths in the United States. Methods: Data was collected for approximately 3,000 counties in the United States (98% of the population) up to April 04, 2020. We fit zero-inflated negative binomial mixed models using county level COVID-19 deaths as the outcome and county level long-term average of PM 2.5 as the exposure. We adjust by population size, hospital beds, number of individuals tested, weather, and socioeconomic and behavioral variables including, but not limited to obesity and smoking. We include a random intercept by state to account for potential correlation in counties within the same state. We found that an increase of only 1 ߤ g/m 3 in PM 2.5 is associated with a 15% increase in the COVID-19 death rate, 95% confidence interval (CI) (5%, 25%). Results are statistically significant and robust to secondary and sensitivity analyses. Conclusions: A small increase in long-term exposure to PM 2.5 leads to a large increase in COVID-19 death rate, with the magnitude of increase 20 times that observed for PM 2.5 and allcause mortality. The study results underscore the importance of continuing to enforce existing air pollution regulations to protect human health both during and after the COVID-19 crisis. The data and code are publicly available. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '75216c0238ddc17dec8ef66adcc84bd48922cae4',\n","  'section': 'Abstract',\n","  'text': 'Precise estimation of genetic substitution patterns is critical for accurate reconstruction of 18 pathogen phylogenies. Few studies of viral evolution account for variations of mutation rate 19 across a single gene. This is especially true when considering evolution of segmented viruses 20 where individual segments are short, encoding for few proteins. However, the structural and 21 functional partitions of these proteins could provide valuable information for more accurate 22 38 All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '754315299d847600d6c5d414665c728d40bf731d',\n","  'section': 'Abstract',\n","  'text': 'A novel coronavirus pneumonia initially identified in Wuhan, China and provisionally named 2019-nCoV has surged in the public. In anticipation of substantial burdens on healthcare system following this human-to-human spread, we aim to scrutinise the currently available information and evaluate the burden of healthcare systems during this outbreak in Wuhan. We applied a modified SIR model to project the actual number of infected cases and the specific burdens on isolation wards and intensive care units (ICU), given the scenarios of different diagnosis rates as well as different public health intervention efficacy. Our estimates suggest, assuming 50% diagnosis rate if no public health interventions were implemented, that the actual number of infected cases could be much higher than the reported, with estimated 88,075 cases (as of 31 st January, 2020), and projected burdens on isolation wards and ICU would be 34,786 and 9,346 respectively The estimated burdens on healthcare system could be largely reduced if at least 70% efficacy of public health intervention is achieved. The health system burdens arising from the actual number of cases infected by the novel coronavirus appear to be considerable if no effective public health interventions were implemented. This calls for continuation of implemented anti-transmission measures (e.g., closure of schools and facilities, suspension of public transport, lockdown of city) and further effective large-scale interventions spanning all subgroups of populations (e.g., universal : bioRxiv preprint -2 -facemask wear) aiming at obtaining overall efficacy with at least 70% to ensure the functioning of and to avoid the breakdown of health system. '},\n"," {'paper_id': '75e68869f9b65bca661e768402395d9dede6de2c',\n","  'section': 'Abstract',\n","  'text': 'and more than 30 countries over last two months. COVID-19 has multiple characteristics distinct from other infectious diseases, including high infectivity during incubation, time delay between real dynamics and daily observed number of confirmed cases, and the intervention effects of implemented quarantine and control measures. We develop a Susceptible, Un-quanrantined infected, Quarantined infected, Confirmed infected (SUQC) model to characterize the dynamics of COVID-19 and explicitly parameterize the intervention effects of control measures, which is more suitable for analysis than other existing epidemic models. '},\n"," {'paper_id': '7607a4ad84452e29998a80f44bfd6bb2f5f68a7f',\n","  'section': 'Abstract',\n","  'text': 'Since December 2019, the outbreak of Coronavirus Disease 2019 (COVID-19) spread from Wuhan, China to the world, it has caused more than 87,000 diagnosed cases and more than 3,000 deaths globally. To fight against COVID-19, we carried out research for the near native SARS-CoV-2 and report here our preliminary results obtained. The pathogen of the COVID-19, the native SARS-CoV-2, was isolated, amplified and purified in a BSL-3 laboratory. The whole viral architecture of SARS-CoV-2 was examined by transmission electron microscopy (both negative staining and cryo-EM). We observed that the virion particles are roughly spherical or moderately pleiomorphic. Spikes have nail-like shape towards outside with a long body embedded in the envelope. CoV-2 is in post-fusion state, with S1 disassociated. This state revealed by cryo-EM first time could provide an important information for the identification and relevant clinical research of this new coronavirus. '},\n"," {'paper_id': '764849aa6da9b0adbdd86b017f9ee1beaf0aee6b',\n","  'section': 'Abstract',\n","  'text': 'now have more than 100,000 confirmed cases worldwide. Diagnosis of COVID-19 is currently performed by RT-qPCR methods, but the capacity of RT-qPCR methods is limited by its requirement of high-level facilities and instruments. Here, we developed and evaluated RT-LAMP assays to detect genomic RNA of SARS-CoV-2, the causative virus of COVID-19. RT-LAMP assays in this study can detect as low as 100 copies of SARS-CoV-2 RNA. Cross-reactivity of RT-LAMP assays to other human Coronaviruses was not observed. We also adapted a colorimetric detection method for our RT-LAMP assay so that the tests potentially performed in higher throughput. '},\n"," {'paper_id': '7677310b4e43cfaff6fd4ad53636fa80de589fd2',\n","  'section': 'Abstract',\n","  'text': 'The rapid expansion of coronavirus has been observed in many parts of the world. Many newly reported cases of this new coronavirus during early outbreak phases have been associated with travel history from an epidemic region (identified as imported cases). For those cases without travel history, the risk of wider spreads through community contact is even higher. However, most population models assume a homogeneous infected population without considering that the imported and secondary cases contracted by the imported cases can pose a different risk to community spread. We have developed an \"easy-to-use\" mathematical framework extending from a meta-population model embedding city-to-city connections to stratify the dynamics of transmission waves caused by imported, secondary, and others from an outbreak source region when control measures are considered. Using the dynamics of the secondary cases, we are able to determine the probability of community spread. Using the top 10 visiting cities from Wuhan in China as an example, we first demonstrated that the arrival time and the dynamics of the outbreaks at these cities can be successfully predicted under the reproductive number R 0 = 2.92 and latent period τ = 5.2 days. Next, we showed that although control measures can gain extra 32.5 and 44.0 days in arrival time through a high intensive border control measure and a shorter time to quarantine under a low R 0 (1.4), if the R 0 is higher (2.92), only 10 extra days can be gained for each of the same measures. This suggests the importance of lowering the incidence at source regions together with infectious disease control measures in susceptible regions. The study allows us to assess the effects of border control and quarantine measures on the emergence and the global spread in a fully connected world using the dynamics of the secondary cases. 2 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': '7698ec48a89a682cb52c066e293567fa3e83b7dd',\n","  'section': 'Abstract',\n","  'text': 'The likelihood of cross-species transmission is related to phenotypic plasticity of a virus and its capacity to produce genetically variable progeny. Herein a model is described connecting the production of genetic variability with increasing genome size. Comparing all known zoonotic viral genome sizes to known non-zoonotic viral genome sizes demonstrates that zoonotic viruses have significantly larger genomes. These results support the notion that large viral genomes are important in producing new zoonotic disease, and suggest that genome size may be a useful surrogate in screening for potential zoonotic viruses. '},\n"," {'paper_id': '77a04c5091c0457ad305691d4d4f24354d6bae79',\n","  'section': 'Abstract',\n","  'text': 'Recent history is punctuated by the emergence of highly pathogenic coronaviruses such as SARS-and MERS-CoV into human circulation. Upon infecting host cells, coronaviruses assemble a multi-subunit RNA-synthesis complex of viral non-structural proteins (NSP) responsible for the replication and transcription of the viral genome. Here, we present the 3.1 Å resolution structure of the SARS-CoV NSP12 polymerase bound to its essential co-factors, NSP7 and NSP8, using single particle cryo-electron microscopy. NSP12 possesses an architecture common to all viral polymerases as well as a large N-terminal extension containing a kinase-like fold and is unexpectedly bound by two NSP8 co-factors. This structure illuminates the assembly of the coronavirus core RNA-synthesis machinery, provides key insights into NSP12 polymerase catalysis and fidelity and acts as a template for the design of novel antiviral therapeutics. '},\n"," {'paper_id': '77edcf4facb96d75ec81182ed64227647d2ef4e3',\n","  'section': 'Abstract',\n","  'text': 'After the spread of SARS-CoV-2 epidemic out of China, evolution in the pandemic worldwide shows dramatic differences among countries. In Europe, the situation of Italy first and later Spain has generated great concern, and despite other countries show better prospects, large uncertainties yet remain on the future evolution and the efficacy of containment, mitigation or attack strategies. Here we applied a modified SEIR compartmental model accounting for the spread of infection during the latent period, in which we also incorporate effects of varying proportions of containment. We fit data to quarantined populations in order to account for the uncertainties in case reporting and study the scenario projections for the 17 individual regions (CCAA). Results indicate that with data for March 23, the epidemics follows an evolution similar to the isolation of 1, 5 percent of the population and if there were no effects of intervention actions it might reach a maximum over 1.4M infected around April27. The effect on the epidemics of the ongoing partial confinement measures is yet unknown, but increasing the isolation around ten times more could drastically reduce the peak to over 100k cases by early April, while each day of delay in taking this hard containment scenario represents an 90 percent increase of the infected population at the peak. Dynamics at the sub aggregated levels of CCAA show epidemics at the different levels of progression with the most worrying situation in Madrid an Catalonia. Increasing alpha values up to 10 times, in addition to a drastic reduction in clinical cases, would also more than halve the number of deaths. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '77efa10bbf0d22795315f61ace2dc95804d269fb',\n","  'section': 'Abstract',\n","  'text': 'Background-Patients with the 2019 coronavirus disease have different degrees of psychological pain, such as anxiety and depression, which may related to their prognosis. Psychological intervention can be conducted in different ways to improve psychological pain and improve the treatment effect. Objective-The present study aimed to investigate and analyze the psychological status of patients with COVID-19 during the course of illness, and to evaluate the effect and influencing factors of psychological crisis intervention, so as to explore the effective mode of clinical psychological intervention in acute patients under isolation environment. Methods-A total of 143 persons participated in the study, including 26 patients diagnosed with COVID-19 in the isolation ward (COVID-19 group), 87 patients with general pneumonia in the observation ward (General Pneumonia group) and 30 healthy volunteers (Normal group). All the patients in the ward received comprehensive psychological intervention, including telephone psychological counseling (active and passive), self-adjustment of written materials and one-to-one psychological crisis intervention. Hamilton depression scale (HAMD) and Hamilton anxiety scale (HAMA) were used to evaluate the mental health status of all patients on the day of admission and 1 week after treatment. Results-The scores of HAMA and HAMD of all patients (including isolation ward and observation ward) were significantly higher than the healthy volunteers at the time of admission. The total score of HAMA and HAMD in CVOID-19 group were both higher than that General Pneumonia group. After 1 week`s hospitalization with comprehensive psychological intervention, the scores of HAMA and HAMD in CVOID-19 group were significantly decreased. Conclusion-Patients those who diagnosed with COVID-19 in the isolation ward and/or general pneumonia in observation ward have different degrees of anxiety, depression and sleep problems. While receiving antiviral treatment, patients also need psychological intervention. Comprehensive psychological intervention model has been proved to be effective. '},\n"," {'paper_id': '7808e4cd470a379b12470833f4a6fc7e1b567c96',\n","  'section': 'Abstract',\n","  'text': 'Recent outbreaks of African swine fever virus (ASFV) in China severely 17 influenced the swine industry of the country. Currently, there is no '},\n"," {'paper_id': '781952b55b46350bc31c782515c03743e6095b9a',\n","  'section': 'Abstract',\n","  'text': 'Governments around the world are implementing population-wide physical distancing measures in an effort to control transmission of COVID-19, but metrics to evaluate their effectiveness are not readily available. We used a publicly available mobility index based on the relative frequency of trips planned in a popular transit application to evaluate the effect of physical distancing on infection growth rates and reproductive number in 34 states and countries. We found that a 10% decrease in relative mobility in the 2 nd week of March was associated with a 11.8% relative decrease (exp(β) = 0.882; 95% CI: 0.822, 0.946) in the average daily growth rate in the fourth week of March and a change in the instantaneous reproductive number of -0.054 (95% CI: 0.097, -0.011) in the same period. Our analysis demonstrates that decreases in urban mobility were predictive of declines in epidemic growth at national or subnational scales. Mobility metrics offer an appealing method to calibrate population-level physical distancing policy and implementation. '},\n"," {'paper_id': '7837a1b529936ac78b802e41daa73ddab8bfc41a',\n","  'section': 'Abstract',\n","  'text': 'Viruses maximize their genetic coding capacity through a variety of biochemical mechanisms including programmed ribosomal frameshifting (PRF), which facilitates the production of multiple proteins from a single transcript. PRF is typically stimulated by structural elements within the mRNA that generate mechanical tension between the transcript and ribosome. However, in this work we show that the forces generated by the cotranslational folding of the nascent polypeptide chain can also enhance PRF. Using an array of biochemical, cellular, and computational techniques, we first demonstrate that the Sindbis virus structural polyprotein forms two competing topological isomers during biosynthesis at the ribosome-translocon complex. We then show that the formation of one of these topological isomers is linked to PRF. Coarse-grained molecular dynamic simulations reveal that the translocon-mediated membrane integration of a transmembrane domain upstream from the ribosomal slip-site generates a force on the nascent polypeptide chain that scales with observed frameshifting. Together, our results demonstrate that cotranslational folding of this protein generates a tension that stimulates PRF. To our knowledge, this constitutes the first example in which the conformational state of the nascent chain has been linked to PRF. These findings raise the possibility that, in addition to RNA-mediated translational recoding, a variety of cotranslational folding and/ or binding events may also stimulate PRF. '},\n"," {'paper_id': '7852aafdfb9e59e6af78a47af796325434f8922a',\n","  'section': 'Abstract',\n","  'text': 'Although the SARS-CoV-2 viral load detection of respiratory specimen has been widely used for novel coronavirus disease diagnosis, it is undeniable that serum SARS-CoV-2 nucleic acid (RNAaemia) could be detected in a fraction of the COVID-19 patients. However, it is not clear that if the incidence of RNAaemia could be correlated with the occurrence of cytokine storm or with the specific class of patients. This study enrolled 48 patients with COVID-19 admitted to the General Hospital of Central Theater Command, PLA, a designated hospital in Wuhan, China. The patients were divided into three groups according to the Dia gnosis and Treatment of New Coronavirus Pneumonia (version 6) published by the National Health Commission of China. The clinical and laboratory data were collected. The serum viral load detection and serum IL-6 levels were determined. Except for routine statistical analysis, Generalized Linear Models (GLMs) analysis was used to establish a patient status prediction model based on real-time RT-PCR Ct value. The Result showed that cases with RNAaemia were exclusively confirmed in critically ill patients group and appeared to reflect the illness severity. Further more, the inflammatory cytokine IL-6 levels were significantly elevated in critically ill patients, which is almost 10-folds higher than those in other patients. More importantly, the extremely high IL-6 level was closely correlated with the incidence of RNAaemia (R=0.902) and the vital signs of COVID-19 patients (R= -0.682). Serum SARS-CoV-2 viral load (RNAaemia) is strongly associated with cytokine storm and can be used to predict the poor prognosis of COVID-19 patients. Moreover, our results strongly suggest that cytokine IL-6 should be considered as a therapeutic target in critically ill patients with excessive inflammatory response. All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '78ab35770cc98632c434523df48c6ba290182d37',\n","  'section': 'Abstract',\n","  'text': 'Several French regions where COVID19 has been reported currently show a renewed increase in ILI cases in the general practice based Sentinelles network. Here we computed the number of excess cases by region and found correlation with the number of reported COVID19 cases so far. These data suggest larger circulation of SARS-CoV-2 in the French population than apparent from confirmed cases. '},\n"," {'paper_id': '78ab74f8133c0dfe8fb414c216350dd37f69b0bf',\n","  'section': 'Abstract',\n","  'text': 'Animal relocation data has recently become considerably more ubiquitous, finely structured (collection frequencies measured in minutes) and co-variate rich (physiology of individuals, environmental and landscape information, and accelerometer data). To better understand the impacts of ecological interactions, individual movement and disease on global change ecology, including wildlife management and conservation, it is important to have simulators that will provide demographic, movement, and epidemiology null models against which to compare patterns observed in empirical systems. Such models may then be used to develop quantitative narratives that enhance our intuition and understanding of the relationship between population structure and generative processes: in essence, along with empirical and experimental narratives, quantitative narratives are used to advance ecological epistemology. Here we describe a simulator that accounts for the influence of consumer-resource interactions, existence of social groups anchored around a central location, territoriality, group-switching behavior, and disease dynamics on population size. We use this simulator to develop new and reinforce existing quantitative narratives and point out areas for future study. The health and viability of species are of considerable concern to all nature lovers. Population models are central to our efforts to assess the numerical and ecological status of species and threats posed by climate change. Models, however, are crude caricatures of complex ecological systems. So how do we construct reliable assessment models able to capture processes essential to predicating the impacts of global change on population viability without getting tied up in their vast complexities? We broach this question and demonstrate how models focusing at the level of the individual (i.e., agent-based models) are tools for developing robust, narratives to augment narratives arising purely from empirical data sources and experimental outcomes. We do this in the context of nesting social groups, foraging for food, while exhibiting territoriality and group-switching behavior; and, we evaluate the impact of disease on the viability of such populations. 7 laboratory at the microcosm scale [7], but for populations of most vertebrates such 8 microcosm studies are not applicable, and we need to resort to the mesocosm level [8] or 9 use alternative methods to obtain insights. One of these methods is to build simulation 10 models that incorporate demographic, spatial, movement, and other relevant behavioral 11 and community processes, and then compare simulations of these models across various 12 scenarios [9] or outcomes with and without epidemiological processes added to the mix. 13 We take this latter approach and our primary aim is to develop a tool that can be used 14 to study the impact of disease on colonial populations and metapopulations [10] . This 15 tool must be comprehensive enough to include critical processes needed to address 16 questions that will extend our current knowledge in significant ways. Its application, of 17 course, is limited by the structure of the model. Thus, in building the model, we are 18 guided by principles related to concepts of \"appropriate complexity modeling\" [11, 12] . The standard approach to modeling disease outbreaks caused by 20 directly-transmitted pathogens is through systems of differential or difference equations 21 that model the number or density of susceptible (S), infectious (I), and recovered or 22 removed (R) individuals [13, 14]. These formulations may either be deterministic or 23 stochastic, and elaborated to include time-delays [15], demographic (births, deaths, and 24 recruitment [16]), metapopulation [17] and other spatial processes [18]. Examples of 25 these elaborations can be found in the context of modeling, inter alia, influenza [19, 20], 26 measles [21], SARS [22, 23], and Ebola [24-28] outbreaks. 27 Systems approaches to modeling disease limit our ability to consider individual-level 28 traits, unless these traits can be reasonably well expressed by dividing the population 29 into a limited number of subgroups, such as groups of individuals infected with different 30 strains of pathogens [29]. Including individual-level variation in disease models, however, 31 requires an agent-based approach [30], which then allows both individual behavior (e.g., 32 relevant to evaluating the probability of pathogen transmission) and pathogen exposure 33 history (e.g., relevant to evaluating the immunological state and hence the susceptibility 34 of an individual) to be incorporated into the model. It also allows one to trace the 35 number of individuals the each pathogen infects [25] and, hence, to identify individuals 36 that may be acting as super-spreaders [31]. Here, we develop an agent-based formulation that has application to studying 38 disease outbreaks in wildlife populations whose state of health-and hence, 39 susceptibility-is impacted by various factors, including social factors related to group 40 living and the structure of social networks [32], ecological factors related to foraging 41 success, and the movement behavior of individuals (which influences exposure to 42 pathogens). The model also includes a periodic return to a colony or nest cell, which 43 implies that the model has both a population-level and a group-association level (core 44 or household) structure. Such models have applications in animal production systems 45 (e.g., managing foot and mouth disease [33]), wildlife conservation [2], and disease 46 management at the human-animal interface [34]. The latter involves diseases such as 47 anthrax [35], bovine tuberculosis [36], and brucellosis in cattle and wild ungulates [37], 48 chronic wasting disease in deer [38], and toxoplasmosis in rats, cats and humans [39]. 49 Rather then following a formulaic ODD (Overview, Design concepts, and Design 50 details; [30, 40]) or extended ODD [41] prescription in presenting our model, we take a 51 more expository approach by first crafting a broad outline of our model\\'s general 52 October 1, 2019 2/28 All rights reserved. No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/791400 doi: bioRxiv preprint features. In addition, to keep the paper reader-friendly, we relegate the mathematical 53 details of our model to a Technical Methods section at the end of the paper and the 54 algorithmic details to a supporting online file (SOF). In the Results section, we report 55 the results of simulations of several illustrative scenarios, and in the Discussion section 56 we present the findings in the form of a quantitative narrative that provides insights 57 into the role different processes play in creating the population patterns observed in our 58 simulations. At this point, a few brief remarks on quantitative narratives in disease ecology are in 60 order. Unlike the canonical theories of physics-with kinematics, electromagnetism, 61 general relativity, and quantum mechanics providing prime examples, ecology has no 62 fundamental theories and associated mathematical formulations that can be used to 63 develop an ecological epistemology. Rather, the complexity of the field induced by the 64 plethora of processes involved requires that we focus only on a limited number of these 65 processes when building models, where the selection of process depends on the questions 66 at hand [11]. Our understanding of these processes requires an analysis of associated 67 quantitative formulations, but the soundness of the arguments cannot be adjudicated by 68 comparing, say, the accuracy of Mercury\\'s orbit as predicted by Newtonian versus 69 Einsteinian physics. Rather, ecological epistemology is woven from a collection of 70 quantitative narratives that provide current best explanations for, or fits to, observed 71 empirical and experimental data. While statistical null models are use to develop 72 consilient empirical and experimental narratives, dynamic null models are used to 73 develop quantitative narratives that are consilient with empirical and experimental 74 narratives (Fig. 1). Such dynamic null models have a Goldilocks quality in the sense of 75 containing just the right amount of biological process detail [42]: too little and model 76 simulations are rejected as providing adequate descriptions of observed data; too much 77 process detail and the null-model-parsimony principle, as discussed by Gotteli and 78 Graves ( [42], p. 6), is violated. 79 Fig 1. A sketch of the role of empirical, experimental and quantitative narratives in the formulation of canons and construction of syntheses in the development of an ecological epistemology. In ecology, quantitative narratives, derived from the Gause-Lotka-Volterra 80 competition model, were used to construct the concept of a niche [43]. This narrative 81 begins with insights into competitive exclusion [44, 45], supported by additional 82 quantitative ideas relating to niche breadth and overlap [46, 47]. In epidemiology, 83 quantitative narratives, derived from analyses of SEIR epidemic models (S: susceptible; 84 E: exposed but not yet infectious; I: infections; R: removed-that is, recovered with 85 immunity or dead) [13] , have been used to identify critical differences in the emergence 86 of outbreak thresholds depending on whether pathogen transmission is dependent on quantitative narratives in epidemiology that incorporate aspects of the movement of 89 individuals [50] or particular ecological processes have been used to develop canonical 90 theories [51][52][53][54][55]; however, these are limited in scope and generality by the analytical 91 difficulties of studying dynamical systems, as well as networks that include more than 92 one ecological or behavioral process. The challenge here is to keep the models somewhat 93 general and broadly applicable, while including several key processes such as: spatial 94 demography; consumer-resource ecology with associated reproduction, growth, stress, 95 and mortality processes; movement and contact behavior; and SEIR epidemiology, in 96 ways that are simple enough to interpret the effects of each process. Sensitivity analyses 97 provide one type of approach to separating out effects [56][57][58][59]. Another is to use 98 methods relating to the practice of appropriate complexity modeling [11, 12] . 2 Methods The model we present here only incorporates the focal processes of interest and those 101 needed to attain a level of realism required to adequately address the questions at hand. 102 Thus, for example, we do not include agent age structure in the model even though age 103 structure is known to have important effects on population growth patterns [60], 104 persistence [61], and epidemics [62]. Agent age structure would certainly add complexity 105 to our narratives; but, whether its impacts are critical (i.e., it qualitatively changes the 106 structures we explain) or only elaborative (i.e., has quantitative but not qualitative 107 implications regarding the questions asked) is a issue to be examined in future studies. 108 For clarity, we enumerate below-without mathematical details-the processes 109 incorporated in our model, leaving such details to the Technical Methods Section at the 110 end of the paper. Aspects of the Numerus Model Builder construction of the model and 111 the generated NovaScript (extended JavaScript language) implementation are provided 112 in a Supporting Information file. 113 2.1 Model Outline 114 The model is formulated as a set of agents (individuals) moving over a cellular array. The agents themselves can be differentiated with respect to type (e.g., sex), can undergo 116 discrete state transitions (e.g., disease state) and can also change continuously over time 117 with respect an agent-level state vector. In our case, the latter contains the variables 118 \"mass\" (also a surrogate for fitness) and \"stress\" (which accumulates when the individual 119 is unable to meet its resource needs each time period). The cells in the cellular array 120 may also have relatively complex states that, in our model, include a list of the agents 121 in each cell and the level of a dynamically responding resource, which determines the 122 environmental context for the agents. Movement of agents occurs under the assumption 123 that, apart from reproduction, ecological and epidemiological processes are independent 124 of agent type. More specifically, in our model, the following objects and concepts apply: 125 Simulation world. This is the set of agents moving over a hexagonal cellular array 126 (we use a toroidal topology to eliminate boundary effects). '},\n"," {'paper_id': '78b493ee7a821aca1579e7a85142af53e14d9fce',\n","  'section': 'Abstract',\n","  'text': 'Vascular leak is a key driver of organ injury in diseases such as Acute Respiratory Distress Syndrome caused by viruses, including COVID-19. Strategies that reduce enhanced permeability and vascular inflammation are promising therapeutic targets. Activation of the Angiopoietin-1 (Angpt1)-Tie2 tyrosine kinase signaling pathway is an important regulator of vascular quiescence. Here we describe the design and construction of a new soluble ANGPT1 mimetic that is a potent activator of endothelial Tie2 in vitro and in vivo. Using a chimeric fusion strategy, we replaced the extracellular matrix (ECM) binding and oligomerization domain of ANGPT1 with a heptameric scaffold derived from the C-terminus of serum complement protein C4-binding protein α (C4BP). We refer to this new fusion protein biologic as C4BP-ANG1, which forms a stable heptamer and induces TIE2 phosphorylation in cultured cells, and in the lung following i.v. injection of mice. Injection of C4BP-ANG1 ameliorates VEGF-and lipopolysaccharideinduced vascular leakage, in keeping with the known functions of Angpt1-Tie2 in maintaining quiescent vascular stability, and therefore is a promising candidate treatment for inflammatory endothelial dysfunction. author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '78e49fdb6f0aa9924a5b510341d52b618fff0ca6',\n","  'section': 'Abstract',\n","  'text': 'The respiratory syndrome caused by a new type of coronavirus has been emerging from 2 China and caused more than 1000 death globally since December 2019. This new virus, 3 called 2019 novel coronavirus (2019-nCoV) uses the same receptor called Angiotensin-4 converting enzyme 2 (ACE2) to attack humans as the coronavirus that caused the severe 5 acute respiratory syndrome (SARS) seventeen years ago. Both viruses recognize ACE2 6 through the spike proteins (S-protein) on their surfaces. It was found that the S-protein 7 from the SARS coronavirus (SARS-CoV) bind stronger to ACE2 than 2019-nCoV. 8 However, function of a bio-system is often under kinetic, rather than thermodynamic, 9 control. To address this issue, we constructed a structural model for complex formed 10 between ACE2 and the S-protein from 2019-nCoV, so that the rate of their association 11 can be estimated and compared with the binding of S-protein from SARS-CoV by a 12 multiscale simulation method. Our simulation results suggest that the association of new 13 virus to the receptor is slower than SARS, which is consistent with the experimental data 14 obtained very recently. We further integrated this difference of association rate between 15 virus and receptor into a mathematical model which describes the life cycle of virus in 16 host cells and its interplay with the innate immune system. Interestingly, we found that 17 the slower association between virus and receptor can result in longer incubation period, 18 while still maintaining a relatively higher level of viral concentration in human body. Our 19 computational study therefore provides, from the molecular level, one possible 20 explanation that the new disease by far spread much faster than SARS. 21 22 23 24 25 26 27 author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '793a82a423402d4c2badcc9b37f1b352ecc7e519',\n","  'section': 'Abstract',\n","  'text': 'Background: Some have claimed that the number of Covid-19 deaths is not much greater than would be experienced in the usual course of events. We sought to estimate the number of deaths due to Covid-19 in the United States and compare that number to the number of expected deaths in the same time period. Methods: To obtain the number of Covid-19 deaths, we multiplied published infection fatality rates by age by population numbers and assumed infection rates. The number of Covid-19 deaths was compared to the number of deaths using life table mortality rates. '},\n"," {'paper_id': '79622641e002b2574af9dc7f99d4fbbe192606e6',\n","  'section': 'Abstract',\n","  'text': '125 words). The SARS-CoV-2 pandemic is straining healthcare resources worldwide, prompting social distancing measures to reduce transmission intensity. The amount of social distancing needed to curb the SARS-CoV-2 epidemic in the context of seasonally varying transmission remains unclear. Using a mathematical model, we assessed that one-time interventions will be insufficient to maintain COVID-19 prevalence within the critical care capacity of the United States. Seasonal variation in transmission will facilitate epidemic control during the summer months but could lead to an intense resurgence in the autumn. Intermittent distancing measures can maintain control of the epidemic, but without other interventions, these measures may be necessary into 2022. Increasing critical care capacity could reduce the duration of the SARS-CoV-2 epidemic while ensuring that critically ill patients receive appropriate care. Full text. The COVID-19 pandemic is causing substantial mortality and a major strain on healthcare systems (1,2) . With no pharmaceutical treatments available, interventions have focused on contact tracing, quarantine, and social distancing. Intensive testing, tracing, and isolation of cases has enabled control of transmission in some places, such as Singapore and Hong Kong (3). At the opposite extreme, many countries lack the testing and public health resources to mount similar responses to the COVID-19 epidemic, which could result in unhindered spread and catastrophic outbreaks. Between these responses, many countries are adopting measures termed \"social distancing\" or \"physical distancing,\" closing schools and workplaces and limiting the sizes of gatherings. The goal of these strategies is to slow the spread of infection and reduce the intensity of the epidemic (\"flatten the curve\") (3), thus reducing risk of overwhelming health systems and buying time to develop treatments and vaccines. However, there is concern that such measures may have to stay in place over long . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': '798a9a453732be004bdbbcd01e872abe0e3af90f',\n","  'section': 'Abstract',\n","  'text': 'medRxiv preprint a regional and subregional granularity. We propose a model derived from well-known models in epidemic to estimate the needed number of places in intensive care units. The model will help decision-makers to plan resources in the short and medium-term in order to guarantee appropriate treatments to all patients needing it. We analyse Italian data at regional level up to March 15th aiming to: (i) support health and government decision-makers in gathering rapid and efficient decisions on increasing health structures capacities (in terms of ICU slots) and (ii) define a scalable geographic model to plan emergency and future COVID-19 patients management using reallocating them among health structures. Finally, the here proposed model can be useful in countries where COVID-19 is not yet strongly diffused. '},\n"," {'paper_id': '79fe6bb36a69589bc0969cdb5d3682cff685811d',\n","  'section': 'Abstract',\n","  'text': 'The response to the COVID19 epidemic is generating severe shortages of personal protective equipment around the world. In particular, the supply of N95 respirator masks has become severely depleted with supplies having to be rationed and health care workers having to use masks for prolonged periods in many countries. We sought to test the ability of 4 different decontamination methods including autoclave treatment, ethylene oxide gassing, ionized hydrogen peroxide fogging and vaporized hydrogen peroxide exposure to decontaminate 4 different N95 masks of experimental contamination with SARS-CoV-2 or vesicular stomatitis virus as a surrogate. In addition, we sought to determine whether masks would tolerate repeated cycles of decontamination while maintaining structural and functional integrity. We found that one cycle of treatment with all modalities was effective in decontamination and was associated with no structural or functional deterioration. Vaporized hydrogen peroxide treatment was tolerated to at least 5 cycles by masks. Most notably, standard autoclave treatment was associated with no loss of structural or functional integrity to a minimum of 10 cycles for the 3 pleated mask models. The molded N95 mask however tolerated only 1 cycle. This last finding may be of particular use to institutions globally due to the virtually universal accessibility of autoclaves in health care settings. The COVID19 pandemic is proving to be an exceptional stress on hospital and health systems resources around the world. Many countries are experiencing or imminently expecting shortages for a variety of equipment and disposable supplies. A tightening supply of N95 masks that allow for protection from airborne pathogens and aerosolized viruses including SARS-CoV-2 is of particular and immediate concern. Without an adequate supply of N95 masks, health care providers are at extreme risk of acquisition of COVID19 disease. The occurrence of patient to health care workers spread of SARS-CoV-2 at sufficiently high rates would lead to demoralization of the workforce, depletion of health care workers for quarantine and turn hospitals into extreme hotspots for infection transmission. '},\n"," {'paper_id': '7a17b9b40b960c4b35b633afc63c27bb0d0e4d52',\n","  'section': 'Abstract',\n","  'text': 'A discrete-time deterministic epidemic model is proposed with the aim of reproducing the behaviour observed in the incidence of real infectious diseases. For this purpose, we analyse a SIRS model under the framework of a small world network formulation. Using this model, we make predictions about the peak of the Covid-19 epidemic in Italy. A Gaussian fit is also performed, to make a similar prediction. . CC-BY 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint . '},\n"," {'paper_id': '7a4b1cbed36a78dd075a57d15e8bd672bfec33ea',\n","  'section': 'Abstract',\n","  'text': 'We present here the design of the Mechanical Ventilator Milano (MVM), a novel mechanical ventilator designed for mass scale production in response to the COVID-19 pandemics, to compensate for the dramatic shortage of such ventilators in many countries. This ventilator is an electro-mechanical equivalent of the old, reliable Manley Ventilator. Our design is optimized to permit large sale production in short time and at a limited cost, relying on off-the-shelf components, readily available worldwide from hardware suppliers. Operation of the MVM requires only a source of compressed oxygen (or compressed medical air) and electrical power. The MVM control and monitoring unit can be connected and networked via WiFi so that no additional electrical connections are necessary other than the connection to the electrical power. At this stage the MVM is not a certified medical device. Construction of the first prototypes is starting with a team of engineers, scientists and computing experts. The purpose of this paper is to disseminate the conceptual design of the MVM broadly and to solicit feedback from the scientific and medical community to speed the process of review, improvement and possible implementation. '},\n"," {'paper_id': '7a51b191fd6575831585eee67a0ac67f209d8fcb',\n","  'section': 'Abstract',\n","  'text': '15 '},\n"," {'paper_id': '7a5a8e3436a3ddc960c07cc769fc951551fda68b',\n","  'section': 'Abstract',\n","  'text': 'Non-pharmaceutical interventions have been implemented to reduce transmission of SARS-CoV-2 in the UK. Projecting the size of an unmitigated epidemic and the potential effect of different control measures has been critical to support evidence-based policymaking during the early stages of the epidemic. We used a stochastic age-structured transmission model to explore a range of intervention scenarios, including the introduction of school closures, social distancing, shielding of elderly groups, self-isolation of symptomatic cases, and extreme \"lockdown\"-type restrictions. We simulated different durations of interventions and triggers for introduction, as well as combinations of interventions. For each scenario, we projected estimated new cases over time, patients requiring inpatient and critical care (intensive care unit, ICU) treatment, and deaths. We found that mitigation measures aimed at reducing transmission would likely have decreased the reproduction number, but not sufficiently to prevent ICU demand from exceeding NHS availability. To keep ICU bed demand below capacity in the model, more extreme restrictions were necessary. In a scenario where \"lockdown\"-type interventions were put in place to reduce transmission, these interventions would need to be in place for a large proportion of the coming year in order to prevent healthcare demand exceeding availability. The characteristics of SARS-CoV-2 mean that extreme measures are likely required to bring the epidemic under control and to prevent very large numbers of deaths and an excess of demand on hospital beds, especially those in ICUs. As countries have moved from early containment efforts to planning for the introduction of large-scale non-pharmaceutical interventions to control COVID-19 outbreaks, epidemic modelling studies have explored the potential for extensive social distancing measures to curb transmission. However, it remains unclear how different combinations of interventions, timings, and triggers for the introduction and lifting of control measures may affect the impact of the epidemic on health services, and what the range of uncertainty associated with these estimates would be. Using a stochastic, age-structured epidemic model, we explored how eight different intervention scenarios could influence the number of new cases and deaths, as well as intensive care beds required over the projected course of the epidemic. We also assessed the potential impact of local versus national targeting of interventions, reduction in leisure events, impact of increased childcare by grandparents, and timing of triggers for different control measures. We simulated multiple realisations for each scenario to reflect uncertainty in possible epidemic trajectories. Our results support early modelling findings, and subsequent empirical observations, that in the absence of control measures, a COVID-19 epidemic could quickly overwhelm a healthcare system. We found that even a combination of moderate interventions -such as school closures, shielding of older groups and self-isolation -would be unlikely to prevent an epidemic that would far exceed available ICU capacity in the UK. Intermittent periods of more intensive lockdown-type measures are predicted to be effective for preventing the healthcare system from being overwhelmed. 3 . CC-BY 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint . We collated multiple sources of evidence to estimate key model parameters ( Table S1 ). In a meta-analysis, we estimated that the basic reproduction number, R 0 -which describes the average number of secondary infections caused by a typical primary infection in a completely susceptible population-was 2.7 (95% credible interval: 1.6-3.9) across settings without substantial control measures in place. We derived age-stratified case fatality ratios (CFR) to estimate a CFR that ranged substantially across age groups, from 0.1% in the 20-29 age group to 7.7% in the over-80 age group. Using these values along with the relationship between CFR and severe and critical cases, we also estimated the proportion of 4 . CC-BY 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': '7aa49aa2b76ea92b9265ad9d5a9d9ef0ea585b24',\n","  'section': 'Abstract',\n","  'text': '17 18 Background: Community responses are important for outbreak management during the early phase 19 when non-pharmaceutical interventions are the major preventive options. Therefore, this study aims to 20 examine the psychological and behavioral responses of the community during the early phase of the 21 COVID-19 epidemic in Hong Kong. 22 23 Method: A cross-sectional online survey was launched within 36 hours after confirmed COVID-19 24 cases were first reported. Councilors of all 452 district council constituency areas were approached for 25 survey dissemination. Respondent demographics, anxiety level, risk perception, sources to retrieve 26 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10. 1101 /2020 COVID-19 information, actual adoption and perceived efficacy of precautionary measures were 27 collected. Result: Analysis from 1715 complete responses indicated high perceived susceptibility (89%) and high 30 perceived severity (97%). Most respondents were worried about COVID-19 (97%), and had their daily 31 routines disrupted (slightly/greatly: 98%). The anxiety level, measured by the Hospital Anxiety and 32 Depression Scale, was borderline abnormal (9.01). Nearly all respondents were alert to the disease 33 progression (99.5%). The most trusted information sources were doctors (84%), followed by broadcast 34 (57%) and newspaper (54%), but they were not common information sources (doctor: 5%; broadcast: 35 34%; newspaper: 40%). Only 16% respondents found official websites reliable. Enhanced personal 36 hygiene practices and travel avoidance to China were frequently adopted (>77%) and considered 37 effective (>90%). The adoption of social-distancing measures was lower (39%-88%), and their drivers 38 for greater adoption include: being female (adjusted odds ratio [aOR]:1.27), living in the New 39 Territories (aOR:1.32-1.55), perceived as having good understanding of COVID-19 (aOR:1.84) and 40 being more anxious (aOR:1.07). 41 42 Discussion: Risk perception towards COVID-19 in the community was high. Most respondents are alert 43 to the disease progression, and adopt self-protective measures. This study contributes by examining the 44 psycho-behavioral responses of hosts, in addition to the largely studied mechanistic aspects, during the 45 early phase of the current COVID-19 epidemic. The timely psychological and behavioral assessment of 46 the community is useful to inform subsequent interventions and risk communication strategies as the 47 epidemic progresses. 48 49 '},\n"," {'paper_id': '7ab9f9fcea519ebce527c3ede8091beedbb26ad9',\n","  'section': 'Abstract',\n","  'text': 'The 2019 novel coronavirus (2019-nCoV) is currently causing a widespread outbreak centered on Hubei province, China and is a major public health concern. Taxonomically 2019-nCoV is closely related to SARS-CoV and SARS-related bat coronaviruses, and it appears to share a common receptor with SARS-CoV (ACE-2). Here, we perform structural modeling of the 2019-nCoV spike glycoprotein. Our data provide support for the similar receptor utilization between 2019-nCoV and SARS-CoV, despite a relatively low amino acid similarity in the receptor binding module. Compared to SARS-CoV, we identify an extended structural loop containing basic amino acids at the interface of the receptor binding (S1) and fusion (S2) domains, which we predict to be author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '7adf41d2120f3398e13817367c2120ebac6b99b6',\n","  'section': 'Abstract',\n","  'text': 'Background: Although the COVID-19 has been well controlled in China, it is rapidly spreading outside China that may lead to catastrophic results globally without implementation of necessary mitigation measures. Because the COVID-19 outbreak has made comprehensive and profound impacts on the world, an accurate prediction of its epidemic trend is significant. Although many studies have predicted the COVID-19 epidemic trend, most of these studies have used the early stage data and focused on the China cases. We predicted the COVID-19 epidemic trend in China and across the world using the machine learning approach. We first built the models for predicting the daily numbers of cumulative confirmed cases (CCCs), new cases (NCs), and death cases (DCs) of COVID-19 in China based on the data from '},\n"," {'paper_id': '7af6a15c35df8fed989119cc36b6679175d2a768',\n","  'section': 'Abstract',\n","  'text': '1 2 Effects of a blend of essential oils in milk replacer on performance, rumen fermentation, 3 blood parameters and health scores of dairy heifers 4 5 Abstract 23 The objective of this study was to evaluate how the inclusion of a blend of essential oils 24 in milk replacer (MR) affects different outcomes of dairy heifers. The outcomes 25 evaluated: feed intake, performance, body development, blood cells and metabolites, 26 insulin-like growth factor-1 (IGF-1), rumen fermentation, fecal scores and respiratory . CC-BY 4.0 : bioRxiv preprint 2 27 scores. All outcomes were evaluated during pre-weaning (4 -60 d of age), and 28 carryover effects on post-weaning (61 -90 d of age) periods. The experimental units 29 utilized were 29 newborn Holstein × Gyr crossbred dairy heifers, with genetic 30 composition of 5/8 or more Holstein and 3/8 or less Gyr and body weight (BW) at birth 31 of 32.2 ± 5.2 kg. Experimental units were randomly assigned to either a control group 32 (CON, n = 15) or a treatment group, consisting of supplementation of a blend of 33 essential oils (BEO, n = 14) with1 g/d/calf (Apex Calf, Adisseo, China). During the pre-34 weaning phase, all heifers were fed 5 L of MR/d reconstituted at 15% (dry matter basis), 35 divided into two equal meals. Water and starter were provided ad libitum. During the 36 post-weaning, animals received a maximum 3 kg of starter/d, and ad libitum corn silage, 37 divided into two meals. The outcomes feed intake, fecal and respiratory scores were 38 evaluated daily. BW was measured every three days, while body development was 39 recorded weekly. Blood samples were collected on 0, 30 and 60 d of age for total blood 40 cell count, weekly to determinate ß-hydroxybutyrate, urea and glucose, and biweekly 41 for IGF-1. Ruminal parameters (pH, volatile fatty acids, ammonia-N and 42 acetate:proprionate proportion -C2:C3) were measured each 14 days. A randomized 43 complete block design with an interaction between treatment and week was the 44 experimental method of choice to test the hypothesis of effect of BEO on all outcomes. 45 An ANOVA procedure was used for continuous outcomes and a non-parametric test 46 was used for the ordered categorical outcomes both adopting a C.I. = 95%. Results 47 indicated that there was not enough evidence to accept the alternative hypothesis of 48 effect of BEO in MR on: feed intake, performance, body development and blood 49 metabolites during both pre-weaning and post-weaning periods. However, results 50 indicated that the inclusion of BEO in MR significantly affects the proportion of C2:C3 51 during pre and post-weaning (P ≤ 0.05). Similarly, the effect is significant for basophil 52 (P ≤ 0.001) and platelet (P ≤ 0.05) counts in pre-weaning. The interaction between week . CC-BY 4.0 : bioRxiv preprint 3 53 and treatment was also significant for lymphocytes (P ≤ 0.001), revealing a long-term 54 treatment immunological effect. Lastly, the effect on fecal scores was also significant (P 55 ≤ 0.05) during pre-weaning, with lower values for BEO. BEO contributed for ruminal 56 manipulation on pre-weaning and carry over effect on post-weaning; immunity 57 improvement and a decrease morbidity of neonatal diarrhea in pre-weaning phase. 58 59 Introduction 60 A good calf-rearing program should embrace aspects that go from body 61 development, stress reduction, good feed and housing management to optimum health 62 status. Average daily gain (ADG) and body weight (BW) at weaning are important 63 metrics used to measure the success of the rearing program. It is well known that these 64 parameters are related to the heifer\\'s future milk production, as well as to its immune 65 responses. Therefore, a bad life start can impact negatively on animal\\'s adult 66 performance (1). Neonatal diseases, especially diarrhea, and nutritional mistakes can act 67 as stressors, lowering calf immunity, and increasing animal susceptibility to other 68 disorders (2). The changes in intestinal and ruminal microbiota, caused by disease or 70 malnutrition, can influence animal health, mortality rates and performance (3). 71 Therefore, tools that help improving heifer development and health are essential to 72 reduce disease, mortality and morbidity as well as accelerate the calf development. 73 Additionally, since a calf is born functionally non-ruminant, it must have its digestive 74 system and other organs and tissues changed in several weeks as well to adapt to the 75 colonization of specific microbiota. The bacteria in the rumen must start the 76 fermentation of carbohydrates, so the calf can become dependent mostly on volatile 77 fatty acids (VFA) and not on lactose-driven metabolism anymore (4). : bioRxiv preprint 4 78 Since diseases have a negative impact on the heifer\\'s future performance, 79 procedures that reduce the animal\\'s susceptibility to pathogens and stressors may 80 improve future performance and productivity. Thus feed additives, such as growth 81 promoter antibiotics, have been used to improve both rumen development and animal 82 health (5), as well as increase performance and feed efficiency (6). However, the use of 83 antibiotics in animal production for this purpose has been under severe criticism. Due to 84 the increase of bacterial resistance and impact over human health it has been banned in 85 several countries. Furthermore, the increase of organic dairy farms in the past years 86 revealed a change in milk consumers\\' preference with increased demand for \"natural\" 87 and healthier foods. An alternative to the use of these additives is the utilization of 88 herbal products, such as natural additives, for ruminant production (7). Essential oils are herbal plant chemical components, constituted by volatile or 90 ethereal oils that have been applied as a natural and safe alternative for antibiotics (8). 91 Some of their properties are antiseptic and antimicrobial activities, that interfere on 92 bacterial, fungal and protozoa cell functioning (9), presenting the similar efficiency to 93 treat some diseases as antibiotics (10). They also contribute for prevention of oxidative 94 stress (11), and help immune response to change leucocyte phagocytic activity and 95 inhibiting complement system (12). Lastly, essential oils have a function similar to 96 ionophores, influencing the gastrointestinal tract development, rumen microbiological 97 activity, improving feed efficiency and decreasing neonatal diseases (9,13). Previous studies have shown that essential oil supplementation in calf solid starter 99 improves performance (14,15), rumen fermentation (16) and diarrhea severity (17). 100 However, the effects over liquid diet supplementation are scarce. The aim of this study 101 was to evaluate how adding a commercial blend of essential oils (BEO) in milk replacer 102 (MR) affects feed intake, performance, feed efficiency, body development, blood cells 103 and metabolites, insulin-like growth factor-1 (IGF-1), ruminal parameters, fecal and . CC-BY 4.0 '},\n"," {'paper_id': '7b21130d26a97ff62ef1810505eb083ec77657f8',\n","  'section': 'Abstract',\n","  'text': 'A total of 116 hospitalized patients with confirmed COVID-19 pneumonia and SARS-CoV-2-specific immunoglobulins tested in Tongji hospital were retrospectively investigated. Clinical, laboratory, radiological characteristics and outcomes data were compared between mild-moderate group and died group. Further, a paired case-control study was conducted where each deceased case was matched to three mild-moderate patients of similar age. Among 116 subjects included, 101 mild-moderate patients survived and 15 cases died. SARS-CoV-2-specific IgM levels peaked in forth week after onset of COVID-19 pneumonia, while serum IgG levels increased over 8 weeks. Serum IgM levels were higher in deceased patients than mild-moderate patients (P = 0.024), but not IgG. Serum IgM levels were negatively correlated with clinical outcome, eosinophil count and albumin levels (r = -0.269, P = 0.003; r = -0.188, P = 0.043; and r = -0.198, P = 0.033, resp.). The area under the ROC curve (AUC) for IgM antibody was 0.681 (95% CI: 0.517-0.845, P = 0.024). In case-control study paired by age, serum IgM was higher in deceased patients than mild-moderate patients (P = 0.019), positively correlated with leucocyte count (r = 0.260, P = 0.045), while negatively correlated with clinical outcome and albumin levels (r = -0.337, P = 0.008; r = -0.265, P = 0.041). AUC for IgM levels was 0.704 (95% CI: 0.534-0.873, P = 0.019). These results indicate that dynamics of SARS-CoV-2 specific IgM and IgG antibodies was similar with that of SARS-CoV, while elevated serum IgM levels indicate poor outcome in patients with COVID-19 pneumonia. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '7b30d043f8f04a83259f35c9528879ecfe948ee8',\n","  'section': 'Abstract',\n","  'text': 'The Coronavirus (COVID-19) has advanced into 197 countries and territories leaving behind a total of 372,757 confirmed cases and 16231 deaths. One the basis of WHO situation reports data of COVID-19 along with daily official reports from the Japan, China and the Kore we modelled the spread of COVID19 by using the Successive Approximation Method. We defined the two state of data to find the mean ratio (η) of the present cases count to the sum of previous and present cases. This ratio further predicts the future state of COVID-19 pandemic. Results: The mean ratio (η) of expected cases were found 0.485, while the mean ratio for deaths was found to be 0.49. We calculated worldwide expected lower bound value for confirmed cases 247007 cases with maximum limit of 1667719 cases and minimum deaths count 8660 with upper limit of 117397 deaths in next 30 days. While in the case of Iran, a large increase in the number of deaths are expected in the upcoming 30 days with lower bound value of 1140 deaths and maximum value of 598478 deaths. Interpretation: Iran whole population is on risk. '},\n"," {'paper_id': '7b32551cd8d200eaebeb9439a5c8ef2e4c31250d',\n","  'section': 'Abstract',\n","  'text': 'Coronavirus infection spreads in clusters and therefore early identification of these clusters is critical for slowing down the spread of the virus. Here, we propose that daily population-wide surveys that assess the development of symptoms caused by the virus could serve as a strategic and valuable tool for identifying such clusters to inform epidemiologists, public health officials, and policy makers. We show preliminary results from a survey of over 58,000 Israelis and call for an international consortium to extend this concept in order to develop predictive models. We expect such data to allow: Faster detection of spreading zones and patients; Obtaining a current snapshot of the number of people in each area who have developed symptoms; Predicting future spreading zones several days before an outbreak occurs; Evaluating the effectiveness of the various social distancing measures taken, and their contribution to reduce the number of symptomatic people. Such information can provide a valuable tool for decision makers to decide which areas need strengthening of social distancing measures and which areas can be relieved. Preliminary analysis shows that in neighborhoods with confirmed COVID-19 patient history, more responders report on COVID-19 associated symptoms, demonstrating the potential utility of our approach for detection of outbreaks. Researchers from other countries including the U.S '},\n"," {'paper_id': '7b5be48b7769ccad2f44a8c54efcea5281a6c633',\n","  'section': 'Abstract',\n","  'text': '20 BACKGROUND The COVID-19 epidemic, first emerged in Wuhan during December 2019, has 21 spread globally. While the mass population movement for Chinese New Year has significantly 22 influenced spreading the disease, little direct evidence exists about the relevance to epidemic and its 23 control of population movement from Wuhan, local emergency response, and medical resources in 24 China. fatality rate was 2.84%, much higher in Hubei than in other regions (3.27% vs 0.73%). The index of 29 population inflow from Hubei was positively correlated with total (Provincial r=0.9159, p<0.001; 30 City r=0.6311, p<0.001) and primary cases (Provincial r=0.8702, p<0.001; City r=0.6358, p<0.001). 31 The local health emergency measures (eg, city lockdown and traffic control) were associated with 32 reduced infections nationwide. Moreover, the number of public health employees per capita was 33 inversely correlated with total cases (r=-0.6295, p <0.001) and infection rates (r =-0.4912, p <0.01). 34 Similarly, cities with less medical resources had higher fatality (r =-0.4791, p<0.01) and lower cure 35 rates (r = 0.5286, p<0.01) among the confirmed cases. 36 CONCLUSIONS The spread of the COVID-19 in China in its early phase was attributed primarily 37 to population movement from Hubei, and effective governmental health emergency measures and 38 adequate medical resources played important roles in subsequent control of epidemic and improved 39 prognosis of affected individuals. 40 41 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': '7b6d4793dd3746e9292feba95dab38ff505905be',\n","  'section': 'Abstract',\n","  'text': 'Initial projections from the first generation of COVID-19 models focused public attention on worst-case scenarios in the absence of decisive policy action. These underscored the imperative for strong and immediate measures to slow the spread of infection. In the coming weeks, however, as policymakers continue enlisting models to inform decisions on COVID-19, answers to the most difficult and pressing policy questions will be much more sensitive to underlying uncertainties. In this study, we demonstrate a model-based approach to assessing the potential value of reducing critical uncertainties most salient to COVID-19 decision-making and discuss priorities for acquiring new data to reduce these uncertainties. We demonstrate how information about the impact of non-pharmaceutical interventions could narrow prediction intervals around hospitalizations over the next few weeks, while information about the prevalence of undetected cases could narrow prediction intervals around the timing and height of the peak of the epidemic. '},\n"," {'paper_id': '7b85fdd826ea9d3f259967d8028b3cf735c80845',\n","  'section': 'Abstract',\n","  'text': \"The novel coronavirus 2019-nCoV/SARS-CoV-2 infection has shown discernible variability across the globe. While in some countries people are recovering relatively quicker, in others, recovery times have been comparatively longer and numbers of those succumbing to it high. In this study, we aimed to evaluate the likely association between an individual's ancestry and the extent of COVID-19 manifestation employing Europeans as the case study. We employed 10,215 ancient and modern genomes across the globe assessing 597,573 single nucleotide polymorphisms (SNPs). Pearson's correlation coefficient (r) between various ancestry proportions of European genomes and COVID-19 death/recovery ratio was calculated and its significance was statistically evaluated. We found significant positive correlation (p=0.03) between European Mesolithic hunter gatherers (WHG) ancestral fractions and COVID-19 death/recovery ratio and a marginally significant negative correlation (p=0.06) between Neolithic Iranian ancestry fractions and COVID-19 death/recovery ratio. We further identified 404 immune response related single nucleotide polymorphisms (SNPs) by comparing publicly available 753 genomes from various European countries against 838 genomes from various Eastern Asian countries in a genome wide association study (GWAS). Prominently we identified that SNPs associated with Interferon stimulated antiviral response, Interferon-stimulated gene 15 mediated antiviral mechanism and 2′-5′ oligoadenylate synthase mediated antiviral response show large differences in allele frequencies between Europeans and East Asians. Overall, to the best of our knowledge, this is the first study evaluating the likely association between genetic ancestry and COVID-19 manifestation. While our current findings improve our overall understanding of the COVID-19, we note that the development of effective therapeutics will benefit immensely from a more detailed analyses of individual genomic sequence data from COVID-19 patients of varied ancestries. \"},\n"," {'paper_id': '7b9e74201824157c69d4989779b00ded3caa2dba',\n","  'section': 'Abstract',\n","  'text': 'bioRxiv preprint Serine Protease 2) in the host cell. Importantly, beside respiratory symptoms, consistent with other common respiratory virus infection when patients become viraemic, a significant number of COVID-19 patients also develop liver comorbidities. We explored if specific target cell-type in the mammalian liver, could be implicated in disease pathophysiology other than the general deleterious response to cytokine storms. Here we employed single-cell RNA-seq (scRNA-seq) to survey the human liver and identified potentially implicated liver cell-type for viral ingress. We report the co-expression of ACE2 and TMPRSS2 in a TROP2 + liver progenitor population. Importantly, we fail to detect the expression of ACE2 in hepatocyte or any other liver (immune and stromal) cell types. These results indicated that in COVID-19 associated liver dysfunction and cell death, viral infection of TROP2 + progenitors in liver may significantly impaired liver regeneration and could lead to pathology. -EPCAM + Liver progenitors co-express ACE2 and TMPRSS2 -ACE2 and TMPRSS2 expression is highest in TROP2 high progenitors -ACE2 and TMPRSS2 cells express cholangiocyte biased fate markers -ACE2 and TMPRSS2 positive cells are absent in human fetal liver . '},\n"," {'paper_id': '7bc87a77a7aba144248cfa3bc514541f09047edf',\n","  'section': 'Abstract',\n","  'text': 'To the memory of Rossiter H. Crozier , an evolutionary biologist, who with his generosity and inquisitiveness inspired many students and scientists, in Australia and abroad. doi: bioRxiv preprint 2 Molecular phylogenetics plays a key role in comparative genomics and has an increasinglysignificant impacts on science, industry, government, public health, and society. We posit that the current phylogenetic protocol is missing two critical steps, and that their absence allows model misspecification and confirmation bias to unduly influence our phylogenetic estimates. Based on the potential offered by well-established but under-used procedures (e.g., assessment of phylogenetic assumptions and tests of goodness-of-fit), we introduce a new phylogenetic protocol that will reduce confirmation bias and increase the accuracy of phylogenetic estimates. doi: bioRxiv preprint CONFIRMATION BIAS IN PHYLOGENETICS Bayesian methods (77-82)), or using non-parametric phylogenetic methods (i.e., parsimony methods (66,68,70,71,(83)(84)(85)). In reality, many researchers analyze their data using a range of model-based phylogenetic methods, and reports that only use parsimony methods are increasingly rare. Depending on the chosen phylogenetic method, researchers may have to select a suitable model of sequence evolution (i.e., a model that combines the substitution model and the rates-across-sites model) to apply to the sub-MSA. This choice is often made by using model-selection methods (6,86-97). Having chosen a phylogenetic method and, in relevant cases, a suitable model of sequence evolution, the next step involves obtaining accurate estimates of the tree and evolutionary processes that led to the data. Phylogenetic methods are implemented in many programs (66-85) and, depending on the methods chosen, users often also obtain the nonparametric bootstrap probability (98) or clade credibility (99) to measure support for divergence events in the phylogeny. '},\n"," {'paper_id': '7bdceb3b947ec934199d0c97fb9ec5a0e2bac776',\n","  'section': 'Abstract',\n","  'text': 'Coronavirus Disease 2019 and severe acute respiratory syndrome (SARS)-related coronaviruses (e.g., 2019-nCoV and SARS-CoV) are phylogenetically distantly related, but both are capable of infecting human hosts via the same receptor, angiotensin-converting enzyme 2, and cause similar clinical and pathological features, suggesting their phenotypic convergence. Yet, the molecular basis that underlies their phenotypic convergence remains unknown. Here, we used a recently developed molecular phyloecological approach to examine the molecular basis leading to their phenotypic convergence. Our genome-level analyses show that the spike protein, which is responsible for receptor binding, has undergone significant Darwinian selection along the branches related to 2019-nCoV and SARS-CoV. Further examination shows an unusually high proportion of evolutionary convergent amino acid sites in the receptor binding domain (RBD) of the spike protein between COVID-19 and SARS-related CoV clades, leading to the phylogenetic uniting of their RBD protein sequences. In addition to the spike protein, we also find the evolutionary convergence of its partner protein, ORF3a, suggesting their possible co-evolutionary convergence. Our results demonstrate a strong adaptive evolutionary convergence between COVID-19 and SARS-related CoV, possibly facilitating their adaptation to similar or identical receptors. Finally, it should be noted that many observed bat SARS-like CoVs that have an evolutionary convergent RBD sequence with 2019-nCoV and SARS-CoV may be preadapted to human host receptor ACE2, and hence would be potential new coronavirus sources to infect humans in the future. '},\n"," {'paper_id': '7c3b4190dd65bd11e2ea7bc133a583f5352a5961',\n","  'section': 'Abstract',\n","  'text': 'have set up a public forum on www.LAMP-Seq.org. The ongoing COVID-19 pandemic has already caused devastating losses. Early evidence shows that the exponential spread of COVID-19 can be slowed by restrictive isolation measures, but these place a tremendous burden on society. Moreover, once these restrictions are lifted, the exponential spread is likely to re-emerge. It has been suggested that populationscale testing can help break the cycle of isolation and spread, but current detection methods are not capable of such large-scale processing. Here we propose LAMP-Seq, a barcoded Reverse-Transcription Loop-mediated Isothermal Amplification (RT-LAMP) protocol that could dramatically reduce the cost and complexity of population-scale testing. In this approach, individual samples are processed in a single heat step, producing barcoded amplicons that can be shipped to a sequencing center, pooled, and analyzed en masse. Using unique barcode combinations per sample from a compressed barcode space enables extensive pooling, significantly reducing cost and organizational efforts. Given the low cost and scalability of next-generation sequencing, we believe that this method can be affordably scaled to analyze millions of samples per day using existing sequencing infrastructure. '},\n"," {'paper_id': '7c3c810250221b9947a2848a87a2b9126c545939',\n","  'section': 'Abstract',\n","  'text': 'The novel Coronavirus SARS-CoV-2 (CoV) has induced a world-wide pandemic and subsequent non-pharmaceutical interventions (NPI) in order to control the spreading of the virus. NPIs are considered to be critical in order to at least delay the peak number of infected individuals and to prevent the health care system becoming overwhelmed by the number of patients to treat in hospitals or in intensive care units (ICUs). However, there is also increasing concern that the NPIs in place would increase mortality because of other diseases, increase the frequency of suicide and increase the risk of an economic recession with unforeseeable implications. It is therefore instrumental to evaluate the necessity of NPIs and to monitor the progress of containment of the virus spreading. We used a data-driven estimation of the evolution of the reproduction number for viral spreading in Germany as well as in all its federal states. Based on an extended infectionepidemic model, parameterized with data from the Robert Koch-Institute and, alternatively, with parameters stemming from a fit to the initial phase of CoV spreading in different regions of Italy, we consistently found that the reproduction number was turned down to a range near 1 in all federal states. We used the latest reproduction number as a starting point for the simulation of epidemic progression and varied the reproduction number, mimicking either release or strengthening of NPIs. Germany is currently, April 3rd, 2020, at the border line of a reproduction number between the scenarios of major immunisation of the population or eradication of the virus. We strongly recommend to keep all NPIs in place and suggest to even strengthen the measures in order to accelerate reaching the state of full control, thus, also limiting collateral damage of the NPIs in time. : medRxiv preprint 1 day, although the range varies between 0 -3 days (IQR) depending on age. The time span spent during hospitalization to admission in ICU is likely shorter as the patients get older. For Germany where the majority of reported infections are in middle-aged groups, we assume the median value of 1/R7 ~ 2.5 days. We have varied 1/R7 within a range of 1 -3.5 days. The inverse of R8 depicts the time span spent in ICU before discharge. Here, we assume that patients who are recovering in ICU do not spend much time in general hospital care. 1/R8 is also presumably dependent on age, with more severe courses and thus prolonged ICU admission needed for high risk individuals, such as aged patients. WHO-China joint mission on COVID-19 outbreak (WHO-China 2020) reports that the total duration of being in hospital for the severely ill patients can be around 3 to 6 weeks. This gives us a range of possible values because: . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': '7c8152250ebc8a8b31066de90c3bdaebc076365b',\n","  'section': 'Abstract',\n","  'text': 'Objective: To analyse the clinical features of COVID-19 parturients, and to compare anaesthetic regimen and clinical outcomes in parturients with or without COVID-19 undergoing cesarean delivery. Methods: Data were extracted from the electronic medical record of 3 medical institutions in Hubei Province, China, from June 1, 2019 to March 20, 2020 according to inclusion and exclusion criteria. After propensity score matching with demographics, the clinical and laboratory characteristics of parturients with or without COVID-19 were analysed. The anaesthetic regimen and clinical outcomes of themselves and their infants were compared in these two groups of parturients. Results: A total of 1,588 patients without SARS-CoV-2 infection undergoing cesarean delivery were retrospectively included. After achieving a balanced cohort through propensity score matching, 89 patients (COVID-19 group), who were diagnosed with COVID-19 by SARS-CoV-2 nucleic acid test and CT scan matched with 173 patients without COVID-19 (Control group). The SARS-CoV-2 infected parturients in the early stages of COVID-19 outbreak was much more than during the later stage. The main clinical characteristics of parturients with COVID-19 were fever (34.8%), cough (33.7%), an increased plasma CRP (52.8%) and a decreased lymphocyte counting (33.7%). A high rate of emergency and a high incidence of anaesthesia-related complications, such as pharyngalgia, multiple puncture, intraoperative hypotension, nausea, vomiting, vertigo and chills in the COVID-19 parturients. In addition, the parturients with COVID-19 had a long duration of operation and hospital stay, and an increased intraoperative oxytocin utilization and postoperative oxygen therapy. The newborns from the SARS-CoV-2 infected mothers, who received general anaesthesia, had a high risk of Apgar score ≤ 8 at 1 and 5 minutes after delivery and a higher rate of neonatal intensive care unit (NICU) admission. Conclusions: Anaesthesia-related complications occur more frequently in the COVID-19 parturients and their newborns have a high risk of distress. '},\n"," {'paper_id': '7cb5dcb9696e0fea95feaf9bd9ea236be5e42bc1',\n","  'section': 'Abstract',\n","  'text': 'The COVID-19 pandemic has resulted in increased need for diagnostic testing using reverse transcriptase real-time PCR (RT-PCR). An exponential increase in demand has resulted in a shortage of numerous reagents in particular those associated with the lysis buffer required to extract the viral RNA. Herein, we describe a rapid collective effort by hospital laboratory scientists, academic researchers and the biopharma industry to generate a validated lysis buffer. We have formulated a 4M Guanidinium thiocyanate (GITC)/ Triton X-100 Lysis buffer which provides comparable results with the recommended reagents. This buffer will ease the burden on hospital labs in their heroic efforts diagnose a large population of patients. '},\n"," {'paper_id': '7cdd50888f639629380fa9906d73f97010158127',\n","  'section': 'Abstract',\n","  'text': 'Background. T cell immunoglobulin mucin domain-1 (TIM-1) is a phosphatidylserine (PS) 28 receptor, mediating filovirus entry into cells through interactions with PS on virions. TIM-1 29 expression has been implicated in Ebola virus (EBOV) pathogenesis; however, it remains 30 unclear whether this is due to TIM-1 serving as a filovirus receptor in vivo or, as others have 31 suggested, TIM-1 induces a cytokine storm elicited by T cell/virion interactions. Here, we use a 32 BSL2 model virus that expresses EBOV glycoprotein and demonstrate the importance of TIM-1 33 as a virus receptor late during in vivo infection. 34 Methodology/Principal findings. We used an infectious, recombinant vesicular stomatitis virus 35 expressing EBOV glycoprotein (EBOV GP/rVSV) to assess the role of TIM-1 during in vivo 36 infection. TIM-1-sufficient or TIM-1-deficient BALB/c interferon α/β receptor -/mice were 37 challenged with EBOV GP/rVSV-GFP or G/rVSV-GFP. While G/rVSV caused profound 38 morbidity and mortality in both mouse strains, TIM-1-deficient mice had significantly better 39 survival than TIM-1-expressing mice following EBOV GP/rVSV challenge. EBOV GP/rVSV 40 load in spleen was high and unaffected by expression of TIM-1. However, infectious virus in 41 serum, liver, kidney and adrenal gland was reduced late in infection in the TIM-1-deficient mice, 42 suggesting that virus entry via this receptor contributes to virus load. Consistent with higher 43 virus loads, proinflammatory chemokines trended higher in organs from infected TIM-1-44 sufficient mice compared to the TIM-1-deficient mice, but proinflammatory cytokines were more 45 modestly affected. To assess the role of T cells in EBOV GP/rVSV pathogenesis, T cells were 46 depleted in TIM-1-sufficient and -deficient mice and the mice were challenged with virus. 47 Depletion of T cells did not alter the pathogenic consequences of virus infection. 48 . CC-BY 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/466102 doi: bioRxiv preprint Conclusions. Our studies provide evidence that at late times during EBOV GP/rVSV infection, 49 TIM-1 increased virus load and associated mortality, consistent with an important role of this 50 receptor in virus entry. This work suggests that inhibitors which block TIM-1/virus interaction 51 may serve as effective antivirals, reducing virus load at late times during EBOV infection. 52 53 Author summary 54 T cell immunoglobulin mucin domain-1 (TIM-1) is one of a number of phosphatidylserine (PS) 55 receptors that mediate clearance of apoptotic bodies by binding PS on the surface of dead or 56 dying cells. Enveloped viruses mimic apoptotic bodies by exposing PS on the outer leaflet of the 57 viral membrane. While TIM-1 has been shown to serve as an adherence factor/receptor for 58 filoviruses in tissue culture, limited studies have investigated the role of TIM-1 as a receptor in 59 vivo. Here, we sought to determine if TIM-1 was critical for Ebola virus glycoprotein-mediated 60 infection using a BSL2 model virus. We demonstrate that loss of TIM-1 expression results in 61 decreased virus load late during infection and significantly reduced virus-elicited mortality. 62 These findings provide evidence that TIM-1 serves as an important receptor for Ebola virus in 63 vivo. Blocking TIM-1/EBOV interactions may be effective antiviral strategy to reduce viral load 64 and pathogenicity at late times of EBOV infection. 65 66 . CC-BY 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/466102 doi: bioRxiv preprint 129 for maEBOV we observed that EBOV GP/rVSV was less pathogenic in TIM-1-deficient mice 130 compared to control mice. The impact of the loss of TIM-1 was specific for EBOV GP-131 expressing virus since wild-type VSV was equally virulent in TIM-1-deficient and TIM-1-132 sufficient mice over a wide range of challenge doses. Importantly, reduced mortality observed 133 in the virus-infected TIM-1 -/mice was associated with lower virus load at late time points 134 during infection in multiple tissues previously appreciated to be important in EBOV 135 . CC-BY 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/466102 doi: bioRxiv preprint 157 Havcr1, were replaced with a LacZ gene, generating a TIM-1-null mouse (TIM-1 -/-) BALB/c 158 IFN-αβ receptor-deficient (Ifnar -/-) mice were a kind gift from Dr. Joan Durbin, NYU Langone 159 . CC-BY 4.0 International license is made available under a '},\n"," {'paper_id': '7cf99cc9d70acc8343ca92edc26a5a5bad134cf0',\n","  'section': 'Abstract',\n","  'text': 'The global outbreak of coronavirus disease 2019 has placed an unprecedented burden on healthcare systems as the virus spread from the initial 27 reported cases in the city of Wuhan, China to a global pandemic in under three months 1 . Resources essential to monitoring virus transmission have been challenged with a demand for expanded surveillance. The CDC 2019-nCoV Real-Time Diagnostic Panel uses a real-time reverse transcription quantitative polymerase chain reaction (RT-qPCR) consisting of two TaqMan probe and primer sets specific for the 2019-nCoV N gene, which codes for the nucleocapsid structural protein that encapsulates viral RNA, for the qualitative detection of 2019-nCoV viral RNA in respiratory samples. To isolate RNA from respiratory samples, the CDC lists RNA extraction kits from three manufacturers. In anticipation of a limited supply chain of RNA extraction kits and the need for test scalability, we sought to identify alternative RNA extraction methods. Here we show that direct lysis of respiratory samples can be used in place of RNA extraction kits to run the CDC 2019-nCoV Real-Time Diagnostic assay with the additional benefits of higher throughput, lower cost, faster turnaround and possibly higher senitivity and improved saftey. '},\n"," {'paper_id': '7cfaa2540d3c8eea0982b3b1147884f125f67ff2',\n","  'section': 'Abstract',\n","  'text': 'Recent reports suggest an association between COVID-19 and altered olfactory function. Here we analyze bulk and single cell RNA-Seq datasets to identify cell types in the olfactory epithelium that express molecules that mediate infection by SARS-CoV-2 (CoV-2), the causal agent in COVID-19. We find in both mouse and human datasets that olfactory sensory neurons do not express two key genes involved in CoV-2 entry, ACE2 and TMPRSS2. In contrast, olfactory epithelial support cells and stem cells express both of these genes, as do cells in the nasal respiratory epithelium. Taken together, these findings suggest possible mechanisms through which CoV-2 infection could lead to anosmia or other forms of olfactory dysfunction. author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '7d91bba210d98289dac830135c56c13e206837ec',\n","  'section': 'Abstract',\n","  'text': 'Plants have a highly sophisticated endomembrane system that is targeted by plant viruses for cell-to-cell movement. The movement protein (MP) of Ourmia melon virus (OuMV) is targeted to plasmodesmata (PD) and forms tubules to facilitate cell-to-cell movement. Despite a number of functionally important regions for correct subcellular localization of OuMV MP has been identified, little is known about the pathways OuMV MP hijacks to reach PD. Here, we demonstrate that OuMV MP localizes to the trans-Golgi network (TGN), but not to the multivesicular body/prevacuolar compartment or Golgi, and carries two putative sorting motifs, a tyrosine (Y) and a dileucine (LL) motif, near its N-terminus. Introducing glycine substitutions in these motifs results in loss of OuMV infectivity in Nicotiana benthamiana and Arabidopsis (Arabidopsis thaliana). Live cell imaging of GFP-labeled sorting motif mutants shows that Y motif mutants fail to localize to the TGN, plasma membrane, and PD. Mutations in the LL motif do not impair plasma membrane targeting of MP, but affect its ability to associate with callose deposits at PD. Taken together, these data suggest that both Y and LL motifs are indispensable for targeting of OuMV MP to PD and for efficient systemic infection, but show differences in functionality. This study provides new insights into the role of sorting motifs in intracellular targeting of MPs and vesicle trafficking pathways that plant viruses hijack for cell-to-cell movement. '},\n"," {'paper_id': '7d98a73314cd9f923aaa367f3f4de056b8cedec5',\n","  'section': 'Abstract',\n","  'text': \"word count: 197 Text word count: 9541 author/funder. All rights reserved. No reuse allowed without permission. Interferon-induced proteins with tetratricopeptide repeats (IFITs) are highly expressed during the cell-intrinsic immune response to viral infection. IFIT1 inhibits translation by binding directly to the 5' end of foreign RNAs, particularly those with non-self cap structures, precluding the recruitment of the cap-binding eukaryotic translation initiation factor 4F and subsequent 40S recruitment. Interaction of different IFIT family members is well described, but little is known of the molecular basis of IFIT association or its impact on function. Here, we reconstituted different complexes of IFIT1, IFIT2 and IFIT3 in vitro, which enabled us to reveal critical aspects of IFIT complex assembly. IFIT1 interacts rapidly and strongly with IFIT3 forming a stable heterotetramer. IFIT2 and IFIT3 homodimers dissociate to form a more stable heterodimer that associates with IFIT1, forming an IFIT1:IFIT2:IFIT3 trimer. Site-directed mutagenesis revealed a C-terminal 'YxxxL' motif in IFIT1 that mediates its association with IFIT3. Using various reporter mRNAs, we demonstrate for the first time that IFIT3 stabilises IFIT1 binding to cap0-mRNA and enhances its translation inhibition activity. Disrupting the binding interface between IFIT1 and IFIT3 abrogated this enhancement. This work reveals molecular aspects of IFIT assembly and provides an important 'missing link' between IFIT interaction and function. author/funder. All rights reserved. No reuse allowed without permission. doi: bioRxiv preprint PCR amplified to contain 5' NdeI and 3' XhoI restriction sites for cloning into pET28b (Novagen) producing a full-length protein with thrombin cleavable N-terminal 6-His tag. For reporter RNA transcription, the firefly luciferase reporter gene (Fluc) was PCR amplified using primers containing the 5' UTR and 3' UTR sequences of human \\uf062-globin (NM_000518.4), including a 5' T7 promotor and EcoRI and PstI sites to facilitate cloning into pUC57. pUC57-ZIKV-Fluc was previously described (30). Recombinant IFITs were expressed in Rosetta 2 (DE3) pLysS (Novagen). Cells were grown to an OD600 of approximately 1 in 2x TY media at 37 \\uf0b0C. Expression was induced by adding 1 mM isopropyl β-D-1thiogalactopyranoside. The induced culture was incubated at 22 \\uf0b0C for 16 hours. Cells were harvested and lysed in a buffer containing 20 mM Tris-Cl pH 7.5, 400 mM KCl, 5 % glycerol, 1 mM DTT and 0.5 mM phenylmethylsulfonyl fluoride and 0.5 mg/ml lysozyme (from hen egg). IFITs were isolated by affinity chromatography on Ni-NTA Agarose beads (Qiagen). IFIT1 and IFIT1 mutants were additionally purified by FPLC on MonoQ (Q buffer: 20 mM Tris-Cl pH 7.5, 5 % glycerol, 1 mM DTT and 100-500 mM KCl), followed by MonoS 5/50 GL (S buffer: 30 mM HEPES pH 7.5, 5 % glycerol, 1 mM DTT and 100-500 mM KCl). IFIT2 and IFIT3 were treated with thrombin (from bovine plasma). author/funder. All rights reserved. No reuse allowed without permission. \"},\n"," {'paper_id': '7dd547f2db271b15f7ba1ea69583ec5d2b38ef27',\n","  'section': 'Abstract',\n","  'text': 'Short running head: Sensitive detection of SARS-CoV-2 using double-quencher probe ORCID : 0000-0002-8002-834X (Yosuke Hirotsu) Abstract Background Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) emerges in Wuhan City, Hubei Province, spreads worldwide, and threats the human life. The detection of SARS-CoV-2 is important for the prevention of the outbreak and management of patients. Real-time reverse-transcription polymerase chain reaction (RT-PCR) assay detected the virus in clinical laboratory. This study utilized primers and single-quencher probes in accordance with the Centers for Disease Control and Prevention (CDC) in the USA and the National Institute of Infectious Diseases (NIID) in Japan. Moreover, we designed the double-quencher probes (YCH assay) according to the oligonucleotide sequence established by NIID. Using these assays, we conducted a one-step real-time RT-PCR with serial DNA positive control to assess the detection sensitivity. The threshold cycle (Ct) value of RT-PCR was relatively low in CDC and YCH assays compared to NIID assay. Serial dilution assay showed that both CDC and YCH assays could detect a low-copy number of DNA positive control. The background fluorescent signal at the baseline was lower in YCH than that of NIID. Double-quencher probes decreased background fluorescent signal and improved detection sensitivity of SARS-CoV-2. '},\n"," {'paper_id': '7e6bd4f9d794f87445f67c9ed9d5699706040fa4',\n","  'section': 'Abstract',\n","  'text': 'doi: bioRxiv preprint molecular biology. Our high-resolution data have enabled a precise study of transcription and translation in both pathogenic and attenuated forms of IBV, and expand our understanding of gammacoronaviral gene expression. We demonstrate that gene expression shows considerable intraspecies variation, with single nucleotide polymorphisms associated with altered production of sgmRNA transcripts, and our RiboSeq data sets enabled us to uncover novel ribosomally occupied ORFs in both strains. We also identify numerous cellular genes and gene networks that are differentially expressed during virus infection, giving insights into the host cell reponse to IBV infection. 3 47 48 49 50 51 52 53 54 55 56 author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '7e80848c7b22e870b0e0a458dce96f6955576f04',\n","  'section': 'Abstract',\n","  'text': \"As a heritable sequence-specific adaptive immune system, CRISPR-Cas is a powerful force shaping strain diversity in host-virus systems. While the diversity of CRISPR alleles has been explored, the associated structure and dynamics of host-virus interactions has not. We develop theory on the role of CRISPR immunity in mediating the interplay between host-virus interaction structure and eco-evolutionary dynamics in a computational model and three natural systems. We show that the structures of networks describing who infects whom and who is protected from whom are modular and weighted-nested, respectively. The dynamic interplay between these networks influences transitions between dynamical regimes of virus diversification and host control,leading to eventual virus extinction. The three empirical systems exhibit weighted nestedness, a pattern our theory shows is indicative of viral control by hosts. Previously missing from studies of microbial host-pathogen systems, the protection network plays a key role in the coevolutionary dynamics. The structure of complex ecological communities is clearly non-random. It is generally argued that interaction structure affects the stability of communities to perturbations [1-3] and arises from coevolutionary dynamics between interacting partners [4] [5] [6] [7] , yet the structure-dynamics nexus remains a long-standing central question in community and network ecology. Host-parasite infection networks, in which interaction structure represents 'who infects whom', have been typically described as modular, nested, or both [8, 9] . Modularity concerns patterns of specificity, in which the network is partitioned into modules of hosts and parasites that interact densely with each other but sparsely with those outside the group. Nestedness concerns instead patterns of specialization, and describes a network structure in which specialist hosts are infected by subsets of parasites that in turn also infect the more generalist hosts [8, 9] . Common to all host-parasite network studies is a dominant focus on patterns of infection. Infection structure should critically depend however on the genetic basis of resistance of hosts to pathogens [10] . The emergence of network structure from immune selection has been shown in pathogens of humans such as Plasmodium falciparum [11, 12]. In particular, strain theory developed for pathogens with multilocus encoding of antigens posits that negative frequency dependent selection, mediated by competition for hosts through cross-immunity, can structure pathogen populations into clusters of strains with limited overlap of antigenic repertoires [11] [12] [13] [14] [15] . Parasites with 2 . CC-BY-NC-ND 4.0 International license is made available under a resistance by surface mutation can arise quickly to take over CRISPR or restriction-modification systems in providing immunity. This experimental and theoretical evidence for advantages of surface modification may entail strong fitness costs under natural conditions because mutations in surface receptors such as those in flagella or pili can be detrimental for microbial movement and biofilm formation [21] . Investigations of natural populations have indeed shown that surface resistance is costly in the wild [27, 28] . These observations underlie the need for complementing experimental approaches by developing theory on the kinds of 'macroscopic', whole system, diversity patterns expected under 'microscopic' CRISPR-induced coevolutionary processes. These patterns can serve as signatures in both nature and experiments for inferring whether specific underlying processes of resistance are at play. Resistance mechanisms can operate alone or in tandem to affect the diversification of microbes and phages. Nonetheless, not all of these are relevant for linking between host/phage genotypes and immune phenotype as observed in the structure of immunity and infection networks. Surface modification, for instance, is a result of mutation and selection that will provide advantage to one or few mutated strains periodically sweeping through the population. Restriction-modification systems, being unspecific to viral genetic sequence, also do not provide such link because they do not result in the creation of new strains. In contrast, CRISPR is a heritable adaptive immune system conferring sequence-specific protection against viruses, plasmids, and other mobile elements [23, 29] . As such, it not only provides a direct sequence-based link, but also allows consideration of concomitant structures for both infection and immunity, and from both host and parasite perspectives. The diversity of CRISPR alleles in natural, experimental and simulated populations has been explored [30] [31] [32] [33] [34] . It has been shown, for example, that increased strain diversity of Pseudomonas aeruginosa (with strains being clones with different spacer compositions) promotes the ability of the bacterium to control the DMS3vir virus [32] . Nevertheless, how such strain diversity evolves, the emergence of structure of host-virus interaction networks, and how they are assembled and disassembled remain unexplored. Advancing this predictive framework for CRISPR-Cas in microbial populations provides a basis for further study of the relative importance and interactions of different resistance mechanisms, and is essential for effective experimental manipulation of microbial systems, including applications to health and engineering [35, 36] . It can open new frontiers in studies of immune diversity and, more generally, contribute to tighten the connection between microbiology and evolutionary biology [37] . \"},\n"," {'paper_id': '7e9cd4bbf0fba1cc0bcded40041fd01b9dfb683f',\n","  'section': 'Abstract',\n","  'text': '162 laboratory-confirmed cases related to COVID-19. We described the demographic 15 characteristics of the cases and estimated the incubation period, serial interval and 16 basic reproduction number. With an estimation of 8 days incubation period and 6 days 17 serial interval, our results indicate that there may exist infectiousness during the 18 incubation period for 2019-nCoV. This increases the difficulty of screening or '},\n"," {'paper_id': '7ea2f1ca7b3c2370d413ae990eb71b2b990c91fe',\n","  'section': 'Abstract',\n","  'text': 'Drug susceptibility, defined by the minimal inhibitory concentration (MIC), often does not 25 predict whether fungal infections will respond to therapy in the clinic. Tolerance at supra-MIC 26 antifungal drug concentrations is rarely quantified and current clinical recommendations suggest 27 it be ignored. Here, we measured and characterized drug-response variables that could 28 influence the outcomes of fungal infections and be generalizable across major clades of 29 Candida albicans, one of the most frequently isolated human fungal pathogens. We quantified 30 antifungal tolerance as the fraction of growth (FoG) above the MIC and found that it is clearly 31 distinct from susceptibility/resistance measured as MIC. Instead, tolerance is due to the slow 32 growth of subpopulations of cells that overcome drug stress more efficiently than the rest of the 33 population, and correlates inversely with the accumulation of intracellular drug. Importantly, 34 many adjuvant drugs used together with fluconazole, a fungistatic drug, reduce tolerance 35 without affecting resistance. These include inhibitors of major stress response hubs such as 36 Hsp90, calcineurin, PKC1 and TOR. Accordingly, in an invertebrate infection model, adjuvant 37 combination therapy was significantly more effective than fluconazole alone in treating highly 38 tolerant isolates and did not improve the treatment of isolates with low tolerance levels. Furthermore, isolates recovered from immunocompetent patients with persistent candidemia 40 displayed significantly higher tolerance than isolates that were readily cleared by fluconazole. 41 Thus, tolerance correlates with the response to fluconazole therapy in patients and may help 42 predict whether infections will respond to fluconazole alone. Similarly, measuring tolerance may 43 provide a useful clinical parameter for choosing appropriate therapeutic strategies to overcome 44 persistent clinical candidemia. 45 46 47 48 49 All rights reserved. No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/206359 doi: bioRxiv preprint 50 A goal of antimicrobial susceptibility testing is to predict the clinical success or failure of 51 antibiotic therapy. Some infections are recalcitrant to drug treatment due to \\'resistance\\', which 52 refers to microbial growth in the presence of drug concentrations that inhibit susceptible 53 isolates 1,2 . Susceptibility is commonly measured as the Minimal Inhibitory Concentration (MIC) 54 after 24 h of growth in the presence of drug 3,4 . Fungal infections generally follow the \"90/60\" rule 55 for predicting therapeutic outcomes based on in vitro susceptibility testing: ~90% of susceptible 56 isolates and 60% of resistant isolates respond to therapy 5-9 . This implies that infection 57 outcomes are influenced by host factors as well as features of the pathogen that are not 58 reflected by the MIC 10,11 . In effect, organisms that cause persistent infections, defined as those 59 that are not cleared by a course of antifungal treatment, have similar susceptibilities to 60 organisms that are readily cleared by a course of antifungal treatment 12 . Accordingly, it is 61 important to identify measurable parameters that can contribute to disease severity. Only four classes of antifungals are currently in clinical use, and resistance to azoles, 63 including fluconazole (FLC), the most commonly administered antifungal against Candida 64 species, is an increasing problem. Altered drug uptake/drug efflux and changes in ergosterol 65 biosynthesis (the target for azole drugs) 13 are the major known mechanisms of azole resistance. Stress responses have been proposed as a third mechanism of antifungal resistance 14 . A 67 broad range of small molecules enhance antifungal activity in vitro and in vivo, with inhibitors of 68 Hsp90, calcineurin and TOR the most prominent among them 15-18 . Combination therapy using 69 antifungals together with such inhibitors has been proposed as a promising strategy to extend 70 the efficacy of current drugs [19][20][21][22][23][24]25,26,27 . In addition, several psychotherapeutic agents such as 71 fluoxetine, fluphenazine or sertraline can enhance FLC activity against fungal species 21,28-30 . Whether such adjuvants affect therapeutic outcomes remains to be addressed. Persistent candidemia, defined as the failure to clear a bloodstream infection caused by 74 a susceptible organism 12,31,32 , is associated with increased mortality. In one study, the mortality 75 All rights reserved. No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/206359 doi: bioRxiv preprint rate was 54% among infections with persistent candidemia and only 3% among those with non-76 persistent candidemia 33 . Mechanisms underlying persistent candidemia may include variability 77 in the pharmacology of the drug, suboptimal dosing, presence of fungal biofilms on indwelling 78 catheters, and reduced immunity. We posit that some responses to the drug are not captured 79 by measuring the MIC alone and that additional parameters could be used to predict the 80 likelihood that a clinical isolate might respond poorly to antifungal drugs. Furthermore, 81 understanding the mechanisms that underlie these parameters is critical for the development of 82 new therapeutic approaches against persistent Candida infections. 83 MIC measurements have been optimized to minimize or ignore residual fungal 84 growth 9,34,35 , termed \\'tolerance\\' or \\'trailing growth\\', which has been discussed in the literature for 85 over 20 years 36,37 and is detected in 25-60% of clinical isolates 4,10,19,38-44 . This recommendation 86 is based upon studies of acute infections using the mouse model of bloodstream candidiasis 45-87 47 , and the observation that isolates with high trailing growth in mucosal infections respond 88 positively to short term antifungal treatment, despite later recurrence of infection 48 . Trailing 89 growth is sensitive to environmental conditions, including pH, temperature and nutrients 3,49,50 90 and is usually detected in liquid cultures. Definitions of \\'tolerance\\' vary and generally describe 91 survival or growth above inhibitory concentrations 2,10,28,38,51-56 , detected as slow growth within 92 the zone of inhibition using E-strips or disk diffusion assays 57,58 , or in broth microdilution 93 assays 38,39 . Tolerance can be affected by several adjuvant drugs 19-24 , iron levels 19,59 , genes 94 involved in vacuolar protein sorting 38,47 , as well as calcium flux 28,55,59-61 . However, the precise 95 relationship between the outcome of fungal infections and tolerance or trailing growth has not 96 been determined. Because tolerant cells continue to divide in the presence of antifungals, we 97 postulated that they contribute to the persistence and/or recurrence of fungal infections. Here, we measured susceptibility and tolerance in a broad range of clinical isolates 99 spanning the major C. albicans clades 62-64 and asked how these parameters influence fungal 100 infection outcomes. We analyzed disk diffusion assays using diskImageR, and quantified the 101 All rights reserved. No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/206359 doi: bioRxiv preprint radius (RAD) of the zone of inhibition, a parameter that relates to the MIC 65 , and the fraction of 102 growth (FoG) within the zone of inhibition, a parameter that measures tolerance 19,38,50,66-68 . We 103 found that a range of adjuvant drugs, used in combination with FLC, increased drug cidality by 104 reducing FoG and not MIC and were effective both in vitro and in vivo at inhibiting strains with 105 high (and not low) FoG levels. Finally, highly tolerant isolates were associated with persistent 106 candidemia, suggesting that knowing the tolerance level of an infecting isolate may have 107 important clinical implications and may inform treatment options. 108 109 Results Tolerance measured as FoG is distinct from drug resistance. We quantified drug 111 responses in C. albicans isolates from different genetic backgrounds and types of infections 112 using diskImageR, a quantitative analysis tool that measures RAD, the radius of the zone of 113 inhibition, an indicator of susceptibility that relates to MIC 69,70 , and FoG, the fraction of growth 114 within the zone of inhibition, relative to the maximum possible growth 65 (Fig. 1a) . A screen of 115 219 clinical isolates (Supplementary Table 1 ) revealed that FoG levels ranged widely from 0.10 116 to 0.85 and did not correlate with RAD levels (Fig. 1b,c) , indicating that FoG and RAD measure 117 independent drug responses. FoG was detected in response to other drugs including fungistatic 118 antifungals (azoles, 5-fluorocytosine) and, to a lesser degree, to fungicidal agents 119 '},\n"," {'paper_id': '7eb6cd9756309f5a1d2fd29e417e10886cc84c0e',\n","  'section': 'Abstract',\n","  'text': \"medRxiv preprint [Abstract]Objective Coronavirus disease 2019 (COVID-19) has become pandemic in the world. The need for IgG-IgM combined antibody test is booming, but data on diagnostic indexes evaluation was inadequate. The aim of this study was to evaluate diagnostic indexes of a rapid IgG-IgM combined antibody test for SARS-CoV-2. Methods A total of 179 patients were enrolled. Serum were collected for IgG-IgM combined antibody test and corresponding nasal and pharyngeal swab specimens were collected for SARS-CoV-2 RT-PCR. According to SARS-CoV-2 RT-PCR results, patients under study were categorized as PCR positive group in 90 patients and PCR negative group in 89 patients. Results 1. Of the 90 PCR positive samples, 77 were tested positive by SARS-CoV-2 IgG-IgM test kit, yielding a sensitivity of 85.6%. Meanwhile, of the 89 PCR negative sample, 8 samples were detected positive, resulting in a specificity of 91%. Positive predictive value, negative predictive value and accuracy of this test kit was 95.1%, 82.7%, and 88.3%, respectively. Kappa efficiency between IgG/IgM test kit and RT-PCR were 0.75. 2. Accuracy in mild/common and severe/critical subgroup were 73.9% and 97.7%, respectively. Accuracy in clinical confirmed, suspected cases and other disease subgroups were 70%, 60%, and 100%, respectively. 3. Patients were further divided into '0 -7', '8 -15' and '>= 16' groups according to the time from illness onset to sample collection. Sensitivity, specificity and accuracy in these three groups were 18.8%, 77.8% and 40%; 100%, 50% and 87.5%; 100%, 64.3%, and 93.9, respectively. Conclusion The sensitivity and specificity of this ease-of-use IgG/IgM combined test kit were adequate, plus short turnaround time, no specific requirements for additional equipment or skilled technicians, all of these collectively contributed to its competence for mass testing. At the current stage, it cannot take the place of SARA-CoV-2 nucleic acid RT-PCR, but can be served as a complementary option for RT-PCR. The combination of RT-PCR and IgG-IgM combined test kit could provide further insight into SARS-CoV-2 infection diagnosis. \"},\n"," {'paper_id': '7ecc2bca3cfb5ebea94cd2601d6d3db60603452f',\n","  'section': 'Abstract',\n","  'text': 'Background Up to 9 March, 2020, 109577 patients were diagnosed with '},\n"," {'paper_id': '7ed2c0786eef3d39fa0435b461aa34ab454a4753',\n","  'section': 'Abstract',\n","  'text': 'The present status of COVID-19 is analyzed and the end of the disease is forecasted. The '},\n"," {'paper_id': '7ef979dc430e365ca94b84ec62bf65b8e96c24f3',\n","  'section': 'Abstract',\n","  'text': 'Background '},\n"," {'paper_id': '7f2df2e8a86da5ade48cd1e7d2a6829e93c2993b',\n","  'section': 'Abstract',\n","  'text': 'SARS-COV2 is infecting more than one million people. Knowing its genome and gene expressions is essential to understand the virus. Here, we propose a computational tool to detect the viral genomic variations as well as viral gene expressions from the sequenced amplicons with Nanopore devices of the multiplex reverse transcription polymerase chain reactions. We applied our tool to 11 samples of terminally ill patients, and discovered that seven of the samples are infected by two viral strains. The expression of viral genes ORF1ab gene, S gene, M gene, and N gene are high among most of the samples. While performing our tests, we noticed the abundances of MUC5B transcript segments to be consistently higher than those of other genes in the host. '},\n"," {'paper_id': '7f73d8a7944270ae9ba4a1793885786d36588bc3',\n","  'section': 'Abstract',\n","  'text': 'Governments around the world are responding to the novel coronavirus (COVID-19) pandemic 1 with unprecedented policies designed to slow the growth rate of infections. Many actions, such as closing schools and restricting populations to their homes, impose large and visible costs on society. In contrast, the benefits of these policies, in the form of infections that did not occur, cannot be directly observed and are currently understood through process-based simulations. [2][3][4] Here, we compile new data on 936 local, regional, and national anti-contagion policies recently deployed in the ongoing pandemic across localities in China, South Korea, Iran, Italy, France, and the United States (US). We then apply reduced-form econometric methods, commonly used to measure the effect of policies on economic growth, to empirically evaluate the effect that these anti-contagion policies have had on the growth rate of infections. In the absence of any policy actions, we estimate that early infections of COVID-19 exhibit exponential growth rates of roughly 45% per day. We find that anti-contagion policies collectively have had significant effects slowing this growth, although policy actions in the US appear to be too recent to have a substantial impact since the magnitude of these effects grows over time. Our results suggest that similar policies may have different impacts on different populations, but we obtain consistent evidence that the policy packages now deployed are achieving large and beneficial health outcomes. We estimate that, to date, current policies have already prevented or delayed on the order of eighty-million infections. These findings may help inform whether or when ongoing policies should be lifted or intensified, and they can support decision-making in the over 150 countries where COVID-19 has been detected but not yet achieved high infection rates. 5 2 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': '7f8c673d136b9bc0aeda55e49cc72aacaa5688de',\n","  'section': 'Abstract',\n","  'text': 'During its first month, the recently emerged 2019 Wuhan novel coronavirus (2019-nCoV) has already infected many thousands of people in mainland China and worldwide and took hundreds of lives. However, the swiftly spreading virus also caused an unprecedentedly rapid response from the research community facing the unknown health challenge of potentially enormous proportions. Unfortunately, the experimental research to understand the molecular mechanisms behind the viral infection and to design a vaccine or antivirals is costly and takes months to develop. To expedite the advancement of our knowledge we leverage the data about the related coronaviruses that is readily available in public databases, and integrate these data into a single computational pipeline. As a result, we provide a comprehensive structural genomics and interactomics road-maps of 2019-nCoV and use these information to infer the possible functional differences and similarities with the related SARS coronavirus. All data are made publicly available to the research community at http://korkinlab.org/wuhan . '},\n"," {'paper_id': '7fb7c0c48b30e66dfaea5fdec16d999ed52d1ee0',\n","  'section': 'Abstract',\n","  'text': 'The novel coronavirus disease 2019 outbreak and Italy has caused 6088 cases and 41 deaths in Republic of Korea and 3144 cases and 107 death in Italy by 5 March 2020. We modeled the transmission process in Republic of Korea and Italy with a stochastic model and estimated the basic reproduction number R0 as 2.6 (95% CI: 2.3-2.9) or 3.2 (95% CI: 2.9-3.5) in Republic of Korea, under the assumption that the exponential growth starting on 31 January or 5 February 2020, and 2.6 (95% CI: 2.3-2.9) or 3.3 (95% CI: 3.0-3.6) in Italy, under the assumption that the exponential growth starting on 5 February or 10 February 2020. Estimates of dispersion term (k) were 10 (95% CI: 5-56) or 22 (95% CI: 8-61) in Republic of Korea, and 13 (95% CI: 5-61) or 37 (95% CI: 13-61) in Italy, and all of which imply few super-spreading events. '},\n"," {'paper_id': '7fd337f16780aba8f1e599ce7516dc9b1d80a546',\n","  'section': 'Abstract',\n","  'text': 'To study the neurological manifestations of patients with coronavirus disease 2019 . DESIGN Retrospective case series Three designated COVID-19 care hospitals Two hundred fourteen hospitalized patients with laboratory confirmed diagnosis of severe acute respiratory syndrome from coronavirus 2 (SARS-CoV-2) infection. Data were collected from 16 January 2020 to 19 February 2020. Clinical data were extracted from electronic medical records and reviewed by a trained team of physicians. Neurological symptoms fall into three categories: central nervous system (CNS) symptoms or diseases (headache, dizziness, impaired consciousness, ataxia, acute cerebrovascular disease, and epilepsy), peripheral nervous system (PNS) symptoms (hypogeusia, hyposmia, hypopsia, and neuralgia), and skeletal muscular symptoms. Data of all neurological symptoms were checked by two trained neurologists. Of 214 patients studied, 88 (41.1%) were severe and 126 (58.9%) were non-severe patients. Compared with non-severe patients, severe patients were older (58.7 ± 15.0 years vs 48.9 ± 14.7 years), had more underlying disorders (42 [47.7%] vs 41 All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint (which was not peer-reviewed) is . https://doi.org/10.1101/2020.02.22.20026500 doi: medRxiv preprint [32.5%]), especially hypertension (32 [36.4%] vs 19 [15.1%]), and showed less typical symptoms such as fever (40 [45.5%] vs 92 [73%]) and cough (30 [34.1%] vs 77 [61.1%]). Seventy-eight (36.4%) patients had neurologic manifestations. More severe patients were likely to have neurologic symptoms (40 [45.5%] vs 38 [30.2%]), such as acute cerebrovascular diseases (5 [5.7%] vs 1 [0.8%]), impaired consciousness (13 [14.8%] vs 3 [2.4%]) and skeletal muscle injury (17 [19.3%] vs 6 [4.8%]). '},\n"," {'paper_id': '7fdd33661d188e3ea250e99f2752d705b5467ef9',\n","  'section': 'Abstract',\n","  'text': 'COVID-19 outbreak is still threatening the public health. Therefore, in the middle of the pandemic, all kind of knowledge on SARS-CoV-2 may help us to find the solution. Determining the 3D structures of the proteins involved in host-pathogen interactions are of great importance in the fight against infection. Besides, post-translational modifications of the protein on 3D structure should be revealed in order to understand the protein function since these modifications are responsible for the host-pathogen interaction. Based on these, we predicted O-glycosylation and phosphorylation positions using full amino acid sequence of S1 protein. Candidate positions were further analyzed with enzyme binding activity, solvent accessibility, surface area parameters and the positions determined with high accuracy rate were used to design full 3D glycoprotein structure of the S1 protein using carbohydrate force field. In addition, the interaction between the C-type lectin CD209L and α -mannose residues was examined and carbohydrate recognition positions were predicted. We suggest these positions as a potential target for the inhibition of the initial binding of SARS-CoV-2 S1 protein to the host cell. '},\n"," {'paper_id': '8044e283ef3a46bf6d3045675a9652401681defd',\n","  'section': 'Abstract',\n","  'text': 'The spread of COVID-19 is posing an unprecedented threat to health systems worldwide 1 . The fast propagation of the disease combined with the existence of covert contagions by asymptomatic individuals make the controlling of this disease particularly challenging. The key parameter to track the progression of the epidemics is the effective reproduction number R, defined as the number of secondary infections generated by an infected individual 2 . The suppression of the epidemics is directly related to this value, and is attained when R < 1. Here, we find an analytical expression for R as a function of mobility restrictions and confinement measures, using an epidemic model tailored for COVID-19. '},\n"," {'paper_id': '80747f397615967a430214adcd1541728934daae',\n","  'section': 'Abstract',\n","  'text': \"Background: Incidence in children was much less than in adults during the COVID-19 outbreak. Sports and entertainment events were canceled (VEC) in Japan for two weeks during 26 February -13 March. Most schools were closed (SC). Object: We construct a susceptible-infected-recovered model using three age classes and estimate the basic reproduction number (R 0 ) and protection level among children simultaneously. Then we simulate SC and VEC effects. Method: We used data of patients with symptoms in Japan during 14 January to assess SC and VEC introduction. Effects of SC and VEC were incorporated into the model through change in the contact pattern or frequencies among age classes. : Results suggest R 0 as 2.86 [95%CI of 2.73, 2.97]. The protection level was estimated as 0.4 [0.2, 0.7]. SC and VEC can reduce the total number of patients significantly, by 6-7%. Discussion and Conclusion: The estimated R 0 was similar to that found from other studies in China and Japan. We found a significant protection level among children, and by effects of SC and VEC. Introduction All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. : medRxiv preprint presumed to be not community-acquired in Japan. Published information about COVID-19 patients with symptoms from the Ministry of Labour, Health and Welfare (MLHW) Japan was usually affected adversely by some delay because of uncertainty during onset to visiting a doctor or in the timing of a physician's suspicion of COVID-19. Therefore, published data of patients must be adjusted at least a few days. To adjust the data, we applied the following regression. We denote Xt-k|t as the number of patients for whom the onset date was t-k published on day t. The dependent variables are the degree of reporting delay, where k>m in several m and k. Here, m denotes the difference of the publishing dates between the two published. Date t represents the publishing date of the latest publishing. The explanatory variables were 1/k, 1/m, and 1/km. The degree of reporting delay was estimated as [estimated coefficient of constant term] + [estimated coefficient of 1/k]/k, when m was sufficiently large and time had passed. Therefore, this estimated degree of reporting delay multiplied by the latest published data is expected to be a prediction of the number of patients for whom the onset date was t-k. We used this adjusted number of patients in the latest few days including those after VEC was adopted. We used published data of 2, 5, 6 and 9-17 March, 2020 provided by MLHW [1]. First, we estimated the joint distribution of R 0 and the protection rate in children in All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. \"},\n"," {'paper_id': '80a77923cc53e58abc35b1b8fcb56d6339055206',\n","  'section': 'Abstract',\n","  'text': '19 Objectives: Development and validation of a single-step and accurate reverse transcriptase loop-20 mediated isothermal amplification technique (RT-LAMP) for rapid identification of SARS COV-2 21 relative to commercial quantitative reverse transcriptase real-time PCR (qRT-PCR) assays to allow 22 prompt initiation of proper medical care and containment of virus spread. 23 24 All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '80df1ce37238d763d313bcd0e673716c7c4458c1',\n","  'section': 'Abstract',\n","  'text': 'With respect to the asymptomatic transmission characteristics of the novel coronavirus that appeared in 2019 (COVID-19), a susceptible-asymptomatic-infected-recovered-death (SAIRD) model that considered human mobility was constructed in this study. The dissemination of COVID-19 was simulated using computational experiments to identify the mechanisms underlying the impact of city and residential lockdowns on controlling the spread of the epidemic. Results: The implementation of measures to lock down cities led to higher mortality rates in these cities, due to reduced mobility. Moreover, implementing city lockdown along with addition of hospital beds led to improved cure and reduced mortality rates. Stringent implementation and early lockdown of residential units effectively controlled the spread of the epidemic, and reduced the number of hospital bed requirements. Collectively, measures to lock down cities and residential units should be taken to prevent the spread of COVID-19. In addition, medical resources should be increased in cities under lockdown. Implementation of these measures would reduce the spread of the virus to other cities and allow appropriate treatment of patients in cities under lockdown. '},\n"," {'paper_id': '811338b8a1cbfa4bee7c50f6358230fe72f479ba',\n","  'section': 'Abstract',\n","  'text': '1 Background: With new cases of COVID-19 surging around the world, many countries have to prepare for mov-2 ing beyond the containment phase. Prediction of the effectiveness of non-case-based interventions for mitigating, 3 delaying or preventing the epidemic is urgent, especially for countries affected by the ongoing seasonal influenza 4 activity. Methods: We developed a transmission model to evaluate the impact of self-imposed prevention measures 6 (handwashing, mask-wearing, and social distancing) due to the spread of COVID-19 awareness and of short-7 term government-imposed social distancing on the peak number of diagnoses, attack rate and time until the 8 peak number of diagnoses. 9 Findings: For fast awareness spread in the population, self-imposed measures can significantly reduce the attack 10 rate, diminish and postpone the peak number of diagnoses. A large epidemic can be prevented if the efficacy 11 of these measures exceeds 50%. For slow awareness spread, self-imposed measures reduce the peak number 12 of diagnoses and attack rate but do not affect the timing of the peak. Early implementation of short-term 13 government interventions can only delay the peak (by at most 7 months for a 3-month intervention). 14 Interpretation: Handwashing, mask-wearing and social distancing as a reaction to information dissemination 15 about COVID-19 can be effective strategies to mitigate and delay the epidemic. We stress the importance of 16 rapidly spreading awareness on the use of these self-imposed prevention measures in the population. Early-17 initiated short-term government-imposed social distancing can buy time for healthcare systems to prepare for 18 an increasing COVID-19 burden. Evidence to date suggests that containment of SARS-CoV-2 using quarantine, travel restrictions, isolation of symp-26 tomatic cases, and contact tracing may need to be supplemented by other interventions. Given its rapid spread 27 across the world and immense implications for public health, it is urgent to understand whether non-case-based 28 interventions can mitigate, delay or even prevent a COVID-19 epidemic. One such strategy is a broader-scale 29 contact rate reduction enforced by governments which was used during previous outbreaks, e.g., the 1918 influenza 30 pandemic and the 2009 influenza A/H1N1 pandemic in Mexico. Alternatively, governments and media may stimu-31 late self-imposed prevention measures (handwashing, mask-wearing, and social distancing) by generating awareness 32 about COVID-19, especially when economic and societal consequences are taken into account. Both of these strate-33 gies may have a significant impact on the outbreak dynamics. Currently, there are no comparative studies that 34 investigate their viability for controlling a COVID-19 epidemic. Added value of this study 36 Using a transmission model parameterized with current best estimates of epidemiological parameters, we evaluated 37 the impact of handwashing, mask-wearing, and social distancing due to COVID-19 awareness and of government-38 imposed social distancing on the peak number of diagnoses, attack rate, and time until the peak number of diagnoses. 39 We show that a short-term (1-3 months) government intervention initiated early into the outbreak can only delay 40 the peak number of diagnoses but neither alters its magnitude nor the attack rate. Our analyses also highlight the 41 importance of spreading awareness about COVID-19 in the population, as the impact of self-imposed measures is 42 strongly dependent on it. When awareness spreads fast, simple self-imposed measures such as handwashing are more 43 effective than short-term government intervention. Self-imposed measures do not only diminish and postpone the 44 peak number of diagnoses, but they can prevent a large epidemic altogether when their efficacy is sufficiently high 45 (above 50%). Qualitatively, these results will aid public health professionals to compare and select interventions 46 for designing effective outbreak control policies. Implications of all available evidence 48 Our results highlight that dissemination of evidence-based information about effective prevention measures (hand-49 washing, mask-wearing, and self-imposed social distancing) can be a key strategy for mitigating and postponing a 50 COVID-19 epidemic. Government interventions (e.g., closing schools and prohibiting mass gatherings) implemented 51 early into the epidemic and lasting for a short-time can only buy time for healthcare systems to prepare for an 52 '},\n"," {'paper_id': '8137e0c8ebd136c177809eb5331cf531a8552ff7',\n","  'section': 'Abstract',\n","  'text': 'We investigate the accumulated microbial and autoantigen antibody repertoire in adult-onset dermatomyositis patients sero-positive for TIF1γ (TRIM33) autoantibodies. We use an untargeted high-throughput approach which combines immunoglobulin disease-specific epitope-enrichment and identification of microbial and human antigens. Increased microbial diversity was observed in dermatomyositis. Viruses were over-represented and species of the Poxviridae family were significantly enriched. The autoantibodies identified recognised a large portion of the human proteome, including interferon regulated proteins; these proteins were clustered in specific biological processes. Apart from TRIM33, autoantibodies against eleven further TRIM proteins, including TRIM21, were identified. Some of these TRIM proteins shared epitope homology with specific viral species including poxviruses. Our data suggest antibody accumulation in dermatomyositis against an expanded diversity of microbial and human proteins and evidence of non-random targeting of specific signalling pathways. Our findings indicate that molecular mimicry and epitope spreading events may play a significant role in the pathogenesis of dermatomyositis. '},\n"," {'paper_id': '815745bf1b522d33fd7371cc9a6561a2a93ef87e',\n","  'section': 'Abstract',\n","  'text': \"Chinese government has taken strong measures in response to the epidemic of new coronavirus (2019-nCoV) from Jan.23, 2020. The number of confirmed infected individuals are still increasing rapidly. Estimating the accurate infected population and the future trend of epidemic spreading under control measures is significant and urgent. There have been reports external icon of spread from an infected patient with no symptoms to a close contact, which means the incubation individuals may has the possibility of infectiousness. However, the traditional transmission model, Susceptible-Exposed-Infectious-Recovered (SEIR) model, assumes that the exposed individual is being infected but without infectiousness. Thus, the estimating infected populations based on SEIR model from the existing literatures seems too far more than the official reported data. Here, we inferred that the epidemic could be spread by exposed (incubation) individuals. Then, we provide a new Exposed-identified-Recovered (EIR) model, and simulated the epidemic spreading processes from free propagation phase to extremely control phase. Then, we estimate of the size of the epidemic and forecast the future development of the epidemics under strong prevention interventions. According to the spread characters of 2019-nCov, we construct a novel EIR compartment system dynamics model. This model integrates two phases of the epidemic spreading: before intervention and after intervention. We assume that 2019-nCov is firstly spread without intervention then the government started to take strong quarantine measures. Use the latest reported data from National Health Commission of the People's Republic of China, we estimate the basic parameters of the model and the basic reproduction number of 2019-nCov. Then, based on this model, we simulate the future spread of the epidemics. Both the infected population and the spreading trend of 2019-nCov under different prevention policy scenarios are estimated. The epidemic spreading trends under different quarantine rate and action starting date of prevention policy are simulated and compared. In our baseline scenario, the government has taken strict prevention actions, and the estimate numbers fit the official numbers very well. Simulation results tells that, if the prevention measures are relaxed or the action starting date of prevention measures is later than Jan. 23, 2020, the peak of identified individuals would be greatly increased, and the elimination date also would be delayed. We estimate the reproductive number for 2019-nCoV was 2.7. And simulation of the baseline scenario tells that, the peak infected individuals will be 49093 at Feb.16, 2020 and the epidemic spreading will be eliminated at the end of March 2020. The simulation results also tell that the quarantine rate and the starting date of intervention action policy have great effect on the epidemic spreading. Specifically, if the quarantine rate is reduced from 100% to less than 63%, which is the threshold of the quarantine rate to control the epidemic . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . \"},\n"," {'paper_id': '815f653470c57a95b92e3857d8f5d038b4000d99',\n","  'section': 'Abstract',\n","  'text': \"Abstract ) is the causative agent of a respiratory disease with a high case fatality rate. During the formation of the coronaviral replication/transcription complex (RTC), essential steps include processing of the conserved polyprotein nsp7-10 region by the main protease M pro and subsequent complex formation of the released nsp's. Here, we analyzed processing of the coronavirus nsp7-10 region using native mass spectrometry showing consumption of substrate, rise and fall of intermediate products and complexation. Importantly, there is a clear order of cleavage efficiencies, which is influenced by the polyprotein tertiary structure. Furthermore, the predominant product is an nsp7+8(2:2) hetero-tetramer with nsp8 scaffold. In conclusion, native MS, opposed to other methods, can expose the processing dynamics of viral polyproteins and the landscape of protein interactions in one set of experiments. Thereby, new insights into protein interactions, essential for generation of viral progeny, were provided, with relevance for development of antivirals. \"},\n"," {'paper_id': '81ea996d3e783bdc0a4ff11e23eec5bbac083fad',\n","  'section': 'Abstract',\n","  'text': 'As reported by Iranian governments, the first cases of coronavirus infections confirmed in Qom, Iran on February 19, 2020 (30 Bahman 1398). The number of identified cases afterward increased rapidly and the novel coronavirus spread to all provinces of the country. This study aimed to fit an epidemic model to the reported cases data to estimate the basic reproduction number (ܴ ) of COVID-19 in Iran. We used data from February 21, 2020, to March 18, 2020, on the number of cases reported by Iranian governments and we employed the SIR (Susceptible-Infectious-Removed) epidemic spreading model to fit the transmission model to the reported cases data by tuning the parameters in order to estimate the basic reproduction number of COVID-19 in Iran. The value of reproduction number was estimated 4.86 in the first week and 4.5 in the second and decreasing from 4.29 to 2.1 in the third and fourth weeks. The results indicate that the basic reproduction number of COVID-19 is significantly larger than one. Based on the results, it is necessary to implement social distancing and a strict restriction on travelling to control the outbreak especially on Nowruz (Persian New Year). '},\n"," {'paper_id': '81f08d610ed4c61197c816bdbe5f23b14670dd47',\n","  'section': 'Abstract',\n","  'text': 'BACKGROUND: In December 2019, a novel strain of SARS-CoV-2 emerged in Wuhan, China. Since then, the city of Wuhan has taken unprecedented measures and efforts in response to the outbreak. China, to assess their effects on the progression of the outbreak. We included the latest estimates of epidemic parameters from a transmission model fitted to data on local and internationally exported cases from Wuhan in the age-structured epidemic framework. Further, we looked at the age-distribution of cases. Lastly, we simulated lifting of the control measures by allowing people to return to work in a phased-in way, and looked at the effects of returning to work at different stages of the underlying outbreak. '},\n"," {'paper_id': '8213acea5f1a592dae49adae07380fea8203910d',\n","  'section': 'Abstract',\n","  'text': 'We model the COVID-19 coronavirus epidemic in South Korea, Italy, France, and Germany. We use early reported case data to predict the cumulative number of reported cases to a final size. The key features of our model are the timing of implementation of major public policies restricting social movement, the identification and isolation of unreported cases, and the impact of asymptomatic infectious cases. '},\n"," {'paper_id': '828759dd16c404fc06efc97cf6473c4fc45abb78',\n","  'section': 'Abstract',\n","  'text': 'The recent spread of severe acute respiratory syndrome coronavirus (SARS-CoV-2) exemplifies the critical need for accurate and rapid diagnostic assays to prompt public health actions. Currently, several quantitative reverse-transcription polymerase chain reaction (qRT-PCR) assays are being used by clinical, research, and public health laboratories for rapid detection of the virus. However, it is currently unclear if results from different tests are comparable. Our goal was to evaluate the primer-probe sets used in four common diagnostic assays available on the World Health Organization (WHO) website. To facilitate this effort, we generated RNA transcripts to create standards and distributed them to other laboratories for internal validation. We then used these RNA transcript standards, full-length SARS-CoV-2 RNA, and RNA-spiked mock samples to determine analytical efficiency and sensitivity of nine primer-probe sets. We show that all primer-probe sets can be used to detect SARS-CoV-2, but there are clear differences in the ability to differentiate between true negatives and positives with low amounts of virus. Adding to this, many primer-probe sets, including the \"N2\" and \"N3\" sets issued by the US Centers for Disease Control and Prevention, have background amplification with SARS-CoV-2-negative nasopharyngeal swabs, which may lead to inconclusive results. Our findings characterize the limitations of commonly used primer-probe sets and can assist other laboratories in selecting appropriate assays for the detection of SARS-CoV-2. '},\n"," {'paper_id': '82ac6362753348f6bd9a46989aaa636f0911d93a',\n","  'section': 'Abstract',\n","  'text': 'words) Main text (2447 words) Background: By early April 2020, the COVID-19 pandemic had infected nearly one million people and had spread to nearly all countries worldwide. It is essential to understand where and how SARS-CoV-2 is transmitted. Methods: Case reports were extracted from the local Municipal Health Commissions of 320 prefectural cities (municipalities) in China, not including Hubei province, between 4 January and 11 February 2020. We identified all outbreaks involving three or more cases and reviewed the major characteristics of the enclosed spaces in which the outbreaks were reported and associated indoor environmental issues. Results: Three hundred and eighteen outbreaks with three or more cases were identified, involving 1245 confirmed cases in 120 prefectural cities. We divided the venues in which the outbreaks occurred into six categories: homes, transport, food, entertainment, shopping, and miscellaneous. Among the identified outbreaks, 53·8% involved three cases, 26·4% involved four cases, and only 1·6% involved ten or more cases. Home outbreaks were the dominant category (254 of 318 outbreaks; 79·9%), followed by transport (108; 34·0%; note that many outbreaks involved more than one venue category). Most home outbreaks involved three to five cases. We identified only a single outbreak in an outdoor environment, which involved two cases. Conclusions: All identified outbreaks of three or more cases occurred in an indoor environment, which confirms that sharing indoor space is a major SARS-CoV-2 infection risk. '},\n"," {'paper_id': '8319d02f4d2aed9d4a5fec3ad09c9570491e4936',\n","  'section': 'Abstract',\n","  'text': 'Background: Coronavirus disease 2019 (COVID-19) '},\n"," {'paper_id': '83323659de2f2d5ed3ca6ada7cf137a8a17bb014',\n","  'section': 'Abstract',\n","  'text': 'The main protease of coronaviruses and the 3C protease of enteroviruses share a similar active-site architecture and a unique requirement for glutamine in the P1 position of the substrate. Because of their unique specificity and essential role in viral polyprotein processing, these proteases are suitable targets for the development of antiviral drugs. In order to obtain nearequipotent, broad-spectrum antivirals against alphacoronaviruses, betacoronaviruses, and enteroviruses, we pursued structure-based design of peptidomimetic a-ketoamides as inhibitors of main and 3C proteases. Six crystal structures of protease:inhibitor complexes were determined as part of this study. Compounds synthesized were tested against the recombinant proteases as well as in viral replicons and virus-infected cell cultures; most of them were not cell-toxic. Optimization of the P2 substituent of the aketoamides proved crucial for achieving near-equipotency against the three virus genera. The best near-equipotent inhibitors, 11u (P2 = cyclopentylmethyl) and 11r (P2 = cyclohexylmethyl), display low-micromolar EC50 values against enteroviruses, alphacoronaviruses, and betacoronaviruses in cell cultures. In Huh7 cells, 11r exhibits three-digit picomolar activity against Middle East Respiratory Syndrome coronavirus. We chose the chemical class of peptidomimetic a-ketoamides to assess the feasibility of achieving antiviral drugs targeting coronaviruses and enteroviruses with near-equipotency. Here we describe the structure-based design, synthesis, and evaluation of inhibitory activity of a series of compounds with broad-spectrum activities afforded by studying the structure-activity relationships mainly with respect to the P2 position of the peptidomimetics. One of the compounds designed and synthesized exhibits excellent activity against MERS-CoV. RESULTS Our efforts to design novel a-ketoamides as broad-spectrum inhibitors of coronavirus M pro s and enterovirus 3C pro s started with a detailed analysis of the following crystal structures of unliganded target enzymes: SARS-CoV M pro (ref. 25-27; PDB entries 1UJ1, 2BX3, 2BX4); bat coronavirus HKU4 M pro as a surrogate for the closely related MERS-CoV protease (our unpublished work (Ma, Xiao et al.; PDB entry 2YNA; see also ref. 27); HCoV-229E M pro (ref. 27,28; PDB entry: 1P9S); Coxsackievirus B3 3C pro (our unpublished work; Tan et al., PDB entry 3ZYD); enterovirus D68 3C pro (ref. 29; PDB entry: 3ZV8); and enterovirus A71 3C pro (ref. 30; PDB entry: 3SJK). During the course of the present study, we determined crystal structures of a number of lead a-ketoamide compounds in complex with SARS-CoV M pro , HCoV-NL63 M pro , and CVB3 3C pro , in support of the design of improvements in the next round of lead optimization. Notably, unexpected differences between alpha-and betacoronavirus M pro were found in this study. The structural foundation of these was elucidated in detail in a subproject involving the M pro of HCoV NL63; because of its volume, this work will be published separately (Zhang et al., in preparation) and only some selected findings are referred to here. The main protease of the newly discovered coronavirus linked to the Wuhan outbreak of respiratory illness is 96% identical (98% similar) in amino-acid sequence to that of SARS-CoV M pro (derived from the RNA genome of BetaCoV/Wuhan/IVDC-HB-01/2019, Genbank accession code: MN908947.2; http://virological.org/t/initialgenome-release-of-novel-coronavirus/319, last accessed on January 11, 2020), so all results reported here for inhibition of SARS-CoV will most likely also apply to the new virus. '},\n"," {'paper_id': '8349823092836fe397a59e38615d1491423dbe70',\n","  'section': 'Abstract',\n","  'text': 'The focused drug repurposing of known approved drugs (such as lopinavir/ritonavir) has been reported failed for curing SARS-CoV-2 infected patients. It is urgent to generate new chemical entities against this virus. As a key enzyme in the life-cycle of coronavirus, the 3C-like main protease (3CL pro or M pro ) is the most attractive for antiviral drug design. Based on a recently solved structure (PDB ID: 6LU7), we developed a novel advanced deep Q-learning network with the fragment-based drug design (ADQN-FBDD) for generating potential lead compounds targeting SARS-CoV-2 3CL pro . We obtained a series of derivatives from those lead compounds by our structure-based optimization policy (SBOP). All the 47 lead compounds directly from our AI-model and related derivatives based on SBOP are accessible in our molecular library at https://github.com/tbwxmu/2019-nCov. These compounds can be used as potential candidates for researchers in their development of drugs against SARS-CoV-2. author/funder. All rights reserved. No reuse allowed without permission. : bioRxiv preprint disease (COVID-19) worldwide. 1 As of March 2, 2020, more than 90,000 people have been infected by SARS-CoV-2 and more than 3000 people have been reported dead according to Johns Hopkins Coronavirus map tracker. 2 The numbers of infection and death are still increasing. To face the considerable threat of SARS-CoV-2, it is urgent to develop new inhibitors or drugs against this deadly virus. Unfortunately, since the outbreak of severe acute respiratory syndrome (SARS) eighteen years ago, there has been no approved treatment against the SARS coronavirus (SARS-CoV), 3 which is similar to SARS-CoV-2. Repurposing potential drugs such as lopinavir and ritonavir also failed to SARS-CoV-2 injected patients. 4 Structure-based antiviral drug design with a new artificial intelligence algorithm may represent a more helpful approach to get the SARS-CoV-2 targeted inhibitors or drugs. Thanks to the prompt efforts of many researchers, we have several pieces of important information about this vital virus genome and protein structures. We now know that the non-structural protein 5 (Nsp5) is the main protease (M pro ) of SARS-CoV-2 and it is a cysteine protease, which also been called \"3C-like protease\" (3CL pro ). Moreover, we know that the 3D structure of 3CL pro is very similar to SARS-CoV with a sequence identity of >96% and 3D structure superposition RMSDCα of 0.44 Å as shown in Figures S1 and S2. 3CL pro has been reported as an attractive target for developing anti-coronaviral drugs: 1) this protease is highly conserved in both sequences and 3D structures; 5 2) 3CL pro is a key enzyme for related virus (including SARS and SARS-CoV-2) replication; 3) it only exists in the virus, not in humans. Developing specific antiviral drugs targeting 3CL pro of the specific virus has shown significant success; for example, both approved drugs lopinavir and ritonavir can completely occupy the substrate-binding site of 3CL pro to break down the replication of human immunodeficiency virus (HIV). However, due to the large difference between HIV and SARS-CoV-2 3CL pro , lopinavir and ritonavir were validated ineffective for inhibiting SARS-Cov-2. 4 On the other hand, the substrate-binding site of 3CL pro is almost the same between the SARS-CoV-2 and SARS as Figure S3 presents. The developed potential inhibitors and drug-design experience targeting SARS-3CL pro may also be applicable to SARS-CoV-2. For author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '836190d366ba8e34c3a5df8a24d17a36fd503505',\n","  'section': 'Abstract',\n","  'text': 'The outbreaks of 2002/2003 SARS, 2012/2015 MERS and 2019/2020 Wuhan respiratory syndrome clearly indicate that genome evolution of an animal coronavirus (CoV) may enable it to acquire human transmission ability, and thereby to cause serious threats to global public health. It is widely accepted that CoV human transmission is driven by the interactions of its spike protein (S-protein) with human receptor on host cell surface; so, quantitative evaluation of these interactions may be used to assess the human transmission capability of CoVs. However, quantitative methods directly using viral genome data are still lacking. Here, we perform large-scale protein-protein docking to quantify the interactions of 2019-nCoV S-protein receptor-binding domain (S-RBD) with human receptor ACE2, based on experimental SARS-CoV S-RBD-ACE2 complex structure. By sampling a large number of thermodynamically probable binding conformations with Monte Carlo algorithm, this approach successfully identified the experimental complex structure as the lowest-energy receptor-binding conformations, and hence established an experiment-based strength reference for evaluating the receptor-binding affinity of 2019-nCoV via comparison with SARS-CoV. Our results show that this binding affinity is about 73% of that of SARS-CoV, supporting that 2019-nCoV may cause human transmission similar to that of SARS-CoV. Thus, this study presents a method for rapidly assessing the human transmission capability of a newly emerged CoV and its mutant strains, and demonstrates that post-genome analysis of protein-protein interactions may provide early scientific guidance for viral prevention and control. '},\n"," {'paper_id': '837056cb9cb01b2f15c39e7e342a3c47bfcc24cf',\n","  'section': 'Abstract',\n","  'text': 'The outbreak of coronavirus disease in China caused by SARS-CoV-2 virus 24 continually lead to worldwide human infections and deaths. It is currently no specific viral protein 25 targeted therapeutics yet. Viral nucleocapsid protein is a potential antiviral drug target, serving 26 multiple critical functions during the viral life cycle. However, the structural information of SARS-27 CoV-2 nucleocapsid protein is yet to be clear. Herein, we have determined the 2.7 Å crystal 28 structure of the N-terminal RNA binding domain of SARS-CoV-2 nucleocapsid protein. 29 Although overall structure is similar with other reported coronavirus nucleocapsid protein N-30 terminal domain, the surface electrostatic potential characteristics between them are distinct. 31 Further comparison with mild virus type HCoV-OC43 equivalent domain demonstrates a unique 32 potential RNA binding pocket alongside the b-sheet core. Complemented by in vitro binding 33 studies, our data provide several atomic resolution features of SARS-CoV-2 nucleocapsid 34 protein N-terminal domain, guiding the design of novel antiviral agents specific targeting to 35 SARS-CoV-2. 36 '},\n"," {'paper_id': '837c77d566651f559ed3f78c2e490213f99212e6',\n","  'section': 'Abstract',\n","  'text': '1. Bees, important pollinators, have declined significantly in recent decades, and human-11 induced changes to nutritional landscapes are partly responsible. Changes to nutritional 12 quality rather than quantity have been overlooked as a threat to bee health. Yet 13 knowledge of bee nutrition is currently largely restricted to adults of social species. 14 Larval stages, where most growth occurs, are relatively understudied -perhaps because 15 most social bees provision progressively and collectively, making nutrition difficult to 16 trace. 17 2. In mass-provisioning solitary bees (Osmia bicornis L.), we can manipulate and follow 18 larval nutrition, and thereby determine effects of changes in diet quality. Under the 19 Geometric Framework for Nutrition, we restricted larvae to 6 diets: 3 20 protein:carbohydrate ratios and 2 nutrient concentrations. We asked: (a) which diets 21 maximise body size and survival, (b) what consumption rules do larvae follow when 22 nutrients are imbalanced? Finally, (c) given a choice of complementary diets, are larvae 23 able to select a dietary balance? 24 3. Larvae pupated after consuming a fixed carbohydrate amount, but tolerated a wide range 25 of protein. Body size and survival were maximised on our lowest P:C ratio diet, and 26 '},\n"," {'paper_id': '83cca3779f8aeea47c5426e1d0fa65f1e9567e0c',\n","  'section': 'Abstract',\n","  'text': 'The G-quadruplex structure forming motifs have recently emerged as a novel therapeutic drug target in various human pathogens. Herein, we report three highly conserved G-quadruplex motifs (SE-PGQ -1, 2, and3) in genome of all the 412 strains of Salmonella enterica. Bioinformatics analysis inferred the presence of SE-PGQ-1 in the regulatory region of mgtA, presence of SE-PGQ-2 in the open reading frame of entA and presence of SE-PGQ-3 in the promoter region of malE and malK genes. The products of mgtA and entA are involved in transport and homeostasis of Mg 2+ and Fe 3+ ion and thereby required for bacterial survival in the presence of reactive nitrogen/oxygen species produced by the host macrophages, whereas, malK and malE genes are involved in transport of maltose sugar, that is one of the major carbon source in the gastrointestinal tract of human. The formation of stable intramolecular G-quadruplex structures by SE-PGQs was confirmed by employing CD, EMSA and NMR spectroscopy. Cellular studies revealed the inhibitory effect of 9-amino acridine on Salmonella enterica growth. Next, CD melting analysis demonstrated the stabilizing effect of 9-amino acridine on SE-PGQs. Further, polymerase inhibition and RT-qPCR assays emphasize the biological relevance of predicted G-quadruplex in the expression of PGQ possessing genes and demonstrate the G-quadruplexes as a potential drug target for the devolping novel therapeutics for combating Salmonella enterica infection. '},\n"," {'paper_id': '841e2b8df37c627db6013f7da95a3d9b7cff81f2',\n","  'section': 'Abstract',\n","  'text': 'The novel Coronavirus disease is being considered as the most serious health threat that the world has never witnessed in the recent times and significantly affecting the daily routine of mankind by emerging as a global pandemic. Yet, as there is no treatment nor a vaccine that was approved so far, universal safety precautions (USPs) and mitigating strategies are the only way to deal with this emergency crisis. However, knowledge and beliefs towards USPs among the general public in countries such as India with a large population are lacking. Methods: A prospective, cross-sectional, web-based online survey was conducted among the general public in India during March 2020. A 20-item self-administered survey questionnaire was developed and randomly distributed among the public using google document forms through social media networks. Descriptive statistics were used in representing the study characteristics, and the Chi-square test was used in assessing the associations among the study variables with a p-value of < 0.05 was considered as statistically significant. Results: Of 1287 participants, 1117 have given their consent of willingness and completed the questionnaire with a response rate of 86.8%. The mean age of the study participants was 28.8 ± 10.9 years, where the majority of them belong to the age category <25 years, and sex was equally distributed. Based upon the socio-demographic information, the majority were post-graduates (32.9%), professional job holders (45%) and belonged to the upper-middle (40%) economic class. Overall, the knowledge and beliefs towards USPs and mitigating strategies among participants varied between moderate to high, with statistically significant associations with their sociodemographic characteristics. Although the knowledge and beliefs of the general public in India towards USPs are encouraging, there is a need for long-term educational interventions as the dynamics and severity of COVID-19 have been changing day-by-day rapidly. The findings of this study could guide the public health authorities in making and implementing decisions to combat this pandemic. '},\n"," {'paper_id': '849f38fd380726f9b8065a51c55e45ae7d0118cf',\n","  'section': 'Abstract',\n","  'text': 'The growth and development of Covid-19 transmission have significantly cut the attention of many societies, particularly Iran that has been struggling with this contagious, infectious disease since late February 2020. In the present study, the known SIR model was used for the dynamics of an epidemic to provide a suitable assessment of the COVID-19 virus epidemic in Iran. The epidemic curve and SIR model parameters were obtained with the use of Iran statistical data. The recovered people were considered alongside the official number of confirmed victims as the reliable long-time statistical data of Iran. The results offered many important predictions of the COVID-19 virus epidemic such as realistic number of victims, infection rate, peak time, and other characteristics. '},\n"," {'paper_id': '84c3cb4cff4551bf95b6f79bd6c52525faee0a44',\n","  'section': 'Abstract',\n","  'text': 'The African swine fever virus (ASFV) has severely influenced the swine industry of the world. Unfortunately, there is no effective antiviral drug or vaccine against the virus until now. Identification of new anti-ASFV drugs is urgently needed. Here, an up-to-date set of protein-protein interactions (PPIs) between ASFV and swine were curated by integration of PPIs from multiple sources. Thirty-two swine proteins were observed to interact with ASFVs and were defined as AIPs. They were found to play a central role in the swine PPI network, with significant larger degree, betweenness and smaller shortest path length than other swine proteins. Some of AIPs also interacted with several other viruses and could be taken as potential targets of drugs for broad-spectrum effect, such as HSP90AB1. Finally, the antiviral drugs which targeted AIPs and ASFV proteins were predicted. Several drugs with either broad-spectrum effect or high specificity on AIPs were identified, such as Polaprezinc. This work could not only deepen our understanding towards the ASFV-swine interactions, but also help for the development of effective antiviral drugs against the ASFVs. author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '85239989835b4d02c0cdece69d0481b641a7d38c',\n","  'section': 'Abstract',\n","  'text': \"Human coronaviruses OC43 and HKU1 are respiratory pathogen of zoonotic origin that have gained 23 worldwide distribution. OC43 apparently emerged from a bovine coronavirus (BCoV) spill-over. All 24 three viruses attach to 9-O-acetylated sialoglycans via spike protein S with hemagglutinin-esterase HE 25 acting as a receptor-destroying enzyme. In BCoV, an HE lectin domain promotes esterase activity 26 towards clustered substrates. OC43 and HKU1, however, lost HE lectin function as an adaptation to 27 humans. Replaying OC43 evolution, we knocked-out BCoV HE lectin function and performed forced 28 evolution-population d ynamics analysis. Loss of HE receptor-binding selected for second-site 29 mutations in S, decreasing S binding affinity by orders of magnitude. Irreversible HE mutations 30 selected for cooperativity in virus swarms with low-affinity S minority variants sustaining propagation 31 of high-affinity majority phenotypes. Salvageable HE mutations induced successive second-site 32 substitutions in both S and HE. Apparently, S and HE are functionally interdependent and co-evolve to 33 optimize the balance between attachment and release. This mechanism of glycan-based receptor 34 usage, entailing a concerted, fine-tuned activity of two envelope protein species, is unique among 35 CoVs, but reminiscent of that of influenza A viruses (IAVs). Apparently, general principles fundamental 36 to virion-sialoglycan interactions prompted convergent evolution of two important groups of human 37 and animal pathogens. The subfamily Orthocoronavirinae comprises a group of enveloped positive-strand RNA viruses of 40 clinical and veterinary significance. Adding to the socio-economic impact of coronaviruses (CoVs) 41 already extant in humans and livestock, the emergence of 'new' CoVs through cross species 42 transmission poses an ever-looming threat to public health, animal health, and food production. 43 Seven coronaviruses are known to infect humans, but not all of them have become established. The 44 introduction of SARS CoV in 2002 from horseshoe bats with masked palm civets as incidental transient 45 hosts, was rapidly contained through quarantine measures 1 . MERS CoV, natural to dromedary camels, 46 causes a classical zoonotic infection with limited human-to-human spread 2 . December 2019, a 47 member of the species Severe acute respiratory syndrome related coronavirus (SARS-CoV), called 48 SARS-CoV-2 and 79.5% identical to the 2002 SARS CoV variant, emerged in Wuhan, China 3,4 to progress 49 to full scale pandemicity. Chances are, SARS-CoV-2 will eventually become established in the human 50 population. Four other respiratory coronaviruses of zoonotic origin already did succeed in becoming true human 52 viruses with world-wide distribution 5-7 . Among them are HKU1 and OC43 (subgenus Embecovirus, 53 genus Betacoronavirus), related yet distinct viruses that arose from different zoonotic progenitors and 54 entered the human population independently. OC43 is far more related to bovine coronavirus (BCoV), 55 its presumptive ancestor, with early isolates sharing 97% genome identity 8,9 . Together with viruses of 56 swine, canines, equines and lagomorphs, OC43 and BCoV are considered host range variants of the 57 virus species Betacoronavirus-1 (collectively referred to as β1CoVs throughout) 7 . OC43 apparently 58 emerged 70 to 130 years ago from a single cross species transmission event that gave rise to a human-59 only virus [8][9][10] . Like OC43, other β1CoVs also exhibit host specificity 8,11 . While these observations attest 60 to the host promiscuity and zoonotic potential of embecoviruses and β1CoVs in particular, they are 61 also indicative for the existence of host barriers, the breaching of which selects for adaptive mutations 62 that result in host specialization and, ultimately, virus speciation. Conceivably, comparative studies of 63 author/funder. All rights reserved. No reuse allowed without permission. : bioRxiv preprint 4 BCoV and OC43 may identify factors that promote or restrict cross species transmission of CoVs and 64 thus further our understanding of the requirements for colonization of the human host. 65 Embecoviruses, OC43 and BCoV included, differ from other CoVs in that they encode two types of 66 surface projections. Homotrimeric 'peplomers' comprised of spike protein S and extending 20 nm 67 from the viral membrane, mediate virion attachment to entry receptors and membrane fusion 12 . 68 Interspersed are stubby 8-nm homodimeric projections comprised of the hemagglutinin-esterase 69 (HE) 13-15 , a dual function protein typically encompassing a receptor-binding lectin domain specific for 70 O-acetylated sialic acid (O-Ac-Sia) and a receptor destroying sialate-O-acetylesterase domain 16-20 . The 71 HE lectin domain contributes to virion attachment, but at the same time enhances sialate-O-72 acetylesterase activity towards clustered sialoglycotopes 11 . 73 Some embecoviruses, like mouse hepatitis virus (MHV) and related CoVs in rodents, attach to 4-or 9-74 O-acetylated sialosides (4-or 9-O-Ac-Sias) via HE 21-25 and to a proteinaceous entry receptor via S 26,27 . 75 Others, animal β1CoVs included, bind to 9-O-Ac-Sias via HE 28 but, remarkably, also via S 29,30 or, in the 76 case of human coronaviruses OC43 and HKU1, even exclusively via S 11,31,32 . 77 Structure function analyses of HE and S proteins have yielded a wealth of data on ligand binding, 78 substrate selection and protein-glycan interactions. The receptor binding sites (RBSs) of CoV HE lectin 79 domains and those in related proteins of toro-and influenza C/D viruses 22,33-35 differ in sequence and 80 structure yet conform to a common architectural design with a deep hydrophobic pocket ('P1') to 81 accommodate the crucial sialate-O-acetyl moiety, and an adjacent pocket or depression ('P2') to 82 accept the 5-N-acyl group 17,21,22,33 . Characteristically, P1 and P2 are separated by an aromatic side 83 chain and binding of the ligand is stabilized further through electrostatic protein-glycan interactions 84 typically involving distinctive Sia functions such as the Sia glycerol side chain, the 5-N-Acyl and/or the 85 Sia carboxylate. The RBS for 9-O-Ac-Sia in the S proteins of BCoV and OC43, identified by comparative 86 structural analysis 32 and confirmed by the cryo-EM holostructure of OC43 S 36 , conforms to this 87 blueprint. Moreover, this site is structurally and functionally conserved in HKU1 32 . : bioRxiv preprint 5 Much less is known about the functional relationship between S and HE, and the role of HE in particular 89 remains poorly understood. In MHV, HE expression is dispensable for replication and rapidly lost 90 during cell culture propagation 15 . Conversely, in β1CoVs, HE seems critical for infection. In OC43, loss 91 of HE-associated acetyl esterase activity abrogates the production of infectious virus and virus 92 dissemination in cell culture 37 . Moreover, acetyl-esterase inhibitors impede BCoV replication 19 , and 93 antibodies against HE neutralize the virus in vitro and in vivo 38-40 . Still, even among β1CoVs there are 94 differences in HE function apparently correlating with host specificity. Whereas HE lectin activity is 95 strictly maintained in BCoV 28 , OC43 lost this function through progressive accumulation of mutations 96 in the HE RBS, apparently as an adaptation to replication in the human respiratory tract 11 . 97 Nevertheless, isolates of either virus propagate in cultured cells. To better understand the 98 consequences of loss of HE lectin function as it occurred during OC43 and also HKU1 evolution, we 99 took a reverse genetics/forced evolution approach with BCoV as a model. The findings reveal that HE 100 and S are functionally interdependent and that the acquisition of HE by a proto-embecovirus allowed 101 its β1CoV descendants to adopt strategies for reversible virion-sialoglycan attachment, remarkably 102 similar to those of influenza A viruses. RESULTS 104 Disruption of HE lectin function selects for mutations in S. To study the role and importance of HE in 105 β1CoV propagation, we developed a reverse genetics system for BCoV strain Mebus (sFig. 1A) based 106 on targeted RNA recombination 41,42 . Recombinant 'wildtype' BCoVs (rBCoV) with parental type HE and 107 S, but with accessory ORF4a replaced by the Renilla luciferase gene (rBCoV-Rluc), were readily 108 generated upon seeding acceptor-virus-infected, donor RNA-transfected mouse LR7 cells onto 109 monolayers of feeder HRT18 cells. rBCoV-Rluc arose and within seven days grew to final titers 110 routinely obtained for wildtype BCoV (~10 8 TCID50/ml). 111 Generating BCoV-Rluc derivatives defective in HE lectin function proved more cumbersome. To 112 abolish the HE RBS, we substituted Phe 211 , which is key to ligand binding 17 (sFig. 1B), by Ala via two 113 nucleotide substitutions to block reversion. Mutant viruses were recovered eventually, but, in 3 out 114 of 4 successful trials, a multistep 160-hr rescue did not suffice and an additional 72-96 hr blind passage 115 was required (Fig. 1A) . 116 Sequence analysis of clonal virus populations, obtained by endpoint dilution, confirmed the presence 117 of the HE Phe 211 Ala substitution in all cases. Surprisingly, the purified viruses all suffered single site 118 mutations in S, clustering in domain S1 A (aa 15-302) in proximity of the RBS (Figs. 1A-C; sFig. 2). Two 119 of the trials yielded multiple S variants and some variants -Thr 83 Ile and Leu 89 Pro-emerged 120 independently in separate experiments (Fig. 1A) . The mutations map to three distinct S RBS elements 121 (Fig. 1C, sFig. 2C ; nomenclature according to 32 ). Ile 26 Ser and Asn 27 Ser locate in the S1 A β1 element 122 within the N-terminal L1-β1-L2 segment (aa 15-33) that walls pocket P1; P1 is crucial for ligand binding 123 as it accommodates the all-important sialate-9-O-acetyl moiety 32,36 . Moreover, in the OC43 S cryo-EM 124 holostructure, the Asn 27 side chain hydrogen bonds with the 9-O-acetyl carbonyl 36 . Leu 89 Pro in S1 A 125 element 3101 is immediately adjacent to Trp 90 . The latter is arguably the most critical residue in the 126 RBS as its indole side chain separates the P1 and P2 pockets, and its replacement precludes receptor-127 binding and virus infectivity 32 . Finally, Gly 82 Glu and Thr 83 Ile substitutions occurred in S1 A element β5 128 that interacts with the sialate carboxylate through hydrogen bonding with Lys 81 and Thr 83 . : bioRxiv preprint 7 As measured by solid phase lectin binding assay (sp-LBA) with S1 A -Fc fusion proteins and bovine 130 submaxillary mucin (BSM) as ligand, all mutations significantly reduced S binding to 9-O-Ac-Sia albeit 131 to widely different extents. S1 A -Fc binding affinities of the mutants were 500-fold (Asn 27 Ser) to more 132 than 10.000-fold (Ile 26 Ser; Gly 82 Glu) lower than that of parental BCoV S1 A -Fc (Fig. 1D) . 133 Loss of the HE lectin RBS would both abolish HE-assisted virion attachment to 9-O-Ac-Sia receptor 134 determinants as well as reduce virion-associated receptor-destroying sialate-O-acetylesterase activity 135 towards clustered glycotopes 11 . The findings suggest that the HE-Phe 211 Ala substitution creates a 136 fitness defect that can be alleviated not by increasing but by dramatically reducing the affinity of the 137 S RBS. Thus, the defect apparently is caused by loss of virion-associated RDE activity rather than by 138 loss of overall virion avidity. 139 140 downregulation of HE esterase activity consequential to loss of lectin function, could affect virus 141 propagation at two stages of the infectious cycle, namely virus release, which would require depletion 142 of intracellular and cell surface receptor pools, and (pre)attachment. We recently reported that G-143 deficient vesicular stomatitis (VSV) virions pseudotyped with wildtype BCoV S require exogenous HE 144 for efficient infection 32 . S1 A mutations that reduce S affinity inhibit infection, but, as we now show, 145 only when HE is present. VSV virions pseudotyped with low affinity mutant S proteins were less reliant 146 on or even inhibited by exogenous HE with decreasing S RBS affinity (Fig. 1E) . The findings provide 147 direct proof that the S1 A mutations act at the level of virion attachment and support the notion that 148 the S1 A mutations, selected for in HE-defective rBCoVs, restore reversibility of receptor binding by 149 lowering S affinity and thereby protect against inadvertent attachment to decoy receptors. 150 HE lectin-deficient recombinant BCoVs are genetically stable when grown in the presence of 151 exogenous receptor-destroying enzyme. To test whether loss of virion-associated RDE activity in 152 rBCoV-HE-F 211 A/S wt /Rluc might be compensated for by adding exogenous soluble HE to the culture 153 medium, we seeded infected/transfected LR7 cells onto HRT18 cell monolayers, supplemented the 154 author/funder. All rights reserved. No reuse allowed without permission. : bioRxiv preprint 157 1 μg/ml promoted virus growth ( Fig. 2A) . 158 To determine whether these conditions would allow isolation of rBCoV-HE-F 211 A without mutations in 159 S1 A , we performed targeted recombination and rescued recombinant viruses by 160 hr multistep 160 propagation as before, but now with culture supernatant supplemented with 100 ng/ml HE-Fc (Fig.   161   2B) . Sanger sequence analysis of RT-PCR amplicons showed that all viruses cloned by endpoint dilution 162 of the 160-hr stock (n=4) coded for mutant HE-Phe 211 Ala in combination with wildtype S1 A . To assess 163 the stability of clonal rBCoV-HE-F 211 A/S wt /Rluc, the virus population resulting from a subsequent 120-164 hr amplification in the presence of exogenous HE-Fc was analyzed by Next-Generation Sequencing 165 (NGS), which allows for the detection of low frequency mutants. Sequence variation in HE and S1 A was 166 distributed randomly and did not exceed background levels (<0.15%). More than 99.5% of the viruses 167 coded for HE-Phe 211 Ala, while preserving parental type S1 A (Fig. 2B) . 168 169 among S affinity variants. With a clonal, virtually pure stock of rBCoV-HE-F 211 A/S wt available, we 170 performed controlled forced evolution experiments. The virus was serially passaged involving three 171 consecutive 120-hr multistep propagation rounds in HRT18 cells but now in the absence of exogenous 172 HE-Fc, with the initial infection at multiplicity of infection (MOI) of 0.005 (Fig. 2C) . In trial 1, viral titers 173 in passage 1 (p1) increased only slowly to 3 x 10 4 and 2 x 10 4 TCID50/ml (measured with or without 174 exogenous HE-Fc, respectively). The withdrawal of exogenous RDE during viral passage immediately 175 selected for mutations in S1 A . Virus cloning by endpoint dilution of the 120-hr p1 sample yielded S RBS 176 mutants Asn 27 Ser, Thr 83 Ile and Ile 26 Ser (Fig. 2C ) -all three of which had been seen before (Fig. 1A) . 177 NGS analysis revealed the true complexity of the p1 population (Fig. 2C) and identified two additional 178 S1 A variants with substitutions -His 173 Tyr and Arg 197 Cys-more distal from the RBS (sFigs. 2D, 3A, B) . 179 His 173 Tyr also reduced the relative binding affinity of S1 A -Fc albeit less dramatically than the other 180 author/funder. All rights reserved. No reuse allowed without permission. : bioRxiv preprint 9 mutations, namely by 30-fold ( Table 1) . The Arg 197 Cys mutation seemingly falls in a separate category 181 as it reduced S1 A -Fc expression levels by more than 90% suggestive of defective folding (sFig. 3C). 182 Apparently, aberrant disulfide-bonding causes most of the fusion protein to be retained in the ER with 183 only a minor, presumably properly folded fraction slipping through to become secreted. All in all, the p1 population was comprised for virtually 100% of HE-Phe 211 Ala mutants, 40% of which 185 in combination with parental BCoV S, the remaining 60% with second-site mutations in S1 A ( Fig. 2C; 186 Table 1 ). Of the latter, the ultra-low affinity variant S1 A -Ile 26 Ser was the most abundant at 46% and 187 the Asn 27 Ser variant the least at less than 1%. However, upon a subsequent round of 120-hr multistep 188 propagation, the tables were turned with S1 A -Asn 27 Ser now comprising almost 40% of the p2 189 population and the Ile 26 Ser variant reduced to 0.7%. In addition, four other S1 A variants emerged. One 190 of these had a mutation in S1 A RBS loop L1, Val 29 Gly, and a relative binding affinity close to that of the 191 Asn 27 Ser mutant ( Fig. 2C; Table 1 ). We also identified at position 75 a second S1 A Cys-substitution 192 mutant, which like Arg 197 Cys, presumably disrupts the RBS through aberrant disulfide bonding (sFig. 193 3). Remarkably, two other S1 A variants arose with mutations -Arg 88 Thr and Pro 174 Leu-that affected 194 the relative binding affinity only modestly to 0.25 and 0.5 of that of wildtype S1 A -Fc, respectively 195 ( Table 1) . Even more remarkably, upon further passage these mutants increased to dominate the p3 196 population, effectively outcompeting variants with low affinity spikes as well as those with parental 197 spikes (Fig. 2C, D) . However, when the p1, p2 and p3 stocks were cloned by endpoint dilution in the 198 absence of exogenous HE-Fc, only virus variants with low affinity mutations in S1 A were isolated (Fig.  \"},\n"," {'paper_id': '854e3c7cd6e602307ef25e827baaa5b45a6ddcd3',\n","  'section': 'Abstract',\n","  'text': 'To reliably estimate the demand on regional health systems and perform public health planning, it is necessary to have a good estimate of the prevalence of infection with SARS-CoV-2 (the virus that causes COVID-19) in the population. In the absence of wide-spread testing, we provide one approach to infer prevalence based on the assumption that the fraction of true infections needing hospitalization is fixed and that all hospitalized cases of COVID-19 in Santa Clara are identified. Our goal is to estimate the prevalence of SARS-CoV-2 infections, i.e. the true number of people currently infected with the virus, divided by the total population size. '},\n"," {'paper_id': '8575c06c64fdd0bb43efc46cd994f557698c755f',\n","  'section': 'Abstract',\n","  'text': 'Quick, simple and accurate diagnosis of suspected COVID-19 is very important for the screening and therapy of patients. Although several methods were performed in clinical practice, however, the IgM and IgG diagnostic value evaluation was little performed. 57 suspected COVID-19 infection patients were enrolled in our study. 24 patients with positive and 33 patients with negative nucleic acid test. The positive rate of COVID-19 nucleic acid was 42.10%. The positive detection rate of combination of IgM and IgG for patients with COVID-19 negative and positive nucleic acid test was 72.73% and 87.50%. The results were significantly higher than the nucleic acid or IgM, IgG single detection. hsCRP in the COVID-19 nucleic acid negative group showed significantly higher than the positive groups (P=0.0298). AST in the COVID-19 IgM negative group showed significantly lower than the positive groups (P=0.0365). We provided a quick, simple, accurate aided detection method for the suspected patients and on-site screening in close contact with the population. '},\n"," {'paper_id': '85e0c4e5071189adeb037b757a37f092a8f52145',\n","  'section': 'Abstract',\n","  'text': '200 22 Main (MAX): 2,584 23 24 15 Abstract 25 Background: In Korea, a total of 8,799 confirmed cases of COVID-19 including 102 deaths have been 26 recorded as of Mar 21, 2020. The time-delay adjusted case fatality risk of COVID-19 in Korea is yet to be 27 estimated. 28 29 Methods: We obtained the daily series of confirmed cases and deaths in Korea reported prior to March 21, 30 2020. Using statistical methods, we estimated the time-delay adjusted risk for death from COVID-19 in the 31 city of Daegu, Gyeongsangbuk-do, other regions in Korea, as well as for the entire country. 32 33 Results: Our model-based crude CFR fitted the observed data well throughout the course of the epidemic 34 except for the very early stage in Gyeongsangbuk-do, partially due to the reporting delay. Our estimates of 35 the risk for death in Gyeongsangbuk-do reached 2.4% (95% CrI: 1.6-3.4%), 1.3% (95% CrI: 1.0-1.6%) in 36 Daegu and 0.7% (95% CrI: 0.3-1.4%) in other regions, whereas the national estimate of the risk for death 37 was estimated at 1.4% (95% CrI: 1.2-1.7%) in Korea. 38 39 Conclusions: The relatively low CFRs are associated with the early implementation of public health 40 interventions including widespread testing, social distancing, and delayed school openings in Korea. 41 Geographic differences in CFR are likely influenced by clusters of nosocomial transmission. 42 43 '},\n"," {'paper_id': '85e97e7b856aa681dd67c802852d39daf052f487',\n","  'section': 'Abstract',\n","  'text': 'In viruses, post-translational modifications (PTMs) are essential for their life cycle. Recognizing viral PTMs is very important for better understanding the mechanism of viral infections and finding potential drug targets. However, a database of comprehensive PTM information on viruses is not available. Results: To fill this gap, we developed a viral post-translational modification database (VPTMdb) for collecting systematic information of viral PTM data. The VPTMdb contains 912 PTM sites that integrate 414 experimental-confirmed PTM sites with 98 proteins in 45 human viruses manually extracted from 162 publications and 498 PTMs extracted from UniProtKB/Swiss-Prot. Moreover, we created a novel sequence-based classifier named VPTMpre to help users predict viral protein phosphorylated sites. Finally, an online web server was implemented for users to download viral PTM data and predict phosphorylated sites of interest. Availability and Implementation: VPTMdb can be accessed at http://vptmdb.com:8787/VPTMdb/ author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '8695528795dd02723afd17afba6e768d08372aba',\n","  'section': 'Abstract',\n","  'text': 'Severe COVID-19 patients account for most of the mortality of this disease. 25 Early detection and effective treatment of severe patients remain major 26 challenges. Here, we performed proteomic and metabolomic profiling of sera 27 from 46 COVID-19 and 53 control individuals. We then trained a machine 28 learning model using proteomic and metabolomic measurements from a 29 training cohort of 18 non-severe and 13 severe patients. The model correctly 30 classified severe patients with an accuracy of 93.5%, and was further 31 validated using ten independent patients, seven of which were correctly 32 classified. We identified molecular changes in the sera of COVID-19 patients 33 implicating dysregulation of macrophage, platelet degranulation and 34 complement system pathways, and massive metabolic suppression. This 35 study shows that it is possible to predict progression to severe COVID-19 36 disease using serum protein and metabolite biomarkers. Our data also 37 uncovered molecular pathophysiology of COVID-19 with potential for 38 developing anti-viral therapies. All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '86aa76ed7c5e77e02cb11831696e5f72ebc06690',\n","  'section': 'Abstract',\n","  'text': 'The COVID-19 pandemic has crudely demonstrated the value of massive and rapid 18 diagnostics. By the first week of April, more than 900,000 positive cases of COVID-19 : medRxiv preprint the case of an epidemic emergency, the first line of response should be based on 21 commercially available and validated resources. Here, we demonstrate the combined use of 22 the miniPCR®, a commercial compact and portable PCR device recently available on the 23 market, and a commercial well-plate reader as a diagnostic system for detecting SARS-24 CoV2 nucleic acids. We used the miniPCR to detect and amplify SARS-CoV2 DNA 25 sequences using the sets of initiators recommended by the World Health Organization for 26 targeting three different regions that encode for the N protein. Prior to amplification, 27 samples were combined with a DNA intercalating reagent (i.e., EvaGreen® Dye). Sample 28 fluorescence after amplification was then read using a commercial 96-well plate reader. 29 This straightforward method allows the detection and amplification of SARS-CoV2 nucleic 30 acids in the range of ~625 to 2×10 5 DNA copies. The accuracy and simplicity of this 31 diagnostics strategy may provide a cost-efficient and reliable alternative for COVID-19 32 pandemic testing, particularly in underdeveloped regions where RT-QPCR instrument 33 availability may be limited. In addition, the portability, ease of use, and reproducibility of 34 the miniPCR® makes it a reliable alternative for deployment in point-of-care SARS-CoV2 35 detection efforts during pandemics. 36 37 : medRxiv preprint [1,2]. The recent pandemic/epidemic episodes associated with viral diseases (e.g., 44 pandemic Influenza A/H1N1/2009 [3], Ebola in West Africa in 2013-2015 [4], and Zika in 45 Southeast Asia and Latin-America in 2016 [5,6]) were clear reminders of the need for 46 portable, low-cost, and easy-to-use diagnostic systems that can effectively address epidemic 47 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': '875b7c463f00772fa0dc18ada678bc1ff16a4274',\n","  'section': 'Abstract',\n","  'text': 'Objective: To evaluate the spectrum of comorbidities and its impact on the clinical outcome in patients with coronavirus disease 2019 . Setting: 575 hospitals in 31 province/autonomous regions/provincial municipalities across China Participants: 1,590 laboratory-confirmed hospitalized patients. were extracted from medical charts. The disease severity was categorized based on the American Thoracic Society guidelines for community-acquired pneumonia. The primary endpoint was the composite endpoints, which consisted of the admission to intensive care unit (ICU), or invasive ventilation, or death. The risk of reaching to the composite endpoints was compared among patients with COVID-19 according to the presence and number of comorbidities. Results: Of the 1,590 cases, the mean age was 48.9 years. 686 patients (42.7%) were females. 647 (40.7%) patients were managed inside Hubei province, and 1,334 (83.9%) patients had a contact history of Wuhan city. Severe cases accounted for 16.0% of the study population. 131 (8.2%) patients reached to the composite endpoints. 399 (25.1%) reported having at least one comorbidity. : medRxiv preprint 5 malignancy and immunodeficiency, respectively. 130 (8.2%) patients reported having two or more comorbidities. Patients with two or more comorbidities had significantly escalated risks of reaching to the composite endpoint compared with those who had a single comorbidity, and even more so as compared with those without (all P<0.05). After adjusting for age and smoking status, patients with COPD (HR 2.681, 95%CI 1.424-5.048), diabetes (HR 1.59, 95%CI 1.03-2.45), hypertension (HR 1.58, 95%CI 1.07-2.32) and malignancy (HR 3.50, 95%CI 1.60-7.64) were more likely to reach to the composite endpoints than those without. As compared with patients without comorbidity, the HR (95%CI) was 1.79 (95%CI 1.16-2.77) among patients with at least one comorbidity and 2.59 (95%CI 1.61-4.17) among patients with two or more comorbidities. Comorbidities are present in around one fourth of patients with COVID-19 in China, and predispose to poorer clinical outcomes. FUNDING: Supported by National Health Commission, Department of Science and Technology of Guangdong Province. The funder had no role in the conduct of the study. '},\n"," {'paper_id': '8762adc9311e5803d1de438493772269acfe7fa3',\n","  'section': 'Abstract',\n","  'text': \"Background. The novel coronavirus (2019-nCoV) epidemic has spread to 23 countries from China. Local cycles of transmission already occurred in 7 countries following case importation. No African country has reported cases yet. The management and control of 2019-nCoV introductions heavily relies on country's health capacity. Here we evaluate the preparedness and vulnerability of African countries against their risk of importation of 2019-nCoV. \"},\n"," {'paper_id': '87b89b038b28d2ba0bcfd2203845bf15f922c1b5',\n","  'section': 'Abstract',\n","  'text': 'The COVID-19 epidemic originated in Wuhan City of Hubei Province in December 2019 and has spread throughout China. Understanding the fast evolving epidemiology and transmission dynamics of the outbreak beyond Hubei would provide timely information to guide intervention policy. We collected individual information on 8,579 laboratory-confirmed cases from official publically sources reported outside Hubei in mainland China, as of February 17, 2020. We estimated the temporal variation of the demographic characteristics of cases and key time-to-event intervals. We used a Bayesian approach to estimate the dynamics of the net reproduction number (Rt) at the provincial level. '},\n"," {'paper_id': '87bfa13221ccf5a02d4a888ef1320b57f2c2d5e2',\n","  'section': 'Abstract',\n","  'text': 'Bats are unique among mammals due to the ability of powered flight and exceptional longevity. They 2 are also asymptomatic hosts for numerous viruses, including recently emerged zoonotic 3 Henipaviruses Nipah and Hendra, which are highly pathogenic for humans and other mammals. 4 Better understanding of how bats control viral infection requires development of relevant permissive 5 cellular experimental models. By applying a somatic reprogramming protocol to Pteropus bat primary 6 cells, using a novel combination of ESRRB, CDX2, and c-MYC transcription factors, we generated 7 bat reprogrammed cells exhibiting stem cell-like characteristics and a neural stem cell-like molecular 8 signature. These cells present a unique interferon-stimulated transcriptomic signature and both 9 produce and respond to interferon type-I, highlighting differences between stem cells from bats and 10 other mammals. In contrast to primary bat cells, these reprogrammed cells are highly susceptible to 11 infection by Henipavirus, thereby enabling isolation of new bat viruses, study of virus-bat 12 interactions, and better understanding of bat biology. 13 14 '},\n"," {'paper_id': '87c582dc954f6bb05f5b8276f7385221cdcb8a91',\n","  'section': 'Abstract',\n","  'text': 'medRxiv preprint Severe acute respiratory syndrome coronavirus 2 (SARS-Cov-2) quickly became a major epidemic threat in the whole China. We analysed SARS-Cov-2 infected cases from Tibetan Autonomous Prefecture, and noted divergent characteristics of these Tibetans infected cases compared to Han Chinese, characterizing by a considerable proportion of asymptomatic carriers (21.7%), and few symptomatic patients with initial symptom of fever (7.7%). Here, we did a descriptive study on clinical characteristics of 18 asymptomatic individuals with SARS-CoV-2 infection. The median age of these asymptomatic carriers was 31 years and one third of them were students, aged under 20 years. Notably, some of asymptomatic carriers had recognizable changes in radiological and laboratory indexes. Our finding indicates a potentially big number of SARS-CoV-2 asymptomatic carriers in prevalent area, highlighting a necessity of screening individuals with close contact of infected patients, for a better control on the spread of SARS-CoV-2 infection. '},\n"," {'paper_id': '882add17d7b4155f04b40c4e25da480c5c4a7a33',\n","  'section': 'Abstract',\n","  'text': 'The SARS-CoV-2 pandemic demonstrates the need for accurate and convenient approaches to 2 diagnose and therapeutically monitor respiratory viral infections. We demonstrated that self-3 sampling with foam swabs is well-tolerated and provides quantitative viral output concordant 4 with flocked swabs. Using longitudinal home-based self-sampling, we demonstrate nasal 5 cytokine levels correlate and cluster according to immune cell of origin. Periods of stable viral 6 loads are followed by rapid elimination, which could be coupled with cytokine expansion and 7 contraction using mathematical models. Nasal foam swab self-sampling at home provides a 8 precise, mechanistic readout of respiratory virus shedding and local immune responses. 9 . CC-BY-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': '88bdd99c41f943dd769a81f74a53434f3eb4f371',\n","  'section': 'Abstract',\n","  'text': 'Seasonal variation in COVID-19 incidence could impact the trajectory of the pandemic. Using global linelist data on COVID-19 cases reported until 29 th February 2020 and global gridded temperature data, and after adjusting for surveillance capacity and time since first imported case, higher average temperature was strongly associated with lower COVID-19 incidence for temperatures of 1°C and higher. However, temperature explained a relatively modest amount of the total variation in COVID-19 incidence. These preliminary findings support stringent containment efforts in Europe and elsewhere. '},\n"," {'paper_id': '88e3b43319399c339186c02c44d369228c35eef4',\n","  'section': 'Abstract',\n","  'text': 'Cancer gene therapy is a great promising tool for cancer therapeutics due to the specific targeting based on the cancerous gene expression background. Binary systems based on sitespecific recombination are one of the most effective potential approaches for cancer gene therapy. In these systems, a cancer specific promoter expresses a site-specific recombinase/integrase that in turn controls the expression of a toxin gene. In the current study, we have developed a new HK022 bacteriophage Integrase (Int) based binary system activating a Diphtheria toxin (DTA) gene expression specifically in cancer cells. We have demonstrated the efficiency, and the high specificity of the system in vitro in cell cultures and in vivo in a lung cancer mouse model. Strikingly, different apoptotic and anti-apoptotic factors demonstrated a remarkable efficacy killing capability of the Int-based binary system compared to the conventional hTERT-DTA mono system in the LLC-Kat lung cancer mice model; we observed that the active hTERT promoter down regulation by the transcription factors Mad-1 is the cornerstone of this phenomenon. The new Int-based binary system offers advantages over already known counterparts and may therefore be developed into a safer and efficient cancer treatment technology. '},\n"," {'paper_id': '891750a4ccc2abe138f88668ea6dcb75260f7cbf',\n","  'section': 'Abstract',\n","  'text': 'Predicting the course of an epidemic is difficult, predicting the course of a pandemic from an emerging virus even more so. The validity of most predictive models relies on numerous parameters, involving biological and social characteristics often unknown or highly uncertain. COVID-19 pandemic brings additional factors such as population density and movements, 5 behaviours, quality of the health system. Data from the COVID-19 epidemics in China, Japan and South Korea were used to build up data-driven deterministic models. Epidemics occurring in selected European countries rapidly evolved to overtake most Chinese provinces, to overtake South Korean model for France and even Hubei in the case of Italy and Spain. This approach was applied to other European countries and provides relevant information to inform disease 10 control decision-making. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '896cb03362facd2a670bee13dc5368b81768b2ba',\n","  'section': 'Abstract',\n","  'text': 'Zika virus (ZIKV) infection in humans has been associated with congenital malformations and other neurological disorders, such as Guillain-Barré syndrome. The mechanism(s) of ZIKV intrauterine transmission, the cell types involved, the most vulnerable period of pregnancy for severe outcomes from infection and other physiopathological aspects remain unknown. In this study, we analyzed placental samples obtained at the time of delivery from a group of twenty-four women diagnosed with ZIKV infection during the first, second or third trimesters of pregnancy. Villous immaturity was the main histological finding in the placental tissues, although placentas without alterations were also frequently observed. Significant enhancement of the number of syncytial sprouts was observed in the placentas of women infected during the third trimester, indicating the development of placental abnormalities after ZIKV infection. Hyperplasia of Hofbauer cells (HCs) was also observed in these thirdtrimester placental tissues, and remarkably, HCs were the only ZIKV-positive fetal cells found in the placentas studied that persisted until birth, as revealed by immunohistochemical (IHC) analysis. Thirty-three percent of women infected during pregnancy delivered infants with congenital abnormalities, although no pattern correlating the gestational stage at infection, the IHC positivity of HCs in placental tissues and the presence of congenital malformations at birth was observed. Placental tissue analysis enabled us to confirm maternal ZIKV infection in cases where serum from the acute infection phase was not available, which reinforces the importance of this technique in identifying possible causal factors of birth defects. The results we observed in the samples from naturally infected pregnant women may contribute to the understanding of some aspects of the pathophysiology of ZIKV. '},\n"," {'paper_id': '8a1fde8c65e439496ac5810504de23ef77312f28',\n","  'section': 'Abstract',\n","  'text': 'As the infection of 2019-nCoV coronavirus is quickly developing into a global pneumonia epidemic, careful analysis of its transmission and cellular mechanisms is sorely needed. In this report, we re-analyzed the computational approaches and findings presented in two recent manuscripts by Ji et al. (https://doi.org/10.1002/jmv.25682) and by Pradhan et al. (https://doi.Global protein sequence alignment of the full-length coronavirus spike proteins was performed by MUSCLE 17 and visualized by SeaView 18 . We use C-I-TASSER 5 to create structural models of the full-length spike protein. Here, C-I-TASSER is an extended pipeline of I-TASSER 19 and utilizes the deep convolutional neuralnetwork based contact-maps 20 to guide the Monte Carlo fragment assembly simulations. Since the RBD domain of spike exhibits different conformations relative to the remaining portion of . CC-BY 4.0 '},\n"," {'paper_id': '8a73ab00fcfa1789da4d0755f67bc5da48846ace',\n","  'section': 'Abstract',\n","  'text': 'medRxiv preprint 5 significantly higher scores of PSQI (p=0.022) and higher prevalence of sleep disturbances with PSQI > 7 points (p<0.0001). Interpretation fMW had higher prevalence of sleep disturbances and worse sleep quality than non-fMW. Female fMW were more vulnerable to having sleep disturbances than male fMW. Funding None. '},\n"," {'paper_id': '8aa021696a6accab9053f1d779eef6710439bfa5',\n","  'section': 'Abstract',\n","  'text': \"Understanding and quantifying the transmission of zoonotic pathogens is essential for 24 directing public health responses, especially for pathogens capable of transmission between 25 humans. However, determining a pathogen's transmission dynamics is complicated by 26 challenges often encountered in zoonotic disease surveillance, including unobserved sources of 27 transmission (both human and zoonotic), limited spatial information, and unknown scope of 28 surveillance. In this work, we present a model-based inference method that addresses these 29 challenges for subcritical zoonotic pathogens using a spatial model with two levels of mixing. 30 After demonstrating the robustness of the method using simulation studies, we apply the new 31 method to a dataset of human monkeypox cases detected during an active surveillance program 32 from 1982-1986 in the Democratic Republic of the Congo (DRC). Our results provide estimates 33 of the reproductive number and spillover rate of monkeypox during this surveillance period and 34 suggest that most human-to-human transmission events occur over distances of 30km or less. 35 Taking advantage of contact-tracing data available for a subset of monkeypox cases, we find that 36 around 80% of contact-traced links could be correctly recovered from transmission trees inferred 37 using only date and location. Our results highlight the importance of identifying the appropriate 38 spatial scale of transmission, and show how even imperfect spatiotemporal data can be 39 incorporated into models to obtain reliable estimates of human-to-human transmission patterns. 40 Author Summary Surveillance datasets are often the only sources of information about the ecology and 42 epidemiology of zoonotic infectious diseases. Methods that can extract as much information as 43 possible from these datasets therefore provide a key advantage for informing our understanding 44 . CC-BY 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/677021 doi: bioRxiv preprint 3 of the disease dynamics and improving our ability to choose the optimal intervention strategy. 45 We developed and tested a likelihood-based inference method based on a mechanistic model of 46 the spillover and human-to-human transmission processes. We first used simulated datasets to 47 explore which information about the disease dynamics of a subcritical zoonotic pathogen could 48 be successfully extracted from a line-list surveillance dataset with non-localized spatial 49 information and unknown geographic coverage. We then applied the method to a dataset of 50 human monkeypox cases detected during an active surveillance program in the Democratic 51 Republic of the Congo between 1982 and 1986 to obtain estimates of the reproductive number, 52 spillover rate, and spatial dispersal of monkeypox in humans. 53 [1][2] [3] [4] [5][6][7][8]. By definition, zoonotic pathogens can transmit from animals to humans; those also 58 capable of human-to-human transmission are of particular public health concern [5, 9] . Infectious 59 disease surveillance serves a crucial role for detecting and gathering information on zoonotic 60 pathogens: data obtained through surveillance are often the primary resource available for 61 informing public health management decisions [10]. Developing methods that improve our 62 ability to infer information about a pathogen's transmission dynamics from available 63 surveillance data is therefore an essential frontier for understanding and ultimately combating 64 these pathogens [11, 12]. 65 . CC-BY 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/677021 doi: bioRxiv preprint 4 For zoonoses, three epidemiological measures are crucial for summarizing transmission 66 dynamics and informing risk assessments. The first of these is the spillover rate, which indicates 67 how frequently the pathogen is transmitted from the animal reservoir into humans and helps 68 inform the total expected disease incidence [13]. The second measure describes the pathogen's 69 potential for further spread once in the human population and is commonly assessed using the 70 reproductive number (R), which gives the average number of secondary human cases caused by 71 an infectious individual [14, 15]. Values of R greater than one indicate that the pathogen is 72 capable of sustained (i.e. 'supercritical') transmission in humans. Pathogens with subcritical 73 transmission (R less than one but greater than zero) can cause limited chains of transmission in 74 humans after a zoonotic introduction, and they pose a risk of acquiring ability for supercritical 75 transmission via evolutionary or environmental change [2,5,16]. The third epidemiological 76 measure is the distance over which human-to-human transmission occurs, which informs how 77 the disease will spread spatially and the risk of it being introduced into new populations. 78 Combined, these three measures can help evaluate the current public health threat posed by the 79 pathogen, the risk of future emergence, and the most effective approaches for disease 80 management. 81 Estimating epidemiological measures is a challenging task in any pathogen system, and 82 the unique properties of zoonotic diseases can exacerbate these difficulties. Infectious disease 83 surveillance often records temporal information and certain aspects of spatial information about 84 human cases, but the underlying transmission events are seldom observed. In a zoonotic system, 85 this means that an observed human infection could have been caused by a previous human case 86 or by zoonotic spillover. Without intensive contact tracing, or sequence data in the case of fast-87 evolving pathogens, quantifying the relative contribution of zoonotic versus human-to-human 88 . CC-BY 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/677021 doi: bioRxiv preprint 5 transmission is a major challenge; identifying the source of infection for specific individuals is 89 an even bigger one. 90 Epidemiological analyses are often hindered by data truncation and unknown 91 denominators [17,18]. In many disease surveillance systems, the total set of localities under 92 surveillance (i.e. those that would appear in the dataset if a case occurred there) can be separated 93 into 'observed localities,' which appear in the dataset because they reported one or more cases, 94 and 'silent localities,' which have no cases during the surveillance period and therefore do not 95 appear in the dataset. This form of truncation, where localities with zero cases are absent from 96 the dataset, obscures the true scope of the surveillance effort. Without knowledge of the total 97 number of localities under observation (the 'unknown denominator'), accurately estimating the 98 spillover rate and probability of human-to-human transmission between localities is not 99 straightforward. Simply disregarding these silent localities in the analysis is the functional 100 equivalent of selectively removing zeros from the dataset and can lead to problematic inference 101 biases. 102 Complicating inference efforts further is the fact that surveillance datasets often report 103 the geographic location of cases only at a coarse resolution, obscuring information about a 104 transmission process that occurs on a much finer scale [19] [20][21]. Precise spatial information is 105 often absent from historic datasets and data collected in remote or low-resource areas, replaced 106 by the names of the locality and broader administrative units where the case occurred. For 107 example, only the village name and the region and country to which the village belongs may be 108 recorded in a dataset. Furthermore, linking a village name to spatial coordinates is often 109 impossible when maps of the region do not exist or only unofficial local names are used. 110 Although collecting exact spatial coordinates has become more practical in contemporary disease 111 . CC-BY 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/677021 doi: bioRxiv preprint 6 surveillance, privacy and confidentiality concerns can arise in both human and agricultural 112 contexts when data contains high-resolution spatial information [19, 20,[22][23][24][25], leading to data 113 being reported in a non-localized manner. Methods that can use this inexact spatial information 114 are especially needed for zoonotic diseases, where any additional information about the 115 proximity of human cases to one another can improve the power to distinguish between human-116 to-human transmission and zoonotic spillover. 117 Despite these challenges, a series of research efforts have expanded our ability to 118 estimate the transmission properties of zoonotic pathogens from case onset data. A key set of 119 methods revolve around inferring R from the sizes of case clusters (a cluster is defined as a group 120 of cases that occur in close spatiotemporal proximity to one another) or from the proportion of 121 observed cases that were infected by zoonotic spillover [16,[26][27][28][29][30]. However, these approaches 122 either require detailed case investigations to determine whether a case was infected by a zoonotic 123 or human source or assume that each cluster is caused by one single spillover event followed by 124 human-to-human transmission. A likelihood-based approach for estimating R for human-to-125 human transmission using only symptom onset dates of cases was introduced by Wallinga and 126 Teunis [31]. This method was extended to apply to zoonotic systems by Lo Iacono et al. [32], but 127 the extension requires that chains of exclusively human-to-human transmission can be identified, 128 and is thus not applicable to many zoonotic surveillance systems where human and zoonotic 129 transmissions are intermixed. A different approach was taken by White and Pagano [33], who 130 introduced a different likelihood-based method that compares the observed number of cases on 131 each day with the expected number, as calculated using the number and timing of previous cases. 132 Though the White and Pagano approach was only applicable to human-to-human transmission, it 133 was expanded by Kucharski et al. [34] to work in zoonotic spillover systems in scenarios where a 134 . CC-BY 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/677021 doi: bioRxiv preprint precise geographic information should it be available, in this study we focus on the more 150 challenging scenario in which only the names of the locality and broader administrative units 151 where a case occurred are known. To make use of this form of non-localized spatial data, our 152 model considers two scales of spatial mixing and transmission (Fig 1A) , reminiscent of the 153 'epidemics with two levels of mixing' structure utilized in Ball et al. [41] and Demiris and 154 O'Neill [42]. The first mixing level is the locality in which the case occurred, such as a village, 155 conceptualized as a group of individuals geographically separated from other localities. We 156 assume that individuals within the same locality have more frequent contact with one another 157 . CC-BY 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/677021 doi: bioRxiv preprint human-to-human transmission, and orange if infected by between-locality human-to-human 172 transmission. Individuals who are not infected are colored gray and do not appear in the 173 surveillance dataset. Similarly, if zero individuals in a locality are infected, that 'silent locality' 174 does not appear in the dataset (represented by the gray locality in the broader contact zone). B. 175 The possible sources of human infection, which in aggregate determine the number of new 176 infections on day t, locality v. The number of cases arising from spillover and human-to-human 177 transmissions follow Poisson distributions with means λ Z and λ {s,w},{t,v} , respectively. 178 179 . CC-BY 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/677021 doi: bioRxiv preprint for model comparison. In the monkeypox dataset, contact-tracing data are available for a subset 199 of the cases, providing a rare opportunity to compare inferred transmission sources with those 200 suggested by epidemiological investigation. In addition, some localities were associated with 201 known GPS coordinates, enabling us to estimate the spatial transmission kernel in greater detail. 202 . CC-BY 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/677021 doi: bioRxiv preprint validated method to a dataset on human monkeypox cases to estimate key epidemiological 216 parameters and the spatial scale of transmission. To generate simulated test datasets and perform 217 parameter inference, we used a mathematical model of the zoonotic pathogen's transmission into 218 and among humans. The model tracks the number of human cases that occur in each locality on 219 each day; infections can arise from spillover from the zoonotic reservoir or from human-to-220 human transmission (Fig 1B) . Three key parameters govern the behavior of the system. The 221 spillover rate (λ z ) describes the average number of human cases caused by animal-to-human 222 transmission ('primary cases') in each locality per day. The reproductive number of the pathogen 223 . CC-BY 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/677021 doi: bioRxiv preprint correspond to 59 to 415 expected spillover events in the five year simulation period, across all 240 localities). Transmission events between humans had a probability σ=0.75 of occurring within a 241 locality and otherwise were equally likely between any localities in the same broader contact 242 zone. We were interested in seeing how well the inference methods are able to use the spatial-243 temporal arrangement of cases to estimate the true parameter values. 244 . CC-BY 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/677021 doi: bioRxiv preprint We tested our method using 125 simulated datasets, with 25 datasets simulated for each of five 307 sets of true parameter values (these are the same datasets as discussed above, simulated with R 308 between 0.2 and 0.6 and spillover rate between 0.0001 and 0.0007). Two hundred plausible 309 transmission trees were sampled for each simulated dataset. 310 \"},\n"," {'paper_id': '8b2e50eb3ea84225580fbdbccccb4fcd3f062feb',\n","  'section': 'Abstract',\n","  'text': 'Background 33 An outbreak of a novel coronavirus (SARS-CoV-2)-infected pneumonia was first occurred in Wuhan, China, in December 2019 and then spread rapidly to 35 other regions. It has been declared that at least one confirmed case infected by 36 SARS-CoV-2 was found in each province of China by late January 2020. 37 We collected the time series data of the cumulative number of confirmed infected, 39 dead, and cured cases from the health commissions in 31 provinces in mainland China. 40 A simple descriptive model in a logistic form was formulated to infer the intrinsic 41 epidemic rules of COVID-19. Furthermore, we compared the intrinsic epidemic rules 42 of COVID-19 in Hubei with that of severe acute respiratory syndrome (SARS) in 43 Beijing, which was obtained from the Ministry of Public Health of China in 2003. 44 The feature of data of three population groups in all the provinces could be well 46 captured (R 2 >0.95) by the proposed descriptive model with slight variation among 47 (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': '8b3d4a33d0cba9d1875dcd658df6c3268b62fbb7',\n","  'section': 'Abstract',\n","  'text': \"The COVID-19 pandemic is spreading worldwide. Italy emerged early on as the country with the largest outbreak outside Asia. The outbreak in Northern Italy demonstrates that it is fundamental to contain the virus' spread at a very early stage of diffusion. At later stages, no containment measure, even if strict, can prevent the saturation of the hospitals and of the intensive care units in any country. Here we show that it is possible to predict when the intensive care units will saturate, within a few days from the first cases of COVID-19 intensive care patients. Using early counts of intensive care patients, we predict the saturation for Lombardy, Italy. Governments should use the Italian precursor to control the outbreak of COVID-19 and prevent the saturation of their intensive care units to protect their population. \"},\n"," {'paper_id': '8b71ce92ec03e066eca7792bfd17e2f210d50ce3',\n","  'section': 'Abstract',\n","  'text': 'COVID-19 caused by a novel coronavirus SARS-CoV-2 emerged in Wuhan, Hubei province since December 2019, and caused a rapid outbreak throughout China and globally. Cities outside Hubei are also facing great challenge and require implementing of effective and feasible strategy in precision diagnosing novel coronavirus pneumonia (NCP). We described a multicenter prospective study on diagnostic strategy of suspected NCP patients from January 22 nd to February 9 th , 2020 in Eastern China cities. Nasopharyngeal swabs were collected from the patients. The epidemiological characteristics, clinical symptoms, laboratory assessments, and computed tomographic (CT) scans were obtained. Pathogen screen were performed including RT-PCR, multiplex PCR, rapid flu antigen tests and mNGS. We enrolled 53 suspected NCP patients, among whom 20 were laboratory-confirmed. Fourteen (70%) and 3 (15%) patients were positive for the first and second SARS-CoV-2 RT-PCR test, respectively. All NCP patients were positive for mNGS. Chest CT images and the symptoms of early stage NCP patients were similar to other viral pneumonia patients. We identified 11 of 20 co-infections in NCP cases, including regular respiratory virus, fungi and bacteria synchronously. Genomic analysis showed that 8 of 10 cases had no mutation in virus genome, while 2 cases had only one single mutation in N gene. Our study discovered that a combination of chest CT, SARS-CoV-2 RT-PCR and multi-plex PCR is recommended in regions outside Hubei province. Co-infection of other pathogens with SARS-CoV-2 exists and should be acknowledged. Repeated sampling, change of specimen type or metagenomics sequencing could further facilitate during critical clinical cases. '},\n"," {'paper_id': '8b88901d0bb033af3a18ebcdf7f2aa7110b21504',\n","  'section': 'Abstract',\n","  'text': '1 The outbreak of novel coronavirus pneumonia (NCP) caused by '},\n"," {'paper_id': '8b88d01563538e4abd29c3ba493c534af30664cc',\n","  'section': 'Abstract',\n","  'text': 'bioRxiv preprint 2 this study, we developed a Bats-Hosts-Reservoir-People transmission network model for simulating the potential transmission from the infection source (probable be bats) to the human infection. Since the Bats-Hosts-Reservoir network was hard to explore clearly and public concerns were focusing on the transmission from a seafood market (reservoir) to people, we simplified the model as Reservoir-People transmission network model. The basic reproduction number (R 0 ) was calculated from the RP model to assess the transmissibility of the 2019-nCoV. '},\n"," {'paper_id': '8bb82a543720a74015bce24945dc0955b666ae7d',\n","  'section': 'Abstract',\n","  'text': 'Background: Sustained spread of SARS-CoV-2 has happened in major US cities. Capacity needs in Chinese cities could inform the planning of local healthcare resources. '},\n"," {'paper_id': '8bd8158e4a47c65b12a462b49b4b8b4af1488dc3',\n","  'section': 'Abstract',\n","  'text': 'In this short paper, the logistic growth model and classic susceptible-infected-recovered dynamic model are used to estimate the final size of the coronavirus epidemic. '},\n"," {'paper_id': '8c04d0f21f205b1813637225ca11b61ed11f1a49',\n","  'section': 'Abstract',\n","  'text': 'On the basis of a semi-realistic SIR microsimulation for Germany and Poland, we show that the R 0 parameter interval for which the COVID-19 epidemic '},\n"," {'paper_id': '8c0b398abd2ce9151a30e14e324a2d91b6b850ce',\n","  'section': 'Abstract',\n","  'text': 'Since the sudden outbreak of coronavirus disease 2019 , it has rapidly evolved into a momentous global health concern. Due to the lack of constructive information on the pathogenesis of COVID-19 and specific treatment, it highlights the importance of early diagnosis and timely treatment. In this study, 11 key blood indices were extracted through random forest algorithm to build the final assistant discrimination tool from 49 clinical available blood test data which were derived by commercial blood test equipments. The method presented robust outcome to accurately identify COVID-19 from a variety of suspected patients with similar CT information or similar symptoms, with accuracy of 0.9795 and 0.9697 for the cross-validation set and test set, respectively. The tool also demonstrated its outstanding performance on an external validation set that was completely independent of the modeling process, with sensitivity, specificity, and overall accuracy of 0.9512, 0.9697, and 0.9595, respectively. Besides, 24 samples from overseas infected patients with COVID-19 were used to make an in-depth clinical assessment with accuracy of 0.9167. After multiple verification, the reliability and repeatability of the tool has been fully evaluated, and it has the potential to develop into an emerging technology to identify COVID-19 and lower the burden of global public health. The . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint . https://doi.org/10. 1101 proposed tool is well-suited to carry out preliminary assessment of suspected patients and help them to get timely treatment and quarantine suggestion. The assistant tool is now available online at '},\n"," {'paper_id': '8c210df7072bc88fa7d2da72b25c84430c16f5e8',\n","  'section': 'Abstract',\n","  'text': 'The recent outbreak of a new coronavirus (SARS-CoV-2) in Wuhan, China, underscores the need for understanding the evolutionary processes that drive the emergence and adaptation of zoonotic viruses in humans. Here, we show that recombination in betacoronaviruses, including human-infecting viruses like SARS-CoV and MERS-CoV, frequently encompasses the Receptor Binding Domain (RBD) in the Spike gene. We find that this common process likely led to a recombination event at least 11 years ago in an ancestor of the SARS-CoV-2 involving the RBD. As a result of this recombination event, SARS-CoV and SARS-CoV-2 share a similar genotype in RBD, including two insertions (positions 432-436 and 460-472), and alleles 427N and 436Y. Both 427N and 436Y belong to a helix that interacts with the human ACE2 receptor. Ancestral state analyses revealed that SARS-CoV-2 differentiated from its most recent common ancestor with RaTG13 by accumulating a significant number of amino acid changes in the RBD. In sum, we propose a two-hit scenario in the emergence of the SARS-CoV-2 virus whereby the SARS-CoV-2 ancestors in bats first acquired genetic characteristics of SARS-CoV by incorporation of a SARS-like RBD through recombination before 2009, and subsequently, the lineage that led to SARS-CoV-2 accumulated further, unique changes specifically in the RBD. '},\n"," {'paper_id': '8c79b740891714d155a31f1a4fa969f6732fd6ef',\n","  'section': 'Abstract',\n","  'text': 'Coronavirus disease 2019 is an infectious disease caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), it was first identified in 2019 in Wuhan, China and has resulted in the 2019-20 coronavirus pandemic. As of March 1, 2020, 79,968 patients in China and 7169 outside of China had tested positive for COVID-19 and a mortality rate of 3.6% has been observed amongst Chinese patients. Its primary mode of transmission is via respiratory droplets from coughs and sneezes. The virus can remain viable for up to three days on plastic and stainless steel or in aerosols for upto 3 hours and is relatively more stable than the known human coronaviruses. It is stable in faeces at room temperature for at least 1-2 days and can be stable in infected patients for up to 4 days. Heat at 56°C kills the SARS coronavirus at around 10000 units per 15 minutes. Thus, temperature is an important factor in survival of COVID-19 virus and this article focuses on understanding the relationship between temperature and COVID-19 transmission from the data available '},\n"," {'paper_id': '8c900409a679dc1f16ef13c73500cbb53605b683',\n","  'section': 'Abstract',\n","  'text': 'The severity and outcome of respiratory viral infections is partially determined by the cellular 9 response mounted by infected lung epithelial cells. Disease prevention and treatment is dependent on our 10 understanding of the shared and unique responses elicited by diverse viruses, yet few studies compare host 11 responses to different viruses while controlling other experimental parameters. We used microarray analysis 12 to compare changes in gene expression of murine lung epithelial cells infected individually by three respiratory 13 viruses causing mild (rhinovirus, RV1B), moderate (coronavirus, MHV-1), and severe (influenza A virus, 14 PR8) disease in mice. RV1B infection caused numerous gene expression changes, but the differential effect 15 peaked at 12 hours post-infection. PR8 altered an intermediate number of genes whose expression continued 16 to change through 24 hours. MHV-1 had comparatively few effects on host gene expression. The viruses 17 elicited highly overlapping responses in antiviral genes, though MHV-1 induced a lower type I interferon 18 response than the other two viruses. Signature genes were identified for each virus and included host defense 19 genes for PR8, tissue remodeling genes for RV1B, and transcription factors for MHV-1. Our comparative 20 approach identified universal and specific transcriptional signatures of virus infection that can be used to 21 discover mechanisms of pathogenesis in the respiratory tract. 22 42 murine cells. Because RV1B can replicate in mouse cells, it has been used to study immune responses and/or 43 mechanisms of asthma exacerbation in infected mice [1-5]. Intranasal inoculation of mice with a high dose of 44 RV1B results in viral replication and an early inflammatory response in the respiratory tract without clinical 45 signs of disease [1, 3-5]. RV1B antigens have been detected in cells of the epithelia and lamina propria of the 46 upper respiratory tract of infected mice [2, 4]. 47 MHV-1 is a strain of mouse hepatitis virus that preferentially replicates and causes disease in the 48 respiratory tract of specific mouse strains and thus serves as a model for respiratory coronaviral infections [6, 49 7]. MHV-1 causes severe disease in A/J-strain mice and milder pathology in other mouse strains [6, 8]. Mouse 50 strain-dependent disease severity corresponds to inflammatory responses, fibrin deposition, and reduced type I 51 interferon (IFN) production [6]. Further, type I IFN-mediated signaling is required for protection from severe 52 disease during MHV-1 infection of resistant mice [8]. MHV-1 has been detected in pulmonary macrophages of 53 infected mice, but has not been reported to infect respiratory epithelial cells in vivo [6]. Although primary 54 murine alveolar epithelial cells are susceptible to MHV-1 infection in vitro, their role during in vivo infection 55 is not clear [9]. '},\n"," {'paper_id': '8cc6848b097f6243b230c2559b3a599a700b5d89',\n","  'section': 'Abstract',\n","  'text': 'In the COVID-19 coronavirus pandemic, currently vaccines and specific anti-viral treatment are not yet available. Thus, preventing viral transmission by case isolation, quarantine, and social distancing is essential to slowing its spread. Here we model social networks using weighted graphs, where vertices represent individuals and edges represent contact. As public health measures are implemented, connectivity in the graph decreases, resulting in lower effective reproductive numbers, and reduced viral transmission. For COVID-19, model parameters were derived from the coronavirus epidemic in China, validated by epidemic data in Italy, then applied to the United States. We calculate that, in the U.S., the public is able to contain viral transmission by limiting the average number of contacts per person to less than 7 unique individuals over each 5 day period. This increases the average social distance between individuals to 10 degrees of separation. '},\n"," {'paper_id': '8cca7482514d4947367c4d538d7d33300449b90e',\n","  'section': 'Abstract',\n","  'text': 'European healthcare systems face extreme pressure from COVID-19. We estimate such pressure by relating both country-specific accumulated COVID-19 deaths (intensity-approach) and active COVID-19 cases (magnitude-approach) to measures of healthcare system capacity: hospital beds, healthcare workers and healthcare expenditure. On March 25, 2020 -relative to Italy on March 11-we found Spain, The Netherlands and France to experience the highest pressure using the intensity-approach with a composite measure for healthcare capacity. For updates see www.covid-hcpressure.org '},\n"," {'paper_id': '8cfb8dfc2f8b2c6e1d60ff48da02de6670694869',\n","  'section': 'Abstract',\n","  'text': 'The avian coronavirus infectious bronchitis virus (IBV) typically infects the respiratory tract of chickens, but can also spread to other organs. However, the mechanisms of virus dissemination are presently unclear. We show that peripheral blood--derived monocytes/macrophages from chickens (chPBMC) are productively infected by clinical strains of IBV, accompanied by induction of apoptosis. Our data suggest that chPBMCs play a role in the dissemination of IBV, and may be important for viral pathogenesis. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '8d07b32de2cd946099966ca7e46c3e9ee9659069',\n","  'section': 'Abstract',\n","  'text': 'A SEIR simulation model for the COVID-19 pandemic was developed (http://covidsim.eu) and applied to a hypothetical European country of 10 million population. Our results show which interventions potentially push the epidemic peak into the subsequent year (when vaccinations may be available) or which fail. Different levels of control (via contact reduction) resulted in 22% to 63% of the population sick, 0.2% to 0.6% hospitalised, and 0.07% to 0.28% dead (n=6,450 to 28,228). '},\n"," {'paper_id': '8d6d3be5f9353eb3358b44e646c83f1a90861b4d',\n","  'section': 'Abstract',\n","  'text': 'Background. Administration of convalescent plasma may be of clinical benefit for treatment of severe acute viral respiratory infections. However, no clear evidence exists to support or oppose convalescent plasma use in clinical practice. We conducted a systematic review and meta-analysis to assess the evidence of randomized controlled trials (RCTs) in the convalescent plasma for the treatment of severe influenza. Methods. Healthcare databases were searched in February 2020. All records were screened against the eligibility criteria. Data extraction and risk of bias assessments were undertaken. The primary outcome was case-fatality rates by influenza. Results. We identified 5 RCTs of severe influenza. The pooled analyses showed no evidence for a reduction in mortality (Odds Ratio (OR) 1.06; 95% confidence interval (CI) 0.51-2.23; p = 0.87; I 2 = 35%). We also found non-significant reductions in days in ICU and hospital, and days on mechanical ventilation. There seemed to have a biological benefit of increasing HAI titer levels and decreasing influenza B virus loads and cytokines after convalescent plasma treatment. No serious adverse events was reported between two groups. Studies were commonly of low risk of bias with high quality. Conclusions. Convalescent plasma appears safe but may not reduce mortality in severe influenza. This therapy should be studied within the context of a well-designed clinical trial for treatment of SARS-Cov-2 infection. CR, Convalescent Plasma Study G. The effectiveness of convalescent plasma and hyperimmune immunoglobulin for the treatment of severe acute respiratory infections of viral etiology: a systematic review and exploratory meta-analysis. J Infect Dis 2015; 211: 80-90. '},\n"," {'paper_id': '8df83cc50766ef007425f79d6220d66c19f73f63',\n","  'section': 'Abstract',\n","  'text': '80 % with an exposure time of 30 seconds. Viral titers are displayed as 50 % tissue culture infectious dose 50 (TCID 50 /mL) values. Cytotoxic effects are displayed as dashed bars are and were calculated analogous to virus infectivity. RFs are included above the bar. The mean of two -three independent experiments with standard deviation are shown. LLOQ: lower limit of quantification. Top inserts: Regression analysis of the inactivation of SARS-CoV-2, bovine CoV (BCoV), SARS-CoV and MERS-CoV by WHO formulation I (A) and II (B). Depicted is the percentage of inactivation at different concentrations. '},\n"," {'paper_id': '8e7c51977c38407e742eb6900ce8be5300cef64f',\n","  'section': 'Abstract',\n","  'text': 'The spread of COVID-19 is expected to put a large strain on many hospital resources, including ICU bed space, and mortuary capacity. In this report we study the possible demands on ICU and mortuary capacity in Sydney, Australia, using an adapted SEIR epidemiological model. * Corresponding author. Author order alphabetical. '},\n"," {'paper_id': '8e91912fad635341704d5df8f36415f6522e98e2',\n","  'section': 'Abstract',\n","  'text': 'Background: Since January 2020, coronavirus disease 2019 has spread rapidly and developing the pandemic model around the world. Data have been needed on the clinical characteristics of the affected patients in an imported cases as model in island outside Wuhan. We conducted a retrospective study included all 168 confirmed cases of Covidconfirmed by real-time RT-PCR and were analysed for demographic, clinical, radiological and laboratory data. Results: Of 168 patients, 160 have been discharged, 6 have died and 2 remain hospitalized. The median age was 51.0 years and 51.8% were females. 129 (76.8%) patients were imported cases, and 118 (70.2%), 51 (30.4%) and 52 (31%) of patients lived in Wuhan or traveled to Wuhan, had contact with Covid-19 patients, or had contact with Wuhan residents, respectively. The most common symptoms at onset of illness were fever (65.5%), dry cough (48.8%) and expectoration (32.1%). On admission, ground-glass opacity was the most common radiologic finding on chest computed tomography (60.2%). The elderly people with diabetes, hypertension and CVD are more likely to develop severe cases. Follow-up of 160 discharged patients found that 20 patients (12.5%) had a positive RT-PCR test results of pharyngeal swabs or anal swabs or fecal. '},\n"," {'paper_id': '8eb99586e9103599ab2f58cc33ca94cd0b60ae53',\n","  'section': 'Abstract',\n","  'text': 'Respiratory tract infections (RTI) are responsible for over 4 million deaths per year worldwide with pathobiont carriage a required precursor to infection. Through a crosssectional community-based nasal self-swabbing study we sought to determine carriage epidemiology for respiratory pathogens amongst bacteria (Streptococcus pneumoniae, Haemophilus influenza, Moraxella catarrhalis, Staphylococcus aureus, Pseudomonas aeruginosa and Neisseria meningitidis) and viruses (RSV, Influenza viruses A and B, Rhinovirus/Enterovirus, Coronavirus, Parainfluenza viruses 1-3 and Adenovirus (ADV)). Carriage of bacterial and viral species was shown to vary with participant age, recent RTI and the presence of other species. The spatial structure of microbial respiratory communities was less nested (more disordered) in the young (0-4 years) and those with recent RTI. Species frequency distributions were flatter than random expectation in young individuals (X 2 = 20.42, p = 0.002), indicating spatial clumping of species consistent with facilitative relationships amongst them. Deviations from a neutral model of ecological niches were observed for samples collected in the summer and from older individuals (those aged 5-17, 18-64 and ≥ 65 years) but not in samples collected from winter, younger individuals (those aged 0-4 years), individuals with recent RTI and individuals without recent RTI, demonstrating the importance of both neutral and niche processes in respiratory community assembly. The application of epidemiological methods and ecological theory to sets of respiratory tract samples has yielded novel insights into the factors that drive microbial community composition, such as seasonality and age, as well as species patterns and interactions within the nose. '},\n"," {'paper_id': '8ecda545cc20407a108cb59018ec349541259da3',\n","  'section': 'Abstract',\n","  'text': 'Province, China has been identified. According to the high prevalence of new coronavirus, further investigation on the clinical and paraclinical features of this disease seems essential. Hence, we carried out this systematic review and meta-analysis to figure out the unknown features. Methods: This study was performed using databases of Web of Science, Scopus and PubMed. We considered English cross-sectional and case-series papers which reported clinical, radiological, and laboratory characteristics of patients with COVID-19. We used STATA v.11 and random effect model for data analysis. In the present meta-analysis, 32 papers including 49504 COVID-19 patients were studied. The most common clinical symptoms were fever (84%), cough (65%) and fatigue (42%), respectively. The most common radiological and paraclinical features were bilateral pneumonia (61%), ground-glass opacity (50%), thrombocytopenia (36%) and lymphocytopenia (34%). The study also showed that the frequency of comorbidities and early symptoms was higher in critically severe patients. Moreover, we found the overall mortality rate of three percent. Conclusion: According to that there are many cases without Computed Tomography Scan findings or clear clinical symptoms, it is recommended to use other confirming methods such RNA sequencing in order to identification of suspicious undiagnosed patients. Moreover, while there is no access to clinical and paraclinical facilities in in public places such as airports and border crossings, it is recommended to consider factors such as fever, cough, sputum and fatigue. '},\n"," {'paper_id': '8ed2df683ca3cde36ebee8185754abf4c44b717b',\n","  'section': 'Abstract',\n","  'text': 'medRxiv preprint 3. Chen N, Zhou M, Dong X., et al., Epidemiological and clinical characteristics of 99 cases of 2019 novel coronavirus pneumonia in Wuhan, China: a descriptive study. Lancet. 2020; 395: 507-513. 4. Yan L, Zhang H-T, Xiao Y, et al, Prediction of survival for severe Covid-19 patients with three clinical features: development of a machine learning-based prognostic model with clinical data in Wuhan, medRxiv, '},\n"," {'paper_id': '8eef6fec694446e150e95dfbef3a3579ae8806d3',\n","  'section': 'Abstract',\n","  'text': 'Hand hygiene is considered as an efficient and cost-effective way to limit the spread of diseases and, as such, it is recommended by both the World Health Organization (WHO) and the Centres for Disease Control and Prevention (CDC). While the effect of hand washing on individual transmissibility of a disease has been studied through medical and public-health research, its potential as a mitigation strategy against a global pandemic has not been fully explored yet. In this study, we investigate contagion dynamics through the world air transportation network and analyze the impact of hand-hygiene behavioural changes of airport population against the spread of infectious diseases worldwide. Using a granular dataset of the world air transportation traffic, we build a detailed individual mobility model that controls for the correlated and recurrent nature of human travel and the waiting-time distributions of individuals at different locations. We perform a Monte-Carlo simulation study to assess the impact of different hand-washing mitigation strategies at the early stages of a global epidemic. From the simulation results we find that increasing the hand cleanliness homogeneously at all airports in the world can inhibit the impact of a potential pandemic by 24 to 69%. By quantifying and ranking the contribution of the different airports to the mitigation of an epidemic outbreak, we identify ten key airports at the core of a cost-optimal deployment of the hand-washing strategy: increasing the engagement rate at those locations alone could potentially reduce a world pandemic by 8 to 37%. This research provides evidence of the effectiveness of hand hygiene in airports on the global spread of infectious diseases, and has important implications for the way public-health policymakers may design new effective strategies to enhance hand hygiene in airports through behavioral changes. All rights reserved. No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/530618 doi: bioRxiv preprint continents 5, 6 . The H1N1 flu, which caused around 300,000 deaths worldwide 7 , had a similar timeline. The first confirmed case of H1N1 was reported in Veracruz, Mexico on April 2009, while within few days the infection migrated to the US and Europe, and two months later the World Health Organisation (WHO) and the Centers for Disease Control and Prevention (CDC) declared the disease as a global pandemic. Bacteria, pathogens and viruses are also transmitted easily at airports or during flights, causing infections and bacterial diseases that can expand to global epidemics. The pathogens are transmitted through the air 8 , resulting in the contagion of airborne diseases, or through physical contact between individuals. The transmission is accelerated when a dense population of people is concentrated in a confined area 9 like an airport, with lack of good hygiene and efficient air ventilation. After an outbreak, infections diffuse while infected individuals transmit the disease to susceptible individuals. Airports play a major role in such contagious dynamics 10 , as they contribute daily to the contact of people from all over the world, some of whom may be carrying endemic infections and bacteria from their country of origin. In addition, there are numerous highly contaminated surfaces which are frequently touched by the passengers at airports and inside aircrafts 11 . Self-service check-in screens, gate bench armrests, water fountain buttons and door handles at airports, as well as seats, tray tables and handles of lavatories in aircrafts, are all known to have high microbial contamination 12-14 . Mitigation strategies are designed and implemented to inhibit a global pandemic. At the individual level there is a focus on behavioral change towards adopting different interventions in the case of a health emergency 15, 16 . Along with other developments in medicine, vaccination has made a big contribution in that direction leading to the extinction of past epidemics and a significant reduction of mortality due to specific infections 17 . Vaccination has a substantial mitigating effect when effective vaccines are available soon enough after the emergence of a new disease and when vaccination campaigns cover about 70% of a susceptible population 18 . However, despite the known impact of vaccines on the reduction of infections, the rate of vaccination in the population has remained unchanged over the past decade 19 . Social nudges such as peer effects or education on vaccination benefits, and changes in the design of vaccination campaigns, can be deployed to change human behavior towards the increase of influenza vaccination rates 20 . In addition of preventing disease spreading by vaccination, isolating patients at home or closure of high-risk places like schools can moderate the transmission of disease-causing pathogenic microorganisms. Several actions within the world air transportation network can be implemented to control a disease spreading in the case of a health emergency 21 . At the global scale, mobility-driven interventions such as airport closures and deliberate rerouting of the travelers, can reduce the number of individuals passing through or traveling from/to regions where dangerous diseases prevail 22 . At the local scale, actions at individual airports such as frequent cleaning of public areas (e.g. toilets, gates, check-in desks, etc.), efficient air ventilation and enhanced sensitization of frequently touched surfaces can reduce the risk of contamination and the transmission of infections. Furthermore, personal hygiene is the most important factor to prevent the spread of an infection 23 . Coughing etiquette, face masks, no face touch and hand hygiene are the most common actions that air travelers can easily adopt. From those actions, hand washing is the simplest and most effective component for preventing the transmission of viruses 24-27 and bacteria, and is regularly mentioned as the first recommendation of precautions in health care (see for example 28 ). A scientific study on the effects of hand washing on the bacterial contamination of hands, showed that after a deliberate contamination of individuals by touching door handles and railings in public places, bacteria were found in 44% of the sample, while this percentage was reduced to 23% after hand washing with water only, and to 8% after hand washing with water and plain soap 29 . The same study concluded also that the effect of hand washing does not depend on the bacteria species. While hand hygiene is considered as the first prevention step in the case of an epidemic emergency, there is lack of evidence for its effects as a mitigation strategy against global epidemic spreading. In this work, we study contagion dynamics through the world air transportation network and we elucidate the impact of hand-hygiene behavioral changes on the diffusion of infections worldwide. We develop a computational model to track the realistic mobility patterns of air travelers, their hand washing behavior and the dynamics of epidemic spreading. Human mobility is modelled by a stochastic agent-base system that accounts for the spatial distribution of airports, the realistic human mobility through the world air transportation network, and the waiting-time distributions of individuals at origin, destination and connecting airports. In addition, we develop a compartmental epidemic model which relates the disease spreading with hand-hygiene behaviour. Using Monte-Carlo simulations we assess the impact of hand washing at the early stages of a global epidemic spreading. From the simulation results, we measure the early-time spreading power of the 40 busiest airports under four different intervention scenarios: 1. increase of hand washing engagement homogeneously at all airports; 2. increase of hand washing engagement only at the source of the disease; 3. increase of hand washing engagement at the ten most important airports of the world air transportation network; and 4. increase of hand washing engagement at the ten most important airports for each source of the disease. The aim of this study is to identify the most effective mitigation strategy of hand hygiene contributing the most to the reduction of global epidemic spreading. '},\n"," {'paper_id': '8f1a1ebca4f4c9bcbc88f193f26f33f908eeabac',\n","  'section': 'Abstract',\n","  'text': \"We calculated the structural properties of >2500 coronaviruses and computed >100000 human 26 protein interactions with severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Using 27 the CROSS method, we found that the SARS-CoV-2 region encompassing nucleotides 23000 and 28 24000 is highly conserved at the structural level, while the region 1000 nucleotides up-stream varies 29 significantly. The two sequences code for a domain of the spike S protein that binds to the host 30 receptor angiotensin-converting enzyme 2 (ACE2) that mediates human infection and in the 31 homologue from Middle East respiratory syndrome coronavirus (MERS-CoV) interacts with sialic 32 acids. Highly structured regions are predicted at the 5' and 3' where our calculations also indicate 33 strong propensity to bind to human proteins. Using the catRAPID method, we calculated 3500 34 interactions with the 5' and identified Cyclin T1 CCNT1, ATP-dependent RNA helicase DDX1, 35 author/funder. All rights reserved. No reuse allowed without permission. \"},\n"," {'paper_id': '8f33e07fdd716b8edeab35563c9f27b63389debc',\n","  'section': 'Abstract',\n","  'text': 'Elucidating antiviral CD8 T lymphocyte responses to SARS-CoV-2 may shed light on the heterogeneity of clinical outcomes and inform vaccine or therapeutic approaches. To facilitate the evaluation of antiviral CD8 T cell responses to SARS-CoV-2, we generated a publicly accessible database of epitopes predicted to bind any class I HLA protein across the entire SARS-CoV-2 proteome. While a subset of epitopes from earlier betacoronaviruses, such as SARS-CoV (SARS), have been validated experimentally, validation systems are often biased toward specific HLA haplotypes (notably HLA-A*02:01) that only account for a fraction of the haplotypes of individuals affected by the SARS-CoV-2 pandemic. To enable evaluation of epitopes across individuals with a variety of HLA haplotypes, we computed the predicted binding affinities between 9-mer peptides derived from the annotated SARS-CoV-2 peptidome across 9,360 MHC class I HLA-A, -B, and -C alleles. There were 6,748 unique combinations of peptides and HLA alleles (pMHCs) with a predicted binding affinity of less than 500nM, including 1,103 unique peptides and 1,022 HLA alleles, spanning 11 annotated superfamilies. These peptides were derived from all 11 proteins spanning the SARS-CoV-2 peptidome, including peptides that have previously been validated experimentally. We also show evidence that these previously '},\n"," {'paper_id': '8f4b98da50277e6c9516aef0227f5cca9af6b5e8',\n","  'section': 'Abstract',\n","  'text': 'Zika virus is spread mainly by the bite of an infected mosquito, which can be passed from a pregnant woman to her fetus, thus leading to birth defects including more than microcephaly. It has been recently estimated that one-third of the world population will be infected by Zika in the near future, but unfortunately so far there is no vaccine or medicine for Zika. In particular, the special concern on the vaccine treatment to Zika and Dengue arising from antibody-dependent enhancement strongly emphasizes the key role of its NS2B-NS3 protease (NS2B-NS3pro) as a target for anti-Zika drug discovery/design due to its absolutely-essential role in viral replication. In response to the current global health emergency triggered by the Zika outbreak, we successfully obtained several active forms of Zika NS2B-NS3pro and further attempted to discover its inhibitors from eatable plants and traditional herbal medicines to immediately fight Zika. Here, for the first time, we discovered that quercetin, a flavonoid extensively existing in many fruits and vegetables, effectively inhibits Zika NS2B-NS3pro. We further quantify its inhibitory activity with IC 50 of 26.0 ± 0.1 µM; and K i of 23.0 ± 1.3 µM. As quercetin has been extensively found in fruits, vegetables, leaves and grains, our discovery would benefit the public to immediately fight Zika. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '8f6833be2037907eecb0edd1d2b536c583d36444',\n","  'section': 'Abstract',\n","  'text': 'The coronavirus disease 2019 ; one of the most hazardous threats that the world has ever been faced, is now increasing exponentially worldwide. An increasing proportion of the women are now infected with this virus during their pregnancy, which may put them in danger in terms of adverse maternal and newborn outcomes. The aims of this systematic review were to summarize the possible symptoms, treatments, and pregnancy outcomes if the women are infected with COVID-19 during their pregnancy. '},\n"," {'paper_id': '8fdf4054b1e75ce6ec43d6284f16e9b49d6f3e1d',\n","  'section': 'Abstract',\n","  'text': 'Although genome sequences of SARS-CoV-2 are growing daily and contain sufficient phylogenetic information, country-specific data still present severe limitations and should be interpreted with caution. The SARS-CoV-2 pandemic has been growing exponentially, affecting nearly 900 thousand people and causing enormous distress to economies and societies worldwide. A plethora of analyses based on viral sequences has already been published, in scientific journals as well as through non-peer reviewed channels, to investigate SARS-CoV-2 genetic heterogeneity and spatiotemporal dissemination. We examined full genome sequences currently available to assess : bioRxiv preprint the presence of sufficient information for reliable phylogenetic and phylogeographic studies in countries with the highest toll of confirmed cases. Although number of-available full-genomes is growing daily, and the full dataset contains sufficient phylogenetic information that would allow reliable inference of phylogenetic relationships, country-specific SARS-CoV-2 datasets still present severe limitations. Studies assessing within country spread or transmission clusters should be considered preliminary at best, or hypothesis generating. Hence the need for continuing concerted efforts to increase number and quality of the sequences required for robust tracing of the epidemic. '},\n"," {'paper_id': '901fdffc89f57ad2efc39cf3e06c3b28ae154427',\n","  'section': 'Abstract',\n","  'text': 'Objective To review and critically appraise published and preprint reports of models that aim to predict either (i) presence of existing COVID-19 infection, (ii) future complications in individuals already diagnosed with or (iii) models to identify individuals at high risk for COVID-19 in the general population. Design Rapid systematic review and critical appraisal of prediction models for diagnosis or prognosis of COVID-19 infection. Data sources PubMed, EMBASE via Ovid, Arxiv, medRxiv and bioRxiv until 24 th March 2020. Study selection Studies that developed or validated a multivariable COVID-19 related prediction model. Two authors independently screened titles, abstracts and full text. Data from included studies were extracted independently by at least two authors based on the CHARMS checklist, and risk of bias was assessed using PROBAST. Data were extracted on various domains including the participants, predictors, outcomes, data analysis, and prediction model performance. Results 2696 titles were screened. Of these, 27 studies describing 31 prediction models were included for data extraction and critical appraisal. We identified three models to predict hospital admission from pneumonia and other events (as a proxy for covid-19 pneumonia) in the general population; 18 diagnostic models to detect COVID-19 infection in symptomatic individuals (13 of which were machine learning utilising computed tomography (CT) results); and ten prognostic models for predicting mortality risk, progression to a severe state, or length of hospital stay. Only one of these studies used data on COVID-19 cases outside of China. Most reported predictors of presence of COVID-19 in suspected patients included age, body temperature, and signs and symptoms. Most reported predictors of severe prognosis in . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint . https://doi.org/10.1101/2020.03.24.20041020 doi: medRxiv preprint 3 infected patients included age, sex, features derived from CT, C-reactive protein, lactic dehydrogenase, and lymphocyte count. Estimated C-index estimates for the prediction models ranged from 0.73 to 0.81 in those for the general population (reported for all 3 general population models), from 0.81 to > 0.99 in those for diagnosis (reported for 13 of the 18 diagnostic models), and from 0.85 to 0.98 in those for prognosis (reported for 6 of the 10 prognostic models). All studies were rated at high risk of bias, mostly because of non-representative selection of control patients, exclusion of patients who had not experienced the event of interest by the end of the study, and poor statistical analysis, including high risk of model overfitting. Reporting quality varied substantially between studies. A description of the study population and intended use of the models was absent in almost all reports, and calibration of predictions was rarely assessed. Conclusion COVID-19 related prediction models are quickly entering the academic literature, to support medical decision making at a time where this is urgently needed. Our review indicates proposed models are poorly reported and at high risk of bias. Thus, their reported performance is likely optimistic and using them to support medical decision making is not advised. We call for immediate sharing of the individual participant data from COVID-19 studies to support collaborative efforts in building more rigorously developed prediction models and validating (evaluating) existing models. The aforementioned predictors identified in multiple included studies could be considered as candidate predictors for new models. We also stress the need to follow methodological guidance when developing and validating prediction models, as unreliable predictions may cause more harm than benefit when used to guide clinical decisions. Finally, studies should adhere to the TRIPOD statement to facilitate validating, appraising, advocating and clinically using the reported models. '},\n"," {'paper_id': '905603e9db862f9b4b42a95f51e9c8d0015d1dff',\n","  'section': 'Abstract',\n","  'text': 'Since the first identified individual of 2019 novel coronavirus (COVID-19) infection on Jan 20, 2020 in South Korea, the number of confirmed cases rapidly increased. As of Feb 26, 2020, 1,261 cases of COVID-19 including 12 deaths were confirmed in South Korea. Using the incidence data of COVID-19, we estimate the reproduction number at 1.5 (95% CI: 1.4-1.6), which indicates sustained transmission and support the implementation of social distancing measures to rapidly control the outbreak. '},\n"," {'paper_id': '9059282d74c84d0d388790ce43fccaef4389641e',\n","  'section': 'Abstract',\n","  'text': 'The purpose of the present paper is to present simple estimation and prediction methods for basic quantities in an emerging epidemic like the ongoing covid-10 pandemic. The simple methods have the advantage that relations between basic quantities become more transparent, thus shedding light to which quantities have biggest impact on predictions, with the additional conclusion that uncertainties in these quantities carry over to high uncertainty also in predictions. A simple non-parametric prediction method for future cumulative case fatalities, as well as future cumulative incidence of infections (assuming a given infection fatality risk f ), is presented. The method uses cumulative reported case fatalities up to present time as input data. It is also described how the introduction of preventive measures of a given magnitude ρ will affect the two incidence predictions, using basic theory of epidemic models. This methodology is then reversed, thus enabling estimation of the preventive magnitude ρ, and of the resulting effective reproduction number R E . However, the effects of preventive measures only start affecting case fatalities some 3-4 weeks later, so estimates are only available after this time has elapsed. The methodology is applicable in the early stage of an outbreak, before, say, 10% of the community have been infected. Beside giving simple estimation and prediction tools for an ongoing epidemic, another important conclusion lies in the observation that the two quantities f (infection fatality risk) and ρ (the magnitude of preventive measures) have very big impact on predictions. Further, both of these quantities currently have very high uncertainty: current estimates of f lie in the range 0.2% up to 2% ([9], [7]), and the overall effect of several combined preventive measures is clearly very uncertain. The two main findings from the paper are hence that, a) any prediction containing f , and/or some preventive measures, contain a large amount of uncertainty (which is usually 2 Two very small changes from version dated March 27: We stress that the community conidered is suitable for an urban area without big spatial distances, and that uncertainty in s d affects time locations but not predicited numbers. '},\n"," {'paper_id': '905ff29124af7e2b7698a6816dee5f1ee7065a85',\n","  'section': 'Abstract',\n","  'text': \"This paper provides a model-based method for the forecast of the total number of currently CoVoD-19 positive individuals and of the occupancy of the available Intensive Care Units in Italy. The predictions obtained -for a time horizon of 10 days starting from March 29th -will be provided at a national as well as at a more disaggregate levels, following a criterion based on the magnitude of the phenomenon. While the Regions which have been hit the most by the pandemic have been kept separated, the less affected ones have been aggregated into homogeneous macro-areas. Results show that -within the forecast period considered (March 29 th -April 7 th )all of the Italian regions will show a decreasing number of CoViD-19 positive people. Same for the number of people who will need to be hospitalized in a Intensive Care Unit (ICU). These estimates are valid under constancy of the Government's current containment policies. In this scenario, Northern Regions will remain the most affected ones and no significant outbreak are foreseen in the southern regions. \"},\n"," {'paper_id': '906b616191eea5e5e724d47546f7d2d9d096ed7a',\n","  'section': 'Abstract',\n","  'text': 'The role of microRNAs (miRNAs) in cellular processes captured the attention of many researchers, since their dysregulation is shown to affect the cancer disease landscape by sustaining proliferative signaling, evading program cell death, and inhibiting growth suppressors. Thus, miRNAs have been considered important diagnostic and prognostic biomarkers for several types of tumors. Machine learning algorithms have proven to be able to exploit the information contained in thousands of miRNAs to accurately predict and classify cancer types. Nevertheless, extracting the most relevant miRNA expressions is fundamental to allow human experts to validate and make sense of the results obtained by automatic algorithms. We propose a novel feature selection approach, able to identify the most important miRNAs for tumor classification, based on consensus on feature relevance from high-accuracy classifiers of different typologies. The proposed methodology is tested on a real-world dataset featuring 8,129 patients, 29 different types of tumors, and 1,046 miRNAs per patient, taken from The Cancer Genome Atlas (TCGA) database. A new miRNA signature is suggested, containing the 100 most important oncogenic miRNAs identified by the presented approach. Such a signature is proved to be sufficient to identify all 29 types of cancer considered in the study, with results nearly identical to those obtained using all 1,046 features in the original dataset. Subsequently, a meta-analysis of the medical literature is performed to find references to the most important biomarkers extracted by the methodology. Besides known oncomarkers, 15 new miRNAs previously not ranked as important biomarkers for diagnosis and prognosis in cancer pathologies are uncovered. Such miRNAs, considered relevant by the machine learning algorithms, but still relatively unexplored by specialized literature, could provide further insights in the biology of cancer. MicroRNAs (miRNAs) are non-coding RNA molecules that regulate gene expression. In the last years, the under and over expression of miRNAs has been related to the diagnosis and prognosis of specific cancer types. While machine learning techniques can efficiently exploit the information contained in thousands of miRNAs to detect the '},\n"," {'paper_id': '907da65e13cde5d7ad71f304967a7ab9dcb5efa1',\n","  'section': 'Abstract',\n","  'text': 'PARP2 belongs to a family of proteins involved in cell differentiation, DNA 15 damage repair, cellular energy expenditure, chromatin modeling and cell differentiation. 16 In addition to these overlapping functions with PARP1, PARP2 participates in 17 spermatogenesis, T-cell maturation, extraembryonic endoderm formation and 18 adipogenesis. The function(s) of PARP2 is far from complete, and the mechanism(s) by 19 which the gene and protein are regulated are unknown. In this study, we found that two 20 different mechanisms are used in vitro to regulate PARP2 levels. In the presence of serum, 21 PARP2 is degraded through the ubiquitin-proteasome pathway, however, when serum is 22 removed, PARP2 is rapidly sequestered into an SDS-and urea-insoluble fraction. This 23 sequestration is relieved by serum in a dose-dependent manner, and again PARP2 is 24 detected by immunoblotting. Furthermore, and despite the presence of a putative serum 25 response element in the PARP2 gene, transcription is not affected by serum deprivation. 26 These observations that PARP2 is tightly regulated by distinct pathways highlights the 27 critical roles PARP2 plays under different physiological conditions. 28 29 '},\n"," {'paper_id': '90e28ff462882ca7a9329beb879f73c2e99430e4',\n","  'section': 'Abstract',\n","  'text': 'Cilia are organelles specialized for movement and signaling. To infer when during animal 24 evolution signaling pathways became associated with cilia, we characterized the proteomes of 25 cilia from three organisms: sea urchins, sea anemones and choanoflagellates. From these 26 ciliomes, we identified 437 high confidence ciliary candidate proteins conserved in mammals, 27 including known regulators of Hh, GPCR and TRP channel signaling. The phylogenetic profiles 28 of their ciliary association indicate that the Hh and GPCR pathways were linked to cilia before 29 the origin of bilateria and TRP channels before the origin of animals. We demonstrated that some 30 of the candidates not previously implicated in ciliary biology localized to cilia and further 31 investigated ENKUR, a TRP channel-interacting protein that we identified in the cilia of all three 32 organisms. In animals, ENKUR is expressed by cells with motile cilia, ENKUR localizes to cilia 33 in diverse organisms and, in both Xenopus laevis and mice, ENKUR is required for patterning 34 the left/right axis. Moreover, mutation of ENKUR causes situs inversus in humans. Thus, 35 proteomic profiling of cilia from diverse eukaryotes defines a conserved ciliary proteome, 36 reveals ancient connections to Hh, GPCR and TRP channel signaling, and uncovers a novel 37 ciliary protein that controls vertebrate development and human disease. 38 39 such as light and chemical cues such as odors (Bloodgood, 2010; Malicki and Johnson, 2016) . 48 Cilia and the related structures, flagella, are comprised of microtubular cores called 49 axonemes that emanate from basal bodies and are ensheathed by membranes that are 50 compositionally distinct from the contiguous plasma membrane 51 '},\n"," {'paper_id': '91098f6fe46a21565bf0cb06fe960cdb2c3f5e38',\n","  'section': 'Abstract',\n","  'text': '1 Background Since December 2019, pneumonia associated with the 2019 novel 2 coronavirus (2019-nCoV) has emerged in Wuhan, China. The exponential increase of 3 the confirmed number of cases of 2019n-CoV is of great concern to the global 4 community. The fears and panic among residents in the epicenters have prompted 5 diverse responses, which are understudied. During such a crisis, community trust and 6 support for the government and health authorities are important to contain the outbreak. 7 We aimed to investigate the influence of institutional trust on public responses to the 8 2019-nCoV outbreak. 9 Methods An anonymous Internet-based, cross-sectional survey was administered on 10 January 29, 2020. The study population comprised all residents currently residing or 11 working in the province of Hubei, where Wuhan is the capital city. The level of trust in 12 information provision and preventive instructions, individual preventive behaviors and 13 treatment-seeking behaviors were queried. 14 Findings The majority of the participants expressed a great extent of trust in the 15 information and preventive instructions provided by the central government than by the 16 local government. A high uptake of 2019-nCoV preventive measures was found, 17 particularly among people who had been placed under quarantine. Being under 18 quarantine (adjusted odds ratio [OR] = 2.35, 95% confidence interval [CI] 1.80 to 3.08) 19 and having a high institutional trust score (OR = 2.23, 95% CI 1.96 to 2.53) were both 20 strong and significant determinants of higher preventive behavior scores. The majority 21 of study participants (85.7%, n = 3,640) reported that they would seek hospital 22 treatment if they suspected themselves to have been infected with 2019 n-CoV. Few of 23 the participants from Wuhan (16.6%, n = 475) and those participants who were under 24 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': '91233a0c35364955d1fdbf2f980f7220f9bafc70',\n","  'section': 'Abstract',\n","  'text': 'Commercial multiplex assays, built on different chemistries and platforms are widely 25 available for simultaneous detection of pathogens that cause respiratory infections. However, 26 these tests are often difficult to implement in a resource limited setting because of high cost. In 27 this study, we developed and validated a method for simultaneous testing of common respiratory 28 pathogens (Respanel) by real-time PCR in a convenient, strip-tube array format. Primers and 29 probes for sixteen PCR assays were selected from the literature or newly designed. Following 30 optimization of individual PCR assays, strip-tube arrays were prepared by dispensing primer-31 probe mixes (PPM) into two sets of 8-tube strips. Nucleic acid extracts from specimens were 32 mixed with PCR master mix, and dispensed column-wise into 2X8-wells of a 96-well plate. PPMs from strip-tubes were then added to the wells using a multichannel pipette for real-time 34 PCR. Individual PCR assays were optimized using previously known specimens (n=397) with 35 91%-100% concordance with culture, DFA or PCR results. Respanel was then tested in a routine 36 manner at two different sites using specimens (n=147) previously tested by Qiagen Resplex I&II 37 or Fast-Track Diagnostics Respiratory Pathogens 21 assays. The sensitivity, specificity and 38 accuracy of Respanel were 94%, 95% and 95%, respectively, against Resplex and 88%, 100% 39 and 99%, respectively, against FTDRP21. Respanel detected 48% more pathogens (p<0.05) than 40 Resplex but the rate of pathogen detection was not significantly different from FTDRP21. Respanel is a convenient and inexpensive assay that is more sensitive than Resplex and 42 comparable to FTDRP21 for the detection of common respiratory pathogens. 43 44 45 46 . CC-BY-NC-ND 4.0 International license is made available under a '},\n"," {'paper_id': '9172627d76e007ee80e252c0b40204be0d9f09a2',\n","  'section': 'Abstract',\n","  'text': 'A novel pneumonia-associated respiratory syndrome named coronavirus disease-2019 (COVID-19), which caused by SARS-CoV-2 and broken in Wuhan, China in the end of 2019. Unfortunately, there is no specific antiviral agent or vaccine available to treat SARS-CoV-2 infections. Also, information regarding the immunological '},\n"," {'paper_id': '9211b1703c76e20afdee8fcb9f7996edadef0e9e',\n","  'section': 'Abstract',\n","  'text': 'The Middle East respiratory syndrome coronavirus (MERS-CoV) developed infections that caused serious epidemics. Special priority of awareness and prompt initiative involvement of health workers (HW) during such intensified health situation necessitated an assessment of their preparedness, appropriate attitudes and protective strategies for better efficiency modes and acceleration during emergency. Methods Sultan Military Medical city in Riyadh, Saudi Arabia were reviewed through specifically designed questionnaires to acquire the demanded data. It included clinical and demographic information about the viral diseases, associated signs and symptoms, transmission and protection, and attitudes about the MERS-CoV disease. The study was accomplished between August and November, 2017 and 477 participants of the medical city, successfully completed the study questionnaire (Appendix I ). Females represented a majority and there was an indirectly proportional decrease with the increasing age. Gradual educational increase levels reflected dominance of the university degree holders. Jobs were dominated by nurses and non-Saudis were a majority whilst, the highly experienced, (>10 years) were a minority. A majority recognized the viral transmission methods, popular information sources of MERS-CoV and associated medical terms. Highest scores were observed in dealing with protective aids and recognizing symptoms of disease. High adherence to hand hygiene protocols and correct washing steps were recorded. Correct and high levels were observed in taking preventive measures and avoiding infection. Participants responded correctly to negative and wrong actions that patients should refrain from. High scores were observed in taking appreciable attitudes towards oneself and towards others. Expatriates were majority and nurses were dominant which, necessitates Saudization of this sector. Ministry of health pamphlets and seminars were of less impact in invigilating HW, hence, more attention and efforts are demanded. HW were quite aware of the basic and emergent health policies during epidemic episodes of MERS-CoV. '},\n"," {'paper_id': '9231e9d6dd98dca26e67eaf02eea8c0ad0bcce3b',\n","  'section': 'Abstract',\n","  'text': \"Emergency school closures are often used as public health interventions during infectious disease outbreaks in an attempt to minimise the spread of infection. However, if children continue to mix with others outside the home during the closures, these measures are unlikely to be effective. This review aimed to summarise existing literature on children's activities and contacts made outside the home during unplanned school closures. We searched four databases from inception to February 2020 for relevant literature. Main findings were extracted. \"},\n"," {'paper_id': '923c5ba5383afc638125021aed55691fa4ebccbe',\n","  'section': 'Abstract',\n","  'text': 'The recent outbreak by SARS-CoV-2 has generated a chaos in global health and economy and claimed/infected a large number of lives. Closely resembling with SARS CoV, the present strain has manifested exceptionally higher degree of spreadability, virulence and stability possibly due to some unidentified mutations. The viral spike glycoprotein is very likely to interact with host Angiotensin-Converting Enzyme 2 (ACE2) and transmits its genetic materials and hijacks host machinery with extreme fidelity for self propagation. Few attempts have been made to develop a suitable vaccine or ACE2 blocker or virus-receptor inhibitor within this short period of time. Here, attempt was taken to develop some therapeutic and vaccination strategies with a comparison of spike glycoproteins among SARS-CoV, MERS-CoV and the SARS-CoV-2. We verified their structure quality (SWISS-MODEL, Phyre2, Pymol) topology (ProFunc), motifs (MEME Suite, GLAM2Scan), gene ontology based conserved domain (InterPro database) and screened several epitopes (SVMTrip) of SARS CoV-2 based on their energetics, IC50 and antigenicity with regard to their possible glycosylation and MHC/paratopic binding (Vaxigen v2.0, HawkDock, ZDOCK Server) effects. We screened here few pairs of spike protein epitopic regions and selected their energetic, IC50, MHC II reactivity and found some of those to be very good target for vaccination. A possible role of glycosylation on epitopic region showed profound effects on epitopic recognition. The present work might be helpful for the urgent development of a suitable vaccination regimen against SARS CoV-2. '},\n"," {'paper_id': '925b9d04db0132612913108f5a119d9b5a5aa877',\n","  'section': 'Abstract',\n","  'text': \"Accurate and rapid diagnosis of COVID-19 suspected cases plays a crucial role in timely quarantine and medical treatment. Developing a deep learning-based model for automatic COVID-19 detection on chest CT is helpful to counter the outbreak of SARS-CoV-2. A weakly-supervised deep learning-based software system was developed using 3D CT volumes to detect COVID-19. For each patient, the lung region was segmented using a pre-trained UNet; then the segmented 3D lung region was fed into a 3D deep neural network to predict the probability of COVID-19 infectious. 499 CT volumes collected from Dec. 13, 2019, to Jan. 23, 2020, were used for training and 131 CT volumes collected from Jan 24, 2020, to Feb 6, 2020, were used for testing. The deep learning algorithm obtained 0.959 ROC AUC and 0.976 PR AUC. There was an operating point with 0.907 sensitivity and 0.911 specificity in the ROC curve. When using a probability threshold of 0.5 to classify COVID-positive and COVID-negative, the algorithm obtained an accuracy of 0.901, a positive predictive value of 0.840 and a very high negative predictive value of 0.982. The algorithm took only 1.93 seconds to process a single patient's CT volume using a dedicated GPU. Our weaklysupervised deep learning model can accurately predict the COVID-19 infectious probability in chest CT volumes without the need for annotating the lesions for training. The easily-trained and highperformance deep learning algorithm provides a fast way to identify COVID-19 patients, which is beneficial to control the outbreak of SARS-CoV-2. The developed deep learning software is available at https://github.com/sydney0zq/covid-19-detection. \"},\n"," {'paper_id': '9261e7a9a4d3febda29d574074a79f02376249a3',\n","  'section': 'Abstract',\n","  'text': 'The authors find that human cells re-entering the cell cycle from quiescence have both an impaired p53-dependent DNA replication origin licensing checkpoint and slow origin licensing. This combination makes every first S phase underlicensed and hypersensitive to replication stress. To maintain tissue homeostasis, cells transition between cell cycle quiescence and proliferation. An essential G1 process is Minichromosome Maintenance complex (MCM) loading at DNA replication origins to prepare for S phase, known as origin licensing. A p53-dependent origin licensing checkpoint normally ensures sufficient MCM loading prior to S phase entry. We used quantitative flow cytometry and live cell imaging to compare MCM loading during the long first G1 upon cell cycle entry and the shorter G1 phases in the second and subsequent cycles. We discovered that despite the longer G1 phase, the first G1 after cell cycle re-entry is significantly underlicensed. As a result, the first S phase cells are hypersensitive to replication stress. This underlicensing is from a combination of slow MCM loading with a severely compromised origin licensing checkpoint. The hypersensitivity to replication stress increases over repeated rounds of quiescence. Thus, underlicensing after cell cycle re-entry from quiescence distinguishes a higher risk cell cycle that promotes genome instability. '},\n"," {'paper_id': '92860a0f4425d12aafb8e096d8e65d38eb67dff5',\n","  'section': 'Abstract',\n","  'text': 'Ebolaviruses are non-segmented, negative-sense RNA viruses (NNSVs) within the order 19 Mononegavirales that possess the multifunctional virion protein 35 (VP35), a major determinant 20 of virulence and pathogenesis that is indispensable for viral replication and host innate immune 21 evasion. VP35 is functionally equivalent to the phosphoprotein (P) of other mononegaviruses 22 such as rhabdoviruses and paramyxoviruses. Phosphorylation of the P protein is universally 23 regarded as functionally important however, a regulatory role(s) of phosphorylation on VP35 24 function remains unexplored. Here, we identified a highly conserved Ser129 residue near the 25 homo-oligomerization coiled coil motif, which is essential for VP35 functions. Affinity-purification 26 MS followed by post-translational modification (PTM) analysis predicted phosphorylation of 27 Ser129. Co-immunoprecipitation, cross-linking, and biochemical characterization studies 28 revealed a moderately decreased capacity of VP35-S129A to oligomerize. Functional analysis 29 showed that Ser-to-Ala substitution of Ebola virus (EBOV) VP35 did not affect IFN inhibitory 30 activity but nearly abolished EBOV minigenome activity. Further coimmunoprecipitation studies 31 demonstrated a lost interaction between VP35-S129A and the amino terminus of the viral 32 polymerase but not between viral nucleoprotein (NP) or VP35-WT. Taken together, our findings 33 provide evidence that phosphorylation modulates VP35 function, supporting VP35 as a NNSV P 34 protein and providing a potentially valuable therapeutic target. 35 Importance 36 Ebola virus (EBOV) can cause severe disease in humans. The 2013-2016 West African 37 epidemic and the two recent outbreaks in the Democratic Republic of the Congo underscore the 38 urgent need for effective countermeasures, which remain lacking. A better understanding of 39 EBOV biology and the modulation of multifunctional viral proteins is desperately needed to 40 develop improved therapeutics. We provide evidence here that function of virion protein 35 41 (VP35) is modulated by phosphorylation of Ser129, a conserved residue among other 42 ebolavirus species. These findings shed light on EBOV biology and present a potential target for 43 broad acting anti-ebolavirus therapeutics. 44 45 46 47 48 49 '},\n"," {'paper_id': '92acb561ba0ebfbeb8efb529279ae659dcd9a548',\n","  'section': 'Abstract',\n","  'text': 'A SIRU-type epidemic model is proposed for the prediction of COVID-19 spreading within Brasil, and analyse the influence of public health measures on simulating the control of this infectious disease. Since the reported cases are typically only a fraction of the total number of the symptomatic infectious individuals, the model accounts for both reported and unreported cases. Also, the model allows for the time variation of both the transmission rate and the fraction of asymptomatic infectious that become reported symptomatic individuals, so as to reflect public health interventions, towards its control, along the course of the epidemic evolution. An analytical exponential behaviour for the accumulated reported cases evolution is assumed at the onset of the epidemy, for explicitly estimating initial conditions, while a Bayesian inference approach is adopted for parametric estimations employing the present direct problem model with the data from the known portion of the epidemics evolution, represented by the time series for the reported cases of infected individuals. The direct-inverse problem analysis is then employed with the actual data from China, with the first half been employed for the parametric estimation and the second half for validation of the predictive capability of the proposed approach. The full dataset for China is then employed in another parameter identification, aimed at refining the values for the average times that asymptomatic infectious individuals and that symptomatic individuals remain infectious. Following this validation, the available data on reported cases in Brasil from February 15 th till March 29 th , 2020, is used for estimating parameters and then predict the epidemy evolution under these conditions. Finally, public health interventions are simulated, aimed at diminishing the effects of the disease spreading, by acting on both the transmission rate and the fraction of the total number of the symptomatic infectious individuals, considering time variable exponential behaviours for these two parameters, usually assumed constant in epidemic evolutions without intervention. It is demonstrated that a combination of actions to affect both parameters can have a much faster and effective result in the control of the epidemy dynamics. : medRxiv preprint the analysis of the impact of public health measures [1], which however did not consider in the modelling the presence of unreported infection cases, which are in practice inherent to this process. The present work is first based on the SIR-type model proposed in [2] , which deals with the epidemic outbreak in Wuhan by introducing the unreported cases in the modelling, and evaluating the consequences of public health interventions. It was a direct application of previous developments [3, 4] on the fundamental problem of parameter identification in mathematical epidemic models, accounting for unreported cases. This same modelling approach was more recently employed in the analysis of the epidemic outbreak in different countries, '},\n"," {'paper_id': '92c020779e68e82d9a5f5898435e618caa63a099',\n","  'section': 'Abstract',\n","  'text': 'Faced with the current large-scale public health emergency, collecting, sorting, and analyzing biomedical information related to the \"coronavirus\" should be done as quickly as possible to gain a global perspective, which is a basic requirement for strengthening epidemic control capacity. However, for human researchers studying the viruses and the hosts, the vast amount of information available cannot be processed effectively and in a timely manner, particularly when the scientific understanding may be limited, which can further lower the information processing efficiency. We present TWIRLS, a method that can automatically acquire, organize, and classify information. Additionally, independent functional data sources can be added to build an inference system using a machine-based approach, which can provide relevant knowledge to help human researchers quickly establish subject cognition and to make more effective decisions. TWIRLS can automatically analyze more than three million words in more than 14,000 literature articles in only 4 hours. Combining with generalized gene interaction databases creates a data interface that can help researchers to further analyze the information. Using the TWIRLS system, we found that an important regulatory factor angiotensin-converting enzyme 2 (ACE2) may be involved in the author/funder. All rights reserved. No reuse allowed without permission. : bioRxiv preprint host pathological changes on binding to the coronavirus after infection. After triggering functional changes in ACE2/AT2R, an imbalance in the steady-state cytokine regulatory axis involving the Renin-Angiotensin System and IP-10 leads to a cytokine storm. '},\n"," {'paper_id': '931fd6847689ae16129aa6b52df3ee368354cbc7',\n","  'section': 'Abstract',\n","  'text': 'Background: The first outbreaks of COVID-19 in Italy occurred during the second half of February 2020 in some areas in the North of the country. Due to the high contagiousness of the infection, further spread by asymptomatic people, Italy has become in a few weeks the country with the greatest number of infected people after China. The large number of severe cases among infected people in Italy led to the hospitalization of thousands of patients, with a heavy burden on the National Health Service. February 2020 to 26 March 2020. Considering such information, we developed a forecast model in real-time, based on the cumulative logistic distribution. We then produced an estimate of the overall number of potentially infected individuals and epidemic duration at a national and Regional level, for the most affected Regions. We reported the daily distribution of performed swabs and confirmed cases, and the cumulative distribution of confirmed cases, of patients quarantined at home (42%), hospitalized in non-intensive care (31%), recovered or discharged (13%), deceased (10%), and hospitalized in intensive care (4%). The forecast model estimated a number of infected persons for Italy of 115,000 about, and a duration of the epidemic not less than 2 months. Once month after the first outbreaks there seems to be the first signs of a decrease in the number of infections, showing that we could be now facing the descending phase of the epidemic. The forecast obtained thanks to our model could be used by decisionmakers to implement coordinative and collaborative efforts in order to control the epidemic. '},\n"," {'paper_id': '9350071c50e046209486fa3cafcfa0757a5e6163',\n","  'section': 'Abstract',\n","  'text': 'The need for a fast and reliable test for COVID-19 is paramount in managing the current pandemic. A cost effective and efficient diagnostic tool as near to the point of care (PoC) as possible would be a game changer in current testing. We tested reverse transcription loop mediated isothermal amplification (RT-LAMP), a method which can produce results in under 30 minutes, alongside standard methods in a real-life clinical setting. Methods: This service improvement project piloted a research RT-LAMP method on nasal and pharyngeal swabs on 21 residents in a high dependency care home, with two index COVID-19 cases, and compared it to multiplex tandem reverse transcription polymerase chain reaction (RT-PCR). We calculated the sensitivity, specificity, positive and negative predictive values of a single RT-LAMP swab compared to RT-PCR, as per STARD guidelines. We also recorded vital signs of patients to correlate clinical and laboratory information. The novel method accurately detected 8/10 PCR positive cases and identified a further 3 positive cases. Eight further cases were negative using both methods. Using repeated RT-PCR as a \"gold standard\", the sensitivity and specificity of the novel test were 80% and 73% respectively. Positive predictive value (PPV) was 73% and negative predictive value (NPV) was 83%. We also observed hypothermia to be a significant early clinical sign in a number of COVID-19 patients in this setting. Interpretation: RT-LAMP testing for SARS-CoV-2 was found to be promising, fast, easy to use and to work equivalently to RT-PCR methods. Definitive studies to evaluate this method in larger cohorts are underway. RT-LAMP has the potential to transform COVID-19 detection, bringing rapid and accurate testing to the point of care. This method could be deployed in mobile testing units in the community, care homes and hospitals to detect disease early and prevent spread. . CC-BY 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': '93d47e78b27a6291cb5bfc81c11f1dacfd11376b',\n","  'section': 'Abstract',\n","  'text': 'We evaluated the impact of early social distancing on the COVID-19 transmission in the São Paulo metropolitan area. Using an age-stratified SEIR model, we determined the time-dependent reproductive number, and forecasted the ICU beds necessary to tackle this epidemic. Within 60 days, these measures might prevent 89,450 deaths. '},\n"," {'paper_id': '93d8f1552a81587e2f88e38580cbe4753b9934f1',\n","  'section': 'Abstract',\n","  'text': 'COVID-19 is a complex disease phenotype where the underlying microbiome could influence morbidity and mortality. Amplicon and metagenomic MinION based sequencing was used to rapidly (within 8 hours) identify SARS-CoV-2 and assess the microbiome in nasopharyngeal swabs obtained from patients with COVID-19 by the ISARIC 4C consortium. '},\n"," {'paper_id': '93f67ffe7803061de9b19c4dfa346b3aa97aa4eb',\n","  'section': 'Abstract',\n","  'text': 'Single particle cryo-EM is a powerful method for studying proteins and other biological macromolecules. Many of these molecules comprise regions with varying structural properties including disorder, flexibility, and partial occupancy. These traits make computational 3D reconstruction from 2D images challenging. Detergent micelles and lipid nanodiscs, used to keep membrane proteins in solution, are common examples of locally disordered structures that can negatively affect existing iterative refinement algorithms which assume rigidity (or spatial uniformity). We introduce a cross-validation approach to derive non-uniform refinement, an algorithm that automatically regularizes 3D density maps during iterative refinement to account for spatial variability, yielding dramatically improved resolution and 3D map quality. We find that in common iterative refinement methods, regularization using spatially uniform filtering operations can simultaneously over-and under-regularize local regions of a 3D map. In contrast, non-uniform refinement removes noise in disordered regions while retaining signal useful for aligning particle images. Our results include state-of-the-art resolution 3D reconstructions of multiple membrane proteins with molecular weight as low as 90kDa. These results demonstrate that higher resolutions and improved 3D density map quality can be achieved even for small membrane proteins, an important use case for single particle cryo-EM, both in structural biology and drug discovery. Non-uniform refinement is implemented in the cryoSPARC software package and has already been used successfully in several notable structural studies. '},\n"," {'paper_id': '9444dbf96374f2efc528fb4ede46ecf41a2b69be',\n","  'section': 'Abstract',\n","  'text': 'Direct RNA sequencing using an Oxford Nanopore MinION characterised the transcriptome of SARS-CoV-2 grown in Vero E6 cells. This cell line is being widely used to propagate the novel coronavirus. The viral transcriptome was analysed using a recently developed ORF-centric pipeline. This revealed the pattern of viral transcripts, (i.e. subgenomic mRNAs), generally fitted the predicted replication and transcription model for coronaviruses. A 24 nt in-frame deletion was detected in subgenomic mRNAs encoding the spike (S) glycoprotein. This feature was identified in over half of the mapped transcripts and was predicted to remove a proposed furin cleavage site from the S glycoprotein. This motif directs cleavage of the S glycoprotein into functional subunits during virus entry or exit. Cleavage of the S glycoprotein can be a barrier to zoonotic coronavirus transmission and affect viral pathogenicity. Allied to this transcriptome analysis, tandem mass spectrometry was used to identify over 500 viral peptides and 44 phosphopeptides, covering almost all of the proteins predicted to be encoded by the SARS-CoV-2 genome, including peptides unique to the deleted variant of the S glycoprotein. Detection of an apparently viable deletion in the furin cleavage site of the S glycoprotein reinforces the point that this and other regions of SARS-CoV-2 proteins may readily mutate. This is of clear significance given the interest in the S glycoprotein as a potential vaccine target and the observation that the furin cleavage site likely contributes strongly to the pathogenesis and zoonosis of this virus. The viral genome sequence should be carefully monitored during the growth of viral stocks for research, animal challenge models and, potentially, in clinical samples. Such variations may result in different levels of virulence, morbidity and mortality. '},\n"," {'paper_id': '949feecec7c10e71532e860f7f1230bd696e8234',\n","  'section': 'Abstract',\n","  'text': 'The outbreak of the 2019 novel coronavirus (SARS-CoV-2) has infected thousands of people with a large number of deaths across 26 countries. The sudden appearance of the virus leads to the limited existing therapies for SARS-CoV-2. Therefore, vaccines and antiviral medicines are in desperate need. This study took immune-informatics approaches to identify B-and T-cell epitopes for surface glycoprotein (S) of SARS-CoV-2, followed by estimating their antigenicity and interactions with the human leukocyte antigen (HLA) alleles. We identified four B cell epitopes, two MHC class-I and nine MHC class-II binding T-cell epitopes, which showed highly antigenic features. Allergenicity, toxicity and physiochemical properties analysis confirmed the specificity and selectivity of epitopes. The stability and safety of epitopes were confirmed by digestion analysis. No mutations were observed in all the selected B-and T-cell epitopes across all isolates from different locations worldwide. Epitopes were thus identified and some of them can be potential candidates for vaccine development. : bioRxiv preprint Methods Primary sequence of SARS-CoV-2 protein was retrieved from NCBI database using accession number MN908947.3 [3]. Experimentally known 3D structure of SARS-CoV-2 S protein was retrieved from Protein Data Bank (PDB ID: 6VSB)[1]. Protein sequence was analyzed for its chemicals and physical properties including GRAVY (Grand average of hydropathicity), half-life, molecular weight, stability index and amino acid atomic composition via an online tool Protparam [13]. TMHMM v2.0 (http://www.cbs.dtu.dk/services/TMHMM/) was applied to examine the transmembrane topology of S protein. Secondary structure of SARS-CoV-2 protein was analyzed by PSIPRED [14]. Existence of disulphide-bonds were examined through an online tool DIANNA v1.1 which uses trained neural system to make predictions [15]. Antigenicity of full-length S protein was evaluated by vaxijen v2.0 [16]. B-cell epitope prediction IEDB (Immune-Epitope-Database And Analysis-Resource) [17] with default parameter settings were used to predict B-cell epitopes. Epitopes predicted by linear epitope prediction of Bepipred and Bepipred2.0, Kolaskar and Tongaonkar antigenicity, Parker hydrophilicity, Chou and Fasman beta turn, and Karplus and Schulz flexibility. BcePred [18] was also used to predict B-cell epitopes using accessibility, antigenic propensity, exposed surface, flexibility, hydrophilicity, polarity and turns. Predicted B-cell epitopes by IEDB and BcePred were combined to the B-cell epitope candidate list. Based on transmembrane topology of S protein predicted by TMHMM v2.0, only epitopes on the outer surface were remained, and other intracellular epitopes were eliminated. VaxiJen 2.0 [16] was applied to evaluate the antigenicity of the remained epitopes. A stringent criteria was used to have antigenicity score of 0.9 and 14 residue length epitopes viewed adequate to start a defensive immune reaction. A B-cell discontinuous epitope forms the antigen-binding interface through fragments scattered along the protein sequence. DiscoTope2.0[19] with discotope score threshold of -3.7 was used to predict discontinuous epitopes. Pymol was used to examine the positions of selected linear and discontinuous epitopes on the 3D structure of SARS-CoV-2 protein. '},\n"," {'paper_id': '94dd454a02cb13481f099002f164e1b4f1590cd7',\n","  'section': 'Abstract',\n","  'text': 'Rapid screening and low-cost diagnosis play a crucial role in choosing the correct course of intervention e.g., drug therapy, quarantine, no action etc. when dealing with highly infectious pathogens. This is especially important if the disease-causing agent has no effective treatment, such as the novel coronavirus SARS-CoV-2 (the pathogen causing COVID-19), and shows no or similar symptoms to other common infections. We report a silicon-based integrated Point-of-Need (PoN) transducer (TriSilix) that can chemically-amplify and detect pathogen-specific sequences of nucleic acids (NA) quantitatively in real-time. Unlike other silicon-based technologies, TriSilix can be produced at wafer-scale in a standard laboratory; we have developed a series of methodologies based on metal-assisted chemical (wet) etching, electroplating, thermal bonding and laser-cutting to enable a cleanroom-free low-cost fabrication that does not require processing in an advanced semiconductor foundry. TriSilix is, therefore, resilient to disruptions in the global supply chain as the devices can be produced anywhere in the world. To create an ultra-low-cost device, the architecture proposed exploits the intrinsic properties of silicon and integrates three modes of operation in a single chip: i) electrical (Joule) heater, ii) temperature sensor (i.e. thermistor) with a negative temperature coefficient that can provide the precise temperature of the sample solution during reaction and iii) electrochemical sensor for detecting target NA. Using TriSilix, the sample solution can be maintained at a single, specific temperature (needed for isothermal amplification of NA such as Recombinase Polymerase Amplification (RPA) or cycled between different temperatures (with a precision of ±1.3 °C) for Polymerase Chain Reaction (PCR) while the exact concentration of amplicons is measured quantitatively and in real-time electrochemically. A single 4-inch Si wafer yields 37 TriSilix chips of 10×10×0.65 mm in size and can be produced in 7 hours, costing ~US $0.35 per device. The system is operated digitally, portable and low powercapable of running up to 35 tests with a 4000 mAh battery (a typical battery capacity of a modern smartphone). We were able to quantitatively detect a 563-bp fragment (Insertion Sequence IS900) of the genomic DNA of M. avium subsp. paratuberculosis (extracted from cultured field samples) through PCR in real-time with a Limit-of-author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '94df380f8a3bab7e30d39cc9d7e95d1f302f5f9f',\n","  'section': 'Abstract',\n","  'text': 'The introduction of electronic patient records in the ambulance service provides new opportunities to monitor the population. Most patients presenting to British ambulance services are discharged at scene. Ambulance records are therefore an ideal data source for syndromic early event detection systems to monitor infectious disease in the prehospital population. It has been previously found that tympanic temperature records can be used to detect influenza outbreaks in emergency departments. This study investigated whether routine tympanic temperature readings collected by ambulance crews can be used to detect seasonal influenza. Here we show that these temperature readings do allow the detection of seasonal influenza before methods applied to conventional data sources. The counts of pyretic patients were used to calculate a sliding case ratio (CR) as a measurement to detect seasonal influenza outbreaks. This method does not rely on conventional thresholds and can be adapted to the data. The data collected correlated with seasonal influenza. The 2016/17 outbreak was detected with high specificity and sensitivity, up to 9 weeks before other surveillance programs. An unanticipated outbreak of E. coli was detected in the same dataset. Our results show that ambulance records can be a useful data source for biosurveillance systems. Two outbreaks caused by different infectious agents have been successfully detected. The routine ambulance records allowed to use tympanic temperature readings that can be used as surveillance tool for febrile diseases. Therefore, this method is a valuable addition to the current surveillance tools. '},\n"," {'paper_id': '94e62a756a8f77d56f465194b69e260afad78eea',\n","  'section': 'Abstract',\n","  'text': 'Objective: Coronavirus disease 2019 (COVID-19) is an escalating global epidemic caused by SARS-CoV-2, with a high mortality in critical patients. Effective indicators for predicting disease severity in SARS-CoV-2 infected patients are urgently needed. Methods: In this study, 43 COVID-19 patients admitted in Chongqing Public Health Medical Center were involved. Demographic data, clinical features, and laboratory examinations were obtained through electronic medical records. Peripheral blood specimens were collected from COVID-19 patients and examined for lymphocyte subsets and cytokine profiles by flow cytometry. Potential contributing factors for prediction of disease severity were further analyzed. Results: A total of 43 COVID-19 patients were included in this study, including 29 mild patients and 14 sever patients. Severe patients were significantly older (61.9±9.4 vs 44.4±15.9) and had higher incidence in co-infection with bacteria compared to mild group (85.7%vs27.6%). Significantly more severe patients had the clinical symptoms of anhelation (78.6%) and asthma (71.4%). For laboratory examination, 57.1% severe cases showed significant reduction in lymphocyte count. The levels of Interluekin-6 (IL6), IL10, erythrocyte sedimentation rate (ESR) and D-Dimer (D-D) were significantly higher in severe patients than mild patients, while the level of albumin (ALB) was remarkably All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. : medRxiv preprint lower in severe patients. Further analysis demonstrated that ESR, D-D, age, ALB and IL6 were the major contributing factors for distinguishing severe patients from mild patients. Moreover, ESR was identified as the most powerful factor to predict disease progression of COVID-19 patients. : Age and the levels of ESR, D-D, ALB and IL6 are closely related to the disease severity of COVID-19 patients. ESR can be used as a valuable indicator for distinguishing severe COVID-19 patients in early stage, so as to increase the survival of severe patients. '},\n"," {'paper_id': '954576043076819041bde838775dc7929be81f5b',\n","  'section': 'Abstract',\n","  'text': 'COVID-19 is now a pandemic and many of the affected countries face severe shortages of 17 hospital resources. In Brazil, the first case was reported on February 26. As the number of cases 18 grows in the country, there is a concern that the health system may become overwhelmed, 19 resulting in shortages of hospital beds, intensive care unit beds, and mechanical ventilators. The 20 timing of shortage is likely to vary geographically depending on the observed onset and pace of 21 transmission observed, on the availability of resources, and on the actions implemented. Here we 22 resources to 34 All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '95687de5baf7b91c20058f1c819580ab872ddc6a',\n","  'section': 'Abstract',\n","  'text': \"Newly available data sets present an exciting opportunity to investigate how human population movement contributes to the spread of infectious diseases across large geographical distances. It is now possible to construct realistic models of infectious disease dynamics for the purposes of understanding global-scale epidemics. Nevertheless, a remaining unanswered question is how best to leverage the new data to parameterize models of movement, and whether one's choice of movement model impacts modeled disease outcomes. We adapt three well-studied models of infectious disease dynamics, the SIR model; the SIS model; and the Ross-Macdonald model, to incorporate either of two candidate movement models. We describe the effect that the choice of movement model has on each disease model's results, finding that in all cases there are parameter regimes where choosing one movement model instead of another has a profound impact on epidemiological outcomes. We further demonstrate the importance of choosing an appropriate movement model using the applied case of malaria transmission and importation on Bioko Island, Equatorial Guinea, finding that one model produces intelligible predictions of R 0 while the other produces nonsensical results. Mathematical modeling | Infectious disease dynamics | Human population movement | Malaria DT Citron developed the project and carried out the analysis. DL Smith provided guidance and context for the project. CA Guerra provided the data for building the Bioko Island model. SL Wu, JM Henry, HM Sánchez C, and D Dolgert discussed and reviewed the analysis. \"},\n"," {'paper_id': '956d0469d076bfebc265cf58743281c109706391',\n","  'section': 'Abstract',\n","  'text': 'A fully Integrated Micropillar Polydimethylsiloxane Accurate CRISPR Detection (IMPACT) system is developed for viral DNA detection. This powerful system is patterned with high-aspect ratio micropillars to enhance reporter probe binding. After surface modification and probe immobilization, CRISPR Cas12a/crRNA complex is injected into the fully enclosed system. With the presence of double-stranded DNA target, the CRISPR enzyme is activated and non-specifically cleaves the ssDNA reporters initially immobilized on the micropillars. This collateral cleavage releases fluorescence dyes into the assay, and the intensity is linearly proportional to the target DNA concentration ranging from 0.1 to 10 nM. Importantly, this system does not rely on traditional dye-quencher labeled probe thus eliminating the fluorescence background presented in the assay. Furthermore, our one-step detection protocol is performed at isothermal conditions (37°C) without using complicated and time-consuming off-chip probe hybridization and denaturation. This miniaturized and fully packed IMPACT chip demonstrates rapid, sensitive, and simple nucleic acid detection and is an ideal candidate for the next generation molecular diagnostic platform for point-of-care (POC) applications, responding to emerging and deadly pathogen outbreaks. author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '95754ec84bdee0637765a5ee34983a5f6cc86eed',\n","  'section': 'Abstract',\n","  'text': 'B-cell lymphoma 2 (Bcl-2) has a dual function, acting both as an oncogene 20 and an anti-tumor gene. It is well known that Bcl-2 exerts its tumor promoting 21 function through the mitochondrial pathway. However, the mechanism by which 22 Bcl-2 suppresses tumor formation is not well understood. We have previously shown 23 that Bcl-2 inhibits cell cycle progression from the G 0 /G 1 to the S phase after serum 24 starvation, and that quiescent Bcl-2 expressing cells maintained a significant lower 25 level of mitochondrial reactive oxygen species (ROS) than the control cells. Based on 26 the fact that ROS mediate cell cycle progression, and are controlled by peroxisome 27 proliferator-activated receptor-γ co-activator 1α (PGC-1α), a key molecule induced by 28 . CC-BY 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/111567 doi: bioRxiv preprint 2 prolonged starvation and involved in mitochondrial metabolism, we hypothesized that 29 PGC-1α might be related with the cell cycle function of Bcl-2. Here, we showed that 30 PGC-1α was upregulated upon Bcl-2 overexpression and downregulated following 31 Bcl-2 knockdown during serum starvation. Knockdown of PGC-1α activated Bcl-2 32 expression. Taken together, our results suggest that after serum depletion, PGC-1α 33 might coordinate with Bcl-2 to reduce ROS, which in turn delay cell cycle 34 progression. 35 36 '},\n"," {'paper_id': '957e71389292e0881dff2bbe0e87fe6bf8b2d381',\n","  'section': 'Abstract',\n","  'text': 'On January 23, 2020, China quarantined Wuhan to contain an emerging coronavirus . We estimated the probability of transportation of COVID-19 from Wuhan to 369 cities in China before the quarantine. The expected risk is >50% in 130 (95% CI 89-190) cities and >99% in the 4 largest metropolitan areas of China. '},\n"," {'paper_id': '95e959217bbe5b03a533352920dbf0d7d222b238',\n","  'section': 'Abstract',\n","  'text': 'An elementary model of COVID-19 dynamics-based on time-delay differential equations with a steplike survival function-is shown to be in good agreement with data from China and South Korea. The timedelal approach overcomes the major limitation of standard Susceptible-Exposed-Infected-Recovered (SEIR) models based on ordinary differential equations, namely their inability to predict the observed curve of infected individuals as a function of time. The model is also applied to countries where the epidemic is in earlier stages, such as Italy and Spain, to obtain estimates of the total number of cases and peak number of infected people that might be observed. '},\n"," {'paper_id': '95eb583d0840350cb9f7b6ed6f6601158a9fa6b1',\n","  'section': 'Abstract',\n","  'text': 'The corona virus disease 2019 shows unusually high transmission rate and unique clinical characteristics, with key pathological mechanism remaining unclear. Here, we analysed the laboratory data based on clinical samples from '},\n"," {'paper_id': '96250a5e9ece21b2415b83d6756a3fcd07123ee7',\n","  'section': 'Abstract',\n","  'text': 'COVID-19 is the most rapidly expanding coronavirus outbreak in the past two decades. To provide a swift response to a novel outbreak, prior knowledge from similar outbreaks is essential. Here, we study the volume of research conducted on previous coronavirus outbreaks, specifically SARS and MERS, relative to other infectious diseases by analyzing over 45 million papers from the last 20 years. Our results demonstrate that previous coronavirus outbreaks have been understudied compared to other viruses. We also show that the research volume of emerging infectious diseases is very high after an outbreak and drops drastically upon the containment of the disease. This can yield inadequate research and limited investment in gaining a full understanding of novel coronavirus management and prevention. Independent of the outcome of the current COVID-19 outbreak, we believe that measures should be taken to encourage sustained research in the field. '},\n"," {'paper_id': '96ef1767754a53f792951ba1752440ae94e90c60',\n","  'section': 'Abstract',\n","  'text': 'Viral escape from CD8 + cytotoxic T lymphocyte responses correlates with disease progression and 6 represents a significant challenge for vaccination. Here, we demonstrate that CD8 + T cell 7 recognition of the naturally occurring MHC-I-restricted LCMV-associated immune escape variant 8 T cell epitope associated with impaired peptide processing TEIPP 40 41 . '},\n"," {'paper_id': '9701a8c529cd8c18124da4cd61c1165a64b50281',\n","  'section': 'Abstract',\n","  'text': '2-18. We compared epidemiological characteristics across periods and different demographic groups. We developed a susceptible-exposed-infectious-recovered model to study the epidemic and evaluate the impact of interventions. The median age of the cases was 57 years and 50.3% were women. The attack rate peaked in the third period and substantially declined afterwards across geographic regions, sex and age groups, except for children (age <20) whose attack rate continued to increase. Healthcare workers and elderly people had higher attack rates and severity risk increased with age. The effective reproductive number dropped from 3.86 (95% credible interval 3.74 to 3.97) before interventions to 0.32 (0.28 to 0.37) post interventions. The interventions were estimated to prevent 94.5% (93.7 to 95.2%) infections till February 18. We found that at least 59% of infected cases were unascertained in Wuhan, potentially including asymptomatic and mild-symptomatic . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': '97027e08cb413795cf19064338efd049fe5d491b',\n","  'section': 'Abstract',\n","  'text': 'Adenovirus is a common human pathogen that relies on host cell processes for production 21 and processing of viral RNA. Although adenoviral promoters, splice junctions, and cleavage and 22 proteins by linking separate gene transcription units. Our work highlights how long-read 33 sequencing technologies can reveal further complexity within viral transcriptomes. : bioRxiv preprint another multifunctional protein that can cooperate with E1A to alter cellular gene expression 61 downstream of p53 as well as form the targeting component of a viral ubiquitin ligase [18][19][20][21][22][23]. 62 The remaining early transcription units are all transcriptionally activated by E1A and encode for 63 products of related function. The E2 region on the reverse strand of the AdV genome has both an 64 early and a late promoter, as well as two distinct polyadenylation sites, leading to upstream E2A 65 and downstream E2B transcripts [24]. E2A encodes for the viral DNA-binding protein (DBP), while 66 alternative splicing to E2B encodes for the protein-priming terminal protein (pTP) as well as the 67 AdV DNA polymerase (AdPol) [25][26][27]. The E3 region encoded on the top strand also has two 68 '},\n"," {'paper_id': '975349ac1c79fc64525ee17e7d088c88c2c2071b',\n","  'section': 'Abstract',\n","  'text': 'Infectious-disease outbreaks are often characterized by the reproductive number R and exponential rate of growth r. R provides information about outbreak control and predicted final size. Directly estimating R is difficult, while r can often be estimated from incidence data. These quantities are linked by the generation interval -the time between when an individual is infected by an infector, and when that infector was infected. It is often infeasible to obtain the exact shape of a generation-interval distribution, and to understand how this shape affects estimates of R. We show that estimating generation interval mean and variance provides insight into the relationship between R and r. We use examples based on Ebola, rabies and measles to explore approximations based on gamma-distributed generation intervals, and find that use of these simple approximations are often sufficient to capture the r-R relationship and provide robust estimates of R. '},\n"," {'paper_id': '97a34a062a7ad09f7633d4a0e76d013b28b9b582',\n","  'section': 'Abstract',\n","  'text': 'Along with tRNAs, enzymes that modify anticodon bases are a key aspect of translation across the 12 tree of life. tRNA modifications extend wobble pairing, allowing specific (\"target\") tRNAs to 13 recognize multiple codons and cover for other (\"non-target\") tRNAs, often improving translation 14 efficiency and accuracy. However, the detailed evolutionary history and impact of tRNA modifying 15 enzymes has not been analyzed. Using ancestral reconstruction of five tRNA modifications across 16 1093 bacteria, we show that most modifications were ancestral to eubacteria, but were repeatedly 17 lost in many lineages. Most modification losses coincided with evolutionary shifts in non-target 18 tRNAs, often driven by increased bias in genomic GC and associated codon use, or by genome 19 reduction. In turn, the loss of tRNA modifications stabilized otherwise highly dynamic tRNA gene 20 repertoires. Our work thus traces the complex history of bacterial tRNA modifications, providing 21 the first clear evidence for their role in the evolution of bacterial translation. 22 23 '},\n"," {'paper_id': '982689f92bf67e90a43fc142c605da9a5045f756',\n","  'section': 'Abstract',\n","  'text': 'Background The outbreak of coronavirus disease 2019 initially appeared and has most rapidly spread in Wuhan, China. The case fatality rate is the most direct indicator to assess the hazards of an infectious disease. We aimed to estimate the instant fatality rate and cure rate of COVID-19 in Wuhan City and its affiliated Hubei Province. We collected the daily case notification data of COVIDin Wuhan City and Hubei Province officially announced by the Chinese authority. The numbers of daily confirmed/deaths/cured cases and the numbers of daily cumulative confirmed/deaths/cured cases were obtained. The death time and cure time of COVID-19 patients were calculated based on the dates of diagnosis, death and discharge of individual cases. Then the estimated diagnosis dates of deaths and cured cases were obtained on the basis of the median death or medium cure time, respectively. Finally, the instant fatality rate of COVID-19 was calculated according to the numbers of deaths and cured cases on the same estimated diagnosis dates. From Jan 1, 2020 to Feb 22, 2020 in Wuhan City, the instant case fatality rate of COVID-19 was 3.4%～19.5% and the instant cured rate was 80.0%～96.6%. The average fatality rate reached 11.4% while the average cure rate was 88.6%. During the same period in Hubei Province, the instant case fatality rate was 3.8%～16.6% and the instant cured rate was 83.4%～96.6%. The average fatality rate and the average cure rate were 9.2% and 91.8%, respectively. The fatality rate and cure rate of COVID-19 in Wuhan City and Hubei Province were underestimated. Wuhan showed higher fatality rate and cure rate than the whole Hubei Province did. '},\n"," {'paper_id': '982a6fe2fdba30430b547741811ff4411f1485f1',\n","  'section': 'Abstract',\n","  'text': 'Metals are a limiting resource for pathogenic bacteria and must be scavenged from host proteins. 22 '},\n"," {'paper_id': '9844f67f10c860b4525c462b9d8f13f3f6b3290d',\n","  'section': 'Abstract',\n","  'text': 'Background: COVID-19 originated in China and has quickly spread worldwide causing a pandemic. Countries need rapid data on the prevalence of the virus in communities to enable rapid containment. However, the equipment, human and laboratory resources required for conducting individual RT-PCR is prohibitive. One technique to reduce the number of tests required is the pooling of samples for analysis by RT-PCR prior to testing. We conducted a mathematical analysis of pooling strategies for infection rate classification using group testing and for the identification of individuals by testing pooled clusters of samples. Findings: On the basis of the proposed pooled testing strategy we calculate the probability of false alarm, the probability of detection, and the average number of tests required as a function of the pool size. We find that when the sample size is 256, using a maximum pool size of 64, with only 7.3 tests on average, we can distinguish between prevalences of 1% and 5% with a probability of detection of 95% and probability of false alarm of 4%. Interpretation: The pooling of RT-PCR samples is a cost-effective technique for providing much-needed course-grained data on the prevalence of COVID-19. This is a powerful tool in providing countries with information that can facilitate a response to the pandemic that is evidence-based and saves the most lives possible with the resources available. '},\n"," {'paper_id': '984e07e449dde660098275470dd952cc88df3223',\n","  'section': 'Abstract',\n","  'text': 'Background: At present, PCR-based nucleic acid detection cannot meet the demands 22 for coronavirus infectious disease (COVID-19) diagnosis. Immunosorbent Assay (ELISA) kits based on recombinant SARS-CoV-2 nucleocapsid 27 protein (rN) and spike protein (rS) were used for detecting IgM and IgG antibodies, 28 and their diagnostic feasibility was evaluated. 29 Results: Among the 214 patients, 146 (68.2%) and 150 (70.1%) were successfully 30 diagnosed with the rN-based IgM and IgG ELISAs, respectively; 165 (77.1%) and 31 159 (74.3%) were successfully diagnosed with the rS-based IgM and IgG ELISAs, 32 respectively. The positive rates of the rN-based and rS-based ELISAs for antibody 33 (IgM and/or IgG) detection were 80.4% and 82.2%, respectively. The sensitivity of 34 the rS-based ELISA for IgM detection was significantly higher than that of the 35 rN-based ELISA. We observed an increase in the positive rate for IgM and IgG with 36 an increasing number of days post-disease onset (d.p.o.), but the positive rate of IgM 37 dropped after 35 d.p.o. The positive rate of rN-based and rS-based IgM and IgG 38 ELISAs was less than 60% during the early stage of the illness 0-10 d.p.o., and that of 39 IgM and IgG was obviously increased after 10 d.p.o. 40 Conclusions: ELISA has a high sensitivity, especially for the detection of serum 41 samples from patients after 10 d.p.o, it can be an important supplementary method for 42 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': '9856876e0f976c1535272fc84aed92d3e2834bed',\n","  'section': 'Abstract',\n","  'text': 'Based on the official data modeling, this paper studies the transmission process of the Corona Virus Disease 2019 (COVID-19). The error between the model and the official data curve is within 3%. At the same time, it realized forward prediction and backward inference of the epidemic situation, and the relevant analysis help relevant countries to make decisions. '},\n"," {'paper_id': '9857c6dc40cf3cc8b02eff38cf13e2e77c4e1b59',\n","  'section': 'Abstract',\n","  'text': 'Rapid and widespread implementation of infectious disease surveillance is a critical 9 '},\n"," {'paper_id': '9867fe1531c1541ca2b25a82d7adb62d575798be',\n","  'section': 'Abstract',\n","  'text': 'SARS-CoV-2 is a betacoronavirus that is responsible for the COVID-19 2 pandemic. The genome of SARS-CoV-2 was reported recently, but its 3 transcriptomic architecture is unknown. Utilizing two complementary 4 sequencing techniques, we here present a high-resolution map of the 5 SARS-CoV-2 transcriptome and epitranscriptome. DNA nanoball sequencing 6 shows that the transcriptome is highly complex owing to numerous 7 recombination events, both canonical and noncanonical. In addition to the 8 genomic RNA and subgenomic RNAs common in all coronaviruses, 9 SARS-CoV-2 produces a large number of transcripts encoding unknown ORFs 10 with fusion, deletion, and/or frameshift. Using nanopore direct RNA 11 sequencing, we further find at least 41 RNA modification sites on viral 12 transcripts, with the most frequent motif being AAGAA. Modified RNAs have 13 shorter poly(A) tails than unmodified RNAs, suggesting a link between the 14 internal modification and the 3 ′ tail. Functional investigation of the unknown 15 ORFs and RNA modifications discovered in this study will open new directions 16 to our understanding of the life cycle and pathogenicity of SARS-CoV-2. 17 Kim et al. Highlights 21 • We provide a high-resolution map of SARS-CoV-2 transcriptome and 22 epitranscriptome using nanopore direct RNA sequencing and DNA 23 nanoball sequencing. 24 • The transcriptome is highly complex owing to numerous recombination 25 events, both canonical and noncanonical. 26 • In addition to the genomic and subgenomic RNAs common in all 27 coronaviruses, SARS-CoV-2 produces transcripts encoding unknown 28 ORFs. 29 • We discover at least 41 potential RNA modification sites with an AAGAA 30 motif. 31 Kim et al. designated as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) 34 (Zhou et al., 2020; Zhu et al., 2020). Like other coronaviruses (order Nidovirales, 35 family Coronaviridae, subfamily Coronavirinae), SARS-CoV-2 is an enveloped 36 virus with a positive-sense, single-stranded RNA genome of~30 kb. 37 SARS-CoV-2 belongs to the genus betacoronavirus, together with SARS-CoV 38 and Middle East respiratory syndrome coronavirus (MERS-CoV) (with 80% and 39 50% homology, respectively) (Kim et al., 2020; Zhou et al., 2020). Coronaviruses 40 (CoVs) were thought to primarily cause enzootic infections in birds and 41 mammals. But, the recurring outbreaks of SARS, MERS, and now COVID-19 42 have clearly demonstrated the remarkable ability of CoVs to cross species 43 barriers and transmit between humans (Menachery et al., 2017). 44 CoVs carry the largest genomes (26-32 kb) among all RNA virus families 45 (Figure 1). Each viral transcript has a 5 ′ -cap structure and a 3 ′ poly(A) tail (Lai 46 and Stohlman, 1981; Yogo et al., 1977). Upon cell entry, the genomic RNA is 47 translated to produce nonstructural proteins (nsps) from two open reading 48 frames (ORFs), ORF1a, and ORF1b. The ORF1a produces polypeptide 1a (pp1a, 49 440-500 kDa) that is cleaved into 11 nsps. The −1 ribosome frameshift occurs 50 immediately upstream of the ORF1a stop codon, which allows continued 51 translation of ORF1b, yielding a large polypeptide (pp1ab, 740-810 kDa) which 52 is cleaved into 16 nsps. The proteolytic cleavage is mediated by viral proteases 53 nsp3 and nsp5 that harbor a papain-like protease domain and a 3C-like protease 54 domain, respectively. 55 The viral genome is also used as the template for replication and 56 transcription, which is mediated by nsp12 harboring RNA-dependent RNA 57 polymerase (RdRP) activity (Snijder et al., 2016; Sola et al., 2015) . 58 Negative-strand RNA intermediates are generated to serve as the templates for 59 the synthesis of positive-sense genomic RNA (gRNA) and subgenomic RNAs 60 (sgRNAs). The gRNA is packaged by the structural proteins to assemble progeny 61 virions. Shorter sgRNAs encode conserved structural proteins (spike protein (S), 62 Kim et al. '},\n"," {'paper_id': '993a9b572dd0a0052a07fd7850ade2e1c7debb0c',\n","  'section': 'Abstract',\n","  'text': 'Equal contributions. We thank our families for their patience with this project during a trying period. We received helpful comments and feedback from '},\n"," {'paper_id': '99a46ba28f5811c112876ec77f570d6c10778db6',\n","  'section': 'Abstract',\n","  'text': 'We plotted daily data on the frequencies of keywords related to severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) from WeChat, a Chinese social media. Using \"Feidian\", Chinese abbreviation for SARS, may detect the SARS-CoV-2 outbreak in 2019 two weeks earlier. WeChat offered a new approach to early detect disease outbreaks. '},\n"," {'paper_id': '99dab788e984601ee4efcd1138926caf10c00dcf',\n","  'section': 'Abstract',\n","  'text': 'Because of the potential link between −1 programmed ribosomal frameshifting and response of a pseudoknot (PK) RNA to force, a number of single molecule pulling experiments have been performed on PKs to decipher the mechanism of programmed ribosomal frameshifting. Motivated in part by these experiments, we performed simulations using a coarse-grained model of RNA to describe the response of a PK over a range of mechanical forces (f s) and monovalent salt concentrations (Cs). The coarse-grained simulations quantitatively reproduce the multistep thermal melting observed in experiments, thus validating our model. The free energy changes obtained in simulations are in excellent agreement with experiments. By varying f and C, we calculated the phase diagram that shows a sequence of structural transitions, populating distinct intermediate states. As f and C are changed, the stem-loop tertiary interactions rupture first, followed by unfolding of the 3 -end hairpin (I F). Finally, the 5 -end hairpin unravels, producing an extended state (E I). A theoretical analysis of the phase boundaries shows that the critical force for rupture scales as (log Cm) α with α = 1 (0.5) for E I (I F) transition. This relation is used to obtain the preferential ion-RNA interaction coefficient, which can be quantitatively measured in single-molecule experiments, as done previously for DNA hairpins. A by-product of our work is the suggestion that the frameshift efficiency is likely determined by the stability of the 5 end hairpin that the ribosome first encounters during translation. '},\n"," {'paper_id': '99f3c26b58af60c98a7dd24b8bc421de5a9f49f4',\n","  'section': 'Abstract',\n","  'text': 'Prolonged viral shedding is associated with severe status and poor prognosis of COVID-19 patients. Unexpectedly, here we report a non-severe patient with the longest duration of viral shedding. According to the investigation on the clinical and epidemiological information of this case, we concluded that this type of virus might have a low toxicity and transmissibility, but have a prolonged infective ability and was hardly to be eliminated in the body with regular therapy. However, infusion of plasma from recovered patients showed high efficiency in elimination of this virus. Our findings might shed light on the management of COVID-19. '},\n"," {'paper_id': '9a077c5ff1dcbb9104feeaa832e73ccdcf5fdcc9',\n","  'section': 'Abstract',\n","  'text': 'The SARS-CoV-2 outbreak is causing widespread infections and significant mortality. Previous studies describing clinical characteristics of the disease contained small cohorts from individual centers or larger series consisting of mixed cases from different hospitals. We report analyses of mortality and disease severity among 402 patients from a single hospital. The cohort included 297 patients with confirmed and 105 with suspected diagnosis. The latter group met the criteria for clinical diagnosis but nucleic acid tests results were initially interpreted as suspicious. Data were compared between genders and among different age groups. The overall case fatality is 5.2%. However, patients 70 years of age or older suffered a significantly higher mortality (17.8%), associated with more patients having severe or critical illness (57.5%). Patients 50 years of age or older had a mortality of 8.0%, and those younger than 50 years, 1.2%. Male patients had a mortality of 7.6% versus 2.9% in females. '},\n"," {'paper_id': '9a097f4bfa3cbb71a7efad65d1c9280955799979',\n","  'section': 'Abstract',\n","  'text': '18 SARS-CoV-2 is a novel coronavirus currently causing a pandemic. We show 19 that the majority of amino acid positions, which differ between SARS-CoV-2 and the 20 closely related SARS-CoV, are differentially conserved suggesting differences in 21 biological behaviour. In agreement, novel cell culture models revealed differences 22 between the tropism of SARS-CoV-2 and SARS-CoV. Moreover, cellular ACE2 23 (SARS-CoV-2 receptor) and TMPRSS2 (enables virus entry via S protein cleavage) 24 levels did not reliably indicate cell susceptibility to SARS-CoV-2. SARS-CoV-2 and 25 SARS-CoV further differed in their drug sensitivity profiles. Thus, only drug testing 26 using SARS-CoV-2 reliably identifies therapy candidates. Therapeutic concentrations 27 of the approved protease inhibitor aprotinin displayed anti-SARS-CoV-2 activity. The 28 efficacy of aprotinin and of remdesivir (currently under clinical investigation against 29 SARS-CoV-2) were further enhanced by therapeutic concentrations of the proton 30 pump inhibitor omeprazole (aprotinin 2.7-fold, remdesivir 10-fold). Hence, our study 31 has also identified anti-SARS-CoV-2 therapy candidates that can be readily tested in 32 patients. 33 34 '},\n"," {'paper_id': '9a0c89a9b8ff3969d97b7cc4245489ff69f3a615',\n","  'section': 'Abstract',\n","  'text': 'Since the outbreak of COVID-19, there has been a pressing need for development of a prognostic tool that is easy for clinicians to use. Recently, a Lancet publication showed that in a cohort of 191 patients with COVID-19, age, SOFA score and D-dimer measurements were associated with mortality. No other publication involving prognostic factors or models has been identified to date. In our cohorts of 444 patients from two hospitals, SOFA scores were low in the majority of patients on admission. The relevance of D-dimer could not be verified, as it is not included in routine laboratory tests. In this study, we have established a multivariable clinical prediction model using a development cohort of 299 patients from one hospital. After backwards selection, four variables, including age, lymphocyte count, lactate dehydrogenase and SpO2 remained in the model to predict mortality. This has been validated internally and externally with a cohort of 145 patients from a different hospital. Discrimination of the model was excellent in both internal (c=0·89) and external (c=0·98) validation. Calibration plots showed excellent agreement between predicted and observed probabilities of mortality after recalibration of the model to account for underlying differences in the risk profile of the datasets. This . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': '9a3de625119a83aa1401eeb297937d9605f0d1f1',\n","  'section': 'Abstract',\n","  'text': 'Rationale: Pneumonia is a leading cause of mortality in infants and young children. The mechanisms that lead to mortality in these children are poorly understood. Studies of the cellular immunology of the infant airway have traditionally been hindered by the limited sample volumes available from the young, frail children who are admitted to hospital with pneumonia. This is further compounded by the relatively low frequencies of certain immune cell phenotypes that are thought to be critical to the clinical outcome of pneumonia. To address this, we developed a novel in-silico deconvolution method for inferring the frequencies of immune cell phenotypes in the airway of children with different survival outcomes using proteomic data. Methods: Using high-resolution mass spectrometry, we identified > 1,000 proteins expressed in the airways of children who were admitted to hospital with clinical pneumonia. 61 of these children were discharged from hospital and survived for more than 365 days after discharge, while 19 died during admission. We used machine learning by random forest to derive protein features that could be used to deconvolve immune cell phenotypes in paediatric airway samples. We applied these phenotype-specific signatures to identify airway-resident immune cell phenotypes that were differentially enriched by survival status and validated the findings using a large retrospective pneumonia cohort. We identified immune-cell phenotype classification features for 33 immune cell types. Eosinophil-associated features were significantly elevated in airway samples obtained from pneumonia survivors and were downregulated in children who subsequently died. To confirm these results, we analyzed clinical parameters from >10,doi: bioRxiv preprint results of this retrospective analysis mirrored airway deconvolution data and showed that survivors had significantly elevated eosinophils at admission compared to fatal pneumonia. Using a proteomics bioinformatics approach, we identify airway eosinophils as a critical factor for pneumonia survival in infants and young children. doi: bioRxiv preprint 1ml of nasopharyngeal and oropharyngeal swab samples obtained from children was centrifuged at 17,000xg for 7 minutes, after which 800 l of the supernatant was removed and discarded. The remaining 200µl were split into two aliquots of 100µl each. The first aliquot was used for neutrophil phenotyping assays and the other was used for neutrophil phagocytosis assays. 20µl of a pre-constituted cocktail of the following antibodies (from ThermoFisher) was used to label both aliquots -CD45, CD16, CD14, CD3, CD19, HLA-DR, CD66b, CD11b and a Live-dead marker. With the exception of the live/dead marker, all other antibodies were diluted 1:100 in FACS buffer. The live/dead marker was prepared at a 1:1000 dilution in FACS buffer. For the phagocytosis assay tube, 20 l of opsonised Escherichia coli (E.coli) was added to the tube (pHrodo Red E. coli BioParticles; ThermoFisher). The bacteria was initially prepared by mixing the E.coli strain with newborn calf sera followed by a 30-minute incubation at 37 o C. After this step, both tubes were incubated at 37 o C for 35 minutes. After the incubation, 20 l of a live-dead marker was added to each tube and incubated for 10 minutes at 37 o C. The reaction was stopped by adding 500 l of 1X RBC lysis buffer followed by a 5-minute incubation. Cells in each tube were then spun down at 2,700xg for 1 minute and the supernatant discarded. Cells were then washed twice with FACS buffer, after which 350 l of FACS flow was added. Cells were then analysed immediately on a BD LSR Fortessa instrument. The following gating strategy was used to detect airway-resident neutrophils: Debris were excluded on the basis of their forward (FSC-A) and side (SSC-A) scatter characteristics, doublets were excluded using FSC-A versus FSC-H and dead cells were excluded using the live-dead marker. '},\n"," {'paper_id': '9ad252c633b7596a4b64ca9f1699a0db5ce99c05',\n","  'section': 'Abstract',\n","  'text': 'Currently, there are no approved specific antiviral agents for 2019 novel coronavirus disease . In this study, ten severe patients confirmed by real-time viral RNA test were enrolled prospectively. One dose of 200 mL convalescent plasma (CP) derived from recently recovered donors with the neutralizing antibody titers above 1:640 was transfused to the patients as an addition to maximal supportive care and antiviral agents. The primary endpoint was the safety of CP transfusion. The second endpoints were the improvement of clinical symptoms and laboratory parameters within 3 days after CP transfusion. The median time from onset of illness to CP transfusion was 16.5 days. After CP transfusion, the level of neutralizing antibody increased rapidly up to 1:640 in five cases, while that of the other four cases maintained at a high level (1:640). The clinical symptoms were significantly improved along with increase of oxyhemoglobin saturation within 3 days. Several parameters tended to improve as compared to pre-transfusion, including increased lymphocyte counts (0.65×10 9 /L vs. 0.76×10 9 /L) and decreased C-reactive protein (55.98 mg/L vs. 18.13 mg/L). Radiological examinations showed varying degrees of absorption of lung lesionswithin 7 days. The viral load was undetectable after transfusion in seven patients who had previous viremia. No severe adverse effects were observed. This study showed CP therapy was welltolerated and could potentially improve the clinical outcomes through neutralizing viremia in severe COVID-19 cases. The optimal dose and time point, as well as the clinical benefit of CP therapy, needs further investigation in larger well-controlled trials. COVID-19is currently a big threat to global health. However, no specific antiviral agents are available for its treatment. In this work, we explored the feasibility of convalescent plasma (CP) transfusion to rescue severe patients. The results from 10 severe adult cases showed that one dose (200 mL) of CP was welltolerated and could significantly increase or maintain the neutralizing antibodies at a high level, leading to disappearance of viremia in 7 days. Meanwhile, clinical symptoms and paraclinical criteria rapidly improved within 3 days. Radiological examination All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '9ae476404f7ef1ec1ede965f0b898f31a5bf5a81',\n","  'section': 'Abstract',\n","  'text': 'Backgrounds Since December 2019, a novel coronavirus epidemic has emerged in Wuhan city, China and then rapidly spread to other areas. As of 20 Feb 2020, a total of 2,055 medical staff confirmed with coronavirus disease 2019 (COVID-19) caused by SARS-Cov-2 in China had been reported. We sought to explore the epidemiological, clinical characteristics and prognosis of novel coronavirus-infected medical staff. In this retrospective study, 64 confirmed cases of novel coronavirus-infected medical staff admitted to Union Hospital, Wuhan between 16 Jan, 2020 to 15 Feb, 2020 were included. Two groups concerned were extracted from the subjects based on duration of symptoms: group 1 (≤10 days) and group 2 (>10 days). Epidemiological and clinical data were analyzed and compared across groups. The Kaplan-Meier plot was used to inspect the change in hospital discharge rate. The Cox regression model was utilized to identify factors associated with hospital discharge. The median age of medical staff included was 35 years old. 64% were female and 67% were nurses. None had an exposure to Huanan seafood wholesale market or wildlife. A small proportion of the cohort had contact with specimens (5%) as well as patients in fever clinics (8%) and isolation wards (5%). Fever (67%) was the most common symptom, followed by cough (47%) and fatigue (34%). The median time interval between symptoms onset and admission was 8.5 days. On admission, 80% of medical staff showed abnormal IL-6 levels and 34% had lymphocytopenia. Chest CT mainly manifested as bilateral (61%), septal/subpleural (80%) and ground-glass (52%) opacities. During the study period, no patients was transferred to intensive care unit or died, and 34 (53%) had been discharged. Higher body mass index (BMI) (≥ 24 kg/m 2 ) (HR 0.14; 95% CI 0.03-0.73), fever (HR 0.24; 95% CI 0.09-0.60) and higher levels of IL-6 on admission (HR 0.31; 95% CI 0.11-0.87) were unfavorable factors for discharge. Interpretation In this study, medical staff infected with COVID-19 have relatively milder symptoms and favorable clinical course, which may be partly due to their . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.09.20033118 doi: medRxiv preprint medical expertise, younger age and less underlying diseases. Smaller BMI, absence of fever symptoms and normal IL-6 levels on admission are favorable for discharge for medical staff. Further studies should be devoted to identifying the exact patterns of SARS-CoV-2 infection among medical staff. '},\n"," {'paper_id': '9aef9e1c22896809852ec6c94e413aff33b3d9cf',\n","  'section': 'Abstract',\n","  'text': 'Disease 2019. Nucleic acid detection of 2019-nCoV is one of the key indicators for clinical diagnosis. However, the positive rate is only 30-50%. Currently, fluorescent quantitative RT-PCR technology is mainly used to detect 2019-nCoV. According to \"The Laboratory Technical Guidelines for Detection 2019-nCoV (Fourth Edition)\" issued by National Health and Commission of China and \"The Experts\\' Consensus on Nucleic Acid Detection of 2019-nCoV\" released by Chinese Society of Laboratory Medicine, the human samples must be placed under 56 o C or higher to inactivate the viruses in order to keep the inspectors from virus infection before the nucleic acids were isolated as the template of qRT-PCR. In this study, we demonstrated that the virus inactivation treatment disrupts its genome integrity seriously when using porcine epidemic diarrhea virus (vaccine), a kind of coronavirus, as a model. Our results showed that only 50.11% of the detectable viral templates left after the inactivation of 56 o C for 30 minutes and only 3.36% left after the inactivation of 92 o C for 5 minutes when the samples were preserved by Hank\\'s solution, one of an isotonic salt solutions currently suggested. However, the detectable templates of viral nucleic acids can be unchanged after the samples were incubated at 56 o C or higher if the samples were preserved with an optimized solution to protect the RNA from being disrupted. We therefore highly recommend to carry out systematic investigation on the impact of high temperature inactivation on the integrity of 2019-nCoV genome and develop a sample preservation solution to protect the detectable templates of 2019-nCoV nucleic acids from high temperature inactivation damage. Key words: coronavirus, 2019-nCoV, virus inactivation by high temperature, nucleic acid degradation, false negative, fluorescent quantitative RT-PCR author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '9af0b8e13e1e5ac714135dca8ba6d5216966066b',\n","  'section': 'Abstract',\n","  'text': 'Intrinsically disordered regions (IDRs) challenge the well-established sequence-structurefunction paradigm for describing protein function and evolution. Here, we direct a combination of biophysical and cellular studies to further our understanding of how the intrinsically disordered C-terminal tail of FtsZ contributes to cell division in rod-shaped bacteria. FtsZ is a modular protein that encompasses a conserved GTPase domain and a highly variable intrinsically disordered C-terminal tail (CTT). The CTT is essential for forming the cytokinetic Z-ring. Despite poor sequence conservation of the CTT, the patterning of oppositely charged residues, which refers to the extent of linear mixing / segregation of oppositely charged residues within CTT sequences is bounded within a narrow range. To assess the impact of evolutionary bounds on charge patterning within CTT sequences we performed experiments, aided by sequence design, to quantify the impact of changing the patterning of oppositely charged residues within the CTT on the functions of FtsZ from B. subtilis. Z-ring formation is robust if and only if the extent of linear mixing / segregation of oppositely charged residues within the CTT sequences is within evolutionarily observed bounds. Otherwise, aberrant, CTT-mediated, FtsZ assemblies impair Zring formation. The complexities of CTT sequences also have to be above a threshold value because FtsZ variants with low complexity CTTs are not tolerated in cells. Taken together, our results suggest that CTT sequences have evolved to be \"just right\" and that this is achieved through an optimal extent of charge patterning while maintaining the sequence complexity above a threshold value. '},\n"," {'paper_id': '9b17a9fa6be11d8b3dbed9f3dea2516020083bd3',\n","  'section': 'Abstract',\n","  'text': 'Identifying viruses and phages in a metagenomics sample has important implication in improving human health, preventing viral outbreaks, and developing personalized medicine. With the rapid increase in data files generated by next generation sequencing, existing tools for identifying and annotating viruses and phages in metagenomics samples suffer from expensive running time. In this paper, we developed a standalone pipeline, FastViromeExplorer, for rapid identification and abundance quantification of viruses and phages in big metagenomic data. Both real and simulated data validated FastViromeExplorer as a reliable tool to accurately identify viruses and their abundances in large data, as well as in a time efficient manner. '},\n"," {'paper_id': '9b1e1c869a26b53973683d284250c2f4a8a339f6',\n","  'section': 'Abstract',\n","  'text': 'The medical, health service, societal and economic impact of the COVID-19 emergency has unknown effects on overall population mortality. Previous models of population mortality are based on death over days among infected people, nearly all of whom (to date at least) have underlying conditions. Models have not incorporated information on high risk conditions or their longer term background (pre-COVID-19) mortality. We estimated the excess number of deaths over 1 year under different COVID-19 incidence rates and differing mortality impacts. Methods: Using population based linked primary and secondary care electronic health records in England (HDR UK -CALIBER), we report the prevalence of underlying conditions defined by UK Public Health England COVID-19 guidelines (16 March 2020) in 3,862,012 individuals aged ≥30 years from 1997-2017. We used previously validated phenotypes, openly available (https://caliberresearch.org/portal), for each condition using ICD-10 diagnosis, Read, procedure and medication codes. We estimated the 1-year mortality in each condition, and developed simple models of excess COVID-19-related deaths assuming relative risk (RR) of the impact of the emergency (compared to background mortality) of 1.2, 1.5 and 2.0. '},\n"," {'paper_id': '9b409562ec542a173815830a3ea64a7570780952',\n","  'section': 'Abstract',\n","  'text': 'Severe Acute Respiratory Syndrome Coronavirus-2 (SARS-CoV-2) has recently caused a pandemic that has involved Italy as the second worldwide nation in terms of infected patients and deaths. The clinical manifestation of Covid-19 ranges from asymptomatic carrier status to severe pneumonia. Asymptomatic individuals in Covid-19 are those who are carriers of the virus but do not show clinical symptoms and are able to transmit the disease in the same degree as symptomatic carriers. In order to contain contagions is of supreme importance to identify asymptomatic patients because this subpopulation is one of the main factors contributing to the spread of this disease. We report on six Italian patients with COVID-19 who presented sudden hyposmia as the only or most prominent disease manifestation, without upper or lower respiratory tract involvement or other major features of the disease. A supra-threshold olfaction test confirmed the hyposmia in all patients. '},\n"," {'paper_id': '9b5f5119bbfbded3245acc37859cefde967458e7',\n","  'section': 'Abstract',\n","  'text': 'Strategies for containing an emerging infectious disease outbreak must be nonpharmaceutical when drugs or vaccines for the pathogen do not yet exist or are unavailable. The success of these non-pharmaceutical strategies will depend not only on the effectiveness of quarantine or other isolation measures but also on the epidemiological characteristics of the infection. However, there is currently no systematic framework to assess the relationship between different containment strategies and the natural history and epidemiological dynamics of the pathogen. Here, we compare the effectiveness of quarantine and symptom monitoring, implemented via contact tracing, in controlling epidemics using an agent-based branching model. We examine the relationship between epidemic containment and the disease dynamics of symptoms and infectiousness for seven case study diseases with diverse natural histories including Ebola, Influenza A, and Severe Acute Respiratory Syndrome (SARS). We show that the comparative effectiveness of symptom monitoring and quarantine depends critically on the natural history of the infectious disease, its inherent transmissibility, and the intervention feasibility in the particular healthcare setting. The benefit of quarantine over symptom monitoring is generally maximized for fast-course diseases, but we show the conditions under which symptom monitoring alone can control certain outbreaks. This quantitative framework can guide policy-makers on how best to use non-pharmaceutical interventions to contain emerging outbreaks and prioritize research during an outbreak of a novel pathogen. '},\n"," {'paper_id': '9bbfd3d34ee18ea1b9f4669331a6cee9c5992893',\n","  'section': 'Abstract',\n","  'text': 'emerged in late 2019 1,2 . Initial outbreaks in China involved 13.8% cases with severe-, and 6.1% with critical courses 3 . This severe presentation corresponds to the usage of a virus receptor that is expressed predominantly in the lung 2,4 . By causing an early onset of severe symptoms, this same receptor tropism is thought to have determined pathogenicity but also aided the control of severe acute respiratory syndrome (SARS) in 2003 5 . However, there are reports of COVID-19 cases with mild upper respiratory tract symptoms, suggesting a potential for pre-or oligosymptomatic transmission 6-8 . There is an urgent need for information on body site -specific virus replication, immunity, and infectivity. Here we provide a detailed virological analysis of nine All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '9bf1677e5f822110e736076a4ba6960a80f2f348',\n","  'section': 'Abstract',\n","  'text': 'Objective: We aim to summarize reliable evidences of evidence-based medicine for the treatment and prevention of the 2019 novel coronavirus (2019-nCoV) by analyzing all the published studies on the clinical characteristics of patients with 2019-nCoV. Methods: PubMed, Cochrane Library, Embase, and other databases were searched. Several studies on the clinical characteristics of 2019-nCoV infection were collected for Meta-analysis. Results: Ten studies were included in Meta-analysis, including a total number of 50466 patients with 2019-nCoV infection. Meta-analysis shows that, among these patients, the incidence of fever was 89.1%, the incidence of cough was 72.2%, and the incidence of muscle soreness or fatigue was 42.5%. The incidence of acute respiratory distress syndrome (ARDS) was 14.8%, the incidence of abnormal chest computer tomography (CT) was 96.6%, the percentage of severe cases in all infected cases was 18.1%, and the case fatality rate of patients with 2019-nCoV infection was 4.3%. Conclusion: Fever and cough are the most common symptoms in patients with 2019-nCoV infection, and most of these patients have abnormal chest CT examination. Several people have muscle soreness or fatigue as well as ARDS. Diarrhea, hemoptysis, headache, sore throat, shock, and other symptoms only occur in a small number of patients. The case fatality rate of patients with 2019-nCoV infection is lower than that of Severe Acute Respiratory Syndrome (SARS) and Middle East Respiratory Syndrome (MERS). '},\n"," {'paper_id': '9bf64a354621d022d264c06e8c608c7c06ad1e6e',\n","  'section': 'Abstract',\n","  'text': 'SARS-CoV-2 has infected over 140,000 people as of March 14, 2020. We use a mathematical model to investigate the effectiveness of social distancing interventions lasting six weeks in a middle-sized city in the US. We explore four social distancing strategies by reducing the contacts of adults over 60 years old, adults over 60 years old and children, all adults (25, 75 or 95% compliance), and everyone in the population. Our results suggest that social distancing interventions can avert cases by 20% and hospitalizations and deaths by 90% even with modest compliance within adults as long as the intervention is kept in place, but the epidemic is set to rebound once the intervention is lifted. Our models suggest that social distancing interventions will buy crucial time but need to occur in conjunction with testing and contact tracing of all suspected cases to mitigate transmission of SARS-CoV-2. '},\n"," {'paper_id': '9bf79ddffb9b8f84da316dbb52d09d960f164739',\n","  'section': 'Abstract',\n","  'text': 'Coronavirus disease 19 is an emerging global health crisis. With over 200,000 29 confirmed cases to date, this pandemic continues to expand, spurring research to discover 30 vaccines and therapies. SARS-CoV-2 is the novel coronavirus responsible for this disease. It 31 initiates entry into human cells by binding to angiotensin-converting enzyme 2 (ACE2) via the 32 receptor binding domain (RBD) of its spike protein (S). Disrupting the SARS-CoV-2-RBD binding 33 to ACE2 with designer drugs has the potential to inhibit the virus from entering human cells, 34 presenting a new modality for therapeutic intervention. Peptide-based binders are an attractive 35 solution to inhibit the RBD-ACE2 interaction by adequately covering the extended protein contact 36 interface. Using molecular dynamics simulations based on the recently solved ACE2 and SARS-37 CoV-2-RBD co-crystal structure, we observed that the ACE2 peptidase domain (PD) α1 helix is 38 important for binding SARS-CoV-2-RBD. Using automated fast-flow peptide synthesis, we 39 chemically synthesized a 23-mer peptide fragment of the ACE2 PD α1 helix composed entirely of 40 proteinogenic amino acids. Chemical synthesis of this human derived sequence was complete in 41 1.5 hours and after work up and isolation >20 milligrams of pure material was obtained. Bio-layer 42 interferometry revealed that this peptide specifically associates with the SARS-CoV-2-RBD with 43 low nanomolar affinity. This peptide binder to SARS-CoV-2-RBD provides new avenues for 44 COVID-19 treatment and diagnostic modalities by blocking the SARS-CoV-2 spike protein 45 interaction with ACE2 and thus precluding virus entry into human cells. 46 47 '},\n"," {'paper_id': '9c016d8d0a0e01eaa2f449c5c779a6ecce82ff17',\n","  'section': 'Abstract',\n","  'text': 'Limited data exist on how SARS-CoV-2 enters and spreads in the general population. We used two strategies for SARS-CoV-2 testing: targeted testing of high-risk individuals (n=4,551) and a population screening (n=5,502). We sequenced SARS-CoV-2 from 340 individuals. RESULTS March 22 2020, 528 had tested positive for SARS-CoV-2 in the targeted testing (11.6%) and 50 in the population screening (0.9%); approximately 0.2% of the Icelandic population. Large fractions of positives had travelled outside Iceland (38.4% and 34.0%). Fewer under 10 years old were positive than those older: 2.8% vs. 12.3% for targeted testing (P=1.6e-9) and 0.0% vs. 1.0% for population screening (P=0.031). Fewer females were positive in the targeted testing than males (9.5% vs. 14.6%, P=6.8e-9). SARS-CoV-2 came from eight clades, seven A clades and one B clade. The clade composition differed between the testing groups and changed with time. In the early targeted testing, 65.0% of clades were A2a1 and A2a2 derived from Italian and Austrian skiing areas, but in the later targeted testing went down to 30.6% and were overtaken by A1a and A2a, the most common clades in the population screening. SARS-CoV-2 has spread widely in Iceland outside of the high-risk groups. Several strains cause these infections and their relative contribution changed rapidly. '},\n"," {'paper_id': '9c127198193a8b257fe885ad0fabf886019ae47e',\n","  'section': 'Abstract',\n","  'text': 'The rapid global spread of the novel, pathogenic, SARS-CoV-2 causing the severe acute respiratory disease COVID-19, becomes a major health problem worldwide and pose the need for international predictive programs. Given the lack of both specific drugs and an efficient preventive vaccine, the expectation that the SARS-CoV-2 transmission rate might decrease in temperate regions during summer, dominated in the social scene. Here, we attempted a prediction of the worldwide spread of the infections based on climatic data, expressed by bioclimatic variables. The calculated probability maps shown that potential areas of infection follow a shift from the Tropical to Temperate and Mediterranean Bioclimatic regions. Maps show an increased probability of infections in Europe, followed by an expansion covering areas of the Middle East and Northern Africa, as well as Eastern coastal areas of North America, South-Eastern coastal areas of Latin America and two areas of Southern Australia. Our approach may, therefore, be of value for the incorporation of climatic influences in the design and implementation of public health policies. Maps are available (constantly updated) at https://navaak.shinyapps.io/CVRisk/. '},\n"," {'paper_id': '9c32d461dc9d4737756a990cf13bae1a03e078a9',\n","  'section': 'Abstract',\n","  'text': 'The respiratory tract surface is protected from inhaled pathogens by a secreted layer of mucus that is rich in mucin glycoproteins. Disrupted mucus production is a cardinal feature of chronic respiratory diseases but how this alteration affect interactions between mucins and pathogens is complex and poorly understood. Here, we identify a central and unexpected role for the major airway mucin MUC5AC in pathogenesis of virus-induced exacerbations of chronic obstructive pulmonary disease (COPD). Virus induction of MUC5AC is augmented in COPD compared to healthy subjects, is enhanced in frequent exacerbators and correlates with inflammation, symptom severity and secondary bacterial infection during exacerbation. MUC5AC is functionally related to inflammation as MUC5ACdeficient (Muc5ac-/-) mice had attenuated rhinovirus-induced airway inflammation whilst exogenous MUC5AC glycoprotein administration augmented virus-induced inflammatory responses and bacterial load. Mechanistically, MUC5AC-augmentation of rhinovirus-induced inflammation occurred through release of extracellular adenosine triphosphate (ATP). Therapeutic suppression of virus-induced MUC5AC release using an epidermal growth factor receptor (EGFR) inhibitor ameliorated exaggerated pro-inflammatory responses in a mouse COPD exacerbation model. Collectively, these studies demonstrate previously unrecognised pro-inflammatory effects of MUC5AC during infection and thus highlight a key unforeseen role in driving COPD exacerbation severity. author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '9c33486a49de4aea64ce61c0a2c21a88c316b6a8',\n","  'section': 'Abstract',\n","  'text': 'Objective: Establishing a social contact data sharing initiative and an online tool to assess mitigation strategies for COVID-19. Results: Using our online tool and the available social contact data, we illustrate that social distancing could have a considerable impact on reducing transmission for COVID-19. The effect itself depends on assumptions made about disease-specific characteristics and the choice of intervention(s). '},\n"," {'paper_id': '9c597bdfdf62cc41f668803f35d24b9aba9e1893',\n","  'section': 'Abstract',\n","  'text': 'We analyze the early data on COVID-19 expansion in selected European countries using an analytical parametric model. A description of the time dependence of the disease expansion and a method to evaluate trends of the expansion are proposed. Several features are observed in the data, namely a high predictability of the expansion of disease in Italy and a convergence of the \"pushback\" parameter towards a limiting value in all the countries where restrictive measures have been adopted. Basic predictions for the evolution of the disease expansion are made for selected countries with a stable evolution in the parametric space of the model. The findings presented here should contribute to the understanding of the behavior of the disease expansion and the role of the restrictive measures on the evolution of the expansion. '},\n"," {'paper_id': '9c5cd8e866b0bbec819ac143d39e19b04416cf10',\n","  'section': 'Abstract',\n","  'text': 'Background & Aims '},\n"," {'paper_id': '9ca81dfa68fcbff13166d810aca929027bb3763b',\n","  'section': 'Abstract',\n","  'text': 'While numerous studies have explored single-word naming, few have evaluated the behavioral and neural correlates of more naturalistic language, like connected speech, which we produce every day. Here, in a retrospective analysis of 120 participants at least six months following left hemisphere stroke, we evaluated the distribution of word errors (paraphasias) and associated brain damage during connected speech (picture description) and object naming. While paraphasias in connected speech and naming shared underlying neural substrates, analysis of the distribution of paraphasias suggested that lexicalsemantic load is likely reduced during connected speech. Using voxelwise lesion-symptom mapping (VLSM), we demonstrated that verbal (real word: semantically related and unrelated) and sound (phonemic and neologistic) paraphasias during both connected speech and naming loaded onto the left hemisphere ventral and dorsal streams of language, respectively. Furthermore, for the first time using both connected speech and naming data, we localized semantically related paraphasias to more anterior left hemisphere temporal cortex and unrelated paraphasias to more posterior left temporal and temporoparietal cortex. The connected speech results, in particular, highlight a gradient of specificity as one translates visual recognition from left temporo-occipital cortex to posterior and subsequently anterior temporal cortex. The robustness of VLSM results for sound paraphasias derived during connected speech was notable, in that analyses performed on sound paraphasias from the connected speech task, and not the naming task, demonstrated significant results following removal of lesion volume variance and related apraxia of speech variance. Therefore, connected speech may be a particularly sensitive task on which to evaluate further lexical-phonological processing in the brain. The results presented here demonstrate the related, though different, distribution of paraphasias during connected speech, confirm that paraphasias arising in connected speech and single-word naming likely share neural origins, and endorse the need for continued evaluation of the neural substrates of connected speech processes. '},\n"," {'paper_id': '9cdceb3cf6324b569f5b4cda124a427648f082c4',\n","  'section': 'Abstract',\n","  'text': 'COVID-19 is a global pandemic with high infectivity and pathogenicity, accounting for tens of thousands of deaths worldwide. Recent studies have found that the pathogen of COVID-19, SARS-CoV-2, shares the same cell receptor Angiotensin converting enzyme II (ACE2) with SARS-CoV. The pathological investigation of COVID-19 death showed that the lung had the characteristics of pulmonary fibrosis. However, how SARS-CoV-2 spreads from the lungs to other organs has not yet been determined. Here, we performed an unbiased evaluation of cell-type specific expression of ACE2 in healthy and fibrotic lungs, as well as in normal and failed adult human hearts, using published single-cell RNA-seq data. We found that ACE2 expression in fibrotic lungs mainly locates in arterial vascular cells, which might provide the route for bloodstream spreading of SARS-CoV-2. The failed human hearts have a higher percentage of ACE2expressing cardiomyocytes, and SARS-CoV-2 might attack cardiomyocytes through the bloodstream in patients with heart failure. Moreover, ACE2 was highly expressed in cells infected by RSV or MERS-CoV and in mice treated by LPS. Our findings indicate that patients with pulmonary fibrosis, heart failure, and virus infection have a higher risk and are more susceptible to SARS-CoV-2 infection. SARS-CoV-2 might attack other organs by getting into the bloodstream. This work provides new insights into SARS-CoV-2 blood entry and heart injury and might propose a therapeutic strategy to prevent patients from developing severe complications. All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '9cfdce851951c817101b36f7b292e744906474c4',\n","  'section': 'Abstract',\n","  'text': \"We live in a world of emerging new diseases and old diseases resurging in more aggressive forms. Drug development by pharmaceutical companies is a market-driven and costly endeavor, and thus it is often a challenge when drugs are needed for diseases endemic only to certain regions or which affect only a few patients. However, biomedical open data is accessible and reusable for reanalysis and generation of a new hypotheses and discovery. In this study, we leverage biomedical data and tools to analyze available data on Nipah Virus (NiV) infection. NiV infection is an emerging zoonosis that is transmissible to humans and is associated with high mortality rates. In this study, explored the application of computational drug repositioning and chemogenomic enrichment analyses using host transcriptome data to match drugs that could reverse the virus-induced gene signature. We performed analyses using two gene signatures: i) A previously published gene signature (n=34), and ii) a gene signature generated using the characteristic direction method (n= 5,533). Our predictive framework suggests that several drugs including FDA approved therapies like beclometasone, trihexyphenidyl, S-propranolol etc. could modulate the NiV infection induced gene signatures in endothelial cells. A target specific analysis of CXCL10 also suggests the potential application of Eldelumab, an investigative therapy for Crohn's disease and ulcerative colitis, as a putative candidate for drug repositioning. To conclude, we also discuss challenges and opportunities in clinical trials (n-of-1 and adaptive trials) for repositioned drugs. Further follow-up studies including biochemical assays and clinical trials are required to identify effective therapies for clinical use. Our proof-of-concept study highlights that translational bioinformatics methods including gene expression analyses and computational drug repositioning could augment epidemiological investigations in the context of an emerging disease with no effective treatment. All rights reserved. No reuse allowed without permission. \"},\n"," {'paper_id': '9d4146eb31b060c8b7295306b21ce8148d941981',\n","  'section': 'Abstract',\n","  'text': 'Decoding disease mechanisms for addressing unmet clinical need demands the rapid assimilation of the exponentially growing biomedical knowledge. These are either inherently unstructured and non-conducive to current computing paradigms or siloed into structured databases requiring specialized bioinformatics. Despite the recent renaissance in unsupervised neural networks for deciphering unstructured natural languages and the availability of numerous bioinformatics resources, a holistic platform for real-time synthesis of the scientific literature and seamless triangulation with deep omic insights and real-world evidence has not been advanced. Here, we introduce the nferX platform that makes the highly unstructured biomedical knowledge computable and supports the seamless visual triangulation with statistical inference from diverse structured databases. The nferX platform will accelerate and amplify the research potential of subject-matter experts as well as non-experts across the life science ecosystem (https://academia.nferx.com/). '},\n"," {'paper_id': '9d50dd6de221e2286fbe407cc3dc42b606315469',\n","  'section': 'Abstract',\n","  'text': 'In December 2019, an initial cluster of unexpected interstitial bilateral pneumonia emerged in Wuhan, Hubei province. A human-to-human transmission was immediately assumed and a previously unrecognized entity, termed coronavirus disease 19 (COVID-19) due to a novel coronavirus (2019-nCov) was suddenly described. The infection has rapidly spread out all over the world and Italy has been the first European Country experiencing the endemic wave with unexpected clinical severity in comparison with Asian countries. It has recently been shown that 2019-nCov utilizes host receptors namely angiotensin converting enzyme 2 (ACE2) as host receptor and host proteases for cell surface binding and internalization. Thus, a predisposing genetic background can give reason for interindividual disease susceptibility and/or severity. Taking advantage of the Network of Italian Genomes (NIG), here we mined around 7000 exomes from 5 different Centers looking for ACE2 variants. A number of variants with a potential impact on protein stability were identified. Among these, three missense changes, p.Asn720Asp, p.Lys26Arg, p.Gly211Arg (MAF 0.002 to 0.015), which have never been reported in the Eastern Asia population, were predicted to interfere with protein u and stabilization. Rare truncating variants likely interfering with the internalization process and one missense variant, p.Trp69Cys, predicted to interfere with 2019-nCov spike protein binding were also observed. These findings suggest that a predisposing genetic background may contribute to the observed inter-individual clinical variability associated with COVID-19. They allow an evidence-based risk assessment opening up the way to personalized preventive measures and therapeutic options. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': '9d8f7b4f40c0790576b6d698b0daf9ad8f078338',\n","  'section': 'Abstract',\n","  'text': 'Background: On 12 March, the World Health Organization Director-General declared that \"the threat of a global pandemic has become a reality\", and the disease caused by the novel coronavirus, known as COVID-19, has become a global concern. Chinese efforts in curbing the virus have widely been recognized. Even the WHO has lauded the efforts of the Chinese government and advised the world to learn from China in fighting the disease. Since the outbreak of COVID-19, to curb the spread of the epidemic, the Chinese government has implemented unprecedented prevention interventions at the nationwide level. Currently, the outbreak in Wuhan is changing in a positive direction and has been effectively controlled. However, it is not clear what these measures were and how these measures changed to curb the outbreak of . CC-BY 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint . cumulates cases to compare the control effects and changing trends. We performed a retrospective description of these intervention strategies from three stages. Besides, from the perspective of public health, the experiences and lessons exposed by this outbreak were roughly summarized. Results: These non-pharmacology interventions measures adopted by the Chinese government by the instruction and spirit of President Xi Jinping were timely and efficient. The present study was conducted to comprehensively analyze from a social epidemiology context. The results confirmed that these radical interventions taken by the Chinese government were effective, ambitious, and agile. However, we must be aware that the epidemic situation in Wuhan is still challenging. '},\n"," {'paper_id': '9d9d513cec6b87c5b9c00215261e00ca946648a5',\n","  'section': 'Abstract',\n","  'text': 'Background: Twitter has been used to track trends and disseminate health information during viral epidemics. On January 21, 2020, the CDC activated its Emergency Operations Center and the WHO released its first situation report about Coronavirus disease 2019 (COVID-19), sparking significant media attention. How Twitter content and sentiment has evolved in the early stages of any outbreak, including the COVID-19 epidemic, has not been described. Objective: To quantify and understand early changes in Twitter activity, content, and sentiment about the COVID-19 epidemic. Design: Observational study. Setting: Twitter platform. Participants: All Twitter users who created or sent a message from January 14th to 28th, 2020. Measurements: We extracted tweets matching hashtags related to COVID-19 and measured frequency of keywords related to infection prevention practices, vaccination, and racial prejudice. We performed a sentiment analysis to identify emotional valence and predominant emotions. We conducted topic modeling to identify and explore discussion topics over time. Results: We evaluated 126,049 tweets from 53,196 unique users. The hourly number of COVID-19-related tweets starkly increased from January 21, 2020 onward. Nearly half (49.5%) of all tweets expressed fear and nearly 30% expressed surprise. The frequency of racially charged tweets closely paralleled the number of newly diagnosed cases of COVID-19. The economic and political impact of the COVID-19 was the most commonly discussed topic, while public health risk and prevention were among the least discussed. Conclusion: Tweets with negative sentiment and emotion parallel the incidence of cases for the COVID-19 outbreak. Twitter is a rich medium that can be leveraged to understand public sentiment in real-time and target public health messages based on user interest and emotion. '},\n"," {'paper_id': '9da357734a3bfc5b51dc2c63b6b4ab28930767ab',\n","  'section': 'Abstract',\n","  'text': 'well as cytokines important for immune reactions and inflammation. Our data indicate that patients with severe COVID-19 exhibited an overall decline of lymphocytes including CD4+ and CD8+ T cells, B cells, and NK cells. The number of immunosuppressive regulatory T cells was moderately increased in patients with mild COVID-19. IL-2, IL-6, and IL-10 were remarkably up-regulated in patients with severe COVID-19. The levels of IL-2 and IL-6 relative to the length of hospital stay underwent a similar \" rise-decline\"pattern, probably reflecting the therapeutic effect. In conclusion, our study shows that the comprehensive decrease of lymphocytes, and the elevation of IL-2 and IL-6 are reliable indicators of severe COVID-19. '},\n"," {'paper_id': '9daed5e675ac113d1939056dcebbb1f32ec0d7ec',\n","  'section': 'Abstract',\n","  'text': 'Over the past 20 years, several coronaviruses have crossed the species barrier into humans, causing outbreaks of severe, and often fatal, respiratory illness. Since SARS-CoV was first identified in animal markets, global viromics projects have discovered thousands of coronavirus sequences in diverse animals and geographic regions. Unfortunately, there are few tools available to functionally test these novel viruses for their ability to infect humans, which has severely hampered efforts to predict the next zoonotic viral outbreak. Here we developed an approach to rapidly screen lineage B betacoronaviruses, such as SARS-CoV and the recent 2019-nCoV, for receptor usage and their ability to infect cell types from different species. We show that host protease processing during viral entry is a significant barrier for several lineage B viruses and that bypassing this barrier allows several lineage B viruses to enter human cells through an unknown receptor. We also demonstrate how different lineage B viruses can recombine to gain entry into human cells and confirm that human ACE2 is the receptor for the recently emerging 2019-nCoV. author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '9e0395780f1083684db7a28b9c912337a85ce4d4',\n","  'section': 'Abstract',\n","  'text': \"The current COVID-19 pandemic has led to a dramatic shortage of masks and other personal protective equipment (PPE) in hospitals around the globe [1] . One component of PPE that is in particular demand are disposable N95 face masks. To alleviate this, many methods of N95 mask sterilization have been studied and proposed with the hope of being able to safely reuse masks [2] . Two major considerations must be made when re-sterilizing masks: (1) the sterilization method effectively kills pathogens, penetrating into the fibers of the mask, and (2) the method does not degrade the operational integrity of the N95 filters. We studied Cobalt-60 ( 60 Co) gamma irradiation as a method of effective sterilization without inducing mask degradation. Significant literature exists supporting the use of gamma radiation as a sterilization method, with viral inactivation of SARS-CoV reported at doses of at most 10 kGy [3] , with other studies supporting 5 kGy for many types of viruses [4] . However, concerns have been raised about the radiation damaging the fiber material within the mask, specifically by causing cross-linking of polymers, leading to cracking and degradation during fitting and/or deployment [5, 6] . A set of 3M 8210 and 9105 masks were irradiated using MIT's 60 Co irradiator. Three masks of each type received 0 kiloGray (kGy), 10 kGy and 50 kGy of approximately 1.3 MeV gamma radiation from the circular cobalt sources, at a dose rate of 2.2kGy per hour. Following this sterilization procedure, the irradiated masks passed a OSHA Gerson Qualitative Fit Test QLFT 50 (saccharin apparatus) [7] when donned correctly, performed at the Brigham and Women's Hospital, in a blinded study repeated in triplicate. However, the masks' filtration of 0.3 µm particles was significantly degraded, even at 10 kGy. These results suggest against gamma, and possibly all ionizing radiation, as a method of disposable N95 sterilization. Even more importantly, they argue against using the qualitative fit test alone to assess mask integrity. \"},\n"," {'paper_id': '9e470a251b0e5f7225c8e2acd5a82d06392c165f',\n","  'section': 'Abstract',\n","  'text': 'An automated statistical and error analysis of 45 countries or regions with more than 1000 cases of COVID-19 as of March 28, 2020, has been performed. This study reveals differences in the rate of disease spreading rate over time in different countries. This survey observes that most countries undergo a beginning exponential growth phase, which transitions into a power-law phase, as recently suggested by Ziff and Ziff. Tracking indicators of growth, such as the power-law exponent, are a good indication of the relative danger different countries are in and show when social measures are effective towards slowing the spread. The data compiled here are usefully synthesizing a global picture, identifying country to country variation in spreading, and identifying countries most at risk. This analysis may factor into how best to track the effectiveness of social distancing policies and quarantines in real-time as data is updated each day. '},\n"," {'paper_id': '9e4b67ef6613c31882bb561d28770203da19771b',\n","  'section': 'Abstract',\n","  'text': 'Background: In December 2019, an outbreak of new type of coronavirus named COVID-19 occurred in Wuhan, Hubei Province, China. In a very short time, this virus spread rapidly over China, greatly threatening public health and economic development. The Chinese government acted quickly and implemented a series of strategies to prevent diffusion of this disease. We therefore sought to evaluate the effects of these Chinese strategies for controlling the spread of COVID-19. Commission websites of China, we performed model fitting and calculated the growth speed of cumulative confirmed patients. We further analyzed the time when this growth speed, the rate of the number of new cases, reached its maximum (Speedmax). Comparing different times to Speedmax of different areas in China, we calculated the dates at which the growth speed began to decline in different areas. Also, The number of plateaus were analyzed. : The quartic model showed the best fit. For almost all areas in mainland China, the speed of infections reached Speedmax and began to decline within 14 days; exceptions were Hebei, Heilongjiang, Hainan, Guizhou, and Hubei. The number of plateaus was significantly correlated with the emigration index. However, the distance from other areas to Hubei and the number of plateaus had little influence on when a province or area arrived at Speedmax. Once strict intervention strategies were implemented, diffusion and deterioration of COVID-19 were inhibited quickly and effectively over China. Conclusion: Our study suggests that Chinese strategies are highly effective on controlling the diffusion and deterioration of the novel coronavirus-infected pneumonia. These strategies supply experience and guidelines for other countries to control the COVID-19 epidemic. : medRxiv preprint the diffusion and deterioration of COVID-19 were inhibited quickly and effectively over China. With the exception of Wuhan, even Hubei, where the cumulative confirmed patients reached 17,680 on February 23, 2020, the speed of growth still reached Speedmax and began to decline in 14-15 days. '},\n"," {'paper_id': '9e780780dbaf35c0d537ddb290dfd484148a3c55',\n","  'section': 'Abstract',\n","  'text': 'Estimating infectious disease parameters such as the serial interval (time between symptom onset in primary and secondary cases) and reproductive number (average number of secondary cases produced by a primary case) are important to understand infectious disease dynamics. Many estimation methods require linking cases by direct transmission, a difficult task for most diseases. Using a subset of cases with detailed genetic or contact investigation data to develop a training set of probable transmission events, we build a model to estimate the relative transmission probability for all case-pairs from demographic, spatial and clinical data. Our method is based on naive Bayes, a machine learning classification algorithm which uses the observed frequencies in the training dataset to estimate the probability that a pair is linked given a set of covariates. In simulations we find that the probabilities estimated using genetic distance between cases to define training transmission events are able to distinguish between truly linked and unlinked pairs with high accuracy (area under the receiver operating curve value of 95%). Additionally only a subset of the cases, 10-50% depending on sample size, need to have detailed genetic data for our method to perform well. We show how these probabilities can be used to estimate the average effective reproductive number and apply our method to a tuberculosis outbreak in Hamburg, Germany. All rights reserved. No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/612945 doi: bioRxiv preprint 3 Our method is a novel way to infer transmission dynamics in any dataset when only a subset of cases has rich contact investigation and/or genetic data. '},\n"," {'paper_id': '9e94f9379fd74fcacc4f3a57e03cbe9035efee8e',\n","  'section': 'Abstract',\n","  'text': 'Three anti-HIV drugs, ritonavir, lopinavir and darunavir, might have therapeutic effect on coronavirus disease 2019 . In this study, the structure models of two severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) proteases, coronavirus endopeptidase C30 (CEP_C30) and papain like viral protease (PLVP), were built by homology modeling. Ritonavir, lopinavir and darunavir were then docked to the models, respectively, followed by energy minimization of the protease-drug complexes. In the simulations, ritonavir can bind to CEP_C30 most suitably, and induce significant conformation changes of CEP_C30; lopinavir can also bind to CEP_C30 suitably, and induce significant conformation changes of CEP_C30; darunavir can bind to PLVP suitably with slight conformation changes of PLVP. It is suggested that the therapeutic effect of ritonavir and lopinavir on COVID-19 may be mainly due to their inhibitory effect on CEP_C30, while ritonavir may have stronger efficacy; the inhibitory effect of darunavir on SARS-CoV-2 and its potential therapeutic effect may be mainly due to its inhibitory effect on PLVP. author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': '9f5625d9287d8c6729215a20add99c8cd4ba08cb',\n","  'section': 'Abstract',\n","  'text': 'As of early March, 2019-nCoV has infected more than one hundred thousand people and claimed thousands of lives. 2019-nCoV is a novel form of coronavirus that causes COVID-19 and has high similarity with SARS-CoV. No approved vaccine yet exists for any form of coronavirus. Here we use computational tools from structural biology and machine learning to identify 2019-nCoV T-cell and B-cell epitopes based on viral protein antigen presentation and antibody binding properties. These epitopes can be used to develop more effective vaccines and identify neutralizing antibodies. We identified 405 viral peptides with good antigen presentation scores for both human MHC-I and MHC-II alleles, and two potential neutralizing Bcell epitopes near the 2019-nCoV spike protein receptor binding domain (440-460 and 494-506). Analyzing mutation profiles of 68 viral genomes from four continents, we identified 96 coding-change mutations. These mutations are more likely to occur in regions with good MHC-I presentation scores (p=0.02). No mutations are present near the spike protein receptor binding domain. Based on these findings, the spike protein is likely immunogenic and a potential vaccine candidate. We validated our computational pipeline with SARS-CoV experimental data. '},\n"," {'paper_id': '9f85a1904bda3efb0c3c84f2043bd35986801912',\n","  'section': 'Abstract',\n","  'text': 'We constructed a simple Susceptible-Infected-Infectious-Excluded model of the spread of COVID-19. The model is parametrised only by the average incubation period, τ, and two rate parameters: contact rate, r C , and exclusion rate, r E . The rates can be manipulated by non-therapeutic interventions and determine the basic reproduction number, R = r C /r E , and, together with τ, the daily multiplication coefficient at the early exponential phase, β. Initial β determines the reduction of r C required to contain epidemic spread. In the long-term, we consider a scenario based on typical social behaviours, in which r C first decreases in response to a surge of daily new cases, forcing people to self-isolate, and then slowly increases when people gradually accept higher risk. Consequently, initial abrupt epidemic spread is followed by a plateau and slow regression. This scenario, although economically and socially devastating, will grant time to develop, produce, and distribute a vaccine, or at least limit daily cases to a manageable number. '},\n"," {'paper_id': '9fb5ca87ca47d35d87161158ae37fc0ca642c51e',\n","  'section': 'Abstract',\n","  'text': '10 '},\n"," {'paper_id': 'a01672035e0b9d21f9a934b9c9071610513325f2',\n","  'section': 'Abstract',\n","  'text': 'Note: this report is a pre-print and has not yet been peer-reviewed. As the number of novel coronavirus cases grows both inside and outside of China, public health authorities require evidence on the effectiveness of control measures such as thermal screening of arrivals at airports. We evaluated the effectiveness of exit and entry screening for 2019-nCoV infection. In our baseline scenario, we estimated that 46.5% (95%CI: 35.9 to 57.7) of infected travellers would not be detected, depending on the incubation period, sensitivity of exit and entry screening, and the proportion of cases which are asymptomatic. Airport screening is unlikely to detect a sufficient proportion of 2019-nCoV infected travellers to avoid entry of infected travellers. We developed an online tool so that results can be updated as new information becomes available. As of 29 Jan 2019, 5,997 confirmed and 9,239 suspected cases of 2019-nCoV have been reported from China with 132 deaths confirmed so far 1 . There were cases in at least 15 other countries, identified due to symptoms plus recent travel history to Hubei province, China, which strongly suggests that the reported cases constitute only a small fraction of the actual number of infected individuals in China 2 . While the most affected region, Hubei province, has now ceased air travel and closed major public transport routes 3 it is unlikely that these measures will fully contain the outbreak. Despite limited evidence for its effectiveness 4,5 , airport screening is frequently implemented to limit the probability of infected cases entering other regions or countries. Here we use the available evidence to evaluate the effectiveness of exit and entry screening for detecting travellers with 2019-nCoV. We also provide an online app so that results can be updated as new information becomes available. . CC-BY 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': 'a05179c57d5e1cdddc839ff3c0b70a614efbb6cb',\n","  'section': 'Abstract',\n","  'text': 'The recent outbreak of pneumonia in Wuhan, China caused by the 2019 Novel Coronavirus (2019-nCoV) emphasizes the importance of detecting novel viruses and predicting their risks of infecting people. In this report, we introduced the VHP (Virus Host Prediction) to predict the potential hosts of viruses using deep learning algorithm. Our prediction suggests that 2019-nCoV has close infectivity with other human coronaviruses, especially the severe acute respiratory syndrome coronavirus (SARS-CoV), Bat SARS-like Coronaviruses and the Middle East respiratory syndrome coronavirus (MERS-CoV). Based on our prediction, compared to the Coronaviruses infecting other vertebrates, bat coronaviruses are assigned with more similar infectivity patterns with 2019-nCoVs. Furthermore, by comparing the infectivity patterns of all viruses hosted on vertebrates, we found mink viruses show a closer infectivity pattern to 2019-nCov. These consequences of infectivity pattern analysis illustrate that bat and mink may be two candidate reservoirs of 2019-nCov.These results warn us to beware of 2019-nCoV and guide us to further explore the properties and reservoir of it. One Sentence Summary: It is of great value to identify whether a newly discovered virus has the risk of infecting human. Guo et al. proposed a virus host prediction method based on deep learning to detect what kind of host a virus can infect with DNA sequence as input. '},\n"," {'paper_id': 'a09c1ad0f05bbcc256cd667ad7411e6690134370',\n","  'section': 'Abstract',\n","  'text': 'To analyze the impact of different patterns of migration flow in two cities, Hefei and Shenzhen, on the epidemic and disease control of Coronavirus Disease 2019 , in order to provide insight for making differentiated controlling policies. Hefei and Shenzhen between January 19 and February 11, 2020, from data officially published by the provincial and municipal Centers for Disease Control and Prevention (CDC). From these data we calculated basic reproduction number R0 to reflect the rate of spread of COVID-19 in these cities. Aggregated data of population migration during the same period was extracted from Baidu Migration. The change of R0 in the two cites were analyzed and compared. Spearman correlation analysis between R0 and population inflow from epidemic focus were performed. '},\n"," {'paper_id': 'a0ecc9b46addbc1faab491434b5331702f7883b8',\n","  'section': 'Abstract',\n","  'text': 'Background: A novel coronavirus named as \"SARS-CoV-2\" has spread widely in many countries since 38 December 2019, especially in China. This study aimed to quantify the age-specific transmissibility by 39 using a mathematical model. '},\n"," {'paper_id': 'a11f079f3652b13e131525ee5b691525fac47382',\n","  'section': 'Abstract',\n","  'text': \"The key parameter that characterizes the transmissibility of a disease is the reproduction number . If it exceeds 1, the number of incident cases will inevitably grow over time, and a large epidemic is possible. To prevent the expansion of an epidemic, must be reduced to a level below 1. To estimate the reproduction number, the probability distribution function of the generation interval of an infectious disease is required to be available; however, this distribution is often unknown. In this letter, given the incomplete information for the generation interval, we propose a maximum entropy method to estimate the reproduction number. Based on this method, given the mean value and variance of the generation interval, we first determine its probability distribution function and in turn estimate the real-time values of reproduction number of COVID-19 in China. By applying these estimated reproduction numbers into the susceptible-infectious-removed epidemic model, we simulate the evolutionary track of the epidemic in China, which is well in accordance with that of the real incident cases. The simulation results predict that China's epidemic will gradually tend to disappear by May 2020 if the quarantine measures can continue to be executed. \"},\n"," {'paper_id': 'a14e3a346bac0f3033de8004525192f0b2687d4a',\n","  'section': 'Abstract',\n","  'text': 'On January 20, 2020, the first COVID-19 case was confirmed in South Korea. After a rapid outbreak, the number of incident cases has been consistently decreasing since early March; this decrease has been widely attributed to its intensive testing. We report here on the likely role of social distancing in reducing transmission in South Korea. Our analysis suggests that transmission may still be persisting in some regions. '},\n"," {'paper_id': 'a1538022297166192b106cda55a1840ffb92f743',\n","  'section': 'Abstract',\n","  'text': '5 '},\n"," {'paper_id': 'a1bff76ce360e8990b0a4ee2a5228a6e6e63d9c1',\n","  'section': 'Abstract',\n","  'text': '1 Objective: To estimate the serial interval of novel coronavirus (COVID-19) 2 from information on 28 infector-infectee pairs. 3 We collected dates of illness onset for primary cases (infectors) and 4 Acknowledgments '},\n"," {'paper_id': 'a1e6af006b51248af8af96c34c0e2f452d764a5c',\n","  'section': 'Abstract',\n","  'text': 'The ongoing COVID-19 pandemic has caused an unprecedented need for rapid diagnostic testing. The Centers for Disease Control and Prevention (CDC) and the World Health Organization (WHO) recommend a standard assay that includes an RNA extraction step from a nasopharyngeal (NP) swab followed by reverse transcription-quantitative polymerase chain reaction (RT-qPCR) to detect the purified SARS-CoV-2 RNA. The current global shortage of RNA extraction kits has caused a severe bottleneck to COVID-19 testing. We hypothesized that SARS-CoV-2 RNA could be detected from NP samples via a direct RT-qPCR assay that omits the RNA extraction step altogether, and tested this hypothesis on a series of blinded clinical samples. The direct RT-qPCR approach correctly identified 92% of NP samples (n = 155) demonstrated to be positive for SARS-CoV-2 RNA by traditional clinical diagnostic RT-qPCR that included an RNA extraction. Thus, direct RT-qPCR could be a front-line approach to identify the substantial majority of COVID-19 patients, reserving a repeat test with RNA extraction for those individuals with high suspicion of infection but an initial negative result. This strategy would drastically ease supply chokepoints of COVID-19 testing and should be applicable throughout the world. The ongoing COVID-19 pandemic has put exceptional strain on public health laboratories, hospital laboratories, and commercial laboratories as they attempt to keep up with demands for SARS-CoV-2 testing. The current diagnostic testing methods recommended by the Centers for Disease Control and Prevention (CDC) in the United States and the World Health Organization (WHO) are traditional RT-qPCR assays that require two steps: first, an RNA extraction from patient nasopharyngeal (NP) swab material, followed by RT-qPCR amplification of the extracted RNA to detect viral RNA 1-3 . The major bottleneck to widespread SARS-CoV-2 testing lies at the RNA extraction step. The simplest manual kit (the Qiagen Viral RNA Mini) is no longer available, and reagents and supplies for the larger automated instruments are extremely limited with uncertain supply chains. While substitution of other RNA extraction kits 4,5 is possible, they too are in limited supply. The current bottleneck is not simply the availability of RNA extraction kits, but also the cost of the extraction assay, the labor and time required to perform it, and the fact that it is rate limiting compared to the downstream RT-qPCR analysis. To address this issue, we tested the unconventional approach of skipping the RNA extraction step altogether and directly loading patient swab material into the RT-qPCR mix. Herein, we report that this approach (which we refer to hereafter as \"direct RT-qPCR\") correctly identified 92% of samples (n =155) previously shown to be positive for SARS-CoV-2 RNA by conventional RT-qPCR featuring an RNA extraction. Thus, our results suggest that this streamlined assay could greatly alleviate constraints to COVID-19 testing in many regions of the world. We initially conducted a pilot experiment using NP swabs from two COVID-19 patients who had previously been verified for SARS-CoV-2 infection by the Vermont Department of Health Laboratory (VDHL) using the CDC\\'s recommended RT-qPCR test. Both patient samples, which were originally collected as NP swabs in 3 mL of M6 viral transport medium (termed diluent hereafter), were pooled (equi-volume). RNA was extracted from 140 µL of the pooled sample using the QIAamp Viral RNA Mini kit, and purified RNA representing 11.3 µL of the original swab diluent was detected as positive via standard RT-qPCR using the CDC 2019-nCoV N3 primer/probe set, with a CT of 18.7. In parallel, we added 7 µL of the pooled COVID-19 patient NP swab diluent directly to the RT-qPCR reaction, and found that SARS-CoV-2 RNA was . '},\n"," {'paper_id': 'a1eaa043e663ce32ae55d870e63f0c08e6a2db8e',\n","  'section': 'Abstract',\n","  'text': 'In this work we propose a data-driven age-structured census-based SIRD-like epidemiological model capable of forecasting the spread of COVID-19 in Brazil. We model the current scenario of closed schools and universities, social distancing of people above sixty years old and voluntary home quarantine to show that it is still not enough to protect the health system by explicitly computing the demand for hospital intensive care units. We also show that an urgent intense quarantine might be the only solution to avoid the collapse of the health system and, consequently, to minimize the quantity of deaths. On the other hand, we demonstrate that the relaxation of the already imposed control measures in the next days would be catastrophic. . CC-BY 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': 'a22ceac8d5e1cfaf2e79557d3c794458d0e1cdbe',\n","  'section': 'Abstract',\n","  'text': 'In numerous instances, tracking the biological significance of a nucleic acid sequence 15 can be augmented through the identification of environmental niches in which the sequence 16 of interest is present. Many metagenomic datasets are now available, with deep sequencing of 17 samples from diverse biological niches. While any individual metagenomic dataset can be 18 readily queried using web-based tools, meta-searches through all such datasets are less 19 accessible. In this brief communication, we demonstrate such a meta-meta-genomic 20 approach, examining close matches to the Wuhan coronavirus 2019-nCoV in all high-21 throughput sequencing datasets in the NCBI Sequence Read Archive accessible with the 22 keyword \"virome\". In addition to the homology to bat coronaviruses observed in descriptions ), we note a strong 25 homology to numerous sequence reads in a metavirome dataset generated from the lungs of 26 deceased Pangolins reported by Liu et al. (Viruses 11:11, 2019, 27 http://doi.org/10.3390/v11110979). Our observations are relevant to discussions of the 28 derivation of 2019-nCoV and illustrate the utility and limitations of meta-metagenomic 29 search tools in effective and rapid characterization of potentially significant nucleic acid 30 sequences. : bioRxiv preprint 65 Read Archive (SRA) using the SRA-tools package (version 2.9.1). The latter sequence data 66 were downloaded as .sra files using the prefetch tool, with extraction to readable format 67 (.fasta.gz) using the NCBI fastq-dump tool. Each of these manipulations can fail some 68 fraction of the time. Obtaining the sequences can fail due to network issues, while extraction 69 in readable format occasionally fails for unknown reasons. Thus the workflow continually 70 requests .sra files with ncbi-prefetch until at least some type of file is obtained, followed by 71 attempts to unpack into .fasta.gz format until one such file is obtained from each .sra file. Metagenomic datasets for analysis were chosen through a keyword search of the SRA 73 : bioRxiv preprint 274 2020. Full-genome evolutionary analysis of the novel corona virus (2019-nCoV) rejects 275 the hypothesis of emergence as a result of a recent recombination event. Infection, 276 Genetics and Evolution 79:104212. '},\n"," {'paper_id': 'a2ba0a34df31fd8059ed22787331a75b65eafa74',\n","  'section': 'Abstract',\n","  'text': \"Background: As of February 25, 2019, 875 cases of Ebola virus disease (EVD) were reported in North Kivu and Ituri Provinces, Democratic Republic of Congo. Since the beginning of October, the outbreak has largely shifted into regions in which active armed conflict has occurred, and in which EVD cases and their contacts have been difficult for health workers to reach. We used available data on the current outbreak with case-count time series from prior outbreaks to project the short-term and long-term course of the outbreak. Methods: For short-and long-term projections, we modeled Ebola virus transmission using a stochastic branching process that assumes gradually quenching transmission rates estimated from past EVD outbreaks, with outbreak trajectories conditioned on agreement with the course of the current outbreak, and with multiple levels of vaccination coverage. We used two regression models to estimate similar projection periods. Short-and long-term projections were estimated using negative binomial autoregression and Theil-Sen regression, respectively. We also used Gott's rule to estimate a baseline minimum-information projection. We then constructed an ensemble of forecasts to be compared and recorded for future evaluation against final outcomes. From August 20, 2018 to February 25, 2019, short-term model projections were validated against known case counts. Results: During validation of short-term projections, from one week to four weeks, we found models consistently scored higher on shorter-term forecasts. Based on case counts as of February 25, the stochastic model projected a median case count of 933 cases by February 18 (95% prediction interval: 872-1054) and 955 cases by March 4 (95% prediction interval: 874-1105), while the auto-regression model projects median case counts of 889 (95% prediction interval: 876-933) and 898 (95% prediction interval: 877-983) cases for those dates, respectively. Projected median final counts range from 953 to 1,749. Although the outbreak is already larger than all past Ebola outbreaks other than the 2013-2016 outbreak of over 26,000 cases, our models do not project that it is likely to grow to that scale. The stochastic model estimates that vaccination coverage in this outbreak is lower than reported in its trial setting in Sierra Leone. Conclusions: Our projections are concentrated in a range up to about 300 cases beyond those already reported. While a catastrophic outbreak is not projected, it is not ruled out, and prevention and vigilance are warranted. Prospective validation of our models in real time allowed us to generate more accurate short-term forecasts, and this process may prove useful for future real-time short-term forecasting. We estimate that transmission rates are higher than would be seen under target levels of 62% coverage due to contact tracing and vaccination, and this model estimate may offer a surrogate indicator for the outbreak response challenges. Author summary Kivu and Ituri Provinces, Democratic Republic of Congo. Since the beginning of October 2018, the outbreak has largely shifted into regions in which active armed conflict has been reported, and in which EVD cases and their contacts have been difficult for health workers to reach. We used an ensemble of models to estimate EVD transmission rates and to forecast the short-and long-term course of the outbreak. Our models project that a final size of roughly up to 300 additional cases is most likely, and estimate that transmission rates are higher than would be seen under optimal levels of contact tracing and vaccination. While a catastrophic outbreak is not projected, is it not ruled out, and prevention and vigilance are warranted. 4 30 and identified the initial epicenter to be Mabalako. This region of DRC (Fig 1) has 5 over five million inhabitants, some of whom suffer from armed conflict, humanitarian 6 crisis, and displacement to the bordering countries of Uganda and Rwanda [2]. Since 7 the outbreak began, its magnitude has surpassed all other EVD outbreaks in DRC, 8 becoming the second largest in history to date. As of February 25, 2019, 875 EVD cases 9 had been reported (65 probable and 810 confirmed), and the Ministry of Health of DRC, 10 \"},\n"," {'paper_id': 'a3116a1ea2e157c8855023d968cc0f0bd6c9846f',\n","  'section': 'Abstract',\n","  'text': 'There is concern about a new coronavirus, the 2019-nCoV, as a global public health threat. In this article, we provide a preliminary evolutionary and molecular epidemiological analysis of this new virus. A phylogenetic tree has been built using the 15 available whole genome sequence of 2019-nCoV and 12 whole genome sequences highly similar sequences available in gene bank (5 from SARS, 2 from MERS and 5 from Bat SARS-like Coronavirus). FUBAR analysis shows that the Nucleocapsid and the Spike Glycoprotein has some sites under positive pressure while homology modelling helped to explain some molecular and structural differences between the viruses. The phylogenetic tree showed that 2019.nCoV significantly clustered with Bat SARS-like Coronavirus sequence isolated in 2015, whereas structural analysis revealed mutation in S and nucleocapsid proteins. From these results, 2019nCoV could be considered a coronavirus distinct from SARS virus, probably transmitted from bats or another host where mutations conferred upon it the ability to infect humans. author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': 'a33f451d8e2caff57c1133f14198454a64993c47',\n","  'section': 'Abstract',\n","  'text': '1 '},\n"," {'paper_id': 'a34d2c1876ffedc4f95f3af13ac06039cd38a5b0',\n","  'section': 'Abstract',\n","  'text': 'Adjusting for delay from confirmation-to-death, we estimated case and infection fatality ratios (CFR, IFR) for COVID-19 on the Diamond Princess ship as 2.3% (0.75%-5.3%) and 1.2% (0.38-2.7%). Comparing deaths onboard with expected deaths based on naive CFR estimates using China data, we estimate IFR and CFR in China to be 0.5% (95% CI: 0.2-1.2%) and 1.1% (95% CI: 0.3-2.4%) respectively. '},\n"," {'paper_id': 'a36bfdd1c9a1666401269fb0c08a3c2922bd6812',\n","  'section': 'Abstract',\n","  'text': 'The asymptomatic carriers identified from close contacts were prone to be mildly ill during hospitalization. However, the communicable period could be up to three weeks and the communicated patients could develop severe illness. These results highlighted the importance of close contact tracing and longitudinally surveillance via virus nucleic acid tests. Further isolation recommendation and continuous nucleic acid tests may also be recommended to the patients discharged. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': 'a37c116f62a9187496a8d5ad0bbbb193b7fd77e0',\n","  'section': 'Abstract',\n","  'text': 'We investigate the effects of social distancing in controlling the impact of the COVID-19 epidemic using a simple susceptible-infected-removed epidemic model. We show that an alternative or complementary approach based on targeted isolation of the vulnerable subpopulation may provide a more efficient and robust strategy at a lower economic and social cost within a shorter timeframe resulting in a collectively immune population. . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint . '},\n"," {'paper_id': 'a388cca261e8af7936cf3b79c35f43442f65706b',\n","  'section': 'Abstract',\n","  'text': \"SARS-CoV-2 and SARS-CoV share a common human receptor ACE2. Protein-protein interaction structure modeling indicates that spike-RBD of the two viruses also has similar overall binding conformation and binding free energy to ACE2. In vitro assays using recombinant ACE2 proteins and ACE2 expressing cells confirmed the two coronaviruses' similar binding affinities to ACE2. The above studies provide experimental supporting evidences and possible explanation for the high transmissibility observed in the SARS-CoV-2 outbreak. Potent ACE2-blocking SARS-CoV neutralizing antibodies showed limited cross-binding and neutralizing activities to SARS-CoV-2. ACE2-non-blocking SARS-CoV RBD antibodies, though with weaker neutralizing activities against SARS-CoV, showed positive cross-neutralizing activities to SARS-CoV-2 with an unknown mechanism. These findings suggest a trade-off between the efficacy and spectrum for therapeutic antibodies to different coronaviruses, and hence highlight the possibilities and challenges in developing broadly protecting antibodies and vaccines against SARS-CoV-2 and its future mutants. Various estimates and analyses suggest SARS-CoV-2 and Severe Acute Respiratory Syndrome coronavirus (SARS-CoV) may have similar transmissibility with an estimated reproductive number (R 0 ) of 3.77 (2.23-4.82) for SARS-CoV-2 and between 2.9-3.3 for SARS-CoV 8 . SARS-CoV-2 and SARS-CoV share a common host-cell receptor protein, angiotensin converting enzyme 2 (ACE2), expressed on epithelial cells in the respiratory track system and various human organs, such as the lung 9,10 . Receptor recognition by coronaviruses is the first and essential step for infecting host cells 11,12 . An envelope-anchored trimeric spike (S) protein mediates the binding of the two coronaviruses to human ACE2 13,14 , and is cleaved by the host protease into two separate polypeptides as S1, which contains the receptor binding domain (RBD), and S2, which is responsible for the homotrimer formation and mediates fusion of the virion with cellular membranes 15 . The receptor binding motif (RBM) in RBD is responsible for direct binding to ACE2 and its binding affinity may directly affects the virus infectivity and transmissibility 16, 17 . The amino acid sequence homology between SARS-CoV and SARS-CoV-2 approximates to 75% for the spike proteins, and are 73.7% and 50.0% for RBDs and RBMs, respectively. Here we investigated the optimized complex structure conformations between the RBDs of SARS-CoV-2 \"},\n"," {'paper_id': 'a3b3d6bd47a42fed114e4b0af5c15b4d7ffdc9f4',\n","  'section': 'Abstract',\n","  'text': \"Although the rapidly rising transmission trend of COVID-19 in Wuhan has been controlled in late February 2020, the outbreak still caused a global pandemic afterward. Understanding Wuhan's COVID-19 transmission dynamics and the effects of prevention approaches is of significant importance for containing virus global transmission. However, most of the recent studies focused on the early outbreaks without considering improvements in diagnostic capability and effects of prevention measures together, thus the estimated results may only reflect the facts in a given period of time. We constructed a stochastic susceptible-exposed-infected-quarantined-recovered (SEIQR) model, embedding with latent periods under different prevention measures and proportions of documented infections to characterize the Wuhan COVID-19 transmission cross different stages of the outbreak. The epidemiological parameters were estimated using a particle filtering approach. Our model successfully reproduced the dynamics of the Wuhan local epidemic with two peaks on February 4 and February 12 separately. Prevention measures determined the time of reaching the first peak and caused an 87% drop in the R t from 3.09 (95% CI, 2.10 to 3.63) to 0.41 (95% CI, 0.18 to 0.66). An improved diagnostic capability created the second peak and increased the number of documented infections. The proportion of documented infections changed from 23% (95% CI, 20% to 26%) to 37% (95% CI, 33% to 41%) when the detection kits were released after January 26, and later up to 73% (95% CI, 64% to 80%) after the diagnostic criteria were improved. \"},\n"," {'paper_id': 'a3dcd6b3f0551e607d0235616109ab8d506f81a2',\n","  'section': 'Abstract',\n","  'text': 'word count: 200 Text word count: 2304 . CC-BY-ND 4.0 International license is made available under a ABSTRACT Vaccine efficacy against susceptibility to infection (VES), regardless of symptoms, is an important endpoint of vaccine trials for pathogens with a high proportion of asymptomatic infection, as such infections may contribute to onward transmission and outcomes such as Congenital Zika Syndrome. However, estimating VES is resource-intensive. We aim to identify methods to accurately estimate VES when limited information is available and resources are constrained. We model an individually randomized vaccine trial by generating a network of individuals and simulating an epidemic. The disease natural history follows a Susceptible, Exposed, Infectious and Symptomatic or Infectious and Asymptomatic, Recovered model. We then use seven approaches to estimate VES, and we also estimate vaccine efficacy against progression to symptoms (VEP). A corrected relative risk and an interval censored Cox model accurately estimate VES and only require serologic testing of participants once, while a Cox model using only symptomatic infections returns biased estimates. Only acquiring serological endpoints in a 10% sample and imputing the remaining infection statuses yields unbiased VES estimates across values of R0 and accurate estimates of VEP for higher values. Identifying resource-preserving methods for accurately estimating VES is important in designing trials for diseases with a high proportion of asymptomatic infection. '},\n"," {'paper_id': 'a3fa8b724a988e6cf50d25645fe687fe50ba3819',\n","  'section': 'Abstract',\n","  'text': 'Background: An ongoing outbreak of mystery pneumonia in Wuhan was caused by coronavirus disease 2019 . The infectious disease has spread globally and become a major threat to public health. We aim to investigate the ultra-high-resolution CT (UHR-CT) findings of imported COVID-19 related pneumonia from the initial diagnosis to early-phase follow-up. Methods: This retrospective study included confirmed cases with early-stage COVID-19 related pneumonia imported from the epicenter. Initial and early-phase follow-up UHR-CT scans (within 5 days) were reviewed for characterizing the radiological findings. The normalized total volumes of ground-glass opacities (GGOs) and consolidations were calculated and compared during the radiological follow-up by artificial-intelligence-based methods. Results: Eleven patients (3 males and 8 females, aged 32-74 years) with confirmed COVID-19 were evaluated. Subpleural GGOs with inter/intralobular septal thickening were typical imaging findings. Other diagnostic CT features included distinct margins (8/11, 73%), pleural retraction or thickening (7/11, 64%), intralesional vasodilatation (6/11, 55%). Normalized volumes of pulmonary GGOs (p=0.003) and consolidations (p=0.003) significantly increased during the CT follow-up. The abnormalities of GGOs with peripleural distribution, consolidated areas, septal thickening, pleural involvement and intralesional vasodilatation on UHR-CT indicate the diagnosis of COVID-19. COVID-19 cases could manifest significantly progressed GGOs and consolidations with increased volume during the early-phase CT follow-up. '},\n"," {'paper_id': 'a42ff48d50aa3a0ebfe840d46ed2204c49955442',\n","  'section': 'Abstract',\n","  'text': 'December 2019, there have been more than 76,000 cases of COVID-19 in China, causing more than 3,000 medical staff infections. Due to COVID-19 spreads quickly, is highly contagious, and can be fatal in severe cases, and there are no specific medicines, it poses a huge threat to the life and health of nurses and has a large impact on their emotional responses and coping strategies. This study conducted an online questionnaire survey from February 1 to 9, 2020 to investigate the current state of emotional responses and coping strategies of nurses and college nursing students in Anhui Province. This study used a modified Brief COPE (Carver, 1997) and a emotional responses scale. The results found that women showed more severe anxiety and fear than men. Participants from cities showed more anxiety and fear than participants from rural, but rural participants showed more sadness than urban participants. The closer COVID-19 is to the participants, the stronger the anxiety and anger. Compared with Nursing college students, nurses have stronger emotional responses and are more willing to use Problem-focused coping. People may have a cycle of \"the more fear, the more problem-focused coping\". And people may \"The more angry, the more emotion-focused coping\", \"the more problem-focused coping, the more anxious, the more angry, the more sadness\". All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': 'a44334e676e43c1889d282b0f0a8365f0f1e0c52',\n","  'section': 'Abstract',\n","  'text': 'Background: An outbreak of the novel coronavirus SARS-CoV-2 has led to 46,997 confirmed cases as of 13 th February 2020. Understanding the early transmission dynamics of the infection and evaluating the effectiveness of control measures is crucial for assessing the potential for sustained transmission to occur in new areas. We combined a stochastic transmission model with data on cases of novel coronavirus disease in Wuhan and international cases that originated in Wuhan to estimate how transmission had varied over time during January and February 2020. Based on these estimates, we then calculated the probability that newly introduced cases might generate outbreaks in other areas. We estimated that the median daily reproduction number, R t , declined from 2.35 (95% CI: 1.15-4.77) one week before travel restrictions were introduced on 23 rd January to 1.05 (95% CI: 0.413-2.39) one week after. Based on our estimates of R t ,we calculated that in locations with similar transmission potential as Wuhan in early January, once there are at least four independently introduced cases, there is a more than 50% chance the infection will establish within that population. '},\n"," {'paper_id': 'a4d1861ceb0adf8dbdb8aed9465a3487fded5568',\n","  'section': 'Abstract',\n","  'text': 'Binge eating (BE) is a heritable symptom of eating disorders with an unknown genetic etiology. 32 Rodent models for BE of palatable food permit the study of genetic and biological mechanisms. 33 We previously used genetic mapping and transcriptome analysis to map a coding mutation in 34 Cyfip2 associated with increased BE in the BE-prone C57BL/6NJ substrain compared to the BE-35 resistant C57BL/6J substrain. The increase in BE in C57BL/6NJ mice was associated with a 36 decrease in transcription of genes enriched for myelination in the striatum. Here, we tested the 37 hypothesis that decreasing myelin levels with the demyelinating agent cuprizone would enhance 38 BE. Mice were treated with a 0.3% cuprizone home cage diet for two weeks. Following a three-39 week recovery period, mice were trained for BE in an intermittent, limited access procedure. 40 Cuprizone induced similar weight loss in both substrains and sexes that recovered within 48 h 41 after removal of the cuprizone diet. Surprisingly, cuprizone reduced BE in male but not female 42 C57BL/6NJ mice while having no effect in C57BL/6J mice. Cuprizone also reduced myelin basic 43 protein (MBP) at seven weeks post-cuprizone removal while having no effect on myelin-44 associated glycoprotein (MAG) at this time point. C57BL/6N mice also showed less MBP than 45 C57BL/6J mice. There were no statistical interactions of Treatment with Sex on MBP levels, 46 indicating that differences in MBP are unlikely to account for sex differences in BE. To summarize, 47 cuprizone induced an unexpected, male-specific reduction in BE which could indicate sex-specific 48 biological mechanisms that depend on genetic background. 49 50 KEY WORDS: binge eating disorder, demyelination, sex differences, reduced complexity cross, 51 QTL, sex as a biological variable 52 '},\n"," {'paper_id': 'a5039336577ac0fed15b784adb912df2a6a143f3',\n","  'section': 'Abstract',\n","  'text': \"Objective: To analyze the clinical characteristics of patients with novel coronavirus pneumonia in Kunming City, and to study the correlation between nutritional status and immune function. Methods: Clinical data of 36 patients with novel coronavirus pneumonia in isolation area of Kunming Third People's Hospital from January 31 to February 15, 2020 were collected, and the basic situation, clinical characteristics, laboratory examination and CT imaging characteristics were analyzed. Serum albumin (ALB), prealbumin (PAB), hypersensitive c-reactive protein (hs-crp), CD3T cells, CD4T cells, CD8T cells and normal control group were analyzed. A simple linear regression analysis of the relationship between proalbumin and T cell subpopulation counts in the blood of patients. (1) The patients with new coronavirus pneumonia in Kunming were mainly of common type. (2) 50% of the patients' first symptoms were fever and cough; (3) The total number of white blood cells in peripheral blood was normal or decreased in 23 cases (79%), and the lymphocyte count decreased in 5 cases (13.89%), without anemia. Hypersensitive c-reactive protein increased in 19 (52.78%) cases, and procalcitonin increased in 1 case. Albumin decreased in 5 cases (13.89%), proalbumin decreased in 15 cases (41.67%), alanine transaminase increased slightly in 4 cases (11.11%), alanine transaminase increased slightly in 4 cases (11.11%), total bilirubin increased slightly in 11 cases (30.56%), and renal function and blood coagulation were normal. Absolute value of CD3 + T cells is with a decrease in 21 cases (58.3%), CD4 + T in 28 cases (77.8%), CD8 + T in 17 cases (47.2%), and CD4 + / CD8 + inverse in 6 cases (16.7%). (4) The prealbumin, group were significantly lower than those in the normal control group, and the hypersensitive c-reactive protein was higher than that in the normal control group. (5) The levels of PAB in the serum of the patients were linearly correlated with hs-crp, CD3 T cells, CD4 T cells and CD8 T cells, and the correlation coefficients were -0.474, 0.558, 0.467 and 0.613, respectively, showing statistical differences. Conclusion: The clinical characteristics of the novel coronavirus pneumonia in Kunming are different from those in Wuhan. The changes of serum proalbumin and T cell subsets are relatively obvious. Changes in serum proalbumin may contribute to the early warning of novel coronavirus pneumonia. The nutritional status of patients with common and mild pneumonia should be considered. \"},\n"," {'paper_id': 'a555fbeeffc44dfdb5087c83395f0f743654cbba',\n","  'section': 'Abstract',\n","  'text': 'Most fungal viruses have been identified in plant pathogens, whereas the presence of viral 33 particles in human pathogenic fungi is less well studied. In the present study, we observed 34 extrachromosomal double-stranded RNA (dsRNA) segments in various clinical isolates of 35 Malassezia species. Malassezia is the most dominant fungal genus on the human skin surface, 36 and species in this group are considered to be etiological factors in various skin diseases 37 including dandruff, seborrheic dermatitis, and atopic dermatitis. We identified novel dsRNA 38 segments and our sequencing results revealed that the virus, named MrV40, belongs to the 39 Totiviridae family and contains an additional satellite dsRNA segment encoding a novel 40 protein. The transcriptome of virus-infected M. restricta cells was compared to that of virus-41 free cells, and the results showed that transcripts involved in ribosomal biosynthesis were 42 down regulated and those involved in energy production and programmed cell death were 43 increased in abundance. Moreover, transmission electron microscopy revealed significantly 44 larger vacuoles for virus-infected M. restricta cells, indicating that MrV40 infection 45 dramatically altered M. restricta physiology. Our analysis also revealed that a viral nucleic 46 acid from MrV40 induces a TLR3-mediated inflammatory immune response in bone marrow-47 derived dendritic cells (BMDCs) and this result suggests that a viral element contributes to 48 the pathogenesis of Malassezia. Importance 51 Malassezia is the most dominant fungal genus on the human skin surface and is associated 52 with various skin diseases including dandruff and seborrheic dermatitis. Among Malassezia 53 species, M. restricta is the most widely observed species on the human skin. In the current 54 study, we identified a novel dsRNA virus, named MrV40, in M. restricta and characterized 55 the sequences and structure of the viral genome along with an independent satellite dsRNA 56 viral segment. Moreover, we found altered expression of genes involved in ribosomal 57 synthesis and programmed cell death, indicating that virus infection altered the physiology of 58 the fungal host cells. Our data also showed that the viral nucleic acid from MrV40 induces a 59 TLR3-mediated inflammatory immune response in bone marrow-derived dendritic cells 60 (BMDCs), indicating that a viral element likely contributes to the pathogenesis of Malassezia. 61 This is the first study to identify and characterize a novel mycovirus in Malassezia. : bioRxiv preprint 65 mushrooms (1). Fungal viruses, also known as mycoviruses, possess different forms of viral 66 genomes including double-stranded RNA (dsRNA), single-stranded RNA (ssRNA), and 67 single-stranded DNA (ssDNA). It is estimated that 30-80% of all fungal species, mainly 68 endophytic fungi, are infected with viruses. Unlike viruses that infect other organisms, the 69 transmission of fungal virus occurs vertically by cell division or horizontally via mating or 70 hyphal anastomosis, with no extracellular phase of the virus life cycle. dsRNA segments have 71 predominantly been found for fungal viruses and, taxonomically, the fungal dsRNA viruses 72 are classified into seven families: Chrysoviridae, Endornaviridae, Megabirnaviridae, 73 Quadriviridae, Partitiviridae, Reoviridae, and Totiviridae (2). 74 The model fungus Saccharomyces cerevisiae also carries a dsRNA virus that belongs to the 75 Totiviridae family and is known as the L-A virus. A unique feature of fungal viruses of the 76 Totiviridae family is their capability to produce the killer toxin that lyses susceptible neighbor 77 strains, whereas the virus-containing strain (also known as a killer strain) is immune to the 78 toxin. Studies of how the virus produces killer toxins in S. cerevisiae showed that killer toxins 79 are encoded by a satellite dsRNA segment, known as the M satellite, within the L-A virus. To 80 date, four different killer toxins, K1, K2, K28, and Klus have been described (3-6). The S. 81 cerevisiae L-A virus forms icosahedral particles with a diameter of approximately 39 nm (7). 82 The virus possesses a non-segmented 4.6-kb dsRNA genome consisting of two open reading 83 frames (ORFs), gag and pol, which overlap by 130 base pairs (bp) (8). gag encodes a major 84 76-kDa capsid protein (CP), and a 180-kDa minor protein species is encoded as a Gag-Pol : bioRxiv preprint 6 fusion protein possesses viral RNA-dependent RNA polymerase (RDRP) activity (8). The 87 ribosomal frameshift is an interesting feature in a compact viral genome and has been 88 commonly found in various viral genomes as a mechanism to allow viruses to express 89 overlapping ORFs. Studies of S. cerevisiae L-A virus revealed that the mechanism of 90 frameshifting is based on the sequence structures including a canonical slippery heptamer, 5′-91 X XXY YYZ-3′ (X= A, U or G; Y=A or U; Z=A, U, or C) and RNA pseudoknot (11). 92 Fungal viruses have also been considered as biocontrol agents in the field of agriculture. For 93 example, a virus causes hypovirulence in the chestnut blight fungus Cryphonectria parasitica 94 (12, 13), and a virus mediates the biocontrol of other phytopathogenic fungi such as 95 Helminthosporium victoriae, Sclerotinia sclerotiorum, and Botrytis cinerea (14). 96 Although viral infections in fungal cells are widespread, the interactions between the fungal 97 virus and its host are not well understood. One of the most studied host defense mechanisms 98 is RNA silencing. Several studies have shown that RNA silencing functions as an antiviral 99 defense mechanism against C. parasitica in fungi. Disruption of the dicer pathway in C. 100 parasitica increases the susceptibility to virus infections (15), and p29 was identified as a 101 suppressor that inhibits expression of the genes required for RNA silencing-mediated viral 102 defense in the fungus (16). Similarly, conserved RNA silencing-mediated antiviral defense 103 systems have been identified in Aspergillus nidulans, Rosellinia necatrix, and Fusarium 104 graminearum (17-19). 105 Malassezia is the most dominant fungal genus on the human skin surface and is considered as 106 an etiological factor in various skin diseases including dandruff, seborrheic dermatitis, and 107 atopic dermatitis (20-23). Eighteen Malassezia species have been identified; among them, M. 108 restricta is the most abundant on the human skin (20). Recent studies showed an increased : bioRxiv preprint 7 burden of M. restricta on the scalp of patients with dandruff, indicating an association 110 between dandruff and the fungus although its role as a pathogenic organism is still unclear, 111 and the host susceptibility should be taken into consideration (21, 24-27). Most fungal viruses 112 are found in plant pathogenic fungi, whereas few examples of viral particles have been 113 identified in human pathogenic fungi such as Candida albicans (28). In the present study, we 114 observed extrachromosomal dsRNA segments in various M. restricta clinical isolates which 115 represented a novel viral genome and its satellite. Sequence analysis revealed that the virus 116 belongs to the Totiviridae family and that the additional satellite dsRNA segment encodes a 117 novel protein. The interactions between the viral elements and the fungal host, and the impact 118 of the virus on fungal interactions with immune cells were evaluated. 119 120 Results 121 Identification of extrachromosomal dsRNA segments in Malassezia 122 Extrachromosomal nucleic acid bands were observed in total nucleic acid extracts of the M. 123 restricta strains isolated in our recent study (29). Among the strains, M. restricta KCTC 124 27540 was used to identify extrachromosomal segments. Total nucleic acids were extracted 125 from the strain and digested with DNase I, RNase A, and RNase T1. The extrachromosomal 126 segments and ribosomal RNA were resistant to DNase I, indicating that they were neither 127 ssDNA nor dsDNA. RNase A degraded all nucleic acids except for genomic DNA, whereas 128 RNase T1, which catalyzes the degradation of ssRNA, removed ribosomal RNA only (30). 129 These results suggested that the extrachromosomal segments correspond to dsRNA, and that 130 M. restricta KCTC 27540 possesses two separate extrachromosomal segments estimated by 131 agarose gel electrophoresis to be approximately 4.5 and 1.5 kb (Fig. 1A). : bioRxiv preprint 8 To confirm whether the extrachromosomal segments observed in other M. restricta clinical 133 isolates were also dsRNA, total nucleic acid extracts from the strains other than M. restricta 134 KCTC 27540 were treated with DNAse I and RNase T1. The extrachromosomal segments 135 remained unaffected after enzyme treatment, indicating that they are also dsRNA, as in M. 136 restricta KCTC 27540 (Fig. 1B). Moreover, other Malassezia species including M. globosa, 137 M. pachydermatis, and M sympodialis showed similar extrachromosomal dsRNA segments 138 suggesting that these segments are common characteristics of Malassezia species (Fig. 1C). 139 Agarose gel electrophoresis revealed extrachromosomal segments composed of at least two 140 separate dsRNA fragments except for M. restricta KCTC 27543, which showed a single 141 dsRNA fragment. Additionally, the large fragments of dsRNA showed similar sizes (~5.0 kb), 142 whereas the small dsRNA segments varied in size in different strains ( Fig. 1B and 1C) . We 143 hypothesized that the dsRNA segments from Malassezia strains represent the dsRNA 144 elements of mycoviruses, which are prevalent in all major fungal taxa (2). Sucrose gradient ultracentrifugation was conducted to purify virus particles to confirm that 146 the dsRNA segments in the Malassezia strains were indeed viral elements. The separated 147 nucleic acids and proteins in each fraction were analyzed. Two dsRNA fragments (~5.0 and 148 ~1.7 kb) were clearly visible in fractions 1-6 following agarose gel electrophoresis ( Fig. 2A) . Moreover, the results of sodium dodecyl sulfate-poly acrylamide gel electrophoresis (SDS-150 PAGE) showed that fractions 3-6 contained protein bands with an estimated molecular 151 weight of ~77 kDa (Fig. 2B ). This molecular weight is similar to that of the known capsid 152 protein of the S. cerevisiae mycovirus (9, 31, 32). Fractions 3-6 were subsequently evaluated 153 by microscopy to visualize mycovirus particles in M. restricta. Transmission electron 154 microscopy (TEM) images showed virus-like particles with an isometric shape and a : bioRxiv preprint 157 the viral particle as MrV40 (M. restricta KCTC 27540 Mycovirus). The large and small 158 dsRNA viral fragments were named as MrV40L and MrV40S, respectively. In addition to 159 evaluating images of the purified virus particle, we examined the morphology of M. restricta 160 KCTC27540 cells containing the mycovirus for comparison with virus-free cells of M. 161 restricta KCTC 27527 by TEM. The results showed that, in general, the size of vacuoles in 162 the virus-containing strain was significantly larger than those in the virus-free strain, 163 suggesting that the virus influences vacuole size in M. restricta (Fig. 3). 164 165 Determination of dsRNA sequence of MrV40L 166 The complete sequence of MrV40L was determined by a combination of the Illumina MiSeq 167 technique and the Sanger sequencing method using purified viral dsRNA. The length of the 168 complete assembled sequence of MrV40L was 4,606 bp, and two overlapping open reading 169 frames (ORFs), designated as ORF1 and ORF2, were identified (Fig. 4A). ORF1 corresponds 170 to the region from nucleotides (nt) 28 to 2,097 and encodes a polypeptide of 689 amino acids 171 with a molecular weight of 77 kDa. ORF2 corresponds to the region from nt 1,949 to 4,538 172 and encodes a polypeptide of 862 amino acids with a molecular weight of 98 kDa. The results 173 of BLAST analysis showed that the protein sequences of ORF1 and ORF2 were highly 174 similar to the capsid protein (CP; the Pfam families of LA-virus_coat, PF09220) and viral 175 RNA-directed RNA polymerase (RDRP; the Pfam family of RDRP_4, PF02123) of S. 176 segobiensis virus L belonging to the genus Totivirus (Totiviridae family) with 53% 177 (YP_009507830.1) and 52% (YP_009507831.1) identities, respectively (33). Eight conserved : bioRxiv preprint 189 genomes similar to MrV40Ls, while the remaining samples had dissimilar viral genomes (Fig. 190 5B). 191 Furthermore, we performed phylogenetic classification of the same viruses found in other 192 clinical M. restricta isolates by multilocus sequence typing of the 1,075-bp region, 638 bp of 193 ORF1, and 437 bp of ORF2, corresponding to gag and pol, respectively, within the viral 194 genomes. The results revealed that the viruses were classified into three clades: clade Ⅰ 195 (MrV12L, MrV16L, MrV40L, MrV79L, MrV80L, and MrV82L), clade Ⅱ (MrV18L, 196 MrV43L, MrV83L, and MrV50L), and clade Ⅲ (MrV24L) (Fig. 5C). Additionally, the 197 sequences of the 1,075-bp region of MrV40L, MrV79L, and MrV82L in the clade I were 100% 198 identical, suggesting that they originated from the same lineage. : bioRxiv preprint To determine the sequence of MrV40S, the dsRNA segments of MrV40S were extracted from 202 agarose gels and then subjected to cDNA cloning and sequencing (See Materials and 203 Methods). Using the partial sequences obtained from cDNA clones and the sequences 204 obtained from repeated 5′ rapid amplification of cDNA ends (RACE), we successfully 205 determined the complete sequence of MrV40S. The sequence of MrV40S was 1,355 bp, and 206 a single ORF was identified in the 3′ region (from nt 773 to 1,097) which encoded a 207 polypeptide of 124 amino acids with a molecular weight of 15.6 kDa. The ORF was 208 designated as ORF3. Although we obtained the complete sequence of MrV40S, no 209 homologous protein sequence was identified by BLAST analysis using all currently available 210 databases. 211 Mycoviruses belonging to totivirus, particularly the S. cerevisiae LA-virus, often possess a 212 satellite dsRNA segment known as M dsRNA, which is responsible for producing a killer 213 toxin that excludes neighboring yeast cells. Because MrV40L resembles M dsRNA, we 214 predicted that MrV40S produces a protein that inhibits other Malassezia strains and/or other 215 fungal and/or bacterial cells residing on the skin. To test the toxin-like activity of the protein 216 produced from MrV40S, ORF3 was cloned and the protein was heterologously expressed in 217 Escherichia coli and purified (Fig. 6A). The activity of the purified protein was evaluated 218 against several pathogenic fungi and bacteria including M. restricta, C. albicans, 219 Cryptococcus neoformans, E. coli, and Staphylococcus aureus. Unexpectedly, the purified 220 protein displayed no growth inhibitory effect on the microbial cells tested (data not shown). Based on these results, we concluded that the novel protein produced from ORF3 likely has 222 no toxin-like activity against the microorganisms tested, and further functional studies are 223 required to characterize its function. : bioRxiv preprint Notably, small increases in the production of TNF-α and IFN-α in TLR3 knockout mice in 318 response to the cell extract of virus-infected M. restricta KCTC 27540 were ignored in our 319 results because only one sample among the two virus-infected strains displayed an increase. The change in IL-10 in the same mutant BMDCs was also excluded because the differences 321 were statistically insignificant. Overall, our data suggest that dsRNA in MrV40 triggers an 322 increase in the production of cytokines involved in inflammation and that TLR3 plays a 323 central role in the host response. 324 325 Discussion 326 In the current study, we detected dsRNA virus in several clinical isolates of Malassezia 327 species. Among them, MrV40 identified in M. restricta KCTC 27540 was selected, and its 328 genome structure, effects on host gene expression, and influence on the mammalian immune 329 response were evaluated. Our data showed that MrV40 consists of two RNA segments, which 330 we named as MrV40L and MrV40S. The results of genome sequence analysis suggested that 331 these segments were 4,606 and 1,355 bp, respectively, and belong to the totivirus. Typically, 332 the genomes of the viruses belonging to the genus Totivirus consist of non-segmented dsRNA 333 with sizes between 4.6 and 7.0 kb, and contain two ORFs, gag and pol. Studies have 334 specifically examined the genome structure of Totivirus because of the overlapping nature of 335 the two ORFs, where a -1 frameshift occurs, resulting translation of the fusion protein (2). The overlapping ORFs and frameshift were frequently observed in a compact viral genome to 337 translate proteins and were found in several dsRNA and ssRNA viruses; these ORFs allow 338 ribosomes to translate CP and RDRP continuously with a missing CP termination codon (11, : bioRxiv preprint 17 the pseudoknot structure of the mRNA for efficient slipping via a slippery site (57). For 341 example, in the genome of the S. cerevisiae L-A virus, the 5′ ORF (gag) encodes a 76-kDa 342 CP and the 3′ ORF (pol) encodes an RDRP which is expressed as a 180-kDa CP-RDRP 343 fusion protein generated by a -1 ribosomal frameshift (8, 11). In our study, MrV40L (the 344 major dsRNA segment in MrV40) contained two overlapping ORFs, ORF1 and ORF2. We 345 identified a putative slippery site heptamer, 5′-GGGTTTT-3′, at the region from nt 1, 968 to 346 1,974 for the -1 ribosomal frameshift, which may be associated with production of the fusion 347 protein of 170 kDa in MrV40L. A previous study suggested that in the S. cerevisiae L-A virus, 348 the rate of ribosomal frameshifs was approximately 1.8%, giving 120 CP and 2 CP-RDRP 349 fusion protein molecules per virus particle (11). Considering the low efficiency of producing 350 the fusion proteins by ribosomal frameshifting, we expected to observe a significantly lower 351 translation rate of the fusion protein; indeed, the putative 170-kDa band was not detected by 352 SDS-PAGE. In addition to MrV40L, we determined the sequence of the satellite dsRNA 353 segment MrV40S and found that it consists of 1,355 nt containing a single ORF, ORF3, 354 producing a novel 15.6-kDa protein. As observed for other totiviruses, the possible toxin-like 355 activity of the protein was investigated in our study, but no growth inhibitory activity against 356 several bacteria and fungi was observed. 357 It has been estimated that 30-80% of fungal species in nature are infected with viruses, and a 358 fungal host normally shows no specific symptoms upon infection (14). However, several 359 genes were shown to be required for maintaining and propagating viruses in the host fungal 360 cells. In S. cerevisiae, numerous chromosomal genes are known to be involved in viral 361 propagation and expression of the viral killer toxin (35). Furthermore, several genes are 362 known to be responsible for maintaining the L-A virus and M dsRNA in S. cerevisiae. The 363 MAK genes are required for the propagation and maintenance of the L-A virus and M dsRNA : bioRxiv preprint 18 in S. cerevisiae (58). Among the MAK genes, MAK3, MAK10, and PET18 are required for the 365 maintenance of both L-A virus and M dsRNA, whereas all other MAK genes are responsible 366 only for M dsRNA (37-42). Particularly, MAK3 encodes an N-acetyltransferase and is 367 required for N-acetylation of the coat protein (37, 59). A previous study showed that the coat 368 proteins without acetylation failed to self-assemble, resulting in the loss of all dsRNA viruses 369 (38). MAK10 and PET18 (MAK30+MAK31) encode a non-catalytic subunit of N-terminal 370 acetyltransferase and a protein of unknown function, respectively. Mutant strains lacking 371 each gene contained unstable viral particles, indicating that the genes are involved in the 372 structural stability of LA-virus and M dsRNA (39). MAK1 (MAK17, TOP1) encodes DNA 373 topoisomerase I, and other MAK genes including MAK2, MAK5, MAK11, MAK16, MAK21, 374 MAK7 (RPL4A), MAK8 (TCM1), and MAK18 (RPL41B) are related to 60S ribosomal subunit 375 assembly (42, 60, 61). All mutant stains lacking the above genes showed decreased levels of 376 free 60S ribosomal subunits and the inability to maintain M dsRNA, suggesting that stable 377 propagation of the satellite dsRNA depends on 60S ribosome synthesis (41, 42). In addition 378 to the MAK genes, the SKI gene family has been shown to be involved in the maintenance 379 and propagation of virus in S. cerevisiae. SKI1 (XRN1) encoding a 5′-3′ exonuclease is 380 involved in the degradation of uncapped mRNA including viral mRNA, and SKI2, SKI3, 381 SKI6, SKI7, and SKI8 block the translation of viral mRNAs (62-64). In the current study, homologs of most MAK and SKI genes were identified in M. restricta. The results of the transcriptome analysis suggested that most MAK genes were downregulated, 384 which may in turn reduce ribosome synthesis in the M. restricta strain containing MrV40. In 385 contrast, no SKI homolog showed significantly altered transcript levels in the M. restricta 386 strain harboring MrV40. Moreover, we found increased expression of genes involved in : bioRxiv preprint 19 energy metabolism and programmed cell death in the M. restricta strain containing the virus. 388 Overall, our transcriptome analysis revealed that the expression of only a few genes was 389 altered upon virus infection in M. restricta. Although the differential expression patterns 390 differed from that of M. restricta, S. cerevisiae also displayed relatively small changes in 391 fungal host gene expression upon virus infection, possibly because of co-adaptation of the 392 virus within the fungal host (65). Maintenance and propagation of virus within the fungal 393 host may be involved in the post-transcriptional mechanism and may contribute to the 394 minimal changes in host gene expression. Notably, the possibility that an RNA silencing 395 pathway in M. restricta cells influences virus maintenance was excluded because of the 396 absence of homologous genes required for the pathway in the genome of the fungus. 397 In addition to transcriptome analysis, we directly investigated whether the virus influences 398 the cellular morphology of M. restricta and the structures of its intracellular organelles by 399 '},\n"," {'paper_id': 'a5944e859b017fd4504ea042daaec613f3fba167',\n","  'section': 'Abstract',\n","  'text': '32 '},\n"," {'paper_id': 'a59906b732bf4a489e282c3e4f499d4166c622e7',\n","  'section': 'Abstract',\n","  'text': 'Background 3 In silico bacterial, viral, and human truth datasets were generated to evaluate available metagenomics 4 algorithms. Sequenced datasets include background organisms, creating ambiguity in the true 5 source organism for each read. Bacterial and viral datasets were created with even and staggered 6 coverage to evaluate organism identification, read mapping, and gene identification capabilities of 7 available algorithms. These truth datasets are provided as a resource for the development and 8 refinement of metagenomic algorithms. Algorithm performance on these truth datasets can inform 9 decision makers on strengths and weaknesses of available algorithms and how the results may be best 10 leveraged for bacterial and viral organism identification and characterization. 11 12 Source organisms were selected to mirror communities described in the Human Microbiome Project as 13 well as the emerging pathogens listed by the National Institute of Allergy and Infectious Diseases. The 14 six in silico datasets were used to evaluate the performance of six leading metagenomics algorithms: 15 MetaScope, Kraken, LMAT, MetaPhlAn, MetaCV, and MetaPhyler. 16 Results Algorithms were evaluated on runtime, true positive organisms identified to the genus and species 19 levels, false positive organisms identified to genus and species level, read mapping, relative abundance 20 estimation, and gene calling. No algorithm out performed the others in all categories, and the 21 algorithm or algorithms of choice strongly depends on analysis goals. MetaPhlAn excels for bacteria 22 and LMAT for viruses. The algorithms were ranked by overall performance using a normalized 23 weighted sum of the above metrics, and MetaScope emerged as the overall winner, followed by Kraken 24 and LMAT. 25 Conclusions 27 Simulated FASTQ datasets with well--characterized truth data about microbial community composition 28 reveal numerous insights about the relative strengths and weaknesses of the metagenomics algorithms 29 evaluated. The simulated datasets are available to download from the Sequence Read Archive 30 (SRP062063). 31 32 '},\n"," {'paper_id': 'a599e1921b5ebf166b6e33daa58085198d59b1ed',\n","  'section': 'Abstract',\n","  'text': 'Background: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) broke out in Wuhan, Hubei, China. This study sought to elucidate a novel predictor of disease severity in patients with coronavirus disease-19 (COVID-19) cased by SARS-CoV-2. Hospital of Wuhan, China. Clinical features, chronic comorbidities, demographic data, and laboratory and radiological data were reviewed. The outcomes of patients with severe pneumonia and those with non-severe pneumonia were compared using the Statistical Package for the Social Sciences (IBM Corp., Armonk, NY, USA) to explore clinical characteristics and risk factors. The receiver operating characteristic curve was used to screen optimal predictors from the risk factors and the predictive power was verified by internal validation. : A total of 377 patients diagnosed with COVID-19 were enrolled in this study, including 117 with severe pneumonia and 260 with non-severe pneumonia. The independent risk factors for severe pneumonia were age [odds ratio (OR): 1.059, 95% confidence interval (CI): 1.036-1.082; p < 0.001], N/L (OR: 1.322, 95% CI: 1.180-1.481; p < 0.001), CRP (OR: 1.231, 95% CI: 1.129-1.341; p = 0.002), and D-dimer (OR: 1.059, 95% CI: 1.013-1.107; p = 0.011). We identified a product of N/L*CRP*D-dimer as having an important predictive value for the severity of COVID-19. The cutoff value was 5.32. The negative predictive value of less than 5.32 for the N/L*CRP*D-dimer was 93.75%, while the positive predictive value was 46.03% in the test sets. The sensitivity and specificity were 89.47% and 67.42%. In the training sets, the negative and positive predictive values were 93.80% and 41.32%, respectively, with a specificity of 70.76% and a sensitivity of 89.87%. Conclusions: A product of N/L*CRP*D-dimer may be an important predictor of disease severity in patients with COVID-19. '},\n"," {'paper_id': 'a59a1e31a7d2edf1be120c4a13a454bed64b1593',\n","  'section': 'Abstract',\n","  'text': 'Emerging and re-emerging RNA viruses occasionally cause epidemics and pandemics worldwide, such as the on-going outbreak of coronavirus SARS-CoV-2. Existing direct-acting antiviral (DAA) drugs cannot be applied immediately to new viruses because of virus-specificity, and the development of new DAA drugs from the beginning is not timely for outbreaks. Thus, host-targeting antiviral (HTA) drugs have many advantages to fight against a broad spectrum of viruses, by blocking the viral replication and overcoming the potential viral mutagenesis simultaneously. Herein, we identified two potent inhibitors of DHODH, S312 and S416, with favorable drug-like and pharmacokinetic profiles, which all showed broad-spectrum antiviral effects against various RNA viruses, including influenza A virus (H1N1, H3N2, H9N2), Zika virus, Ebola virus, and particularly against the recent novel coronavirus SARS-CoV-2. Our results are the first to validate that DHODH is an attractive host target through high antiviral efficacy in vivo and low virus replication in DHODH knocking-out cells. We also proposed the drug combination of DAA and HTA was a promising strategy for anti-virus treatment and proved that S312 showed more advantageous than Oseltamivir to treat advanced influenza diseases in severely infected animals. Notably, S416 is reported to be the most potent inhibitor with an EC50 of 17nM and SI value >5882 in SARS-CoV-2-infected cells so far. This work demonstrates that both our self-designed candidates and old drugs (Leflunomide/Teriflunomide) with dual actions of antiviral and immuno-repression may have clinical potentials not only to influenza but also to '},\n"," {'paper_id': 'a625e5d6fc8956c231c500a40e887e248a69de61',\n","  'section': 'Abstract',\n","  'text': \"A recently developing pneumonia caused by SARS-CoV-2 was originated in Wuhan, China, and has quickly spread across the world. We reported the clinical characteristics of 82 death cases with COVID-19 in a single center. Clinical data on 82 death cases laboratory-confirmed as SARS-CoV-2 infection were obtained from a Wuhan local hospital's electronic medical records according to previously designed standardized data collection forms. Results: All patients were local residents of Wuhan, and the great proportion of them were diagnosed as severe illness when admitted. Most of the death cases were male (65.9%). More than half of dead patients were older than 60 years (80.5%) and the median age was 72.5 years. The bulk of death cases had comorbidity (76.8%), including hypertension (56.1%), heart disease (20.7%), diabetes (18.3%), cerebrovascular disease (12.2%), and cancer (7.3%). Respiratory failure remained the leading cause of death (69.5%), following by sepsis syndrome/MOF (28.0%), cardiac failure (14.6%), hemorrhage (6.1%), and renal failure (3.7%). Furthermore, respiratory, cardiac, hemorrhage, hepatic, and renal damage were found in 100%, 89%, 80.5%, 78.0%, and 31.7% of patients, respectively. On the admission, lymphopenia (89.2%), neutrophilia (74.3%), and thrombocytopenia (24.3%) were usually observed. Most patients had a high neutrophil-to-lymphocyte ratio of >5 (94.5%), high systemic immune-inflammation index of >500 (89.2%), increased C-reactive protein level (100%), lactate dehydrogenase (93.2%), and D-dimer (97.1%). A high level of IL-6 (>10 pg/ml) was observed in all detected patients. Median time from initial symptom to death was 15 days , and a significant association between aspartate aminotransferase (p=0.002), alanine All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. : medRxiv preprint aminotransferase (p=0.037) and time from initial symptom to death were interestingly observed. Older males with comorbidities are more likely to develop severe disease, even die from SARS-CoV-2 infection. Respiratory failure is the main cause of COVID-19, but either virus itself or cytokine release storm mediated damage to other organ including cardiac, renal, hepatic, and hemorrhage should be taken seriously as well. \"},\n"," {'paper_id': 'a68fdcd0f4e55fda6918c216bc3a5b37ef45289e',\n","  'section': 'Abstract',\n","  'text': 'Given the high need and the absence of specific antivirals for treatment of COVID-19 (the disease caused by severe acute respiratory syndrome-associated coronavirus-2 [SARS-CoV-2]), human immunodeficiency virus (HIV) protease inhibitors are being considered as therapeutic alternatives. Prezcobix/Rezolsta is a fixed-dose combination of 800 mg of the HIV protease inhibitor darunavir (DRV) and 150 mg cobicistat, a CYP3A4 inhibitor, which is indicated in combination with other antiretroviral agents for the treatment of HIV infection. There are currently no definitive data on the safety and efficacy of DRV/cobicistat for treatment of COVID-19. The in vitro antiviral activity of darunavir against a clinical isolate from a patient infected with SARS-CoV-2 was assessed. DRV showed no activity against SARS-CoV-2 at clinically relevant concentrations (EC 50 >100 μ M). Remdesivir, used as a positive control, showed potent antiviral activity (EC 50 = 0.38 μ M). Overall, the data do not support the use of DRV for treatment of COVID-19. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': 'a6af28a7f0a0d8d1e4d5b06f5ae38412298bbcb4',\n","  'section': 'Abstract',\n","  'text': 'The pandemic potential of the novel coronavirus (nCoV) that emerged in Wuhan, China, during December 2019 is strongly tied to the number and contagiousness of undocumented human infections. Here we present findings from a proactive longitudinal sampling study of acute viral respiratory infections that documents rates of asymptomatic infection and clinical care seeking for seasonal coronavirus. We find that the majority of infections are asymptomatic by most symptom definitions and that only 4% of individuals experiencing a seasonal coronavirus infection episode sought medical care for their symptoms. These numbers indicate that a very high percentage of seasonal coronavirus infections are undocumented and provide a reference for understanding the spread of the emergent nCoV. . CC-BY 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': 'a6b05fa393f9404c906099e179ca476294068394',\n","  'section': 'Abstract',\n","  'text': 'For successful infection, viruses must recognize their respective host cells. A common mechanism of host recognition by viruses is to utilize a portion of the host cell as a receptor. Bacteriophage Sf6, which infects Shigella flexneri, uses lipopolysaccharide as a primary receptor and then requires interaction with a secondary receptor, a role that can be fulfilled by either outer membrane proteins (Omp) A or C. Our previous work showed that specific residues in the loops of OmpA mediate Sf6 infection. To better understand Sf6 interactions with OmpA loop variants, we determined the kinetics of these interactions through the use of biolayer interferometry, an optical biosensing technique that yields data similar to surface plasmon resonance. Here, we successfully tethered whole Sf6 virions, determined the binding constant of Sf6 to OmpA to be 36 nM. Additionally, we showed that Sf6 bound to five variant OmpAs and the resulting kinetic parameters varied only slightly. Based on these data, we propose a model in which Sf6: Omp receptor recognition is not solely based on kinetics, but likely also on the ability of an Omp to induce a conformational change that results in productive infection. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': 'a6b0f6b85b36a9f15b8dd7293ff9c68d7f10f932',\n","  'section': 'Abstract',\n","  'text': 'The coronavirus disease 2019 outbreak is an unprecedented global public health challenge, leading to thousands of deaths every day worldwide. Despite the epidemiological importance, clinical patterns of children with COVID-19 remain unclear. Objective: To describe the clinical, laboratorial and radiological characteristics of children with COVID-19. Data Sources: The Medline database was searched between December 1 st 2019 and March 30 th 2020. Study Selection: Inclusion criteria were: (1) studied patients younger than 18 years old; (2) presented original data from cases of COVID-19 confirmed by reverse-transcription polymerase chain reaction; and (3) contained descriptions of clinical manifestations, laboratory tests or radiological examinations. Data Extraction: Number of cases, gender, age, clinical manifestations, laboratory tests, radiological examinations and outcomes. Results: A total of 34 studies (1,118 cases) were included. From all the cases, 1,111 had their severity classified: 14.3% were asymptomatic, 36.4% were mild, 46.0% were moderate, 2.2% were severe and 1.2% were critical. The most prevalent symptom was fever (16.3%), followed by cough (14.4%), nasal symptoms (3.6%), diarrhea (2.7%) and nausea/vomiting (2.5%). One hundred forty-five (12.9%) children were diagnosed with pneumonia and 43 (3.8%) upper airway infections were reported. Reduced lymphocyte count were reported in 13.1% of cases. Abnormalities on computed tomography was reported in 62.7% of cases. The most prevalent abnormalities reported were ground glass opacities, patchy shadows and consolidations. Only one death was reported. '},\n"," {'paper_id': 'a6b762a20337862a7383ef9e8dbaa882a97bc389',\n","  'section': 'Abstract',\n","  'text': 'Objectives: To report our experiences screening and managing patients with suspected or confirmed novel coronavirus disease using a hospital-specific protocol. Design: Longitudinal cohort study. Setting: A 1,200 bed tertiary care teaching hospital in Chengdu, Sichuan, China. Participants: 802 adults presenting to hospital with concerns of having COVID-19, 1,246 inpatients and 2,531 hospital visitors. Interventions: Screening and management of patients using a hospital-specific protocol, which included fever triage, monitoring visitors and patients, emergency response, personnel training for healthcare team members, health education for patients and family, medical materials management, disinfection and wastes disposal protocols. : Between 23 January and 28 February 2020, 73 people were identified as having fever plus respiratory signs with/without a history of exposure and were tested for the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by our hospital lab using RT PCR. Forty-five of these 73 people were subsequently excluded based on one negative RT PCR result plus positive results to quick screening tests for flu or other respiratory viruses. The remaining 28 people received a second RT PCR test 24 h later. Three people were confirmed positive for COVID-19 based on two consecutive positive RT PCR tests whilst 25 people were excluded based on two consecutive negative tests. The three COVID-19 confirmed cases received non-critical care. There were no new infections of medical staff or new infections of other hospital inpatients. Conclusions: A hospital-specific protocol for screening and management is necessary for reliably identifying suspected or confirmed COVID-19 patients during an outbreak. All three cases were detected as a result of vigilant monitoring of hospital visitors. Whilst screening out-patients presenting to a fever clinic remains important, monitoring visitors must not be overlooked. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': 'a6cba8f9a9f89a08bb87e30a209389c9f1ca8c3d',\n","  'section': 'Abstract',\n","  'text': 'In this paper, ACE2 gene of pigs was cloned and the purified protein was obtained via 12 the prokaryotic expression system. Polyclonal antibody of high titer and sensitivity 13 was obtained using Wastar rats immunization method and is then used to determine of 14 the expression of ACE2 using immunohistochemistry. The sequence of ACE2 in pigs 15 covered 2418 nucleotides and coded 805 amino acid (aa) residues. Sequence 16 homology analysis showed that the ACE2 sequence in pigs is highly conserved 17 among species at the nucleotide and amino acid levels. Genetic evolution analysis 18 revealed that ACE2 gene in pigs has the shortest genetic distance with that in goats 19 while residing in a totally different branch from that in zebra fishes. Analysis of 20 protein structure predicted that ACE2 protein is a transmembrane secreted protein 21 with high hydrophilicity, containing a signal peptide sequences locating between 1aa 22 to 17aa. The ACE2 fusion protein expressed (under the induction with 1.0 mmol/L 23 IPTG for 10 h) was of approximately 100 kDa and mainly existed in inclusion body. Wastar rats immunization showed that the titer of the anti-ACE2 antiserum in rats was '},\n"," {'paper_id': 'a6d2875c8b70a41ea815511417546a0689816964',\n","  'section': 'Abstract',\n","  'text': 'Background '},\n"," {'paper_id': 'a751eee7f538e9e54dbf1b65b0f90cc4579faec8',\n","  'section': 'Abstract',\n","  'text': 'The ongoing COVID-19 pandemic has reached more than 200 countries and territories worldwide. Given the large requirement of SARS-CoV-2 diagnosis and considering that RNA extraction kits are in short supply, we investigated whether two commercial RT-qPCR kits were compatible with direct SARS-CoV-2 detection from nasopharyngeal swab samples. We show that one of the tested kits is fully compatible with direct SARS-CoV-2 detection suggesting that omission of an RNA extraction step should be considered in SARS-CoV-2 diagnosis. Two coronaviruses, SARS-CoV (severe acute respiratory syndrome coronavirus) and MERS-CoV (Middle East respiratory syndrome coronavirus) caused large-scale severe respiratory syndromes in humans in 2002 and 2013, respectively 1, 2 . A recent outbreak of pneumonia in Wuhan, China declared in December 12 th 2019 led to the isolation and identification of a new human coronavirus, 2019-nCoV (SARS-CoV-2), as the responsible agent of this new acute respiratory syndrome 3 . Since, SARS-CoV-2 and the Coronavirus disease 19 (COVID-19) have spread to more than 200 countries and territories worldwide affecting more than 600,000 people (laboratory-confirmed cases) from which around 28,000 have died by COVID-19 4 . As such, the WHO declared COVID-19 outbreak as a pandemic on March 11 th 2020 5 . Given the rapid spread of the SARS-CoV-2 virus and that an important fraction of the infected individuals remains asymptomatic but still spread the virus 6 , rapid and extensive diagnosis together with the isolation of infected individuals remains as one of the most powerful ways to avoid the spread of new COVID-19 cases. There are several RT-qPCR protocols from different countries that are currently approved by the WHO in order to detect SARS-CoV-2 7 . However, one of the limiting steps of such protocols is RNA extraction from clinical samples, which is currently performed by the use of a limited quantity of approved RNA extraction kits. Since the current demand for those kits largely exceeds their offer, it is necessary to develop alternative strategies allowing a rapid detection of the virus. This becomes more important in middle-and low-income countries, which will not have massive access to kit-based diagnosis 8, 9 . In this sense, it was recently reported that the detection of SARS-CoV-2 by RT-qPCR from a mixed sample could be achieved with non-approved kits but also in the absence of an RNA extraction process 10 . Here, we have extended these preliminary observations by using an RNA extraction-free SARS-CoV-2 detection protocol from nasopharyngeal swab samples from two COVID-19 diagnosed individuals using two commercially available and broadly used RT-qPCR kits. Our data indicate that RNA extraction can be omitted from the protocol at least with one of the tested kits allowing the rapid and reliable detection of SARS-CoV-2 directly from nasopharyngeal swabs samples. '},\n"," {'paper_id': 'a7a4a5568e5b7c921bc616eedf532a393229decb',\n","  'section': 'Abstract',\n","  'text': 'The same nucleotide sequence can encode two protein products in different reading frames. Overlapping gene regions encode higher levels of intrinsic structural disorder (ISD) than non-overlapping genes (39% vs. 25% in our viral dataset). This might be because of the intrinsic properties of the genetic code, because one member per pair was recently born de novo in a process that favors high ISD, or because high ISD relieves increased evolutionary constraint imposed by dual-coding. Here we quantify the relative contributions of these three alternative hypotheses. We estimate that the recency of de novo gene birth explains 32% or more of the elevation in ISD in overlapping regions of viral genes. While the two reading frames within a same-strand overlapping gene pair have markedly different ISD tendencies that must be controlled for, their effects cancel out to make no net contribution to ISD. The remaining elevation of ISD in the older members of overlapping gene pairs, presumed due to the need to alleviate evolutionary constraint, was already present prior to the origin of the overlap. Same-strand overlapping gene birth events can occur in two different frames, favoring high ISD either in the ancestral gene or in the novel gene; surprisingly, most de novo gene birth events contained completely within the body of an ancestral gene favor high ISD in the ancestral gene (23 phylogenetically independent events vs. 1). This can be explained by mutation bias favoring the frame with more start codons and fewer stop codons. '},\n"," {'paper_id': 'a7c641688e1032b963ca6efc8b2a96a8310d1623',\n","  'section': 'Abstract',\n","  'text': 'Programmed ribosomal frameshifting is the controlled slippage of the translating ribosome to an 11 alternative frame. This tightly regulated process is widely employed by human viruses such as HIV 12 and SARS coronavirus and is critical for their life cycle and virulence. It is also utilized from yeast to 13 human to implement a feedback control mechanism to regulate polyamine levels. Here, we developed 14 a high-throughput, fluorescence-based approach to assess the frameshifting potential of a sequence. 15 We designed and tested >12,000 sequences based on 15 viral and human frameshifting events, 16 allowing us to elucidate the rules governing ribosomal frameshifting in a systematic way and to 17 discover novel regulatory inputs based on amino acid properties and tRNA availability. We assessed 18 the natural variation in HIV gag-pol frameshifting rates by testing >500 clinical isolates and identified 19 subtype-specific differences as well as associations between viral load in patients and the optimality 20 of gag-pol frameshifting rates. We further devised computational models that accurately predict 21 frameshifting potential (up to auROC=0.93) and frameshifting rates (up to Pearson r=0.81) of novel 22 '},\n"," {'paper_id': 'a7d9440cb3a45994c274d4de41cbaf14343c8a89',\n","  'section': 'Abstract',\n","  'text': 'We have analyzed the COVID19 epidemic data of more than 174 countries (excluding China) in the period between January 22 and March 28, 2020. We found that some countries (such as the US, the UK, and Canada) follow an exponential epidemic growth, while others (like Italy and several other European countries) show a power law like growth. At the same time, regardless of the best fitting law, most countries can be shown to follow a trajectory similar to that of Italy, but with varying degrees of delay. We found that countries with \"younger\" epidemics tend to exhibit more exponential like behavior, while countries that are closer behind Italy tend to follow a power law growth. We hypothesize that there is a universal growth pattern of this infection that starts off as exponential and subsequently becomes more power law like. Although it cannot be excluded that this growth pattern is a consequence of social distancing measures, an alternative explanation is that it is an intrinsic epidemic growth law, dictated by a spatially distributed community structure, where the growth in individual highly mixed communities is exponential but the longer term, local geographical spread (in the absence of global mixing) results in a power-law. This is supported by computer simulations of a metapopulation model that gives rise to predictions about the growth dynamics that are consistent with correlations found in the epidemiological data. Therefore, seeing a deviation from straight exponential growth may not be a consequence of working nonpharmaceutical interventions (except for, perhaps, restricting the air travel). Instead, this is a normal course of raging infection spread. On the practical side, this cautions us against overly optimistic interpretations of the countries epidemic development and emphasizes the need to continue improving the compliance with social distancing behavior recommendations. '},\n"," {'paper_id': 'a7ed0aa9fe3b699189671e0e917d0135a5958210',\n","  'section': 'Abstract',\n","  'text': 'The number of confirmed COVID-19 cases admitted in hospitals is continuously increasing in the Philippines. Frontline health care workers are faced with imminent risks of getting infected. In this study, we formulate a theoretical model to calculate the risk of being infected in health care facilities considering the following factors: the average number of encounters with a suspected COVID-19 patient per hour; interaction time for each encounter; work shift duration or exposure time; crowd density, which may depend on the amount of space available in a given location; and availability and effectiveness of protective gears and facilities provided for the frontline health care workers. Based on the simulation results, we recommend the following: (i) decrease the rate of patient encounter per frontline health care worker, e.g., maximum of three encounters per hour in a 12-hour work shift duration; (ii) decrease the interaction time between the frontline health care worker and the patients, e.g., less than 40 minutes for the whole day; (iii) increase the clean and safe space for social distancing, e.g., maximum of 10% crowd density, and if possible, implement compartmentalization of patients; and/or (iv) provide effective protective gears and facilities, e.g., 95% effective, that the frontline health care workers can use during their shift. Moreover, the formulated model can be used for other similar scenarios, such as identifying infection risk in public transportation, school classroom settings, offices, and mass gatherings. '},\n"," {'paper_id': 'a7f473df655c67cbdf1b5f61c5a7458bc2a9385c',\n","  'section': 'Abstract',\n","  'text': 'The ongoing coronavirus disease pandemic caused by Severe Acute Respiratory Syndrome Coronavirus-2 (SARS-CoV-2) is a global public health concern due to relatively easy person-to-person transmission and the current lack of effective antiviral therapy. However, the exact molecular mechanisms of SARS-CoV-2 pathogenesis remain largely unknown. We exploited an integrated proteomics approach to systematically investigate intra-viral and virus-host interactomes for the identification of unrealized SARS-CoV-2 host targets and participation of cellular proteins in the response to viral infection using peripheral blood mononuclear cells (PBMCs) isolated from COVID-19 patients. Using this approach, we elucidated 251 host proteins targeted by SARS-CoV-2 and more than 200 host proteins that are significantly perturbed in COVID-19 derived PBMCs. From the interactome, we further identified that non-structural protein nsp9 and nsp10 interact with NKRF, a NF-kB repressor, and may precipitate the strong IL-8/IL-6 mediated chemotaxis of neutrophils and overexuberant host inflammatory response observed in COVID-19 patients. Our integrative study not only presents a systematic examination of SARS-CoV-2-induced perturbation of host targets and cellular networks to reflect disease etiology, but also reveals insights into the mechanisms by which SARS-CoV-2 triggers cytokine storms and represents a powerful resource in the quest for therapeutic intervention. '},\n"," {'paper_id': 'a83b9f98ccbd6dd76d47c49cfe5785f9c2ecc09e',\n","  'section': 'Abstract',\n","  'text': \"Background Lassa fever (LF) is increasingly recognized as an important rodent-borne viral hemorrhagic fever presenting a severe public health threat to sub-Saharan West Africa. In 2018, LF caused an unprecedented outbreak in Nigeria, and the situation was worse in 2019. This work aims to study the epidemiological features of outbreaks in different Nigerian regions and quantify the association between reproduction number (R) and local rainfall by using modeling analysis. We quantify the infectivity of LF by the reproduction numbers estimated from four different growth models: the Richards, three-parameter logistic, Gompertz, and Weibull growth models. LF surveillance data are used to fit the growth models and estimate the Rs and epidemic turning points (τ) in different regions at different time periods. Cochran's Q test is further applied to test the spatial heterogeneity of the LF epidemics. A linear random-effect regression model is adopted to quantify the association between R and local rainfall with various lag terms. Our estimated Rs for 2017-18 (1.33 with 95% CI: [1.29, 1.37]) and 2018-19 (1.29 with 95% CI: [1.27, 1.32]) are significantly higher than those for 2016-17 (1.23 with 95% CI: [1.22, 1.24]) . We report spatial heterogeneity in the Rs for outbreaks in different Nigerian regions. For the association between rainfall and R, we find that a one unit (mm) increase in . CC-BY 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/602706 doi: bioRxiv preprint average rainfall over the past 7 months could cause a 0.62% (95% CI: [0.20%, 1.05%]) rise in R. There is significant spatial heterogeneity in the LF epidemics in different Nigerian regions. We report clear evidence of rainfall impacts on LF outbreaks in Nigeria and quantify the impact. \"},\n"," {'paper_id': 'a8573e6963125b8842707d449bf5232185042208',\n","  'section': 'Abstract',\n","  'text': 'The known: COVID-19 has been diagnosed in over 4,000 Australians. Up until mid-March, most were from international travel, but now we are seeing a rise in locally acquired cases. This study uses a simple transmission dynamic model to demonstrate the difference between moderate changes to the reproduction number and forcing the reproduction number below one. Background: Around the world there are examples of both effective control (e.g., South Korea, Japan) and less successful control (e.g., Italy, Spain, United States) of COVID-19 with dramatic differences in the consequent epidemic curves. Models agree that flattening the curve without controlling the epidemic completely is insufficient and will lead to an overwhelmed health service. A recent model, calibrated for the UK and US, demonstrated this starkly. Methods: We used a simple compartmental deterministic model of COVID-19 transmission in Australia, to illustrate the dynamics resulting from shifting or flattening the curve versus completely squashing it. Results: We find that when the reproduction number is close to one, a small decrease in transmission leads to a large reduction in burden (i.e., cases, deaths and hospitalisations), but achieving this early in the epidemic through social distancing interventions also implies that the community will not reach herd immunity. '},\n"," {'paper_id': 'a891d059692c951d1fb0fb36914fe451f973684a',\n","  'section': 'Abstract',\n","  'text': 'Human coronavirus HKU1 (HCoV-HKU1) is associated with respiratory disease and is 36 prevalent worldwide, but in vitro model for virus replication is lacking. Interaction between the 37 coronaviral spike (S) protein and its receptor is the major determinant of virus tissue and host 38 specificity, but virus entry is a complex process requiring a concerted action of multiple cellular 39 elements. Here, we show that KLK13 is required for the infection of the human respiratory 40 epithelium and is sufficient to mediate the entry of HCoV-HKU1 to non-permissive RD cells. 41 We also demonstrated HCoV-HKU1 S protein cleavage by KLK13 in the S1/S2 region, proving 42 that KLK13 is the priming enzyme for this virus. Summarizing, we show for the first time that 43 protease distribution and specificity predetermines the tissue and cell specificity of the virus 44 and may also regulate interspecies transmission. It is also of importance that presented data may 45 be relevant for the emerging coronaviruses, including SARS-CoV-2 and may help to understand 46 the differences in their zoonotic potential. : bioRxiv preprint Coronaviruses are the largest group within the order Nidovirales. Mainly, they cause 49 respiratory and enteric diseases in humans and animals, but some can cause more serious 50 conditions such as hepatitis, peritonitis, or neurological disease. Seven coronaviruses infect 51 humans, four of which (human coronavirus [HCoV]-229E, HCoV-NL63, HCoV-OC43, and 52 HCoV-HKU1) cause relatively mild upper and lower respiratory tract disease and two (SARS-53 CoV and MERS-CoV) are associated with severe, life-threatening respiratory infections and 54 multiorgan failure (1-6). Furthermore, in December 2019 a novel coronavirus SARS-CoV-2 55 emerged in Hubei province, China, causing pneumonia. To date, almost 90,000 cases were 56 identified and 3,000 patients died worldwide. 57 Coronaviral infection is initiated by interaction between the trimeric spike (S) protein 58 and its receptor, which is expressed on the surface of the susceptible cell. A number of adhesion 59 and entry receptors have been described for coronaviruses. For example, HCoV-229E (similar 60 to many other alphacoronaviruses) utilizes aminopeptidase N (APN) as the primary entry port 61 (7). Surprisingly, its cousin HCoV-NL63 shares receptor specificity with the evolutionarily 62 distant SARS-CoV and SARS-CoV-2: all hijack angiotensin-converting enzyme 2 (ACE2) (8-63 11). HCoV-NL63 was also shown to use heparan sulfate as a primary attachment site (12-14). 64 A very different receptor is recognized by MERS-CoV, which binds to dipeptidyl-peptidase 4 65 (DPP4) (9, 15, 16). Another betacoronavirus, HCoV-OC43, binds to N-acetyl-9-O-66 acetylneuraminic acid (17, 18). HCoV-HKU1 remains the great unknown because its cellular 67 receptor has not been identified and all efforts to culture the virus in vitro have failed. 68 HCoV-HKU1 was identified in Hong Kong in 2004. The virus was present in a sample 69 obtained from an elderly patient with severe pneumonia (19). Epidemiological studies show a 70 high prevalence of the pathogen in humans; this is because the majority of children seroconvert and others reported that ex vivo fully differentiated human airway epithelium (HAE) and human 73 alveolar type II cells support the infection (22-25). A thorough study by Huang X et al. 74 demonstrated that HCoV-HKU1 binds to target cells via O-acetylated sialic acids on the cell 75 surface; however, this interaction is not sufficient for the infection. The study also showed that 76 the hemagglutinin-esterase (HE) protein of HCoV-HKU1 exhibits sialate-O-acetylesterase 77 activity and may act as a receptor-destroying enzyme, thereby facilitating the release of viral 78 progeny (26). Bakkers et al. proposed that, in order to adapt to the sialoglycome of the human 79 respiratory tract over the evolutionary timescale, HCoV-HKU1 lost the ability to bind to 80 attachment receptors via the HE protein (27). Recently, Hulswit et al. mapped the virus binding 81 site to O-acetylated sialic acids, demonstrating that the S1 domain A is responsible for binding 82 to the attachment receptor (28). The S protein is the main player during coronavirus entry, and its characteristics 84 determine the host range. Coronaviral S proteins are class I fusion proteins comprising a large 85 N-terminal ectodomain, a hydrophobic trans-membrane region, and a small C-terminal 86 endodomain. The ectodomain is highly glycosylated and is composed of S1 and S2 domains. The globular S1 domain is highly variable and carries the receptor-binding site, whereas the 88 more conserved rod-like S2 domain undergoes structural rearrangement during entry, which 89 brings the cellular and viral membranes into close proximity. Such a structural switch may be 90 triggered by different stimuli, including receptor binding, proteolytic cleavage of the S protein, 91 and/or a reduction in pH. Because different species require different stimuli, coronaviruses enter 92 cells at different subcellular sites. Some coronaviruses fuse at the plasma membrane, whereas 93 others are believed to enter the cell through receptor-mediated endocytosis, followed by fusion 94 deep within the endosomal compartments (29-35). Furthermore, recent reports show that the 95 author/funder. All rights reserved. No reuse allowed without permission. entry portal may vary depending on tissue/cell characteristics. These differences may affect the 96 host range, pathogenicity, and cell/tissue specificity (1). 97 Host proteases prime coronaviral S proteins. For example, trypsin-mediated cleavage in 98 the small intestine is required for entry of porcine epidemic diarrhea virus (36), while a number 99 of coronaviruses from different genera (including HCoV-OC43, HCoV-HKU1, murine 100 hepatitis virus [MHV], MERS-CoV, and infectious bronchitis virus [IBV]) possess a furin 101 cleavage site (37-41). Kam et al. showed that SARS-CoV S protein can be cleaved by plasmin; 102 however, there is almost no biological evidence for its role in vivo (42). Cathepsins may also 103 act as S protein-activating enzymes. Indeed, cathepsin L processes the S proteins of SARS-104 CoV, MERS-CoV, HCoV-229E, and MHV-2 (43-46). However, recent reports show that, in 105 vivo, respiratory coronaviruses may be activated by the TMPRSS2 protease, which enables 106 endocytosis-independent internalization, thereby re-shaping the entry process (45, 47-50). 107 While laboratory strains require priming by cathepsins, S proteins of clinical isolates (e.g., 108 HCoV-229E and HCoV-OC43) undergo TMPRSS2-mediated cleavage at the cell surface, 109 which enables them to fuse with the cell membrane on the cell surface (51). A recent study by 110 Shirato et al. demonstrates that coronaviruses may lose their ability to infect naturally 111 permissive HAE cultures during cell culture adaptation because the S gene evolves and adjusts 112 to the proteolytic landscape of the immortalized cells (51). 113 Here, we identified a protease belonging to the tissue kallikrein (KLK) family as a new 114 player essential for HCoV-HKU1 entry to the target cell. The KLK family comprises 15 closely 115 related serine proteases with trypsin-or chymotrypsin-like specificity. The expression of these 116 enzymes is tightly regulated, and each tissue has its own unique KLK expression profile. These 117 enzymes play a role in a diverse range of processes during embryonic development to adulthood 118 (52-56), and some have been linked to human cancers (57-60). The function of some KLKs 119 remains to be elucidated, but obtained results suggest that protease distribution may be an 120 author/funder. All rights reserved. No reuse allowed without permission. important factor pre-determining the cell and tissue specificity of the virus, but also regulating 121 the interspecies transfers. Further, the data presented herein bring us a step closer to developing a convenient 123 in vitro culture model and possibly identifying the cellular receptor for this virus. It is also of 124 importance that presented data may be relevant for the emerging coronaviruses and may help 125 to understand the differences in their zoonotic potential. 128 First, we asked whether HCoV-HKU1 infection modulates the expression of human 129 KLKs. HAE cultures were infected with HCoV-HKU1 or mock-inoculated. At 120 h post-130 inoculation (p.i.), cells were collected and the expression of mRNAs encoding KLKs was 131 analyzed. We detected the expression of KLK7, KLK8, KLK10, KLK11, and KLK13 in 132 non-infected fully differentiated cultures. However, the pattern in HCoV-HKU1-infected cells 133 was different: we detected an upregulation of KLK7, KLK8, KLK10, KLK11 and KLK13. 134 Further, KLK1, KLK5, KLK6, KLK9, KLK12 and KLK14 were expressed in the infected cells. 135 KLK2, KLK3, and KLK15 were not expressed (Fig. 1). 136 137 KLK13 is essential for HCoV-HKU1 infection 138 S protein priming is a prerequisite for coronavirus entry; therefore, we tested whether 139 KLKs take part in this process by culturing cells in the presence/absence of KLK inhibitors ( 140 Table 1 ) (61). For this, HAE cultures were pre-incubated with each inhibitor (10 µM) or with 141 vehicle control (DMSO) and mock-inoculated or inoculated with the virus (10 6 RNA copies per 142 ml) in the presence of the inhibitor. Apical washes were collected each day for analysis of virus 143 replication. Subsequently, viral RNA was isolated and reverse transcribed (RT), and the HCoV-144 HKU1 yield was determined by quantitative real-time PCR (qPCR). The results showed that 145 HCoV-HKU1 replication was inhibited in the presence of a KLK13 inhibitor; this was not the 146 case for cells treated with DMSO or with inhibitors specific for KLK7 or KLK8 ( Fig. 2A) . 147 Next, we analyzed HCoV-HKU1 replication in the presence of a family-specific KLK 148 inhibitor SPINK6 at a concentration of 10 µg/ml (62, 63) or 100 µM camostat (a broad inhibitor 149 of serine proteases, also a potent inhibitor of KLK13) (34, 64). We noted inhibition of 150 author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': 'a8be6ba53508e988c2c8c153478ff615b0d77687',\n","  'section': 'Abstract',\n","  'text': 'RNA knockdown in vivo carries significant potential for disease modelings and therapies. Despite the emerging approaches of CRISPR/Cas9-mediated permanent knock out of targeted genes, strategies targeting RNA for disruption are advantageous in the treatment of acquired metabolic disorders when permanent modification of the genome DNA is not appropriate, and RNA virus infection diseases when pathogenic DNA is not available (such as SARS-Cov-2 and MERS infections). Recently, Cas13d, a family of RNA-targeting CRISPR effectors, has been shown to accomplish robust down-regulation of cellular RNAs in mammalian cells in vitro. Among the various Cas13d subtypes, CasRx (RfxCas13d) showed the most potent RNA knockdown efficiency in HEK293T cells. However, the RNA-targeting activity of Cas13d still needs to be verified in vivo. In this study, the CasRx system was demonstrated to efficiently and functionally knock down genes related to metabolism functions, including Pten, Pcsk9 and lncLstr, in mouse hepatocytes. CasRx-mediated simultaneous knockdown of multiple genes was also achieved by sgRNA arrays, providing a useful strategy to modulate complex metabolism networks. Moreover, the AAV (adeno-associated virus)-mediated delivery of CasRx and Pcsk9 sgRNAs into mouse liver successfully decreased serum PCSK9, resulting in significant reduction of serum cholesterol levels. Importantly, CasRx-mediated knockdown of Pcsk9 is author/funder. All rights reserved. No reuse allowed without permission. : bioRxiv preprint 3 reversible and Pcsk9 could be repeatedly down-regulated, providing an effective strategy to reversibly modulate metabolic genes. The present work supplies a successful proof-of-concept trial that suggests efficient and regulatory knockdown of target metabolic genes for a designed metabolism modulation in the liver. '},\n"," {'paper_id': 'a8e53c77bc72f467723a7d5b8020875c0e96f8aa',\n","  'section': 'Abstract',\n","  'text': ', China, at the end of 2019, has caused at least infected 11,844 as of Feb 1, 2020. However, there is no specific antiviral treatment or vaccine currently. Very recently report had suggested that novel CoV would use the same cell entry receptor, ACE2, as the SARS-CoV. In this report, we generated a novel recombinant protein by connecting the extracellular domain of human ACE2 to the Fc region of the human immunoglobulin IgG1. An ACE2 mutant with low catalytic activity was also used in the study. The fusion proteins were then characterized. Both fusion proteins has high affinity binding to the receptor-binding domain (RBD) of SARS-CoV and 2019-nCoV and exerted desired pharmacological properties. Moreover, fusion proteins potently neutralized SARS-CoV and 2019-nCoV in vitro. As these fusion proteins exhibit cross-reactivity against coronaviruses, they could have potential applications for diagnosis, prophylaxis, and treatment of 2019-nCoV. author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': 'a8e64e868372614e2c7adb12a9c8a319830a5500',\n","  'section': 'Abstract',\n","  'text': \"Background: The first case of COVID-19 in sub-Saharan Africa (SSA) was reported by Nigeria on February 27, 2020. While case counts in the entire region remain considerably less than those being reported by individual countries in Europe, Asia, and the Americas, SSA countries remain vulnerable to significant COVID morbidity and mortality due to systemic healthcare weaknesses, less financial resources and infrastructure to address the new crisis, and untreated comorbidities. Variation in preparedness and response capacity as well as in data availability has raised concerns about undetected transmission events. Methods: Confirmed cases reported by SSA countries were line-listed to capture epidemiological details related to early transmission events into and within countries. Data were retrieved from publicly available sources, including institutional websites, situation reports, press releases, and social media accounts, with supplementary details obtained from news articles. A data availability score was calculated for each imported case in terms of how many indicators (sex, age, travel history, date of arrival in country, reporting date of confirmation, and how detected) could be identified. We assessed the relationship between time to first importation and overall Global Health Security Index (GHSI) using Cox regression. K-means clustering grouped countries according to healthcare capacity and health and demographic risk factors. Findings: A total of 2417 confirmed cases of COVID-19 were reported by 40 countries in sub-Saharan Africa during the 30 days after the first known introduction to the region. Out of the 876 cases for which information was publicly available, 677 (77.3%) were considered importation events. At the regional level, imported cases tended to be male (67.3%), were a median 43.0 years old (Range: 6 weeks -88 years), and most frequently had recent travel history from Europe (43.3%). The median time to reporting an introduction was 19 days; a country's time to report its first importation was not related to GHSI. Mean data availability scores were lowest for countries that had, on average, the highest case fatality rates, lowest healthcare capacity, and highest probability of premature death due to non-communicable diseases. Countries with systemic, demographic, and pre-existing health vulnerabilities to severe COVID-related morbidity and mortality are less likely to report any cases or are reporting with limited public availability of information. Reporting of information on COVID detection and response efforts, as well as on trends in non-COVID illness and care-seeking behavior, is critical to assessing direct and indirect consequences and capacity needs in resourceconstrained settings. Such assessments aid in the ability to make data-driven decisions about interventions, country priorities, and risk assessment. . CC-BY 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . \"},\n"," {'paper_id': 'a99f02efcd401fc017aab682571c7af17c29b634',\n","  'section': 'Abstract',\n","  'text': 'We model the evolution of the number of individuals that are reported to be sick with COVID-19 in Germany. Our theoretical framework builds on a continuous time Markov chain with four states: healthy without infection, sick, healthy after recovery or after infection but without symptoms and dead. Our quantitative solution matches the number of sick individuals up to the most recent observation and ends with a share of sick individuals following from infection rates and sickness probabilities. We employ this framework to study inter alia the expected peak of the number of sick individuals in a scenario without public regulation of social contacts. We also study the e¤ects of public regulations. For all scenarios we report the expected end of the CoV-2 epidemic. We have four general …ndings: First, current epidemiological thinking implies that the long-run e¤ects of the epidemic only depend on the aggregate long-run infection rate and on the individual risk to turn sick after an infection. Any measures by individuals and the public therefore only in ‡uence the dynamics of spread of CoV-2. Second, predictions about the duration and level of the epidemic must strongly distinguish between the o¢ cially reported numbers (Robert Koch Institut, RKI) and actual numbers of sick individuals. Third, given the current (scarce) medical knowledge about long-run infection rate and individual risks to turn sick, any prediction on the length (duration in months) and strength (e.g. maximum numbers of sick individuals on a given day) is subject to a lot of uncertainty. Our predictions therefore o¤er robustness analyses that provide ranges on how long the epidemic will last and how strong it will be. Fourth, public interventions that are already in place and that are being discussed can lead to more and less severe outcomes of the epidemic. If an intervention takes place too early, the epidemic can actually be stronger than with an intervention that starts later. Interventions should therefore be contingent on current infection rates in regions or countries. of the modelling initiative of the Deutsche Gesellschaft für Epidemiologie for comments and discussions. We are deeply indebted to Damir Stijepic for the initial numerical solutions and the initial calibration. Our biggest debt of gratitude is owed to Matthias Birkner who has been advising and teaching us on stochastic models for many years. '},\n"," {'paper_id': 'a9a978c056422f979395ce56c96738d33b104a3e',\n","  'section': 'Abstract',\n","  'text': 'The flavivirus Zika virus (ZIKV) activates ribonuclease L (RNase L) catalytic antiviral 26 function during infection, yet deletion of RNase L decreases ZIKV production, suggesting 27 a proviral role of RNase L. In this study, we reveal that latent RNase L supports ZIKV 28 replication factory (RF) assembly. Deletion of RNase L induced broader cellular 29 distribution of ZIKV dsRNA and NS3 compared with densely concentrated RFs detected 30 in WT cells. An inactive form of RNase L was sufficient to contain ZIKV genome and 31 dsRNA within a smaller area, which increased levels of viral RNA within RFs as well as 32 infectious ZIKV released from the cell. We used a microtubule stabilization drug to 33 demonstrate that RNase L deletion impaired the cytoskeleton rearrangements that are 34 required for proper generation of RFs. During infection with dengue or West Nile Kunjin 35 viruses, RNase L decreased virus production, suggesting that RNase L proviral function 36 is specific to ZIKV. 37 38 39 Keywords 40 Zika virus, flavivirus, RNase L, replication factories, dengue virus, West Nile Kunjin virus 41 42 43 44 45 46 47 All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': 'a9c58042555c0d56b8e94bc5558db0c3dd697dcc',\n","  'section': 'Abstract',\n","  'text': 'Retrospective case series. Methods were admitted to five hospitals in Zhejiang province, China. Data were Of all 91 patients, 88 (96.70%) were laboratory-confirmed COVID-19 with throat swab samples that tested positive for SARS-Cov-2 while 3 (3.30%) were clinical-diagnosed COVID-19 cases. The median age of the patients was 50 (36.5-57) years, and female accounted for 59.34%. In this sample 40 (43.96%) patients had contracted the diseases from local cases, 31 (34.07%) patients had been to Wuhan/Hubei, 8 (8.79%) cases had contacted with people from Wuhan, 11 (12.09%) cases were confirmed aircraft transmission. In particular within the city of Ningbo, 60.52% cases can be traced back to an event held in a temple. The most common symptoms were fever (71.43%), cough (60.44%) and fatigue (43.96%). The median of incubation period was 6 (IQR, 3-8) days and the median time from first visit to a doctor to confirmed diagnosis was 1 (1-2) days. According to the Chest computed tomography scans, 67.03% cases had bilateral pneumonia. . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint . https://doi.org/10. 1101 /2020 Conclusions Social activity cluster, family cluster and travel by airplane were how COVID-19 patients get transmitted and could be rapidly diagnosed COVID-19 in Zhejiang. '},\n"," {'paper_id': 'a9c98dc9001a93b57599a06a1145df12751ccad6',\n","  'section': 'Abstract',\n","  'text': 'The COVID-19 disease is rapidly spreading in whole globe, affecting millions of people and pushing governments to take drastic measures to contain the outbreaks. The understanding of the dynamics of the epidemic is of great interest for the governments and health authorities that are facing COVID-19 outbreaks. The scarce presence of epidemiologic data, due to the still ongoing outbreaks, makes prediction difficult and mainly based on heuristic (fitting) models. However, these models with non-physical based parameters, can only give limited insight in the evolution of the outbreaks. In this work a SIRD compartmental model was developed to describe and predict the evolution of the Chinese and Italian outbreaks. Exploiting the similarities of the measures taken by the governments to contain the virus and of the total population number of Hubei province and Italy, the model was tuned on the Chinese outbreak (almost extinguished) and by perturbation the Italian outbreak was describe and predicted. With this approach the lack of data due to the still ongoing Italian epidemic was compensated by the Chinese experience, allowing to obtain a SIRD model for this relatively new COVID-19 outbreak. According to the modeling prediction the peak of infection should be on the 28 th of March, with almost 42,000 infected people. '},\n"," {'paper_id': 'aa12917f77de45b88f1c9a06fba69686863b81ae',\n","  'section': 'Abstract',\n","  'text': 'In the current COVID-19 pandemic context, proposing and validating effective treatments represents a major challenge. However, the lack of biologically relevant pre-clinical experimental models of SARS-CoV-2 infection as a complement of classic cell lines represents a 35 major barrier for scientific and medical progress. Here, we advantageously used human reconstituted airway epithelial models of nasal or bronchial origin to characterize viral infection kinetics, tissue-level remodeling of the cellular ultrastructure and transcriptional immune signatures induced by SARS-CoV-2. Our results underline the relevance of this model for the preclinical evaluation of antiviral candidates. Foremost, we provide evidence on the antiviral 40 efficacy of remdesivir and the therapeutic potential of the remdesivir-diltiazem combination as a rapidly available option to respond to the current unmet medical need imposed by COVID-19. : bioRxiv preprint 65 reported the advantage of using more physiological models such as in-house or commercially available reconstituted human airway epithelia (HAE) to isolate, culture and study a wide range of respiratory viruses (8, 9) . Developed from biopsies of nasal or bronchial cells differentiated in the air/liquid interphase, these models reproduce with high fidelity most of the main structural, '},\n"," {'paper_id': 'aa4423bd6bbf65648e770756f17092913a5a79a6',\n","  'section': 'Abstract',\n","  'text': 'The World Health Organisation has declared a pandemic caused by the newly discovered SARS-CoV-2. Due to growing demand for reagents used for SARS-CoV-2 RNA extraction for subsequent molecular diagnostics, there is a worldwide risk of kit-and/or reagent-shortages for extraction. With a detection sensitivity of 97.4% (95% CI=86.2-99.9%), we describe a simple, fast, alternative workflow for molecular detection of SARS-CoV-2, where samples are simply heat-processed for 5 minutes at 98˚C prior to the RT-qPCR reaction. '},\n"," {'paper_id': 'aa507fcd3876f8c422993c8dd13847b1b8e77471',\n","  'section': 'Abstract',\n","  'text': '1 To explore potential intermediate host of a novel coronavirus is vital to rapidly control 2 continuous COVID-19 spread. We found genomic and evolutionary evidences of the 3 occurrence of 2019-nCoV-like coronavirus (named as Pangolin-CoV) from dead Malayan 4 Pangolins. Pangolin-CoV is 91.02% and 90.55% identical at the whole genome level to 5 2019-nCoV and BatCoV RaTG13, respectively. Pangolin-CoV is the lowest common 6 ancestor of 2019-nCoV and RaTG13. The S1 protein of Pangolin-CoV is much more 7 closely related to 2019-nCoV than RaTG13. Five key amino-acid residues involved in the 8 interaction with human ACE2 are completely consistent between Pangolin-CoV and 2019-9 nCoV but four amino-acid mutations occur in RaTG13. It indicates Pangolin-CoV has 10 similar pathogenic potential to 2019-nCoV, and would be helpful to trace the origin and 11 probable intermediate host of 2019-nCoV. '},\n"," {'paper_id': 'aa57784258b47b91d9283af65802f27e0b36b013',\n","  'section': 'Abstract',\n","  'text': 'As the number of cases of COVID-19 continues to grow exponentially, local health services are likely to be overwhelmed with patients requiring intensive care. We develop and implement an algorithm to provide optimal re-routing strategies to transfer patients requiring Intensive Care Units (ICU) between hospitals within NHS trusts, constrained by feasibility of transfer. We coarse-grain the NHS system at the level of NHS trusts and, subsequently cover the whole set of geopositioned trusts to extract a 4-regular geometric graph which indicates, for a given trust, its four nearest neighbors. Estimates of weekly ICU demand can be extrapolated from an age structured epidemiological model by considering contagion-to-ICU likelihood estimates, and through random search optimisation we identify the best load sharing strategy. The cost function to minimise is based on the total number of ICU units above capacity and we implement and test two optimisation strategies. Our framework is flexible allowing for additional criteria, different cost functions, and this methodology is general enough that it can easily be extended to optimise other resources beyond ICU units or ventilators. Assuming a uniform ICU demand across trusts, we show that using our method it is possible to enable access to ICU treatment to up to 1000 cases nationally in a single step of the algorithm -leading to potentially saving a large percentage of these lives that would otherwise not have access to ICU if no load sharing was implemented. '},\n"," {'paper_id': 'aa5a99e67d3e887b3fa88575fb0d4a662d0a60ce',\n","  'section': 'Abstract',\n","  'text': 'Epidemiological models are dominated by SEIR (Susceptible, Exposed, Infected and Removed) dynamical systems formulations and their elaborations. These formulations can be continuous or discrete, deterministic or stochastic, or spatially homogeneous or heterogeneous, the latter often embracing a network formulation. Here we review the continuous and discrete deterministic and discrete stochastic formulations of the SEIR dynamical systems models, and we outline how they can be easily and rapidly constructed using the Numerus Model Builder, a graphicallydriven coding platform. We also demonstrate how to extend these models to a metapopulation setting using both the Numerus Model Builder network and geographical mapping tools. '},\n"," {'paper_id': 'aa8b881bfc32fb55003961f75e201d9d847cd1b6',\n","  'section': 'Abstract',\n","  'text': 'The pandemic caused by the Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) has resulted in several thousand deaths worldwide in just a few months. Patients who died from Coronavirus disease 2019 (COVID-19) often had comorbidities, such as hypertension, diabetes, and chronic obstructive lung disease. The angiotensin-converting enzyme 2 (ACE2) was identified as a crucial factor that facilitates SARS-CoV2 to bind and enter host cells. To date, no study has assessed the ACE2 expression in the lungs of patients with these diseases. Here, we analyzed over 700 lung transcriptome samples of patients with comorbidities associated with severe COVID-19 and found that ACE2 was highly expressed in these patients, compared to control individuals. This finding suggests that patients with such comorbidities may have higher chances of developing severe COVID-19. We also found other genes, such as RAB1A, that can be important for SARS-CoV-2 infection in the lung. Correlation and network analyses revealed many potential regulators of ACE2 in the human lung, including genes related to histone modifications, such as HAT1, HDAC2, and KDM5B. In fact, epigenetic marks found in ACE2 locus were compatible to with those promoted by KDM5B. Our systems biology approach offers a possible explanation for increase of COVID-19 severity in patients with certain comorbidities. '},\n"," {'paper_id': 'aac94361fff1512d252a8e580f8bb56bad25c8cb',\n","  'section': 'Abstract',\n","  'text': 'Background: COVID-19 outbreak poses an unprecedented challenge for societies, healthcare organizations and economies. In the present analysis we coupled climate data with COVID-19 spread rates worldwide, and in a single country (USA). two measures: replication rate (RR), the slope of the logarithmic curve of confirmed cases, and the rate of spread (RoS), the slope of the linear regression of the logarithmic curve. : Based on predefined criteria, the mean COVID-19 RR was significantly lower in warm climate countries (0.12±0.02) compared with cold countries (0.24±0.01), (P<0.0001). Similarly, RoS was significantly lower in warm climate countries 0.12±0.02 vs. 0.25 ± 0.01 than in cold climate countries (P<0.001). In all countries (independent of climate classification) both RR and RoS displayed a moderate negative correlation with temperature R= -0.69, 95% confidence interval [CI], -0.87 to -0.36; P<0.001 and R= -0.72, 95% confidence interval [CI], -0.87 to -0.36; P<0.001, respectively. We identified a similar moderate negative correlation with the dew point temperature. Additional climate variables did not display a significant correlation with neither RR nor RoS. Finally, in an ancillary analysis, COVID-19 intra-country model using an inter-state analysis of the USA did not identify yet correlation between climate parameters and RR or RoS as of March, 19, 2020. Our analysis suggests a plausible negative correlation between warmer climate and COVID-19 spread rate as defined by RR and RoS worldwide. This initial correlation should be interpreted cautiously and be further validated over time, the pandemic is at different stages in various countries as well as in regions within these countries. As such, some associations may be more affected by local transmission patterns rather than by climate. Importantly, we provide an All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': 'aacc3acdd6a928fd820d51a2d1ea98c164b6abd4',\n","  'section': 'Abstract',\n","  'text': \"The emergence of intestinal organoids, as a stem cell-based self-renewable model system, has led to many studies on intestinal development and cell-cell signaling. However, potential issues regarding the phenotypic stability and reproducibility of the methodology during culture still needs to be addressed for different organoids. Here we investigated the transcriptomes of intestinal organoids derived from the same pig as well as batch-to-batch variation of organoids derived from different pigs over long-term passage. The set of genes expressed in organoids closely resembled that of the tissue of origin, including location specific functions, for at least 17 passages. Minor differences in gene expression were observed between individual organoid cultures. In contrast, most tissue-specific genes were not expressed in the transformed jejunum cell line IPECJ2, which also showed gene expression consistent with cancer phenotypes. We conclude that intestinal organoids provide a robust and stable model for translational research with clear advantages over transformed cells. Methods Jejunum tissue segments were obtained from control piglets used for another study, following guidelines of the animal ethics committee of Wageningen University. Two 5-week-old piglets were used for generating organoids following procedures previously described [11, 13] . Briefly, a 2 cm section of the mid-jejunum was dissected and placed in ice-cold PBS. After opening the sections longitudinally, jejunum segments were washed three times in ice-cold PBS and villi removed by carefully scraping with a scalpel. Small sections of the mucosa were cut from the muscle layer, divided into small cubes, and transferred into ice-cold PBS containing 30 mM EDTA and incubated with rotation at room temperature for 15 min. The PBS -EDTA was then replaced and incubation continued for 10 min at 37 °C. After washing in ice-cold DMEM supplemented with 5% penicillin/streptomycin (Gibco, Thermo Fisher Scientific), the crypts were dissociated by rigorous vortexing and passed through a 100 µm strainer into cold DMEM containing 5% foetal bovine serum (FBS, v/v). Crypts were pelleted by centrifugation at 300 x g for 5 min, and suspended in Matrigel (Basement Membrane, Growth factor reduced, REF 356231, Corning, Bedford, MA, USA). To improve surface tension, empty 24-well plates were pre-incubated overnight at 37 °C. Matrigel containing crypts was then plated at 7 domes per well (approx. 35 µl per well) and inverted to polymerize at 37 °C. After polymerization, 600 µl F12 cell culture medium (Gibco) was added, supplemented with 100 μg/ml primocin (Invivogen), 10 mM HEPES (HyClone), 1 × B-27 (Gibco), 1.25 mM N-acetylcysteine (Sigma), 50 ng/ml human epidermal growth factor (R&D systems), 15 nM gastrin, 10 mM nicotinamide, 10 μM p38 MAPK inhibitor (Sigma), 600 nM TGFβ receptor inhibitor A83-01, and conditioned media for recombinant Noggin (15% v/v), Spondin (15% v/v), and Wnt3A (30% v/v) provided by dr. Kuo and Hubrecht Institute (Utrecht, the Netherlands). Organoids were passaged at a 1:5 ratio every 5 days using ice-cold DMEM by mechanical dissociation, centrifuged at 500 x g for 5 min, and plating in fresh Matrigel matrix droplets as previously described [11] . Directly after crypt isolation, organoid cultures were separated into three batches per animal and grown independently for 17 passages. Jejunum organoids were grown for 3 and 12 weeks (4-17 passages) and extracted using ice-cold PBS. After washing twice in PBS, intact organoids were pelleted at 300 x g for 5 min and suspended in RLT lysis buffer and stored at -80°C prior to RNA isolation. Porcine jejunum epithelial cell line IPEC-J2 (ACC-701) was grown in DMEM F12 medium supplemented with 10% FBS and 5% penicillin/streptomycin (P/S, Gibco) in 75 cm 2 culture flasks. Data for one IPEC-J2 transcriptome was kindly provided by dr. Richard Crooijmans, via the Functional Annotation of ANimal Genomes (FAANG, BioSamples accession SAMEA4447551) [14] . For the analysis of the remaining two lines, IPEC-J2 at passage 87 and 91 were seeded at 5 x 10 4 cells/well in 24-well plates and grown to confluence within 48h with reduced P/S (1%). The monolayers were subsequently left to differentiate for 5 days, lysed using RLT-buffer, and stored at -80 °C prior to RNA isolation. Total RNA was extracted using an RNeasy mini-kit (Qiagen) following manufacturer's instructions including a 15 min on-column DNAse step. Preliminary tRNA concentrations, contamination and degradation were identified using Qubit (Thermo-Fisher) and gel-electrophoresis. The quantity and integrity of RNA was measured using a Bioanalyzer 2100 (Agilent). \"},\n"," {'paper_id': 'aad76905ce54679c80b75e4ee35717c30e7e1099',\n","  'section': 'Abstract',\n","  'text': \"In this study, we used pan RNA-seq analysis to reveal the ubiquitous existence of 5' end and 3' end 23 small RNAs. 5' and 3' sRNAs alone can be used to annotate mitochondrial with 1-bp resolution and nuclear 24 non-coding genes and identify new steady-state RNAs, which are usually from functional genes. Using 5', 3' 25 and intronic sRNAs, we revealed that the enzymatic dsRNA cleavage and RNAi could involve in the RNA 26 degradation and gene expression regulation of U1 snRNA in human. The further study of 5', 3' and intronic 27 sRNAs help rediscover double-stranded RNA (dsRNA) cleavage, RNA interference (RNAi) and the 28 regulation of gene expression, which challenges the classical theories. In this study, we provided a simple 29 and cost effective way for the annotation of mitochondrial and nuclear non-coding genes and the 30 identification of new steady-state RNAs, particularly long non-coding RNAs (lncRNAs). We also provided 31 a different point of view for cancer and virus, based on the new discoveries of dsRNA cleavage, RNAi and 32 the regulation of gene expression. \"},\n"," {'paper_id': 'aafa6cdfe96a5cdaf7b7c2f04b11a5dbdd73b2df',\n","  'section': 'Abstract',\n","  'text': 'Background: Recently, Coronavirus Disease 2019 (COVID-19) outbreak started in Wuhan, China. Although the clinical features of have been reported previously, data regarding the risk factors associated with the clinical outcomes are lacking. Objectives: To summary and analyze the clinical characteristics and identify the predictors of disease severity and mortality. The PubMed, Web of Science Core Collection, Embase, Cochrane and MedRxiv databases were searched through February 25, 2020. Meta-analysis of Observational Studies in Epidemiology (MOOSE) recommendations were followed. We extracted and pooled data using random-e ects meta-analysis to summary the clinical feature of the confirmed COVID-19 patients, and further identify risk factors for disease severity and death. Heterogeneity was evaluated using the I² method and explained with subgroup analysis and meta-regression. A total of 30 studies including 53000 patients with were included in this study, the mean age was 49.8 years (95% CI, 47.5-52.2 yrs) and 55.5% were male. The pooled incidence of severity and mortality were 20.2% (95% CI, 15.1-25.2%) and 3.1% (95% CI, 1.9-4.2%), respectively. The predictor for disease severity included old age (≥ 50 yrs, odds ratio [OR] = 2.61; 95% CI, 2.29-2.98), male (OR =1.348, 95% CI, 1.195-1.521), smoking (OR =1.734, 95% CI, All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. : medRxiv preprint 1.146-2.626) and any comorbidity (OR = 2.635, 95% CI, 2.098-3.309), especially chronic kidney disease (CKD, OR = 6.017; 95% CI,, chronic obstructive pulmonary disease (COPD, OR = 5.323; 95% CI, 2.613-10.847) and cerebrovascular disease (OR = 3.219; 95% CI, 1.486-6.972). In terms of laboratory results, increased lactate dehydrogenase (LDH), C-reactive protein (CRP) and D-dimer and decreased blood platelet and lymphocytes count were highly associated with severe COVID-19 (all for P < 0.001). Meanwhile, old age (≥ 60 yrs, RR = 9.45; 95% CI, 8.09-11.04), followed by cardiovascular disease (RR = 6.75; 95% CI, 5.40-8.43) hypertension (RR = 4.48; 95% CI, and diabetes (RR = 4.43; 95% CI, 3.49-5.61) were found to be independent prognostic factors for the COVID-19 related death. To our knowledge, this is the first evidence-based medicine research to explore the risk factors of prognosis in patients with COVID-19, which is helpful to identify early-stage patients with poor prognosis and adapt effective treatment. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': 'ab0693fcea66cf43ac63de91e546fcfb96daba6d',\n","  'section': 'Abstract',\n","  'text': 'The literature reports several symptoms of SARS-CoV-2 in humans such as fever, cough, fatigue, pneumonia, and headache. Furthermore, patients infected with similar strains (SARS-CoV and MERS-CoV) suffered testis, liver, or thyroid damage. Angiotensin-converting enzyme 2 (ACE2) serves as an entry point into cells for some strains of coronavirus (SARS-CoV, MERS-CoV, SARS-CoV-2). Our hypothesis was that as ACE2 is essential to the SARS-CoV-2 virus invasion, then brain regions where ACE2 is the most expressed are more likely to be disturbed by the infection. Thus, the expression of other genes which are also over-expressed in those damaged areas could be affected. We used mRNA expression levels data of genes provided by the Allen Human Brain Atlas (ABA), and computed spatial correlations with the LinkRbrain platform. Genes whose co-expression is spatially correlated to that of ACE2 were then clustered into 16 groups, depending on the organ in which they are the most expressed (as described by the NCBI genes database). The list of organs where genes sharing local over-expression with the ACE2 gene are the most expressed is astonishingly similar to the organs affected by Covid-19. '},\n"," {'paper_id': 'ab1692d56f32fdb0922e5f323f6ca385fb1e0808',\n","  'section': 'Abstract',\n","  'text': 'Virophages are satellite-like double stranded DNA viruses whose replication requires the presence of two biological entities, a giant virus and a protist. In this report, we present the first evidence of host range expansion in a virophage. We demonstrated that the Guarani virophage was able to spontaneously expand its viral host range to replicate with two novel giant viruses that were previously nonpermissive to this virophage. We were able to characterize a potential genetic determinant of this cross-species infection. We then highlighted the relevant impact of this host adaptation on giant viruses and protists by demonstrating that coinfection with the mutant virophage abolishes giant virus production and rescues the host cell population from lysis. The results of our study help to elucidate the parasitic lifestyle of virophages and their interactions with giant viruses and protists. author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': 'ab8c488ab6d262aaaa2dda1f953257c5af588064',\n","  'section': 'Abstract',\n","  'text': 'Coronavirus disease 2019 (COVID-19) has been rapidly spreading throughout China and other countries including South Korea. As of March 12, 2020, a total number of 7,869 cases and 66 deaths had been documented in Using spatial visualization, this paper identified two early transmission clusters in South Korea (Daegu cluster and capital area cluster). Using a degree-weighted centrality measure, this paper proposes potential super-spreaders of the virus in the visualized clusters. Compared to various epidemiological measures such as the basic reproduction number, spatial visualizations of the cluster-specific transmission networks and the proposed centrality measure may be more useful to characterize super-spreaders and the spread of the virus especially in the early epidemic phase. '},\n"," {'paper_id': 'abcfffafab399149d4adadd6bb458c4994e2025d',\n","  'section': 'Abstract',\n","  'text': 'Rationale: Several studies have estimated basic production number of novel coronavirus pneumonia (NCP). However, the time-varying transmission dynamics of NCP during the outbreak remain unclear. Objectives: We aimed to estimate the basic and time-varying transmission dynamics of NCP across China, and compared them with SARS. Methods: Data on NCP cases by February 7, 2020 were collected from epidemiological investigations or official websites. Data on severe acute respiratory syndrome (SARS) cases in Guangdong Province, Beijing and Hong Kong during 2002-2003 were also obtained. We estimated the doubling time, basic reproduction number (R0) and time-varying reproduction number (Rt) of NCP and SARS. Measurements and main results: As of February 7, 2020, 34,598 NCP cases were identified in China, and daily confirmed cases decreased after February 4. The doubling time of NCP nationwide was 2.4 days which was shorter than that of SARS in Guangdong (14.3 days), Hong Kong (5.7 days) and Beijing (12.4 days). The R0 of NCP cases nationwide and in Wuhan were 4.5 and 4.4 respectively, which were higher than R0 of SARS in Guangdong (R0=2.3), Hongkong (R0=2.3), and Beijing (R0=2.6). The Rt for NCP continuously decreased especially after January 16 nationwide and in Wuhan. The R0 for secondary NCP cases in Guangdong was 0.6, and the Rt values were less than 1 during the epidemic. NCP may have a higher transmissibility than SARS, and the efforts of containing the outbreak are effective. However, the efforts are needed to persist in for reducing time-varying reproduction number below one. now named as Novel Coronavirus Pneumonia (NCP), occurred in Wuhan, Hubei Province, China. The disease has rapidly spread from Wuhan to other areas. As a novel virus, the time-varying transmission dynamics of NCP remain unclear, and it is also important to compare it with SARS. What This Study Adds to the Field: We compared the transmission dynamics of NCP with SARS, and found that NCP has a higher transmissibility than SARS. Time-varying production number indicates that rigorous control measures taken by governments are effective across China, and persistent efforts are needed to be taken for reducing instantaneous reproduction number below one. author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': 'ac0197045a3bcf94e272bcf02145c760f04beb9a',\n","  'section': 'Abstract',\n","  'text': 'C3A is a sub-clone of human hepatoblastoma HepG2 cell line with the strong contact inhibition 26 of growth. We fortuitously found that C3A was more susceptible to human coronavirus HCoV-27 OC43 infection than HepG2, which was attributed to the increased efficiency of virus entry into 28 C3A cells. In an effort to search for the host cellular protein(s) mediating the differential 29 susceptibility of the two cell lines to HCoV-OC43 infection, we found that ADAP2, GILT and 30 LY6E, three cellular proteins with known activity of interfering virus entry, expressed at 31 significantly higher levels in HepG2 cells. Functional analyses revealed that ectopic expression 32 of LY6E, but not GILT or ADAP2, in HEK 293 cells inhibited the entry of HCoV-OC43. While 33 overexpression of LY6E in C3A and A549 cells efficiently inhibited the infection of HCoV-34 OC43, knockdown of LY6E expression in HepG2 significantly increased its susceptibility to 35 HCoV-OC43 infection. Moreover, we found that LY6E also efficiently restricted the entry 36 mediated by the envelope spike proteins of other human coronaviruses, including the currently 37 pandemic SARS-CoV-2. Interestingly, overexpression of serine protease TMPRSS2 or 38 amphotericin treatment significantly neutralized the IFITM3 restriction of human coronavirus 39 entry, but did not compromise the effect of LY6E on the entry of human coronaviruses. The 40 work reported herein thus demonstrates that LY6E is a critical antiviral immune effector that 41 controls CoV infection and pathogenesis via a distinct mechanism. '},\n"," {'paper_id': 'ac35eeb6eaaf382fe0e123719f7dc409046e5d87',\n","  'section': 'Abstract',\n","  'text': \"Estimating the key epidemiological features of the novel coronavirus (2019-nCoV) epidemic proves to be challenging, given incompleteness and delays in early data reporting, in particular, the severe under-reporting bias in the epicenter, Wuhan, Hubei Province, China. As a result, the current literature reports widely varying estimates. We developed an alternative geostratified debiasing estimation framework by incorporating human mobility with case reporting data in three stratified zones, i.e., Wuhan, Hubei Province excluding Wuhan, and mainland China excluding Hubei. We estimated the latent infection ratio to be around 0.12% (18,556 people) and the basic reproduction number to be 3.24 in Wuhan before the city's lockdown on January 23, 2020. The findings based on this debiasing framework have important implications to prioritization of control and prevention efforts. One Sentence Summary: A geo-stratified debiasing approach incorporating human movement data was developed to improve modeling of the 2019-nCoV epidemic. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. \"},\n"," {'paper_id': 'ac3e33b6ed12e43208d76bf455f3b81067e6ac6d',\n","  'section': 'Abstract',\n","  'text': 'We conducted a comparative study of COVID-19 epidemic in three different settings: mainland China, the Guangdong province of China and South Korea, by formulating two disease transmission dynamics models incorporating epidemic characteristics and setting-specific interventions, and fitting the models to multi-source data to identify initial and effective reproduction numbers and evaluate effectiveness of interventions. We estimated the initial basic reproduction number for South Korea, the Guangdong province and mainland China as 2.6 (95% confidence interval (CI): (2.5, 2.7)), 3.0 (95%CI: (2.6, 3.3)) and 3.8 (95%CI: (3.5,4.2)), respectively, given a serial interval with mean of 5 days with standard deviation of 3 days. We found that the effective reproduction number for the Guangdong province and mainland China has fallen below the threshold 1 since February 8 th and 18 th respectively, while the effective reproduction number for South Korea remains high, suggesting that the interventions implemented need to be enhanced in order to halt further infections. We also project the epidemic trend in South Korea under different scenarios where a portion or the entirety of the integrated package of interventions in China is used. We show that a coherent and integrated approach with stringent public health interventions is the key to the success of containing the epidemic in China and specially its provinces outside its epicenter, and we show that this approach can also be effective to mitigate the burden of the COVID-19 epidemic in South Korea. The experience of outbreak control in mainland China should be a guiding reference for the rest of the world including South Korea. '},\n"," {'paper_id': 'ac45d1c2865c21a16912e73a857060bacce4c95a',\n","  'section': 'Abstract',\n","  'text': 'Many detection methods have been used or reported for the diagnosis and/or surveillance of SARS-CoV-2. Among them, reverse transcription polymerase chain reaction (RT-PCR) is the most sensitive, claiming detection of about 5 copies of viruses. However, it has been reported that only 47-59% of the positive cases were identified by RT-PCR, probably due to loss or degradation of virus RNA in the sampling process, or even mutation of the virus genome. Therefore, developing highly sensitive methods is imperative to ensure robust detection capabilities. With the goal of improving sensitivity and accommodate various application settings, we developed a multiplex-PCR-based method comprised of 172 pairs of specific primers, and demonstrated its efficiency to detect SARS-CoV-2 at low copy numbers. The assay produced clean characteristic target peaks of defined sizes, which allowed for direct identification of positives by electrophoresis. In addition, optional sequencing can provide further confirmation as well as phylogenetic information of the identified virus(es) for specific strain discrimination, which will be of paramount importance for surveillance purposes that represent a global health imperative. Finally, we also developed in parallel a multiplex-PCR-based metagenomic method that is amenable to detect SARS-CoV-2, with the additional benefit of its potential for uncovering mutational diversity and novel pathogens at low sequencing depth. '},\n"," {'paper_id': 'ac4a3bbe27272cd6780e7168906b0bf1413fc749',\n","  'section': 'Abstract',\n","  'text': \"The detection of nucleic acids by pattern recognition receptors is an ancient and 27 conserved component of the innate immune system. Notably, RNA virus genomes 28 are sensed by mammalian cytosolic RIG-I-like receptors, thereby activating 29 interferon-stimulated gene (ISG) expression to restrict viral replication. However, 30 recent evidence indicates that the cGAS-STING DNA sensing pathway also 31 protects against RNA viruses. So far, the mechanisms responsible for DNA sensing 32 of RNA viruses, which replicate without known DNA intermediates, remain unclear. 33 By using cGAS gene knockout and reconstitution in human and mouse cell 34 , we discovered that DNA sensing and cGAMP synthase activities are 35 required for cGAS-mediated restriction of vesicular stomatitis virus and Sindbis 36 virus. The level of cGAMP produced in response to RNA virus infection was below 37 the threshold of detection, suggesting that only transient and/or low levels of 38 cGAMP are produced during RNA virus infections. To clarify the DNA ligands that 39 activate cGAS activity, we confirmed that cGAS binds mitochondrial DNA in the 40 cytosol of both uninfected and infected cells; however, the amount of 41 cGAS-associated mitochondrial DNA did not change in response to virus infection. 42 Rather, a variety of pre-existing cytosolic DNAs, including mitochondrial DNA and 43 endogenous cDNAs, may serve as stimuli for basal cGAS activation. Importantly, 44 cGAS knockout and reconstitution experiments demonstrated that cGAS drives 45 low-level ISG expression at steady state. We propose that cGAS-STING restricts 46 RNA viruses by promoting a preparatory immune activation state within cells, likely 47 primed by endogenous cellular DNA ligands. 48 . Importance 49 Many medically important RNA viruses are restricted by the cGAS-STING 50 DNA-sensing pathway of innate immune activation. Since these viruses replicate 51 without DNA intermediates, it is unclear what DNA ligand(s) are responsible for 52 triggering this pathway. We show here that cGAS's DNA binding and signaling 53 activities are required for RNA virus restriction, similar to the mechanisms by which it 54 restricts DNA viruses. Furthermore, we confirmed that cGAS continuously binds host 55 DNA, which was unaffected by RNA virus infection. Finally, cGAS expression 56 correlated with the low-level expression of interferon-stimulated genes in uninfected 57 cells, both in vitro and in vivo. We propose that cGAS-mediated sensing of 58 endogenous DNA ligands contributes to RNA virus restriction by establishing a 59 baseline of innate immune activation. doi: bioRxiv preprint 61 A key feature of innate immunity is the detection of pathogen-associated 62 molecular patterns (PAMPs) by pattern recognition receptors (PRRs) (1). For 63 mammalian cells, viral nucleic acids are detected by distinct PRRs, triggering 64 interferon-stimulated gene (ISG) expression to set up an antiviral state. During RNA 65 virus infections, uncapped and double-stranded RNAs are detected in the cytosol by 66 the PRRs retinoic acid-inducible gene I (RIG-I) and related RIG-I-like receptors 67 (RLRs). However, the recent discovery of the cGAS-STING cytosolic DNA sensing 68 pathway, and the observation that it can also restrict RNA viruses (2), reveals a need 69 to further investigate the mechanisms of nucleic acid sensing during RNA virus 70 infection. 71 The stimulator of interferon genes (STING) is an endoplasmic reticulum-and 72 mitochondrial-bound protein that spontaneously activates ISG expression when 73 overexpressed (2). Although STING is involved in DNA sensing, STING -/mice and 74 mouse endothelial fibroblasts (MEFs) are more permissive for vesicular stomatitis 75 virus (VSV), a negative-stand RNA virus (2, 3). Additionally, studies in MEFs 76 deficient in three prime repair exonuclease 1 (TREX1), a nuclease important for the 77 turnover of cytosolic retroelement cDNAs (4), have described enhanced antiviral 78 phenotypes in response to a wide array of RNA viruses and retroviruses, 79 presumably due to the accumulation of DNA in the cytosol (5, 6). It appears that this 80 DNA-based restriction is broad, as many RNA viruses have evolved mechanisms to 81 subvert the cGAS-STING pathway, including flaviviruses (7-9), hepaciviruses (10, 82 STING does not directly interact with cytosolic DNA, but functions as an innate 84 immune adaptor protein to transduce signals between cyclic GMP-AMP synthase 85 (cGAS) and Tank-binding kinase 1, which subsequently phosphorylates the 86 transcription factor interferon regulatory factor 3 (IRF3) to initiate an ISG response doi: bioRxiv preprint 157 158 growth of VSV-GFP, VSVΔM51A-GFP, and SINV-GFP, while THP-1 KO cells or 159 THP-1 KO cells reconstituted with inactive hcGAS-HA3x mutants showed little or no 160 restriction (Figure 3D-3F). As observed previously in MEFs, VSVΔM51A-GFP was 161 more potently restricted than VSV-GFP, but unlike in MEFs, infected fewer cells 162 expressing mutant cGAS. This was also true for SINV-GFP, albeit restriction with 163 WT hcGAS-HA3x was extremely potent, comparatively. It is unclear whether these 164 modest decreases in infection of the cGAS mutants was due to hcGAS-HA3x 165 overexpression in THP-1 cells, residual hcGAS activities, or normal clonal 166 variation of cells. Nevertheless, these results are most consistent with an integral 167 role for cGAS DNA binding and cGAMP synthase activities in RNA virus restriction. 168 Detection of cGAMP produced in response to DNA transfection but not 169 RNA virus infection. Because cGAMP synthesis activity was essential for RNA 170 virus restriction, we next sought to identify cGAMP produced in response to RNA . 676 2013. The papain-like protease of porcine epidemic diarrhea virus negatively 677 regulates type I interferon pathway by acting as a viral deubiquitinase. The 678 Journal of General Virology 94:1554-1567. 679 JL, Mar KB, Richardson RB, Ratushny AV, Litvak V, Dabelic R, 704 Manicassamy B, Aitchison JD, Aderem A, Elliott RM, Garcia-Sastre A, 705 Racaniello V, Snijder EJ, Yokoyama WM, Diamond MS, Virgin HW, Rice 706 CM. 2014. Pan-viral specificity of IFN-induced genes reveals new roles for 707 cGAS in innate immunity. Nature 505:691-695. 708 . CC-BY-NC-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/434027 doi: bioRxiv preprint Flamand A. 1990. Genetic evidence for multiple functions of the matrix 761 protein of vesicular stomatitis virus. J Gen Virol 71 ( Pt 4):991-996. 762 39. Li L, Yin Q, Kuss P, Maliga Z, Millan JL, Wu H, Mitchison TJ. 2014. 763 Hydrolysis of 2'3'-cGAMP by ENPP1 and design of nonhydrolyzable analogs. 764 P, Shu H-B. 2008. The Adaptor Protein MITA Links Virus-Sensing Receptors 794 to IRF3 Transcription Factor Activation. Immunity 29:538-550. 795 48. Castanier C, Garcin D, Vazquez A, Arnoult D. 2010. Mitochondrial 796 dynamics regulate the RIG-I-like receptor antiviral pathway. EMBO Rep 797 11:133-138. 798 49. Horner SM, Liu HM, Park HS, Briley J, Gale M. 2011. 799 Mitochondrial-associated endoplasmic reticulum membranes (MAM) form 800 innate immune synapses and are targeted by hepatitis C virus. Proceedings 801 of the National Academy of Sciences 108:14590-14595. 802 50. Schoggins JW, MacDuff DA, Imanaka N, Gainey MD, Shrestha B, Eitson 803 JL, Mar KB, Richardson RB, Ratushny AV, Litvak V, Dabelic R, 804 Manicassamy B, Aitchison JD, Aderem A, Elliott RM, Garcia-Sastre A, 805 Racaniello V, Snijder EJ, Yokoyama WM, Diamond MS, Virgin HW, Rice 806 CM. 2014. Pan-viral specificity of IFN-induced genes reveals new roles for 807 cGAS in innate immunity. Nature 505:691-695. 808 51. Liu Y, Liu Y, Wu J, Roizman B, Zhou GG. 2018. Innate responses to gene 809 knockouts impact overlapping gene networks and vary with respect to 810 resistance to viral infection. Proceedings of the National Academy of 811 Sciences 115:E3230. 812 52. Schoggins JW, Wilson SJ, Panis M, Murphy MY, Jones CT, Bieniasz P, 813 Rice CM. 2011. A diverse range of gene products are effectors of the type I 814 interferon antiviral response. Nature 472:481-485. 815 53. \"},\n"," {'paper_id': 'ac81102667b0d56edeb8ab0044765dc49a19f374',\n","  'section': 'Abstract',\n","  'text': 'Positive-sense RNA viruses hijack intracellular membranes that provide niches for viral RNA 23 synthesis and a platform for interactions with host proteins. However, little is known about host 24 factors at the interface between replicase complexes and the host cytoplasm. We engineered a 25 biotin ligase into a coronaviral replication/transcription complex (RTC) and identified >500 26 host proteins constituting the RTC microenvironment. siRNA-silencing of each host factor demonstrated importance of vesicular trafficking pathways, ubiquitin-dependent 28 and autophagy-related processes, and translation initiation factors. Notably, detection of 29 translation initiation factors at the RTC was instrumental to visualize and demonstrate active 30 translation proximal to replication complexes of several coronaviruses. 31 Collectively, we establish a spatial link between viral RNA synthesis and diverse host factors 32 of unprecedented breadth. Our data may serve as a paradigm for other positive-strand RNA 33 viruses and provide a starting point for a comprehensive analysis of critical virus-host 34 interactions that represent targets for therapeutic intervention. Positive-strand RNA viruses replicate at membranous structures that accommodate the viral 37 replication complex and facilitate RNA synthesis in the cytosol of infected host cells (1-5). 38 Rewiring host endomembranes is hypothesized to provide a privileged microenvironment 39 physically separated from the cytosol, thereby ensuring adequate concentrations of 40 macromolecules for viral RNA synthesis, preventing recognition of replication intermediates 41 such as double-stranded RNA (dsRNA) by cytosolic innate immune receptors (6, 7), and 42 providing a platform that facilitates molecular interactions with host cell proteins. 43 Ultrastructural studies have reported the origin, nature, and extent of membrane modifications 44 induced by coronaviruses (order Nidovirales, family Coronaviridae), which materialize as an 45 ER-derived network of interconnected double-membrane vesicles (DMVs) and convoluted 46 membranes (CM) in perinuclear regions of infected cells to which the viral 47 replication/transcription complex (RTC) is anchored (4, 8, 9). The RTC is generated by 48 translation of the genomic RNA into two large polyproteins that are extensively auto-49 proteolytically processed by viral proteases to give rise to 16 processing end-products, termed 50 non-structural proteins (nsps) 1-16. Nsp1 is rapidly cleaved from the polyproteins and not 51 considered an integral component of the coronaviral RTC, but interferes with host cell 52 translation by inducing degradation of cellular mRNAs (10-12). Although it has not yet been 53 formally demonstrated, the remaining nsps (2-16) are thought to comprise the RTC and harbor 54 multiple enzymes and functions, such as de-ubiquitination, proteases, helicase, polymerase, 55 exo-and endonuclease, and N7-and 2\\'O-methyltransferases (13-18). Many of these functions 56 have been studied using reverse genetic approaches, which revealed their importance in virus-57 host interactions (19-23). In most cases phenotypes were described via loss-of-function 58 mutagenesis, however, in the context of virus infection, the specific interactions of RTC 59 components with host cell factors remain largely unknown. 60 . CC-BY-NC-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/417907 doi: bioRxiv preprint 4 A number of individual host cell proteins have been shown to impact coronavirus replication 61 by using various screening methods, such as genome-wide siRNA, kinome, and yeast-two-62 hybrid screens (24-28). Likewise, genome-wide CRISPR-based screens have been applied to 63 other positive-stranded RNA viruses, such as flaviviruses, and identified critical host proteins 64 required for replication (29, 30). Some of these proteins were described in the context of 65 distinct ER processes, such as N-linked glycosylation, ER-associated protein degradation 66 (ERAD), and signal peptide insertion and processing. Although individual proteins identified 67 by these screens may interact with viral replication complexes, they likely constitute only a 68 small fraction of the global replicase microenvironment. 69 To capture the full breadth of host cell proteins and cellular pathways that are spatially 70 associated with viral RTCs, we employed a proximity-based labeling approach involving a 71 promiscuous E. coli-derived biotin ligase (BirAR118G). BirAR118G biotinylates proximal (<10 72 nm) proteins in live cells without disrupting intracellular membranes or protein complexes, and 73 hence, does not rely on high affinity protein-protein interactions, but is able to permanently tag 74 transient interactions (31). Covalent protein biotinylation allows stringent lysis and washing 75 conditions during affinity purification and subsequent mass spectrometric identification of 76 captured factors. By engineering a recombinant MHV harboring BirAR118G as an integral 77 component of the RTC we identified >500 host proteins reflecting the molecular 78 microenvironment of MHV replication structures. siRNA-mediated silencing of each of these 79 factors highlighted, amongst others, the functional importance of vesicular ER-Golgi apparatus 80 trafficking pathways, ubiquitin-dependent and autophagy-related catabolic processes, and 81 translation initiation factors. Importantly, the detection of active translation in close proximity 82 to the viral RTC highlighted the critical involvement of translation initiation factors during 83 coronavirus replication. Collectively, the determination of the coronavirus RTC-associated 84 microenvironment provides a functional and spatial link between conserved host cell processes 85 . CC-BY-NC-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/417907 doi: bioRxiv preprint 5 and viral RNA synthesis, and highlights potential targets for the development of novel antiviral 86 agents. 87 88 Results 89 Engineering the BirAR118-biotin ligase into the MHV replicase transcriptase complex 90 To insert the promiscuous biotin ligase BirAR118G as an integral subunit of the MHV RTC, we 91 used a vaccinia virus-based reverse genetic system (32, 33) to generate a recombinant MHV 92 harboring a myc-tagged BirAR118G fused to nsp2. This strategy was recently employed by 93 Freeman et al. for a fusion of green fluorescent protein (GFP) with nsp2 (34). MHV-BirAR118G-94 nsp2 retained the cleavage site between nsp1 and BirAR118G, while a deleted cleavage site 95 between BirAR118G and nsp2 ensured the expression of a BirAR118G-nsp2 fusion protein (Fig.  96   1a) . 97 MHV-BirAR118G-nsp2 replicated to comparable peak titers and replication kinetics as the 98 parental wild-type MHV-A59 (Fig. 1b) . MHV-EGFP-nsp2, which was constructed in parallel 99 and contained the coding sequence of EGFP (34) instead of BirAR118G, was used as a control 100 and also reached wild-type virus peak titers, with slightly reduced viral titers at 9 hours post-101 infection (h.p.i.) compared to MHV-A59 and MHV-BirAR118G-nsp2 (Fig. 1b) . 102 To confirm the accommodation of BirAR118G within the viral RTC, MHV-A59-, MHV-103 BirAR118G-nsp2-, and mock-infected L929 fibroblasts were visualized using indirect 104 immunofluorescence microscopy. BirAR118G-nsp2 remained strongly associated with the MHV 105 RTC, as indicated by the co-localization of BirAR118G-nsp2 with established markers of the 106 MHV replicase, such as nsp2/3 and nsp8 ( Fig. 1c; Supplemental Fig. S1 ). This observation 107 corroborates previous studies demonstrating that nsp2, although not required for viral RNA 108 synthesis, co-localizes with other nsps of the coronaviral RTC (35-37). Importantly, by 109 . CC-BY-NC-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/417907 doi: bioRxiv preprint 6 supplementing the culture medium with biotin, we could readily detect biotinylated proteins 110 with fluorophore-coupled streptavidin that appeared close to the MHV RTC in MHV-111 BirAR118G-nsp2-infected cells, demonstrating efficient proximity-dependent biotinylation of 112 RTC-proximal host factors ( Fig. 1c; Supplemental Fig. S1 ). Furthermore, to define the 113 localization of the nsp2 fusion protein at the ultrastructural level, we replaced the BirAR118G 114 biotin ligase with the APEX2 ascorbate peroxidase to generate recombinant MHV-APEX2-115 nsp2. APEX2 mediates the catalysis of 3,3\\'-diaminobenzidine (DAB) into an insoluble 116 polymer that can be readily observed by electron microscopy (38). As shown in figure 1d , 117 APEX2-catalized DAB polymer deposition was readily detectable at characteristic coronavirus 118 replication compartments, such as DMVs and CM, categorically demonstrating that the nsp2 119 fusion proteins localize to known sites of coronavirus replication (4, 8). 120 Importantly, our collective results establish that the recombinant MHV-BirAR118G-nsp2 121 replicates with comparable kinetics to wild-type MHV-A59, expresses a functional BirAR118G 122 biotin ligase that is tightly associated with the MHV RTC, and that biotinylated, RTC-proximal 123 proteins can be readily detected in MHV-BirAR118G-nsp2 infected cells. 124 125 To further demonstrate the efficiency and specificity of BirAR118G-mediated biotinylation we 127 assessed, by western blot analysis, fractions of biotinylated proteins derived from MHV-A59-128 , MHV-BirAR118G-nsp2-, or non-infected cells that were grown with or without the addition of 129 biotin (Fig. 2a,b) . A characteristic pattern of endogenously biotinylated proteins was observed 130 under all conditions where no exogenous biotin was added to the culture medium (Fig. 2b) . 131 The same pattern was detectable in non-infected and wild-type MHV-A59-infected cells when 132 the culture medium was supplemented with biotin, suggesting that the addition of biotin in the 133 . CC-BY-NC-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/417907 doi: bioRxiv preprint 7 absence of the BirAR118G biotin ligase does not recognizably change the fraction of 134 endogenously biotinylated proteins. In contrast, we observed a greatly increased fraction of 135 biotinylated proteins in lysates derived from MHV-BirAR118G-nsp2-infected cells treated with 136 biotin. This result demonstrates that virus-mediated expression of the BirAR118G biotin ligase 137 results in efficient biotinylation when biotin is added to the culture medium. Moreover, we 138 could readily affinity purify, enrich, and recover the fraction of biotinylated proteins under 139 stringent denaturing lysis and washing conditions by using streptavidin-coupled magnetic 140 beads (Fig. 2b) . 141 Affinity purified proteins derived from biotin-treated MHV-A59-and MHV-BirAR118G-nsp2-142 infected cells were subjected to mass spectrometric analysis (n=3). Liquid chromatography 143 tandem-mass spectrometry (LC-MS/MS) was performed from in-gel digested samples and log-144 transformed label free quantification (LFQ) levels were used to compare protein enrichment 145 between samples (Fig. 2c) . Overall, 1381 host proteins were identified, of which 513 were 146 statistically significantly enriched in MHV-BirAR118G-nsp2-infected samples over MHV-A59-147 infected samples. These host proteins represent a comprehensive repertoire of RTC-proximal 148 factors throughout MHV infection (Fig. 2c, table S1 ). Importantly, viral replicase gene 149 products nsp2-10 and nsp12-16, and the nucleocapsid protein were significantly enriched in 150 fractions derived from MHV-BirAR118G-nsp2-infected cells (Fig. 2c, d) . This is in agreement 151 with studies demonstrating co-localization and interactions amongst individual nsps, and with 152 studies showing association of the nucleocapsid protein with the coronavirus RTC (8,[39][40][41]. 153 It also highlights the specificity and effectiveness of the labeling approach in live cells and is 154 the first experimental evidence showing that collectively these viral nsps and the nucleocapsid 155 (N) protein are subunits of the coronavirus RTC. Furthermore, these results corroborate 156 previous reports that nsp1 is likely not an integral component of the coronavirus RTC (10-12, 157 42). Amongst the \"not detected\" or \"not enriched\" viral proteins are (i) nsp11, which is a short 158 . CC-BY-NC-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/417907 doi: bioRxiv preprint number of host proteins from our candidate list in MHV-infected cells. Accordingly, we 172 identified RTC-proximal host proteins displaying a pronounced co-localization with the MHV 173 RTC, such as the ER protein reticulon 4 (rtn4; Fig. 2e ), and host proteins where co-localization 174 by indirect immunofluorescence microscopy was not readily detectable, such as the eukaryotic 175 translation initiation factor 3E (eIF3E; Fig. 2e ). However, in the latter case, a more sensitive 176 detection technique, such as a proximity ligation assay that relies on proximity-dependent 177 antibody-coupled DNA probe amplification (43), demonstrated proximity of eIF3E and 178 dsRNA in MHV-infected cells (Fig. 2f) . 179 Collectively, our results show that the approach of integrating a promiscuous biotin ligase as 180 an integral subunit into a coronavirus RTC revealed a comprehensive list of host cell proteins 181 that comprises the RTC microenvironment. The efficacy and specificity of our approach is best 182 illustrated by the fact that we were able to identify all expected viral components of the MHV 183 . CC-BY-NC-ND 4.0 '},\n"," {'paper_id': 'acb0e11e5763dbf7588ec435751a2ab705a783f1',\n","  'section': 'Abstract',\n","  'text': 'Novel Coronavirus (nCoV) outbreak in the city of Wuhan, China during December 2019, has now spread to various countries across the globe triggering a heightened containment effort. This human pathogen is a member of betacoronavirus genus carrying 30 kilobase of single positive-sense RNA genome. Understanding the evolution, zoonotic transmission, and source of this novel virus would help accelerating containment and prevention efforts. The present study reported detailed analysis of 2019-nCoV genome evolution and potential candidate peptides for vaccine development. This nCoV genotype might have been evolved from a bat-CoV by accumulating non-synonymous mutations, indels, and recombination events. Structural proteins Spike (S), and Membrane (M) had extensive mutational changes, whereas Envelope (E) and Nucleocapsid (N) proteins were very conserved suggesting differential selection pressures exerted on 2019-nCoV during evolution. Interestingly, 2019-nCoV Spike protein contains a 39 nucleotide sequence insertion relative to SARSlike bat-SL-CoVZC45/2017. Furthermore, we identified eight high binding affinity (HBA) CD4 T-cell epitopes in the S, E, M and N proteins, which can be commonly recognized by HLA-DR alleles of Asia and Asia-Pacific Region population. These immunodominant epitopes can be incorporated in universal subunit CoV vaccine. Diverse HLA types and variations in the epitope binding affinity may contribute to the wide range of immunopathological outcomes of circulating virus in humans. Our findings emphasize the requirement for continuous surveillance of CoV strains in live animal markets to better understand the viral adaptation to human host and to develop practical solutions to prevent the emergence of novel pathogenic CoV strains. '},\n"," {'paper_id': 'acd5cb2ba08da5c0f6bc310d74d497d38aa86be2',\n","  'section': 'Abstract',\n","  'text': 'word count: 225 22 Importance word count: 144 23 . CC-BY-NC-ND 4.0 International license is made available under a '},\n"," {'paper_id': 'acd8d28ae75dc1ec5fea2d767e3158545011c1f7',\n","  'section': 'Abstract',\n","  'text': \"Mycetoma is a neglected tropical disease, endemic in many tropical and subtropical regions, 49 characterised by massive deformity and disability and can be fatal if untreated early and 50 appropriately. Interleukins (IL) -35 and IL-37 are newly discovered cytokines that play an 51 important role in suppressing the immune system. However, the expression of these 52 interleukins in patients with Madurella mycetomatis (M. mycetomatis) induced eumycetoma 53 has not yet been explored. This study aims to determine the levels of the IL-1 family (IL-1β, 54 IL-37) and IL-12 family (IL-12, IL-35) in a group of these patients and the association 55 between these cytokines levels and the patients' demographic characteristics. The present, 56 a case-control study was conducted at the Mycetoma Research Centre, Soba University 57 Hospital, University of Khartoum, Sudan and it included 140 individuals. They were divided 58 into two groups; group I: healthy controls [n = 70; median age 25 years (range 12 to 70 59 years)]. Group II: mycetoma patients [n = 70 patients; median age 25 (range 13 to 70 60 years)]. Cytokines levels were measured in sera using enzyme-linked immunosorbent assay 61 (ELISA). 62 There was no significant correlation between the IL-1β and IL-12 levels and the lesions size 63 and disease duration, whereas levels of IL-37 and IL-35 were significantly correlated with 64 that. The analysis of the risk factors of higher circulatory levels of IL-37 in patients of 65 mycetoma showed a significant negative association with IL-1β cytokine, where a unit 66 increment in IL-1β will decrease the levels of IL-37 by 35.28 pg/ml. The levels of IL-37 67 among the patients with a duration of mycetoma infection ≤ one year had significantly 68 decreased by an average of 18.45 compared to patients with a mycetoma infection's 69 duration of ≥ 5years (reference group). Furthermore, the risk factors of higher levels of IL-35 . CC-BY 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/502609 doi: bioRxiv preprint 70 in mycetoma patients revealed a significant negative association with IL-12, as a unit 71 increment in IL-12 decreases the levels of IL-35by 8.99 pg/ml (p < 0.001). Levels of IL-35 72 among the patients with duration of mycetoma infection ≤ one year had significantly 73 decreased (p-value = 0.002) on average by 41.82 compared to patients with a duration of 74 mycetoma infection ≥ five years (reference group). In conclusion, this study indicates that 75 both IL-35 and IL-37 are negatively associated with the levels of IL-1β and IL-12 in 76 eumycetoma mycetoma infection; and high levels of IL-37 and IL-35 may have a negative 77 impact on disease progression. 78 79 Authors Summary 80 Mycetoma is a progressive chronic granulomatous fungal or bacterial infection that may 81 result in massive destruction of subcutaneous tissues, muscles and bones. Mycetoma is 82 a neglected disease which is endemic in many tropical and subtropical areas. If the 83 disease is not treated properly, eventually it ends up with amputation and adverse 84 medical, health and socioeconomic effects on patients and the community. 85 Previous data suggested a crucial role of adaptive immunity in host resistance to 86 causative agents and the disease progression. The recently identified IL -35 and IL-37 87 cytokines revealed an important role in immune suppression. Nevertheless, the 88 expression of these interleukins in patients with mycetoma has not yet been 89 investigated. Therefore, the present case-control study aimed to determine the levels of 90 IL-1 family (IL-1β, IL-37) and IL-12 family (IL-12, IL-35) in these patients and the 91 association between these cytokines levels and the patients' demographic 92 characteristics. . CC-BY 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/502609 doi: bioRxiv preprint The results of this study showed that the levels of IL-37 and IL-35 were consistently 94 positively correlated with different diameters of mycetoma lesions as well as its duration. 95 However, the levels of IL-1β and IL-12 were consistently negatively correlated with 96 different diameters of lesions and the duration of mycetoma infection. The analysis of 97 the risk factors of higher circulatory levels of IL-37 in patients of mycetoma showed a 98 significant negative association with IL-1β cytokine. Furthermore, the risk factors of 99 higher levels of IL-35 in patients of mycetoma revealed a significant negative 100 association with IL-12. These findings uncover a possible the role of IL-35 and IL-37 in 101 \"},\n"," {'paper_id': 'ad2ac6290bd819dc91338a17298350d0abc17922',\n","  'section': 'Abstract',\n","  'text': 'Objective: To screen out COVID-19 patients in entrying population from abroad with the minimum number of nucleic acid testing (NAT). In the first stage, the nasopharyngeal swab samples of the entrying population are numbered and grouped. After the samples in the group are mixed together, one NAT was performed. When the test result is negative, it shows that all of the peple in the group are not infected and the test for the group is complete. On the contrary, when the test result is positive, the group enters the second stage. In the second stage, all samples in the positive group will be tested one by one using the NAT. The advantages and precautions of the method were discussed. The incidence rate of entrying population is the determinant of the number of samples in the group. The lower the incidence rate, the larger the number of samples in the group, and the greater the percentage saved. The method has obvious efficiency and cost advantages in COVID-19 testing. It can also be used to screen other populations such as community populations, high risk of infection populations. '},\n"," {'paper_id': 'ad357dd76eea58e5a29727e554d3ff431cbade5d',\n","  'section': 'Abstract',\n","  'text': 'Objectives: The 2019 novel coronavirus (COVID-19) has been declared a public health 3 emergency worldwide. The objective of this systematic review was to characterize the clinical, 4 diagnostic, and treatment characteristics of patients presenting with COVID-19. 5 '},\n"," {'paper_id': 'ad4025bd13daeb0f9649b39f8c72cc55c96959ce',\n","  'section': 'Abstract',\n","  'text': '162/150) 39 40 With two-thirds of the global population projected to be living in urban areas by 2050, 41 understanding the transmission patterns of viral pathogens within cities is crucial for effective 42 prevention strategies. Here, in unprecedented spatial resolution, we analysed the socioeconomic 43 determinants of influenza transmission in a European city. We combined geographical and 44 epidemiological data with whole genome sequencing of influenza viruses at the scale of urban 45 quarters and statistical blocks, the smallest geographic subdivisions within a city. We observed 46 annually re-occurring geographic clusters of influenza incidences, mainly associated with net 47 income, and independent of population density and living space. Vaccination against influenza 48 was also mainly associated with household income and was linked to the likelihood of influenza-49 like illness within an urban quarter. Transmissions patterns within and between quarters were 50 complex. High-resolution city-level epidemiological studies combined with social science surveys 51 such as this will be essential for understanding seasonal and pandemic transmission chains and 52 delivering tailored public health information and vaccination programs at the municipal level. '},\n"," {'paper_id': 'ad47f8d46f41eeb6559d89a1addba67699813a6a',\n","  'section': 'Abstract',\n","  'text': 'Objective: To examine risk perceptions and behavioural responses of the UK adult population during the early phase of the COVID-19 epidemic in the UK. Setting: Conducted with a nationally representative sample of UK adults within 48 hours of the UK Government advising the public to stop non-essential contact with others and all unnecessary travel. Participants: 2,108 adults living in the UK aged 18 years and over. Data were collected between March 17 and 18 2020. Descriptive statistics for all survey questions, including the number of respondents and the weighted percentages. Logistic regression was used to identify sociodemographic variation in: (1) adoption of social-distancing measures, (2) ability to work from home, and (3) willingness and (4) ability to self-isolate. Overall, 1,992 (94.2%) respondents reported taking at least one preventive measure: 85.8% washed their hands with soap more frequently; 56.5% avoided crowded areas and 54.5% avoided social events. Adoption of social-distancing measures was higher in those aged over 70 compared to younger adults aged 18 to 34 years (aOR:1.9; 95% CI:1.1 to 3.4). Those with the lowest household income were six times less likely to be able to work from home (aOR:0.16; 95% CI:0.09 to 0.26) and three times less likely to be able to self-isolate (aOR:0.31; 95% CI:0.16 to 0.58). Ability to self-isolate was also lower in black and minority ethnic groups (aOR:0.47; 95% CI:0.27 to 0.82). Willingness to self-isolate was high across all respondents. '},\n"," {'paper_id': 'adb09e4e5c7331b2aa661b3d3bb0a643e00d11bc',\n","  'section': 'Abstract',\n","  'text': 'A novel coronavirus (nCoV-2019) was the cause of an outbreak of respiratory illness detected in Wuhan, Hubei Province, China in December of 2019. Genomic analyses of nCoV-2019 determined a 96% resemblance with a coronavirus isolated from a bat in 2013 (RaTG13); however, the receptor binding motif (RBM) of these two genomes share low sequence similarity. This divergence suggests a possible alternative source for the RBM coding sequence in nCoV-2019. We identified high sequence similarity in the RBM between nCoV-2019 and a coronavirus genome reconstructed from a viral metagenomic dataset from pangolins possibly indicating a more complex origin for nCoV- '},\n"," {'paper_id': 'adb3a6501ad731eda95d0a1a182a793a0dcd58b0',\n","  'section': 'Abstract',\n","  'text': \"Risk of COVID-19 infection in Wuhan has been estimated using imported case counts of international travelers, often under the assumption that all cases in travelers are ascertained. Recent work indicates variation among countries in detection capacity for imported cases. Singapore has historically had very strong epidemiological surveillance and contact-tracing capacity and has shown in the COVID-19 epidemic evidence of a high sensitivity of case detection. We therefore used a Bayesian modeling approach to estimate the relative imported case detection capacity for other countries compared to that of Singapore. We estimate that the global ability to detect imported cases is 38% (95% HPDI 22% -64%) of Singapore's capacity. Equivalently, an estimate of 2.8 (95% HPDI 1.5 -4.4) times the current number of imported cases, could have been detected, if all countries had had the same detection capacity as Singapore. Using the second component of the Global Health Security index to stratify likely case-detection capacities, we found that the ability to detect imported cases relative to Singapore among high surveillance locations is 40% (95% HPDI 22% -67%), among intermediate surveillance locations it is 37% (95% HPDI 18% -68%), and among low surveillance locations it is 11% (95% HPDI 0% -42%). Using a simple mathematical model, we further find that treating all travelers as if they were residents (rather than accounting for the brief stay of some of these travelers in Wuhan) can modestly contribute to underestimation of prevalence as well. We conclude that estimates of case counts in Wuhan based on assumptions of perfect detection in travelers . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. : medRxiv preprint term can be well approximated as We plot this approximation of V given doubling times . V ≈ 1 − e −(r+γ)d aligned with 4 , a range of durations of detectable infection and a range of lengths of stay 2 ( Figure 2 ). We estimate that the global ability to detect imported cases is 38% ( 95% HPDI 22% -64%) of Singapore's capacity. Equivalently, an estimate of 2.8 (95% HPDI 1.5 -4.4) times the current number of imported cases, could have been detected, if all countries had had the same detection capacity as Singapore, which leads to 1.8 (95% CI 0.5 -3.4) undetected cases per detected case. The ability to detect imported cases among high surveillance countries is 40% (95% HPDI 22% -67%), among intermediate surveillance countries it is 37% (95% HPDI 18% -68%), and among low surveillance countries it is 11% (95% HPDI 0% -42%). : medRxiv preprint 7 In this paper, we have aimed to test two assumptions underlying the estimation of incidence at the epicentre of the SARS-Cov2 outbreak. The first of these is that the capacity for detection of international imported cases is 100% sensitive and specific across locations. While we know of no reason to doubt specificity of detection, we tested the assumption of perfect sensitivity. Based on our previous estimates 5 we assumed Singapore has the highest capacity of surveillance with respect to COVID-19. We regressed the cumulative cases against Wuhan-to-location air travel volume considering Singapore to have the greatest detection capacity and estimating the relative underdetection compared to Singapore in the remaining 190 locations classified according to the GHS 2 index 9 . While it is unlikely that this index reflects the true ranking of countries for any specific epidemic, such as the current one of COVID-19, we assume that it captures different levels of surveillance capacity. We therefore grouped the remaining locations into three big classes high, medium, and low surveillance capacity instead of using exact ranking. We estimated that detection of exported cases from Wuhan worldwide is 38% (95% HPDI 22%-64%) as sensitive as it has been in Singapore. Put another way, this implies that the true number of cases in travelers is at least 2.8 (95% HPDI 1.5 -4.4) times the number that has been detected. Equivalently, for each detected case there are at least 1.8 (95% CI 0.5-3.4) undetected cases. If the model is correct, this is an upper bound on the detection frequency because (1) Singapore's detection is probably not 100% efficient. Singapore had as of 12 February 2020 eight \"},\n"," {'paper_id': 'ae27bdd56a8f734f970bae64463b7f68e689f365',\n","  'section': 'Abstract',\n","  'text': 'Background: The number of patients of COVID-19 in Tokyo has been increasing gradually through the end of March, 2020. Object: Support for policymaking requires forecasting of the entire course and outcome of the outbreak including the date of collapse of medical facilities if a lockdown is not initiated. Moreover, the effects of a lockdown must be considered when choosing to initiate one. Method: Data of Tokyo patients with symptoms during January 14 -March 28, 2020 were used to formulate a susceptible-infected-recovered (SIR) model using three age classes and to estimate the basic reproduction number (R 0 ). Based on the estimated R 0 , We inferred outbreak outcomes including the date of collapse of medical facilities if a lockdown were not enacted. Then we estimate the lockdown effects. Results: Results suggest R 0 as 2.86, with a 95% confidence interval of [2.73, 2.97]. Collapse of medical facilities can be expected to occur on April 26 if no lockdown occurs. The total number of deaths can be expected to be half a million people. If a lockdown were enacted from April 6, and if more than 60% of trips outside the home were restricted voluntarily, then a collapse of medical facilities could be avoided. Discussion and Conclusion: The estimated R 0 was similar to that found from other studies conducted in China and Japan. Results demonstrate that a lockdown with reasonable cooperation of residents can avoid a collapse of medical facilities and save 0.25 million mortality cases. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': 'ae61300f4e7a12a68baf114f199eb51940c7aad3',\n","  'section': 'Abstract',\n","  'text': 'To predict the epidemic of COVID-19 based on quarantined surveillance from real world in China by modified SEIR model different from the previous simply mathematical model. We forecasted the epidemic of COVID-19 based on current clinical and epidemiological data and built a modified SEIR model to consider both the infectivity during incubation period and the influence on the epidemic from strict quarantined measures. '},\n"," {'paper_id': 'ae66f22f886d3b019934ae237c11a5fb53dbefb2',\n","  'section': 'Abstract',\n","  'text': 'The virus receptor is key for viral infection of host cells. Identification of the virus receptor is still challenging at present. Our previous study has shown that human virus receptor proteins have some unique features including high level of N-glycosylation, high number of interaction partners and high expressions. Here, we built a random-forest model to identify human virus receptorome from human cell membrane proteins with accepted accuracy based on combination of unique features of human virus receptors and protein sequences. A total of 729 human cell membrane proteins were predicted to constitute the receptorome for the human-infecting virome. Combination of the random-forest model with protein-protein interactions between human and viruses predicted in previous studies further predicted '},\n"," {'paper_id': 'ae6fc64042b050df93ebb8f8045892952f18510f',\n","  'section': 'Abstract',\n","  'text': 'A symptom-based mass screening and testing intervention (MSTI) can identify a large fraction '},\n"," {'paper_id': 'aeb0c3a2994cf6075961a930c84245563aedea3e',\n","  'section': 'Abstract',\n","  'text': 'This is a preprint version, not peer-reviewed yet, and readers should treat it accordingly. 2 Summary Background: The COVID-19 disease has turned into a global pandemic with unprecedented challenges for the global community. Understanding the state of the disease and planning for future trajectories relies heavily on data on the spread and mortality. Yet official data coming from various countries are highly unreliable: symptoms similar to common cold in majority of cases and limited screening resources and delayed testing procedures may contribute to underestimation of the burden of disease. Anecdotal and more limited data are available, but few have systematically combined those with official statistics into a coherent view of the epidemic. This study is a modeling-in-real-time of the emerging outbreak for understanding the state of the disease. Our focus is on the case of the spread of disease in Iran, as one of the epicenters of the disease in the first months of 2020. We develop a simple dynamic model of the epidemic to provide a more reliable picture of the state of the disease based on existing data. Building on the generic SEIR (Susceptible, Exposed, Infected, and Recovered) framework we incorporate two behavioral and logistical considerations. First we capture the endogenous changes in contact rate (average contact per person) as more death are reported. As a result the reproduction number changes endogenously in the model. Second we differentiate reported and true cases by including simple formulations for how only a fraction of cases might be diagnosed, and how that fraction changes in response to epidemic\\'s progression. In estimating the model we use both the official data as well as the discovered infected travelers and unofficial medical community estimates and triangulate these sources to build a more complete picture. Calibration is completed by forming a likelihood function for observing the actual time series data conditional on model parameters, and conducting a Markov Chain Monte Carlo simulations. The model is used to estimate current \"true\" cases of infection and death. We analyze the future trajectory of the disease under six conditions related to the seasonal effects and policy measures targeting social distancing. The model closely replicates the past data but also shows the true number of cases is likely far larger. We estimate about 493,000 current infected cases (90% CI: 271K-810K) as 1 A Persian summary of the article is provided as an appendix. \\u202bاست\\u202c \\u202bموجود\\u202c \\u202bانتها\\u202c \\u202bضمیمه\\u202c \\u202bدر\\u202c \\u202bفارسی\\u202c \\u202bبه\\u202c \\u202bمقاله\\u202c \\u202bخالصه\\u202c '},\n"," {'paper_id': 'af266fac8970a7960e96630a67d91bec5dda0335',\n","  'section': 'Abstract',\n","  'text': 'Estimating key infectious disease parameters from the COVID-19 outbreak is quintessential for modelling studies and guiding intervention strategies. Whereas different estimates for the incubation period distribution and the serial interval distribution have been reported, estimates of the generation interval for COVID-19 have not been provided. Methods: We used outbreak data from clusters in Singapore and Tianjin, China to estimate the generation interval from symptom onset data while acknowledging uncertainty about the incubation period distribution and the underlying transmission network. From those estimates we obtained the proportions pre-symptomatic transmission and reproduction numbers. Results: The mean generation interval was 5.20 (95%CI 3.78-6.78) days for Singapore and 3.95 (95%CI 3.01-4.91) days for Tianjin, China when relying on a previously reported incubation period with mean 5.2 and SD 2.8 days. The proportion of pre-symptomatic transmission was 48% (95%CI 32-67%) for Singapore and 62% (95%CI 50-76%) for Tianjin, China. Estimates of the reproduction number based on the generation interval distribution were slightly higher than those based on the serial interval distribution. Conclusions: Estimating generation and serial interval distributions from outbreak 1 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': 'af5d562d78a7441932170a3c39b5eaea7e75e016',\n","  'section': 'Abstract',\n","  'text': 'The 2019-2020 influenza sentinel surveillance data exhibits unexpected trends. Typical influenza seasons have a small herald wave, followed by a decrease due to school closure during holidays, and then a main post-holiday peak that is significantly larger than the pre-holiday wave. During the 2019-2020 influenza season, influenza-like illness data in the United States appears to have a markedly lower main epidemic peak compared to what would be expected based on the pre-holiday peak. We hypothesize that the 2019-2020 influenza season does have a lower than expected burden and that this deflation is due to a behavioral or ecological interaction with COVID-19. We apply an intervention analysis to assess if this influenza season deviates from expectations, then we compare multiple hypothesized drivers of the decrease in influenza in a spatiotemporal regression model. Lastly, we develop a mechanistic metapopulation model, incorporating transmission reduction that scales with COVID-19 risk perception. We find that the 2019-2020 ILI season is smaller and decreases earlier than expected based on prior influenza seasons, and that the increase in COVID-19 risk perception is associated with this decrease. Additionally, we find that a 5% average reduction in transmission is su cient to reproduce the observed flu dynamics. We propose that precautionary behaviors driven by COVID-19 risk perception or increased isolation driven by undetected COVID-19 spread dampened the influenza season. We suggest that when surveillance for a novel pathogen is limited, surveillance streams of co-circulating infections may provide a signal. '},\n"," {'paper_id': 'af7d038be406d73b373f05a9cdfaf62d98e26cbb',\n","  'section': 'Abstract',\n","  'text': \"We evaluated how non-pharmaceutical interventions could be used to control the COVID-19 pandemic and reduce the burden on the healthcare system. Using an age-structured compartmental model of COVID-19 transmission in the population of Ontario, Canada, we compared a base case with limited testing, isolation, and quarantine to scenarios with: enhanced case finding; restrictive social distancing measures; or a combination of enhanced case finding and less restrictive social distancing. Interventions were either implemented for fixed durations or dynamically cycled on and off, based on projected ICU bed occupancy. We present median and credible intervals (CrI) from 100 replicates per scenario using a two-year time horizon. We estimated that 56% (95% CrI: 42-63%) of the Ontario population would be infected over the course of the epidemic in the base case. At the epidemic peak, we projected 107,000 (95% CrI: 60,760-149,000) cases in hospital and 55,500 (95% CrI: 32,700-75,200) cases in ICU. For fixed duration scenarios, all interventions were projected to delay and reduce the height of the epidemic peak relative to the base case, with restrictive social distancing estimated to have the greatest effect. Longer duration interventions were more effective. Dynamic interventions were projected to reduce the proportion of the population infected at the end of the two-year period. Dynamic social distancing interventions could reduce the median number of cases in ICU below current estimates of Ontario's ICU capacity. Interpretation: Without significant social distancing or a combination of moderate social distancing with enhanced case finding, we project that ICU resources would be overwhelmed. Dynamic social distancing could maintain health system capacity and also allow periodic psychological and economic respite for populations. \"},\n"," {'paper_id': 'b026bbd2c2afbbf5f36cfdae973657aaca4153a7',\n","  'section': 'Abstract',\n","  'text': 'COVID-19, the first pandemic of this decade and the second in less than 15 years, has 15 harshly taught us that viral diseases do not recognize boundaries; however, they truly do 16 discriminate between aggressive and mediocre containment responses. 17 We present a simple epidemiological model that is amenable to implementation in Excel 18 spreadsheets and sufficiently accurate to reproduce observed data on the evolution of the 19 COVID-19 pandemics in different regions (i.e., Italy, Spain, and New York City (NYC)). 20 We also show that the model can be adapted to closely follow the evolution of COVID-19 21 in any large city by simply adjusting two parameters related to (a) population density and 22 (b) aggressiveness of the response from a society/government to epidemics. Moreover, we 23 show that this simple epidemiological simulator can be used to assess the efficacy of the 24 response of a government/society to an outbreak. 25 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': 'b04157f0041a2404b8c0d8403e5fe9313488f8f1',\n","  'section': 'Abstract',\n","  'text': 'XQ(2); Lee WL(2); Kauffman K (3); Hanage WP(4); Matus M (5); Ghaeli N(5); Endo N(5); Duvallet C(5); Moniz K(1); Erickson TB(6); Chai PR (6); Thompson J(7); Alm EJ(1,2,5) Abstract. Wastewater surveillance may represent a complementary approach to measure the presence and even prevalence of infectious diseases when the capacity for clinical testing is limited. Moreover, aggregate, population-wide data can help inform modeling efforts. We tested wastewater collected at a major urban treatment facility in Massachusetts and found the presence of SARS-CoV-2 at high titers in the period from March 18 -25 using RT-qPCR. We then confirmed the identity of the PCR product by direct DNA sequencing. Viral titers observed were significantly higher than expected based on clinically confirmed cases in Massachusetts as of March 25. The reason for the discrepancy is not yet clear, and until further experiments are complete, these data do not necessarily indicate that clinical estimates are incorrect. Our approach is scalable and may be useful in modeling the SARS-CoV-2 pandemic and future outbreaks. . CC-BY 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': 'b07bb8f6466cb60a903182960e1e57f5cdbc7780',\n","  'section': 'Abstract',\n","  'text': 'Many countries have banned groups and gatherings as part of their response to the pandemic caused by the coronavirus, SARS-CoV-2. Although there are outbreak reports involving mass gatherings, the contribution to overall transmission is unknown. We used data from a survey of social contact behaviour that specifically asked about contact with groups to estimate the Population Attributable Fraction (PAF) due to groups as the relative change in the Basic Reproduction Number when groups are prevented. We estimate that PAF due to groups of 50+ people is 2.2% (95%CI 1.1%, 3.6%); the PAF due to groups of 20+ people is 6.4% (5.0%, 8.0%); the PAF due to groups of 10+ is 11.3% (9.9%, 13.0%) Interpretation Large groups of individuals have a small epidemiological impact; small and medium sized groups between 10 and 50 people have a larger impact on an epidemic. '},\n"," {'paper_id': 'b0899a264af548a89d649154aec569889717b295',\n","  'section': 'Abstract',\n","  'text': \"Predicting the effect of different programmes designed to control both the morbidity induced by helminth infections and parasite transmission is greatly facilitated by the use of mathematical models of transmission and control impact. In such models, it is essential to account for as many sources of uncertainty-natural, or otherwise -to ensure robustness in prediction and to accurately depict variation around an expected outcome. In this paper, we investigate how well the standard deterministic models match the predictions made using individual-based stochastic simulations. We also explore how well concepts which derive from deterministic models, such as 'breakpoints' in transmission, apply in the stochastic world. Employing an individual based stochastic model framework we also investigate how transmission and control are affected by the migration of infected people into a defined community. To give our study focus we consider the control of soil-transmitted helminths (STH) by mass drug administration (MDA), though our methodology is readily applicable to the other helminth species such as the schistosome parasites and the filarial worms. We show it is possible to define a 'stochastic breakpoint' where much noise surrounds the expected deterministic breakpoint. We also discuss the concept of the 'interruption of transmission' independent of the 'breakpoint' concept where analyses of model behaviour illustrate the current limitations of deterministic models to account for the 'fade-out' or transmission extinction behaviour in simulations. The analyses based on migration confirm a relationship between the infected human migration rate per unit of time and the death rate of infective stages that are released into the free-living environment (eggs or larvae depending on the STH species) that create the reservoir of infection which in turn determines the likelihood that control activities aim at chemotherapeutic treatment of the human host will eliminate transmission. The development of a new stochastic simulation code for STH in the form of a publicly-available open-source python package which includes features to incorporate many population stratifications, different control interventions including mass drug administration (with defined frequency, coverage levels and compliance patterns) and inter-village human migration is also described. : medRxiv preprint Following past publications [12, 23] , the STH transmission model we introduce here is a stochastic individual-based analogue to the deterministic original given in Refs. [26, 11, 7] . : medRxiv preprint ∂ ∂Z G jn [Z(t), t] = λ jn Λ j (t) [Z(t) − 1] G jn [Z(t), t] , whose solution is given by Eq. (A.4) when one replaces Z(t ) − 1 with the inverted form of the equation above. \"},\n"," {'paper_id': 'b0d841f7331c175952da753a086795a5287c61e0',\n","  'section': 'Abstract',\n","  'text': 'A mathematical model is developed to study the spread of the COVID-19 epidemics in France. Data from French Public Agency of Health are considered to calibrate the model. The spread of the epidemics strongly depends on confinement measures. The aim of the paper is to predict the evolution of the epidemics under various scenarios that could be taken to quit confinement. The spread of the disease and its re-emergence strongly depends on these scenarios. '},\n"," {'paper_id': 'b117016a887f05386acc4387dab5bf8c769cf90e',\n","  'section': 'Abstract',\n","  'text': 'An ongoing outbreak of atypical pneumonia caused by the 2019 novel coronavirus 2) is hitting Wuhan City and has spread to other provinces/cities of China and overseas. It very 29 urgent to forecast the future course of the outbreak. Here, we provide an estimate of the potential 30 total number of confirmed cases in mainland China by applying Boltzmann-function based 31 regression analyses. We found that the cumulative number of confirmed cases from Jan 21 to Feb 32 14, 2020 for mainland China, Hubei Province, Wuhan City and other provinces were all well 33 fitted with the Boltzmann function (R 2 being close to 0.999). The potential total number of 34 confirmed cases in the above geographic regions were estimated at 95% confidence interval (CI) 35 as 79589 (71576, 93855), 64817 (58223, 77895), 46562 (40812, 57678) and 13956 (12748, 36 16092), respectively. Notably, our results suggest that the number of daily new confirmed cases '},\n"," {'paper_id': 'b12f20ae397aa504f852fc25caeb3e3b0e5b1942',\n","  'section': 'Abstract',\n","  'text': 'Coronavirus(COVID-19) registered in the China Clinical Trial Registry (ChiCTR), and provide data bases and information references for clinical treatment '},\n"," {'paper_id': 'b1a5096c2bfbb4ab9826c36e3961929a744134ca',\n","  'section': 'Abstract',\n","  'text': 'Generation intervals, defined as the time between when an individual is infected and when 14 that individual infects another person, link two key quantities that describe an epidemic: 15 the reproductive number, R, and the rate of exponential growth, r. Generation intervals 16 are often measured through contact tracing by identifying who infected whom. We study 17 how observed intervals differ from \"intrinsic\" intervals that could be estimated by tracing 18 individual-level infectiousness, and identify both spatial and temporal effects, including cen-19 soring (due to observation time), and the effects of susceptible depletion at various spatial 20 scales. Early in an epidemic, we expect the variation in the observed generation intervals to 21 be mainly driven by the censoring and the population structure near the source of disease 22 spread; therefore, we predict that correcting observed intervals for the effect of temporal 23 censoring but not for spatial effects will provide a spatially informed \"effective\" generation-24 interval distribution, which will correctly link r and R. We develop and test statistical 25 methods for temporal corrections of generation intervals, and confirm our prediction using 26 individual-based simulations on an empirical network. An epidemic can be characterized by the exponential growth rate, r, and the reproductive 32 number, R. The reproductive number, defined as the average number of secondary cases 33 arising from a primary case, is of particular interest as it provides information about the 34 final size of an epidemic (1; 2). However, estimating the reproductive number directly from 35 disease natural history requires detailed knowledge which is not often available, particularly 36 early in an outbreak (3). Instead, the reproductive number can be indirectly estimated from 37 the exponential growth rate, which can be estimated from incidence data (4; 5; 6; 7; 8). These two quantities are linked by generation-interval distributions (9; 10; 11; 12; 13). 39 A generation interval is defined as the time between when a person becomes infected and 40 when that person infects another person (10). Due to individual variation in infection time, 41 the observed generation-interval distribution can change depending on when and how it is 42 measured (10; 14; 15; 16). There are important distinctions to be made when estimating 43 generation intervals: intrinsic generation intervals measure the infectiousness of an infected 44 individual, while observed generation intervals refer to the time between actual infection 45 events. 46 While the intrinsic generation-interval distribution is often assumed to be fixed, the 47 shape of the observed generation-interval distribution depends on when and how intervals 48 are measured (14; 15; 17; 16; 18). When an epidemic is growing exponentially, as often occurs 49 near the beginning of an outbreak, the number of newly infected individuals will be large 50 relative to the number infected earlier on. A susceptible individual is thus relatively more 51 likely to be infected by a newly infected individual. Thus, \"backward\" generation intervals 52 (which look at a cohort of individuals, and ask when their infectors were infected) will be 53 shorter on average than intrinsic generation intervals -the converse is true when an epidemic 54 is subsiding (15; 16; 18). In practice, generation intervals are often measured by aggregating available information 56 from contact tracing. While an epidemic is ongoing, these measurements are \"censored\": we 57 don\\'t know what happens after the observation time. These censored intervals are similar 58 to backward intervals: there is a bias towards observing shorter intervals, which are more 59 likely to have concluded by the observation time. Realized generation intervals are also affected by spatial structure. In a population 61 that does not mix homogeneously, susceptibility will tend to decrease more quickly in the 62 neighbourhood of infected individuals than in the general population. This means that 63 contacts made late in an individual\\'s infection are more likely to be ineffective due to contacts 64 that were made earlier (because the contactee may have been infected already) As a result, 65 realized generation intervals will have shorter mean than the intrinsic generation-interval 66 distribution. This perspective allows us to reinterpret the finding of (19) that, given an 67 intrinsic generation interval and an observed growth rate, the reproductive number on various 68 network structures is always smaller on a network than would be predicted from homogeneous 69 mixing. These observations allow us to make the following prediction: removing the time-70 censoring bias from observed generation intervals will yield a spatially corrected \"effective\" 71 distribution that contains information about the population structure and will allow us to 72 2 '},\n"," {'paper_id': 'b1a8aaa839328820b23cfa0de1d6427ebc0fb2ef',\n","  'section': 'Abstract',\n","  'text': 'Severe Acute Respiratory Syndrome coronavirus 2 (SARS-CoV-2) causes Coronavirus disease 2019 (COVID-19), a respiratory tract infection. The standard molecular diagnostic test is a multistep process involving viral RNA extraction and real-time quantitative reverse transcriptase PCR (qRT-PCR). Laboratories across the globe face constraints on equipment and reagents during the COVID-19 pandemic. We have developed a simplified qRT-PCR assay that removes the need for an RNA extraction process and can be run on a real-time thermal cycler. The assay uses custom primers and probes, and maintains diagnostic sensitivity within 98.0% compared to the assay run on a high-throughput, random-access automated platform, the Panther Fusion (Hologic). This assay can be used to increase capacity for '},\n"," {'paper_id': 'b1b9d81726e56c2781ce6b63ae5015cf2f314c6f',\n","  'section': 'Abstract',\n","  'text': 'Background: Since the 2019-nCoV outbreaks in Wuhan, China, the cumulative number of confirmed cases is increasing every day, and a large number of populations all over the world are at risk. The quarantine and traffic blockage can alleviate the risk of the epidemic and the infections, henceforth evaluating the efficacy of such actions is essential to inform policy makers and raise the public awareness of the importance of self-isolation and quarantine. We collected confirmed case data and the migration data, and introduced the quarantine factor and traffic blockage factor to the Flow-SEIR model. By varying the quarantine factor and traffic blockage factor, we simulated the change of the peak number and arrival time of infections, then the efficacy of these two intervation measures can be analyzed in our simulation. In our study, the self-protection at home is also included in quarantine. In the simulated results, the quarantine and traffic blockage are effective for epidemic control. For Hubei province, the current quarantine factor is estimaed to be 0.405, which means around 40.5% of suceptibles who are close contacting with are in quarantine, and the current traffic blockage factor is estimaed to be 0.66, which indicates around 34% of suceptibles who had flowed out from Hubei. For the other provinces outside Hubei, the current quarantine factor is estimated to be 0.285, and the current traffic blockage factor is estimated to be 0.26. With the quarantine and traffic blockage factor increasing, the number of infections decrease dramatically. We also simulated the start dates of quarantine and traffic blockage at four time points, the simulated results show that the early of warning is also effective for epidemic containing. However, provincial level traffic blockage can only alleviate 21.06% -22.38% of the peak number of infections. In general, the quarantine is much more effective than the traffic blockage control. '},\n"," {'paper_id': 'b1ce6ca5960cb799403188ca303a364428f812d8',\n","  'section': 'Abstract',\n","  'text': \"Manipulation of natural mosquito populations using the endosymbiotic bacteria Wolbachia is being 21 investigated as a novel strategy to reduce the burden of mosquito-borne viruses. To evaluate the 22 efficacy of these interventions, it will be critical to determine Wolbachia infection frequencies in 23 Aedes aegypti mosquito populations. However, current diagnostic tools are not well-suited to fit this 24 need. Morphological methods cannot identify Wolbachia, immunoassays often suffer from low 25 sensitivity and poor throughput, while PCR and spectroscopy require complex instruments and 26 technical expertise, which restrict their use to centralized laboratories. To address this unmet need, 27 we have used loop-mediated isothermal amplification (LAMP) and oligonucleotide strand 28 displacement (OSD) probes to create a one-pot sample-to-answer nucleic acid diagnostic platform 29 for vector and symbiont surveillance. LAMP-OSD assays can directly amplify target nucleic acids 30 from macerated mosquitoes without requiring nucleic acid purification and yield specific single 31 endpoint yes/no fluorescence signals that are observable to eye or by cellphone camera. We 32 demonstrate cellphone-imaged LAMP-OSD tests for two targets, the Aedes aegypti cytochrome 33 oxidase I (coi) gene and the Wolbachia surface protein (wsp) gene, and show a limit of detection 34 of 4 and 40 target DNA copies, respectively. In a blinded test of 90 field-caught mosquitoes, the 35 coi LAMP-OSD assay demonstrated 98% specificity and 97% sensitivity in identifying Ae. aegypti 36 mosquitoes even after 3 weeks of storage without desiccant at 37 °C. Similarly, the wsp LAMP-37 OSD assay readily identified the wAlbB Wolbachia strain in field-collected Aedes albopictus 38 mosquitoes without generating any false positive signals. Modest technology requirements, 39 minimal execution steps, simple binary readout, and robust accuracy make the LAMP-OSD-to-40 cellphone assay platform well suited for field vector surveillance in austere or resource-limited 41 conditions. 42 43 All rights reserved. No reuse allowed without permission. Mosquitoes spread many human pathogens and novel approaches are required to reduce the 45 burden of mosquito-borne disease. One promising approach is transferring Wolbachia into 46 Aedes aegypti mosquitoes where it blocks transmission of arboviruses like dengue, Zika and 47 Yellow fever viruses and spreads through mosquito populations. For effective evaluation of this 48 approach, regular surveillance of Wolbachia infections in Ae. aegypti is required, but current 49 diagnostic tools are not well suited to support these critical surveillance needs. To fill this need 50 we developed a simple, robust and inexpensive assay to identify Ae. aegypti mosquitoes and 51 Wolbachia using our unique one-pot assay platform, LAMP-OSD, which uses loop-mediated 52 isothermal amplification to amplify nucleic acid targets at a single temperature. Unlike other 53 LAMP-based tests, our assays assure accuracy by coupling amplification with novel nucleic acid 54 strand displacement (OSD) probes that hybridize to specific sequences in LAMP amplification 55 products and thereby generate simple yes/no readout of fluorescence readable by human eye 56 and by off-the-shelf cellphones. To facilitate field use, we developed our assays so they are 57 compatible with crushed mosquito homogenate as the template, meaning no nucleic acid 58 extraction is required. In blinded tests using field collected mosquitoes, LAMP-OSD-cellphone 59 tests performed robustly to identify 29 of 30 Ae. aegypti even after 3 weeks of storage at 37 °C 60 while producing only one false positive out of 60 non-specific mosquitoes. Similarly, our assay 61 could identify Wolbachia in field-caught Aedes albopictus without producing any false positives. 62 Our easy to use and easy to interpret assays should facilitate widespread field mosquito 63 surveillance with minimal instrumentation and high accuracy. 64 65 66 All rights reserved. No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/291849 doi: bioRxiv preprint diseases [1]. Traditionally considered a problem for tropical regions, mosquitoes are increasingly 69 becoming a global public health challenge [2, 3] due to a changing global environment, urbanization, 70 increases in the global movement of populations, and the emergence of insecticide resistance [4]. 71 Estimates suggest nearly half the world's population is at risk for mosquito-borne diseases [5, 6], 72 and as such, there is an urgent need for novel approaches to reduce the burden of disease. All rights reserved. No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/291849 doi: bioRxiv preprint 4 traditional morphological-based approaches cannot detect associated symbionts or pathogens. 91 These limitations restrict widespread accessibility and necessitate sample preservation and 92 transport. On the other end of the spectrum, immunoassay-based tools for identifying pathogen-93 infected mosquitoes, such as VecTest TM dipsticks (Medical Analysis Systems Inc.), are portable 94 and inexpensive. However, these tests have poor sensitivity [23-25] and are not necessarily 95 available for distinguishing mosquito species or identifying Wolbachia endosymbionts. 96 Nucleic acid tests can provide the necessary sensitivity and versatility for identifying both 97 Wolbachia and mosquito species. However, since molecular testing is currently heavily reliant on 98 PCR [26-28], opportunities for field-based determinations are limited, leading to significant delays 99 and gaps in actionable surveillance. To support widespread vector surveillance inexpensive, 100 portable, nucleic acid diagnostic platforms are needed that rapidly produce accurate results without 101 requiring complex procedures, instruments, and laboratory infrastructure. In this regard, isothermal 102 nucleic acid amplification assays such as loop-mediated isothermal amplification (LAMP) have 103 begun to be employed because they do not require complex thermocycling instruments [29][30][31]. 104 However, although LAMP can rival PCR for sensitivity it often produces spurious amplicons, which 105 in turn lead to false positive readouts with non-specific reporters such as Mg 2+ precipitation or 106 fluorescent dye intercalation [32][33][34]. 107 To mitigate the spurious signals that arise with LAMP, we have previously applied principles that 108 were developed for nucleic acid strand exchange circuits [35][36][37][38] to the design of short hemiduplex 109 oligonucleotide strand displacement (OSD) probes for LAMP [39]. The single stranded 'toehold' 110 regions of OSD probes bind to LAMP amplicon loop sequences, and then signal via strand 111 exchange [40] that leads to separation of a fluorophore and quencher [39]. OSDs are the 112 functional equivalents of TaqMan probes and can specifically report single or multiplex LAMP 113 amplicons without interference from non-specific nucleic acids or inhibitors [39, 41]. OSDs 114 significantly enhance the diagnostic applicability of LAMP, allowing it to match the allelic specificity 115 All rights reserved. No reuse allowed without permission. \"},\n"," {'paper_id': 'b1e7752cbf528db99723a3a67b49b9b5c6e449af',\n","  'section': 'Abstract',\n","  'text': 'Host shifts, where a pathogen invades and establishes in a new host species, are a major source of 17 emerging infectious diseases. They frequently occur between related host species and often rely on 18 the pathogen evolving adaptations that increase their fitness in the novel host species. To 19 investigate genetic changes in novel hosts, we experimentally evolved replicate lineages of an RNA 20 virus (Drosophila C Virus) in 19 different species of Drosophilidae and deep sequenced the viral 21 genomes. We found a strong pattern of parallel evolution, where viral lineages from the same host 22 were genetically more similar to each other than to lineages from other host species. When we 23 compared viruses that had evolved in different host species, we found that parallel genetic changes 24 were more likely to occur if the two host species were closely related. This suggests that when a 25 virus adapts to one host it might also become better adapted to closely related host species. This 26 may explain in part why host shifts tend to occur between related species, and may mean that when 27 a new pathogen appears in a given species, closely related species may become vulnerable to the 28 new disease. '},\n"," {'paper_id': 'b21be53f027cbee5bfb70762cb7b008d9cdf78f5',\n","  'section': 'Abstract',\n","  'text': 'The reasons for a wide variation in severity of coronavirus disease 2019 (COVID-19) across the affected countries of the world are not known. Two recent studies have suggested a link between the BCG vaccination policy and the morbidity and mortality due to COVID-19. In the present study we compared the impact of COVID-19 in terms of case fatality rates (CFR) between countries with high disease burden and those with BCG revaccination policies presuming that revaccination practices would have provided added protection to the population against severe COVID-19. We found a significant difference in the CFR between the two groups of countries. Our data further supports the view that universal BCG vaccination has a protective effect on the course of COVID-19 probably preventing progression to severe disease and death. Clinical trials of BCG vaccine are urgently needed to establish its beneficial role in COVID-19 as suggested by the epidemiological data, especially in countries without a universal BCG vaccination policy. '},\n"," {'paper_id': 'b23ef04fdeffdf21cf7c32a977c96d5b6a2c194c',\n","  'section': 'Abstract',\n","  'text': 'COVID-19 is spreading all across the globe. Up until March 23, 2020, the confirmed cases in 173 12 countries and regions of the globe had surpassed 346,000, and more than 14,700 deaths had resulted. The 13 confirmed cases outside of China had also reached over 81,000, with over 3,200 deaths. In this study, a 14 Convolutional Neural Network (CNN) was proposed to analyze and predict the number of confirmed cases. 15 Several cities with the most confirmed cases in China were the focus of this study, and a COVID-19 16 forecasting model, based on the CNN deep neural network method, was proposed. To compare the overall 17 efficacies of different algorithms, the indicators of mean absolute error and root mean square error were 18 applied in the experiment of this study. The experiment results indicated that compared with other deep 19 learning methods, the CNN model proposed in this study has the greatest prediction efficacy. The feasibility 20 and practicality of the model in predicting the cumulative number of COVID-19 confirmed cases were also 21 verified in this study. 22 '},\n"," {'paper_id': 'b244c4ca5ceaf0e609cfcefe8f6bf471a9cb279a',\n","  'section': 'Abstract',\n","  'text': 'Background: Italy is bearing the brunt of the COVID-19 pandemic, as the death toll there has already surpassed that in Wuhan, the city in China where the coronavirus emerged in December 2019. Here we employ statistical methods to assess the severity of COVID-19 pandemic across different regions of Italy. We manually retrieved the daily cumulative numbers of laboratory-confirmed cases and deaths attributed to COVID-19 stratified by region in Italy from March 2, 2020 to March 28, 2020. We estimated both the crude and time-delay adjusted case fatality ratio across five geographic regions of Italy: Northwest, Northeast, Center, South and Island. : The Northwest that includes Lombardy exhibited the highest time-delay-adjusted CFR at 31.4%% [95% credible interval: 30.8, 32.1%] followed by the Northeast (20.3%) [95% CrI: 19.4, 21.1], the Center region (16.5%) [95% CrI: 15.4, 17.7] and the South region (16.8%) [95% CrI: 15.0, 18.7]. The Island region has exhibited the lowest CFR (11.8%) [95% CrI: 9.0, 14.8]. Conclusion: Italy is experiencing one of the highest CFR from the COVID-19 pandemic in the world. Time delay adjusted CFR in all the regions in Italy except the Island region were higher than that estimated for Wuhan, China. The CFR in Northwest Italy is 2.5-fold higher than that in Wuhan. Our finding reflects the need of urgent medical support in the Northwest region and the appropriate planning and supplies procurement in other regions of Italy with . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': 'b2668487afe5796f5a35f8c5f8c49827dc87c2bd',\n","  'section': 'Abstract',\n","  'text': 'COVID-19 pandemic has become a major threat to the country. Till date, well tested medication or antidote is not available to cure this disease. According to WHO reports, COVID-19 is a severe acute respiratory syndrome which is transmitted through respiratory droplets and contact routes. Analysis of this disease requires major attention by the Government to take necessary steps in reducing the effect of this global pandemic. In this study, outbreak of this disease has been analysed for India till 30 th March 2020 and predictions have been made for the number of cases for the next 2 weeks. SEIR model and Regression model have been used for predictions based on the data collected from John Hopkins University repository in the time period of 30 th January 2020 to 30 th March 2020. The performance of the models was evaluated using RMSLE and achieved 1.52 for SEIR model and 1.75 for the regression model. The RMSLE error rate between SEIR model and Regression model was found to be 2.01. Also, the value of R 0 which is the spread of the disease was calculated to be 2.02. Expected cases may rise between 5000-6000 in the next two weeks of time. This study will help the Government and doctors in preparing their plans for the next two weeks. Based on the predictions for short-term interval, these models can be tuned for forecasting in long-term intervals. '},\n"," {'paper_id': 'b2de691570027e843e3af655efc42bf87ac56c7f',\n","  'section': 'Abstract',\n","  'text': 'Background: Current outbreaks of COVID-19 are threatening the health care systems of several countries around the world. Control measures, based on isolation and quarantine, have been shown to decrease and delay the burden of the ongoing epidemic. With respect to the ongoing COVID-19 epidemic, recent modelling work shows that this intervention technique may be inadequate to control local outbreaks, even when perfect isolation is assumed. Furthermore, the effect of infectiousness prior to symptom onset combined with a significant proportion of asymptomatic infectees further complicates the use of contact tracing. Antivirals, which decrease the viral load and reduce the infectiousness, could be integrated in the control measures in order to augment the feasibility of controlling the epidemic. Methods: Using a simulation-based model of viral transmission we tested the efficacy of different intervention measures for the control of COVID-19. For individuals that were identified through contact tracing, we evaluate two procedures: monitoring individuals for symptoms onset and testing of individuals. Moreover, we investigate the effect of a potent antiviral compound on the contact tracing process. Findings: The use of an antiviral drug, in combination with contact tracing, quarantine and isolation, results in a significant decrease of the final size, the peak incidence, and increases the probability that the outbreak will fade out. Interpretation: For an infectious disease in which presymptomatic infections are plausible, an intervention measure based on contact tracing performs better when realized together with testing instead of monitoring, provided that the test is able to detect infections during the incubation period. In addition, in all tested scenarios, the model highlights the benefits of the administration of an antiviral drug in addition to quarantine, isolation and contact tracing. The resulting control measure, could be an effective strategy to control local and re-emerging outbreaks of COVID-19. '},\n"," {'paper_id': 'b30b43ac5cf19932f2b3c3034e2f4ea34a69e901',\n","  'section': 'Abstract',\n","  'text': \"Analysis of genetic sequence data from the pandemic SARS Coronavirus 2 can provide 18 insights into epidemic origins, worldwide dispersal, and epidemiological history. With few 19 exceptions, genomic epidemiological analysis has focused on geographically distributed data sets 20 with few isolates in any given location. Here we report an analysis of 20 whole SARS-CoV 2 21 genomes from a single relatively small and geographically constrained outbreak in Weifang, 22 People's Republic of China. Using Bayesian model-based phylodynamic methods, we estimate the 23 reproduction number for the outbreak to be 2.6 (95% CI:1.5-5). We further estimate the number of 24 infections through time and compare these estimates to confirmed diagnoses by the Weifang 25 Centers for Disease Control. We find that these estimates are consistent with reported cases and 26 there is unlikely to be a large undiagnosed burden of infection over the period we studied. 27 28 194 We gratefully acknowledge China National GeneBank at Shenzhen, China for the sequencing strategy 195 and capacity support. We also gratefully acknowledge the laboratories that have contributed \"},\n"," {'paper_id': 'b323d6bf1873bf0b94bcae0be2052d9e335bf1af',\n","  'section': 'Abstract',\n","  'text': 'The COVID-19 pandemic demands assimilation of all available biomedical knowledge to decode its mechanisms of pathogenicity and transmission. Despite the recent renaissance in unsupervised neural networks for decoding unstructured natural languages, a platform for the real-time synthesis of the exponentially growing biomedical literature and its comprehensive triangulation with deep omic insights is not available. Here, we present the nferX platform for dynamic inference from over 45 quadrillion possible conceptual associations extracted from unstructured biomedical text, and their triangulation with Single Cell RNA-sequencing based insights from over 25 tissues. Using this platform, we identify intersections between the pathologic manifestations of COVID-19 and the comprehensive expression profile of the SARS-CoV-2 receptor ACE2. We find that tongue keratinocytes, airway club cells, and ciliated cells are likely underappreciated targets of SARS-CoV-2 infection, in addition to type II pneumocytes and olfactory epithelial cells. We further identify mature small intestinal enterocytes as a possible hotspot of COVID-19 fecal-oral transmission, where an intriguing maturation-correlated transcriptional signature is shared between ACE2 and the other coronavirus receptors DPP4 (MERS-CoV) and ANPEP (ɑ-coronavirus). This study demonstrates how a holistic data science platform can leverage unprecedented quantities of structured and unstructured publicly available data to accelerate the generation of impactful biological insights and hypotheses. The nferX Platform Single-cell resource -https://academia.nferx.com/ '},\n"," {'paper_id': 'b3a334560f9a7a4b35a60415c87d41d658a89ebb',\n","  'section': 'Abstract',\n","  'text': 'Severe acute respiratory syndrome coronavirus 2 (SARS-COV-2), previously known as 2019-nCoV, is responsible for the atypical pneumonia pandemic designated as Coronavirus Disease 2019 . The number of cases continues to grow exponentially reaching 492,000 people in 175 countries as of March 25, 2020. 22,169 people (~4.5%) infected with SARS-COV-2 virus have died. We have developed an exponential regression model using the COVID-19 case data (Jan 22 -Mar 22, 2020). Our primary model uses designated Phase 1 countries, who exceed 2500 cases on Mar 22. The model is then applied to Phase 2 countries: those that escaped the initial Phase 1 global expansion of COVID-19. With the exception of stabilizing countries (South Korea, Japan, and Iran) all Phase 1 countries are growing exponentially, as per 2500 ( ) = 120.4 × 0.238 , with a rate, r = 0.238 ± 0.068. Excluding China, the BRICS developing nations and Australia are in Phase 2. Case data from Phase 2 countries are following the model derived from Phase 1 countries. In the absence of measures employed to flatten the curve including social distancing, quarantine, and healthcare expansion, our model projects over 274,000 cases and 12,300 deaths in the US by Mar 31. India can expect 123,000 cases by April 16. By flattening the curve to the growth rate of stabilizing countries (r = 0. '},\n"," {'paper_id': 'b3b84e6844412c5aba86d285a15a9bca706c3fe8',\n","  'section': 'Abstract',\n","  'text': 'bioRxiv preprint TATA box, CCAAT box, GC box and DNA stability are shown. Blue and red boxes indicate positions of those putative elements and related transcription factors predicted by GPMiner. A lack of CCAAT box was observed in both bats. '},\n"," {'paper_id': 'b3bad2ef157e371fea5d06500f90c0f1a74d95f2',\n","  'section': 'Abstract',\n","  'text': '42 The impact of COVID-19 has been profound, and the public health challenge 44 seem to be the most serious regarding respiratory viruses since the 1918 H1N1 45 influenza pandemic. In the absence of effective vaccine or biomedical treatment, the 46 basic rules of public health measures have not changed, namely public distancing. 47 '},\n"," {'paper_id': 'b3cb1d91dcd7f78dd9bbc88b93d2403eb6869746',\n","  'section': 'Abstract',\n","  'text': \"High-density lipoproteins (HDL) are nanoparticles with >80 associated proteins, phospholipids, cholesterol and cholesteryl esters. A comprehensive genetic analysis of the regulation of proteome of HDL isolated from a panel of 100 diverse inbred strains of mice, Hybrid Mouse Diversity Panel (HMDP), revealed widely varied HDL protein levels across the strains. Some of this variation was explained by local, cis-acting regulation, termed cis-protein quantitative trait loci. Variations in apolipoprotein A-II and apolipoprotein C-3 affected the abundance of multiple HDL proteins indicating a coordinated regulation. We identified modules of co-varying proteins and define a protein-protein interaction network describing the protein composition of the naturally occurring subspecies of HDL in mice. Sterol efflux capacity varied up to 3-fold across the strains and HDL proteins displayed distinct correlation patterns with macrophage and ABCA1 specific cholesterol efflux capacity and cholesterol exchange, suggesting that subspecies of HDL participate in discrete functions. The baseline and stimulated sterol efflux capacity phenotypes associated with distinct QTLs with smaller effect size suggesting a multi genetic regulation. Our results highlight the complexity of HDL particles by revealing high degree of heterogeneity and intercorrelation, some of which is associated with functional variation, supporting the concept that HDL-cholesterol alone is not an accurate measure of HDL's properties such as protection against CAD. \"},\n"," {'paper_id': 'b3e877acb1b2ec719f6e094f7900cc3bf865457c',\n","  'section': 'Abstract',\n","  'text': 'The emergence of SARS-CoV-2 has resulted in more than 200,000 infections and nearly 9,000 deaths globally so far. This novel virus is thought to have originated from an animal reservoir, and acquired the ability to infect human cells using the SARS-CoV cell receptor hACE2. In the wake of a global pandemic it is essential to improve our understanding of the evolutionary dynamics surrounding the origin and spread of a novel infectious disease. One way theory predicts selection pressures should shape viral evolution is to enhance binding with host cells. We first assessed evolutionary dynamics in select betacoronavirus spike protein genes to predict where these genomic regions are under directional or purifying selection between divergent viral lineages at various scales of relatedness. With this analysis, we determine a region inside the receptor-binding domain with putative sites under positive selection interspersed among highly conserved sites, which are implicated in structural stability of the viral spike protein and its union with human receptor hACE2. Next, to gain further insights into factors associated with coronaviruses recognition of the human host receptor, we performed modeling studies of five different coronaviruses and their potential binding to hACE2. Modeling results indicate that interfering with the salt bridges at hot spot 353 could be an effective strategy for inhibiting binding, and hence for the prevention of coronavirus infections. We also propose that a glycine residue at the receptor binding domain of the spike glycoprotein can have a critical role in permitting bat variants of the coronaviruses to infect human cells. '},\n"," {'paper_id': 'b463aa228141ea5c920755e57bf9c49a55adf141',\n","  'section': 'Abstract',\n","  'text': 'Importance: To predict the diagnosed COVID-19 patients and the trend of the epidemic in China. It may give the public some scientific information to ease the fear of the epidemic. Objective: In December 2019, pneumonia infected with the novel coronavirus burst in Wuhan, China. We aimed to use a mathematical model to predict number of diagnosed patients in future to ease anxiety on the emergent situation. Design: According to all diagnosis number from WHO website and combining with the transmission mode of infectious diseases, the mathematical model was fitted to predict future trend of outbreak. Setting: Our model was based on the epidemic situation in China, which could provide referential significance for disease prediction in other countries, and provide clues for prevention and intervention of relevant health authorities. '},\n"," {'paper_id': 'b465e0f32b19ffb9def3921db0f63f2d62a2fbe5',\n","  'section': 'Abstract',\n","  'text': 'The virus that causes COVID-19, SARS-CoV-2, has a large RNA genome that encodes numerous proteins that might be targets for antiviral drugs. Some of these proteins, such as the RNA-dependent RNA polymers, helicase and main protease, are well conserved between SARS-CoV-2 and the original SARS virus, but several others are not. This study examines one of the most novel proteins encoded by SARS-CoV-2, a macrodomain of nonstructural protein 3 (nsp3). Although 26% of the amino acids in this SARS-CoV-2 macrodomain differ from those seen in other coronaviruses, the protein retains the ability to bind ADPribose, which is an important characteristic of beta coronaviruses, and potential therapeutic target. author/funder. All rights reserved. No reuse allowed without permission. '},\n"," {'paper_id': 'b47f37acf862906eeae49368e09f6b397194cdd7',\n","  'section': 'Abstract',\n","  'text': 'The Coronavirus disease 19 pandemic has led to widespread concerns about the risk of infection in patients with rheumatic diseases (RD) receiving disease modifying antrheumatic drugs (DMARDs) and other immunosuppressants (IS). A SurveyMonkey® based electronic survey was conducted amongst members of the Indian Rheumatology Association to understand the need for changes in prevailing practices. Of the 861 invitees, 221 responded. In the wake of the pandemic, 47.5% would reduce biological DMARDs (bDMARDs) while only 12.2% would reduce the use of conventional synthetic DMARDs. 64.2% were likely to defer change in IS, the reluctance being most with rituximab (58.3%) followed by cyclophosphamide (53.3%), anti-tumor necrosis factor alpha agents (52.4%) and Janus kinase inhibitors (34.39%). Hydroxychloroquine was the preferred choice (81.9%) for the treatment of COVID-19 followed by protease inhibitors (22.1%) and intravenous immunoglobulin (8.1%). Chloroquine was less preferred (19%). More than two-thirds (70.5%) believed that COVID-19 might trigger macrophage activation syndrome. Social distancing (98.1%) and hand hygiene (74.6%) were recommended by majority. 62.8% would avoid touch for clinical examination whenever feasible. Most rheumatologists perceived the need to change treatment of RDs during the COVID-19 pandemic; reduce immunosuppression and defer the usage of rituximab and bDMARDs. '},\n"," {'paper_id': 'b4d9a00412c65725de441654454803bb20ecb931',\n","  'section': 'Abstract',\n","  'text': 'The novel coronavirus disease first identified in 2019 in Wuhan, has become a serious global public health concern. One current issue is the ability to adequately screen for the virus causing COVID-2 (SARS-CoV-2). Here we demonstrate the feasibility of shotgun proteomics as a SARS-CoV-2 screening method, through the detection of viral peptides in proteolytically digested body fluids. Using in silico methods, we generated trypsin-based shotgun proteomics methods optimized for LCMS systems from 5 commercial instrument vendors (Thermo, SCIEX, Waters, Shimadzu, and Agilent). First, we generated protein FASTA files and their protein digest maps. Second, the FASTA files were used to generate spectral libraries based on experimental data. Third, transition lists were derived from the '},\n"," {'paper_id': 'b4edd81e0d2770b0d6e0e3c93c7fffc43e3caa80',\n","  'section': 'Abstract',\n","  'text': 'COVID-19 is having a great impact on public health, mortality and economy worldwide, in spite of the efforts to prevent its epidemy. The SARS-CoV-2 genome is different from that of MERS-CoV and SARS-CoV, although also expected to spread differently according to meteorological conditions. Our main goal is to investigate the role of some meteorological variables on the expansion of this outbreak. In this study, an exponential model relating the number of accumulated confirmed cases and time was considered. The rate of COVID-19 spread, using as criterion the doubling time of the number of confirmed cases, was used as dependent variable in a linear model that took four independent meteorological variables: temperature, humidity, precipitation and wind speed. Only China cases were considered, to control both cultural aspects and containment policies. Confirmed cases and the 4 meteorological variables were gathered between January 23 and March 1 (39 days) for the 31 provinces of Mainland China. Several periods of time were sampled for each province, obtaining more than one value for the rate of disease progression. Two different periods of time were tested, of 12 and 15 days, along with 3 and 5 different starting points in time, randomly chosen. The median value for each meteorological variable was computed, using the same time period; models with > 0.75 were selected. The rate of progression and doubling time were computed and used to fit a linear regression model. Models were evaluated using = 0.05. . CC-BY 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint . https: //doi.org/10.1101//doi.org/10. /2020 Results indicate that the doubling time correlates positively with temperature and inversely with humidity, suggesting that a decrease in the rate of progression of COVID-19 with the arrival of spring and summer in the north hemisphere. A 20ºC increase is expected to delay the doubling time in 1.8 days. Those variables explain 18% of the variation in disease doubling time; the remaining 82% may be related to containment measures, general health policies, population density, transportation or cultural aspects. '},\n"," {'paper_id': 'b4fc44eabb1f640124fff98990c85dc5c9984aee',\n","  'section': 'Abstract',\n","  'text': \"14 Zika virus is a single-stranded, positive-sense RNA virus of the family Flaviviridae, which has 15 recently undergone a rapid expansion among humans in the Western Hemisphere. Here, we report 16 a high-resolution map of ribosomal occupancy of the Zika virus genome during infection of 17 mammalian and insect cells, obtained by ribosome profiling. In contrast to some other flaviviruses 18 such as West Nile, we find no evidence for substantial frameshift-induced ribosomal drop-off 19 during translation of the viral polyprotein, indicating that Zika virus must use alternative 20 mechanisms to downregulate levels of catalytically active viral polymerase. We also show that 21 high levels of ribosome-protected fragments map in-frame to two previously overlooked upstream 22 open reading frames (uORFs) initiating at CUG and UUG codons, with likely consequences for 23 the efficiency of polyprotein expression. Curiously, in African isolates of Zika virus, the two 24 uORFs are fused in-frame into a single uORF. A parallel RNA-Seq analysis reveals the 5' end 25 . CC-BY 4.0 International license is made available under a \"},\n"," {'paper_id': 'b5161b031c7f720562e94735a018d1c3c8be3ae5',\n","  'section': 'Abstract',\n","  'text': 'During outbreaks of deadly emerging pathogens (e.g., Ebola, MERS-CoV) and bioterror threats (e.g., smallpox), actively monitoring potentially infected individuals aims to limit disease transmission and morbidity. Guidance issued by CDC on active monitoring was a cornerstone of its response to the West Africa Ebola outbreak. There are limited data on how to balance the costs and performance of this important public health activity. We present a framework that estimates the risks and costs of specific durations of active monitoring for pathogens of significant public health concern. We analyze data from New York City\\'s Ebola active monitoring program over a 16-month period in 2014-2016. For monitored individuals, we identified unique durations of active monitoring that minimize expected costs for those at \"low (but not zero) risk\" and \"some '},\n"," {'paper_id': 'b51d1c7287ac75790a52752489577d33d522898b',\n","  'section': 'Abstract',\n","  'text': 'Based on results of inter-group difference test, we found that the incidence of diarrhea differed in three recent reports. As 2019-nCoV utilizes the same cell entry receptor ACE2 as severe acute respiratory syndrome coronavirus (SARS-CoV) and ACE2 tightly controls intestinal inflammation, to trace the route of infection mediated by 2019-nCoV, we used the single-cell RNA sequencing data for analysis. We found that the ACE2 mRNA was highly expressed in the healthy human small intestine rather than the lung. Besides, single-cell RNA sequencing data showed that ACE2 was significantly elevated in the proximal and distal enterocytes, where the small intestinal epithelium is exposed to the foreign pathogen. Thus, we suspect that ACE2-expressing small intestinal epithelium cells might be vulnerable to 2019-nCoV infection when people eat infected wild animals and diarrhea may serve as an indicator for infection, suggesting that clinicians should pay more attention to patients with diarrhea during the outbreak of pneumonia. of exported infection cases have been identified in Japan, Korea, the United States, Canada, France, etc. On January 30, 2020, the WHO has confirmed 7,818 cases worldwide, and 7,736 cases were reported in China. According to clinical observation, the most common symptoms were fever, cough, and myalgia or fatigue. Less common symptoms included sputum production, headache, hemoptysis, and diarrhea. Based on three recent publications 1-3 , we compared the incidence of these common symptoms and found that the incidence of diarrhea significantly differed in these three reports. As some patients suffering from diarrhea will first seek help from the gastroenterologist, incorrect estimation of diarrhea may increase the risk of 2019-nCoV infection in the gastroenterology department. Therefore, a comprehensive understanding of the clinical features of pneumonia patients will further improve hospital infection control and reduce healthcare-associated infection. By using the next-generation sequencing technology, scientists revealed that the sequence and genome structure of 2019-nCoV is highly homologous to that of SARS-CoV 4,5 . It has been reported that Angiotensin-converting enzyme 2 (ACE2) serves as a receptor-binding domain complexed with receptor. Science 2005;309:1864-8. 7. Wan Y, Shang J, Graham R, Baric RS, Li F. Receptor recognition by novel coronavirus from Wuhan: An analysis based on decade-long structural studies of SARS. Journal of virology 2020. '},\n"," {'paper_id': 'b52e0f732cefa36aae4d45ebc13208fba190b5af',\n","  'section': 'Abstract',\n","  'text': \"In 2014, A novel tick-borne virus of the genus Flavivirus was first reported from the Mogiana region in Brazil. This virus was named the Mogiana tick virus (MGTV). Later, MGTV was also named as Jingmen tick virus (JMTV), Kindia tick virus (KDTV), Guangxi tick virus (GXTV) etc. In the present study, we used small RNA sequencing (sRNA-seq) to detect viruses in ticks and detected MGTV in Amblyomma testudinarium ticks, which had been captured in Yunnan province of China in the year of 2016. The fulllength genome sequence of a new MGTV strain Yunnan2016 (GenBank: MT080097, MT080098, MT080099 and MT080100) was obtained and recommended to be included into the NCBI RefSeq database for the future studies of MGTV. Our phylogenetic analyses showed that viruses named MGTV, JMTV, KDTV and GXTV are monophyletic: the MGTV group (lineage) of viruses. We show, for the first time, that 5′ and 3′ sRNAs can be used to obtain full-length sequences of the 5' and 3' ends of, but not limited to genome sequences of RNA viruses. And we proved the feasibility of using the sRNA-seq based method for the detection of viruses in a sample containing miniscule RNA. \"},\n"," {'paper_id': 'b579c3547bba2a33057373d57b7c05f37dd4cfc3',\n","  'section': 'Abstract',\n","  'text': 'BACKGROUND: In December 2019, a novel coronavirus (SARS-CoV-2)-infected pneumonia (COVID-19) occurred in Wuhan, China. Travel-associated cases have also been reported in other countries. The number of cases has increased rapidly but laboratory diagnosis is limited. METHODS: We collect two groups of cases diagnosed with COVID-19 for experiments. One group collected 63 samples for Enzyme-linked immunosorbent assay (ELISA) IgG and IgM antibodies. The other group collected 91 plasma samples for colloidal gold-immunochromatographic assay (GICA). RESULTS: The sensitivity of the combined ELISA IgM and ELISA IgG detection was 55/63 ( 87.3%), The sensitivity of the combined GICA IgM and GICA IgG detection was 75/91 ( 82.4%), Both methods are negative for healthy controls, specificity of 100% .There is no significant difference between the sensitivity of between ELISA and GICA (IgM+ IgG). CONCLUSIONS: ELISA and GICA for specific IgM and IgG antibodies are conventional serological assays, they are simple, fast, and safe, the results can be used for clinical reference, and the huge clinical diagnosis and treatment pressure can be .The disease has rapidly spread from Wuhan to other areas. By February 27, 2020, there were 78630 confirmed cases reported in China, and tragically we have now surpassed 2747 people in China have lost their lives because of this virus. Most of the cases and most of the deaths are in Hubei province, Wuhan. Outside China, there are 3545 cases in 40 countries, and 35 death. The type of pneumonia caused by the 2019 All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '},\n"," {'paper_id': 'b5b029e65b963ae092dd88b21911e74b0cf557cc',\n","  'section': 'Abstract',\n","  'text': 'Antibiotics are widely used to treat infections in humans. However, the impact of antibiotic 27 use on host cells is understudied. We have identified a novel antiviral effect of commonly 28 used aminoglycoside antibiotics. We show that mucosal application of aminoglycosides 29 increased host resistance to a broad range of viral infections including herpes simplex 30 viruses, influenza A virus and Zika virus. Aminoglycoside treatment also reduced viral 31 replication in primary human cells. This antiviral activity was independent of the microbiota 32 as aminoglycoside treatment protected germ-free mice. Microarray analysis uncovered a 33 marked upregulation of transcripts for interferon-stimulated genes (ISGs) following 34 aminoglycoside application. ISG induction was mediated by TLR3, and required TIR-35 domain-containing adapter-inducing interferon-β (TRIF), signaling adaptor, and interferon 36 regulatory factors 3 (IRF3) and IRF7, transcription factors that promote ISG expression. 37 XCR1+ dendritic cells, which uniquely express TLR3, were recruited to the vaginal mucosa 38 upon aminoglycoside treatment and were required for ISG induction. These results 39 highlight an unexpected ability of aminoglycoside antibiotics to confer broad antiviral 40 resistance in vivo. '},\n"," {'paper_id': 'b5be10942198c0a79dff8aea2253b77ae2401882',\n","  'section': 'Abstract',\n","  'text': 'Background: Severe patients with 2019 novel coronavirus (2019-nCoV) pneumonia progressed rapidly to acute respiratory failure. We aimed to evaluate the definite efficacy and safety of corticosteroid in the treatment of severe COVID-19 pneumonia. Methods: Forty-six hospitalized patients with severe COVID-19 pneumonia hospitalized at Wuhan Union Hospital from January 20 to February 25, 2020, were retrospectively reviewed. The patients were divided into two groups based on whether they received corticosteroid treatment. The clinical symptoms and chest computed tomography(CT) results were compared. : A total of 26 patients received intravenous administration of methylprednisolone with a dosage of 1-2mg/kg/d for 5-7 days, while the remaining patients not. There was no significant difference in age, sex, comorbidities, clinical or laboratory parameters between the two groups on admission. The average number of days for body temperature back to the normal range was significantly shorter in patients with administration of methylprednisolone when compared to those without administration of methylprednisolone (2.06±0.28 vs. 5.29±0.70, P=0.010). The patients with administration of methylprednisolone had a faster improvement of SpO2, while patients without administration of methylprednisolone had a significantly longer interval of using supplemental oxygen therapy (8.2days[IQR 7.0-10.3] vs. 13.5days(IQR 10.3-16); P<0.001). In terms of chest CT, the absorption degree of the focus was significantly better in patients with administration of methylprednisolone. Our data indicate that in patients with severe COVID-19 pneumonia, early, lowdose and short-term application of corticosteroid was associated with a faster improvement of clinical symptoms and absorption of lung focus. . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . '},\n"," {'paper_id': 'b5e1d36298a78690140d5ccb9673dbd48a0ddcd3',\n","  'section': 'Abstract',\n","  'text': \"The genus Torovirus (subfamily Torovirinae, family Coronaviridae, order Nidovirales) 12 encompasses a range of species that infect domestic ungulates including cattle, 13 sheep, goats, pigs and horses, causing an acute self-limiting gastroenteritis. Using the 14 prototype species equine torovirus (EToV) we performed parallel RNA sequencing 15 (RNA-seq) and ribosome profiling (Ribo-seq) to analyse the relative expression levels 16 of the known torovirus proteins and transcripts, chimaeric sequences produced via 17 discontinuous RNA synthesis (a characteristic of the nidovirus replication cycle) and 18 changes in host transcription and translation as a result of EToV infection. RNA 19 sequencing confirmed that EToV utilises a unique combination of discontinuous and 20 non-discontinuous RNA synthesis to produce its subgenomic RNAs; indeed, we 21 identified transcripts arising from both mechanisms that would result in sgRNAs 22 encoding the nucleocapsid. Our ribosome profiling analysis revealed that ribosomes 23 efficiently translate two novel CUG-initiated ORFs, located within the so-called 5' 24 UTR. We have termed the resulting proteins U1 and U2. Comparative genomic 25 analysis confirmed that these ORFs are conserved across all available torovirus 26 . CC-BY 4.0 \"},\n"," {'paper_id': 'b62add31f59fb244e0721a204bdec76888be7e16',\n","  'section': 'Abstract',\n","  'text': 'The outbreak of the novel coronavirus disease COVID-19, caused by the SARS-CoV-2 virus has spread rapidly around the globe during the past 3 months. As the virus infected cases and mortality rate of this disease is increasing exponentially, scientists and researchers all over the world are relentlessly working to understand this new virus along with possible treatment regimens by discovering active therapeutic agents and vaccines. So, there is an urgent requirement of new and effective medications that can treat the disease caused by SARS-CoV-2. The research includes the study of drugs that are already available in the market and being used for other diseases to accelerate clinical recovery, in other words repurposing of existing drugs. The vast complexity in drug design and protocols regarding clinical trials often prohibit developing various new drug combinations for this epidemic disease in a limited time. Recently, remarkable improvements in computational power coupled with advancements in Machine Learning (ML) technology have been utilized to revolutionize the drug development process. Consequently, a detailed study using ML for the repurposing of therapeutic agents is urgently required. Here, we report the ML model based on the Naïve Bayes algorithm, which has an accuracy of around 73% to predict the drugs that could be used for the treatment of COVID-19. Our study predicts around ten FDA approved commercial drugs that can be used for repurposing. Among all, we suggest that the antiretroviral drug Saquinavir (DrugBank ID -DB01232) would probably be one of the most effective drugs based on the selected criterions. Our study can help clinical scientists in being more selective in identifying and testing the therapeutic agents for COVID-19 treatment. The ML based approach for drug discovery as reported here can be a futuristic smart drug designing strategy for community applications. '},\n"," {'paper_id': 'b696d208705fcb1925693c5f0d118733bb557ea6',\n","  'section': 'Abstract',\n","  'text': 'The novel coronavirus 2019-nCoV has caused major outbreaks in many parts of the world. A better understanding of the pathophysiology of COVID-19 is urgently needed. Clinically, it is important to identify who may be susceptible to infection and identify treatments for the disease. There is good evidence that ACE2 is a receptor for 2019-nCoV, and studies also suggested that high expression of ACE2 may increase susceptibility to infection. Here we conducted a phenome-wide Mendelian randomization (MR) study to prioritize diseases/traits and blood proteins that may be causally linked to ACE2 expression in the lung. Expression data was based on GTEx. We also explored drug candidates whose targets overlapped with the top-ranked proteins in MR analysis, as these drugs could potentially alter ACE2 expression and may be clinically relevant. Notably, MR is much less vulnerable to confounding and reverse causality compared to observational studies. The most consistent finding was a tentative causal association between diabetes-related traits and increased ACE2 expression. Based on one of the largest GWAS on type II diabetes (T2DM) to date (N=898,130), we found that T2DM is causally linked to raised ACE2 expression (beta=0.1835, 95% CI 0.0853-0.2817; p=2.49E-4; GSMR method). Significant associations (at nominal level; p<0.05) was also observed across multiple datasets, with different analytic methods, and for both type I and II diabetes. Other diseases/traits having nominal significant associations with increased ACE2 included inflammatory bowel disease, (ER+) breast and lung cancers, asthma, smoking and elevated ALT, among others. We also uncovered a number of plasma/serum proteins potentially linked to altered ACE2 expression, and the top enriched pathways included cytokine-cytokine-receptor interaction, VEGF signaling, JAK-STAT signaling etc. We also explored drugs that target some of the top-ranked proteins in the MR analysis. '},\n"," {'paper_id': 'b6d2052bb9aef4f3b1d21e86ca164cd43d020857',\n","  'section': 'Abstract',\n","  'text': \"The human microbiome, described as an accessory organ because of the crucial functions it 24 provides, is composed of species that are uniquely found in humans 1,2 . Yet, surprisingly little is known 25 about the impact of routine interpersonal contacts in shaping microbiome composition. In a relatively 26 'closed' cohort of 287 people from the Fiji Islands, where common barriers to bacterial transmission are 27 absent, we examine putative bacterial transmission in individuals' gut and oral microbiomes using strain-28 level data from both core SNPs and flexible genomic regions. We find a weak signal of transmission, 29 defined by the inferred sharing of genotypes, across many organisms that, in aggregate, reveals strong 30 transmission patterns, most notably within households and between spouses. We find that women harbor 31 strains more closely related to those harbored by their familial and social contacts than men; and that 32 transmission patterns of oral-and gut-associated microbiota need not be the same. Using strain-level data 33 alone, we are able to confidently predict a subset of spouses, highlighting the role of shared 34 susceptibilities, behaviors or social interactions that distinguish specific links in the social network. \"},\n"," {'paper_id': 'b6e92eda6c26b456165422d4e5552b81754c4a82',\n","  'section': 'Abstract',\n","  'text': '12 '},\n"," ...]"]},"metadata":{"tags":[]},"execution_count":31}]},{"cell_type":"code","metadata":{"id":"t4SWjDBBFZaS","colab_type":"code","outputId":"aceaa6e7-0e6b-4cf2-99ee-5fabe098b583","executionInfo":{"status":"ok","timestamp":1588770226410,"user_tz":-120,"elapsed":682,"user":{"displayName":"Marko Cvitković","photoUrl":"","userId":"14016951488981011334"}},"colab":{"base_uri":"https://localhost:8080/","height":34}},"source":["len(abstracts)"],"execution_count":0,"outputs":[{"output_type":"execute_result","data":{"text/plain":["29306"]},"metadata":{"tags":[]},"execution_count":32}]},{"cell_type":"code","metadata":{"id":"6mr2eCWAFbwd","colab_type":"code","colab":{}},"source":["paper_ids = [abstract['paper_id'] for abstract in abstracts]"],"execution_count":0,"outputs":[]},{"cell_type":"code","metadata":{"id":"12VqiV5QF-1q","colab_type":"code","outputId":"67b48bdf-973e-4b0e-9d48-0b4cdb35a3dc","executionInfo":{"status":"ok","timestamp":1588771574186,"user_tz":-120,"elapsed":808922,"user":{"displayName":"Marko Cvitković","photoUrl":"","userId":"14016951488981011334"}},"colab":{"base_uri":"https://localhost:8080/","height":34}},"source":["query_engine = QueryEngine_BERT()\n","query_engine.fit(abstracts, paper_ids)\n","    \n","query_engine.save('./', 'abstracts_query_engine')"],"execution_count":0,"outputs":[{"output_type":"stream","text":["Writing object to ./abstracts_query_engine.dat\n"],"name":"stdout"}]},{"cell_type":"code","metadata":{"id":"qv0I6BNPKzTu","colab_type":"code","colab":{}},"source":["query_engine = QueryEngine_BERT.load('abstracts_query_engine.dat')"],"execution_count":0,"outputs":[]},{"cell_type":"code","metadata":{"id":"TL0wmR6lMBDX","colab_type":"code","colab":{}},"source":["pd.set_option(\"display.max_rows\", None, \"display.max_columns\", None)\n","pd.set_option(\"display.max_colwidth\",None)"],"execution_count":0,"outputs":[]},{"cell_type":"code","metadata":{"id":"_JiTyaFqLS3L","colab_type":"code","outputId":"e4c1f885-e8a5-4a49-a3bb-da5f6c9a539a","executionInfo":{"status":"ok","timestamp":1588773501017,"user_tz":-120,"elapsed":2928,"user":{"displayName":"Marko Cvitković","photoUrl":"","userId":"14016951488981011334"}},"colab":{"base_uri":"https://localhost:8080/","height":771}},"source":["query_engine.run_query('Physical science of the coronavirus',10)[['text','sim']].style.set_properties(**{'font-size': '7pt'})"],"execution_count":0,"outputs":[{"output_type":"execute_result","data":{"text/html":["<style  type=\"text/css\" >\n","    #T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row0_col0 {\n","            font-size:  7pt;\n","        }    #T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row0_col1 {\n","            font-size:  7pt;\n","        }    #T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row1_col0 {\n","            font-size:  7pt;\n","        }    #T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row1_col1 {\n","            font-size:  7pt;\n","        }    #T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row2_col0 {\n","            font-size:  7pt;\n","        }    #T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row2_col1 {\n","            font-size:  7pt;\n","        }    #T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row3_col0 {\n","            font-size:  7pt;\n","        }    #T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row3_col1 {\n","            font-size:  7pt;\n","        }    #T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row4_col0 {\n","            font-size:  7pt;\n","        }    #T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row4_col1 {\n","            font-size:  7pt;\n","        }    #T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row5_col0 {\n","            font-size:  7pt;\n","        }    #T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row5_col1 {\n","            font-size:  7pt;\n","        }    #T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row6_col0 {\n","            font-size:  7pt;\n","        }    #T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row6_col1 {\n","            font-size:  7pt;\n","        }    #T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row7_col0 {\n","            font-size:  7pt;\n","        }    #T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row7_col1 {\n","            font-size:  7pt;\n","        }    #T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row8_col0 {\n","            font-size:  7pt;\n","        }    #T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row8_col1 {\n","            font-size:  7pt;\n","        }    #T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row9_col0 {\n","            font-size:  7pt;\n","        }    #T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row9_col1 {\n","            font-size:  7pt;\n","        }</style><table id=\"T_a5566540_8fa1_11ea_a1d6_0242ac1c0002\" ><thead>    <tr>        <th class=\"blank level0\" ></th>        <th class=\"col_heading level0 col0\" >text</th>        <th class=\"col_heading level0 col1\" >sim</th>    </tr></thead><tbody>\n","                <tr>\n","                        <th id=\"T_a5566540_8fa1_11ea_a1d6_0242ac1c0002level0_row0\" class=\"row_heading level0 row0\" >25204</th>\n","                        <td id=\"T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row0_col0\" class=\"data row0 col0\" >Coronaviruses (CoVs), enveloped positive-sense RNA viruses, are characterized by club-like spikes that project from their surface, an unusually large RNA genome, and a unique replication strategy. Coronaviruses cause a variety of diseases in mammals and birds ranging from enteritis in cows and pigs and upper respiratory disease in chickens to potentially lethal human respiratory infections. Here we provide a brief introduction to coronaviruses discussing their replication and pathogenicity, and current prevention and treatment strategies. We also discuss the outbreaks of the highly pathogenic Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) and the recently identifi ed Middle Eastern Respiratory Syndrome Coronavirus (MERS-CoV). Coronavirus virions are spherical with diameters of approximately 125 nm as depicted in recent studies by cryo-electron tomography and cryo-electron microscopy [ 2 , 3 ]. The most prominent feature of coronaviruses is the club-shaped spike projections emanating from the surface of the virion. These spikes are a defi ning feature of the virion and give them the appearance of a solar corona, prompting the name, coronaviruses. Within the envelope of the virion is the nucleocapsid. Coronaviruses have helically symmetrical nucleocapsids, which is uncommon among positive-sense RNA viruses, but far more common for negative-sense RNA viruses. </td>\n","                        <td id=\"T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row0_col1\" class=\"data row0 col1\" >0.631149</td>\n","            </tr>\n","            <tr>\n","                        <th id=\"T_a5566540_8fa1_11ea_a1d6_0242ac1c0002level0_row1\" class=\"row_heading level0 row1\" >12368</th>\n","                        <td id=\"T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row1_col0\" class=\"data row1 col0\" >Coronaviruses are etiologic agents of respiratory and enteric diseases in humans and in animals. In this study, a one-step real-time reverse transcriptionpolymerase chain reaction (RT-PCR) assay based on SYBR Green chemistry and degenerate primers was developed for the generic detection of coronaviruses. The primers, designed in the open reading frame 1b, enabled the detection of 32 animal coronaviruses including strains of canine coronavirus, feline coronavirus, transmissible gastroenteritis virus (TGEV), bovine coronavirus (BCoV), murine hepatitis virus (MHV) and infectious bronchitis virus (IBV). A specific amplification was also observed with the human coronaviruses (HCoV) HCoV-NL63, HCoV-OC43, HCoV-229E and severe acute respiratory syndrome coronavirus (SARS-CoV). The real-time RT-PCR detected down to 10 cRNA copies from TGEV, BCoV, SARS-CoV and IBV. In addition, the assay exhibited a high sensitivity and specificity on clinical samples from different animal species. The developed assay represents a potential tool for laboratory diagnostics and for detecting still uncharacterized coronaviruses. </td>\n","                        <td id=\"T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row1_col1\" class=\"data row1 col1\" >0.611510</td>\n","            </tr>\n","            <tr>\n","                        <th id=\"T_a5566540_8fa1_11ea_a1d6_0242ac1c0002level0_row2\" class=\"row_heading level0 row2\" >4371</th>\n","                        <td id=\"T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row2_col0\" class=\"data row2 col0\" >In 2012, a novel coronavirus, initially named as human coronavirus EMC (HCoV-EMC) but recently renamed as Middle East respiratory syndrome human coronavirus (MERS-CoV), was identified in patients who suffered severe acute respiratory infection and subsequent renal failure that resulted in death. Ongoing epidemiological investigations together with retrospective studies have found 61 laboratory-confirmed cases of infection with this novel coronavirus, including 34 deaths to date. This novel coronavirus is culturable and two complete genome sequences are now available. Furthermore, molecular detection and indirect immunofluorescence assay have been developed. The present paper summarises the limited recent advances of this novel human coronavirus, including its discovery, genomic characterisation and detection. HCoV-EMC, MERS-CoV, genomic characterisation, molecular detection Geng H Y, Tan W J. A novel human coronavirus: Middle East respiratory syndrome human coronavirus. </td>\n","                        <td id=\"T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row2_col1\" class=\"data row2 col1\" >0.609226</td>\n","            </tr>\n","            <tr>\n","                        <th id=\"T_a5566540_8fa1_11ea_a1d6_0242ac1c0002level0_row3\" class=\"row_heading level0 row3\" >17116</th>\n","                        <td id=\"T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row3_col0\" class=\"data row3 col0\" >Severe acute respiratory syndrome (SARS) was first described during a 2002-2003 global outbreak of severe pneumonia associated with human deaths and person-toperson disease transmission. The etiologic agent was initially identified as a coronavirus by thin-section electron microscopic examination of a virus isolate. Virions were spherical, 78 nm in mean diameter, and composed of a helical nucleocapsid within an envelope with surface projections. We show that infection with the SARS-associated coronavirus resulted in distinct ultrastructural features: double-membrane vesicles, nucleocapsid inclusions, and large granular areas of cytoplasm. These three structures and the coronavirus particles were shown to be positive for viral proteins and RNA by using ultrastructural immunogold and in situ hybridization assays. In addition, ultrastructural examination of a bronchiolar lavage specimen from a SARS patient showed numerous coronavirus-infected cells with features similar to those in infected culture cells. Electron microscopic studies were critical in identifying the etiologic agent of the SARS outbreak and in guiding subsequent laboratory and epidemiologic investigations. </td>\n","                        <td id=\"T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row3_col1\" class=\"data row3 col1\" >0.600576</td>\n","            </tr>\n","            <tr>\n","                        <th id=\"T_a5566540_8fa1_11ea_a1d6_0242ac1c0002level0_row4\" class=\"row_heading level0 row4\" >14400</th>\n","                        <td id=\"T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row4_col0\" class=\"data row4 col0\" >Despite years of research, the precise determinants of coronavirus replication and pathogenesis remain unidentified. What is known of the pathogenesis of the severe acute respiratory syndrome coronavirus (SARS-CoV) is limited, but clinical observations suggest that both viral-induced cytotoxicity and host immune-mediated destruction contribute to the severity of disease. This summary discusses recent advances in coronavirus research that will facilitate the identification of crucial molecular targets for the rational design of SARS therapeutics. When the SARS coronavirus came dramatically to attention as a fatal and potentially pandemic respiratory disease, the scientific response was swift and effective. The etiological agent was identified, the genome was cloned and sequenced, and large strides have been made in understanding key points in the viral life cycle, including recent identification of a functional host cell receptor and solution of the three-dimensional structure of the viral attachment protein. Much of this progress was possible only because a foundation of basic coronavirus biology existed. The Denison laboratory has been at the forefront of the small group of investigators who have dissected the basic features of coronaviruses. Denison has discovered critical mechanisms in the life cycle of the coronaviruses and is a key figure in the effective and urgent application of this knowledge to SARS. </td>\n","                        <td id=\"T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row4_col1\" class=\"data row4 col1\" >0.594483</td>\n","            </tr>\n","            <tr>\n","                        <th id=\"T_a5566540_8fa1_11ea_a1d6_0242ac1c0002level0_row5\" class=\"row_heading level0 row5\" >13216</th>\n","                        <td id=\"T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row5_col0\" class=\"data row5 col0\" >Doughnut-shaped particles, 55-65 nm in diameter, were revealed by electron microscopy in the cisterns of the rough endoplasmic reticulum of cells from an active lesion in autopsied brain tissue from a multiple sclerosis patient. The morphology of the particles closely resembled that of coronaviruses. </td>\n","                        <td id=\"T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row5_col1\" class=\"data row5 col1\" >0.591290</td>\n","            </tr>\n","            <tr>\n","                        <th id=\"T_a5566540_8fa1_11ea_a1d6_0242ac1c0002level0_row6\" class=\"row_heading level0 row6\" >24715</th>\n","                        <td id=\"T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row6_col0\" class=\"data row6 col0\" >Coronaviruses have been closely related with mankind for thousands of years. Communityacquired human coronaviruses have long been recognized to cause common cold. However, zoonotic coronaviruses are now becoming more a global concern with the discovery of highly pathogenic severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS) coronaviruses causing severe respiratory diseases. Infections by these emerging human coronaviruses are characterized by less robust interferon production. Treatment of patients with recombinant interferon regimen promises beneficial outcomes, suggesting that compromised interferon expression might contribute at least partially to the severity of disease. The mechanisms by which coronaviruses evade host innate antiviral response are under intense investigations. This review focuses on the fierce arms race between host innate antiviral immunity and emerging human coronaviruses. Particularly, the host pathogen recognition receptors and the signal transduction pathways to mount an effective antiviral response against SARS and MERS coronavirus infection are discussed. On the other hand, the counter-measures evolved by SARS and MERS coronaviruses to circumvent host defense are also dissected. With a better understanding of the dynamic interaction between host and coronaviruses, it is hoped that insights on the pathogenesis of newly-identified highly pathogenic human coronaviruses and new strategies in antiviral development can be derived. </td>\n","                        <td id=\"T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row6_col1\" class=\"data row6 col1\" >0.585042</td>\n","            </tr>\n","            <tr>\n","                        <th id=\"T_a5566540_8fa1_11ea_a1d6_0242ac1c0002level0_row7\" class=\"row_heading level0 row7\" >24504</th>\n","                        <td id=\"T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row7_col0\" class=\"data row7 col0\" >Development of vaccination strategies for emerging pathogens are particularly challenging because of the sudden nature of the emergence of these viruses and the long process needed for traditional vaccine development. Therefore, there is a need for development of a rapid method of vaccine development that can respond to emerging pathogens in a short time frame. The emergence of severe acute respiratory syndrome coronavirus (SARS-CoV) in 2003 and Middle East respiratory syndrome (MERS-CoV) in late 2012 demonstrate the importance of coronaviruses as emerging pathogens. The spike glycoproteins of coronaviruses reside on the surface of the virion and are responsible for virus entry. The spike glycoprotein is the major immunodominant antigen of coronaviruses and has proven to be an excellent target for vaccine designs that seek to block coronavirus entry and promote antibody targeting of infected cells. Vaccination strategies for coronaviruses have involved live attenuated virus, recombinant viruses, non-replicative virus-like particles expressing coronavirus proteins or DNA plasmids expressing coronavirus genes. None of these strategies has progressed to an approved human coronavirus vaccine in the ten years since SARS-CoV emerged. Here we describe a novel method for generating MERS-CoV and SARS-CoV full-length spike nanoparticles, which in combination with adjuvants are able to produce high titer antibodies in mice. </td>\n","                        <td id=\"T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row7_col1\" class=\"data row7 col1\" >0.581003</td>\n","            </tr>\n","            <tr>\n","                        <th id=\"T_a5566540_8fa1_11ea_a1d6_0242ac1c0002level0_row8\" class=\"row_heading level0 row8\" >23009</th>\n","                        <td id=\"T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row8_col0\" class=\"data row8 col0\" >Human coronaviruses are known to be a common cause of respiratory infections in man. However, the diagnosis of human coronavirus infections is not carried out routinely, primarily because the isolation and propagation of these viruses in tissue culture is difficult and time consuming. The aim of this study was to evaluate the use of recombinant, bacterial expressed proteins in the serodiagnosis of coronavirus infections. Two proteins were examined: the human coronavirus 229E nucleocapsid protein (N), expressed as a fusion protein in the vector pUR and the coronavirus 229E surface glycoprotein (S), expressed as a fusion protein in the vector PROS. The recombinant proteins were used as antigens in Western blot (WB) assays to detect the 229E-specific IgG antibodies and the results were compared with a standard serological method, indirect immunofluorescence. Serum samples of 51 paediatric patients, suffering from acute respiratory illness, and 10 adults, voluntarily infected with human coronavirus, were tested. The semm samples of the adult group had coronavims-specific IgG antibodies in both test systems. In contrast, only 8/51 sera of the paediatric group were positive for coronavirus-specific IgG by both WB and IF and 20/51 sera were positive by WB, but not by IF. The overall incidence of human coronavims infections in the paediatric age group was 55% evaluated by WB analysis and 16% evaluated by IF. This study shows that recombinant human coronavirus 229E proteins are suitable reagents for the epidemiological screening of coronavirus 229E infections. 0166-0934/95/$09.50 0 1995 Elsevier Science B.V. All rights reserved SSDIO166-0934(95)00041-O </td>\n","                        <td id=\"T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row8_col1\" class=\"data row8 col1\" >0.578708</td>\n","            </tr>\n","            <tr>\n","                        <th id=\"T_a5566540_8fa1_11ea_a1d6_0242ac1c0002level0_row9\" class=\"row_heading level0 row9\" >14512</th>\n","                        <td id=\"T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row9_col0\" class=\"data row9 col0\" >Development of vaccination strategies for emerging pathogens are particularly challenging because of the sudden nature of their emergence and the long process needed for traditional vaccine development. Therefore, there is a need for development of a rapid method of vaccine development that can respond to emerging pathogens in a short time frame. The emergence of severe acute respiratory syndrome coronavirus (SARS-CoV) in 2003 and Middle East Respiratory Syndrome Coronavirus (MERS-CoV) in late 2012 demonstrate the importance of coronaviruses as emerging pathogens. The spike glycoproteins of coronaviruses reside on the surface of the virion and are responsible for virus entry. The spike glycoprotein is the major immunodominant antigen of coronaviruses and has proven to be an excellent target for vaccine designs that seek to block coronavirus entry and promote antibody targeting of infected cells. Vaccination strategies for coronaviruses have involved live attenuated virus, recombinant viruses, nonreplicative virus-like particles expressing coronavirus proteins or DNA plasmids expressing coronavirus genes. None of these strategies has progressed to an approved human coronavirus vaccine in the ten years since SARS-CoV emerged. Here we describe a novel method for generating MERS-CoV and SARS-CoV full-length spike nanoparticles, which in combination with adjuvants are able to produce high titer antibodies in mice. </td>\n","                        <td id=\"T_a5566540_8fa1_11ea_a1d6_0242ac1c0002row9_col1\" class=\"data row9 col1\" >0.575677</td>\n","            </tr>\n","    </tbody></table>"],"text/plain":["<pandas.io.formats.style.Styler at 0x7f729e62c9b0>"]},"metadata":{"tags":[]},"execution_count":69}]},{"cell_type":"code","metadata":{"id":"ff-0ZxjOPp2W","colab_type":"code","colab":{}},"source":["\"\"\"\n","TODO: analogno za body_texts\n","\"\"\""],"execution_count":0,"outputs":[]}]}